{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import json\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_json('/Users/s0c02nj/Downloads/BioASQ-67b-15Oct2019/BioASQ-7b/train/Full-Abstract/BioASQ-train-factoid-7b-full-annotated.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'paragraphs': [{'qas': [{'id': '56bc751eac7ad10019000013_001',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 31}]}],\n",
       "   'context': 'Acrokeratosis paraneoplastica (Bazex syndrome): report of a case associated with small cell lung carcinoma and review of the literature. Acrokeratosis paraneoplastic (Bazex syndrome) is a rare, but distinctive paraneoplastic dermatosis characterized by erythematosquamous lesions located at the acral sites and is most commonly associated with carcinomas of the upper aerodigestive tract. We report a 58-year-old female with a history of a pigmented rash on her extremities, thick keratotic plaques on her hands, and brittle nails. Chest imaging revealed a right upper lobe mass that was proven to be small cell lung carcinoma. While Bazex syndrome has been described in the dermatology literature, it is also important for the radiologist to be aware of this entity and its common presentations.'},\n",
       "  {'qas': [{'id': '56bc751eac7ad10019000013_002',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 167}]}],\n",
       "   'context': 'Acrokeratosis paraneoplastica (Bazex syndrome): report of a case associated with small cell lung carcinoma and review of the literature. Acrokeratosis paraneoplastic (Bazex syndrome) is a rare, but distinctive paraneoplastic dermatosis characterized by erythematosquamous lesions located at the acral sites and is most commonly associated with carcinomas of the upper aerodigestive tract. We report a 58-year-old female with a history of a pigmented rash on her extremities, thick keratotic plaques on her hands, and brittle nails. Chest imaging revealed a right upper lobe mass that was proven to be small cell lung carcinoma. While Bazex syndrome has been described in the dermatology literature, it is also important for the radiologist to be aware of this entity and its common presentations.'},\n",
       "  {'qas': [{'id': '56bc751eac7ad10019000013_003',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 0}]}],\n",
       "   'context': 'Bazex syndrome (acrokeratosis paraneoplastica): persistence of cutaneous lesions after successful treatment of an associated oropharyngeal neoplasm. Acrokeratosis paraneoplastica is a rare paraneoplastic syndrome commonly affecting males over 40 years of age. There exists a strong association with squamous cell carcinoma (SCC) of the upper aerodigestive tract or cervical metastatic disease originating from an unknown primary. We report a case associated with SCC of the right tonsil with persistent paraneoplastic cutaneous lesions 2 years after successful treatment of the underlying neoplasm.'},\n",
       "  {'qas': [{'id': '56bc751eac7ad10019000013_004',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 30}]}],\n",
       "   'context': 'Acrokeratosis paraneoplastica Bazex syndrome associated with esophageal squamocellular carcinoma. BACKGROUND: Acrokeratosis paraneoplastica Bazex (APB) is a very rare disease in the group of obligate paraneoplastic dermatoses, associated mostly with squamous cell carcinoma of the upper aerodigestive tract and metastatic cervical lymphadenopathy. The disease is characterized by violaceous erythemosquamous changes on the acral regions. This entity was first reported by Bazex in 1965. About 160 cases have been presented so far. CASE REPORT: We presented a patient with a three-month history of violaceous erythema, edema, erosions and scaling on the acral regions, elbows and knees and severe nail dystrophy. When the diagnosis was established, he did not have any symptom of internal malignancy. Esophagogastroscopy revealed ulcerovegetant lesion of the esophagus, while histology showed squamocellular invasive carcinoma. Surgical tumor removal resulted in significant improvement of skin changes in 15 days. Unfortunately, four months later, extensive skin lesions pointed to metastasis of squamous cell carcinoma. CONCLUSION: Skin changes can precede a few years the first manifestations of neoplasia. The course of the disease in our patient proved that APB is a specific marker of underlying malignancy.'},\n",
       "  {'qas': [{'id': '56bc751eac7ad10019000013_005',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 31}]}],\n",
       "   'context': 'Acrokeratosis paraneoplastica (Bazex syndrome) with oropharyngeal squamous cell carcinoma. A 65-year-old white man presented with all the clinical features of acrokeratosis paraneoplastica of Bazex, characterized by violaceous erythema and scaling of the nose, aural helices, fingers, and toes, with keratoderma and severe nail dystrophy. Examination of the patient for possible associated malignancy disclosed an asymptomatic squamous cell carcinoma at the oropharyngeal region. The skin lesions resolved almost completely following radiation therapy of the neoplasm, but the onychodystrophy persisted. This case report illustrates the importance of early recognition of Bazex syndrome.'},\n",
       "  {'qas': [{'id': '56bc751eac7ad10019000013_006',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 31}]}],\n",
       "   'context': 'Acrokeratosis paraneoplastica: Bazex syndrome. Bazex syndrome, or acrokeratosis paraneoplastica, is a cutaneous paraneoplastic syndrome characterized by psoriasiform lesions associated with, usually, a squamous cell carcinoma of the upper aerodigestive tract. We present a case of Bazex syndrome associated with metastatic cervical squamous cell carcinoma with an unknown primary. The features of the condition are discussed in the light of current knowledge.'},\n",
       "  {'qas': [{'id': '56bc751eac7ad10019000013_007',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 0}]}],\n",
       "   'context': 'Bazex syndrome: acrokeratosis paraneoplastica. The focus of this article is acrokeratosis paraneoplastica, one of two disorders that have acquired the eponym Bazex syndrome. To date, all of the patients reported in the literature have had an underlying neoplasm, most commonly squamous cell carcinoma of the upper aerodigestive tract. In this review of 113 cases of acrokeratosis paraneoplastica (mean age, 61 years; 105 males, 8 females), the psoriasiform lesions preceded the diagnosis of the associated malignancy in 73 (67%) of 109 patients, whereas the cutaneous manifestations followed the diagnosis of the neoplasm in only 16 (15%) of 109; in the remainder, the onset of the skin lesions and the diagnosis of the tumor occurred simultaneously. Therefore, awareness of the cutaneous signs of Bazex syndrome is of obvious importance to dermatologists. Evidence in favor of the paraneoplastic nature of this disease is as follows: in 81 (93%) of 87 patients with adequate clinical descriptions, the skin lesions either improved significantly (or resolved) when the underlying neoplasm was treated or they remained unchanged in the setting of persistent disease. Occasionally, the reappearance of skin lesions has signaled a recurrence of the tumor.'},\n",
       "  {'qas': [{'id': '56bc751eac7ad10019000013_008',\n",
       "     'question': 'Name synonym of Acrokeratosis paraneoplastica.',\n",
       "     'answers': [{'text': 'Bazex syndrome', 'answer_start': 150}]}],\n",
       "   'context': \"Acrokeratosis paraneoplastica (Bazex syndrome) presenting in a patient with metastatic breast carcinoma: possible etiologic role of zinc. BACKGROUND: Bazex syndrome (acrokeratosis paraneoplastica) is a rare paraneoplastic syndrome that usually occurs in males over 40 years old and is particularly associated with squamous cell carcinoma of the upper aerodigestive tract and adenopathy above the diaphragm. OBJECTIVE: The objectives of our article are (1) to describe a unique case of acrokeratosis paraneoplastica and (2) to review the current literature regarding skin findings, commonly associated neoplasms, and treatment options relative to this condition. PATIENT: We describe a 68-year-old female with lobular breast carcinoma, complicated by local and distant recurrences, who presented with a 1-year history of prominent acral skin and nail changes. RESULTS: Our patient's clinical skin findings improved significantly following treatment and partial remission of her underlying malignancy. CONCLUSIONS: Our patient represents one of few females described with this syndrome, which is especially rare in association with lobular breast carcinoma. Further, the patient's presentation is unique as she was discovered to demonstrate laboratory findings consistent with coexistent porphyria cutanea tarda and relative zinc deficiency.\"},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_001',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 73}]}],\n",
       "   'context': 'Orteronel plus prednisone in patients with chemotherapy-naive metastatic castration-resistant prostate cancer (ELM-PC 4): a double-blind, multicentre, phase 3, randomised, placebo-controlled trial. BACKGROUND: Orteronel is an investigational, partially selective inhibitor of CYP 17,20-lyase in the androgen signalling pathway, a validated therapeutic target for metastatic castration-resistant prostate cancer. We assessed orteronel in chemotherapy-naive patients with metastatic castration-resistant prostate cancer. METHODS: In this phase 3, double-blind, placebo-controlled trial, we recruited patients with progressive metastatic castration-resistant prostate cancer and no previous chemotherapy from 324 study centres (ie, hospitals or large urologic or group outpatient offices) in 43 countries. Eligible patients were randomly assigned in a 1:1 ratio to receive either 400 mg orteronel plus 5 mg prednisone twice daily or placebo plus 5 mg prednisone twice daily. Randomisation was done centrally with an interactive voice response system and patients were stratified by region (Europe, North America, and not Europe or North America) and the presence or absence of radiographic disease progression at baseline. The two primary endpoints were radiographic progression-free survival and overall survival, determined in the intention-to-treat population. This trial is registered with ClinicalTrials.gov, number NCT01193244. FINDINGS: From Oct 31, 2010, to June 29, 2012, 2353 patients were assessed for eligibility. Of those, 1560 were randomly assigned to receive either orteronel plus prednisone (n=781) or placebo plus prednisone (n=779). The clinical cutoff date for the final analysis was Jan 15, 2014 (with 611 deaths). Median follow-up for radiographic progression-free survival was 8·4 months (IQR 3·7-16·6). Median radiographic progression-free survival was 13·8 months (95% CI 13·1-14·9) with orteronel plus prednisone and 8·7 months (8·3-10·9) with placebo plus prednisone (hazard ratio [HR] 0·71, 95% CI 0·63-0·80; p<0·0001). After a median follow-up of 20·7 months (IQR 14·2-25·4), median overall survival was 31·4 months (95% CI 28·6-not estimable) with orteronel plus prednisone and 29·5 months (27·0-not estimable) with placebo plus prednisone (HR 0·92, 95% CI 0·79-1·08; p=0·31). The most common grade 3 or worse adverse events were increased lipase (137 [17%] of 784 patients in the orteronel plus prednisone group vs 14 [2%] of 770 patients in the placebo plus prednisone group), increased amylase (77 [10%] vs nine [1%]), fatigue (50 [6%] vs 14 [2%]), and pulmonary embolism (40 [5%] vs 27 [4%]). Serious adverse events were reported in 358 [46%] patients receiving orteronel plus prednisone and in 292 [38%] patients receiving placebo plus prednisone. INTERPRETATION: In chemotherapy-naive patients with metastatic castration-resistant prostate cancer, radiographic progression-free survival was prolonged with orteronel plus prednisone versus placebo plus prednisone. However, no improvement was noted in the other primary endpoint, overall survival. Orteronel plus prednisone was associated with increased toxic effects compared with placebo plus prednisone. On the basis of these and other data, orteronel is not undergoing further development in metastatic castration-resistant prostate cancer. FUNDING: Millennium Pharmaceuticals, Inc, a wholly owned subsidiary of Takeda Pharmaceutical Company Limited.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_002',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 374}]}],\n",
       "   'context': 'Orteronel plus prednisone in patients with chemotherapy-naive metastatic castration-resistant prostate cancer (ELM-PC 4): a double-blind, multicentre, phase 3, randomised, placebo-controlled trial. BACKGROUND: Orteronel is an investigational, partially selective inhibitor of CYP 17,20-lyase in the androgen signalling pathway, a validated therapeutic target for metastatic castration-resistant prostate cancer. We assessed orteronel in chemotherapy-naive patients with metastatic castration-resistant prostate cancer. METHODS: In this phase 3, double-blind, placebo-controlled trial, we recruited patients with progressive metastatic castration-resistant prostate cancer and no previous chemotherapy from 324 study centres (ie, hospitals or large urologic or group outpatient offices) in 43 countries. Eligible patients were randomly assigned in a 1:1 ratio to receive either 400 mg orteronel plus 5 mg prednisone twice daily or placebo plus 5 mg prednisone twice daily. Randomisation was done centrally with an interactive voice response system and patients were stratified by region (Europe, North America, and not Europe or North America) and the presence or absence of radiographic disease progression at baseline. The two primary endpoints were radiographic progression-free survival and overall survival, determined in the intention-to-treat population. This trial is registered with ClinicalTrials.gov, number NCT01193244. FINDINGS: From Oct 31, 2010, to June 29, 2012, 2353 patients were assessed for eligibility. Of those, 1560 were randomly assigned to receive either orteronel plus prednisone (n=781) or placebo plus prednisone (n=779). The clinical cutoff date for the final analysis was Jan 15, 2014 (with 611 deaths). Median follow-up for radiographic progression-free survival was 8·4 months (IQR 3·7-16·6). Median radiographic progression-free survival was 13·8 months (95% CI 13·1-14·9) with orteronel plus prednisone and 8·7 months (8·3-10·9) with placebo plus prednisone (hazard ratio [HR] 0·71, 95% CI 0·63-0·80; p<0·0001). After a median follow-up of 20·7 months (IQR 14·2-25·4), median overall survival was 31·4 months (95% CI 28·6-not estimable) with orteronel plus prednisone and 29·5 months (27·0-not estimable) with placebo plus prednisone (HR 0·92, 95% CI 0·79-1·08; p=0·31). The most common grade 3 or worse adverse events were increased lipase (137 [17%] of 784 patients in the orteronel plus prednisone group vs 14 [2%] of 770 patients in the placebo plus prednisone group), increased amylase (77 [10%] vs nine [1%]), fatigue (50 [6%] vs 14 [2%]), and pulmonary embolism (40 [5%] vs 27 [4%]). Serious adverse events were reported in 358 [46%] patients receiving orteronel plus prednisone and in 292 [38%] patients receiving placebo plus prednisone. INTERPRETATION: In chemotherapy-naive patients with metastatic castration-resistant prostate cancer, radiographic progression-free survival was prolonged with orteronel plus prednisone versus placebo plus prednisone. However, no improvement was noted in the other primary endpoint, overall survival. Orteronel plus prednisone was associated with increased toxic effects compared with placebo plus prednisone. On the basis of these and other data, orteronel is not undergoing further development in metastatic castration-resistant prostate cancer. FUNDING: Millennium Pharmaceuticals, Inc, a wholly owned subsidiary of Takeda Pharmaceutical Company Limited.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_003',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 481}]}],\n",
       "   'context': 'Orteronel plus prednisone in patients with chemotherapy-naive metastatic castration-resistant prostate cancer (ELM-PC 4): a double-blind, multicentre, phase 3, randomised, placebo-controlled trial. BACKGROUND: Orteronel is an investigational, partially selective inhibitor of CYP 17,20-lyase in the androgen signalling pathway, a validated therapeutic target for metastatic castration-resistant prostate cancer. We assessed orteronel in chemotherapy-naive patients with metastatic castration-resistant prostate cancer. METHODS: In this phase 3, double-blind, placebo-controlled trial, we recruited patients with progressive metastatic castration-resistant prostate cancer and no previous chemotherapy from 324 study centres (ie, hospitals or large urologic or group outpatient offices) in 43 countries. Eligible patients were randomly assigned in a 1:1 ratio to receive either 400 mg orteronel plus 5 mg prednisone twice daily or placebo plus 5 mg prednisone twice daily. Randomisation was done centrally with an interactive voice response system and patients were stratified by region (Europe, North America, and not Europe or North America) and the presence or absence of radiographic disease progression at baseline. The two primary endpoints were radiographic progression-free survival and overall survival, determined in the intention-to-treat population. This trial is registered with ClinicalTrials.gov, number NCT01193244. FINDINGS: From Oct 31, 2010, to June 29, 2012, 2353 patients were assessed for eligibility. Of those, 1560 were randomly assigned to receive either orteronel plus prednisone (n=781) or placebo plus prednisone (n=779). The clinical cutoff date for the final analysis was Jan 15, 2014 (with 611 deaths). Median follow-up for radiographic progression-free survival was 8·4 months (IQR 3·7-16·6). Median radiographic progression-free survival was 13·8 months (95% CI 13·1-14·9) with orteronel plus prednisone and 8·7 months (8·3-10·9) with placebo plus prednisone (hazard ratio [HR] 0·71, 95% CI 0·63-0·80; p<0·0001). After a median follow-up of 20·7 months (IQR 14·2-25·4), median overall survival was 31·4 months (95% CI 28·6-not estimable) with orteronel plus prednisone and 29·5 months (27·0-not estimable) with placebo plus prednisone (HR 0·92, 95% CI 0·79-1·08; p=0·31). The most common grade 3 or worse adverse events were increased lipase (137 [17%] of 784 patients in the orteronel plus prednisone group vs 14 [2%] of 770 patients in the placebo plus prednisone group), increased amylase (77 [10%] vs nine [1%]), fatigue (50 [6%] vs 14 [2%]), and pulmonary embolism (40 [5%] vs 27 [4%]). Serious adverse events were reported in 358 [46%] patients receiving orteronel plus prednisone and in 292 [38%] patients receiving placebo plus prednisone. INTERPRETATION: In chemotherapy-naive patients with metastatic castration-resistant prostate cancer, radiographic progression-free survival was prolonged with orteronel plus prednisone versus placebo plus prednisone. However, no improvement was noted in the other primary endpoint, overall survival. Orteronel plus prednisone was associated with increased toxic effects compared with placebo plus prednisone. On the basis of these and other data, orteronel is not undergoing further development in metastatic castration-resistant prostate cancer. FUNDING: Millennium Pharmaceuticals, Inc, a wholly owned subsidiary of Takeda Pharmaceutical Company Limited.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_004',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 2843}]}],\n",
       "   'context': 'Orteronel plus prednisone in patients with chemotherapy-naive metastatic castration-resistant prostate cancer (ELM-PC 4): a double-blind, multicentre, phase 3, randomised, placebo-controlled trial. BACKGROUND: Orteronel is an investigational, partially selective inhibitor of CYP 17,20-lyase in the androgen signalling pathway, a validated therapeutic target for metastatic castration-resistant prostate cancer. We assessed orteronel in chemotherapy-naive patients with metastatic castration-resistant prostate cancer. METHODS: In this phase 3, double-blind, placebo-controlled trial, we recruited patients with progressive metastatic castration-resistant prostate cancer and no previous chemotherapy from 324 study centres (ie, hospitals or large urologic or group outpatient offices) in 43 countries. Eligible patients were randomly assigned in a 1:1 ratio to receive either 400 mg orteronel plus 5 mg prednisone twice daily or placebo plus 5 mg prednisone twice daily. Randomisation was done centrally with an interactive voice response system and patients were stratified by region (Europe, North America, and not Europe or North America) and the presence or absence of radiographic disease progression at baseline. The two primary endpoints were radiographic progression-free survival and overall survival, determined in the intention-to-treat population. This trial is registered with ClinicalTrials.gov, number NCT01193244. FINDINGS: From Oct 31, 2010, to June 29, 2012, 2353 patients were assessed for eligibility. Of those, 1560 were randomly assigned to receive either orteronel plus prednisone (n=781) or placebo plus prednisone (n=779). The clinical cutoff date for the final analysis was Jan 15, 2014 (with 611 deaths). Median follow-up for radiographic progression-free survival was 8·4 months (IQR 3·7-16·6). Median radiographic progression-free survival was 13·8 months (95% CI 13·1-14·9) with orteronel plus prednisone and 8·7 months (8·3-10·9) with placebo plus prednisone (hazard ratio [HR] 0·71, 95% CI 0·63-0·80; p<0·0001). After a median follow-up of 20·7 months (IQR 14·2-25·4), median overall survival was 31·4 months (95% CI 28·6-not estimable) with orteronel plus prednisone and 29·5 months (27·0-not estimable) with placebo plus prednisone (HR 0·92, 95% CI 0·79-1·08; p=0·31). The most common grade 3 or worse adverse events were increased lipase (137 [17%] of 784 patients in the orteronel plus prednisone group vs 14 [2%] of 770 patients in the placebo plus prednisone group), increased amylase (77 [10%] vs nine [1%]), fatigue (50 [6%] vs 14 [2%]), and pulmonary embolism (40 [5%] vs 27 [4%]). Serious adverse events were reported in 358 [46%] patients receiving orteronel plus prednisone and in 292 [38%] patients receiving placebo plus prednisone. INTERPRETATION: In chemotherapy-naive patients with metastatic castration-resistant prostate cancer, radiographic progression-free survival was prolonged with orteronel plus prednisone versus placebo plus prednisone. However, no improvement was noted in the other primary endpoint, overall survival. Orteronel plus prednisone was associated with increased toxic effects compared with placebo plus prednisone. On the basis of these and other data, orteronel is not undergoing further development in metastatic castration-resistant prostate cancer. FUNDING: Millennium Pharmaceuticals, Inc, a wholly owned subsidiary of Takeda Pharmaceutical Company Limited.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_005',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 3289}]}],\n",
       "   'context': 'Orteronel plus prednisone in patients with chemotherapy-naive metastatic castration-resistant prostate cancer (ELM-PC 4): a double-blind, multicentre, phase 3, randomised, placebo-controlled trial. BACKGROUND: Orteronel is an investigational, partially selective inhibitor of CYP 17,20-lyase in the androgen signalling pathway, a validated therapeutic target for metastatic castration-resistant prostate cancer. We assessed orteronel in chemotherapy-naive patients with metastatic castration-resistant prostate cancer. METHODS: In this phase 3, double-blind, placebo-controlled trial, we recruited patients with progressive metastatic castration-resistant prostate cancer and no previous chemotherapy from 324 study centres (ie, hospitals or large urologic or group outpatient offices) in 43 countries. Eligible patients were randomly assigned in a 1:1 ratio to receive either 400 mg orteronel plus 5 mg prednisone twice daily or placebo plus 5 mg prednisone twice daily. Randomisation was done centrally with an interactive voice response system and patients were stratified by region (Europe, North America, and not Europe or North America) and the presence or absence of radiographic disease progression at baseline. The two primary endpoints were radiographic progression-free survival and overall survival, determined in the intention-to-treat population. This trial is registered with ClinicalTrials.gov, number NCT01193244. FINDINGS: From Oct 31, 2010, to June 29, 2012, 2353 patients were assessed for eligibility. Of those, 1560 were randomly assigned to receive either orteronel plus prednisone (n=781) or placebo plus prednisone (n=779). The clinical cutoff date for the final analysis was Jan 15, 2014 (with 611 deaths). Median follow-up for radiographic progression-free survival was 8·4 months (IQR 3·7-16·6). Median radiographic progression-free survival was 13·8 months (95% CI 13·1-14·9) with orteronel plus prednisone and 8·7 months (8·3-10·9) with placebo plus prednisone (hazard ratio [HR] 0·71, 95% CI 0·63-0·80; p<0·0001). After a median follow-up of 20·7 months (IQR 14·2-25·4), median overall survival was 31·4 months (95% CI 28·6-not estimable) with orteronel plus prednisone and 29·5 months (27·0-not estimable) with placebo plus prednisone (HR 0·92, 95% CI 0·79-1·08; p=0·31). The most common grade 3 or worse adverse events were increased lipase (137 [17%] of 784 patients in the orteronel plus prednisone group vs 14 [2%] of 770 patients in the placebo plus prednisone group), increased amylase (77 [10%] vs nine [1%]), fatigue (50 [6%] vs 14 [2%]), and pulmonary embolism (40 [5%] vs 27 [4%]). Serious adverse events were reported in 358 [46%] patients receiving orteronel plus prednisone and in 292 [38%] patients receiving placebo plus prednisone. INTERPRETATION: In chemotherapy-naive patients with metastatic castration-resistant prostate cancer, radiographic progression-free survival was prolonged with orteronel plus prednisone versus placebo plus prednisone. However, no improvement was noted in the other primary endpoint, overall survival. Orteronel plus prednisone was associated with increased toxic effects compared with placebo plus prednisone. On the basis of these and other data, orteronel is not undergoing further development in metastatic castration-resistant prostate cancer. FUNDING: Millennium Pharmaceuticals, Inc, a wholly owned subsidiary of Takeda Pharmaceutical Company Limited.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_006',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 158}]}],\n",
       "   'context': 'Phase III, randomized, double-blind, multicenter trial comparing orteronel (TAK-700) plus prednisone with placebo plus prednisone in patients with metastatic castration-resistant prostate cancer that has progressed during or after docetaxel-based therapy: ELM-PC 5. PURPOSE: Orteronel (TAK-700) is an investigational, nonsteroidal, reversible, selective 17,20-lyase inhibitor. This study examined orteronel in patients with metastatic castration-resistant prostate cancer that progressed after docetaxel therapy. PATIENTS AND METHODS: In our study, 1,099 men were randomly assigned in a 2:1 schedule to receive orteronel 400 mg plus prednisone 5 mg twice daily or placebo plus prednisone 5 mg twice daily, stratified by region (Europe, North America [NA], and non-Europe/NA) and Brief Pain Inventory-Short Form worst pain score. Primary end point was overall survival (OS). Key secondary end points (radiographic progression-free survival [rPFS], > 50% decrease of prostate-specific antigen [PSA50], and pain response at 12 weeks) were to undergo statistical testing only if the primary end point analysis was significant. RESULTS: The study was unblinded after crossing a prespecified OS futility boundary. The median OS was 17.0 months versus 15.2 months with orteronel-prednisone versus placebo-prednisone (hazard ratio [HR], 0.886; 95% CI, 0.739 to 1.062; P = .190). Improved rPFS was observed with orteronel-prednisone (median, 8.3 v 5.7 months; HR, 0.760; 95% CI, 0.653 to 0.885; P < .001). Orteronel-prednisone showed advantages over placebo-prednisone in PSA50 rate (25% v 10%, P < .001) and time to PSA progression (median, 5.5 v 2.9 months, P < .001) but not pain response rate (12% v 9%; P = .128). Adverse events (all grades) were generally more frequent with orteronel-prednisone, including nausea (42% v 26%), vomiting (36% v 17%), fatigue (29% v 23%), and increased amylase (14% v 2%). CONCLUSION: Our study did not meet the primary end point of OS. Longer rPFS and a higher PSA50 rate with orteronel-prednisone indicate antitumor activity.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_007',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 435}]}],\n",
       "   'context': 'Phase III, randomized, double-blind, multicenter trial comparing orteronel (TAK-700) plus prednisone with placebo plus prednisone in patients with metastatic castration-resistant prostate cancer that has progressed during or after docetaxel-based therapy: ELM-PC 5. PURPOSE: Orteronel (TAK-700) is an investigational, nonsteroidal, reversible, selective 17,20-lyase inhibitor. This study examined orteronel in patients with metastatic castration-resistant prostate cancer that progressed after docetaxel therapy. PATIENTS AND METHODS: In our study, 1,099 men were randomly assigned in a 2:1 schedule to receive orteronel 400 mg plus prednisone 5 mg twice daily or placebo plus prednisone 5 mg twice daily, stratified by region (Europe, North America [NA], and non-Europe/NA) and Brief Pain Inventory-Short Form worst pain score. Primary end point was overall survival (OS). Key secondary end points (radiographic progression-free survival [rPFS], > 50% decrease of prostate-specific antigen [PSA50], and pain response at 12 weeks) were to undergo statistical testing only if the primary end point analysis was significant. RESULTS: The study was unblinded after crossing a prespecified OS futility boundary. The median OS was 17.0 months versus 15.2 months with orteronel-prednisone versus placebo-prednisone (hazard ratio [HR], 0.886; 95% CI, 0.739 to 1.062; P = .190). Improved rPFS was observed with orteronel-prednisone (median, 8.3 v 5.7 months; HR, 0.760; 95% CI, 0.653 to 0.885; P < .001). Orteronel-prednisone showed advantages over placebo-prednisone in PSA50 rate (25% v 10%, P < .001) and time to PSA progression (median, 5.5 v 2.9 months, P < .001) but not pain response rate (12% v 9%; P = .128). Adverse events (all grades) were generally more frequent with orteronel-prednisone, including nausea (42% v 26%), vomiting (36% v 17%), fatigue (29% v 23%), and increased amylase (14% v 2%). CONCLUSION: Our study did not meet the primary end point of OS. Longer rPFS and a higher PSA50 rate with orteronel-prednisone indicate antitumor activity.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_008',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 69}]}],\n",
       "   'context': 'A phase 1 multiple-dose study of orteronel in Japanese patients with castration-resistant prostate cancer. PURPOSE: Orteronel (TAK-700) is a non-steroidal, selective, reversible inhibitor of 17,20-lyase. We evaluated the safety, tolerability, pharmacokinetics, pharmacodynamics, and antitumor effect of orteronel with or without prednisolone in Japanese patients with castration-resistant prostate cancer (CRPC). METHODS: We conducted a phase 1 study in men with progressive and chemotherapy-naïve CRPC. Patients received orteronel orally at doses of 200-400 mg twice daily (BID) with or without oral prednisolone (5 mg BID). Dose-limiting toxicity (DLT) was assessed during Cycle 1 (28 days). Patients could continue study treatment until any of criteria for treatment discontinuation were met. Gonadotropin-releasing hormone therapy was continued in patients without prior orchidectomy. RESULTS: Fifteen patients were enrolled and administered at least one dose of orteronel. No DLTs were reported during Cycle 1 in this study. Adverse events (AEs) were reported in all 15 patients. Most common AEs (>30%) were hyperlipasemia (47%), hyperamylasemia (40%), and constipation (33%). Acute pancreatitis (Grades 2 and 3) and pancreatitis (Grade 1) were complicated in three patients during the study. Dose-dependent increase in plasma orteronel concentrations was indicated over the 200-400 mg BID dose range. Prednisolone coadministered did not alter PK of orteronel. Serum testosterone was rapidly suppressed below the lower limit of quantification across all doses. Of 15 subjects, 13 achieved at least a 50% reduction from baseline in prostate-specific antigen. CONCLUSIONS: Orteronel at doses up to 400 mg BID was tolerable in Japanese CRPC patients. The present results support further evaluation of orteronel with or without prednisolone.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_009',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 368}]}],\n",
       "   'context': 'A phase 1 multiple-dose study of orteronel in Japanese patients with castration-resistant prostate cancer. PURPOSE: Orteronel (TAK-700) is a non-steroidal, selective, reversible inhibitor of 17,20-lyase. We evaluated the safety, tolerability, pharmacokinetics, pharmacodynamics, and antitumor effect of orteronel with or without prednisolone in Japanese patients with castration-resistant prostate cancer (CRPC). METHODS: We conducted a phase 1 study in men with progressive and chemotherapy-naïve CRPC. Patients received orteronel orally at doses of 200-400 mg twice daily (BID) with or without oral prednisolone (5 mg BID). Dose-limiting toxicity (DLT) was assessed during Cycle 1 (28 days). Patients could continue study treatment until any of criteria for treatment discontinuation were met. Gonadotropin-releasing hormone therapy was continued in patients without prior orchidectomy. RESULTS: Fifteen patients were enrolled and administered at least one dose of orteronel. No DLTs were reported during Cycle 1 in this study. Adverse events (AEs) were reported in all 15 patients. Most common AEs (>30%) were hyperlipasemia (47%), hyperamylasemia (40%), and constipation (33%). Acute pancreatitis (Grades 2 and 3) and pancreatitis (Grade 1) were complicated in three patients during the study. Dose-dependent increase in plasma orteronel concentrations was indicated over the 200-400 mg BID dose range. Prednisolone coadministered did not alter PK of orteronel. Serum testosterone was rapidly suppressed below the lower limit of quantification across all doses. Of 15 subjects, 13 achieved at least a 50% reduction from baseline in prostate-specific antigen. CONCLUSIONS: Orteronel at doses up to 400 mg BID was tolerable in Japanese CRPC patients. The present results support further evaluation of orteronel with or without prednisolone.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_010',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 341}]}],\n",
       "   'context': \"Assessment of cytochrome P450-mediated drug-drug interaction potential of orteronel and exposure changes in patients with renal impairment using physiologically based pharmacokinetic modeling and simulation. Orteronel is a nonsteroidal, selective inhibitor of 17,20-lyase that was recently in phase 3 clinical development as a treatment for castration-resistant prostate cancer. In humans, the primary clearance route for orteronel is renal excretion. Human liver microsomal studies indicated that orteronel weakly inhibits CYP1A2, 2C8, 2C9 and 2C19, with IC50 values of 17.8, 27.7, 30.8 and 38.8 µm, respectively, whereas orteronel does not inhibit CYP2B6, 2D6 or 3A4/5 (IC50 > 100 µm). Orteronel also does not exhibit time-dependent inhibition of CYP1A2, 2B6, 2C8, 2C9, 2C19, 2D6 or 3A4/5. The results of a static model indicated an [I]/Ki ratio >0.1 for CYP1A2, 2C8, 2C9 and 2C19. Therefore, a physiologically based pharmacokinetic (PBPK) model was developed to assess the potential for drug-drug interactions (DDIs) between orteronel and theophylline, repaglinide, (S)-warfarin and omeprazole, which are sensitive substrates of CYP1A2, 2C8, 2C9 and 2C19, respectively. Simulation of the area under the plasma concentration-time curve (AUC) of these four CYP substrates in the presence and absence of orteronel revealed geometric mean AUC ratios <1.25. Therefore, in accordance with the 2012 US FDA Draft Guidance on DDIs, orteronel can be labeled a 'non-inhibitor' and further clinical DDI evaluation is not required. In PBPK models of moderate and severe renal impairment, the AUC of orteronel was predicted to increase by 52% and 83%, respectively. These results are in agreement with those of a clinical trial in which AUC increases of 38% and 87% were observed in patients with moderate and severe renal impairment, respectively.\"},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_011',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 932}]}],\n",
       "   'context': 'Discovery of orteronel (TAK-700), a naphthylmethylimidazole derivative, as a highly selective 17,20-lyase inhibitor with potential utility in the treatment of prostate cancer. A novel naphthylmethylimidazole derivative 1 and its related compounds were identified as 17,20-lyase inhibitors. Based on the structure-activity relationship around the naphthalene scaffold and the results of a docking study of 1a in the homology model of 17,20-lyase, the 6,7-dihydro-5H-pyrrolo[1,2-c]imidazole derivative (+)-3c was synthesized and identified as a potent and highly selective 17,20-lyase inhibitor. Biological evaluation of (+)-3c at a dose of 1mg/kg in a male monkey model revealed marked reductions in both serum testosterone and dehydroepiandrosterone concentrations. Therefore, (+)-3c (termed orteronel [TAK-700]) was selected as a candidate for clinical evaluation and is currently in phase III clinical trials for the treatment of castration-resistant prostate cancer.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_012',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 481}]}],\n",
       "   'context': 'Orteronel plus prednisone in patients with chemotherapy-naive metastatic castration-resistant prostate cancer (ELM-PC 4): a double-blind, multicentre, phase 3, randomised, placebo-controlled trial. BACKGROUND: Orteronel is an investigational, partially selective inhibitor of CYP 17,20-lyase in the androgen signalling pathway, a validated therapeutic target for metastatic castration-resistant prostate cancer. We assessed orteronel in chemotherapy-naive patients with metastatic castration-resistant prostate cancer. METHODS: In this phase 3, double-blind, placebo-controlled trial, we recruited patients with progressive metastatic castration-resistant prostate cancer and no previous chemotherapy from 324 study centres (ie, hospitals or large urologic or group outpatient offices) in 43 countries. Eligible patients were randomly assigned in a 1:1 ratio to receive either 400 mg orteronel plus 5 mg prednisone twice daily or placebo plus 5 mg prednisone twice daily. Randomisation was done centrally with an interactive voice response system and patients were stratified by region (Europe, North America, and not Europe or North America) and the presence or absence of radiographic disease progression at baseline. The two primary endpoints were radiographic progression-free survival and overall survival, determined in the intention-to-treat population. This trial is registered with ClinicalTrials.gov, number NCT01193244. FINDINGS: From Oct 31, 2010, to June 29, 2012, 2353 patients were assessed for eligibility. Of those, 1560 were randomly assigned to receive either orteronel plus prednisone (n=781) or placebo plus prednisone (n=779). The clinical cutoff date for the final analysis was Jan 15, 2014 (with 611 deaths). Median follow-up for radiographic progression-free survival was 8·4 months (IQR 3·7-16·6). Median radiographic progression-free survival was 13·8 months (95% CI 13·1-14·9) with orteronel plus prednisone and 8·7 months (8·3-10·9) with placebo plus prednisone (hazard ratio [HR] 0·71, 95% CI 0·63-0·80; p<0·0001). After a median follow-up of 20·7 months (IQR 14·2-25·4), median overall survival was 31·4 months (95% CI 28·6-not estimable) with orteronel plus prednisone and 29·5 months (27·0-not estimable) with placebo plus prednisone (HR 0·92, 95% CI 0·79-1·08; p=0·31). The most common grade 3 or worse adverse events were increased lipase (137 [17%] of 784 patients in the orteronel plus prednisone group vs 14 [2%] of 770 patients in the placebo plus prednisone group), increased amylase (77 [10%] vs nine [1%]), fatigue (50 [6%] vs 14 [2%]), and pulmonary embolism (40 [5%] vs 27 [4%]). Serious adverse events were reported in 358 [46%] patients receiving orteronel plus prednisone and in 292 [38%] patients receiving placebo plus prednisone. INTERPRETATION: In chemotherapy-naive patients with metastatic castration-resistant prostate cancer, radiographic progression-free survival was prolonged with orteronel plus prednisone versus placebo plus prednisone. However, no improvement was noted in the other primary endpoint, overall survival. Orteronel plus prednisone was associated with increased toxic effects compared with placebo plus prednisone. On the basis of these and other data, orteronel is not undergoing further development in metastatic castration-resistant prostate cancer. FUNDING: Millennium Pharmaceuticals, Inc, a wholly owned subsidiary of Takeda Pharmaceutical Company Limited.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_013',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 341}]}],\n",
       "   'context': \"Assessment of cytochrome P450-mediated drug-drug interaction potential of orteronel and exposure changes in patients with renal impairment using physiologically based pharmacokinetic modeling and simulation. Orteronel is a nonsteroidal, selective inhibitor of 17,20-lyase that was recently in phase 3 clinical development as a treatment for castration-resistant prostate cancer. In humans, the primary clearance route for orteronel is renal excretion. Human liver microsomal studies indicated that orteronel weakly inhibits CYP1A2, 2C8, 2C9 and 2C19, with IC50 values of 17.8, 27.7, 30.8 and 38.8 µm, respectively, whereas orteronel does not inhibit CYP2B6, 2D6 or 3A4/5 (IC50 > 100 µm). Orteronel also does not exhibit time-dependent inhibition of CYP1A2, 2B6, 2C8, 2C9, 2C19, 2D6 or 3A4/5. The results of a static model indicated an [I]/Ki ratio >0.1 for CYP1A2, 2C8, 2C9 and 2C19. Therefore, a physiologically based pharmacokinetic (PBPK) model was developed to assess the potential for drug-drug interactions (DDIs) between orteronel and theophylline, repaglinide, (S)-warfarin and omeprazole, which are sensitive substrates of CYP1A2, 2C8, 2C9 and 2C19, respectively. Simulation of the area under the plasma concentration-time curve (AUC) of these four CYP substrates in the presence and absence of orteronel revealed geometric mean AUC ratios <1.25. Therefore, in accordance with the 2012 US FDA Draft Guidance on DDIs, orteronel can be labeled a 'non-inhibitor' and further clinical DDI evaluation is not required. In PBPK models of moderate and severe renal impairment, the AUC of orteronel was predicted to increase by 52% and 83%, respectively. These results are in agreement with those of a clinical trial in which AUC increases of 38% and 87% were observed in patients with moderate and severe renal impairment, respectively.\"},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_014',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 111}]}],\n",
       "   'context': 'Phase I/II trial of orteronel (TAK-700)--an investigational 17,20-lyase inhibitor--in patients with metastatic castration-resistant prostate cancer. PURPOSE: The androgen receptor pathway remains active in men with prostate cancer whose disease has progressed following surgical or medical castration. Orteronel (TAK-700) is an investigational, oral, nonsteroidal, selective, reversible inhibitor of 17,20-lyase, a key enzyme in the production of androgenic hormones. EXPERIMENTAL DESIGN: We conducted a phase I/II study in men with progressive, chemotherapy-naïve, metastatic castration-resistant prostate cancer, and serum testosterone <50 ng/dL. In the phase I part, patients received orteronel 100 to 600 mg twice daily or 400 mg twice a day plus prednisone 5 mg twice a day. In phase II, patients received orteronel 300 mg twice a day, 400 mg twice a day plus prednisone, 600 mg twice a day plus prednisone, or 600 mg once a day without prednisone. RESULTS: In phase I (n = 26), no dose-limiting toxicities were observed and 13 of 20 evaluable patients (65%) achieved > 50% prostate-specific antigen (PSA) decline from baseline at 12 weeks. In phase II (n = 97), 45 of 84 evaluable patients (54%) achieved a > 50% decline in PSA and at 12 weeks, substantial mean reductions from baseline in testosterone (-7.5 ng/dL) and dehydroepiandrosterone-sulfate (-45.3 μg/dL) were observed. Unconfirmed partial responses were reported in 10 of 51 evaluable phase II patients (20%). Decreases in circulating tumor cells were documented. Fifty-three percent of phase II patients experienced grade > 3 adverse events irrespective of causality; most common were fatigue, hypokalemia, hyperglycemia, and diarrhea. CONCLUSIONS: 17,20-Lyase inhibition by orteronel was tolerable and results in declines in PSA and testosterone, with evidence of radiographic responses.'},\n",
       "  {'qas': [{'id': '56c1f01def6e394741000045_015',\n",
       "     'question': 'Orteronel was developed for treatment of which cancer?',\n",
       "     'answers': [{'text': 'castration-resistant prostate cancer',\n",
       "       'answer_start': 3289}]}],\n",
       "   'context': 'Orteronel plus prednisone in patients with chemotherapy-naive metastatic castration-resistant prostate cancer (ELM-PC 4): a double-blind, multicentre, phase 3, randomised, placebo-controlled trial. BACKGROUND: Orteronel is an investigational, partially selective inhibitor of CYP 17,20-lyase in the androgen signalling pathway, a validated therapeutic target for metastatic castration-resistant prostate cancer. We assessed orteronel in chemotherapy-naive patients with metastatic castration-resistant prostate cancer. METHODS: In this phase 3, double-blind, placebo-controlled trial, we recruited patients with progressive metastatic castration-resistant prostate cancer and no previous chemotherapy from 324 study centres (ie, hospitals or large urologic or group outpatient offices) in 43 countries. Eligible patients were randomly assigned in a 1:1 ratio to receive either 400 mg orteronel plus 5 mg prednisone twice daily or placebo plus 5 mg prednisone twice daily. Randomisation was done centrally with an interactive voice response system and patients were stratified by region (Europe, North America, and not Europe or North America) and the presence or absence of radiographic disease progression at baseline. The two primary endpoints were radiographic progression-free survival and overall survival, determined in the intention-to-treat population. This trial is registered with ClinicalTrials.gov, number NCT01193244. FINDINGS: From Oct 31, 2010, to June 29, 2012, 2353 patients were assessed for eligibility. Of those, 1560 were randomly assigned to receive either orteronel plus prednisone (n=781) or placebo plus prednisone (n=779). The clinical cutoff date for the final analysis was Jan 15, 2014 (with 611 deaths). Median follow-up for radiographic progression-free survival was 8·4 months (IQR 3·7-16·6). Median radiographic progression-free survival was 13·8 months (95% CI 13·1-14·9) with orteronel plus prednisone and 8·7 months (8·3-10·9) with placebo plus prednisone (hazard ratio [HR] 0·71, 95% CI 0·63-0·80; p<0·0001). After a median follow-up of 20·7 months (IQR 14·2-25·4), median overall survival was 31·4 months (95% CI 28·6-not estimable) with orteronel plus prednisone and 29·5 months (27·0-not estimable) with placebo plus prednisone (HR 0·92, 95% CI 0·79-1·08; p=0·31). The most common grade 3 or worse adverse events were increased lipase (137 [17%] of 784 patients in the orteronel plus prednisone group vs 14 [2%] of 770 patients in the placebo plus prednisone group), increased amylase (77 [10%] vs nine [1%]), fatigue (50 [6%] vs 14 [2%]), and pulmonary embolism (40 [5%] vs 27 [4%]). Serious adverse events were reported in 358 [46%] patients receiving orteronel plus prednisone and in 292 [38%] patients receiving placebo plus prednisone. INTERPRETATION: In chemotherapy-naive patients with metastatic castration-resistant prostate cancer, radiographic progression-free survival was prolonged with orteronel plus prednisone versus placebo plus prednisone. However, no improvement was noted in the other primary endpoint, overall survival. Orteronel plus prednisone was associated with increased toxic effects compared with placebo plus prednisone. On the basis of these and other data, orteronel is not undergoing further development in metastatic castration-resistant prostate cancer. FUNDING: Millennium Pharmaceuticals, Inc, a wholly owned subsidiary of Takeda Pharmaceutical Company Limited.'},\n",
       "  {'qas': [{'id': '56af9f130a360a5e45000015_001',\n",
       "     'question': 'Where is the protein Pannexin1 located?',\n",
       "     'answers': [{'text': 'plasma membrane', 'answer_start': 127}]}],\n",
       "   'context': 'Pannexin2 oligomers localize in the membranes of endosomal vesicles in mammalian cells while Pannexin1 channels traffic to the plasma membrane. Pannexin2 (Panx2) is the largest of three members of the pannexin proteins. Pannexins are topologically related to connexins and innexins, but serve different functional roles than forming gap junctions. We previously showed that pannexins form oligomeric channels but unlike connexins and innexins, they form only single membrane channels. High levels of Panx2 mRNA and protein in the Central Nervous System (CNS) have been documented. Whereas Pannexin1 (Panx1) is fairly ubiquitous and Pannexin3 (Panx3) is found in skin and connective tissue, both are fully glycosylated, traffic to the plasma membrane and have functions correlated with extracellular ATP release. Here, we describe trafficking and subcellular localizations of exogenous Panx2 and Panx1 protein expression in MDCK, HeLa, and HEK 293T cells as well as endogenous Panx1 and Panx2 patterns in the CNS. Panx2 was found in intracellular localizations, was partially N-glycosylated, and localizations were non-overlapping with Panx1. Confocal images of hippocampal sections immunolabeled for the astrocytic protein GFAP, Panx1 and Panx2 demonstrated that the two isoforms, Panx1 and Panx2, localized at different subcellular compartments in both astrocytes and neurons. Using recombinant fusions of Panx2 with appended genetic tags developed for correlated light and electron microscopy and then expressed in different cell lines, we determined that Panx2 is localized in the membrane of intracellular vesicles and not in the endoplasmic reticulum as initially indicated by calnexin colocalization experiments. Dual immunofluorescence imaging with protein markers for specific vesicle compartments showed that Panx2 vesicles are early endosomal in origin. In electron tomographic volumes, cross-sections of these vesicles displayed fine structural details and close proximity to actin filaments. Thus, pannexins expressed at different subcellular compartments likely exert distinct functional roles, particularly in the nervous system.'},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_001',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 170}]}],\n",
       "   'context': \"Wilson's disease: an update. Wilson's disease (WD) is an inborn error of copper metabolism caused by a mutation to the copper-transporting gene ATP7B. The disease has an autosomal recessive mode of inheritance, and is characterized by excessive copper deposition, predominantly in the liver and brain. Diagnosis of the condition depends primarily on clinical features, biochemical parameters and the presence of the Kayser-Fleischer ring, and a new diagnostic scoring system has recently been proposed. Mutations in ATP7B can occur anywhere along the entire 21 exons, which makes the identification of gene defects particularly challenging. Identification of carriers and presymptomatic family members of affected individuals is achieved by polymerase-chain-reaction-based marker analysis. The traditional treatment for WD is based on copper chelation with agents such as D-penicillamine, but use of this drug has been questioned because of reported side effects. The use of agents such as trientine and ammonium tetrathiomolybdate has been advocated, although results of long-term trials are awaited. In selected cases, orthotropic hepatic transplantation can reverse the basic metabolic abnormality in WD and improve both hepatic and neurological symptoms. Studies of the underlying defects in ATP7B and its suspected modifiers ATOX1 and COMMD1 are expected to unravel the disease's genotype-phenotype correlation, and should lead to the design of improved drugs for ameliorating the suffering of patients.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_002',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 218}]}],\n",
       "   'context': \"[Wilson's disease]. Wilson's disease is an inherited disorder leading to accumulation of copper in tissues, mainly in the liver and brain. Genetic defect is in the gene coding ATPase type P (ATP7B). The inheritance is autosomal recessive. Up to now, more then 500 mutations causing Wilson's disease were described. The most frequent mutation in Central Europe is mutation H1069Q. The manifestation of Wilson's disease is usually hepatic or neurologic. Hepatic form is manifested by acute or chronic hepatitis, steatosis or cirrhosis. Neurologic involvement is manifested usually after 20 year of age by motor disturbances (tremor, disturbed speech, problems with writing), which could progress into severe extrapyramidal syndrome with tremor, rigidity, dysartria, dysfagia and muscle contracture. Diagnosis is based on clinical and laboratory examinations (neurologic symptoms, liver disease, low serum ceruloplasmin levels, elevated free copper concentration in serum, high urine copper excretion, and presence of Kayser-Fleischer rings). Confirmation of diagnosis is done by hepatic copper concentration in liver biopsy or by genetic examination. Untreated disease leads to the death of a patient. Treatment is based on chelating agents decreasing the copper content by excretion into urine (D-penicillamine, trientine) or on agents preventing absorption of copper from food (zinc, ammonium-tetrahiomolybdene). Patients with asymptomatic Wilson's disease have to be treated as well. In Czech Republic either penicillamine or zinc are used. Liver transplantation is indicated in patients with fulminant liver failure or decompensated cirrhosis. Screening in families of affected patients (all siblings) is obvious.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_003',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 132}]}],\n",
       "   'context': \"Wilson's disease: update on integrated Chinese and Western medicine. Wilson's disease (WD), or hepatolenticular degeneration, is an autosomal recessive inheritance disorder of copper metabolism caused by ATP7B gene mutation. As WD is an inherited disease of the nervous system that is not curable; early diagnosis with early and life-long treatment leads to better prognoses. Currently, the recommended treatment for WD is integrated Chinese and Western medicine. A number of studies indicate that treatment of integrative medicine can not only enforce the de-copper effect but also improve liver function, intelligence, and other factors. This article reviewed in detail the advantages of WD treated with Chinese and Western medicine together.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_004',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 272}]}],\n",
       "   'context': 'Familial pediatric rapidly progressive extrapyramidal syndrome: is it Hallervorden-Spatz disease? The clinical features of two children of a family with rapidly progressive extrapyramidal-pyramidal-dementia complex have been described. Inheritance seems most likely to be autosomal recessive. Magnetic resonance imaging results of brain were negative. Even so, the authors argued in favor of a diagnosis of Hallervorden-Spatz disease because the cases fulfilled the clinical criteria for diagnosis of this disease. Apart from the negative magnetic resonance findings, the other unusual feature was the early development of levodopa-induced dyskinesia. Few conditions need to be considered in the differential diagnosis of a childhood-onset rapidly progressive extrapyramidal syndrome. Such conditions include Wilson\\'s disease, Hallervorden-Spatz disease (HSD), juvenile form of Huntington\\'s disease, juvenile neuronal ceroid lipofuscinosis, early-onset Machado-Joseph disease neuroacanthocytosis, storage disorders, and variant form of dopa-response dystonias (DRD). Rarer conditions are Leigh\\'s disease, Lafora body disease, and dentato-rubro-pallido-luysian atrophy. HSD is a rare disorder characterized by progressive extrapyramidal dysfunction and dementia. Onset is most commonly in late childhood or early adolescence. The disease can be familial or sporadic. When familial, it is inherited recessively and has been linked to chromosome 20. Recently, a mutation in the pantothenate kinase (PANK2) gene on band 20pl3 has been described in patients with typical HSD. HSD produces typical magnetic resonance imaging (MRI) changes in brain, aiding in antemortem diagnosis. The typical finding is of bilaterally symmetrical hyperintense signal changes in the external segment of globus pallidus, with surrounding hypointensity on T(2)-weighted image. These imaging features are fairly diagnostic and have been termed the \"eye-of-the tiger sign\". The hyperintensity represents pathologic changes, including gliosis, demyelination, neuronal loss, and axonal swelling, and the surrounding hypointensity is caused by loss of signal secondary to iron deposition. Described herein are the clinical aspects of a family with autosomal recessive inheritance with rapidly progressive extrapyramidal-pyramidal-dementia complex but with negative brain MRI results. The diagnosis should be considered a variant form of HSD.'},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_005',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 270}]}],\n",
       "   'context': 'Wilson disease and idiopathic copper toxicosis. The pathogenic agent of both Wilson disease (WD) and non-Indian childhood cirrhosis (which we term idiopathic copper toxicosis, or ICT) is copper accumulating to excess in the liver. Inheritance of a pair of alleles of an autosomal recessive gene on chromosome 13 is necessary and sufficient to cause such copper accumulation in WD; reducing the dietary intake of copper cannot prevent the development of WD. In contrast, the lethal accumulations of copper in children with ICT have been attributed primarily to an increased dietary intake of copper. However, 64 124 child-year exposures of children under the age of 6 y to drinking water containing a copper concentration of approximately 125.9 micromol/L (8 mg/L) produced no deaths from any form of liver disease. Moreover, the ICT of seven infants was attributed primarily to drinking water containing < 110.2 micromol Cu/L (7 mg/L) despite evidence of the presence of a genetic defect in three of the patients, one of whom was exclusively breast-fed. These data suggest that ICT cannot be caused solely by increased dietary intake of copper and occurs only in children with an identified genetic defect.'},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_006',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 121}]}],\n",
       "   'context': \"Wilson's disease: clinical and radiological features. BACKGROUND: Wilson's disease is a treatable movement disorder with autosomal recessive inheritance which is associated with severe morbidity and mortality if not treated early. MATERIAL AND METHODS: The clinical and radiological features of 22 cases of Wilson's disease seen during January 1984 to December 1993 were analysed for clinical presentation and common radiological features. RESULTS: Among all the patients extrapyramidal features were the commonest (19/22 patients), followed closely by impaired higher mental functions (17/22 patients) and cerebellar signs (11/22 patients). In patients with onset of symptoms before 20 years, the common presentations were impaired higher mental functions, speech disturbance, dystonia and choreo-athetosis; whereas in patients with onset after 20 years cerebellar signs were commonest. The commonest CT head abnormality was basal ganglion hypodensity (10 patients) followed by brain stem hypodensity (6 patients). CONCLUSIONS: The clinical and CT scan features are evaluated and compared with reported series. Hypodensities of brain stem earlier reported a rarity, was seen in 6 out of 22 cases.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_007',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 293}]}],\n",
       "   'context': \"Excessive iron storage in a patient with Wilson's disease. We report on an otherwise healthy female, mother of two children, with severe decompensated liver cirrhosis due to an iron overload and Wilson's disease. The patient was considered heterozygote for hemochromatosis on the basis of the autosomal recessive inheritance for hemochromatosis, the frequency of the hemochromatosis gene, and the laboratory parameters defining her iron overload. The case is interesting because of the coincidence of Wilson's disease and excessive iron storage.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_008',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 167}]}],\n",
       "   'context': \"Diagnosis and characterization of presymptomatic patients with Wilson's disease and the use of molecular genetics to aid in the diagnosis. Wilson's disease (WD) is an autosomal recessive disorder of copper accumulation leading to liver and/or brain damage. Although fatal if untreated, the condition can be treated effectively. Autosomal recessive inheritance indicates that siblings of affected patients are at 25% risk of having the disease. If they are diagnosed prior to becoming symptomatic, affected siblings can be kept free of symptoms by prophylactic therapy. In this paper we have examined the utility of copper-related variables, along with other clinical and molecular findings, in identifying those siblings of affected patients who should be further evaluated with a liver biopsy. Data are presented on a series of 13 presymptomatic patients in whom we have made the diagnosis of WD based on liver biopsy findings. Signs of liver disease were present in 12 out of 13 cases. The classic, noninvasive, screening approaches that we evaluated were not adequate to identify all cases of WD in this group of patients. These included positive Kayser-Fleischer (KF) rings, elevated liver serum alanine transferase, elevated urine copper, or elevated plasma nonceruloplasmin copper. We have introduced the use of molecular genetics for screening siblings of affected patients for WD. We show that a probe from the linked retinoblastoma (RB) gene can be very helpful in problem cases. However, at this time, the quantitative determination of liver copper concentration remains as the definitive diagnostic criterion.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_009',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'Autosomal recessive', 'answer_start': 328}]}],\n",
       "   'context': \"Diagnosis and characterization of presymptomatic patients with Wilson's disease and the use of molecular genetics to aid in the diagnosis. Wilson's disease (WD) is an autosomal recessive disorder of copper accumulation leading to liver and/or brain damage. Although fatal if untreated, the condition can be treated effectively. Autosomal recessive inheritance indicates that siblings of affected patients are at 25% risk of having the disease. If they are diagnosed prior to becoming symptomatic, affected siblings can be kept free of symptoms by prophylactic therapy. In this paper we have examined the utility of copper-related variables, along with other clinical and molecular findings, in identifying those siblings of affected patients who should be further evaluated with a liver biopsy. Data are presented on a series of 13 presymptomatic patients in whom we have made the diagnosis of WD based on liver biopsy findings. Signs of liver disease were present in 12 out of 13 cases. The classic, noninvasive, screening approaches that we evaluated were not adequate to identify all cases of WD in this group of patients. These included positive Kayser-Fleischer (KF) rings, elevated liver serum alanine transferase, elevated urine copper, or elevated plasma nonceruloplasmin copper. We have introduced the use of molecular genetics for screening siblings of affected patients for WD. We show that a probe from the linked retinoblastoma (RB) gene can be very helpful in problem cases. However, at this time, the quantitative determination of liver copper concentration remains as the definitive diagnostic criterion.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_010',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 138}]}],\n",
       "   'context': \"[Variations in the composition of breast milk in Wilson's disease]. Wilson's disease is a rare genetic disorder of copper metabolism with autosomal recessive inheritance. It occurs between the 6th and 45th year of life. An early and reliable diagnosis, if possible in the preclinical stage, is the prerequisite for starting therapy in time. By the treatment the quality of life and the expectation of life are raised considerably. If consistent treatment is given, there will be no objections to pregnancy in Wilson's disease. It should be interrupted only in case of marked portal hypertension and in the presence of oesophageal varices. The examination of the breast milk of a patient suffering from Wilson's disease showed a reduction in the trace elements copper and zinc. It may be necessary to think of copper and zinc substitution in children fed with breast milk only.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_011',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 303}]}],\n",
       "   'context': \"Is Wilson's disease caused by a controller gene mutation resulting in perpetuation of the fetal mode of copper metabolism into childhood? Wilson's disease is an inborn error of copper metabolism, characterised by raised liver-copper concentrations and low serum levels of copper and caeruloplasmin. The autosomal recessive mode of inheritance strongly suggests that mutation of a single gene causes the impairment of both caeruloplasmin synthesis and biliary copper excretion. The normal infant is born with the biochemical features of Wilson's disease (very high liver-copper levels and low serum copper and caeruloplasmin). Induction of normal copper metabolism after birth results in a fall in liver-copper concentrations and rise in serum caeruloplasmin. The repression of normal copper metabolism in the fetus and its induction after birth is probably regulated by a controller gene. It is suggested that mutation of a controller rather than a structural gene underlies the pathogenesis of Wilson's disease and that the disease results from failure to switch from the positive copper balance of the fetus to the normal copper balance of the child.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_012',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 412}]}],\n",
       "   'context': \"Heterogeneity of Wilson's disease in Israel. In a survey in Israel of 50 patients with Wilson's disease, it was found that this disease occurred in all ethnic groups. In the Arab patients there was a significantly early age of onset and the disease followed a more severe course than that in the Jewish patients. The overall sex ratio of patients was nearly 1:1, and genetic analysis of 20 families confirmed an autosomal recessive mode of inheritance. The very similar age of onset and type of disease within sibships and the varying ages of onset noted between the Arab and Jewish patients suggest that the disease is genetically heterogeneous.\"},\n",
       "  {'qas': [{'id': '52bf1b0a03868f1b06000009_013',\n",
       "     'question': \"What is the mode of inheritance of Wilson's disease?\",\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 370}]}],\n",
       "   'context': \"[Dermatoglyphics of homo- and heterozygotes for Wilson's disease (hepatolenticular degeneration) (author's transl)]. Dermatoglyphics of 11 patients with Wilson's disease and 16 of their clinically asymptomatic relatives of first degree were investigated; 11 of the latter ones were heterozygous in agreement with the turn over rates of Cu-67, 12 under the assumption of autosomal recessive inheritance. On the finger tips the Mb. Wilson patients showed 52.7% whorls, their heterozygous relatives about 40%; compared with our controls (males 33.16%, females 28.82%, Aue-Hauser, 1970) that means a strong increase of this pattern type. On the palm the high frequency of hypothenar patterns in homo- and heterozygotes for Wilson's disease and of loops with accessory triradius in the 4th interdigitum of the patients with Wilson's disease was striking.\"},\n",
       "  {'qas': [{'id': '52bf19c503868f1b06000001_001',\n",
       "     'question': 'What is the mode of inheritance of Facioscapulohumeral muscular dystrophy (FSHD)?',\n",
       "     'answers': [{'text': 'autosomal dominant', 'answer_start': 345}]}],\n",
       "   'context': 'Digenic inheritance of an SMCHD1 mutation and an FSHD-permissive D4Z4 allele causes facioscapulohumeral muscular dystrophy type 2. Facioscapulohumeral dystrophy (FSHD) is characterized by chromatin relaxation of the D4Z4 macrosatellite array on chromosome 4 and expression of the D4Z4-encoded DUX4 gene in skeletal muscle. The more common form, autosomal dominant FSHD1, is caused by contraction of the D4Z4 array, whereas the genetic determinants and inheritance of D4Z4 array contraction-independent FSHD2 are unclear. Here, we show that mutations in SMCHD1 (encoding structural maintenance of chromosomes flexible hinge domain containing 1) on chromosome 18 reduce SMCHD1 protein levels and segregate with genome-wide D4Z4 CpG hypomethylation in human kindreds. FSHD2 occurs in individuals who inherited both the SMCHD1 mutation and a normal-sized D4Z4 array on a chromosome 4 haplotype permissive for DUX4 expression. Reducing SMCHD1 levels in skeletal muscle results in D4Z4 contraction-independent DUX4 expression. Our study identifies SMCHD1 as an epigenetic modifier of the D4Z4 metastable epiallele and as a causal genetic determinant of FSHD2 and possibly other human diseases subject to epigenetic regulation.'},\n",
       "  {'qas': [{'id': '52bf19c503868f1b06000001_002',\n",
       "     'question': 'What is the mode of inheritance of Facioscapulohumeral muscular dystrophy (FSHD)?',\n",
       "     'answers': [{'text': 'autosomal dominant', 'answer_start': 180}]}],\n",
       "   'context': \"Recommendations for the management of facioscapulohumeral muscular dystrophy in 2011. Facioscapulohumeral muscular dystrophy (FSHD) is a neuromuscular disease, characterized by an autosomal dominant mode of inheritance, facial involvement, and selectivity and asymmetry of muscle involvement. In general, FSHD typically presents before age 20 years. Usually, FSHD muscle involvement starts in the face and then progresses to the shoulder girdle, the humeral muscles and the abdominal muscles, and then the anterolateral compartment of the leg. Disease severity is highly variable and progression is very slow. About 20% of FSHD patients become wheelchair-bound. Lifespan is not shortened. The diagnosis of FSHD is based on a genetic test by which a deletion of 3.3kb DNA repeats (named D4Z4 and mapping to the subtelomeric region of chromosome 4q35) is identified. The progressive pattern of FSHD requires that the severity of symptoms as well as their physical, social and psychological impact be evaluated on a regular basis. A yearly assessment is recommended. Multidisciplinary management of FSHD--consisting of a combination of genetic counselling, functional assessment, an assessment by a physical therapist, prescription of symptomatic therapies and prevention of known complications of this disease--is required. Prescription of physical therapy sessions and orthopedic appliances are to be adapted to the patient's deficiencies and contractures.\"},\n",
       "  {'qas': [{'id': '52bf19c503868f1b06000001_003',\n",
       "     'question': 'What is the mode of inheritance of Facioscapulohumeral muscular dystrophy (FSHD)?',\n",
       "     'answers': [{'text': 'autosomal dominant', 'answer_start': 1405}]}],\n",
       "   'context': 'Genetically determined neuromuscular disorders of some Roma families living in Hungary. The authors discuss the clinical and molecular genetic aspects of genetically determined neuromuscular disorders of some Roma families living in Hungary. Among the autosomal recessively inherited spinal muscular atrophic (SMA) group, 8 Caucasian children had the typical 7-8 exonal deletions of the SMA gene, but only 2 patients belonged to the Roma population. There was no difference in the molecular genetic findings among the Caucasian and the Roma SMA patients. All of them had 7-8 exonal deletions of the SMA gene. We wanted to call attention to the founder mutation of the Roma population in 7 patients suffering from congenital myasthenia (CMS) from 3 Roma families. The 1267G deletion for CMS was detected by molecular genetic method. Clinical onset was pubertal and relatively slow progression of specific and phenotypic features for this founder mutation of acetyl-cholin receptor epsylon gene. In 2 patients (sister and brother) the sarcoglycanopathy 2C type C283Q mutation was proven in one Roma family suffering from limb-girdle muscular dystrophy (LGMD). Two out of the three facioscapular-humeral dystrophy (FSHD) Roma families carried 21.8 kb and 18.5 kb alleles in FSHD A1 gene (D4S139). In one family together with prenatal diagnosis founder mutation in FSHD A1 gene was detected, according to the autosomal dominant (AD) inheritance. In (F2) prenatal diagnosis was carried out, 18.5 kb/18.5 kb homozygosity was proven in the fetus, so the pregnancy was interrupted. In the CMS, LGMD and FSHD Roma patients ancient typical Roma founder mutations were found.'},\n",
       "  {'qas': [{'id': '52bf19c503868f1b06000001_004',\n",
       "     'question': 'What is the mode of inheritance of Facioscapulohumeral muscular dystrophy (FSHD)?',\n",
       "     'answers': [{'text': 'autosomal dominant', 'answer_start': 170}]}],\n",
       "   'context': '[Genetic investigations in facioscapulohumeral muscular dystrophy: a preliminary report]. Facioscapulohumeral muscular dystrophy (FSHD) is a primary muscle disorder with autosomal dominant inheritance. FSHD was mapped to chromosome 4 locus q35, but the gene is not yet known. It is characterised by progressive, often asymmetric, selective muscular weakness and great clinical variability. The aim of the study was to analyze 62 FSHD cases from 44 Polish families in which the diagnosis was confirmed by DNA analyses. FSHD diagnosis was based on the clinical findings and standardized investigations confirming primary muscular involvement (EMG, muscle biopsy). DNA analysis was based on EcoRI/BlnI restriction enzyme digestion followed by hybridization with P13E-11 molecular probe. In our material, we have found a relatively large percentage (41%) of big deletions (EcoRI/BlnI fragment of 10-15 kb [kilo bases]), which in the majority of cases (67%) was present in isolated cases. In 10 families (23%) the phenotype was assessed as severe. These are cases with the onset before the age of 10 and fast progression. \"Middle sized\" deletions (EcoRI/BlnI fragment of 16-29 kb) were prevalent in familial cases and present in 57% of families. \"Small\" deletion was found in one family (EcoRI/BlnI fragment of 30 kb). Somatic mosaicism was confirmed in one case. De novo mutations were shown in 11% of the examined families. The results of this study indicate that the bigger the deletion, the more severe the FSHD course, however there are some exceptions. A similar relationship has been shown by previous research. Molecular analyses are particularly important in atypical and sporadic cases. It is the first genetic presentation of a group of patients\\' with this kind of dystrophy in the Polish population.'},\n",
       "  {'qas': [{'id': '52bf19c503868f1b06000001_005',\n",
       "     'question': 'What is the mode of inheritance of Facioscapulohumeral muscular dystrophy (FSHD)?',\n",
       "     'answers': [{'text': 'autosomal dominant', 'answer_start': 181}]}],\n",
       "   'context': 'The strength and functional performance in patients with facioscapulohumeral muscular dystrophy. Facioscapulohumeral muscular dystrophy (FSHD) is a slowly progressive myopathy with autosomal dominant inheritance remarkable for its early involvement of facial musculature. The purpose of our study was to assess the rate of strength deterioration, functional condition and performance of activity of daily living of patients with FSHD in Taiwan. Twenty patients diagnosed with FSHD were included in this study. Manual muscle testing (MMT) was used to evaluate muscle strength. The Brooke and Vignos scales were used to assess upper and lower extremity function respectively, and the capability of the activity of daily living was measured by Barthel index. The result of the strength testing was characterized by the presence of a progressive asymmetrical muscular weakness in patients with FSHD. The mean muscular strength of the right extremity was weaker than its left counterparts (p < 0.05) and the shoulder muscle group was the weakest. According to the Brooke functional scale, 20% of our patients were graded as 1, 30% as grade 2, and 50% as grade 3. On the Vignos functional scale, 50% of patients fell into grade 1, 10% in grade 2, and 40% in grades 3-5. Vignos scale was significantly correlated with mean muscle strength (p < 0.05). The average value of Barthel index was 97.8 +/- 4.7. The muscle strength decline in this Taiwanese of FSHD population was more severe in shoulder girdle area. The mean muscle strength of the right extremity was weaker than the left. Most of our patients suffered from mild or moderate physical disability. Finding of these Taiwanese FSHD population is similar to those reported elsewhere in the world.'},\n",
       "  {'qas': [{'id': '52bf19c503868f1b06000001_006',\n",
       "     'question': 'What is the mode of inheritance of Facioscapulohumeral muscular dystrophy (FSHD)?',\n",
       "     'answers': [{'text': 'autosomal dominant', 'answer_start': 287}]}],\n",
       "   'context': 'Extension of the clinical range of facioscapulohumeral dystrophy: report of six cases. Consensual diagnostic criteria for facioscapulohumeral dystrophy (FSHD) include onset of the disease in facial or shoulder girdle muscles, facial weakness in more than 50% of affected family members, autosomal dominant inheritance in familial cases, and evidence of myopathic disease in at least one affected member without biopsy features specific to alternative diagnoses. Six patients did not meet most of these criteria but were diagnosed as FSHD by DNA testing, which showed small EcoRI fragments on chromosome 4q. Their clinical signs and symptoms and results of auxiliary investigations are reported. The patients presented with foot extensor, thigh, or calf muscle weakness. None of them had apparent facial weakness, only one complained of weakness in the shoulders, none had a positive family history. Expert physical examination, however, showed a typical facial expression, an abnormal shoulder configuration on lifting the arms, or scapular winging. This raised the suspicion of FSHD, whereupon DNA analysis was done. In conclusion, the clinical expression of FSHD is much broader than indicated by the nomenclature. The possibility to perform DNA tests is likely to greatly expand the clinical range of FSHD.'},\n",
       "  {'qas': [{'id': '5505edac8e1671127b000005_001',\n",
       "     'question': 'What kind of chromatography is HILIC?',\n",
       "     'answers': [{'text': 'Hydrophilic Interaction Chromatography',\n",
       "       'answer_start': 188}]}],\n",
       "   'context': 'Application of hydrophilic interaction chromatography for the analysis of polar contaminants in food and environmental samples. For the analysis of highly hydrophilic and polar compounds, Hydrophilic Interaction Chromatography (HILIC) has been established as a valuable complementary approach to reversed-phase liquid chromatography (RPLC). Moreover, the use of mobile phases with a high percentage of organic solvent in HILIC separation is beneficial for mass spectrometric (MS) detection, because of enhanced ionization which results in an increased sensitivity. In this review, various applications of HILIC are described for a number of environmental and food contaminants together with detailed methodological descriptions and the advantages or drawbacks of HILIC compared to other LC methods are critically discussed. In the first part of the review, an overview is given of the work that has been carried out with HILIC for the analysis of pharmaceuticals and pesticides in environmental samples. HILIC has shown its applicability for polar pharmaceuticals, such as antibiotics, estrogens and their metabolites, drugs of abuse, cytostatics, metformin and contrast agents. In the pesticide group, HILIC chromatography was helpful for polar phenylurea and organophosphorus pesticides. The second part of the review focuses on the analysis of antibiotic residues in food and feed with HILIC, while in the pesticide group, HILIC experiments have been reported for dithiocarbamates and quaternary ammonium compounds. The last chapter gives an overview of the analysis by HILIC of miscellaneous analytes in aquatic and food/feed samples.'},\n",
       "  {'qas': [{'id': '56e2cec751531f7e33000015_001',\n",
       "     'question': 'What is the structural fold of bromodomain proteins?',\n",
       "     'answers': [{'text': 'all-alpha-helical fold', 'answer_start': 899}]}],\n",
       "   'context': 'Structural ramification for acetyl-lysine recognition by the bromodomain of human BRG1 protein, a central ATPase of the SWI/SNF remodeling complex. Bromodomains represent an extensive family of evolutionarily conserved domains that are found in many chromatin-associated proteins such as histone acetyltransferases (HAT) and subunits of ATP-dependent chromatin-remodeling complexes. These domains are associated with acetylated lysine residues that bind both in vivo and in vitro; for example, they bind to the N-acetylated lysines of the histone tail of nucleosomes. In this report, we determined the structure of the bromodomain from human brahma-related gene 1 (BRG1) protein, a subunit of an ATP-dependent switching/sucrose nonfermenting (SWI/SNF) remodeling complex, and have also characterized its in vitro interaction with N-acetylated lysine peptides from histones. In addition to a typical all-alpha-helical fold that was observed in the bromodomains, we observed for the first time a small beta-sheet in the ZA loop region of the BRG1 protein. The BRG1 bromodomain exhibited binding, albeit weak, to acetylated peptides that were derived from histones H3 and H4. We have compared the acetyl-lysine binding sites of BRG1 bromodomain with the yGCN5 (general control of amino acid biosynthesis). By modeling the acetylated-lysine peptide into the BRG1 bromodomain structure, we were able to explain the weak binding of acetylated-lysine peptides to this bromodomain.'},\n",
       "  {'qas': [{'id': '56e2cec751531f7e33000015_002',\n",
       "     'question': 'What is the structural fold of bromodomain proteins?',\n",
       "     'answers': [{'text': 'all-alpha-helical fold', 'answer_start': 899}]}],\n",
       "   'context': 'Structural ramification for acetyl-lysine recognition by the bromodomain of human BRG1 protein, a central ATPase of the SWI/SNF remodeling complex. Bromodomains represent an extensive family of evolutionarily conserved domains that are found in many chromatin-associated proteins such as histone acetyltransferases (HAT) and subunits of ATP-dependent chromatin-remodeling complexes. These domains are associated with acetylated lysine residues that bind both in vivo and in vitro; for example, they bind to the N-acetylated lysines of the histone tail of nucleosomes. In this report, we determined the structure of the bromodomain from human brahma-related gene 1 (BRG1) protein, a subunit of an ATP-dependent switching/sucrose nonfermenting (SWI/SNF) remodeling complex, and have also characterized its in vitro interaction with N-acetylated lysine peptides from histones. In addition to a typical all-alpha-helical fold that was observed in the bromodomains, we observed for the first time a small beta-sheet in the ZA loop region of the BRG1 protein. The BRG1 bromodomain exhibited binding, albeit weak, to acetylated peptides that were derived from histones H3 and H4. We have compared the acetyl-lysine binding sites of BRG1 bromodomain with the yGCN5 (general control of amino acid biosynthesis). By modeling the acetylated-lysine peptide into the BRG1 bromodomain structure, we were able to explain the weak binding of acetylated-lysine peptides to this bromodomain.'},\n",
       "  {'qas': [{'id': '56e2cec751531f7e33000015_003',\n",
       "     'question': 'What is the structural fold of bromodomain proteins?',\n",
       "     'answers': [{'text': 'all-alpha-helical fold', 'answer_start': 899}]}],\n",
       "   'context': 'Structural ramification for acetyl-lysine recognition by the bromodomain of human BRG1 protein, a central ATPase of the SWI/SNF remodeling complex. Bromodomains represent an extensive family of evolutionarily conserved domains that are found in many chromatin-associated proteins such as histone acetyltransferases (HAT) and subunits of ATP-dependent chromatin-remodeling complexes. These domains are associated with acetylated lysine residues that bind both in vivo and in vitro; for example, they bind to the N-acetylated lysines of the histone tail of nucleosomes. In this report, we determined the structure of the bromodomain from human brahma-related gene 1 (BRG1) protein, a subunit of an ATP-dependent switching/sucrose nonfermenting (SWI/SNF) remodeling complex, and have also characterized its in vitro interaction with N-acetylated lysine peptides from histones. In addition to a typical all-alpha-helical fold that was observed in the bromodomains, we observed for the first time a small beta-sheet in the ZA loop region of the BRG1 protein. The BRG1 bromodomain exhibited binding, albeit weak, to acetylated peptides that were derived from histones H3 and H4. We have compared the acetyl-lysine binding sites of BRG1 bromodomain with the yGCN5 (general control of amino acid biosynthesis). By modeling the acetylated-lysine peptide into the BRG1 bromodomain structure, we were able to explain the weak binding of acetylated-lysine peptides to this bromodomain.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_001',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'c-Jun NH2-terminal kinase',\n",
       "       'answer_start': 1080}]}],\n",
       "   'context': 'Estrogen attenuates lipopolysaccharide-induced nitric oxide production in macrophages partially via the nongenomic pathway. Steroid hormones exert genotropic effects through members of the nuclear hormone receptor family. In the present study, we examined the effects of 17β-estradiol (E2) on nitric oxide (NO) production following lipopolysaccharide (LPS) stimulation and investigated the mechanisms in mouse bone marrow-derived macrophages (BMMs). E2 alone did not affect NO production. In contrast, E2 inhibited LPS-induced production of NO in BMMs. Using a cell-impermeable E2 conjugated to BSA (E2-BSA), which has been used to investigate the nongenomic effects of estrogen, we found that the increase in NO production induced by LPS was also attenuated. In addition, the intracellular estrogen receptor blocker, ICI 182780, only partially antagonized the total effects of E2 on LPS-stimulated NO production capacity. E2 also attenuated the LPS activation of p38 mitogen-activated protein kinase (MAPK) but not that of extracellular-regulated protein kinase 1/2 (ERK1/2) and c-Jun NH2-terminal kinase (JNK). This attenuation was not abrogated by ICI 182780. Moreover, the p38 inhibitor, SB 203580, greatly reduced the LPS-induced NO production, and the remaining NO levels were no longer regulated by E2. Additionally, E2-BSA inhibited LPS-mediated changes in p38 MAPK activation to the same extent as E2. Moreover, E2 and E2-BSA inhibited LPS-induced activation of nuclear factor-kappa B (NF-κB) and activator protein 1 (AP-1). This inhibitory effect of E2 was only partially antagonized by ICI 182780. Taken together, these results suggest that E2 has an inhibitory effect on LPS-induced NO production in BMMs through inhibition of p38 MAPK phosphorylation, and blockade of NF-κB and AP-1 activation. These effects are mediated at least in part via a nongenomic pathway.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_002',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 1107}]}],\n",
       "   'context': 'Estrogen attenuates lipopolysaccharide-induced nitric oxide production in macrophages partially via the nongenomic pathway. Steroid hormones exert genotropic effects through members of the nuclear hormone receptor family. In the present study, we examined the effects of 17β-estradiol (E2) on nitric oxide (NO) production following lipopolysaccharide (LPS) stimulation and investigated the mechanisms in mouse bone marrow-derived macrophages (BMMs). E2 alone did not affect NO production. In contrast, E2 inhibited LPS-induced production of NO in BMMs. Using a cell-impermeable E2 conjugated to BSA (E2-BSA), which has been used to investigate the nongenomic effects of estrogen, we found that the increase in NO production induced by LPS was also attenuated. In addition, the intracellular estrogen receptor blocker, ICI 182780, only partially antagonized the total effects of E2 on LPS-stimulated NO production capacity. E2 also attenuated the LPS activation of p38 mitogen-activated protein kinase (MAPK) but not that of extracellular-regulated protein kinase 1/2 (ERK1/2) and c-Jun NH2-terminal kinase (JNK). This attenuation was not abrogated by ICI 182780. Moreover, the p38 inhibitor, SB 203580, greatly reduced the LPS-induced NO production, and the remaining NO levels were no longer regulated by E2. Additionally, E2-BSA inhibited LPS-mediated changes in p38 MAPK activation to the same extent as E2. Moreover, E2 and E2-BSA inhibited LPS-induced activation of nuclear factor-kappa B (NF-κB) and activator protein 1 (AP-1). This inhibitory effect of E2 was only partially antagonized by ICI 182780. Taken together, these results suggest that E2 has an inhibitory effect on LPS-induced NO production in BMMs through inhibition of p38 MAPK phosphorylation, and blockade of NF-κB and AP-1 activation. These effects are mediated at least in part via a nongenomic pathway.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_003',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 301}]}],\n",
       "   'context': '[Immune regulatory effect of citalopram on microglial cells]. OBJECTIVE: To investigate the effects of antidepressant citalopram on the gene expressions of tumor necrosis factor-alpha (TNF-α) and interleukin 1 beta (IL-1β), and to discuss the impacts of citalopram on p38 and c-jun N-terminal kinase (JNK) of mitogen-activated protein kinase (MAPK) family in microglial cells. METHODS: BV2 cells were induced by lipopolysaccharide (LPS) to produce TNF-α and IL-1β. After pretreatment with citalopram (20 μmol/L) for 4 h, the mRNA levels of TNF-α and IL-1β were measured by quantitative real-time PCR (qRT-PCR); after pretreatment for 24 h, the protein levels of TNF-α and IL-1β were analyzed by ELISA; the effects of citalopram on the phosphorylation of p38MAPK and JNK were observed after pretreatment for 30 min. RESULTS: Citalopram significantly inhibited the mRNA and protein expressions of TNF-α and IL-1β, and the phosphorylation of p38MAPK and JNK. CONCLUSION: Citalopram may play the anti-inflammatory role by inhibiting MAPK pathway in microglial cells.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_004',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 1114}]}],\n",
       "   'context': 'A novel compound RY10-4 induces apoptosis and inhibits invasion via inhibiting STAT3 through ERK-, p38-dependent pathways in human lung adenocarcinoma A549 cells. Previous reports suggested that protoapigenone showed remarkable antitumor activities against a broad spectrum of human cancer cell lines, but had no effect on human lung adenocarcinoma A549 cell. The lack of effective remedies had necessitated the application of new therapeutic scheme. A novel compound RY10-4 which has the similar structure close to protoapigenone showed better antitumor activity. Treatment with RY10-4 inhibited the expression of pro-caspase-3, pro-caspase-9, Bcl-2 as well as phosphorylation of signal transducer and activator of transcription-3 (p-STAT3). It also reduced the expressions of matrix metalloproteinase-2 (MMP-2), matrix metalloproteinase-9 (MMP-9) and increases the expressions of reversion-inducing cysteine-rich protein with kazal motifs (RECK), as well as tissue inhibitor of metalloproteinase (TIMP) via inhibiting STAT3 by activating the mitogen-activated protein (MAP) kinases (the c-Jun N-terminal kinase (JNK), the p38 and extracellular signal-regulated kinase (ERK)) in A549 cells treated with RY10-4. Moreover, the cytotoxic effect of RY10-4 was induction of apoptosis in A549 cells by enhancing production of reactive oxygen species (ROS). Taken together, the observations suggested that RY10-4 had affected Bcl-2 family members, caspases, MMPs, TIMPs expressions and ROS production via inhibiting STAT3 activities through ERK and p38 pathways in A549 cells.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_005',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 994}]}],\n",
       "   'context': 'Jellyfish collagen stimulates production of TNF-α and IL-6 by J774.1 cells through activation of NF-κB and JNK via TLR4 signaling pathway. We previously reported that jellyfish collagen stimulates both the acquired and innate immune responses. In the acquired immune response, jellyfish collagen enhanced immunoglobulin production by lymphocytes in vitro and in vivo. Meanwhile, in the innate immune response jellyfish collagen promoted cytokine production and phagocytotic activity of macrophages. The facts that jellyfish collagen plays several potential roles in stimulating cytokine production by macrophages have further attracted us to uncover its mechanisms. We herein describe that the cytokine production-stimulating activity of jellyfish collagen was canceled by a Toll-like receptor 4 (TLR4) inhibitor. Moreover, jellyfish collagen stimulated phosphorylation of inhibitor of κBα (IκBα), promoted the translocation of nucleus factor-κB (NF-κB), and activated c-Jun N-terminal kinase (JNK). A JNK inhibitor also abrogated the cytokine production-stimulating activity of jellyfish collagen. These results suggest that jellyfish collagen may facilitate cytokine production by macrophages through activation of NF-κB and JNK via the TLR4 signaling pathways.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_006',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 1132}]}],\n",
       "   'context': 'Sinodielide A exerts thermosensitizing effects and induces apoptosis and G2/M cell cycle arrest in DU145 human prostate cancer cells via the Ras/Raf/MAPK and PI3K/Akt signaling pathways. Sinodielide A (SA) is a naturally occurring guaianolide, which is isolated from the root of Sinodielsia yunnanensis. This root, commonly found in Yunnan province, is used in traditional Chinese medicine as an antipyretic, analgesic and diaphoretic agent. A number of studies have reported that agents isolated from a species of Umbelliferae (Apiaceae) have antitumor activities. We previously reported, using combined treatments with this medicinal herb and hyperthermia at various temperatures, an enhanced cytotoxicity in the human prostate cancer androgen - independent cell lines, PC3 and DU145, and analyzed the related mechanisms. In the present study, we investigated the effects of treatment with SA prior to hyperthermia on the thermosensitivity of DU145 cells, and the mechanisms related to the induction of apoptosis and G(2)/M cell cycle arrest via the activation of extracellular-regulated kinase (ERK)1/2, c-Jun N-terminal kinase (JNK) mitogen-activated protein kinase (MAPK) signaling pathways, as well as the phosphoinositide 3-kinase (PI3K)/Akt signaling pathways. Cells were exposed to hyperthermia alone (40-44˚C) or hyperthermia in combination with SA. Lethal damage to cells treated with mild hyperthermia (40 or 42˚C) for up to 6 h was slight; however, hyperthermia in combination with SA synergistically enhanced thermosensivity. Lethal damage to cells treated with acute hyperthermia (43 or 44˚C) was more severe, but these effects were also enhanced and were more significant by the combined treatment with SA. The kinetics of apoptosis induction and cell cycle distribution were analyzed by flow cytometry. In addition, the levels of ERK1/2, JNK and Akt were determined by western blot analysis. The incidence of apoptotic cells after treatment with SA (20.0 µM) at 37˚C for 4 h, hyperthermia (44˚C) alone for 30 min, and the combination in sequence were examined. The sub-G1 division (%) in the diagram obtained by flow cytometry was applied to that assay. The percentage of apoptotic cells (10.53±5.02%) was higher at 48 h as compared to 0, 12 and 24 h after treatment. The distribution of DU145 cells in the G2/M cell cycle phase was markedly increased after 24 h of heating at 44˚C and after the combined treatment with heating and SA. The phosphorylation of ERK1/2 was reduced following treatment with heating and SA, while the levels of phosphorylated JNK (p-JNK) were markedly increased immediately after heating at 44˚C and when heating was combined with SA. By contrast, the levels of phosphorylated Akt (p-Akt) were immediately increased only after heating at 44˚C. Thus, we concluded that SA exerts its thermosensitizing effects on DU145 cells by inhibiting the activation of the MAPK/ERK1/2 and PI3K/Akt signaling pathways.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_007',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 575}]}],\n",
       "   'context': 'Lipopolysaccharide (LPS) induces the apoptosis and inhibits osteoblast differentiation through JNK pathway in MC3T3-E1 cells. Bone degradation is a serious complication of chronic inflammatory diseases such as septic arthritis, osteomyelitis, and infected orthopedic implant failure. Up to date, effective therapeutic treatments for bacteria-caused bone destruction are limited. In our previous study, we found that LPS promoted osteoclast differentiation and activity through activation of mitogen-activated protein kinases (MAPKs) pathway such as c-Jun N-terminal kinases (JNK) and extracellular signal regulated kinase (ERK1/2). The current study was to evaluate the mechanism of LPS on the apoptosis and osteoblast differentiation in MC3T3-E1 cells. MC3T3-E1 osteoblasts were non-treated, treated with LPS. After treatment, the cell viability, the activity of alkaline phosphatase (ALP) and caspase-3 were measured. The expressions of osteoblast-specific genes and Bax, Bcl-2, and caspase-3 were determined by real-time quantitative polymerase chain reaction (qPCR). Protein levels of Bax, Bcl-2, caspase-3, and phosphorylation of MAPKs were measured using Western blotting assays. The MAPK signaling pathway was blocked by pretreatment with JNK inhibitor SP600125. LPS treatment induced a significant decrease in cell metabolism, viability, and ALP activity in MC3T3-E1 cells. LPS also significantly decreased mRNA expressions of osteoblast-related genes in MC3T3-E1 cells. On the other hand, LPS significantly upregulated mRNA expressions and protein levels of Bax and caspase-3 as well as activation of caspase-3, whereas decreased Bcl-2 expression in MC3T3-E1 cells. Furthermore, LPS significantly promoted MAPK pathway including the phosphorylation of JNK and the phosphorylation of ERK1/2; moreover, pretreatment with JNK inhibitor not only attenuated both of phosphorylation-JNK and ERK1/2 enhanced by LPS in MC3T3-E1 cells, but also reversed the downregulated expressions of osteoblast-specific genes including ALP and BSP induced by LPS. In conclusion, LPS could induce osteoblast apoptosis and inhibit osteoblast differentiation via activation of JNK pathway.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_008',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 131}]}],\n",
       "   'context': 'Role of c-Jun N-terminal kinase isoforms in the cellular activity of melanoma cell lines. BACKGROUND: The c-Jun N-terminal kinase (JNK) is thought to be involved in inflammation, proliferation and apoptosis. AIM: To examine the role of JNK isoforms in metastasis, proliferation, migration and invasion of the malignant melanoma (MM) cell lines SK-MEL-28, SK-MEL-3 and WM164, using a kinase-specific inhibitor or isoform-specific small interfering (si)RNAs. RESULTS: SK-MEL-3, a cell line established from metastatic MM, showed slightly increased phosphorylation of both JNK1 and JNK2, whereas WM164, a cell line derived from primary MM, showed significant phosphorylation of JNK1. A JNK inhibitor, SP600125, inhibited cell proliferation of SK-MEL-3 but not SK-MEL-28 or WM164. Transfection of JNK1-specific siRNA reduced the migratory activity of WM164 cells, while silencing of either JNK1 or JNK2 strongly suppressed the invasive activity of SK-MEL-3. CONCLUSIONS: Our study suggests that JNK isoforms have different roles in MM. Metastasis of MM may be regulated by JNK2, while invasion is regulated by both JNK1 and JNK2. JNK1 and JNK2 respectively mediate cell migration and cell proliferation. Further understanding of the specific roles of JNK isoforms in the pathogenesis of MM may lead to the development of therapies targeting specific isoforms.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_009',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 912}]}],\n",
       "   'context': 'Involvement of cardiomyocyte apoptosis in myocardial injury of hereditary epileptic rats. Many clinical cases have been reported where epilepsy profoundly influenced the pathophysiological function of the heart; however, the underlying mechanisms were not elucidated. We use the tremor (TRM) rat as an animal model of epilepsy to investigate the potential mechanisms of myocardial injury. Cardiac functions were assessed by arrhythmia score, heart rate, heart:body mass ratio, and hemodynamic parameters including left ventricular systolic pressure (LVSP), left ventricular end-diastolic pressure (LVEDP), and maximum rate of left ventricular pressure rise and fall (+dp/dtmax and -dp/dtmax). Catecholamine level was detected by HPLC. Apoptotic index was estimated by TUNEL assay. The expressions of Bcl-2, Bax, caspase-3, extracellular signal-regulated protein kinase (ERK), c-Jun NH2-terminal protein kinases (JNK), and p38 were evaluated by Western blot. The results indicated that there existed cardiac dysfunction and cardiomyocyte apoptosis, accompanied by increasing catecholamine levels in TRM rats. Further investigation revealed that apoptosis was mediated by reducing Bcl-2, upregulating Bax, and activating caspase-3. Additional experiments demonstrated that P-ERK1/2 was decreased, whereas P-JNK and P-p38 were up-regulated. Our results suggest that the sympathetic nervous system activation and cardiomyocyte apoptosis are involved in the myocardial injury of TRM rats. The mechanisms of apoptosis might be associated with the activation of the mitochondria-initiated and the mitogen-activated protein kinase pathways.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_010',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 1427}]}],\n",
       "   'context': 'Docosahexaenoic acid reduces cellular inflammatory response following permanent focal cerebral ischemia in rats. Cellular inflammatory response plays an important role in ischemic brain injury and anti-inflammatory treatments in stroke are beneficial. Dietary supplementation with docosahexaenoic acid (DHA) shows anti-inflammatory and neuroprotective effects against ischemic stroke. However, its effectiveness and its precise modes of neuroprotective action remain incompletely understood. This study provides evidence of an alternative target for DHA and sheds light on the mechanism of its physiological benefits. We report a global inhibitory effect of 3 consecutive days of DHA preadministration on circulating and intracerebral cellular inflammatory responses in a rat model of permanent cerebral ischemia. DHA exhibited a neuroprotective effect against ischemic deficits by reduction of behavioral disturbance, brain infarction, edema and blood-brain barrier disruption. The results of enzymatic assay, Western blot, real-time reverse transcriptase polymerase chain reaction and flow cytometric analysis revealed that DHA reduced central macrophages/microglia activation, leukocyte infiltration and pro-inflammatory cytokine expression and peripheral leukocyte activation after cerebral ischemia. In parallel with these immunosuppressive phenomena, DHA attenuated post-stroke oxidative stress, c-Jun N-terminal kinase (JNK) phosphorylation, c-Jun phosphorylation and activating protein-1 (AP-1) activation but further elevated ischemia-induced NF-E2-related factor-2 (Nrf2) and heme oxygenase-1 (HO-1) expression. DHA treatment also had an immunosuppressive effect in lipopolysaccharide/interferon-γ-stimulated glial cultures by attenuating JNK phosphorylation, c-Jun phosphorylation and AP-1 activation and augmenting Nrf2 and HO-1 expression. In summary, we have shown that DHA exhibited neuroprotective and anti-inflammatory effects against ischemic brain injury and these effects were accompanied by decreased oxidative stress and JNK/AP-1 signaling as well as enhanced Nrf2/HO-1 expression.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_011',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 1959}]}],\n",
       "   'context': 'Multisite phosphorylation of c-Jun at threonine 91/93/95 triggers the onset of c-Jun pro-apoptotic activity in cerebellar granule neurons. Cerebellar granule cell (CGC) apoptosis by trophic/potassium (TK) deprivation is a model of election to study the interplay of pro-apoptotic and pro-survival signaling pathways in neuronal cell death. In this model, the c-Jun N-terminal kinase (JNK) induces pro-apoptotic genes through the c-Jun/activator protein 1 (AP-1) transcription factor. On the other side, a survival pathway initiated by lithium leads to repression of pro-apoptotic c-Jun/AP-1 target genes without interfering with JNK activity. Yet, the mechanism by which lithium inhibits c-Jun activity remains to be elucidated. Here, we used this model system to study the regulation and function of site-specific c-Jun phosphorylation at the S63 and T91/T93 JNK sites in neuronal cell death. We found that TK-deprivation led to c-Jun multiphosphorylation at all three JNK sites. However, immunofluorescence analysis of c-Jun phosphorylation at single cell level revealed that the S63 site was phosphorylated in all c-Jun-expressing cells, whereas the response of T91/T93 phosphorylation was more sensitive, mirroring the switch-like apoptotic response of CGCs. Conversely, lithium prevented T91T93 phosphorylation and cell death without affecting the S63 site, suggesting that T91T93 phosphorylation triggers c-Jun pro-apoptotic activity. Accordingly, a c-Jun mutant lacking the T95 priming site for T91/93 phosphorylation protected CGCs from apoptosis, whereas it was able to induce neurite outgrowth in PC12 cells. Vice versa, a c-Jun mutant bearing aspartate substitution of T95 overwhelmed lithium-mediate protection of CGCs from TK-deprivation, validating that inhibition of T91/T93/T95 phosphorylation underlies the effect of lithium on cell death. Mass spectrometry analysis confirmed multiphosphorylation of c-Jun at T91/T93/T95 in cells. Moreover, JNK phosphorylated recombinant c-Jun at T91/T93 in a T95-dependent manner. On the basis of our results, we propose that T91/T93/T95 multiphosphorylation of c-Jun functions as a sensitivity amplifier of the JNK cascade, setting the threshold for c-Jun pro-apoptotic activity in neuronal cells.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_012',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 947}]}],\n",
       "   'context': 'Regulation of RANTES/CCL5 expression in human astrocytes by interleukin-1 and interferon-beta. In the CNS, astrocytes are significant sources of RANTES/CCL5 (regulated upon activation, normal T cell expressed and secreted), a CC-chemokine with important biological function. Astrocyte RANTES/CCL5 has been shown to be induced by interleukin-1 (IL-1), with interferon-gamma (IFNgamma) as a primer, but whether type I interferons play any role in the expression of RANTES/CCL5 is not known. In this report, we studied the detailed mechanism of RANTES/CCL5 induction in primary human astrocytes activated with IL-1 and IFNbeta. Ribonuclease protection assay and ELISA showed that IFNbeta, although not effective alone, increased IL-1-induced RANTES/CCL5 expression, but did not antagonize IFNgamma. IL-1 or IL-1/IFNbeta-induced RANTES/CCL5 expression was inhibited by the super-repressor IkappaBalpha or inhibitors of p38 or c-Jun N-terminal kinase (JNK) MAPKs (mitogen-activated protein kinases), but not by extracellular signal regulated kinases (ERK) inhibitors. IFNbeta enhanced IL-1-induced phosphorylation of p38 MAPK, but was not effective alone. Transfection with mutated RANTES/CCL5 promoter-reporter constructs revealed that kappaB, interferon-stimulated response element (ISRE) and CAATT-enhancer binding protein-beta (C/EBPbeta) sites all contributed to IL-1/IFNbeta-induced RANTES/CCL5 transcription. IFNbeta synergized with IL-1 to induce nuclear accumulation of C/EBPbeta protein. They also synergized to form nuclear ISRE complexes with Stat1, Stat2 and interferon regulatory factor-1 (IRF-1) proteins. Together, our results demonstrate that IFNbeta plays a positive regulatory role in the expression of RANTES/CCL5 in human astrocytes through several distinct mechanisms.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_013',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 110}]}],\n",
       "   'context': 'Identification of a specific domain responsible for JNK2alpha2 autophosphorylation. c-Jun N-terminal kinases (JNKs) are a group of mitogen-activated protein kinase family members that are important in regulating cell growth, proliferation, and apoptosis. Activation of the JNK pathway has been implicated in the formation of several human tumors. We have previously demonstrated that a 55-kDa JNK isoform is constitutively activated in 86% of human brain tumors and more recently demonstrated that this isoform is either JNK2alpha2 or JNK2beta2. Importantly, we have also found that among the 10 known JNK isoforms, the JNK2 isoforms are unique in their ability to autophosphorylate in vitro and in vivo. This does not require the participation of any upstream kinases and also leads to substrate kinase activity in vitro and in vivo. To clarify the mechanism of JNK2alpha2 autoactivation, we have generated a series of chimeric cDNAs joining portions of JNK1alpha2, which does not have detectable autophosphorylation activity, with portions of JNK2alpha2, which has the strongest autophosphorylation activity. Through in vivo and in vitro kinase assays, we were able to define a domain ranging from amino acids 218 to 226 within JNK2alpha2 that is required for its autophosphorylation. Mutation of JNK2alpha2 to its counterpart of JNK1alpha2 in this region abrogated the autophosphorylation activity and c-Jun substrate kinase activity in vivo and in vitro. Notably, switching of JNK1alpha2 to JNK2alpha2 at this 9-amino acid site enabled JNK1alpha2 to gain the autophosphorylation activity in vivo and in vitro. We also found two other functional sites that participate in JNK2alpha2 activity. One site ranging from amino acids 363 to 382 of JNK2alpha2 is required for efficient c-Jun binding in vitro, and a site ranging from amino acids 383 to 424 enhances autophosphorylation intensity, although it is not required for triggering the autophosphorylation in vitro. These findings have uncovered the regions required for JNK2alpha2 autophosphorylation, and this information could be used as potential targets to block JNK2alpha2 activation.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_014',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 883}]}],\n",
       "   'context': 'Phosphorylation of c-Fos by members of the p38 MAPK family. Role in the AP-1 response to UV light. Exposure to sources of UV radiation, such as sunlight, induces a number of cellular alterations that are highly dependent on its ability to affect gene expression. Among them, the rapid activation of genes coding for two subfamilies of proto-oncoproteins, Fos and Jun, which constitute the AP-1 transcription factor, plays a key role in the subsequent regulation of expression of genes involved in DNA repair, cell proliferation, cell cycle arrest, death by apoptosis, and tissue and extracellular matrix remodeling proteases. Besides being regulated at the transcriptional level, Jun and Fos transcriptional activities are also regulated by phosphorylation as a result of the activation of intracellular signaling cascades. In this regard, the phosphorylation of c-Jun by UV-induced JNK has been readily documented, whereas a role for Fos proteins in UV-mediated responses and the identification of Fos-activating kinases has remained elusive. Here we identify p38 MAPKs as proteins that can associate with c-Fos and phosphorylate its transactivation domain both in vitro and in vivo. This phosphorylation is transduced into changes in its transcriptional ability as p38-activated c-Fos enhances AP1-driven gene expression. Our findings indicate that as a consequence of the activation of stress pathways induced by UV light, endogenous c-Fos becomes a substrate of p38 MAPKs and, for the first time, provide evidence that support a critical role for p38 MAPKs in mediating stress-induced c-Fos phosphorylation and gene transcription activation. Using a specific pharmacological inhibitor for p38alpha and -beta, we found that most likely these two isoforms mediate UV-induced c-Fos phosphorylation in vivo. Thus, these newly described pathways act concomitantly with the activation of c-Jun by JNK/MAPKs, thereby contributing to the complexity of AP1-driven gene transcription regulation.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_015',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 436}]}],\n",
       "   'context': 'c-Jun-mediated anticancer mechanisms of tylophorine. Tylophorine, a phenanthroindolizidine alkaloid, is the major medicinal constituent of herb Tylophora indica. Tylophorine treatment increased the accumulation of c-Jun protein, a component of activator protein 1 (AP1), in carcinoma cells. An in vitro kinase assay revealed that the resultant c-Jun phosphorylation was primarily mediated via activated c-Jun N-terminal protein kinase (JNK). Moreover, flow cytometry indicated that ectopically overexpressed c-Jun in conjunction with tylophorine significantly increased the number of carcinoma cells that were arrested at the G1 phase. The tylophorine-mediated downregulation of cyclin A2 protein levels is known to be involved in the primary G1 arrest. Chromatin immunoprecipitation and reporter assays revealed that tylophorine enhanced the c-Jun downregulation of the cyclin A2 promoter activity upon increased binding of c-Jun to the deregulation AP1 site and decreased binding to the upregulation activating transcription factor (ATF) site in the cyclin A2 promoter, thereby reducing cyclin A2 expression. Further, biochemical studies using pharmacological inhibitors and RNA silencing approaches demonstrated that tylophorine-mediated elevation of the c-Jun protein level occurs primarily via two discrete prolonged signaling pathways: (i) the NF-κB/PKCδ_(MKK4)_JNK cascade, which phosphorylates c-Jun and increases its stability by slowing its ubiquitination, and (ii) the PI3K_PDK1_PP2A_eEF2 cascade, which sustains eukaryotic elongation factor 2 (eEF2) activity and thus c-Jun protein translation. To the best of our knowledge, this report is the first to demonstrate the involvement of c-Jun in the anticancer activity of tylophorine and the release of c-Jun translation from a global translational blockade via the PI3K_PDK1_eEF2 signaling cascade.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_016',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 157}]}],\n",
       "   'context': 'High yield purification of JNK1β1 and activation by in vitro reconstitution of the MEKK1→MKK4→JNK MAPK phosphorylation cascade. The c-Jun N-terminal kinase (JNK) pathway forms part of the mitogen-activated protein kinase (MAPK) signaling pathways comprising a sequential three-tiered kinase cascade. Here, an upstream MAP3K (MEKK1) phosphorylates and activates a MAP2K (MKK4 and MKK7), which in turn phosphorylates and activates the MAPK, JNK. The C-terminal kinase domain of MEKK1 (MEKK-C) is constitutively active, while MKK4/7 and JNK are both activated by dual phosphorylation of S/Y, and T/Y residues within their activation loops, respectively. While improvements in the purification of large quantities of active JNKs have recently been made, inadequacies in their yield, purity, and the efficiency of their phosphorylation still exist. We describe a novel and robust method that further improves upon the purification of large yields of highly pure, phosphorylated JNK1β1, which is most suitable for biochemical and biophysical characterization. Codon harmonization of the JNK1β1 gene was used as a precautionary measure toward increasing the soluble overexpression of the kinase. While JNK1β1 and its substrate ATF2 were both purified to >99% purity as GST fusion proteins using GSH-agarose affinity chromatography and each cleaved from GST using thrombin, constitutively-active MEKK-C and inactive MKK4 were separately expressed in E. coli as thioredoxin-His(6)-tagged proteins and purified using urea refolding and Ni(2+)-IMAC, respectively. Activation of JNK1β1 was then achieved by successfully reconstituting the JNK MAPK activation cascade in vitro; MEKK-C was used to activate MKK4, which in turn was used to efficiently phosphorylate and activate large quantities of JNK1β1. Activated JNK1β1 was thereafter able to phosphorylate ATF2 with high catalytic efficiency.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_017',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 94}]}],\n",
       "   'context': 'c-Jun N-Terminal Kinase in Inflammation and Rheumatic Diseases. The c-Jun N-terminal kinases (JNKs) are members of the mitogen-activated protein kinase (MAPK) family and are activated by environmental stress. JNK is also activated by proinflammatory cytokines, such as TNF and IL-1, and Toll-like receptor ligands. This pathway, therefore, can act as a critical convergence point in immune system signaling for both adaptive and innate responses. Like other MAPKs, the JNKs are activated via the sequential activation of protein kinases that includes two dual-specificity MAP kinase kinases (MKK4 and MKK7) and multiple MAP kinase kinase kinases. MAPKs, including JNKs, can be deactivated by a specialized group of phosphatases, called MAP kinase phosphatases. JNK phosphorylates and regulates the activity of transcription factors other than c-Jun, including ATF2, Elk-1, p53 and c-Myc and non-transcription factors, such as members of the Bcl-2 family. The pathway plays a critical role in cell proliferation, apoptosis, angiogenesis and migration. In this review, an overview of the functions that are related to rheumatic diseases is presented. In addition, some diseases in which JNK participates will be highlighted.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_018',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 761}]}],\n",
       "   'context': 'c-Jun N-Terminal Kinase in Inflammation and Rheumatic Diseases. The c-Jun N-terminal kinases (JNKs) are members of the mitogen-activated protein kinase (MAPK) family and are activated by environmental stress. JNK is also activated by proinflammatory cytokines, such as TNF and IL-1, and Toll-like receptor ligands. This pathway, therefore, can act as a critical convergence point in immune system signaling for both adaptive and innate responses. Like other MAPKs, the JNKs are activated via the sequential activation of protein kinases that includes two dual-specificity MAP kinase kinases (MKK4 and MKK7) and multiple MAP kinase kinase kinases. MAPKs, including JNKs, can be deactivated by a specialized group of phosphatases, called MAP kinase phosphatases. JNK phosphorylates and regulates the activity of transcription factors other than c-Jun, including ATF2, Elk-1, p53 and c-Myc and non-transcription factors, such as members of the Bcl-2 family. The pathway plays a critical role in cell proliferation, apoptosis, angiogenesis and migration. In this review, an overview of the functions that are related to rheumatic diseases is presented. In addition, some diseases in which JNK participates will be highlighted.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_019',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'c-Jun NH2-terminal kinase',\n",
       "       'answer_start': 2055}]}],\n",
       "   'context': 'Transcriptional regulation by MAP kinases. Tyrosine kinase growth factor receptors activate MAP kinase by a complex mechanism involving the SH2/3 protein Grb2, the exchange protein Sos, and Ras. The GTP-bound Ras protein binds to the Raf kinase and initiates a protein kinase cascade that leads to MAP kinase activation. Three MAP kinase kinase kinases have been described--c-Raf, c-Mos, and Mekk--that phosphorylate and activate Mek, the MAP kinase kinase. Activated Mek phosphorylates and activates MAP kinase. Subsequently, the activated MAP kinase translocates into the nucleus where many of the physiological targets of the MAP kinase signal transduction pathway are located. These substrates include transcription factors that are regulated by MAP kinase phosphorylation (e.g., Elk-1, c-Myc, c-Jun, c-Fos, and C/EBP beta). Thus the MAP kinase pathway represents a significant mechanism of signal transduction by growth factor receptors from the cell surface to the nucleus that results in the regulation of gene expression. Three MAP kinase homologs have been identified in the rat: Erk1, Erk2, and Erk3. Human MAP kinases that are similar to the rat Erk kinases have also been identified by molecular cloning. The human Erk1 protein kinase has been shown to be widely expressed as a 44-kDa protein in many tissues. The human Erk2 protein kinase is a 41-kDa protein that is expressed ubiquitously. In contrast, a human Erk3-related protein kinase has been found to be expressed at a high level only in heart muscle and brain. The loci of these MAP kinase genes are widely distributed within the human genome: erk2 at 22q11.2; erk1 at 16p11.2; and ek3-related at 18q12-21. In the yeast Saccharomyces cerevisiae, five MAP kinase gene homologs have been described: smkl, mpk1, hog1, fus3, and kss1. Together, these kinases are a more diverse group than the human erks that have been identified. Thus the erks are likely to represent only one subgroup of a larger human MAP kinase gene family. A candidate for this extended family of MAP kinases is the c-Jun NH2-terminal kinase (Jnk), which binds to and phosphorylates the transcription factor c-Jun at the activating sites Ser-63 and Ser-73. Evidence is presented here to demonstrate that Jnk is a distant relative of the MAP kinase group that is activated by dual phosphorylation at Tyr and Thr.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_020',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'Jnk', 'answer_start': 2082}]}],\n",
       "   'context': 'Transcriptional regulation by MAP kinases. Tyrosine kinase growth factor receptors activate MAP kinase by a complex mechanism involving the SH2/3 protein Grb2, the exchange protein Sos, and Ras. The GTP-bound Ras protein binds to the Raf kinase and initiates a protein kinase cascade that leads to MAP kinase activation. Three MAP kinase kinase kinases have been described--c-Raf, c-Mos, and Mekk--that phosphorylate and activate Mek, the MAP kinase kinase. Activated Mek phosphorylates and activates MAP kinase. Subsequently, the activated MAP kinase translocates into the nucleus where many of the physiological targets of the MAP kinase signal transduction pathway are located. These substrates include transcription factors that are regulated by MAP kinase phosphorylation (e.g., Elk-1, c-Myc, c-Jun, c-Fos, and C/EBP beta). Thus the MAP kinase pathway represents a significant mechanism of signal transduction by growth factor receptors from the cell surface to the nucleus that results in the regulation of gene expression. Three MAP kinase homologs have been identified in the rat: Erk1, Erk2, and Erk3. Human MAP kinases that are similar to the rat Erk kinases have also been identified by molecular cloning. The human Erk1 protein kinase has been shown to be widely expressed as a 44-kDa protein in many tissues. The human Erk2 protein kinase is a 41-kDa protein that is expressed ubiquitously. In contrast, a human Erk3-related protein kinase has been found to be expressed at a high level only in heart muscle and brain. The loci of these MAP kinase genes are widely distributed within the human genome: erk2 at 22q11.2; erk1 at 16p11.2; and ek3-related at 18q12-21. In the yeast Saccharomyces cerevisiae, five MAP kinase gene homologs have been described: smkl, mpk1, hog1, fus3, and kss1. Together, these kinases are a more diverse group than the human erks that have been identified. Thus the erks are likely to represent only one subgroup of a larger human MAP kinase gene family. A candidate for this extended family of MAP kinases is the c-Jun NH2-terminal kinase (Jnk), which binds to and phosphorylates the transcription factor c-Jun at the activating sites Ser-63 and Ser-73. Evidence is presented here to demonstrate that Jnk is a distant relative of the MAP kinase group that is activated by dual phosphorylation at Tyr and Thr.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_021',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'JNK', 'answer_start': 436}]}],\n",
       "   'context': 'c-Jun-mediated anticancer mechanisms of tylophorine. Tylophorine, a phenanthroindolizidine alkaloid, is the major medicinal constituent of herb Tylophora indica. Tylophorine treatment increased the accumulation of c-Jun protein, a component of activator protein 1 (AP1), in carcinoma cells. An in vitro kinase assay revealed that the resultant c-Jun phosphorylation was primarily mediated via activated c-Jun N-terminal protein kinase (JNK). Moreover, flow cytometry indicated that ectopically overexpressed c-Jun in conjunction with tylophorine significantly increased the number of carcinoma cells that were arrested at the G1 phase. The tylophorine-mediated downregulation of cyclin A2 protein levels is known to be involved in the primary G1 arrest. Chromatin immunoprecipitation and reporter assays revealed that tylophorine enhanced the c-Jun downregulation of the cyclin A2 promoter activity upon increased binding of c-Jun to the deregulation AP1 site and decreased binding to the upregulation activating transcription factor (ATF) site in the cyclin A2 promoter, thereby reducing cyclin A2 expression. Further, biochemical studies using pharmacological inhibitors and RNA silencing approaches demonstrated that tylophorine-mediated elevation of the c-Jun protein level occurs primarily via two discrete prolonged signaling pathways: (i) the NF-κB/PKCδ_(MKK4)_JNK cascade, which phosphorylates c-Jun and increases its stability by slowing its ubiquitination, and (ii) the PI3K_PDK1_PP2A_eEF2 cascade, which sustains eukaryotic elongation factor 2 (eEF2) activity and thus c-Jun protein translation. To the best of our knowledge, this report is the first to demonstrate the involvement of c-Jun in the anticancer activity of tylophorine and the release of c-Jun translation from a global translational blockade via the PI3K_PDK1_eEF2 signaling cascade.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_022',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'c-Jun NH2-terminal kinase',\n",
       "       'answer_start': 2055}]}],\n",
       "   'context': 'Transcriptional regulation by MAP kinases. Tyrosine kinase growth factor receptors activate MAP kinase by a complex mechanism involving the SH2/3 protein Grb2, the exchange protein Sos, and Ras. The GTP-bound Ras protein binds to the Raf kinase and initiates a protein kinase cascade that leads to MAP kinase activation. Three MAP kinase kinase kinases have been described--c-Raf, c-Mos, and Mekk--that phosphorylate and activate Mek, the MAP kinase kinase. Activated Mek phosphorylates and activates MAP kinase. Subsequently, the activated MAP kinase translocates into the nucleus where many of the physiological targets of the MAP kinase signal transduction pathway are located. These substrates include transcription factors that are regulated by MAP kinase phosphorylation (e.g., Elk-1, c-Myc, c-Jun, c-Fos, and C/EBP beta). Thus the MAP kinase pathway represents a significant mechanism of signal transduction by growth factor receptors from the cell surface to the nucleus that results in the regulation of gene expression. Three MAP kinase homologs have been identified in the rat: Erk1, Erk2, and Erk3. Human MAP kinases that are similar to the rat Erk kinases have also been identified by molecular cloning. The human Erk1 protein kinase has been shown to be widely expressed as a 44-kDa protein in many tissues. The human Erk2 protein kinase is a 41-kDa protein that is expressed ubiquitously. In contrast, a human Erk3-related protein kinase has been found to be expressed at a high level only in heart muscle and brain. The loci of these MAP kinase genes are widely distributed within the human genome: erk2 at 22q11.2; erk1 at 16p11.2; and ek3-related at 18q12-21. In the yeast Saccharomyces cerevisiae, five MAP kinase gene homologs have been described: smkl, mpk1, hog1, fus3, and kss1. Together, these kinases are a more diverse group than the human erks that have been identified. Thus the erks are likely to represent only one subgroup of a larger human MAP kinase gene family. A candidate for this extended family of MAP kinases is the c-Jun NH2-terminal kinase (Jnk), which binds to and phosphorylates the transcription factor c-Jun at the activating sites Ser-63 and Ser-73. Evidence is presented here to demonstrate that Jnk is a distant relative of the MAP kinase group that is activated by dual phosphorylation at Tyr and Thr.'},\n",
       "  {'qas': [{'id': '5518e7da622b194345000004_023',\n",
       "     'question': 'Which MAP kinase phosphorylates the transcription factor c-jun?',\n",
       "     'answers': [{'text': 'Jnk', 'answer_start': 2082}]}],\n",
       "   'context': 'Transcriptional regulation by MAP kinases. Tyrosine kinase growth factor receptors activate MAP kinase by a complex mechanism involving the SH2/3 protein Grb2, the exchange protein Sos, and Ras. The GTP-bound Ras protein binds to the Raf kinase and initiates a protein kinase cascade that leads to MAP kinase activation. Three MAP kinase kinase kinases have been described--c-Raf, c-Mos, and Mekk--that phosphorylate and activate Mek, the MAP kinase kinase. Activated Mek phosphorylates and activates MAP kinase. Subsequently, the activated MAP kinase translocates into the nucleus where many of the physiological targets of the MAP kinase signal transduction pathway are located. These substrates include transcription factors that are regulated by MAP kinase phosphorylation (e.g., Elk-1, c-Myc, c-Jun, c-Fos, and C/EBP beta). Thus the MAP kinase pathway represents a significant mechanism of signal transduction by growth factor receptors from the cell surface to the nucleus that results in the regulation of gene expression. Three MAP kinase homologs have been identified in the rat: Erk1, Erk2, and Erk3. Human MAP kinases that are similar to the rat Erk kinases have also been identified by molecular cloning. The human Erk1 protein kinase has been shown to be widely expressed as a 44-kDa protein in many tissues. The human Erk2 protein kinase is a 41-kDa protein that is expressed ubiquitously. In contrast, a human Erk3-related protein kinase has been found to be expressed at a high level only in heart muscle and brain. The loci of these MAP kinase genes are widely distributed within the human genome: erk2 at 22q11.2; erk1 at 16p11.2; and ek3-related at 18q12-21. In the yeast Saccharomyces cerevisiae, five MAP kinase gene homologs have been described: smkl, mpk1, hog1, fus3, and kss1. Together, these kinases are a more diverse group than the human erks that have been identified. Thus the erks are likely to represent only one subgroup of a larger human MAP kinase gene family. A candidate for this extended family of MAP kinases is the c-Jun NH2-terminal kinase (Jnk), which binds to and phosphorylates the transcription factor c-Jun at the activating sites Ser-63 and Ser-73. Evidence is presented here to demonstrate that Jnk is a distant relative of the MAP kinase group that is activated by dual phosphorylation at Tyr and Thr.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_001',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 279}]}],\n",
       "   'context': 'Lysyl oxidase is downregulated by the EWS/FLI1 oncoprotein and its propeptide domain displays tumor supressor activities in Ewing sarcoma cells. Ewing sarcoma is the second most common bone malignancy in children and young adults. It is driven by oncogenic fusion proteins (i.e. EWS/FLI1) acting as aberrant transcription factors that upregulate and downregulate target genes, leading to cellular transformation. Thus, identificating these target genes and understanding their contribution to Ewing sarcoma tumorigenesis are key for the development of new therapeutic strategies. In this study we show that lysyl oxidase (LOX), an enzyme involved in maintaining structural integrity of the extracellular matrix, is downregulated by the EWS/FLI1 oncoprotein and in consequence it is not expressed in Ewing sarcoma cells and primary tumors. Using a doxycycline inducible system to restore LOX expression in an Ewing sarcoma derived cell line, we showed that LOX displays tumor suppressor activities. Interestingly, we showed that the tumor suppressor activity resides in the propeptide domain of LOX (LOX-PP), an N-terminal domain produced by proteolytic cleavage during the physiological processing of LOX. Expression of LOX-PP reduced cell proliferation, cell migration, anchorage-independent growth in soft agar and formation of tumors in immunodeficient mice. By contrast, the C-terminal domain of LOX, which contains the enzymatic activity, had the opposite effects, corroborating that the tumor suppressor activity of LOX is mediated exclusively by its propeptide domain. Finally, we showed that LOX-PP inhibits ERK/MAPK signalling pathway, and that many pathways involved in cell cycle progression were significantly deregulated by LOX-PP, providing a mechanistic explanation to the cell proliferation inhibition observed upon LOX-PP expression. In summary, our observations indicate that deregulation of the LOX gene participates in Ewing sarcoma development and identify LOX-PP as a new therapeutic target for one of the most aggressive paediatric malignancies. These findings suggest that therapeutic strategies based on the administration of LOX propeptide or functional analogues could be useful for the treatment of this devastating paediatric cancer.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_002',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 674}]}],\n",
       "   'context': \"EWS/FLI1 regulates EYA3 in Ewing sarcoma via modulation of miRNA-708, resulting in increased cell survival and chemoresistance. Ewing sarcoma is an aggressive pediatric cancer of the bone and soft tissue, in which patients whose tumors have a poor histologic response to initial chemotherapy have a poor overall prognosis. Therefore, it is important to identify molecules involved in resistance to chemotherapy. Herein, we show that the DNA repair protein and transcriptional cofactor, EYA3, is highly expressed in Ewing sarcoma tumor samples and cell lines compared with mesenchymal stem cells, the presumed cell-of-origin of Ewing sarcoma, and that it is regulated by the EWS/FLI1 fusion protein transcription factor. We further show that EWS/FLI1 mediates upregulation of EYA3 via repression of miR-708, a miRNA that targets the EYA3 3'-untranslated region, rather than by binding the EYA3 promoter directly. Importantly, we show that high levels of EYA3 significantly correlate with low levels of miR-708 in Ewing sarcoma samples, suggesting that this miR-mediated mechanism of EYA3 regulation holds true in human cancers. Because EYA proteins are important for cell survival during development, we examine, and show, that loss of EYA3 decreases survival of Ewing sarcoma cells. Most importantly, knockdown of EYA3 in Ewing sarcoma cells leads to sensitization to DNA-damaging chemotherapeutics used in the treatment of Ewing sarcoma, and as expected, after chemotherapeutic treatment, EYA3 knockdown cells repair DNA damage less effectively than their control counterparts. These studies identify EYA3 as a novel mediator of chemoresistance in Ewing sarcoma and define the molecular mechanisms of both EYA3 overexpression and of EYA3-mediated chemoresistance.\"},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_003',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 56}]}],\n",
       "   'context': 'The orphan nuclear receptor DAX1 is up-regulated by the EWS/FLI1 oncoprotein and is highly expressed in Ewing tumors. The Ewing family of tumors harbors chromosomal translocations that join the N-terminal region of the EWS gene with the C-terminal region of several transcription factors of the ETS family, mainly FLI1, resulting in chimeric transcription factors that play a pivotal role in the pathogenesis of Ewing tumors. To identify downstream targets of the EWS/FLI1 fusion protein, we established 293 cells expressing constitutively either the chimeric EWS/FLI1 or wild type FLI1 proteins and used cDNA arrays to identify genes differentially regulated by EWS/FLI1. DAX1 (NR0B1), an unusual orphan nuclear receptor involved in gonadal development, sex determination and steroidogenesis, showed a consistent up-regulation by EWS/FLI1 oncoprotein, but not by wild type FLI1. Specific induction of DAX1 by EWS/FLI1 was confirmed in two independent cell systems with inducible expression of EWS/FLI1. We also analyzed the expression of DAX1 in Ewing tumors and derived cell lines, as well as in other nonrelated small round cell tumors. DAX1 was expressed in all Ewing tumor specimens analyzed, and in seven out of eight Ewing tumor cell lines, but not in any neuroblastoma or embryonal rhabdomyosarcoma. Furthermore, silencing of EWS/FLI1 by RNA interference in a Ewing tumor cell line markedly reduced the levels of DAX1 mRNA and protein, confirming that DAX1 up-regulation is dependent upon EWS/FLI1 expression. The high levels of DAX1 found in Ewing tumors and its potent transcriptional repressor activity suggest that the oncogenic effect of EWS/FLI1 may be mediated, at least in part, by the up-regulation of DAX1 expression.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_004',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 464}]}],\n",
       "   'context': 'The orphan nuclear receptor DAX1 is up-regulated by the EWS/FLI1 oncoprotein and is highly expressed in Ewing tumors. The Ewing family of tumors harbors chromosomal translocations that join the N-terminal region of the EWS gene with the C-terminal region of several transcription factors of the ETS family, mainly FLI1, resulting in chimeric transcription factors that play a pivotal role in the pathogenesis of Ewing tumors. To identify downstream targets of the EWS/FLI1 fusion protein, we established 293 cells expressing constitutively either the chimeric EWS/FLI1 or wild type FLI1 proteins and used cDNA arrays to identify genes differentially regulated by EWS/FLI1. DAX1 (NR0B1), an unusual orphan nuclear receptor involved in gonadal development, sex determination and steroidogenesis, showed a consistent up-regulation by EWS/FLI1 oncoprotein, but not by wild type FLI1. Specific induction of DAX1 by EWS/FLI1 was confirmed in two independent cell systems with inducible expression of EWS/FLI1. We also analyzed the expression of DAX1 in Ewing tumors and derived cell lines, as well as in other nonrelated small round cell tumors. DAX1 was expressed in all Ewing tumor specimens analyzed, and in seven out of eight Ewing tumor cell lines, but not in any neuroblastoma or embryonal rhabdomyosarcoma. Furthermore, silencing of EWS/FLI1 by RNA interference in a Ewing tumor cell line markedly reduced the levels of DAX1 mRNA and protein, confirming that DAX1 up-regulation is dependent upon EWS/FLI1 expression. The high levels of DAX1 found in Ewing tumors and its potent transcriptional repressor activity suggest that the oncogenic effect of EWS/FLI1 may be mediated, at least in part, by the up-regulation of DAX1 expression.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_005',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 560}]}],\n",
       "   'context': 'The orphan nuclear receptor DAX1 is up-regulated by the EWS/FLI1 oncoprotein and is highly expressed in Ewing tumors. The Ewing family of tumors harbors chromosomal translocations that join the N-terminal region of the EWS gene with the C-terminal region of several transcription factors of the ETS family, mainly FLI1, resulting in chimeric transcription factors that play a pivotal role in the pathogenesis of Ewing tumors. To identify downstream targets of the EWS/FLI1 fusion protein, we established 293 cells expressing constitutively either the chimeric EWS/FLI1 or wild type FLI1 proteins and used cDNA arrays to identify genes differentially regulated by EWS/FLI1. DAX1 (NR0B1), an unusual orphan nuclear receptor involved in gonadal development, sex determination and steroidogenesis, showed a consistent up-regulation by EWS/FLI1 oncoprotein, but not by wild type FLI1. Specific induction of DAX1 by EWS/FLI1 was confirmed in two independent cell systems with inducible expression of EWS/FLI1. We also analyzed the expression of DAX1 in Ewing tumors and derived cell lines, as well as in other nonrelated small round cell tumors. DAX1 was expressed in all Ewing tumor specimens analyzed, and in seven out of eight Ewing tumor cell lines, but not in any neuroblastoma or embryonal rhabdomyosarcoma. Furthermore, silencing of EWS/FLI1 by RNA interference in a Ewing tumor cell line markedly reduced the levels of DAX1 mRNA and protein, confirming that DAX1 up-regulation is dependent upon EWS/FLI1 expression. The high levels of DAX1 found in Ewing tumors and its potent transcriptional repressor activity suggest that the oncogenic effect of EWS/FLI1 may be mediated, at least in part, by the up-regulation of DAX1 expression.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_006',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 663}]}],\n",
       "   'context': 'The orphan nuclear receptor DAX1 is up-regulated by the EWS/FLI1 oncoprotein and is highly expressed in Ewing tumors. The Ewing family of tumors harbors chromosomal translocations that join the N-terminal region of the EWS gene with the C-terminal region of several transcription factors of the ETS family, mainly FLI1, resulting in chimeric transcription factors that play a pivotal role in the pathogenesis of Ewing tumors. To identify downstream targets of the EWS/FLI1 fusion protein, we established 293 cells expressing constitutively either the chimeric EWS/FLI1 or wild type FLI1 proteins and used cDNA arrays to identify genes differentially regulated by EWS/FLI1. DAX1 (NR0B1), an unusual orphan nuclear receptor involved in gonadal development, sex determination and steroidogenesis, showed a consistent up-regulation by EWS/FLI1 oncoprotein, but not by wild type FLI1. Specific induction of DAX1 by EWS/FLI1 was confirmed in two independent cell systems with inducible expression of EWS/FLI1. We also analyzed the expression of DAX1 in Ewing tumors and derived cell lines, as well as in other nonrelated small round cell tumors. DAX1 was expressed in all Ewing tumor specimens analyzed, and in seven out of eight Ewing tumor cell lines, but not in any neuroblastoma or embryonal rhabdomyosarcoma. Furthermore, silencing of EWS/FLI1 by RNA interference in a Ewing tumor cell line markedly reduced the levels of DAX1 mRNA and protein, confirming that DAX1 up-regulation is dependent upon EWS/FLI1 expression. The high levels of DAX1 found in Ewing tumors and its potent transcriptional repressor activity suggest that the oncogenic effect of EWS/FLI1 may be mediated, at least in part, by the up-regulation of DAX1 expression.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_007',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 1651}]}],\n",
       "   'context': 'The orphan nuclear receptor DAX1 is up-regulated by the EWS/FLI1 oncoprotein and is highly expressed in Ewing tumors. The Ewing family of tumors harbors chromosomal translocations that join the N-terminal region of the EWS gene with the C-terminal region of several transcription factors of the ETS family, mainly FLI1, resulting in chimeric transcription factors that play a pivotal role in the pathogenesis of Ewing tumors. To identify downstream targets of the EWS/FLI1 fusion protein, we established 293 cells expressing constitutively either the chimeric EWS/FLI1 or wild type FLI1 proteins and used cDNA arrays to identify genes differentially regulated by EWS/FLI1. DAX1 (NR0B1), an unusual orphan nuclear receptor involved in gonadal development, sex determination and steroidogenesis, showed a consistent up-regulation by EWS/FLI1 oncoprotein, but not by wild type FLI1. Specific induction of DAX1 by EWS/FLI1 was confirmed in two independent cell systems with inducible expression of EWS/FLI1. We also analyzed the expression of DAX1 in Ewing tumors and derived cell lines, as well as in other nonrelated small round cell tumors. DAX1 was expressed in all Ewing tumor specimens analyzed, and in seven out of eight Ewing tumor cell lines, but not in any neuroblastoma or embryonal rhabdomyosarcoma. Furthermore, silencing of EWS/FLI1 by RNA interference in a Ewing tumor cell line markedly reduced the levels of DAX1 mRNA and protein, confirming that DAX1 up-regulation is dependent upon EWS/FLI1 expression. The high levels of DAX1 found in Ewing tumors and its potent transcriptional repressor activity suggest that the oncogenic effect of EWS/FLI1 may be mediated, at least in part, by the up-regulation of DAX1 expression.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_008',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 674}]}],\n",
       "   'context': \"EWS/FLI1 regulates EYA3 in Ewing sarcoma via modulation of miRNA-708, resulting in increased cell survival and chemoresistance. Ewing sarcoma is an aggressive pediatric cancer of the bone and soft tissue, in which patients whose tumors have a poor histologic response to initial chemotherapy have a poor overall prognosis. Therefore, it is important to identify molecules involved in resistance to chemotherapy. Herein, we show that the DNA repair protein and transcriptional cofactor, EYA3, is highly expressed in Ewing sarcoma tumor samples and cell lines compared with mesenchymal stem cells, the presumed cell-of-origin of Ewing sarcoma, and that it is regulated by the EWS/FLI1 fusion protein transcription factor. We further show that EWS/FLI1 mediates upregulation of EYA3 via repression of miR-708, a miRNA that targets the EYA3 3'-untranslated region, rather than by binding the EYA3 promoter directly. Importantly, we show that high levels of EYA3 significantly correlate with low levels of miR-708 in Ewing sarcoma samples, suggesting that this miR-mediated mechanism of EYA3 regulation holds true in human cancers. Because EYA proteins are important for cell survival during development, we examine, and show, that loss of EYA3 decreases survival of Ewing sarcoma cells. Most importantly, knockdown of EYA3 in Ewing sarcoma cells leads to sensitization to DNA-damaging chemotherapeutics used in the treatment of Ewing sarcoma, and as expected, after chemotherapeutic treatment, EYA3 knockdown cells repair DNA damage less effectively than their control counterparts. These studies identify EYA3 as a novel mediator of chemoresistance in Ewing sarcoma and define the molecular mechanisms of both EYA3 overexpression and of EYA3-mediated chemoresistance.\"},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_009',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 287}]}],\n",
       "   'context': 'Ewing sarcoma EWS protein regulates midzone formation by recruiting Aurora B kinase to the midzone. Ewing sarcoma is a malignant bone cancer that primarily occurs in children and adolescents. Eighty-five percent of Ewing sarcoma is characterized by the presence of the aberrant chimeric EWS/FLI1 fusion gene. Previously, we demonstrated that an interaction between EWS/FLI1 and wild-type EWS led to the inhibition of EWS activity and mitotic dysfunction. Although defective mitosis is considered to be a critical step in cancer initiation, it is unknown how interference with EWS contributes to Ewing sarcoma formation. Here, we demonstrate that EWS/FLI1- and EWS-knockdown cells display a high incidence of defects in the midzone, a midline structure located between segregating chromatids during anaphase. Defects in the midzone can lead to the failure of cytokinesis and can result in the induction of aneuploidy. The similarity among the phenotypes of EWS/FLI1- and EWS siRNA-transfected HeLa cells points to the inhibition of EWS as the key mechanism for the induction of midzone defects. Supporting this observation, the ectopic expression of EWS rescues the high incidence of midzone defects observed in Ewing sarcoma A673 cells. We discovered that EWS interacts with Aurora B kinase, and that EWS is also required for recruiting Aurora B to the midzone. A domain analysis revealed that the R565 in the RGG3 domain of EWS is essential for both Aurora B interaction and the recruitment of Aurora B to the midzone. Here, we propose that the impairment of EWS-dependent midzone formation via the recruitment of Aurora B is a potential mechanism of Ewing sarcoma development.'},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_010',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 674}]}],\n",
       "   'context': \"EWS/FLI1 regulates EYA3 in Ewing sarcoma via modulation of miRNA-708, resulting in increased cell survival and chemoresistance. Ewing sarcoma is an aggressive pediatric cancer of the bone and soft tissue, in which patients whose tumors have a poor histologic response to initial chemotherapy have a poor overall prognosis. Therefore, it is important to identify molecules involved in resistance to chemotherapy. Herein, we show that the DNA repair protein and transcriptional cofactor, EYA3, is highly expressed in Ewing sarcoma tumor samples and cell lines compared with mesenchymal stem cells, the presumed cell-of-origin of Ewing sarcoma, and that it is regulated by the EWS/FLI1 fusion protein transcription factor. We further show that EWS/FLI1 mediates upregulation of EYA3 via repression of miR-708, a miRNA that targets the EYA3 3'-untranslated region, rather than by binding the EYA3 promoter directly. Importantly, we show that high levels of EYA3 significantly correlate with low levels of miR-708 in Ewing sarcoma samples, suggesting that this miR-mediated mechanism of EYA3 regulation holds true in human cancers. Because EYA proteins are important for cell survival during development, we examine, and show, that loss of EYA3 decreases survival of Ewing sarcoma cells. Most importantly, knockdown of EYA3 in Ewing sarcoma cells leads to sensitization to DNA-damaging chemotherapeutics used in the treatment of Ewing sarcoma, and as expected, after chemotherapeutic treatment, EYA3 knockdown cells repair DNA damage less effectively than their control counterparts. These studies identify EYA3 as a novel mediator of chemoresistance in Ewing sarcoma and define the molecular mechanisms of both EYA3 overexpression and of EYA3-mediated chemoresistance.\"},\n",
       "  {'qas': [{'id': '552fac4fbc4f83e828000006_011',\n",
       "     'question': 'Which fusion protein is involved in the development of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'EWS/FLI1', 'answer_start': 287}]}],\n",
       "   'context': 'Ewing sarcoma EWS protein regulates midzone formation by recruiting Aurora B kinase to the midzone. Ewing sarcoma is a malignant bone cancer that primarily occurs in children and adolescents. Eighty-five percent of Ewing sarcoma is characterized by the presence of the aberrant chimeric EWS/FLI1 fusion gene. Previously, we demonstrated that an interaction between EWS/FLI1 and wild-type EWS led to the inhibition of EWS activity and mitotic dysfunction. Although defective mitosis is considered to be a critical step in cancer initiation, it is unknown how interference with EWS contributes to Ewing sarcoma formation. Here, we demonstrate that EWS/FLI1- and EWS-knockdown cells display a high incidence of defects in the midzone, a midline structure located between segregating chromatids during anaphase. Defects in the midzone can lead to the failure of cytokinesis and can result in the induction of aneuploidy. The similarity among the phenotypes of EWS/FLI1- and EWS siRNA-transfected HeLa cells points to the inhibition of EWS as the key mechanism for the induction of midzone defects. Supporting this observation, the ectopic expression of EWS rescues the high incidence of midzone defects observed in Ewing sarcoma A673 cells. We discovered that EWS interacts with Aurora B kinase, and that EWS is also required for recruiting Aurora B to the midzone. A domain analysis revealed that the R565 in the RGG3 domain of EWS is essential for both Aurora B interaction and the recruitment of Aurora B to the midzone. Here, we propose that the impairment of EWS-dependent midzone formation via the recruitment of Aurora B is a potential mechanism of Ewing sarcoma development.'},\n",
       "  {'qas': [{'id': '54cf7051f693c3b16b000013_001',\n",
       "     'question': 'Treatment of which disease was investigated in the MR CLEAN study?',\n",
       "     'answers': [{'text': 'Acute ischemic stroke', 'answer_start': 251}]}],\n",
       "   'context': 'A meta-analysis of prospective randomized controlled trials evaluating endovascular therapies for acute ischemic stroke. INTRODUCTION: A recent randomized controlled trial (RCT), the Multicenter Randomized CLinical trial of Endovascular treatment for Acute ischemic stroke in the Netherlands (MR CLEAN), demonstrated better outcomes with endovascular treatment compared with medical therapy for acute ischemic stroke (AIS). However, previous trials have provided mixed results regarding the efficacy of endovascular treatment for AIS. A meta-analysis of all available trial data was performed to summarize the available evidence. METHODS: A literature search was performed to identify all prospective RCTs comparing endovascular therapies with medical management for AIS. Two datasets were created: (1) all patients randomized after confirmation of large vessel occlusion (LVO) (consistent with the contemporary standard of practice at the majority of centers); and (2) all patients with outcome data who underwent randomization regardless of qualifying vascular imaging. The pre-specified primary outcome measure was modified Rankin Scale score of 0-2 at 90 days. A fixed-effect model was used to determine significance. RESULTS: Five prospective RCTs comparing endovascular therapies with medical management were included in dataset 1 (1183 patients) and six were included in dataset 2 (1903 total patients). Endovascular therapies were associated with significantly improved outcomes compared with medical management (OR 1.67, 95% CI 1.29 to 1.16, p=0.0001) for patients with LVO (dataset 1). This benefit persisted when patients from all six RCTs were included, even in the absence of confirmation of LVO (OR 1.27, 95% CI 1.05 to 1.54, p=0.019; dataset 2). CONCLUSIONS: A meta-analysis of prospective RCTs comparing endovascular therapies with medical management demonstrates superior outcomes in patients randomized to endovascular therapy.'},\n",
       "  {'qas': [{'id': '54cf7051f693c3b16b000013_002',\n",
       "     'question': 'Treatment of which disease was investigated in the MR CLEAN study?',\n",
       "     'answers': [{'text': 'acute ischemic stroke', 'answer_start': 395}]}],\n",
       "   'context': 'A meta-analysis of prospective randomized controlled trials evaluating endovascular therapies for acute ischemic stroke. INTRODUCTION: A recent randomized controlled trial (RCT), the Multicenter Randomized CLinical trial of Endovascular treatment for Acute ischemic stroke in the Netherlands (MR CLEAN), demonstrated better outcomes with endovascular treatment compared with medical therapy for acute ischemic stroke (AIS). However, previous trials have provided mixed results regarding the efficacy of endovascular treatment for AIS. A meta-analysis of all available trial data was performed to summarize the available evidence. METHODS: A literature search was performed to identify all prospective RCTs comparing endovascular therapies with medical management for AIS. Two datasets were created: (1) all patients randomized after confirmation of large vessel occlusion (LVO) (consistent with the contemporary standard of practice at the majority of centers); and (2) all patients with outcome data who underwent randomization regardless of qualifying vascular imaging. The pre-specified primary outcome measure was modified Rankin Scale score of 0-2 at 90 days. A fixed-effect model was used to determine significance. RESULTS: Five prospective RCTs comparing endovascular therapies with medical management were included in dataset 1 (1183 patients) and six were included in dataset 2 (1903 total patients). Endovascular therapies were associated with significantly improved outcomes compared with medical management (OR 1.67, 95% CI 1.29 to 1.16, p=0.0001) for patients with LVO (dataset 1). This benefit persisted when patients from all six RCTs were included, even in the absence of confirmation of LVO (OR 1.27, 95% CI 1.05 to 1.54, p=0.019; dataset 2). CONCLUSIONS: A meta-analysis of prospective RCTs comparing endovascular therapies with medical management demonstrates superior outcomes in patients randomized to endovascular therapy.'},\n",
       "  {'qas': [{'id': '54cf7051f693c3b16b000013_003',\n",
       "     'question': 'Treatment of which disease was investigated in the MR CLEAN study?',\n",
       "     'answers': [{'text': 'Acute ischemic stroke', 'answer_start': 251}]}],\n",
       "   'context': 'A meta-analysis of prospective randomized controlled trials evaluating endovascular therapies for acute ischemic stroke. INTRODUCTION: A recent randomized controlled trial (RCT), the Multicenter Randomized CLinical trial of Endovascular treatment for Acute ischemic stroke in the Netherlands (MR CLEAN), demonstrated better outcomes with endovascular treatment compared with medical therapy for acute ischemic stroke (AIS). However, previous trials have provided mixed results regarding the efficacy of endovascular treatment for AIS. A meta-analysis of all available trial data was performed to summarize the available evidence. METHODS: A literature search was performed to identify all prospective RCTs comparing endovascular therapies with medical management for AIS. Two datasets were created: (1) all patients randomized after confirmation of large vessel occlusion (LVO) (consistent with the contemporary standard of practice at the majority of centers); and (2) all patients with outcome data who underwent randomization regardless of qualifying vascular imaging. The pre-specified primary outcome measure was modified Rankin Scale score of 0-2 at 90 days. A fixed-effect model was used to determine significance. RESULTS: Five prospective RCTs comparing endovascular therapies with medical management were included in dataset 1 (1183 patients) and six were included in dataset 2 (1903 total patients). Endovascular therapies were associated with significantly improved outcomes compared with medical management (OR 1.67, 95% CI 1.29 to 1.16, p=0.0001) for patients with LVO (dataset 1). This benefit persisted when patients from all six RCTs were included, even in the absence of confirmation of LVO (OR 1.27, 95% CI 1.05 to 1.54, p=0.019; dataset 2). CONCLUSIONS: A meta-analysis of prospective RCTs comparing endovascular therapies with medical management demonstrates superior outcomes in patients randomized to endovascular therapy.'},\n",
       "  {'qas': [{'id': '54cf7051f693c3b16b000013_004',\n",
       "     'question': 'Treatment of which disease was investigated in the MR CLEAN study?',\n",
       "     'answers': [{'text': 'acute ischemic stroke', 'answer_start': 395}]}],\n",
       "   'context': 'A meta-analysis of prospective randomized controlled trials evaluating endovascular therapies for acute ischemic stroke. INTRODUCTION: A recent randomized controlled trial (RCT), the Multicenter Randomized CLinical trial of Endovascular treatment for Acute ischemic stroke in the Netherlands (MR CLEAN), demonstrated better outcomes with endovascular treatment compared with medical therapy for acute ischemic stroke (AIS). However, previous trials have provided mixed results regarding the efficacy of endovascular treatment for AIS. A meta-analysis of all available trial data was performed to summarize the available evidence. METHODS: A literature search was performed to identify all prospective RCTs comparing endovascular therapies with medical management for AIS. Two datasets were created: (1) all patients randomized after confirmation of large vessel occlusion (LVO) (consistent with the contemporary standard of practice at the majority of centers); and (2) all patients with outcome data who underwent randomization regardless of qualifying vascular imaging. The pre-specified primary outcome measure was modified Rankin Scale score of 0-2 at 90 days. A fixed-effect model was used to determine significance. RESULTS: Five prospective RCTs comparing endovascular therapies with medical management were included in dataset 1 (1183 patients) and six were included in dataset 2 (1903 total patients). Endovascular therapies were associated with significantly improved outcomes compared with medical management (OR 1.67, 95% CI 1.29 to 1.16, p=0.0001) for patients with LVO (dataset 1). This benefit persisted when patients from all six RCTs were included, even in the absence of confirmation of LVO (OR 1.27, 95% CI 1.05 to 1.54, p=0.019; dataset 2). CONCLUSIONS: A meta-analysis of prospective RCTs comparing endovascular therapies with medical management demonstrates superior outcomes in patients randomized to endovascular therapy.'},\n",
       "  {'qas': [{'id': '54cf7051f693c3b16b000013_005',\n",
       "     'question': 'Treatment of which disease was investigated in the MR CLEAN study?',\n",
       "     'answers': [{'text': 'acute ischemic stroke', 'answer_start': 1948}]}],\n",
       "   'context': 'A randomized trial of intraarterial treatment for acute ischemic stroke. BACKGROUND: In patients with acute ischemic stroke caused by a proximal intracranial arterial occlusion, intraarterial treatment is highly effective for emergency revascularization. However, proof of a beneficial effect on functional outcome is lacking. METHODS: We randomly assigned eligible patients to either intraarterial treatment plus usual care or usual care alone. Eligible patients had a proximal arterial occlusion in the anterior cerebral circulation that was confirmed on vessel imaging and that could be treated intraarterially within 6 hours after symptom onset. The primary outcome was the modified Rankin scale score at 90 days; this categorical scale measures functional outcome, with scores ranging from 0 (no symptoms) to 6 (death). The treatment effect was estimated with ordinal logistic regression as a common odds ratio, adjusted for prespecified prognostic factors. The adjusted common odds ratio measured the likelihood that intraarterial treatment would lead to lower modified Rankin scores, as compared with usual care alone (shift analysis). RESULTS: We enrolled 500 patients at 16 medical centers in The Netherlands (233 assigned to intraarterial treatment and 267 to usual care alone). The mean age was 65 years (range, 23 to 96), and 445 patients (89.0%) were treated with intravenous alteplase before randomization. Retrievable stents were used in 190 of the 233 patients (81.5%) assigned to intraarterial treatment. The adjusted common odds ratio was 1.67 (95% confidence interval [CI], 1.21 to 2.30). There was an absolute difference of 13.5 percentage points (95% CI, 5.9 to 21.2) in the rate of functional independence (modified Rankin score, 0 to 2) in favor of the intervention (32.6% vs. 19.1%). There were no significant differences in mortality or the occurrence of symptomatic intracerebral hemorrhage. CONCLUSIONS: In patients with acute ischemic stroke caused by a proximal intracranial occlusion of the anterior circulation, intraarterial treatment administered within 6 hours after stroke onset was effective and safe. (Funded by the Dutch Heart Foundation and others; MR CLEAN Netherlands Trial Registry number, NTR1804, and Current Controlled Trials number, ISRCTN10888758.).'},\n",
       "  {'qas': [{'id': '54cf7051f693c3b16b000013_006',\n",
       "     'question': 'Treatment of which disease was investigated in the MR CLEAN study?',\n",
       "     'answers': [{'text': 'acute ischemic stroke', 'answer_start': 80}]}],\n",
       "   'context': 'MR CLEAN, a multicenter randomized clinical trial of endovascular treatment for acute ischemic stroke in the Netherlands: study protocol for a randomized controlled trial. BACKGROUND: Endovascular or intra-arterial treatment (IAT) increases the likelihood of recanalization in patients with acute ischemic stroke caused by a proximal intracranial arterial occlusion. However, a beneficial effect of IAT on functional recovery in patients with acute ischemic stroke remains unproven. The aim of this study is to assess the effect of IAT on functional outcome in patients with acute ischemic stroke. Additionally, we aim to assess the safety of IAT, and the effect on recanalization of different mechanical treatment modalities. METHODS/DESIGN: A multicenter randomized clinical trial with blinded outcome assessment. The active comparison is IAT versus no IAT. IAT may consist of intra-arterial thrombolysis with alteplase or urokinase, mechanical treatment or both. Mechanical treatment refers to retraction, aspiration, sonolysis, or use of a retrievable stent (stent-retriever). Patients with a relevant intracranial proximal arterial occlusion of the anterior circulation, who can be treated within 6\\xa0hours after stroke onset, are eligible. Treatment effect will be estimated with ordinal logistic regression (shift analysis); 500 patients will be included in the trial for a power of 80% to detect a shift leading to a decrease in dependency in 10% of treated patients. The primary outcome is the score on the modified Rankin scale at 90\\xa0days. Secondary outcomes are the National Institutes of Health stroke scale score at 24\\xa0hours, vessel patency at 24\\xa0hours, infarct size on day 5, and the occurrence of major bleeding during the first 5\\xa0days. DISCUSSION: If IAT leads to a 10% absolute reduction in poor outcome after stroke, careful implementation of the intervention could save approximately 1% of all new stroke cases from death or disability annually. TRIAL REGISTRATION: NTR1804 (7 May 2009)/ISRCTN10888758 (24 July 2012).'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_001',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 83}]}],\n",
       "   'context': 'Efficacy and safety profile of evolocumab (AMG145), an injectable inhibitor of the proprotein convertase subtilisin/kexin type 9: the available clinical evidence. INTRODUCTION: Despite the proven efficacy of statins, they are often reported to be inadequate to achieve low-density lipoprotein cholesterol (LDL-C) goals (especially in high-risk patients). Moreover, a large number of subjects cannot tolerate statins or full doses of these drugs. Thus, there is a need for additional effective LDL-C reducing agents. AREAS COVERED: Evolocumab (AMG145) is a monoclonal antibody inhibiting the proprotein convertase subtilisin/kexin type 9 that binds to the liver LDL receptor and prevents it from normal recycling by targeting it for degradation. Phase I and II trials revealed that its subcutaneous injection, either alone or in combination with statins, is able to reduce LDL-C from 40 to 80%, apolipoprotein B100 from 30 to 59% and lipoprotein(a) from 18 to 36% in a dose-dependent manner. The incidence of side effects seems to be low and mainly limited to nasopharyngitis, injection site pain, arthralgia and back pain. EXPERT OPINION: Evolocumab is an innovative powerful lipid-lowering drug, additive to statins and with an apparently large therapeutic range associated to a low rate of mild adverse events. If available data will be confirmed in long-term trials with strong outcomes, Evolocumab will provide an essential tool to treat high-risk patients who need to reach ambitious LDL-C target.'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_002',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 591}]}],\n",
       "   'context': 'Efficacy and safety profile of evolocumab (AMG145), an injectable inhibitor of the proprotein convertase subtilisin/kexin type 9: the available clinical evidence. INTRODUCTION: Despite the proven efficacy of statins, they are often reported to be inadequate to achieve low-density lipoprotein cholesterol (LDL-C) goals (especially in high-risk patients). Moreover, a large number of subjects cannot tolerate statins or full doses of these drugs. Thus, there is a need for additional effective LDL-C reducing agents. AREAS COVERED: Evolocumab (AMG145) is a monoclonal antibody inhibiting the proprotein convertase subtilisin/kexin type 9 that binds to the liver LDL receptor and prevents it from normal recycling by targeting it for degradation. Phase I and II trials revealed that its subcutaneous injection, either alone or in combination with statins, is able to reduce LDL-C from 40 to 80%, apolipoprotein B100 from 30 to 59% and lipoprotein(a) from 18 to 36% in a dose-dependent manner. The incidence of side effects seems to be low and mainly limited to nasopharyngitis, injection site pain, arthralgia and back pain. EXPERT OPINION: Evolocumab is an innovative powerful lipid-lowering drug, additive to statins and with an apparently large therapeutic range associated to a low rate of mild adverse events. If available data will be confirmed in long-term trials with strong outcomes, Evolocumab will provide an essential tool to treat high-risk patients who need to reach ambitious LDL-C target.'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_003',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 731}]}],\n",
       "   'context': 'Rationale and design of LAPLACE-2: a phase 3, randomized, double-blind, placebo- and ezetimibe-controlled trial evaluating the efficacy and safety of evolocumab in subjects with hypercholesterolemia on background statin therapy. Low-density lipoprotein cholesterol (LDL-C) levels are significantly associated with atherosclerotic cardiovascular disease (ASCVD) risk, and studies using interventions that lower LDL-C levels have been shown to reduce the risk of ASCVD events and mortality. Statin treatment is the current first-line therapy for lowering LDL-C and reducing ASCVD risk. However, many patients are still unable to reach recommended LDL-C goals on maximally tolerated statin therapy. Monoclonal antibodies that inhibit proprotein convertase subtilisin/kexin type 9, including evolocumab (previously AMG 145), dramatically lowered LDL-C in phase 2 clinical trials when administered alone or in combination with a statin. The aim of this phase 3 study is to evaluate the efficacy of 12 weeks of subcutaneous evolocumab (vs placebo) administered every 2 weeks or every month in combination with a statin in patients with hypercholesterolemia and mixed dyslipidemia. This study will also provide comparative efficacy, safety, and tolerability data between evolocumab and ezetimibe when added to background atorvastatin therapy.'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_004',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 967}]}],\n",
       "   'context': 'Antibodies to watch in 2014. Since 2010, mAbs has documented the biopharmaceutical industry\\'s progress in transitioning antibody therapeutics to first Phase 3 clinical studies and regulatory review, and its success at gaining first marketing approvals for antibody-based products. This installment of the \"Antibodies to watch\" series outlines events anticipated to occur between December 2013 and the end of 2014, including first regulatory actions on marketing applications for vedolizumab, siltuximab, and ramucirumab, as well as the Fc fusion proteins Factor IX-Fc and Factor VIII-Fc; and the submission of first marketing applications for up to five therapeutics (secukinumab, ch14.18, onartuzumab, necitumumab, gevokizumab). Antibody therapeutics in Phase 3 studies are described, with an emphasis on those with study completion dates in 2014, including antibodies targeting interleukin-17a or the interleukin-17a receptor (secukinumab, ixekizumab, brodalumab), proprotein convertase subtilisin/kexin type 9 (alirocumab, evolocumab, bococizumab), and programmed death 1 receptor (lambrolizumab, nivolumab). Five antibodies with US Food and Drug Administration\\'s Breakthrough Therapy designation (obinutuzumab, ofatumumab, lambrolizumab, bimagrumab, daratumumab) are also discussed.'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_005',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 283}]}],\n",
       "   'context': \"Efficacy and safety of longer-term administration of evolocumab (AMG 145) in patients with hypercholesterolemia: 52-week results from the Open-Label Study of Long-Term Evaluation Against LDL-C (OSLER) randomized trial. BACKGROUND: Evolocumab (AMG 145), a monoclonal antibody against proprotein convertase subtilisin/kexin type 9 (PCSK9), significantly reduced low-density lipoprotein cholesterol (LDL-C) in phase 2 studies of 12 weeks' duration. The longer-term efficacy and safety of PCSK9 inhibition remain undefined. METHODS AND RESULTS: Of 1359 randomized and dosed patients in the 4 evolocumab phase 2 parent studies, 1104 (81%) elected to enroll into the Open-Label Study of Long-term Evaluation Against LDL-C (OSLER) study. Regardless of their treatment assignment in the parent study, patients were randomized 2:1 to receive either open-label subcutaneous evolocumab 420 mg every 4 weeks with standard of care (SOC) (evolocumab+SOC, n=736) or SOC alone (n=368). Ninety-two percent of patients in the evolocumab+SOC group and 89% of patients in the SOC group completed 52 weeks of follow-up. Patients who first received evolocumab in OSLER experienced a mean 52.3% [SE, 1.8%] reduction in LDL-C at week 52 (P<0.0001). Patients who received 1 of 6 dosing regimens of evolocumab in the parent studies and received evolocumab+SOC in OSLER had persistent LDL-C reductions (mean reduction, 50.4% [SE, 0.8%] at the end of the parent study versus 52.1% [SE, 1.0%] at 52 weeks; P=0.31). In patients who discontinued evolocumab on entry into OSLER, LDL-C levels returned to near baseline levels. Adverse events and serious adverse events occurred in 81.4% and 7.1% of the evolocumab+SOC group patients and 73.1% and 6.3% of the SOC group patients, respectively. CONCLUSION: Evolocumab dosed every 4 weeks demonstrated continued efficacy and encouraging safety and tolerability over 1 year of treatment in the largest and longest evaluation of a PCSK9 inhibitor in hypercholesterolemic patients to date. CLINICAL TRIAL REGISTRATION URL: http://clinicaltrials.gov. Unique identifier: NCT01439880.\"},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_006',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 480}]}],\n",
       "   'context': 'Inhibition of PCSK9 with evolocumab in homozygous familial hypercholesterolaemia (TESLA Part B): a randomised, double-blind, placebo-controlled trial. BACKGROUND: Homozygous familial hypercholesterolaemia is a rare, serious disorder caused by very low or absent plasma clearance of LDL, substantially raised LDL cholesterol, and accelerated development of cardiovascular disease. Conventional lipid-lowering treatments are modestly effective. Evolocumab, a monoclonal antibody to proprotein convertase subtilisin/kexin type 9 (PCSK9), reduced LDL cholesterol by 16% in a pilot study. We now report results with evolocumab in a randomised, double-blind, placebo-controlled phase 3 trial. METHODS: This randomised, double-blind, placebo-controlled phase 3 trial was undertaken at 17 sites in ten countries in North America, Europe, the Middle East, and South Africa. 50 eligible patients (aged > 12 years) with homozygous familial hypercholesterolaemia, on stable lipid-regulating therapy for at least 4 weeks, and not receiving lipoprotein apheresis, were randomly allocated by a computer-generated randomisation sequence in a 2:1 ratio to receive subcutaneous evolocumab 420 mg or placebo every 4 weeks for 12 weeks. Randomisation was stratified by LDL cholesterol at screening (<11 mmol/L or > 11 mmol/L) and implemented by a computerised interactive voice-response system. Patients, study personnel, and the funder were masked to treatment and to the efficacy results by the central laboratory not returning LDL cholesterol or any lipid results to the clinical sites after the baseline visit. The primary endpoint was percentage change in ultracentrifugation LDL cholesterol from baseline at week 12 compared with placebo, analysed by intention-to-treat. This trial is registered with ClinicalTrials.gov, number NCT01588496. FINDINGS: Of the 50 eligible patients randomly assigned to the two treatment groups, 49 actually received the study drug and completed the study (16 in the placebo group and 33 in the evolocumab group). Compared with placebo, evolocumab significantly reduced ultracentrifugation LDL cholesterol at 12 weeks by 30·9% (95% CI -43·9% to -18·0%; p<0·0001). Treatment-emergent adverse events occurred in ten (63%) of 16 patients in the placebo group and 12 (36%) of 33 in the evolocumab group. No serious clinical or laboratory adverse events occurred, and no anti-evolocumab antibody development was detected during the study. INTERPRETATION: In patients with homozygous familial hypercholesterolaemia receiving stable background lipid-lowering treatment and not on apheresis, evolocumab 420 mg administered every 4 weeks was well tolerated and significantly reduced LDL cholesterol compared with placebo. FUNDING: Amgen Inc.'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_007',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 492}]}],\n",
       "   'context': 'Safety and tolerability of injectable lipid-lowering drugs: a review of available clinical data. INTRODUCTION: To answer the need of a better low-density lipoprotein (LDL) cholesterol control in statin-treated patients at high risk for cardiovascular disease, new injectable lipid-lowering drugs with innovative mechanisms of action are in advanced phase of development or have just been approved. AREAS COVERED: Evolocumab and alirocumab are fully human monoclonal antibodies inhibiting the proprotein convertase subtilisin/kexin type 9 (PCSK9) that binds to hepatic LDL receptor and prevents it from normal recycling by targeting it for degradation. Mipomersen specifically binds to a segment of the human apolipoprotein B100 messenger RNA, blocking the translation of the gene product. Phase II (for evolocumab and alirocumab) and III (for evolocumab) trials show that PCSK9 inhibitors are equally well tolerated, with adverse events mainly limited to mild-to-moderate nasopharyngitis, injection-site pain, arthralgia and back pain. Mipomersen use is mainly associated to hepatosteatosis, increased transaminases (> 3 times the upper limit of normal), mild-to-moderate injection-site reactions and flu-like symptoms. EXPERT OPINION: PCSK9 inhibitors have demonstrated their good safety and tolerability in a large number of subjects with different clinical conditions, including statin-intolerance, enlarging their potential use in a broader range of patients. Further data on long-term mipomersen safety are required.'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_008',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 1204}]}],\n",
       "   'context': 'Familial hypercholesterolemia: etiology, diagnosis and new treatment options. Familial hypercholesterolemia (FH) is a common genetic disorder that presents with robust increases in low-density lipoprotein cholesterol (LDL-C) and can lead to premature cardiovascular disease. There are heterozygous and homozygous forms. The diagnosis is usually made based on blood cholesterol levels, clinical signs and family history. Genetic testing can be used to confirm the diagnosis. Effective lowering of LDL-C in FH can prevent cardiovascular morbidity and mortality, however, the disease remains greatly underdiagnosed. The mainstay of pharmacologic therapy in FH patients is high-dose statins, which are often combined with other lipid-lowering agents. The homozygous form is mainly treated with lipid apheresis. Guideline-recommended target levels of LDL-C are often not reached, making new treatment options desirable. Four classes of newer lipid-lowering drugs offer promising advances in treating FH, namely the apolipoprotein-B synthesis inhibitors (mipomersen), the microsomal transfer protein inhibitors (lomitapide), the cholesterol ester transfer protein inhibitors (anacetrapib, evacetrapib) and the proprotein convertase subtilisin/kexin type 9 inhibitors (evolocumab, alirocumab). In this review, the available evidence regarding the use of these drugs in patients with FH is discussed, with particular focus on their efficacy and safety.'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_009',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 560}]}],\n",
       "   'context': 'Anti-PCSK9 antibody effectively lowers cholesterol in patients with statin intolerance: the GAUSS-2 randomized, placebo-controlled phase 3 clinical trial of evolocumab. OBJECTIVES: This study sought to evaluate the efficacy and safety of subcutaneous evolocumab compared with oral ezetimibe in\\xa0hypercholesterolemic patients who are unable to tolerate effective statin doses. BACKGROUND: Statin intolerance, which is predominantly due to muscle-related side effects, is reported in up to 10% to 20% of patients. Evolocumab, a fully human monoclonal antibody to proprotein convertase subtilisin/kexin type 9 (PCSK9), demonstrated marked reductions in plasma low-density lipoprotein cholesterol (LDL-C) in a phase 2 study in statin-intolerant patients. METHODS: The GAUSS-2 (Goal Achievement after Utilizing an Anti-PCSK9 Antibody in Statin Intolerant Subjects) trial was a 12-week, double-blind study of randomized patients (2:2:1:1) to evolocumab 140 mg every two weeks (Q2W) or evolocumab 420 mg once monthly (QM) both with daily oral placebo or subcutaneous placebo Q2W or QM both with daily oral ezetimibe 10 mg. Co-primary endpoints were percent change from baseline in LDL-C at the\\xa0mean of weeks 10 and\\xa012, and at week 12. RESULTS: Three hundred seven patients (age 62 ± 10 years; LDL-C 193 ± 59 mg/dl) were randomized. Evolocumab reduced LDL-C from baseline by 53% to 56%, corresponding to treatment differences versus ezetimibe of 37% to 39% (p\\xa0<0.001). Muscle adverse events occurred in 12% of evolocumab-treated patients and 23% of ezetimibe-treated patients. Treatment-emergent adverse events and laboratory abnormalities were comparable across treatment groups. CONCLUSIONS: Robust efficacy combined with favorable tolerability makes evolocumab a promising therapy for addressing the\\xa0largely unmet clinical need in high-risk patients with elevated cholesterol who are statin intolerant. (Goal Achievement After Utilizing an Anti-PCSK9 Antibody in Statin Intolerant Subjects-2; NCT01763905).'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_010',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 364}]}],\n",
       "   'context': 'Anti-PCSK9 monotherapy for hypercholesterolemia: the MENDEL-2 randomized, controlled phase III clinical trial of evolocumab. OBJECTIVES: The aim of this study was to compare biweekly and monthly evolocumab with placebo and oral ezetimibe in patients\\xa0with hypercholesterolemia in a phase III trial. BACKGROUND: Evolocumab, a fully human monoclonal antibody against proprotein convertase subtilisin/kexin type 9 (PCSK9), significantly reduced LDL-C in phase II trials. METHODS: Patients 18 to 80 years of age with fasting low-density lipoprotein cholesterol (LDL-C)\\xa0 > 100 and\\xa0<190 mg/dl and Framingham risk scores\\xa0 < 10% were randomized (1:1:1:1:2:2) to oral placebo and subcutaneous (SC) placebo biweekly; oral placebo and SC placebo monthly; ezetimibe and SC placebo biweekly; ezetimibe and SC placebo monthly; oral placebo and evolocumab 140 mg biweekly; or oral placebo and evolocumab 420 mg monthly. RESULTS: A total of 614 patients were randomized and administered doses. Evolocumab treatment reduced LDL-C from baseline, on average, by 55% to 57% more than placebo and 38% to 40% more than ezetimibe (p\\xa0< 0.001 for all comparisons). Evolocumab treatment also favorably altered other lipoprotein levels. Treatment-emergent adverse events (AEs), muscle-related AEs, and laboratory abnormalities were comparable across treatment groups. CONCLUSIONS: In the largest monotherapy trial using a PCSK9 inhibitor to date, evolocumab yielded significant LDL-C reductions compared with placebo or ezetimibe and was well tolerated in patients with hypercholesterolemia. (Monoclonal Antibody Against PCSK9 to Reduce Elevated LDL-C in Subjects Currently Not Receiving Drug Therapy for Easing Lipid\\xa0Levels-2 [MENDEL-2]; NCT01763827).'},\n",
       "  {'qas': [{'id': '54e0d1491388e8454a000014_011',\n",
       "     'question': 'Which enzyme is targeted by Evolocumab?',\n",
       "     'answers': [{'text': 'proprotein convertase subtilisin/kexin type 9',\n",
       "       'answer_start': 128}]}],\n",
       "   'context': 'A 52-week placebo-controlled trial of evolocumab in hyperlipidemia. BACKGROUND: Evolocumab, a monoclonal antibody that inhibits proprotein convertase subtilisin/kexin type 9 (PCSK9), significantly reduced low-density lipoprotein (LDL) cholesterol levels in phase 2 studies. We conducted a phase 3 trial to evaluate the safety and efficacy of 52 weeks of treatment with evolocumab. METHODS: We stratified patients with hyperlipidemia according to the risk categories outlined by the Adult Treatment Panel III of the National Cholesterol Education Program. On the basis of this classification, patients were started on background lipid-lowering therapy with diet alone or diet plus atorvastatin at a dose of 10 mg daily, atorvastatin at a dose of 80 mg daily, or atorvastatin at a dose of 80 mg daily plus ezetimibe at a dose of 10 mg daily, for a run-in period of 4 to 12 weeks. Patients with an LDL cholesterol level of 75 mg per deciliter (1.9 mmol per liter) or higher were then randomly assigned in a 2:1 ratio to receive either evolocumab (420 mg) or placebo every 4 weeks. The primary end point was the percent change from baseline in LDL cholesterol, as measured by means of ultracentrifugation, at week 52. RESULTS: Among the 901 patients included in the primary analysis, the overall least-squares mean (±SE) reduction in LDL cholesterol from baseline in the evolocumab group, taking into account the change in the placebo group, was 57.0±2.1% (P<0.001). The mean reduction was 55.7±4.2% among patients who underwent background therapy with diet alone, 61.6±2.6% among those who received 10 mg of atorvastatin, 56.8±5.3% among those who received 80 mg of atorvastatin, and 48.5±5.2% among those who received a combination of 80 mg of atorvastatin and 10 mg of ezetimibe (P<0.001 for all comparisons). Evolocumab treatment also significantly reduced levels of apolipoprotein B, non-high-density lipoprotein cholesterol, lipoprotein(a), and triglycerides. The most common adverse events were nasopharyngitis, upper respiratory tract infection, influenza, and back pain. CONCLUSIONS: At 52 weeks, evolocumab added to diet alone, to low-dose atorvastatin, or to high-dose atorvastatin with or without ezetimibe significantly reduced LDL cholesterol levels in patients with a range of cardiovascular risks. (Funded by Amgen; DESCARTES ClinicalTrials.gov number, NCT01516879.).'},\n",
       "  {'qas': [{'id': '51404dd723fec90375000002_001',\n",
       "     'question': 'What is the methyl donor of DNA (cytosine-5)-methyltransferases?',\n",
       "     'answers': [{'text': 'S-adenosyl-L-methionine', 'answer_start': 170}]}],\n",
       "   'context': 'Self-methylation of the M.BspRI methyltransferase. In the absence of DNA substrate, the DNA methyltransferase (MTase) M.BspRI can methylate itself using the methyl donor S-adenosyl-L-methionine (AdoMet). The methyl group is transferred to two Cys residues of the MTase.'},\n",
       "  {'qas': [{'id': '51404dd723fec90375000002_002',\n",
       "     'question': 'What is the methyl donor of DNA (cytosine-5)-methyltransferases?',\n",
       "     'answers': [{'text': 'S-adenosyl-L-methionine', 'answer_start': 165}]}],\n",
       "   'context': 'Self-methylation of BspRI DNA-methyltransferase. The DNA (cytosine-5)-methyltransferase (m5C-MTase) M.BspRI is able to accept the methyl group from the methyl donor S-adenosyl-L-methionine (AdoMet) in the absence of DNA. Transfer of the methyl group to the enzyme is a slow reaction relative to DNA methylation. Self-methylation is dependent on the native conformation of the enzyme and is inhibited by S-adenosyl-L-homocysteine, DNA and sulfhydryl reagents. Amino acid sequencing of proteolytic peptides obtained from M.BspRI, which had been methylated with [methyl-3H]AdoMet, and thin layer chromatography of the modified amino acid identified two cysteines, Cys156 and Cys181 that bind the methyl group in form of S-methylcysteine. One of the acceptor residues, Cys156 is the highly conserved cysteine which plays the role of the catalytic nucleophile of m5C-MTases.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_001',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 957}]}],\n",
       "   'context': 'Genetics of pituitary adenomas. Pituitary adenomas are common tumors of the adenohypophysis which can cause considerable morbidity, due to excessive hormonal secretion or compression and local invasion of surrounding structures. The vast majority of pituitary adenomas occur sporadically. Altered gene expression is commonly detected but somatic mutations, epigenetic changes and abnormal microRNAs have also been described. Occurrence of GNAS mutations at a postzygotic stage lead to McCune-Albright syndrome (MAS), a disease causing endocrine hyperfunction and tumors in several organs, including the pituitary. Familial pituitary adenomas occur as part of a syndrome affecting other organs, such as in MEN1 or Carney complex, or occur with pituitary adenomas only as in familial isolated pituitary adenoma (FIPA). FIPA, an autosomal-dominant disease with variable penetrance, is explained in 20% of patients by germline mutations in the tumor suppressor aryl hydrocarbon receptor interacting protein(AIP), while no gene abnormality has been identified to date in the majority of the FIPA families. AIP mutation-positive patients have a characteristic clinical phenotype with usually young- or childhood-onset growth hormone (GH) and/or prolactin (PRL)-secreting adenomas and can be seen in cases with no apparent family history as well. Understanding the tumorigenic process in AIP-positive and AIP-negative FIPA patients could result in better diagnostic and treatment options for both familial and sporadic cases.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_002',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 206}]}],\n",
       "   'context': 'Aryl hydrocarbon receptor interacting protein (AIP) mutations occur rarely in sporadic parathyroid adenomas. CONTEXT: The molecular pathogenesis of primary hyperparathyroidism is still largely unknown. The aryl hydrocarbon receptor interacting protein (AIP) gene has a major role in the pathogenesis of familial isolated pituitary adenoma. OBJECTIVE: We evaluated the involvement of the AIP gene in sporadic parathyroid adenomas. PATIENTS AND DESIGN: We performed direct sequencing and multiplex ligation-dependent probe amplification analyses of the AIP gene in a large series of sporadic parathyroid adenomas. Loss of heterozygosity (LOH) at the AIP locus was studied, and aryl hydrocarbon receptor interacting protein immunostaining was also performed. In addition, alterations in the MEN1 gene were studied. RESULTS: A somatic AIP mutation, substitution of arginine with glutamine at codon 304 (R304Q), was identified in 2 of 132 tumors. The mutation was germline in both cases despite the nonfamilial presentation. Heterozygous AIP large deletions were detected in 29 cases including 1 of the 2 mutated tumors, confirming a biallelic inactivation of the AIP gene. The AIP-mutated tumor with LOH showed decreased AIP immunostaining compared with normal parathyroid. LOH at the MEN1 locus, which often shared LOH at the AIP locus, was found in one third of tumors. Somatic MEN1 mutations were found in the 1 of the 2 AIP-mutated tumors and in 22 parathyroid adenomas. In addition, multiplex ligation-dependent probe amplification analysis revealed 1 large deletion of the MEN1 gene in 1 patient. CONCLUSIONS: The AIP gene is rarely involved in parathyroid adenomas, but the germline nature of the mutations suggests that it might predispose to primary hyperparathyroidism. MEN1 gene alterations occur in a substantial proportion of sporadic parathyroid adenomas.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_003',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 435}]}],\n",
       "   'context': 'Familial isolated pituitary adenomas: an emerging clinical entity. Familial pituitary tumors are increasingly recognized. While some of these cases are related to wellknown syndromic conditions such as multiple endocrine neoplasia type 1 (MEN1) or Carney complex, others belong to the familial isolated pituitary adenoma (FIPA) patient group. The discovery of heterozygous, loss-of-function germline mutations in the gene encoding the aryl hydrocarbon receptor interacting protein (AIP) in 2006 has subsequently enabled the identification of a mutation in this gene in 20% of FIPA families and 20% of childhood-onset simplex soma- totroph adenomas. The exact mechanism by which the lack of AIP leads to pituitary adenomas is not clear. AIP mutations cause a low penetrance autosomal dominant disease with often a distinct phenotype characterized by young-onset, aggressive, large GH, mixed GH and PRL or PRL-secreting adenomas. This review aims to summarize currently available clinical data on AIP mutation-positive and negative FIPA patients.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_004',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'Aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 1}]}],\n",
       "   'context': '[Aryl hydrocarbon receptor interacting protein gene and familial isolated pituitary adenomas]. Familial isolated pituitary adenoma (FIPA) is an autosomal dominant disease, characterized by low penetrance, early-onset disease, more invasive tumor growth, as well as somatotroph and lactotroph adenomas in most cases. It has been indicated that the aryl hydrocarbon receptor interacting protein (AIP) gene is a tumor suppressor gene. Many heterozygous mutations have been discovered in AIP in about 20% of FIPA families. However, the exact molecular mechanism by which its disfunction promotes tumorigenesis of pituitary is unclear.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_005',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 468}]}],\n",
       "   'context': '[Familial isolated pituitary adenoma syndrome]. Familial pituitary adenomas occur in multiple endocrine neoplasia type 1, Carney complex, as well as in familial isolated pituitary adenoma syndrome. Familial isolated pituitary adenoma syndrome is an autosomal dominant disease with incomplete penetrance. Pituitary adenomas occur in familial setting but without any other specific tumors. In 20-40% of families with this syndrome, mutations have been identified in the aryl hydrocarbon receptor interacting protein gene while in the rest of the families the causative gene or genes have not been identified. Families carrying aryl hydrocarbon receptor interacting protein gene mutations have a distinct phenotype with younger age at diagnosis and a predominance of somatotroph and lactotroph adenomas. Germline mutations of the aryl hydrocarbon receptor interacting protein gene can be occasionally identified in usually young-onset seemingly sporadic cases. Genetic and clinical testing of relatives of patients with aryl hydrocarbon receptor interacting protein gene mutations can lead to earlier diagnosis and treatment at an earlier stage of the pituitary tumor.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_006',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'Aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 112}]}],\n",
       "   'context': 'Familial isolated pituitary adenoma: evidence for genetic heterogeneity. The identification of mutations in the Aryl hydrocarbon receptor interacting protein (AIP) gene in a subset of familial isolated pituitary adenoma (FIPA) cases has recently expanded our understanding of the pathophysiology of inherited pituitary adenoma disorders. However, a genetic cause of has not yet been determined in the majority (85%) of FIPA families and half of the families with isolated familial somatotropinoma. Several studies and reviews have assessed the genetic and clinical features of AIP-mutated FIPA patients, which range from a complete lack of symptoms in adult/elderly individuals to large, aggressive early-onset pituitary tumors. In this study, we aimed to briefly revise the data available for the 11q13 locus and other additional loci that have been implicated in genetic susceptibility to FIPA: 2p16-12; 3q28; 4q32.3-4q33; chr 5, 8q12.1, chr 14, 19q13.4 and 21q22.1. These candidate regions may contain unidentified gene(s) that can be potentially disrupted in AIP-negative FIPA families. A better knowledge of these susceptibility loci may disclose modifier genes that are likely to play exacerbating or protective roles in the phenotypic diversity of AIP-mutated families.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_007',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 109}]}],\n",
       "   'context': 'Familial isolated pituitary adenomas (FIPA) and the pituitary adenoma predisposition due to mutations in the aryl hydrocarbon receptor interacting protein (AIP) gene. Pituitary adenomas are one of the most frequent intracranial tumors and occur with a prevalence of approximately 1:1000 in the developed world. Pituitary adenomas have a serious disease burden, and their management involves neurosurgery, biological therapies, and radiotherapy. Early diagnosis of pituitary tumors while they are smaller may help increase cure rates. Few genetic predictors of pituitary adenoma development exist. Recent years have seen two separate, complimentary advances in inherited pituitary tumor research. The clinical condition of familial isolated pituitary adenomas (FIPA) has been described, which encompasses the familial occurrence of isolated pituitary adenomas outside of the setting of syndromic conditions like multiple endocrine neoplasia type 1 and Carney complex. FIPA families comprise approximately 2% of pituitary adenomas and represent a clinical entity with homogeneous or heterogeneous pituitary adenoma types occurring within the same kindred. The aryl hydrocarbon receptor interacting protein (AIP) gene has been identified as causing a pituitary adenoma predisposition of variable penetrance that accounts for 20% of FIPA families. Germline AIP mutations have been shown to associate with the occurrence of large pituitary adenomas that occur at a young age, predominantly in children/adolescents and young adults. AIP mutations are usually associated with somatotropinomas, but prolactinomas, nonfunctioning pituitary adenomas, Cushing disease, and other infrequent clinical adenoma types can also occur. Gigantism is a particular feature of AIP mutations and occurs in more than one third of affected somatotropinoma patients. Study of pituitary adenoma patients with AIP mutations has demonstrated that these cases raise clinical challenges to successful treatment. Extensive research on the biology of AIP and new advances in mouse Aip knockout models demonstrate multiple pathways by which AIP may contribute to tumorigenesis. This review assesses the current clinical and therapeutic characteristics of more than 200 FIPA families and addresses research findings among AIP mutation-bearing patients in different populations with pituitary adenomas.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_008',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 145}]}],\n",
       "   'context': 'Hyperplasia-adenoma sequence in pituitary tumorigenesis related to aryl hydrocarbon receptor interacting protein gene mutation. Mutations of the aryl hydrocarbon receptor interacting protein (AIP) gene are associated with pituitary adenomas that usually occur as familial isolated pituitary adenomas (FIPA). Detailed pathological and tumor genetic data on AIP mutation-related pituitary adenomas are not sufficient. Non-identical twin females presented as adolescents to the emergency department with severe progressive headache caused by large pituitary macroadenomas require emergency neurosurgery; one patient had incipient pituitary apoplexy. Post-surgically, the patients were found to have silent somatotrope adenomas on pathological examination. Furthermore, the light microscopic, immunohistochemical, and electron microscopic studies demonstrated tumors of virtually identical characteristics. The adenomas were accompanied by multiple areas of pituitary hyperplasia, which stained positively for GH, indicating somatotrope hyperplasia. Genetic analyses of the FIPA kindred revealed a novel E216X mutation of the AIP gene, which was present in both the affected patients and the unaffected father. Molecular analysis of surgical specimens revealed loss of heterozygosity (LOH) in the adenoma but showed that LOH was not present in the hyperplastic pituitary tissue from either patient. AIP immunostaining confirmed normal staining in the hyperplastic tissue and decreased staining in the adenoma in the tumors from both patients. These results demonstrate that patients with AIP germline mutation can present with silent somatotrope pituitary adenomas. The finding of somatotrope hyperplasia unaccompanied by AIP LOH suggests that LOH at the AIP locus might be a late event in a potential progression from hyperplastic to adenomatous tissue.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_009',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 122}]}],\n",
       "   'context': 'Clinical experience in the screening and management of a large kindred with familial isolated pituitary adenoma due to an aryl hydrocarbon receptor interacting protein (AIP) mutation. CONTEXT: Germline AIP mutations usually cause young-onset acromegaly with low penetrance in a subset of familial isolated pituitary adenoma families. We describe our experience with a large family with R304* AIP mutation and discuss some of the diagnostic dilemmas and management issues. OBJECTIVE: The aim of the study was to identify and screen mutation carriers in the family. PATIENTS: Forty-three family members participated in the study. SETTING: The study was performed in university hospitals. OUTCOME: We conducted genetic and endocrine screening of family members. RESULTS: We identified 18 carriers of the R304* mutation, three family members with an AIP-variant A299V, and two family members who harbored both changes. One of the two index cases presented with gigantism and pituitary apoplexy, the other presented with young-onset acromegaly, and both had surgery and radiotherapy. After genetic and clinical screening of the family, two R304* carriers were diagnosed with acromegaly. They underwent transsphenoidal surgery after a short period of somatostatin analog treatment. One of these two patients is in remission; the other achieved successful pregnancy despite suboptimal control of acromegaly. One of the A299V carrier family members was previously diagnosed with a microprolactinoma; we consider this case to be a phenocopy. Height of the unaffected R304* carrier family members is not different compared to noncarrier relatives. CONCLUSIONS: Families with AIP mutations present particular problems such as the occurrence of large invasive tumors, poor response to medical treatment, difficulties with fertility and management of pregnancy, and the finding of AIP sequence variants of unknown significance. Because disease mostly develops at a younger age and penetrance is low, the timing and duration of the follow-up of carriers without overt disease requires further study. The psychological and financial impact of prolonged clinical screening must be considered. Excellent relationships between the family, endocrinologists, and geneticists are essential, and ideally these families should be managed in centers with specialist expertise.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_010',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 957}]}],\n",
       "   'context': 'Genetics of pituitary adenomas. Pituitary adenomas are common tumors of the adenohypophysis which can cause considerable morbidity, due to excessive hormonal secretion or compression and local invasion of surrounding structures. The vast majority of pituitary adenomas occur sporadically. Altered gene expression is commonly detected but somatic mutations, epigenetic changes and abnormal microRNAs have also been described. Occurrence of GNAS mutations at a postzygotic stage lead to McCune-Albright syndrome (MAS), a disease causing endocrine hyperfunction and tumors in several organs, including the pituitary. Familial pituitary adenomas occur as part of a syndrome affecting other organs, such as in MEN1 or Carney complex, or occur with pituitary adenomas only as in familial isolated pituitary adenoma (FIPA). FIPA, an autosomal-dominant disease with variable penetrance, is explained in 20% of patients by germline mutations in the tumor suppressor aryl hydrocarbon receptor interacting protein(AIP), while no gene abnormality has been identified to date in the majority of the FIPA families. AIP mutation-positive patients have a characteristic clinical phenotype with usually young- or childhood-onset growth hormone (GH) and/or prolactin (PRL)-secreting adenomas and can be seen in cases with no apparent family history as well. Understanding the tumorigenic process in AIP-positive and AIP-negative FIPA patients could result in better diagnostic and treatment options for both familial and sporadic cases.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_011',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'Aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 112}]}],\n",
       "   'context': 'Familial isolated pituitary adenoma: evidence for genetic heterogeneity. The identification of mutations in the Aryl hydrocarbon receptor interacting protein (AIP) gene in a subset of familial isolated pituitary adenoma (FIPA) cases has recently expanded our understanding of the pathophysiology of inherited pituitary adenoma disorders. However, a genetic cause of has not yet been determined in the majority (85%) of FIPA families and half of the families with isolated familial somatotropinoma. Several studies and reviews have assessed the genetic and clinical features of AIP-mutated FIPA patients, which range from a complete lack of symptoms in adult/elderly individuals to large, aggressive early-onset pituitary tumors. In this study, we aimed to briefly revise the data available for the 11q13 locus and other additional loci that have been implicated in genetic susceptibility to FIPA: 2p16-12; 3q28; 4q32.3-4q33; chr 5, 8q12.1, chr 14, 19q13.4 and 21q22.1. These candidate regions may contain unidentified gene(s) that can be potentially disrupted in AIP-negative FIPA families. A better knowledge of these susceptibility loci may disclose modifier genes that are likely to play exacerbating or protective roles in the phenotypic diversity of AIP-mutated families.'},\n",
       "  {'qas': [{'id': '551c23bc6b348bb82c00000b_012',\n",
       "     'question': 'Mutation of which gene is implicated in the familial isolated pituitary adenoma?',\n",
       "     'answers': [{'text': 'aryl hydrocarbon receptor interacting protein',\n",
       "       'answer_start': 1038}]}],\n",
       "   'context': 'Molecular and trophic mechanisms of pituitary tumourigenesis. BACKGROUND: The paradox of pituitary tumours is that persistent growth is so atypical. By definition, all pituitary microadenomas regain complete trophic stability after an initial period of deregulated growth. Unlike tumours in many other organ systems, concern about significant growth of macroadenoma remnants after debulking is minimal. Despite reports of a relatively high prevalence of aneuploidy and clonal skewing in these tumours, prolonged efforts to implicate classical proto-oncogene activation and tumour suppressor mutations have been of limited success. No histological or molecular markers reliably predict behaviour. To date, the number of molecular genetic factors unequivocally linked to pituitary tumours can be counted on the fingers of one hand: (1) GNAS1 activation in acromegaly; (2) the MENIN and p27Kip1 (CDKN1B) mutations associated with multiple endocrine neoplasia type 1; (3) mutations of PRKA1RA with loss of 17q22-24 in Carney complex, and (4) aryl hydrocarbon receptor interacting protein gene mutations in 15% of familial isolated pituitary adenomas and 50% of familial isolated acromegaly. Together, these account for only a small proportion (<5%) of sporadic pituitary macroadenomas. CONCLUSION: In most instances, we still do not know what causes quantitative aberrations in trophic behaviour.'},\n",
       "  {'qas': [{'id': '56c1f02cef6e39474100004c_001',\n",
       "     'question': 'GV1001 vaccine targets which enzyme?',\n",
       "     'answers': [{'text': 'human telomerase reverse transcriptase',\n",
       "       'answer_start': 113}]}],\n",
       "   'context': 'Novel vaccine peptide GV1001 effectively blocks β-amyloid toxicity by mimicking the extra-telomeric functions of human telomerase reverse transcriptase. GV1001 is a 16-amino-acid vaccine peptide derived from the human telomerase reverse transcriptase sequence. We investigated the effects of GV1001 against β-amyloid (Aβ) oligomer-induced neurotoxicity in rat neural stem cells (NSCs). Primary culture NSCs were treated with several concentrations of GV1001 and/or Aβ₂₅₋₃₅ oligomer for 48 hours. GV1001 protected NSCs against the Aβ₂₅₋₃₅ oligomer in a concentration-dependent manner. Aβ₂₅₋₃₅ concentration dependently decreased viability, proliferation, and mobilization of NSCs and GV1001 treatment restored the cells to wild-type levels. Aβ₂₅₋₃₅ increased free radical levels in rat NSCs while combined treatment with GV1001 significantly reduced these levels. In addition, GV1001 treatment of Aβ₂₅₋₃₅-injured NSCs increased the expression level of survival-related proteins, including mitochondria-associated survival proteins, and decreased the levels of death and inflammation-related proteins, including mitochondria-associated death proteins. Together, these results suggest that GV1001 possesses neuroprotective effects against Aβ₂₅₋₃₅ oligomer in NSCs and that these effects are mediated through mimicking the extra-telomeric functions of human telomerase reverse transcriptase, including the induction of cellular proliferation, anti-apoptotic effects, mitochondrial stabilization, and anti-aging and anti-oxidant effects.'},\n",
       "  {'qas': [{'id': '56c1f02cef6e39474100004c_002',\n",
       "     'question': 'GV1001 vaccine targets which enzyme?',\n",
       "     'answers': [{'text': 'human telomerase reverse transcriptase',\n",
       "       'answer_start': 212}]}],\n",
       "   'context': 'Novel vaccine peptide GV1001 effectively blocks β-amyloid toxicity by mimicking the extra-telomeric functions of human telomerase reverse transcriptase. GV1001 is a 16-amino-acid vaccine peptide derived from the human telomerase reverse transcriptase sequence. We investigated the effects of GV1001 against β-amyloid (Aβ) oligomer-induced neurotoxicity in rat neural stem cells (NSCs). Primary culture NSCs were treated with several concentrations of GV1001 and/or Aβ₂₅₋₃₅ oligomer for 48 hours. GV1001 protected NSCs against the Aβ₂₅₋₃₅ oligomer in a concentration-dependent manner. Aβ₂₅₋₃₅ concentration dependently decreased viability, proliferation, and mobilization of NSCs and GV1001 treatment restored the cells to wild-type levels. Aβ₂₅₋₃₅ increased free radical levels in rat NSCs while combined treatment with GV1001 significantly reduced these levels. In addition, GV1001 treatment of Aβ₂₅₋₃₅-injured NSCs increased the expression level of survival-related proteins, including mitochondria-associated survival proteins, and decreased the levels of death and inflammation-related proteins, including mitochondria-associated death proteins. Together, these results suggest that GV1001 possesses neuroprotective effects against Aβ₂₅₋₃₅ oligomer in NSCs and that these effects are mediated through mimicking the extra-telomeric functions of human telomerase reverse transcriptase, including the induction of cellular proliferation, anti-apoptotic effects, mitochondrial stabilization, and anti-aging and anti-oxidant effects.'},\n",
       "  {'qas': [{'id': '56c1f02cef6e39474100004c_003',\n",
       "     'question': 'GV1001 vaccine targets which enzyme?',\n",
       "     'answers': [{'text': 'human telomerase reverse transcriptase',\n",
       "       'answer_start': 1348}]}],\n",
       "   'context': 'Novel vaccine peptide GV1001 effectively blocks β-amyloid toxicity by mimicking the extra-telomeric functions of human telomerase reverse transcriptase. GV1001 is a 16-amino-acid vaccine peptide derived from the human telomerase reverse transcriptase sequence. We investigated the effects of GV1001 against β-amyloid (Aβ) oligomer-induced neurotoxicity in rat neural stem cells (NSCs). Primary culture NSCs were treated with several concentrations of GV1001 and/or Aβ₂₅₋₃₅ oligomer for 48 hours. GV1001 protected NSCs against the Aβ₂₅₋₃₅ oligomer in a concentration-dependent manner. Aβ₂₅₋₃₅ concentration dependently decreased viability, proliferation, and mobilization of NSCs and GV1001 treatment restored the cells to wild-type levels. Aβ₂₅₋₃₅ increased free radical levels in rat NSCs while combined treatment with GV1001 significantly reduced these levels. In addition, GV1001 treatment of Aβ₂₅₋₃₅-injured NSCs increased the expression level of survival-related proteins, including mitochondria-associated survival proteins, and decreased the levels of death and inflammation-related proteins, including mitochondria-associated death proteins. Together, these results suggest that GV1001 possesses neuroprotective effects against Aβ₂₅₋₃₅ oligomer in NSCs and that these effects are mediated through mimicking the extra-telomeric functions of human telomerase reverse transcriptase, including the induction of cellular proliferation, anti-apoptotic effects, mitochondrial stabilization, and anti-aging and anti-oxidant effects.'},\n",
       "  {'qas': [{'id': '56c1f02cef6e39474100004c_004',\n",
       "     'question': 'GV1001 vaccine targets which enzyme?',\n",
       "     'answers': [{'text': 'human telomerase reverse transcriptase',\n",
       "       'answer_start': 253}]}],\n",
       "   'context': 'Protective effect of peptide GV1001 against renal ischemia-reperfusion injury in mice. BACKGROUND: Ischemia reperfusion injury (IRI) is a common complication after kidney transplantation. Peptide GV1001 is a peptide vaccine representing a 16-amino acid human telomerase reverse transcriptase sequence, which has been reported to possess potential antineoplastic and anti-inflammatory activity. This study aimed to investigate the potential effects of peptide GV1001 on renal IRI. METHODS: Peptide GV1001 was subcutaneously administered to C57BL6/J mice 30 minutes before and 12 hours after bilateral IRI. Sham operation and phosphate-buffered saline (PBS) injection were used as controls. Blood and renal tissues were harvested at 1 day after IRI. RESULTS: Peptide GV1001 treatment significantly attenuated renal functional deterioration after IRI (peptide GV1001 group vs PBS group; blood urea nitrogen, P < .05; creatinine, P < .05). Peptide GV1001 treatment also attenuated renal tissue injury (tubular injury score; the peptide GV1001 group vs PBS group; P < .001). Renal apoptosis was also lower in the peptide GV1001 group. Immunohistochemical studies showed that IRI increased perirenal infiltration of both neutrophils and macrophages, and that peptide GV1001 significantly attenuated this process. Expression of interleukin-6 and monocyte chemotactic protein-1 was significantly reduced by peptide GV1001 treatment. CONCLUSIONS: Peptide GV1001 ameliorates acute renal IRI by reducing inflammation and apoptosis; therefore, it is promising as a potential therapeutic agent for renal IRI. The mechanisms of protection should be explored in further studies.'},\n",
       "  {'qas': [{'id': '56c1f02cef6e39474100004c_005',\n",
       "     'question': 'GV1001 vaccine targets which enzyme?',\n",
       "     'answers': [{'text': 'human telomerase reverse transcriptase',\n",
       "       'answer_start': 253}]}],\n",
       "   'context': 'Protective effect of peptide GV1001 against renal ischemia-reperfusion injury in mice. BACKGROUND: Ischemia reperfusion injury (IRI) is a common complication after kidney transplantation. Peptide GV1001 is a peptide vaccine representing a 16-amino acid human telomerase reverse transcriptase sequence, which has been reported to possess potential antineoplastic and anti-inflammatory activity. This study aimed to investigate the potential effects of peptide GV1001 on renal IRI. METHODS: Peptide GV1001 was subcutaneously administered to C57BL6/J mice 30 minutes before and 12 hours after bilateral IRI. Sham operation and phosphate-buffered saline (PBS) injection were used as controls. Blood and renal tissues were harvested at 1 day after IRI. RESULTS: Peptide GV1001 treatment significantly attenuated renal functional deterioration after IRI (peptide GV1001 group vs PBS group; blood urea nitrogen, P < .05; creatinine, P < .05). Peptide GV1001 treatment also attenuated renal tissue injury (tubular injury score; the peptide GV1001 group vs PBS group; P < .001). Renal apoptosis was also lower in the peptide GV1001 group. Immunohistochemical studies showed that IRI increased perirenal infiltration of both neutrophils and macrophages, and that peptide GV1001 significantly attenuated this process. Expression of interleukin-6 and monocyte chemotactic protein-1 was significantly reduced by peptide GV1001 treatment. CONCLUSIONS: Peptide GV1001 ameliorates acute renal IRI by reducing inflammation and apoptosis; therefore, it is promising as a potential therapeutic agent for renal IRI. The mechanisms of protection should be explored in further studies.'},\n",
       "  {'qas': [{'id': '56c1f02cef6e39474100004c_006',\n",
       "     'question': 'GV1001 vaccine targets which enzyme?',\n",
       "     'answers': [{'text': 'Human telomerase reverse transcriptase',\n",
       "       'answer_start': 265}]}],\n",
       "   'context': 'Cancer vaccination with telomerase peptide GV1001. Telomerase is highly expressed in essentially all cancer forms, while the expression in normal tissues is restricted. Moreover, telomerase activity is considered indispensable for tumor immortalization and growth. Human telomerase reverse transcriptase (hTERT), the rate-limiting subunit of the telomerase complex, is therefore an attractive target for cancer vaccination. The present review provides an update on the development of GV1001, a peptide vaccine representing a 16-aa hTERT sequence. GV1001 binds multiple HLA class II molecules and harbors putative HLA class I epitopes. The peptide may therefore elicit combined CD4/CD8 T-cell responses, considered important to initiate tumor eradication and long-term memory. Phase I/II trials in advanced pancreatic and pulmonary cancer patients have demonstrated GV1001-specific T-cell responses in > 50% of subjects, without clinically important toxicity. The results indicate a correlation between development of GV1001-specific responses and prolonged survival. However, as in most cancer vaccine trials, a large proportion of immune responders experience no clinical benefit. Long-term survivors harbor durable GV1001-specific T-cell responses with high IFN-gamma/IL-10 ratios and polyfunctional cytokine patterns. Interestingly, the cytokine profiles do not follow a T(H)1/T(H)2 delineation. Here, the author discusses how immunomonitoring may be improved to discriminate between efficient and pointless immune responses, and which questions to address in the further development of GV1001.'},\n",
       "  {'qas': [{'id': '56c1f02cef6e39474100004c_007',\n",
       "     'question': 'GV1001 vaccine targets which enzyme?',\n",
       "     'answers': [{'text': 'human telomerase reverse transcriptase',\n",
       "       'answer_start': 212}]}],\n",
       "   'context': 'Novel vaccine peptide GV1001 effectively blocks β-amyloid toxicity by mimicking the extra-telomeric functions of human telomerase reverse transcriptase. GV1001 is a 16-amino-acid vaccine peptide derived from the human telomerase reverse transcriptase sequence. We investigated the effects of GV1001 against β-amyloid (Aβ) oligomer-induced neurotoxicity in rat neural stem cells (NSCs). Primary culture NSCs were treated with several concentrations of GV1001 and/or Aβ₂₅₋₃₅ oligomer for 48 hours. GV1001 protected NSCs against the Aβ₂₅₋₃₅ oligomer in a concentration-dependent manner. Aβ₂₅₋₃₅ concentration dependently decreased viability, proliferation, and mobilization of NSCs and GV1001 treatment restored the cells to wild-type levels. Aβ₂₅₋₃₅ increased free radical levels in rat NSCs while combined treatment with GV1001 significantly reduced these levels. In addition, GV1001 treatment of Aβ₂₅₋₃₅-injured NSCs increased the expression level of survival-related proteins, including mitochondria-associated survival proteins, and decreased the levels of death and inflammation-related proteins, including mitochondria-associated death proteins. Together, these results suggest that GV1001 possesses neuroprotective effects against Aβ₂₅₋₃₅ oligomer in NSCs and that these effects are mediated through mimicking the extra-telomeric functions of human telomerase reverse transcriptase, including the induction of cellular proliferation, anti-apoptotic effects, mitochondrial stabilization, and anti-aging and anti-oxidant effects.'},\n",
       "  {'qas': [{'id': '550af222c2af5d5b7000000b_001',\n",
       "     'question': 'Which is the E3 ubiquitin ligase which ubiquitinates IkB leading to its proteasomal degradation?',\n",
       "     'answers': [{'text': 'SCF(β-TrCP)', 'answer_start': 799}]}],\n",
       "   'context': 'Regulation of NF-κB by ubiquitination and degradation of the IκBs. The nuclear factor-κB (NF-κB) signaling pathway is a busy ground for the action of the ubiquitin-proteasome system; many of the signaling steps are coordinated by protein ubiquitination. The end point of this pathway is to induce transcription, and to this end, there is a need to overcome a major obstacle, a set of inhibitors (IκBs) that bind NF-κB and prohibit either the nuclear entry or the DNA binding of the transcription factor. Two major signaling steps are required for the elimination of the inhibitors: activation of the IκB kinase (IKK) and degradation of the phosphorylated inhibitors. IKK activation and IκB degradation involve different ubiquitination modes; the latter is mediated by a specific E3 ubiquitin ligase SCF(β-TrCP) . The F-box component of this E3, β-TrCP, recognizes the IκB degron formed following phosphorylation by IKK and thus couples IκB phosphorylation to ubiquitination. SCF(β-TrCP) -mediated IκB ubiquitination and degradation is a very efficient process, often resulting in complete degradation of the key inhibitor IκBα within a few minutes of cell stimulation. In vivo ablation of β-TrCP results in accumulation of all the IκBs and complete NF-κB inhibition. As many details of IκB-β-TrCP interaction have been worked out, the development of β-TrCP inhibitors might be a feasible therapeutic approach for NF-κB-associated human disease. However, we may still need to advance our understanding of the mechanism of IκB degradation as well as of the diverse functions of β-TrCP in vivo.'},\n",
       "  {'qas': [{'id': '550af222c2af5d5b7000000b_002',\n",
       "     'question': 'Which is the E3 ubiquitin ligase which ubiquitinates IkB leading to its proteasomal degradation?',\n",
       "     'answers': [{'text': 'SCF(β-TrCP)', 'answer_start': 975}]}],\n",
       "   'context': 'Regulation of NF-κB by ubiquitination and degradation of the IκBs. The nuclear factor-κB (NF-κB) signaling pathway is a busy ground for the action of the ubiquitin-proteasome system; many of the signaling steps are coordinated by protein ubiquitination. The end point of this pathway is to induce transcription, and to this end, there is a need to overcome a major obstacle, a set of inhibitors (IκBs) that bind NF-κB and prohibit either the nuclear entry or the DNA binding of the transcription factor. Two major signaling steps are required for the elimination of the inhibitors: activation of the IκB kinase (IKK) and degradation of the phosphorylated inhibitors. IKK activation and IκB degradation involve different ubiquitination modes; the latter is mediated by a specific E3 ubiquitin ligase SCF(β-TrCP) . The F-box component of this E3, β-TrCP, recognizes the IκB degron formed following phosphorylation by IKK and thus couples IκB phosphorylation to ubiquitination. SCF(β-TrCP) -mediated IκB ubiquitination and degradation is a very efficient process, often resulting in complete degradation of the key inhibitor IκBα within a few minutes of cell stimulation. In vivo ablation of β-TrCP results in accumulation of all the IκBs and complete NF-κB inhibition. As many details of IκB-β-TrCP interaction have been worked out, the development of β-TrCP inhibitors might be a feasible therapeutic approach for NF-κB-associated human disease. However, we may still need to advance our understanding of the mechanism of IκB degradation as well as of the diverse functions of β-TrCP in vivo.'},\n",
       "  {'qas': [{'id': '550af222c2af5d5b7000000b_003',\n",
       "     'question': 'Which is the E3 ubiquitin ligase which ubiquitinates IkB leading to its proteasomal degradation?',\n",
       "     'answers': [{'text': 'beta-TrCP', 'answer_start': 710}]}],\n",
       "   'context': 'Identification of clusterin domain involved in NF-kappaB pathway regulation. Clusterin (CLU) is a ubiquitous protein that has been implicated in tumorigenesis, apoptosis, inflammation, and cell proliferation. We and others have previously shown that CLU is an inhibitor of the NF-kappaB pathway. However, the exact form of CLU and the region(s) of CLU involved in this effect were unknown. Using newly generated molecular constructs encoding for CLU and various regions of the molecule, we demonstrated that the presecretory form of CLU (psCLU) form bears the NF-kappaB regulatory activity. Sequence comparison analysis showed sequence motif identity between CLU and beta-transducin repeat-containing protein (beta-TrCP), a main E3 ubiquitin ligase involved in IkappaB-alpha degradation. These homologies were localized in the disulfide constraint region of CLU. We generated a specific molecular construct of this region, named DeltaCLU, and showed that it has the same NF-kappaB regulatory activity as CLU. Neither the alpha-chain nor the beta-chain of CLU had any NF-kappaB regulatory activity. Furthermore, we showed that following tumor necrosis factor-alpha stimulation of transfected cells, we could co-immunoprecipitate phospho-IkappaB-alpha with DeltaCLU. Moreover, we showed that DeltaCLU could localize both in the cytoplasm and in the nucleus. These results demonstrate the identification of a new CLU activity site involved in NF-kappaB pathway regulation.'},\n",
       "  {'qas': [{'id': '550af222c2af5d5b7000000b_004',\n",
       "     'question': 'Which is the E3 ubiquitin ligase which ubiquitinates IkB leading to its proteasomal degradation?',\n",
       "     'answers': [{'text': 'beta-Trcp', 'answer_start': 612}]}],\n",
       "   'context': 'Oncogenic BRAF regulates beta-Trcp expression and NF-kappaB activity in human melanoma cells. Mutational activation of BRAF is a frequent event in human malignant melanomas suggesting that BRAF-dependent signaling is conducive to melanoma cell growth and survival. Previously published work reported that melanoma cells exhibit constitutive anti-apoptotic nuclear factor kappaB (NF-kappaB) transcription factor activation triggered by proteolysis of its inhibitor IkappaB. IkappaB degradation is dependent upon its phosphorylation by the IkappaB kinase (IKK) complex and subsequent ubiquitination facilitated by beta-Trcp E3 ubiquitin ligase. Here, we report that melanocytes expressing a conditionally oncogenic form of BRAF(V600E) exhibit enhanced beta-Trcp expression, increased IKK activity and a concomitant increase in the rate of IkappaBalpha degradation. Conversely, inhibition of BRAF signaling using either a broad-spectrum Raf inhibitor (BAY 43-9006) or by selective knock-down of BRAF(V600E) expression by RNA interference in human melanoma cells leads to decreased IKK activity and beta-Trcp expression, stabilization of IkappaB, inhibition of NF-kappaB transcriptional activity and sensitization of these cells to apoptosis. Taken together, these data support a model in which mutational activation of BRAF in human melanomas contributes to constitutive induction of NF-kappaB activity and to increased survival of melanoma cells.'},\n",
       "  {'qas': [{'id': '550af222c2af5d5b7000000b_005',\n",
       "     'question': 'Which is the E3 ubiquitin ligase which ubiquitinates IkB leading to its proteasomal degradation?',\n",
       "     'answers': [{'text': 'beta-TrCP', 'answer_start': 2076}]}],\n",
       "   'context': 'Phosphorylation meets ubiquitination: the control of NF-[kappa]B activity. NF-kappaB (nuclear factor-kappaB) is a collective name for inducible dimeric transcription factors composed of members of the Rel family of DNA-binding proteins that recognize a common sequence motif. NF-kappaB is found in essentially all cell types and is involved in activation of an exceptionally large number of genes in response to infections, inflammation, and other stressful situations requiring rapid reprogramming of gene expression. NF-kappaB is normally sequestered in the cytoplasm of nonstimulated cells and consequently must be translocated into the nucleus to function. The subcellular location of NF-kappaB is controlled by a family of inhibitory proteins, IkappaBs, which bind NF-kappaB and mask its nuclear localization signal, thereby preventing nuclear uptake. Exposure of cells to a variety of extracellular stimuli leads to the rapid phosphorylation, ubiquitination, and ultimately proteolytic degradation of IkappaB, which frees NF-kappaB to translocate to the nucleus where it regulates gene transcription. NF-kappaB activation represents a paradigm for controlling the function of a regulatory protein via ubiquitination-dependent proteolysis, as an integral part of a phosphorylationbased signaling cascade. Recently, considerable progress has been made in understanding the details of the signaling pathways that regulate NF-kappaB activity, particularly those responding to the proinflammatory cytokines tumor necrosis factor-alpha and interleukin-1. The multisubunit IkappaB kinase (IKK) responsible for inducible IkappaB phosphorylation is the point of convergence for most NF-kappaB-activating stimuli. IKK contains two catalytic subunits, IKKalpha and IKKbeta, both of which are able to correctly phosphorylate IkappaB. Gene knockout studies have shed light on the very different physiological functions of IKKalpha and IKKbeta. After phosphorylation, the IKK phosphoacceptor sites on IkappaB serve as an essential part of a specific recognition site for E3RS(IkappaB/beta-TrCP), an SCF-type E3 ubiquitin ligase, thereby explaining how IKK controls IkappaB ubiquitination and degradation. A variety of other signaling events, including phosphorylation of NF-kappaB, hyperphosphorylation of IKK, induction of IkappaB synthesis, and the processing of NF-kappaB precursors, provide additional mechanisms that modulate the level and duration of NF-kappaB activity.'},\n",
       "  {'qas': [{'id': '56c1f03bef6e394741000053_001',\n",
       "     'question': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "     'answers': [{'text': 'thyroid transcription factor 1',\n",
       "       'answer_start': 233}]}],\n",
       "   'context': 'NKX2-1 mutations in brain-lung-thyroid syndrome: a case series of four patients. Brain-lung-thyroid syndrome (BLTS) characterized by congenital hypothyroidism, respiratory distress syndrome, and benign hereditary chorea is caused by thyroid transcription factor 1 (NKX2-1/TTF1) mutations. We report the clinical and molecular characteristics of four cases presenting with primary hypothyroidism, respiratory distress, and neurological disorder. Two of the four patients presenting with the triad of BLTS had NKX2-1 mutations, and one of these NKX2-1 [c.890_896del (p.Ala327Glyfs*52)] is a novel variant. The third patient without any identified NKX2-1 mutations was a carrier of mitochondrial mutation; this raises the possibility of mitochondrial mutations contributing to thyroid dysgenesis. Although rare, the triad of congenital hypothyroidism, neurological, and respiratory signs is highly suggestive of NKX2-1 anomalies. Screening for NKX2-1 mutations in patients with thyroid, lung, and neurological abnormalities will enable a unifying diagnosis and genetic counseling for the affected families. In addition, identification of an NKX2-1 defect would be helpful in allaying the concerns about inadequate thyroxine supplementation as the cause of neurological defects observed in some children with congenital hypothyroidism.'},\n",
       "  {'qas': [{'id': '56c1f03bef6e394741000053_002',\n",
       "     'question': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "     'answers': [{'text': 'Thyroid transcription factor 1',\n",
       "       'answer_start': 101}]}],\n",
       "   'context': 'Five new TTF1/NKX2.1 mutations in brain-lung-thyroid syndrome: rescue by PAX8 synergism in one case. Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC). The objectives of the present study were (i) detection of NKX2-1 mutations in patients with CH associated with pneumopathy and/or BHC, (ii) functional analysis of new mutations in vitro and (iii) description of the phenotypic spectrum of brain-lung-thyroid syndrome. We identified three new heterozygous missense mutations (L176V, P202L, Q210P), a splice site mutation (376-2A-->G), and one deletion of NKX2-1 at 14q13. Functional analysis of the three missense mutations revealed loss of transactivation capacity on the human thyroglobulin enhancer/promoter. Interestingly, we showed that deficient transcriptional activity of NKX2-1-P202L was completely rescued by cotransfected PAX8-WT, whereas the synergistic effect was abolished by L176V and Q210P. The clinical spectrum of 6 own and 40 published patients with NKX2-1 mutations ranged from the complete triad of brain-lung-thyroid syndrome (50%), brain and thyroid disease (30%), to isolated BHC (13%). Thyroid morphology was normal (55%) and compensated hypothyroidism occurred in 61%. Lung disease occurred in 54% of patients (IRDS at term 76%; recurrent pulmonary infections 24%). On follow-up, 20% developed severe chronic interstitial lung disease, and 16% died. In conclusion, we describe five new NKX2.1 mutations with, for the first time, complete rescue by PAX8 of the deficient transactivating capacity in one case. Additionally, our review shows that the majority of affected patients display neurological and/or thyroidal problems and that, although less frequent, lung disease is responsible for a considerable mortality.'},\n",
       "  {'qas': [{'id': '56c1f03bef6e394741000053_003',\n",
       "     'question': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "     'answers': [{'text': 'Thyroid transcription factor 1',\n",
       "       'answer_start': 101}]}],\n",
       "   'context': 'Five new TTF1/NKX2.1 mutations in brain-lung-thyroid syndrome: rescue by PAX8 synergism in one case. Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC). The objectives of the present study were (i) detection of NKX2-1 mutations in patients with CH associated with pneumopathy and/or BHC, (ii) functional analysis of new mutations in vitro and (iii) description of the phenotypic spectrum of brain-lung-thyroid syndrome. We identified three new heterozygous missense mutations (L176V, P202L, Q210P), a splice site mutation (376-2A-->G), and one deletion of NKX2-1 at 14q13. Functional analysis of the three missense mutations revealed loss of transactivation capacity on the human thyroglobulin enhancer/promoter. Interestingly, we showed that deficient transcriptional activity of NKX2-1-P202L was completely rescued by cotransfected PAX8-WT, whereas the synergistic effect was abolished by L176V and Q210P. The clinical spectrum of 6 own and 40 published patients with NKX2-1 mutations ranged from the complete triad of brain-lung-thyroid syndrome (50%), brain and thyroid disease (30%), to isolated BHC (13%). Thyroid morphology was normal (55%) and compensated hypothyroidism occurred in 61%. Lung disease occurred in 54% of patients (IRDS at term 76%; recurrent pulmonary infections 24%). On follow-up, 20% developed severe chronic interstitial lung disease, and 16% died. In conclusion, we describe five new NKX2.1 mutations with, for the first time, complete rescue by PAX8 of the deficient transactivating capacity in one case. Additionally, our review shows that the majority of affected patients display neurological and/or thyroidal problems and that, although less frequent, lung disease is responsible for a considerable mortality.'},\n",
       "  {'qas': [{'id': '56c1f03bef6e394741000053_004',\n",
       "     'question': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "     'answers': [{'text': 'thyroid transcription factor 1',\n",
       "       'answer_start': 233}]}],\n",
       "   'context': 'NKX2-1 mutations in brain-lung-thyroid syndrome: a case series of four patients. Brain-lung-thyroid syndrome (BLTS) characterized by congenital hypothyroidism, respiratory distress syndrome, and benign hereditary chorea is caused by thyroid transcription factor 1 (NKX2-1/TTF1) mutations. We report the clinical and molecular characteristics of four cases presenting with primary hypothyroidism, respiratory distress, and neurological disorder. Two of the four patients presenting with the triad of BLTS had NKX2-1 mutations, and one of these NKX2-1 [c.890_896del (p.Ala327Glyfs*52)] is a novel variant. The third patient without any identified NKX2-1 mutations was a carrier of mitochondrial mutation; this raises the possibility of mitochondrial mutations contributing to thyroid dysgenesis. Although rare, the triad of congenital hypothyroidism, neurological, and respiratory signs is highly suggestive of NKX2-1 anomalies. Screening for NKX2-1 mutations in patients with thyroid, lung, and neurological abnormalities will enable a unifying diagnosis and genetic counseling for the affected families. In addition, identification of an NKX2-1 defect would be helpful in allaying the concerns about inadequate thyroxine supplementation as the cause of neurological defects observed in some children with congenital hypothyroidism.'},\n",
       "  {'qas': [{'id': '56c1f03bef6e394741000053_005',\n",
       "     'question': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "     'answers': [{'text': 'Thyroid transcription factor 1',\n",
       "       'answer_start': 101}]}],\n",
       "   'context': 'Five new TTF1/NKX2.1 mutations in brain-lung-thyroid syndrome: rescue by PAX8 synergism in one case. Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC). The objectives of the present study were (i) detection of NKX2-1 mutations in patients with CH associated with pneumopathy and/or BHC, (ii) functional analysis of new mutations in vitro and (iii) description of the phenotypic spectrum of brain-lung-thyroid syndrome. We identified three new heterozygous missense mutations (L176V, P202L, Q210P), a splice site mutation (376-2A-->G), and one deletion of NKX2-1 at 14q13. Functional analysis of the three missense mutations revealed loss of transactivation capacity on the human thyroglobulin enhancer/promoter. Interestingly, we showed that deficient transcriptional activity of NKX2-1-P202L was completely rescued by cotransfected PAX8-WT, whereas the synergistic effect was abolished by L176V and Q210P. The clinical spectrum of 6 own and 40 published patients with NKX2-1 mutations ranged from the complete triad of brain-lung-thyroid syndrome (50%), brain and thyroid disease (30%), to isolated BHC (13%). Thyroid morphology was normal (55%) and compensated hypothyroidism occurred in 61%. Lung disease occurred in 54% of patients (IRDS at term 76%; recurrent pulmonary infections 24%). On follow-up, 20% developed severe chronic interstitial lung disease, and 16% died. In conclusion, we describe five new NKX2.1 mutations with, for the first time, complete rescue by PAX8 of the deficient transactivating capacity in one case. Additionally, our review shows that the majority of affected patients display neurological and/or thyroidal problems and that, although less frequent, lung disease is responsible for a considerable mortality.'},\n",
       "  {'qas': [{'id': '56c1f03bef6e394741000053_006',\n",
       "     'question': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "     'answers': [{'text': 'thyroid transcription factor 1',\n",
       "       'answer_start': 233}]}],\n",
       "   'context': 'NKX2-1 mutations in brain-lung-thyroid syndrome: a case series of four patients. Brain-lung-thyroid syndrome (BLTS) characterized by congenital hypothyroidism, respiratory distress syndrome, and benign hereditary chorea is caused by thyroid transcription factor 1 (NKX2-1/TTF1) mutations. We report the clinical and molecular characteristics of four cases presenting with primary hypothyroidism, respiratory distress, and neurological disorder. Two of the four patients presenting with the triad of BLTS had NKX2-1 mutations, and one of these NKX2-1 [c.890_896del (p.Ala327Glyfs*52)] is a novel variant. The third patient without any identified NKX2-1 mutations was a carrier of mitochondrial mutation; this raises the possibility of mitochondrial mutations contributing to thyroid dysgenesis. Although rare, the triad of congenital hypothyroidism, neurological, and respiratory signs is highly suggestive of NKX2-1 anomalies. Screening for NKX2-1 mutations in patients with thyroid, lung, and neurological abnormalities will enable a unifying diagnosis and genetic counseling for the affected families. In addition, identification of an NKX2-1 defect would be helpful in allaying the concerns about inadequate thyroxine supplementation as the cause of neurological defects observed in some children with congenital hypothyroidism.'},\n",
       "  {'qas': [{'id': '56c1f03bef6e394741000053_007',\n",
       "     'question': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "     'answers': [{'text': 'Thyroid transcription factor 1',\n",
       "       'answer_start': 101}]}],\n",
       "   'context': 'Five new TTF1/NKX2.1 mutations in brain-lung-thyroid syndrome: rescue by PAX8 synergism in one case. Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC). The objectives of the present study were (i) detection of NKX2-1 mutations in patients with CH associated with pneumopathy and/or BHC, (ii) functional analysis of new mutations in vitro and (iii) description of the phenotypic spectrum of brain-lung-thyroid syndrome. We identified three new heterozygous missense mutations (L176V, P202L, Q210P), a splice site mutation (376-2A-->G), and one deletion of NKX2-1 at 14q13. Functional analysis of the three missense mutations revealed loss of transactivation capacity on the human thyroglobulin enhancer/promoter. Interestingly, we showed that deficient transcriptional activity of NKX2-1-P202L was completely rescued by cotransfected PAX8-WT, whereas the synergistic effect was abolished by L176V and Q210P. The clinical spectrum of 6 own and 40 published patients with NKX2-1 mutations ranged from the complete triad of brain-lung-thyroid syndrome (50%), brain and thyroid disease (30%), to isolated BHC (13%). Thyroid morphology was normal (55%) and compensated hypothyroidism occurred in 61%. Lung disease occurred in 54% of patients (IRDS at term 76%; recurrent pulmonary infections 24%). On follow-up, 20% developed severe chronic interstitial lung disease, and 16% died. In conclusion, we describe five new NKX2.1 mutations with, for the first time, complete rescue by PAX8 of the deficient transactivating capacity in one case. Additionally, our review shows that the majority of affected patients display neurological and/or thyroidal problems and that, although less frequent, lung disease is responsible for a considerable mortality.'},\n",
       "  {'qas': [{'id': '532498959b2d7acc7e000017_001',\n",
       "     'question': 'Which enzyme is targeted by the drug Imetelstat?',\n",
       "     'answers': [{'text': 'human telomerase', 'answer_start': 1926}]}],\n",
       "   'context': \"Oligonucleotide n3'-->p5' phosphoramidates and thio-phoshoramidates as potential therapeutic agents. Nucleic acids analogues, i.e., oligonucleotide N3'-->P5' phosphoramidates and N3'-->P5' thio-phosphoramidates, containing 3'-amino-3'-deoxy nucleosides with various 2'-substituents were synthesized and extensively studied. These compounds resist nuclease hydrolysis and form stable duplexes with complementary native phosphodiester DNA and, particularly, RNA strands. An increase in duplexes' melting temperature, DeltaT(m), relative to their phosphodiester counterparts, reaches 2.2-4.0 degrees per modified nucleoside. 2'-OH- (RNA-like), 2'-O-Me-, and 2'-ribo-F-nucleoside substitutions result in the highest degree of duplex stabilization. Moreover, under close to physiological salt and pH conditions, the 2'-deoxy- and 2'-fluoro-phosphoramidate compounds form extremely stable triple-stranded complexes with either single- or double-stranded phosphodiester DNA oligonucleotides. Melting temperature, T(m), of these triplexes exceeds T(m) values for the isosequential phosphodiester counterparts by up to 35 degrees . 2'-Deoxy-N3'-->P5' phosphoramidates adopt RNA-like C3'-endo or N-type nucleoside sugar-ring conformations and hence can be used as stable RNA mimetics. Duplexes formed by 2'-deoxy phosphoramidates with complementary RNA strands are not substrates for RNase H-mediated cleavage in vitro. Oligonucleotide phosphoramidates and especially thio-phosphoramidates conjugated with lipid groups are cell-permeable and demonstrate high biological target specific activity in vitro. In vivo, these compounds show good bioavailability and efficient biodistribution to all major organs, while exerting acceptable toxicity at therapeutically relevant doses. Short oligonucleotide N3'-->P5' thio-phosphoramidate conjugated to 5'-palmitoyl group, designated as GRN163L (Imetelstat), was recently introduced as a potent human telomerase inhibitor. GRN163L is not an antisense agent; it is a direct competitive inhibitor of human telomerase, which directly binds to the active site of the enzyme and thus inhibits its activity. This compound is currently in multiple Phase-I and Phase-I/II clinical trials as potential broad-spectrum anticancer agent.\"},\n",
       "  {'qas': [{'id': '532498959b2d7acc7e000017_002',\n",
       "     'question': 'Which enzyme is targeted by the drug Imetelstat?',\n",
       "     'answers': [{'text': 'human telomerase', 'answer_start': 279}]}],\n",
       "   'context': 'The telomerase antagonist, imetelstat, efficiently targets glioblastoma tumor-initiating cells leading to decreased proliferation and tumor growth. PURPOSE: Telomerase activity is one of the hallmarks of cancer and is a highly relevant therapeutic target. The effects of a novel human telomerase antagonist, imetelstat, on primary human glioblastoma (GBM) tumor-initiating cells were investigated in vitro and in vivo. EXPERIMENTAL DESIGN: Tumor-initiating cells were isolated from primary GBM tumors and expanded as neurospheres in vitro. The GBM tumor-initiating cells were treated with imetelstat and examined for the effects on telomerase activity levels, telomere length, proliferation, clonogenicity, and differentiation. Subsequently, mouse orthotopic and subcutaneous xenografts were used to assess the in vivo efficacy of imetelstat. RESULTS: Imetelstat treatment produced a dose-dependent inhibition of telomerase (IC(50) 0.45 micromol/L). Long-term imetelstat treatment led to progressive telomere shortening, reduced rates of proliferation, and eventually cell death in GBM tumor-initiating cells. Imetelstat in combination with radiation and temozolomide had a dramatic effect on cell survival and activated the DNA damage response pathway. Imetelstat is able to cross the blood-brain barrier in orthotopic GBM xenograft tumors. Fluorescently labeled GBM tumor cells isolated from orthotopic tumors, following systemic administration of imetelstat (30 mg/kg every day for three days), showed approximately 70% inhibition of telomerase activity. Chronic systemic treatment produced a marked decrease in the rate of xenograft subcutaneous tumor growth. CONCLUSION: This preclinical study supports the feasibility of testing imetelstat in the treatment of GBM patients, alone or in combination with standard therapies.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_001',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 414}]}],\n",
       "   'context': 'Rag GTPases mediate amino acid-dependent recruitment of TFEB and MITF to lysosomes. The mTORC1 complex supports cell growth and proliferation in response to energy levels, growth factors, and nutrients. The Rag guanosine triphosphatases (GTPases) activate mTORC1 in response to amino acids by promoting its redistribution to lysosomes. In this paper, we identify a novel role for Rags in controlling activation of transcription factor EB (TFEB), a master regulator of autophagic and lysosomal gene expression. Interaction of TFEB with active Rag heterodimers promoted recruitment of TFEB to lysosomes, leading to mTORC1-dependent phosphorylation and inhibition of TFEB. The interaction of TFEB with Rags required the first 30 residues of TFEB and the switch regions of the Rags G domain. Depletion or inactivation of Rags prevented recruitment of TFEB to lysosomes, whereas expression of active Rags induced association of TFEB with lysosomal membranes. Finally, Rag GTPases bound and regulated activation of microphthalmia-associated transcription factor, suggesting a broader role for Rags in the control of gene expression. Our work provides new insight into the molecular mechanisms that link nutrient availability and TFEB localization and activation.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_002',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 692}]}],\n",
       "   'context': \"Signals from the lysosome: a control centre for cellular clearance and energy metabolism. For a long time, lysosomes were considered merely to be cellular 'incinerators' involved in the degradation and recycling of cellular waste. However, now there is compelling evidence indicating that lysosomes have a much broader function and that they are involved in fundamental processes such as secretion, plasma membrane repair, signalling and energy metabolism. Furthermore, the essential role of lysosomes in autophagic pathways puts these organelles at the crossroads of several cellular processes, with significant implications for health and disease. The identification of a master regulator, transcription factor EB (TFEB), that regulates lysosomal biogenesis and autophagy has revealed how the lysosome adapts to environmental cues, such as starvation, and targeting TFEB may provide a novel therapeutic strategy for modulating lysosomal function in human disease.\"},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_003',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 839}]}],\n",
       "   'context': \"TFEB regulates lysosomal proteostasis. Loss-of-function diseases are often caused by destabilizing mutations that lead to protein misfolding and degradation. Modulating the innate protein homeostasis (proteostasis) capacity may lead to rescue of native folding of the mutated variants, thereby ameliorating the disease phenotype. In lysosomal storage disorders (LSDs), a number of highly prevalent alleles have missense mutations that do not impair the enzyme's catalytic activity but destabilize its native structure, resulting in the degradation of the misfolded protein. Enhancing the cellular folding capacity enables rescuing the native, biologically functional structure of these unstable mutated enzymes. However, proteostasis modulators specific for the lysosomal system are currently unknown. Here, we investigate the role of the transcription factor EB (TFEB), a master regulator of lysosomal biogenesis and function, in modulating lysosomal proteostasis in LSDs. We show that TFEB activation results in enhanced folding, trafficking and lysosomal activity of a severely destabilized glucocerebrosidase (GC) variant associated with the development of Gaucher disease (GD), the most common LSD. TFEB specifically induces the expression of GC and of key genes involved in folding and lysosomal trafficking, thereby enhancing both the pool of mutated enzyme and its processing through the secretory pathway. TFEB activation also rescues the activity of a β-hexosaminidase mutant associated with the development of another LSD, Tay-Sachs disease, thus suggesting general applicability of TFEB-mediated proteostasis modulation to rescue destabilizing mutations in LSDs. In summary, our findings identify TFEB as a specific regulator of lysosomal proteostasis and suggest that TFEB may be used as a therapeutic target to rescue enzyme homeostasis in LSDs.\"},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_004',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 355}]}],\n",
       "   'context': 'TFEB controls cellular lipid metabolism through a starvation-induced autoregulatory loop. The lysosomal-autophagic pathway is activated by starvation and plays an important role in both cellular clearance and lipid catabolism. However, the transcriptional regulation of this pathway in response to metabolic cues is uncharacterized. Here we show that the transcription factor EB (TFEB), a master regulator of lysosomal biogenesis and autophagy, is induced by starvation through an autoregulatory feedback loop and exerts a global transcriptional control on lipid catabolism via Ppargc1α and Ppar1α. Thus, during starvation a transcriptional mechanism links the autophagic pathway to cellular energy metabolism. The conservation of this mechanism in Caenorhabditis elegans suggests a fundamental role for TFEB in the evolution of the adaptive response to food deprivation. Viral delivery of TFEB to the liver prevented weight gain and metabolic syndrome in both diet-induced and genetic mouse models of obesity, suggesting a new therapeutic strategy for disorders of lipid metabolism.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_005',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'Transcription Factor EB (TFEB)',\n",
       "       'answer_start': 344}]}],\n",
       "   'context': 'A lysosome-to-nucleus signalling mechanism senses and regulates the lysosome via mTOR and TFEB. The lysosome plays a key role in cellular homeostasis by controlling both cellular clearance and energy production to respond to environmental cues. However, the mechanisms mediating lysosomal adaptation are largely unknown. Here, we show that the Transcription Factor EB (TFEB), a master regulator of lysosomal biogenesis, colocalizes with master growth regulator mTOR complex 1 (mTORC1) on the lysosomal membrane. When nutrients are present, phosphorylation of TFEB by mTORC1 inhibits TFEB activity. Conversely, pharmacological inhibition of mTORC1, as well as starvation and lysosomal disruption, activates TFEB by promoting its nuclear translocation. In addition, the transcriptional response of lysosomal and autophagic genes to either lysosomal dysfunction or pharmacological inhibition of mTORC1 is suppressed in TFEB-/- cells. Interestingly, the Rag GTPase complex, which senses lysosomal amino acids and activates mTORC1, is both necessary and sufficient to regulate starvation- and stress-induced nuclear translocation of TFEB. These data indicate that the lysosome senses its content and regulates its own biogenesis by a lysosome-to-nucleus signalling mechanism that involves TFEB and mTOR.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_006',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 746}]}],\n",
       "   'context': 'Increased lysosomal biogenesis in activated microglia and exacerbated neuronal damage after traumatic brain injury in progranulin-deficient mice. Progranulin (PGRN) is known to play a role in the pathogenesis of neurodegenerative diseases. Recently, it has been demonstrated that patients with the homozygous mutation in the GRN gene present with neuronal ceroid lipofuscinosis, and there is growing evidence that PGRN is related to lysosomal function. In the present study, we investigated the possible role of PGRN in the lysosomes of activated microglia in the cerebral cortex after traumatic brain injury (TBI). We showed that the mouse GRN gene has two possible coordinated lysosomal expression and regulation (CLEAR) sequences that bind to transcription factor EB (TFEB), a master regulator of lysosomal genes. PGRN was colocalized with Lamp1, a lysosomal marker, and Lamp1-positive areas in GRN-deficient (KO) mice were significantly expanded compared with wild-type (WT) mice after TBI. Expression of all the lysosome-related genes examined in KO mice was significantly higher than that in WT mice. The number of activated microglia with TFEB localized to the nucleus was also significantly increased in KO as compared with WT mice. Since the TFEB translocation is regulated by the mammalian target of rapamycin complex 1 (mTORC1) activity in the lysosome, we compared ribosomal S6 kinase 1 (S6K1) phosphorylation that reflects mTORC1 activity. S6K1 phosphorylation in KO mice was significantly lower than that in WT mice. In addition, the number of nissl-positive and fluoro-jade B-positive cells around the injury was significantly decreased and increased, respectively, in KO as compared with WT mice. These results suggest that PGRN localized in the lysosome is involved in the activation of mTORC1, and its deficiency leads to increased TFEB nuclear translocation with a resultant increase in lysosomal biogenesis in activated microglia and exacerbated neuronal damage in the cerebral cortex after TBI.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_007',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 414}]}],\n",
       "   'context': 'Rag GTPases mediate amino acid-dependent recruitment of TFEB and MITF to lysosomes. The mTORC1 complex supports cell growth and proliferation in response to energy levels, growth factors, and nutrients. The Rag guanosine triphosphatases (GTPases) activate mTORC1 in response to amino acids by promoting its redistribution to lysosomes. In this paper, we identify a novel role for Rags in controlling activation of transcription factor EB (TFEB), a master regulator of autophagic and lysosomal gene expression. Interaction of TFEB with active Rag heterodimers promoted recruitment of TFEB to lysosomes, leading to mTORC1-dependent phosphorylation and inhibition of TFEB. The interaction of TFEB with Rags required the first 30 residues of TFEB and the switch regions of the Rags G domain. Depletion or inactivation of Rags prevented recruitment of TFEB to lysosomes, whereas expression of active Rags induced association of TFEB with lysosomal membranes. Finally, Rag GTPases bound and regulated activation of microphthalmia-associated transcription factor, suggesting a broader role for Rags in the control of gene expression. Our work provides new insight into the molecular mechanisms that link nutrient availability and TFEB localization and activation.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_008',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 418}]}],\n",
       "   'context': 'RRAG GTPases link nutrient availability to gene expression, autophagy and lysosomal biogenesis. When the levels of intracellular amino acids are high, RRAG GTPases recruit MTORC1 to lysosomes and promote its activation. We found that RRAGs also recruit specific MTORC1 substrates to the lysosomal surface, thus facilitating MTORC1-mediated phosphorylation and regulation. In particular, active RRAGs interact with the transcription factor EB (TFEB), the master regulator of a gene network that promotes lysosomal biogenesis and autophagy. Redistribution to lysosomes is critical for MTORC1-dependent inactivation of TFEB under nutrient-rich conditions. Therefore, RRAGs play a critical role coordinating nutrient availability and cellular clearance.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_009',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'Transcription Factor EB (TFEB)',\n",
       "       'answer_start': 344}]}],\n",
       "   'context': 'A lysosome-to-nucleus signalling mechanism senses and regulates the lysosome via mTOR and TFEB. The lysosome plays a key role in cellular homeostasis by controlling both cellular clearance and energy production to respond to environmental cues. However, the mechanisms mediating lysosomal adaptation are largely unknown. Here, we show that the Transcription Factor EB (TFEB), a master regulator of lysosomal biogenesis, colocalizes with master growth regulator mTOR complex 1 (mTORC1) on the lysosomal membrane. When nutrients are present, phosphorylation of TFEB by mTORC1 inhibits TFEB activity. Conversely, pharmacological inhibition of mTORC1, as well as starvation and lysosomal disruption, activates TFEB by promoting its nuclear translocation. In addition, the transcriptional response of lysosomal and autophagic genes to either lysosomal dysfunction or pharmacological inhibition of mTORC1 is suppressed in TFEB-/- cells. Interestingly, the Rag GTPase complex, which senses lysosomal amino acids and activates mTORC1, is both necessary and sufficient to regulate starvation- and stress-induced nuclear translocation of TFEB. These data indicate that the lysosome senses its content and regulates its own biogenesis by a lysosome-to-nucleus signalling mechanism that involves TFEB and mTOR.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_010',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 839}]}],\n",
       "   'context': \"TFEB regulates lysosomal proteostasis. Loss-of-function diseases are often caused by destabilizing mutations that lead to protein misfolding and degradation. Modulating the innate protein homeostasis (proteostasis) capacity may lead to rescue of native folding of the mutated variants, thereby ameliorating the disease phenotype. In lysosomal storage disorders (LSDs), a number of highly prevalent alleles have missense mutations that do not impair the enzyme's catalytic activity but destabilize its native structure, resulting in the degradation of the misfolded protein. Enhancing the cellular folding capacity enables rescuing the native, biologically functional structure of these unstable mutated enzymes. However, proteostasis modulators specific for the lysosomal system are currently unknown. Here, we investigate the role of the transcription factor EB (TFEB), a master regulator of lysosomal biogenesis and function, in modulating lysosomal proteostasis in LSDs. We show that TFEB activation results in enhanced folding, trafficking and lysosomal activity of a severely destabilized glucocerebrosidase (GC) variant associated with the development of Gaucher disease (GD), the most common LSD. TFEB specifically induces the expression of GC and of key genes involved in folding and lysosomal trafficking, thereby enhancing both the pool of mutated enzyme and its processing through the secretory pathway. TFEB activation also rescues the activity of a β-hexosaminidase mutant associated with the development of another LSD, Tay-Sachs disease, thus suggesting general applicability of TFEB-mediated proteostasis modulation to rescue destabilizing mutations in LSDs. In summary, our findings identify TFEB as a specific regulator of lysosomal proteostasis and suggest that TFEB may be used as a therapeutic target to rescue enzyme homeostasis in LSDs.\"},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_011',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 355}]}],\n",
       "   'context': 'TFEB controls cellular lipid metabolism through a starvation-induced autoregulatory loop. The lysosomal-autophagic pathway is activated by starvation and plays an important role in both cellular clearance and lipid catabolism. However, the transcriptional regulation of this pathway in response to metabolic cues is uncharacterized. Here we show that the transcription factor EB (TFEB), a master regulator of lysosomal biogenesis and autophagy, is induced by starvation through an autoregulatory feedback loop and exerts a global transcriptional control on lipid catabolism via Ppargc1α and Ppar1α. Thus, during starvation a transcriptional mechanism links the autophagic pathway to cellular energy metabolism. The conservation of this mechanism in Caenorhabditis elegans suggests a fundamental role for TFEB in the evolution of the adaptive response to food deprivation. Viral delivery of TFEB to the liver prevented weight gain and metabolic syndrome in both diet-induced and genetic mouse models of obesity, suggesting a new therapeutic strategy for disorders of lipid metabolism.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_012',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 451}]}],\n",
       "   'context': 'TFEB links autophagy to lysosomal biogenesis. Autophagy is a cellular catabolic process that relies on the cooperation of autophagosomes and lysosomes. During starvation, the cell expands both compartments to enhance degradation processes. We found that starvation activates a transcriptional program that controls major steps of the autophagic pathway, including autophagosome formation, autophagosome-lysosome fusion, and substrate degradation. The transcription factor EB (TFEB), a master gene for lysosomal biogenesis, coordinated this program by driving expression of autophagy and lysosomal genes. Nuclear localization and activity of TFEB were regulated by serine phosphorylation mediated by the extracellular signal-regulated kinase 2, whose activity was tuned by the levels of extracellular nutrients. Thus, a mitogen-activated protein kinase-dependent mechanism regulates autophagy by controlling the biogenesis and partnership of two distinct cellular organelles.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_013',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 418}]}],\n",
       "   'context': 'RRAG GTPases link nutrient availability to gene expression, autophagy and lysosomal biogenesis. When the levels of intracellular amino acids are high, RRAG GTPases recruit MTORC1 to lysosomes and promote its activation. We found that RRAGs also recruit specific MTORC1 substrates to the lysosomal surface, thus facilitating MTORC1-mediated phosphorylation and regulation. In particular, active RRAGs interact with the transcription factor EB (TFEB), the master regulator of a gene network that promotes lysosomal biogenesis and autophagy. Redistribution to lysosomes is critical for MTORC1-dependent inactivation of TFEB under nutrient-rich conditions. Therefore, RRAGs play a critical role coordinating nutrient availability and cellular clearance.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_014',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 746}]}],\n",
       "   'context': 'Increased lysosomal biogenesis in activated microglia and exacerbated neuronal damage after traumatic brain injury in progranulin-deficient mice. Progranulin (PGRN) is known to play a role in the pathogenesis of neurodegenerative diseases. Recently, it has been demonstrated that patients with the homozygous mutation in the GRN gene present with neuronal ceroid lipofuscinosis, and there is growing evidence that PGRN is related to lysosomal function. In the present study, we investigated the possible role of PGRN in the lysosomes of activated microglia in the cerebral cortex after traumatic brain injury (TBI). We showed that the mouse GRN gene has two possible coordinated lysosomal expression and regulation (CLEAR) sequences that bind to transcription factor EB (TFEB), a master regulator of lysosomal genes. PGRN was colocalized with Lamp1, a lysosomal marker, and Lamp1-positive areas in GRN-deficient (KO) mice were significantly expanded compared with wild-type (WT) mice after TBI. Expression of all the lysosome-related genes examined in KO mice was significantly higher than that in WT mice. The number of activated microglia with TFEB localized to the nucleus was also significantly increased in KO as compared with WT mice. Since the TFEB translocation is regulated by the mammalian target of rapamycin complex 1 (mTORC1) activity in the lysosome, we compared ribosomal S6 kinase 1 (S6K1) phosphorylation that reflects mTORC1 activity. S6K1 phosphorylation in KO mice was significantly lower than that in WT mice. In addition, the number of nissl-positive and fluoro-jade B-positive cells around the injury was significantly decreased and increased, respectively, in KO as compared with WT mice. These results suggest that PGRN localized in the lysosome is involved in the activation of mTORC1, and its deficiency leads to increased TFEB nuclear translocation with a resultant increase in lysosomal biogenesis in activated microglia and exacerbated neuronal damage in the cerebral cortex after TBI.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_015',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 414}]}],\n",
       "   'context': 'Rag GTPases mediate amino acid-dependent recruitment of TFEB and MITF to lysosomes. The mTORC1 complex supports cell growth and proliferation in response to energy levels, growth factors, and nutrients. The Rag guanosine triphosphatases (GTPases) activate mTORC1 in response to amino acids by promoting its redistribution to lysosomes. In this paper, we identify a novel role for Rags in controlling activation of transcription factor EB (TFEB), a master regulator of autophagic and lysosomal gene expression. Interaction of TFEB with active Rag heterodimers promoted recruitment of TFEB to lysosomes, leading to mTORC1-dependent phosphorylation and inhibition of TFEB. The interaction of TFEB with Rags required the first 30 residues of TFEB and the switch regions of the Rags G domain. Depletion or inactivation of Rags prevented recruitment of TFEB to lysosomes, whereas expression of active Rags induced association of TFEB with lysosomal membranes. Finally, Rag GTPases bound and regulated activation of microphthalmia-associated transcription factor, suggesting a broader role for Rags in the control of gene expression. Our work provides new insight into the molecular mechanisms that link nutrient availability and TFEB localization and activation.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_016',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 534}]}],\n",
       "   'context': 'Characterization of the CLEAR network reveals an integrated control of cellular clearance pathways. In metazoans, lysosomes are the center for the degradation of macromolecules and play a key role in a variety of cellular processes, such as autophagy, exocytosis and membrane repair. Defects of lysosomal pathways are associated with lysosomal storage disorders and with several late onset neurodegenerative diseases. We recently discovered the CLEAR (Coordinated Lysosomal Expression and Regulation) gene network and its master gene transcription factor EB (TFEB), which regulates lysosomal biogenesis and function. Here, we used a combination of genomic approaches, including ChIP-seq (sequencing of chromatin immunoprecipitate) analysis, profiling of TFEB-mediated transcriptional induction, genome-wide mapping of TFEB target sites and recursive expression meta-analysis of TFEB targets, to identify 471 TFEB direct targets that represent essential components of the CLEAR network. This analysis revealed a comprehensive system regulating the expression, import and activity of lysosomal enzymes that control the degradation of proteins, glycosaminoglycans, sphingolipids and glycogen. Interestingly, the CLEAR network appears to be involved in the regulation of additional lysosome-associated processes, including autophagy, exo- and endocytosis, phagocytosis and immune response. Furthermore, non-lysosomal enzymes involved in the degradation of essential proteins such as hemoglobin and chitin are also part of the CLEAR network. Finally, we identified nine novel lysosomal proteins by using the CLEAR network as a tool for prioritizing candidates. This study provides potential therapeutic targets to modulate cellular clearance in a variety of disease conditions.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_017',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'Transcription factor EB (TFEB)',\n",
       "       'answer_start': 1212}]}],\n",
       "   'context': 'Induction of lysosomal biogenesis in atherosclerotic macrophages can rescue lipid-induced lysosomal dysfunction and downstream sequelae. OBJECTIVE: Recent reports of a proatherogenic phenotype in mice with macrophage-specific autophagy deficiency have renewed interest in the role of the autophagy-lysosomal system in atherosclerosis. Lysosomes have the unique ability to process both exogenous material, including lipids and autophagy-derived cargo such as dysfunctional proteins/organelles. We aimed to understand the effects of an atherogenic lipid environment on macrophage lysosomes and to evaluate novel ways to modulate this system. APPROACH AND RESULTS: Using a variety of complementary techniques, we show that oxidized low-density lipoproteins and cholesterol crystals, commonly encountered lipid species in atherosclerosis, lead to profound lysosomal dysfunction in cultured macrophages. Disruptions in lysosomal pH, proteolytic capacity, membrane integrity, and morphology are readily seen. Using flow cytometry, we find that macrophages isolated from atherosclerotic plaques also display features of lysosome dysfunction. We then investigated whether enhancing lysosomal function can be beneficial. Transcription factor EB (TFEB) is the only known transcription factor that is a master regulator of lysosomal biogenesis although its role in macrophages has not been studied. Lysosomal stress induced by chloroquine or atherogenic lipids leads to TFEB nuclear translocation and activation of lysosomal and autophagy genes. TFEB overexpression in macrophages further augments this prodegradative response and rescues several deleterious effects seen with atherogenic lipid loading as evidenced by blunted lysosomal dysfunction, reduced secretion of the proinflammatory cytokine interleukin-1β, enhanced cholesterol efflux, and decreased polyubiquitinated protein aggregation. CONCLUSIONS: Taken together, these data demonstrate that lysosomal function is markedly impaired in atherosclerosis and suggest that induction of a lysosomal biogenesis program in macrophages has antiatherogenic effects.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_018',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 593}]}],\n",
       "   'context': 'Activation of peroxisome proliferator-activated receptor α induces lysosomal biogenesis in brain cells: implications for lysosomal storage disorders. Lysosomes are ubiquitous membrane-enclosed organelles filled with an acidic interior and are central to the autophagic, endocytic, or phagocytic pathway. In contrast to its classical function as the waste management machinery, lysosomes are now considered to be an integral part of various cellular signaling processes. The diverse functionality of this single organelle requires a very complex and coordinated regulation of its activity with transcription factor EB (TFEB), a master regulator of lysosomal biogenesis, at its core. However, mechanisms by which TFEB is regulated are poorly understood. This study demonstrates that gemfibrozil, an agonist of peroxisome proliferator-activated receptor (PPAR) α, alone and in conjunction with all-trans-retinoic acid is capable of enhancing TFEB in brain cells. We also observed that PPARα, but not PPARβ and PPARγ, is involved in gemfibrozil-mediated up-regulation of TFEB. Reporter assay and chromatin immunoprecipitation studies confirmed the recruitment of retinoid X receptor α, PPARα, and PGC1α on the PPAR-binding site on the Tfeb promoter as well. Subsequently, the drug-mediated induction of TFEB caused an increase in lysosomal protein and the lysosomal abundance in cell. Collectively, this study reinforces the link between lysosomal biogenesis and lipid metabolism with TFEB at the crossroads. Furthermore, gemfibrozil may be of therapeutic value in the treatment of lysosomal storage disorders in which autophagy-lysosome pathway plays an important role.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_019',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'Transcription Factor EB (TFEB)',\n",
       "       'answer_start': 344}]}],\n",
       "   'context': 'A lysosome-to-nucleus signalling mechanism senses and regulates the lysosome via mTOR and TFEB. The lysosome plays a key role in cellular homeostasis by controlling both cellular clearance and energy production to respond to environmental cues. However, the mechanisms mediating lysosomal adaptation are largely unknown. Here, we show that the Transcription Factor EB (TFEB), a master regulator of lysosomal biogenesis, colocalizes with master growth regulator mTOR complex 1 (mTORC1) on the lysosomal membrane. When nutrients are present, phosphorylation of TFEB by mTORC1 inhibits TFEB activity. Conversely, pharmacological inhibition of mTORC1, as well as starvation and lysosomal disruption, activates TFEB by promoting its nuclear translocation. In addition, the transcriptional response of lysosomal and autophagic genes to either lysosomal dysfunction or pharmacological inhibition of mTORC1 is suppressed in TFEB-/- cells. Interestingly, the Rag GTPase complex, which senses lysosomal amino acids and activates mTORC1, is both necessary and sufficient to regulate starvation- and stress-induced nuclear translocation of TFEB. These data indicate that the lysosome senses its content and regulates its own biogenesis by a lysosome-to-nucleus signalling mechanism that involves TFEB and mTOR.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_020',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 746}]}],\n",
       "   'context': 'Increased lysosomal biogenesis in activated microglia and exacerbated neuronal damage after traumatic brain injury in progranulin-deficient mice. Progranulin (PGRN) is known to play a role in the pathogenesis of neurodegenerative diseases. Recently, it has been demonstrated that patients with the homozygous mutation in the GRN gene present with neuronal ceroid lipofuscinosis, and there is growing evidence that PGRN is related to lysosomal function. In the present study, we investigated the possible role of PGRN in the lysosomes of activated microglia in the cerebral cortex after traumatic brain injury (TBI). We showed that the mouse GRN gene has two possible coordinated lysosomal expression and regulation (CLEAR) sequences that bind to transcription factor EB (TFEB), a master regulator of lysosomal genes. PGRN was colocalized with Lamp1, a lysosomal marker, and Lamp1-positive areas in GRN-deficient (KO) mice were significantly expanded compared with wild-type (WT) mice after TBI. Expression of all the lysosome-related genes examined in KO mice was significantly higher than that in WT mice. The number of activated microglia with TFEB localized to the nucleus was also significantly increased in KO as compared with WT mice. Since the TFEB translocation is regulated by the mammalian target of rapamycin complex 1 (mTORC1) activity in the lysosome, we compared ribosomal S6 kinase 1 (S6K1) phosphorylation that reflects mTORC1 activity. S6K1 phosphorylation in KO mice was significantly lower than that in WT mice. In addition, the number of nissl-positive and fluoro-jade B-positive cells around the injury was significantly decreased and increased, respectively, in KO as compared with WT mice. These results suggest that PGRN localized in the lysosome is involved in the activation of mTORC1, and its deficiency leads to increased TFEB nuclear translocation with a resultant increase in lysosomal biogenesis in activated microglia and exacerbated neuronal damage in the cerebral cortex after TBI.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_021',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 414}]}],\n",
       "   'context': 'Rag GTPases mediate amino acid-dependent recruitment of TFEB and MITF to lysosomes. The mTORC1 complex supports cell growth and proliferation in response to energy levels, growth factors, and nutrients. The Rag guanosine triphosphatases (GTPases) activate mTORC1 in response to amino acids by promoting its redistribution to lysosomes. In this paper, we identify a novel role for Rags in controlling activation of transcription factor EB (TFEB), a master regulator of autophagic and lysosomal gene expression. Interaction of TFEB with active Rag heterodimers promoted recruitment of TFEB to lysosomes, leading to mTORC1-dependent phosphorylation and inhibition of TFEB. The interaction of TFEB with Rags required the first 30 residues of TFEB and the switch regions of the Rags G domain. Depletion or inactivation of Rags prevented recruitment of TFEB to lysosomes, whereas expression of active Rags induced association of TFEB with lysosomal membranes. Finally, Rag GTPases bound and regulated activation of microphthalmia-associated transcription factor, suggesting a broader role for Rags in the control of gene expression. Our work provides new insight into the molecular mechanisms that link nutrient availability and TFEB localization and activation.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_022',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 839}]}],\n",
       "   'context': \"TFEB regulates lysosomal proteostasis. Loss-of-function diseases are often caused by destabilizing mutations that lead to protein misfolding and degradation. Modulating the innate protein homeostasis (proteostasis) capacity may lead to rescue of native folding of the mutated variants, thereby ameliorating the disease phenotype. In lysosomal storage disorders (LSDs), a number of highly prevalent alleles have missense mutations that do not impair the enzyme's catalytic activity but destabilize its native structure, resulting in the degradation of the misfolded protein. Enhancing the cellular folding capacity enables rescuing the native, biologically functional structure of these unstable mutated enzymes. However, proteostasis modulators specific for the lysosomal system are currently unknown. Here, we investigate the role of the transcription factor EB (TFEB), a master regulator of lysosomal biogenesis and function, in modulating lysosomal proteostasis in LSDs. We show that TFEB activation results in enhanced folding, trafficking and lysosomal activity of a severely destabilized glucocerebrosidase (GC) variant associated with the development of Gaucher disease (GD), the most common LSD. TFEB specifically induces the expression of GC and of key genes involved in folding and lysosomal trafficking, thereby enhancing both the pool of mutated enzyme and its processing through the secretory pathway. TFEB activation also rescues the activity of a β-hexosaminidase mutant associated with the development of another LSD, Tay-Sachs disease, thus suggesting general applicability of TFEB-mediated proteostasis modulation to rescue destabilizing mutations in LSDs. In summary, our findings identify TFEB as a specific regulator of lysosomal proteostasis and suggest that TFEB may be used as a therapeutic target to rescue enzyme homeostasis in LSDs.\"},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_023',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'Transcription factor EB (TFEB)',\n",
       "       'answer_start': 1212}]}],\n",
       "   'context': 'Induction of lysosomal biogenesis in atherosclerotic macrophages can rescue lipid-induced lysosomal dysfunction and downstream sequelae. OBJECTIVE: Recent reports of a proatherogenic phenotype in mice with macrophage-specific autophagy deficiency have renewed interest in the role of the autophagy-lysosomal system in atherosclerosis. Lysosomes have the unique ability to process both exogenous material, including lipids and autophagy-derived cargo such as dysfunctional proteins/organelles. We aimed to understand the effects of an atherogenic lipid environment on macrophage lysosomes and to evaluate novel ways to modulate this system. APPROACH AND RESULTS: Using a variety of complementary techniques, we show that oxidized low-density lipoproteins and cholesterol crystals, commonly encountered lipid species in atherosclerosis, lead to profound lysosomal dysfunction in cultured macrophages. Disruptions in lysosomal pH, proteolytic capacity, membrane integrity, and morphology are readily seen. Using flow cytometry, we find that macrophages isolated from atherosclerotic plaques also display features of lysosome dysfunction. We then investigated whether enhancing lysosomal function can be beneficial. Transcription factor EB (TFEB) is the only known transcription factor that is a master regulator of lysosomal biogenesis although its role in macrophages has not been studied. Lysosomal stress induced by chloroquine or atherogenic lipids leads to TFEB nuclear translocation and activation of lysosomal and autophagy genes. TFEB overexpression in macrophages further augments this prodegradative response and rescues several deleterious effects seen with atherogenic lipid loading as evidenced by blunted lysosomal dysfunction, reduced secretion of the proinflammatory cytokine interleukin-1β, enhanced cholesterol efflux, and decreased polyubiquitinated protein aggregation. CONCLUSIONS: Taken together, these data demonstrate that lysosomal function is markedly impaired in atherosclerosis and suggest that induction of a lysosomal biogenesis program in macrophages has antiatherogenic effects.'},\n",
       "  {'qas': [{'id': '56cdf40d5795f9a73e00003d_024',\n",
       "     'question': 'Which transcription factor is considered as a master regulator of lysosomal genes?',\n",
       "     'answers': [{'text': 'transcription factor EB (TFEB)',\n",
       "       'answer_start': 418}]}],\n",
       "   'context': 'RRAG GTPases link nutrient availability to gene expression, autophagy and lysosomal biogenesis. When the levels of intracellular amino acids are high, RRAG GTPases recruit MTORC1 to lysosomes and promote its activation. We found that RRAGs also recruit specific MTORC1 substrates to the lysosomal surface, thus facilitating MTORC1-mediated phosphorylation and regulation. In particular, active RRAGs interact with the transcription factor EB (TFEB), the master regulator of a gene network that promotes lysosomal biogenesis and autophagy. Redistribution to lysosomes is critical for MTORC1-dependent inactivation of TFEB under nutrient-rich conditions. Therefore, RRAGs play a critical role coordinating nutrient availability and cellular clearance.'},\n",
       "  {'qas': [{'id': '5343caffaeec6fbd07000002_001',\n",
       "     'question': 'Which is the human selenoprotein that contains several Se-Cys residues?',\n",
       "     'answers': [{'text': 'selenoprotein P', 'answer_start': 540}]}],\n",
       "   'context': 'Processive selenocysteine incorporation during synthesis of eukaryotic selenoproteins. Selenoproteins are a family of proteins that share the common feature of containing selenocysteine, the \"twenty-first\" amino acid. Selenocysteine incorporation occurs during translation of selenoprotein messages by redefinition of UGA codons, which normally specify termination of translation. Studies of the eukaryotic selenocysteine incorporation mechanism suggest that selenocysteine insertion is inefficient compared with termination. Nevertheless, selenoprotein P and several other selenoproteins are known to contain multiple selenocysteines. The production of full-length (FL) protein from these messages would seem to demand highly efficient selenocysteine incorporation due to the compounding effect of termination at each UGA codon. We present data demonstrating that efficient incorporation of multiple selenocysteines can be reconstituted in rabbit reticulocyte lysate translation reactions. Selenocysteine incorporation at the first UGA codon is inefficient but increases by approximately 10-fold at subsequent downstream UGA codons. We found that ribosomes in the \"processive\" phase of selenocysteine incorporation (i.e., after decoding the first UGA codon as selenocysteine) are fully competent to terminate translation at UAG and UAA codons, that ribosomes become less efficient at selenocysteine incorporation as the distance between UGA codons is increased, and that efficient selenocysteine incorporation is not dependent on cis-acting elements unique to selenoprotein P. Furthermore, we found that the percentage of ribosomes decoding a UGA codon as selenocysteine rather than termination can be increased by 3- to 5-fold by placing the murine leukemia virus UAG read-through element upstream of the first UGA codon or by providing a competing messenger RNA in trans. The mechanisms of selenocysteine incorporation and selenoprotein synthesis are discussed in light of these results.'},\n",
       "  {'qas': [{'id': '5343caffaeec6fbd07000002_002',\n",
       "     'question': 'Which is the human selenoprotein that contains several Se-Cys residues?',\n",
       "     'answers': [{'text': 'selenoprotein P', 'answer_start': 545}]}],\n",
       "   'context': 'Efficient incorporation of multiple selenocysteines involves an inefficient decoding step serving as a potential translational checkpoint and ribosome bottleneck. Selenocysteine is incorporated into proteins via \"recoding\" of UGA from a stop codon to a sense codon, a process that requires specific secondary structures in the 3\\' untranslated region, termed selenocysteine incorporation sequence (SECIS) elements, and the protein factors that they recruit. Whereas most selenoprotein mRNAs contain a single UGA codon and a single SECIS element, selenoprotein P genes encode multiple UGAs and two SECIS elements. We have identified evolutionary adaptations in selenoprotein P genes that contribute to the efficiency of incorporating multiple selenocysteine residues in this protein. The first is a conserved, inefficiently decoded UGA codon in the N-terminal region, which appears to serve both as a checkpoint for the presence of factors required for selenocysteine incorporation and as a \"bottleneck,\" slowing down the progress of elongating ribosomes. The second adaptation involves the presence of introns downstream of this inefficiently decoded UGA which confer the potential for nonsense-mediated decay when factors required for selenocysteine incorporation are limiting. Third, the two SECIS elements in selenoprotein P mRNA function with differing efficiencies, affecting both the rate and the efficiency of decoding different UGAs. The implications for how these factors contribute to the decoding of multiple selenocysteine residues are discussed.'},\n",
       "  {'qas': [{'id': '5343caffaeec6fbd07000002_003',\n",
       "     'question': 'Which is the human selenoprotein that contains several Se-Cys residues?',\n",
       "     'answers': [{'text': 'selenoprotein P', 'answer_start': 93}]}],\n",
       "   'context': '[The apoptosis-inducing activity of human selenoprotein P shorter isoform]. OBJECTIVE: Human selenoprotein P (HSelP) is unique protein that contains 10 selenocysteines encoded by 10 inframe UGA, which typically function as stop codon. The function of HSelP remains unclear, in part due to the inability to express it by gene recombinant technique. This study is to investigate expression and purification of recombinant HSelP in prokaryotic expression system, and its activity to induce apoptosis in vitro. METHODS: The shorter HSelP isoform was cloned. After the selenocysteine (SeCys) at 40th position from N terminus of the HSelP shorter isoform was mutated into cysteine by PCR, it was expressed in E. coli. The expressed product was purified with DEAE column and identified by Western blot. Subsequently, its function on induction of mitochondrial apoptotic activity was studied. RESULTS: The mutant HSelP shorter isoform expressed in prokaryotic system was purified by DEAE column to 90% homogeneity. The purified product, HSelP280m, induced the opening of mitochondrial permeability transition pore (PTP) and decreased the transmembrane potential in a dose-dependent manner. These events could be abolished by PTP specific inhibitors. CONCLUSION: HSelP280m can induce the opening of mitochondrial PTP, which provides a basis for investigating the structure and function of recombinant HSelP.'},\n",
       "  {'qas': [{'id': '5343caffaeec6fbd07000002_004',\n",
       "     'question': 'Which is the human selenoprotein that contains several Se-Cys residues?',\n",
       "     'answers': [{'text': 'selenoprotein P', 'answer_start': 330}]}],\n",
       "   'context': \"Expression and characterization of nonmammalian selenoprotein P in the zebrafish, Danio rerio. BACKGROUND: Selenoprotein P is a protein of considerable intrigue, due to its unusual composition and requirements for its biosynthesis. Whereas most selenoproteins contain a single selenocysteine residue, the human, bovine and rodent selenoprotein P genes encode proteins containing 10-12 selenocysteines. Selenoprotein P genes have, to date, only been reported in mammals, and the function of the protein remains elusive. RESULTS: Herein, we report the identification and characterization of nonmammalian selenoprotein P in the zebrafish Danio rerio. Sequencing of the cDNA revealed the presence of 17 selenocysteine codons, the highest number reported in any protein. Two histidine-rich regions present in the mammalian selenoprotein P sequences are conserved in the zebrafish protein, and two SECIS elements are present in the 3' untranslated region. Whole-mount in situ hybridization of zebrafish embryos revealed high levels of expression of selenoprotein P mRNA in fertilized eggs and in the yolk sac of developing embryos. Transient transfection of the cDNA in mammalian cells resulted in efficient expression of the full-length secreted selenoprotein. A single N-glycosylation site is predicted, and shown to be utilized. CONCLUSIONS: Discovery of selenoprotein P in the zebrafish opens a previously unavailable avenue for genetic investigation of the functions of this unusual protein.\"},\n",
       "  {'qas': [{'id': '5343caffaeec6fbd07000002_005',\n",
       "     'question': 'Which is the human selenoprotein that contains several Se-Cys residues?',\n",
       "     'answers': [{'text': 'Selenoprotein P', 'answer_start': 196}]}],\n",
       "   'context': 'Selenoprotein P expression, purification, and immunochemical characterization. Most selenoproteins contain a single selenocysteine residue per polypeptide chain, encoded by an in-frame UGA codon. Selenoprotein P is unique in that its mRNA encodes 10-12 selenocysteine residues, depending on species. In addition to the high number of selenocysteines, the protein is cysteine- and histidine-rich. The function of selenoprotein P has remained elusive, in part due to the inability to express the recombinant protein. This has been attributed to presumed inefficient translation through the selenocysteine/stop codons. Herein, we report for the first time the expression of recombinant rat selenoprotein P in a transiently transfected human epithelial kidney cell line, as well as the endogenously expressed protein from HepG2 and Chinese hamster ovary cells. The majority of the expressed protein migrates with the predicted 57-kDa size of full-length glycosylated selenoprotein P. Based on the histidine-rich nature of selenoprotein P, we have purified the recombinant and endogenously expressed proteins using nickel-agarose affinity chromatography. We show that the recombinant rat and endogenous human proteins react in Western blotting and immunoprecipitation assays with commercial anti-histidine antibodies. The ability to express, purify, and immunochemically detect the recombinant protein provides a foundation for investigating the functions and efficiency of expression of this intriguing protein.'},\n",
       "  {'qas': [{'id': '5343caffaeec6fbd07000002_006',\n",
       "     'question': 'Which is the human selenoprotein that contains several Se-Cys residues?',\n",
       "     'answers': [{'text': 'selenoprotein P', 'answer_start': 44}]}],\n",
       "   'context': \"Molecular cloning of cDNA encoding a bovine selenoprotein P-like protein containing 12 selenocysteines and a (His-Pro) rich domain insertion, and its regional expression. When cDNA containing proteins enriched in the bovine cerebellar cortex were cloned, a clone which seemed to encode a selenoprotein P-like protein was isolated. The coding nucleotide sequence of its cDNA insert displayed high homology to rat and human selenoprotein P cDNA but contained 12 rather than 10 TGAs (12 rather than 10 selenocysteines in deduced amino acids), a tandem repeat of one CACTCC (His-Ser) and seven CATCCCs (His-Pro), and a 3' untranslated region approximately 890 bases shorter than that of rat liver selenoprotein P. RT-PCR using a set of primers flanking to the repeat displayed the existence of mRNA without the repeat. The tandem repeat and its adjacent region consisted of a similar motif of CAC/TCC/AC/T. Thus, these proteins included a (His-Pro) rich domain with a slightly negative free energy change irrespective of having the tandem repeat or not. Such His-Pro repeats reportedly exist in the segmentation gene paired or homeobox protein Om(1D) of Drosophila. Moreover, both this selenoprotein P-like protein mRNA and selenoprotein P mRNA were expressed in all the areas of the brain but most prominently in the cerebellar cortex, hippocampus, and olfactory bulb. These findings suggest the possibility that these selenoproteins are major selenium carriers in the brain and play a role in the morphological response of nerve or glial cells.\"},\n",
       "  {'qas': [{'id': '5343caffaeec6fbd07000002_007',\n",
       "     'question': 'Which is the human selenoprotein that contains several Se-Cys residues?',\n",
       "     'answers': [{'text': 'selenoprotein P', 'answer_start': 422}]}],\n",
       "   'context': \"Molecular cloning of cDNA encoding a bovine selenoprotein P-like protein containing 12 selenocysteines and a (His-Pro) rich domain insertion, and its regional expression. When cDNA containing proteins enriched in the bovine cerebellar cortex were cloned, a clone which seemed to encode a selenoprotein P-like protein was isolated. The coding nucleotide sequence of its cDNA insert displayed high homology to rat and human selenoprotein P cDNA but contained 12 rather than 10 TGAs (12 rather than 10 selenocysteines in deduced amino acids), a tandem repeat of one CACTCC (His-Ser) and seven CATCCCs (His-Pro), and a 3' untranslated region approximately 890 bases shorter than that of rat liver selenoprotein P. RT-PCR using a set of primers flanking to the repeat displayed the existence of mRNA without the repeat. The tandem repeat and its adjacent region consisted of a similar motif of CAC/TCC/AC/T. Thus, these proteins included a (His-Pro) rich domain with a slightly negative free energy change irrespective of having the tandem repeat or not. Such His-Pro repeats reportedly exist in the segmentation gene paired or homeobox protein Om(1D) of Drosophila. Moreover, both this selenoprotein P-like protein mRNA and selenoprotein P mRNA were expressed in all the areas of the brain but most prominently in the cerebellar cortex, hippocampus, and olfactory bulb. These findings suggest the possibility that these selenoproteins are major selenium carriers in the brain and play a role in the morphological response of nerve or glial cells.\"},\n",
       "  {'qas': [{'id': '56a39d60496b62f23f000006_001',\n",
       "     'question': 'Which package is available for analysing genomic interactions in R/Bioconductor?',\n",
       "     'answers': [{'text': 'r3Cseq', 'answer_start': 0}]}],\n",
       "   'context': 'r3Cseq: an R/Bioconductor package for the discovery of long-range genomic interactions from chromosome conformation capture and next-generation sequencing data. The coupling of chromosome conformation capture (3C) with next-generation sequencing technologies enables the high-throughput detection of long-range genomic interactions, via the generation of ligation products between DNA sequences, which are closely juxtaposed in vivo. These interactions involve promoter regions, enhancers and other regulatory and structural elements of chromosomes and can reveal key details of the regulation of gene expression. 3C-seq is a variant of the method for the detection of interactions between one chosen genomic element (viewpoint) and the rest of the genome. We present r3Cseq, an R/Bioconductor package designed to perform 3C-seq data analysis in a number of different experimental designs. The package reads a common aligned read input format, provides data normalization, allows the visualization of candidate interaction regions and detects statistically significant chromatin interactions, thus greatly facilitating hypothesis generation and the interpretation of experimental results. We further demonstrate its use on a series of real-world applications.'},\n",
       "  {'qas': [{'id': '56a39d60496b62f23f000006_002',\n",
       "     'question': 'Which package is available for analysing genomic interactions in R/Bioconductor?',\n",
       "     'answers': [{'text': 'r3Cseq', 'answer_start': 768}]}],\n",
       "   'context': 'r3Cseq: an R/Bioconductor package for the discovery of long-range genomic interactions from chromosome conformation capture and next-generation sequencing data. The coupling of chromosome conformation capture (3C) with next-generation sequencing technologies enables the high-throughput detection of long-range genomic interactions, via the generation of ligation products between DNA sequences, which are closely juxtaposed in vivo. These interactions involve promoter regions, enhancers and other regulatory and structural elements of chromosomes and can reveal key details of the regulation of gene expression. 3C-seq is a variant of the method for the detection of interactions between one chosen genomic element (viewpoint) and the rest of the genome. We present r3Cseq, an R/Bioconductor package designed to perform 3C-seq data analysis in a number of different experimental designs. The package reads a common aligned read input format, provides data normalization, allows the visualization of candidate interaction regions and detects statistically significant chromatin interactions, thus greatly facilitating hypothesis generation and the interpretation of experimental results. We further demonstrate its use on a series of real-world applications.'},\n",
       "  {'qas': [{'id': '56a39d60496b62f23f000006_003',\n",
       "     'question': 'Which package is available for analysing genomic interactions in R/Bioconductor?',\n",
       "     'answers': [{'text': 'r3Cseq', 'answer_start': 0}]}],\n",
       "   'context': 'r3Cseq: an R/Bioconductor package for the discovery of long-range genomic interactions from chromosome conformation capture and next-generation sequencing data. The coupling of chromosome conformation capture (3C) with next-generation sequencing technologies enables the high-throughput detection of long-range genomic interactions, via the generation of ligation products between DNA sequences, which are closely juxtaposed in vivo. These interactions involve promoter regions, enhancers and other regulatory and structural elements of chromosomes and can reveal key details of the regulation of gene expression. 3C-seq is a variant of the method for the detection of interactions between one chosen genomic element (viewpoint) and the rest of the genome. We present r3Cseq, an R/Bioconductor package designed to perform 3C-seq data analysis in a number of different experimental designs. The package reads a common aligned read input format, provides data normalization, allows the visualization of candidate interaction regions and detects statistically significant chromatin interactions, thus greatly facilitating hypothesis generation and the interpretation of experimental results. We further demonstrate its use on a series of real-world applications.'},\n",
       "  {'qas': [{'id': '5150b807d24251bc05000072_001',\n",
       "     'question': 'How many clinical trials for off-label drugs in neonates are cited in the literature.',\n",
       "     'answers': [{'text': '0', 'answer_start': 455}]}],\n",
       "   'context': 'Paediatric investigation plans for pain: painfully slow! PURPOSE: To examine the early impact of the Paediatric Regulation, which entered into force in Europe on 27 January 2007, on the development of pharmaceutical drugs in the therapeutic field of pain submitted to the Paediatric Committee (PDCO) and to the European Medicines Agency (EMA). METHODS: Paediatric Investigations Plans (PIPs) submitted with a Decision (outcome) reached between September 2007 and March 2010 were included in the analysis. RESULTS: Of the 17 Paediatric Investigation Plans submitted, 14 have resulted in an EMA Decision, 3 were withdrawn by the applicants, 8 were granted a full waiver from development, and 1 resulted in a negative opinion. Decisions as issued included 15 clinical trials, with at least 1,282 children to be recruited into studies across five different products. Neonates were included in four of the products. CONCLUSIONS: The small number of submissions indicates a lack of new drugs being developed for the management of pain. Ethical concerns that too many vulnerable children will be recruited into clinical trials must be balanced against limiting the number of off-label prescribing and obtaining age-appropriate information on paediatric use. Now is an opportune time for clinicians, academics, learned societies and industry to collaborate for the benefit of children in pain.'},\n",
       "  {'qas': [{'id': '5150b807d24251bc05000072_002',\n",
       "     'question': 'How many clinical trials for off-label drugs in neonates are cited in the literature.',\n",
       "     'answers': [{'text': '0', 'answer_start': 456}]}],\n",
       "   'context': 'Paediatric investigation plans for pain: painfully slow! PURPOSE: To examine the early impact of the Paediatric Regulation, which entered into force in Europe on 27 January 2007, on the development of pharmaceutical drugs in the therapeutic field of pain submitted to the Paediatric Committee (PDCO) and to the European Medicines Agency (EMA). METHODS: Paediatric Investigations Plans (PIPs) submitted with a Decision (outcome) reached between September 2007 and March 2010 were included in the analysis. RESULTS: Of the 17 Paediatric Investigation Plans submitted, 14 have resulted in an EMA Decision, 3 were withdrawn by the applicants, 8 were granted a full waiver from development, and 1 resulted in a negative opinion. Decisions as issued included 15 clinical trials, with at least 1,282 children to be recruited into studies across five different products. Neonates were included in four of the products. CONCLUSIONS: The small number of submissions indicates a lack of new drugs being developed for the management of pain. Ethical concerns that too many vulnerable children will be recruited into clinical trials must be balanced against limiting the number of off-label prescribing and obtaining age-appropriate information on paediatric use. Now is an opportune time for clinicians, academics, learned societies and industry to collaborate for the benefit of children in pain.'},\n",
       "  {'qas': [{'id': '5150b807d24251bc05000072_003',\n",
       "     'question': 'How many clinical trials for off-label drugs in neonates are cited in the literature.',\n",
       "     'answers': [{'text': '0', 'answer_start': 470}]}],\n",
       "   'context': 'Paediatric investigation plans for pain: painfully slow! PURPOSE: To examine the early impact of the Paediatric Regulation, which entered into force in Europe on 27 January 2007, on the development of pharmaceutical drugs in the therapeutic field of pain submitted to the Paediatric Committee (PDCO) and to the European Medicines Agency (EMA). METHODS: Paediatric Investigations Plans (PIPs) submitted with a Decision (outcome) reached between September 2007 and March 2010 were included in the analysis. RESULTS: Of the 17 Paediatric Investigation Plans submitted, 14 have resulted in an EMA Decision, 3 were withdrawn by the applicants, 8 were granted a full waiver from development, and 1 resulted in a negative opinion. Decisions as issued included 15 clinical trials, with at least 1,282 children to be recruited into studies across five different products. Neonates were included in four of the products. CONCLUSIONS: The small number of submissions indicates a lack of new drugs being developed for the management of pain. Ethical concerns that too many vulnerable children will be recruited into clinical trials must be balanced against limiting the number of off-label prescribing and obtaining age-appropriate information on paediatric use. Now is an opportune time for clinicians, academics, learned societies and industry to collaborate for the benefit of children in pain.'},\n",
       "  {'qas': [{'id': '5150b807d24251bc05000072_004',\n",
       "     'question': 'How many clinical trials for off-label drugs in neonates are cited in the literature.',\n",
       "     'answers': [{'text': '0', 'answer_start': 472}]}],\n",
       "   'context': 'Paediatric investigation plans for pain: painfully slow! PURPOSE: To examine the early impact of the Paediatric Regulation, which entered into force in Europe on 27 January 2007, on the development of pharmaceutical drugs in the therapeutic field of pain submitted to the Paediatric Committee (PDCO) and to the European Medicines Agency (EMA). METHODS: Paediatric Investigations Plans (PIPs) submitted with a Decision (outcome) reached between September 2007 and March 2010 were included in the analysis. RESULTS: Of the 17 Paediatric Investigation Plans submitted, 14 have resulted in an EMA Decision, 3 were withdrawn by the applicants, 8 were granted a full waiver from development, and 1 resulted in a negative opinion. Decisions as issued included 15 clinical trials, with at least 1,282 children to be recruited into studies across five different products. Neonates were included in four of the products. CONCLUSIONS: The small number of submissions indicates a lack of new drugs being developed for the management of pain. Ethical concerns that too many vulnerable children will be recruited into clinical trials must be balanced against limiting the number of off-label prescribing and obtaining age-appropriate information on paediatric use. Now is an opportune time for clinicians, academics, learned societies and industry to collaborate for the benefit of children in pain.'},\n",
       "  {'qas': [{'id': '56c1d857ef6e394741000033_001',\n",
       "     'question': 'What enzyme is inhibied by Opicapone?',\n",
       "     'answers': [{'text': 'catechol-O-methyltransferase',\n",
       "       'answer_start': 110}]}],\n",
       "   'context': \"Effect of moderate liver impairment on the pharmacokinetics of opicapone. PURPOSE: Opicapone (OPC) is a novel catechol-O-methyltransferase (COMT) inhibitor to be used as adjunctive therapy in levodopa-treated patients with Parkinson's disease. The purpose of this study was to evaluate the effect of moderate liver impairment on the pharmacokinetics (PK) and pharmacodynamics (PD; effect on COMT activity) of OPC. METHODS: An open-label, parallel-group study in patients (n = 8) with moderate liver impairment (Child-Pugh category B, score of 7 to 9) and matched healthy subjects (n = 8, control) with normal liver function. All subjects received a single 50-mg oral dose of OPC, with plasma and urine concentrations of opicapone and its metabolites measured up to 72\\xa0h post-dose, including soluble COMT (S-COMT) activity. A one-way analysis of variance (ANOVA) was used to compare the main PK and PD parameters between groups. Point estimates (PE) of geometric mean ratios (GMR) and corresponding 90\\xa0% confidence intervals (90%CI) for the ratio hepatic/control subjects of each parameter were calculated and compared with the reference interval (80-125\\xa0%). RESULTS: Exposure to opicapone (AUC and Cmax) increased significantly in patients with moderate hepatic impairment (PE [90%CI]: AUC0-∞, 184\\xa0% [135-250\\xa0%]; Cmax, 189\\xa0% [144-249\\xa0%]). Although apparent total clearance (CL/F) of opicapone was decreased by ∼35\\xa0%, similar elimination half-life and unbound/bound fractions of opicapone were observed between the two groups. Both rate and extent of exposure to BIA 9-1103 were higher in the hepatically impaired group, but not statistically significant compared with the control group. Similar to the parent (opicapone), the observed increase in exposure to BIA 9-1106 was statistically significant in the hepatically impaired group over the control group. BIA 9-1106 was the only metabolite detected in urine and its urine PK parameters were in accordance with plasma data. Maximum S-COMT inhibition (Emax) occurred earlier for the hepatically impaired group with values of 100\\xa0% and 91.2\\xa0% for the hepatically impaired and control groups respectively. Both Emax and AUEC for the hepatically impaired group reached statistical significance over the control group. OPC was well tolerated in both hepatically impaired and control groups. CONCLUSION: The bioavailability of an orally administered single dose of 50\\xa0mg OPC was significantly higher in patients with moderate chronic hepatic impairment, perhaps by a reduced first-pass effect. As the tolerability profile of OPC was favourable under the conditions of this study and its exposure is completely purged from systemic circulation before the subsequent dose administration, no OPC dose adjustment is needed in patients with mild to moderate chronic hepatic impairment. However, as OPC is under clinical development for use as adjunctive therapy in levodopa-treated patients with Parkinson's disease, an adjustment of levodopa and/or OPC regimens in patients should be carefully considered based on a potentially enhanced levodopa dopaminergic response and the associated tolerability.\"},\n",
       "  {'qas': [{'id': '56c1d857ef6e394741000033_002',\n",
       "     'question': 'What enzyme is inhibied by Opicapone?',\n",
       "     'answers': [{'text': 'catechol-O-methyltransferase',\n",
       "       'answer_start': 270}]}],\n",
       "   'context': \"Brain and peripheral pharmacokinetics of levodopa in the cynomolgus monkey following administration of opicapone, a third generation nitrocatechol COMT inhibitor. OBJECTIVE: The present study aimed at evaluating the effect of opicapone, a third generation nitrocatechol catechol-O-methyltransferase (COMT) inhibitor, on the systemic and central bioavailability of 3,4-dihydroxy-l-phenylalanine (levodopa) and related metabolites in the cynomolgus monkey. METHODS: Four monkeys, implanted with guiding cannulas for microdialysis probes, in the substantia nigra, dorsal striatum and prefrontal cortex, were randomized in two groups that received, in a crossover design, vehicle or 100 mg/kg opicapone for 14 days. Twenty-three hours after last administration of vehicle or opicapone, animals were challenged with levodopa/benserazide (12/3 mg/kg). Extracellular dialysate and blood samples were collected over 360 min (at 30 min intervals) for the assays of catecholamine and COMT activity. RESULTS: Opicapone increased levodopa systemic exposure by 2-fold not changing Cmax values and reduced both 3-O-methyldopa (3-OMD) exposure and Cmax values by 5-fold. These changes were accompanied by ∼76-84% reduction in erythrocyte COMT activity. In dorsal striatum and substantia nigra, opicapone increased levodopa exposure by 1.7- and 1.4-fold, respectively, reducing 3-OMD exposure by 5- and 7-fold respectively. DOPAC exposure was increased by 4-fold in the substantia nigra. In the prefrontal cortex, opicapone increased levodopa exposure and reduced 3-OMD levels by 2.3- and 2.4-fold, respectively. CONCLUSIONS: Opicapone behaved as long-acting COMT inhibitor that markedly increased systemic and central levodopa bioavailability. Opicapone is a strong candidate to fill the unmet need for COMT inhibitors that lead to more sustained levodopa levels in Parkinson's disease patients.\"},\n",
       "  {'qas': [{'id': '56c1d857ef6e394741000033_003',\n",
       "     'question': 'What enzyme is inhibied by Opicapone?',\n",
       "     'answers': [{'text': 'catechol-O-methyltransferase',\n",
       "       'answer_start': 176}]}],\n",
       "   'context': \"Effect of opicapone and entacapone upon levodopa pharmacokinetics during three daily levodopa administrations. BACKGROUND AND OBJECTIVES: Opicapone is a novel third generation catechol-O-methyltransferase (COMT) inhibitor. The purpose of this study was to compare the levodopa pharmacokinetic profile throughout a day driven by the COMT inhibition either following repeated doses of opicapone or concomitant administration with entacapone. METHODS: A randomized, double-blind, gender-balanced, parallel-group study was performed in 4 groups of 20 healthy subjects each. Four subjects in each group received placebo during the entire study. Sixteen subjects in one group received placebo once daily for 11 days and on day 12, 200 mg entacapone concomitantly with each levodopa/carbidopa dose (three times separated by a 5-h interval). Sixteen subjects in each of the remaining three groups received respectively 25, 50, and 75 mg opicapone once daily for 11 days and on day 12, placebo concomitantly with each levodopa/carbidopa dose. RESULTS: Levodopa minimum plasma concentration (Cmin) for each levodopa/carbidopa dose and for the mean of all levodopa/carbidopa doses increased substantially with all active treatments (entacapone and opicapone) when compared to the control group (placebo), with values ranging from 1.7-fold (200 mg entacapone) to 3.3-fold (75 mg opicapone). No statistical difference was found for levodopa peak of systemic exposure (as assessed by maximum observed plasma concentration (Cmax)) between all active treatments and placebo. A significant increase in the levodopa extent of systemic exposure (as assessed by concentration-time curve (AUC)) occurred with all opicapone treatments in relation to placebo. No statistical difference was found for levodopa AUC when entacapone was compared to placebo. When compared to entacapone, both 50 and 75 mg opicapone presented a significant increase for the levodopa AUC. All active treatments significantly inhibited both peak (as assessed by Emax) and extent (as assessed by effect-time curve (AUEC)) of the COMT activity in relation to placebo. When compared to entacapone, all opicapone treatments significantly decreased the extent (AUEC) of the COMT activity due to a long-lasting and sustained effect. The tolerability profile was favorable for all active treatments. CONCLUSION: Opicapone, a novel third generation COMT inhibitor, when compared to entacapone, provides a superior response upon the bioavailability of levodopa associated to more pronounced, long-lasting, and sustained COMT inhibition. The tolerability profile was favorable. On the basis of the results presented in this study and along with the earlier pharmacology studies, it is anticipated that opicapone adjunct therapy at the dosages of 25 and 50 mg will provide an enhancement in levodopa availability that will translate into clinical benefit for Parkinson's disease patients.\"},\n",
       "  {'qas': [{'id': '56c1d857ef6e394741000033_004',\n",
       "     'question': 'What enzyme is inhibied by Opicapone?',\n",
       "     'answers': [{'text': 'catechol-O-methyltransferase',\n",
       "       'answer_start': 74}]}],\n",
       "   'context': 'Pharmacokinetics, pharmacodynamics and tolerability of opicapone, a novel catechol-O-methyltransferase inhibitor, in healthy subjects: prediction of slow enzyme-inhibitor complex dissociation of a short-living and very long-acting inhibitor. BACKGROUND AND OBJECTIVES: Opicapone is a novel catechol-O-methyltransferase (COMT) inhibitor. The purpose of this study was to evaluate the tolerability, pharmacokinetics (including the effect of food) and pharmacodynamics (effect on COMT activity) following single oral doses of opicapone in young healthy male volunteers. METHODS: Single rising oral doses of opicapone (10, 25, 50, 100, 200, 400, 800 and 1,200 mg) were administered to eight groups of eight subjects per group (two subjects randomized to placebo and six subjects to opicapone), under a double-blind, randomized, placebo-controlled design. In an additional group of 12 subjects, a 50 mg single dose of opicapone was administered on two occasions, once having fasted overnight and once with a high-fat high-calorie meal. RESULTS: Opicapone was well tolerated at all doses tested. The extent of systemic exposure (area under the plasma concentration-time curve and maximum plasma concentration) to opicapone and metabolites increased in an approximately dose-proportional manner and showed a decrease following concomitant ingestion of a high-fat high-calorie meal. The apparent terminal elimination half-life of opicapone was 0.8-3.2 h. Sulphation appeared to be the main metabolic pathway for opicapone, and both opicapone and the main sulphated metabolite are likely excreted by the biliary route. Maximum COMT inhibition by opicapone was dose dependent, ranged from 36.1% (10 mg) to 100% (200 mg and above), and reached statistical significance at all doses tested. The long duration of COMT inhibition by opicapone, however, tended to be independent from the dose taken. The observed half-life of opicapone-induced COMT inhibition in human erythrocytes was 61.6 h (standard deviation [SD] = 37.6 h), which reflects an underlying dissociative process with a kinetic rate constant of 3.1 × 10(-6) s(-1) (SD = 1.9 × 10(-6) s(-1)). Such a process compares well to the estimated dissociation rate constant (k(off)) of the COMT-opicapone molecular complex (k(off) = 1.9 × 10(-6) s(-1)). CONCLUSIONS: Opicapone was well-tolerated and presented dose-proportional kinetics. Opicapone demonstrated marked and sustained inhibition of erythrocyte soluble COMT activity. Based on the observation that the half-life of COMT inhibition is independent of the dose and that it reflects an underlying kinetic process that is consistent with the k(off) value of the COMT-opicapone complex, we propose that the sustained COMT inhibition, far beyond the observable point of clearance of circulating drug, is due to the long residence time of the reversible complex formed between COMT and opicapone. Globally, these promising results provide a basis for further clinical development of opicapone.'},\n",
       "  {'qas': [{'id': '56c1d857ef6e394741000033_005',\n",
       "     'question': 'What enzyme is inhibied by Opicapone?',\n",
       "     'answers': [{'text': 'catechol-O-methyltransferase',\n",
       "       'answer_start': 290}]}],\n",
       "   'context': 'Pharmacokinetics, pharmacodynamics and tolerability of opicapone, a novel catechol-O-methyltransferase inhibitor, in healthy subjects: prediction of slow enzyme-inhibitor complex dissociation of a short-living and very long-acting inhibitor. BACKGROUND AND OBJECTIVES: Opicapone is a novel catechol-O-methyltransferase (COMT) inhibitor. The purpose of this study was to evaluate the tolerability, pharmacokinetics (including the effect of food) and pharmacodynamics (effect on COMT activity) following single oral doses of opicapone in young healthy male volunteers. METHODS: Single rising oral doses of opicapone (10, 25, 50, 100, 200, 400, 800 and 1,200 mg) were administered to eight groups of eight subjects per group (two subjects randomized to placebo and six subjects to opicapone), under a double-blind, randomized, placebo-controlled design. In an additional group of 12 subjects, a 50 mg single dose of opicapone was administered on two occasions, once having fasted overnight and once with a high-fat high-calorie meal. RESULTS: Opicapone was well tolerated at all doses tested. The extent of systemic exposure (area under the plasma concentration-time curve and maximum plasma concentration) to opicapone and metabolites increased in an approximately dose-proportional manner and showed a decrease following concomitant ingestion of a high-fat high-calorie meal. The apparent terminal elimination half-life of opicapone was 0.8-3.2 h. Sulphation appeared to be the main metabolic pathway for opicapone, and both opicapone and the main sulphated metabolite are likely excreted by the biliary route. Maximum COMT inhibition by opicapone was dose dependent, ranged from 36.1% (10 mg) to 100% (200 mg and above), and reached statistical significance at all doses tested. The long duration of COMT inhibition by opicapone, however, tended to be independent from the dose taken. The observed half-life of opicapone-induced COMT inhibition in human erythrocytes was 61.6 h (standard deviation [SD] = 37.6 h), which reflects an underlying dissociative process with a kinetic rate constant of 3.1 × 10(-6) s(-1) (SD = 1.9 × 10(-6) s(-1)). Such a process compares well to the estimated dissociation rate constant (k(off)) of the COMT-opicapone molecular complex (k(off) = 1.9 × 10(-6) s(-1)). CONCLUSIONS: Opicapone was well-tolerated and presented dose-proportional kinetics. Opicapone demonstrated marked and sustained inhibition of erythrocyte soluble COMT activity. Based on the observation that the half-life of COMT inhibition is independent of the dose and that it reflects an underlying kinetic process that is consistent with the k(off) value of the COMT-opicapone complex, we propose that the sustained COMT inhibition, far beyond the observable point of clearance of circulating drug, is due to the long residence time of the reversible complex formed between COMT and opicapone. Globally, these promising results provide a basis for further clinical development of opicapone.'},\n",
       "  {'qas': [{'id': '56c1d857ef6e394741000033_006',\n",
       "     'question': 'What enzyme is inhibied by Opicapone?',\n",
       "     'answers': [{'text': 'catechol-O-methyltransferase',\n",
       "       'answer_start': 176}]}],\n",
       "   'context': \"Effect of opicapone and entacapone upon levodopa pharmacokinetics during three daily levodopa administrations. BACKGROUND AND OBJECTIVES: Opicapone is a novel third generation catechol-O-methyltransferase (COMT) inhibitor. The purpose of this study was to compare the levodopa pharmacokinetic profile throughout a day driven by the COMT inhibition either following repeated doses of opicapone or concomitant administration with entacapone. METHODS: A randomized, double-blind, gender-balanced, parallel-group study was performed in 4 groups of 20 healthy subjects each. Four subjects in each group received placebo during the entire study. Sixteen subjects in one group received placebo once daily for 11 days and on day 12, 200 mg entacapone concomitantly with each levodopa/carbidopa dose (three times separated by a 5-h interval). Sixteen subjects in each of the remaining three groups received respectively 25, 50, and 75 mg opicapone once daily for 11 days and on day 12, placebo concomitantly with each levodopa/carbidopa dose. RESULTS: Levodopa minimum plasma concentration (Cmin) for each levodopa/carbidopa dose and for the mean of all levodopa/carbidopa doses increased substantially with all active treatments (entacapone and opicapone) when compared to the control group (placebo), with values ranging from 1.7-fold (200 mg entacapone) to 3.3-fold (75 mg opicapone). No statistical difference was found for levodopa peak of systemic exposure (as assessed by maximum observed plasma concentration (Cmax)) between all active treatments and placebo. A significant increase in the levodopa extent of systemic exposure (as assessed by concentration-time curve (AUC)) occurred with all opicapone treatments in relation to placebo. No statistical difference was found for levodopa AUC when entacapone was compared to placebo. When compared to entacapone, both 50 and 75 mg opicapone presented a significant increase for the levodopa AUC. All active treatments significantly inhibited both peak (as assessed by Emax) and extent (as assessed by effect-time curve (AUEC)) of the COMT activity in relation to placebo. When compared to entacapone, all opicapone treatments significantly decreased the extent (AUEC) of the COMT activity due to a long-lasting and sustained effect. The tolerability profile was favorable for all active treatments. CONCLUSION: Opicapone, a novel third generation COMT inhibitor, when compared to entacapone, provides a superior response upon the bioavailability of levodopa associated to more pronounced, long-lasting, and sustained COMT inhibition. The tolerability profile was favorable. On the basis of the results presented in this study and along with the earlier pharmacology studies, it is anticipated that opicapone adjunct therapy at the dosages of 25 and 50 mg will provide an enhancement in levodopa availability that will translate into clinical benefit for Parkinson's disease patients.\"},\n",
       "  {'qas': [{'id': '56c1d857ef6e394741000033_007',\n",
       "     'question': 'What enzyme is inhibied by Opicapone?',\n",
       "     'answers': [{'text': 'catechol-O-methyltransferase',\n",
       "       'answer_start': 273}]}],\n",
       "   'context': 'Opicapone: a short lived and very long acting novel catechol-O-methyltransferase inhibitor following multiple dose administration in healthy subjects. AIMS: The aim of this study was to assess the tolerability, pharmacokinetics and inhibitory effect on erythrocyte soluble catechol-O-methyltransferase (S-COMT) activity following repeated doses of opicapone. METHODS: This randomized, placebo-controlled, double-blind study enrolled healthy male subjects who received either once daily placebo or opicapone 5, 10, 20 or 30 mg for 8 days. RESULTS: Opicapone was well tolerated. Its systemic exposure increased in an approximately dose-proportional manner with an apparent terminal half-life of 1.0 to 1.4 h. Sulphation was the main metabolic pathway. Opicapone metabolites recovered in urine accounted for less than 3% of the amount of opicapone administered suggesting that bile is likely the main route of excretion. Maximum S-COMT inhibition (Emax ) ranged from 69.9% to 98.0% following the last dose of opicapone. The opicapone-induced S-COMT inhibition showed a half-life in excess of 100 h, which was dose-independent and much longer than plasma drug exposure. Such a half-life translates into a putative underlying rate constant that is comparable with the estimated dissociation rate constant of the COMT-opicapone complex. CONCLUSION: Despite its short elimination half-life, opicapone markedly and sustainably inhibited erythrocyte S-COMT activity making it suitable for a once daily regimen.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_001',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 469}]}],\n",
       "   'context': 'Accelerated growth in the absence of DNA replication origins. DNA replication initiates at defined sites called origins, which serve as binding sites for initiator proteins that recruit the replicative machinery. Origins differ in number and structure across the three domains of life and their properties determine the dynamics of chromosome replication. Bacteria and some archaea replicate from single origins, whereas most archaea and all eukaryotes replicate using multiple origins. Initiation mechanisms that rely on homologous recombination operate in some viruses. Here we show that such mechanisms also operate in archaea. We use deep sequencing to study replication in Haloferax volcanii and identify four chromosomal origins of differing activity. Deletion of individual origins results in perturbed replication dynamics and reduced growth. However, a strain lacking all origins has no apparent defects and grows significantly faster than wild type. Origin-less cells initiate replication at dispersed sites rather than at discrete origins and have an absolute requirement for the recombinase RadA, unlike strains lacking individual origins. Our results demonstrate that homologous recombination alone can efficiently initiate the replication of an entire cellular genome. This raises the question of what purpose replication origins serve and why they have evolved.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_002',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 90}]}],\n",
       "   'context': \"Specificity and function of archaeal DNA replication initiator proteins. Chromosomes with multiple DNA replication origins are a hallmark of Eukaryotes and some Archaea. All eukaryal nuclear replication origins are defined by the origin recognition complex (ORC) that recruits the replicative helicase MCM(2-7) via Cdc6 and Cdt1. We find that the three origins in the single chromosome of the archaeon Sulfolobus islandicus are specified by distinct initiation factors. While two origins are dependent on archaeal homologs of eukaryal Orc1 and Cdc6, the third origin is instead reliant on an archaeal Cdt1 homolog. We exploit the nonessential nature of the orc1-1 gene to investigate the role of ATP binding and hydrolysis in initiator function in\\xa0vivo and in\\xa0vitro. We find that the ATP-bound form of Orc1-1 is proficient for replication and implicates hydrolysis of ATP in downregulation of origin activity. Finally, we reveal that ATP and DNA binding by Orc1-1 remodels the protein's structure rather than that of the DNA template.\"},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_003',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'Multiple', 'answer_start': 493}]}],\n",
       "   'context': 'Diversity and evolution of multiple orc/cdc6-adjacent replication origins in haloarchaea. BACKGROUND: While multiple replication origins have been observed in archaea, considerably less is known about their evolutionary processes. Here, we performed a comparative analysis of the predicted (proved in part) orc/cdc6-associated replication origins in 15 completely sequenced haloarchaeal genomes to investigate the diversity and evolution of replication origins in halophilic Archaea. RESULTS: Multiple orc/cdc6-associated replication origins were predicted in all of the analyzed haloarchaeal genomes following the identification of putative ORBs (origin recognition boxes) that are associated with orc/cdc6 genes. Five of these predicted replication origins in Haloarcula hispanica were experimentally confirmed via autonomous replication activities. Strikingly, several predicted replication origins in H. hispanica and Haloarcula marismortui are located in the distinct regions of their highly homologous chromosomes, suggesting that these replication origins might have been introduced as parts of new genomic content. A comparison of the origin-associated Orc/Cdc6 homologs and the corresponding predicted ORB elements revealed that the replication origins in a given haloarchaeon are quite diverse, while different haloarchaea can share a few conserved origins. Phylogenetic and genomic context analyses suggested that there is an original replication origin (oriC1) that was inherited from the ancestor of archaea, and several other origins were likely evolved and/or translocated within the haloarchaeal species. CONCLUSION: This study provides detailed information about the diversity of multiple orc/cdc6-associated replication origins in haloarchaeal genomes, and provides novel insight into the evolution of multiple replication origins in Archaea.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_004',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 799}]}],\n",
       "   'context': 'Replication-biased genome organisation in the crenarchaeon Sulfolobus. BACKGROUND: Species of the crenarchaeon Sulfolobus harbour three replication origins in their single circular chromosome that are synchronously initiated during replication. RESULTS: We demonstrate that global gene expression in two Sulfolobus species is highly biased, such that early replicating genome regions are more highly expressed at all three origins. The bias by far exceeds what would be anticipated by gene dosage effects alone. In addition, early replicating regions are denser in archaeal core genes (enriched in essential functions), display lower intergenic distances, and are devoid of mobile genetic elements. CONCLUSION: The strong replication-biased structuring of the Sulfolobus chromosome implies that the multiple replication origins serve purposes other than simply shortening the time required for replication. The higher-level chromosomal organisation could be of importance for minimizing the impact of DNA damage, and may also be linked to transcriptional regulation.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_005',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 437}]}],\n",
       "   'context': 'Extrachromosomal element capture and the evolution of multiple replication origins in archaeal chromosomes. In all three domains of life, DNA replication begins at specialized loci termed replication origins. In bacteria, replication initiates from a single, clearly defined site. In contrast, eukaryotic organisms exploit a multitude of replication origins, dividing their genomes into an array of short contiguous units. Recently, the multiple replication origin paradigm has also been demonstrated within the archaeal domain of life, with the discovery that the hyperthermophilic archaeon Sulfolobus has three replication origins. However, the evolutionary mechanism driving the progression from single to multiple origin usage remains unclear. Here, we demonstrate that Aeropyrum pernix, a distant relative of Sulfolobus, has two origins. Comparison with the Sulfolobus origins provides evidence for evolution of replicon complexity by capture of extrachromosomal genetic elements. We additionally identify a previously unrecognized candidate archaeal initiator protein that is distantly related to eukaryotic Cdt1. Our data thus provide evidence that horizontal gene transfer, in addition to its well-established role in contributing to the information content of chromosomes, may fundamentally alter the manner in which the host chromosome is replicated.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_006',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 47}]}],\n",
       "   'context': 'Archaeal cell cycle progress. The discovery of multiple chromosome replication origins in Sulfolobus species has added yet another eukaryotic trait to the archaea, and brought new levels of complexity to the cell cycle in terms of initiation of chromosome replication, replication termination and chromosome decatenation. Conserved repeated DNA elements--origin recognition boxes--have been identified in the origins of replication, and shown to bind the Orc1/Cdc6 proteins involved in cell cycle control. The origin recognition boxes aid in the identification and characterization of new origins, and their conservation suggests that most archaea have a similar replication initiation mechanism. Cell-cycle-dependent variation in Orc1/Cdc6 levels has been demonstrated, reminiscent of variations in cyclin levels during the eukaryotic cell cycle. Information about archaeal chromosome segregation is also accumulating, including the identification of a protein that binds to short regularly spaced repeats that might constitute centromere-like elements. In addition, studies of cell-cycle-specific gene expression have potential to reveal, in the near future, missing components in crenarchaeal chromosome replication, genome segregation and cell division. Together with an increased number of physiological and cytological investigations of the overall organization of the cell cycle, rapid progress of the archaeal cell cycle field is evident, and archaea, in particular Sulfolobus species, are emerging as simple and powerful models for the eukaryotic cell cycle.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_007',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 1233}]}],\n",
       "   'context': 'Multiple replication origins of the archaeon Halobacterium species NRC-1. The genomic sequence of the halophilic archaeon Halobacterium NRC-1 has been analyzed by the Z curve method. The Z curve is a three-dimensional curve that uniquely represents a given DNA sequence. Based on the known behaviors of the Z curves for the archaea whose replication origins have been identified, the analysis of the Z curve for the genome of Halobacterium NRC-1 strongly suggests that the large genome has two replication origins, oriC1 (921,863-922,014) and oriC2 (1,806,444-1,807,229), which are located at two sharp peaks of the Z curve. These two regions are next to the cdc6 genes and contain multiple copies of stretches of G and C, i.e., ggggtgggg and ccccacccc, which may also be regarded as direct and inverted repeats. Based on the above analysis, a model of replication of Halobacterium NRC-1 with two replication origins and two termini has been proposed. The experimental confirmation of this model would constitute the first example of multiple replication origins of archaea, which will finally provide much insight into the understanding of replication mechanisms of eukaryotic organisms, including human. In addition, the potential multiple replication origins of the archaeon Sulfolobus solfataricus are suggested by the analysis based on the Z curve method.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_008',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 108}]}],\n",
       "   'context': 'Diversity and evolution of multiple orc/cdc6-adjacent replication origins in haloarchaea. BACKGROUND: While multiple replication origins have been observed in archaea, considerably less is known about their evolutionary processes. Here, we performed a comparative analysis of the predicted (proved in part) orc/cdc6-associated replication origins in 15 completely sequenced haloarchaeal genomes to investigate the diversity and evolution of replication origins in halophilic Archaea. RESULTS: Multiple orc/cdc6-associated replication origins were predicted in all of the analyzed haloarchaeal genomes following the identification of putative ORBs (origin recognition boxes) that are associated with orc/cdc6 genes. Five of these predicted replication origins in Haloarcula hispanica were experimentally confirmed via autonomous replication activities. Strikingly, several predicted replication origins in H. hispanica and Haloarcula marismortui are located in the distinct regions of their highly homologous chromosomes, suggesting that these replication origins might have been introduced as parts of new genomic content. A comparison of the origin-associated Orc/Cdc6 homologs and the corresponding predicted ORB elements revealed that the replication origins in a given haloarchaeon are quite diverse, while different haloarchaea can share a few conserved origins. Phylogenetic and genomic context analyses suggested that there is an original replication origin (oriC1) that was inherited from the ancestor of archaea, and several other origins were likely evolved and/or translocated within the haloarchaeal species. CONCLUSION: This study provides detailed information about the diversity of multiple orc/cdc6-associated replication origins in haloarchaeal genomes, and provides novel insight into the evolution of multiple replication origins in Archaea.'},\n",
       "  {'qas': [{'id': '52fe58f82059c6d71c00007a_009',\n",
       "     'question': 'Do archaeal genomes contain one or multiple origins of replication?',\n",
       "     'answers': [{'text': 'multiple', 'answer_start': 1697}]}],\n",
       "   'context': 'Diversity and evolution of multiple orc/cdc6-adjacent replication origins in haloarchaea. BACKGROUND: While multiple replication origins have been observed in archaea, considerably less is known about their evolutionary processes. Here, we performed a comparative analysis of the predicted (proved in part) orc/cdc6-associated replication origins in 15 completely sequenced haloarchaeal genomes to investigate the diversity and evolution of replication origins in halophilic Archaea. RESULTS: Multiple orc/cdc6-associated replication origins were predicted in all of the analyzed haloarchaeal genomes following the identification of putative ORBs (origin recognition boxes) that are associated with orc/cdc6 genes. Five of these predicted replication origins in Haloarcula hispanica were experimentally confirmed via autonomous replication activities. Strikingly, several predicted replication origins in H. hispanica and Haloarcula marismortui are located in the distinct regions of their highly homologous chromosomes, suggesting that these replication origins might have been introduced as parts of new genomic content. A comparison of the origin-associated Orc/Cdc6 homologs and the corresponding predicted ORB elements revealed that the replication origins in a given haloarchaeon are quite diverse, while different haloarchaea can share a few conserved origins. Phylogenetic and genomic context analyses suggested that there is an original replication origin (oriC1) that was inherited from the ancestor of archaea, and several other origins were likely evolved and/or translocated within the haloarchaeal species. CONCLUSION: This study provides detailed information about the diversity of multiple orc/cdc6-associated replication origins in haloarchaeal genomes, and provides novel insight into the evolution of multiple replication origins in Archaea.'},\n",
       "  {'qas': [{'id': '56c5fd325795f9a73e000005_001',\n",
       "     'question': 'Which is the genetic basis of Spinal Muscular Atrophy (SMA)?',\n",
       "     'answers': [{'text': 'The molecular genetic basis of spinal muscular atrophy (SMA), an autosomal recessive neuromuscular disorder, is the loss of function of the survival motor neuron gene (SMN1)',\n",
       "       'answer_start': 51}]}],\n",
       "   'context': 'Spinal muscular atrophy and therapeutic prospects. The molecular genetic basis of spinal muscular atrophy (SMA), an autosomal recessive neuromuscular disorder, is the loss of function of the survival motor neuron gene (SMN1). The SMN2 gene, a nearly identical copy of SMN1, has been detected as a promising target for SMA therapy. Both genes are ubiquitously expressed and encode identical proteins, but markedly differ in their splicing patterns: While SMN1 produces full-length (FL)-SMN transcripts only, the majority of SMN2 transcripts lacks exon 7. Transcriptional SMN2 activation or modulation of its splicing pattern to increase FL-SMN levels is believed to be clinically beneficial and therefore a crucial challenge in SMA research. Drugs such as valproic acid, phenylbutyrate, sodium butyrate, M344 and SAHA that mainly act as histone deacetylase inhibitors can mediate both: they stimulate the SMN2 gene transcription and/or restore the splicing pattern, thereby elevating the levels of FL-SMN2 protein. Preliminary phase II clinical trials and individual experimental curative approaches SMA patients show promising results. However, phase III double-blind placebo controlled clinical trials have to finally prove the efficacy of these drugs.'},\n",
       "  {'qas': [{'id': '5319ac99b166e2b806000034_001',\n",
       "     'question': 'Which is the third subunit of the TSC1-TSC2 complex upstream of mTORC1?',\n",
       "     'answers': [{'text': 'TBC1D7', 'answer_start': 449}]}],\n",
       "   'context': 'TBC1D7 is a third subunit of the TSC1-TSC2 complex upstream of mTORC1. The tuberous sclerosis complex (TSC) tumor suppressors form the TSC1-TSC2 complex, which limits cell growth in response to poor growth conditions. Through its GTPase-activating protein (GAP) activity toward Rheb, this complex inhibits the mechanistic target of rapamycin (mTOR) complex 1 (mTORC1), a key promoter of cell growth. Here, we identify and biochemically characterize TBC1D7 as a stably associated and ubiquitous third core subunit of the TSC1-TSC2 complex. We demonstrate that the TSC1-TSC2-TBC1D7 (TSC-TBC) complex is the functional complex that senses specific cellular growth conditions and possesses Rheb-GAP activity. Sequencing analyses of samples from TSC patients suggest that TBC1D7 is unlikely to represent TSC3. TBC1D7 knockdown decreases the association of TSC1 and TSC2 leading to decreased Rheb-GAP activity, without effects on the localization of TSC2 to the lysosome. Like the other TSC-TBC components, TBC1D7 knockdown results in increased mTORC1 signaling, delayed induction of autophagy, and enhanced cell growth under poor growth conditions.'},\n",
       "  {'qas': [{'id': '5319ac99b166e2b806000034_002',\n",
       "     'question': 'Which is the third subunit of the TSC1-TSC2 complex upstream of mTORC1?',\n",
       "     'answers': [{'text': 'TBC1D7', 'answer_start': 573}]}],\n",
       "   'context': 'TBC1D7 is a third subunit of the TSC1-TSC2 complex upstream of mTORC1. The tuberous sclerosis complex (TSC) tumor suppressors form the TSC1-TSC2 complex, which limits cell growth in response to poor growth conditions. Through its GTPase-activating protein (GAP) activity toward Rheb, this complex inhibits the mechanistic target of rapamycin (mTOR) complex 1 (mTORC1), a key promoter of cell growth. Here, we identify and biochemically characterize TBC1D7 as a stably associated and ubiquitous third core subunit of the TSC1-TSC2 complex. We demonstrate that the TSC1-TSC2-TBC1D7 (TSC-TBC) complex is the functional complex that senses specific cellular growth conditions and possesses Rheb-GAP activity. Sequencing analyses of samples from TSC patients suggest that TBC1D7 is unlikely to represent TSC3. TBC1D7 knockdown decreases the association of TSC1 and TSC2 leading to decreased Rheb-GAP activity, without effects on the localization of TSC2 to the lysosome. Like the other TSC-TBC components, TBC1D7 knockdown results in increased mTORC1 signaling, delayed induction of autophagy, and enhanced cell growth under poor growth conditions.'},\n",
       "  {'qas': [{'id': '5319ac99b166e2b806000034_003',\n",
       "     'question': 'Which is the third subunit of the TSC1-TSC2 complex upstream of mTORC1?',\n",
       "     'answers': [{'text': 'TBC1D7', 'answer_start': 805}]}],\n",
       "   'context': 'TBC1D7 is a third subunit of the TSC1-TSC2 complex upstream of mTORC1. The tuberous sclerosis complex (TSC) tumor suppressors form the TSC1-TSC2 complex, which limits cell growth in response to poor growth conditions. Through its GTPase-activating protein (GAP) activity toward Rheb, this complex inhibits the mechanistic target of rapamycin (mTOR) complex 1 (mTORC1), a key promoter of cell growth. Here, we identify and biochemically characterize TBC1D7 as a stably associated and ubiquitous third core subunit of the TSC1-TSC2 complex. We demonstrate that the TSC1-TSC2-TBC1D7 (TSC-TBC) complex is the functional complex that senses specific cellular growth conditions and possesses Rheb-GAP activity. Sequencing analyses of samples from TSC patients suggest that TBC1D7 is unlikely to represent TSC3. TBC1D7 knockdown decreases the association of TSC1 and TSC2 leading to decreased Rheb-GAP activity, without effects on the localization of TSC2 to the lysosome. Like the other TSC-TBC components, TBC1D7 knockdown results in increased mTORC1 signaling, delayed induction of autophagy, and enhanced cell growth under poor growth conditions.'},\n",
       "  {'qas': [{'id': '5319ac99b166e2b806000034_004',\n",
       "     'question': 'Which is the third subunit of the TSC1-TSC2 complex upstream of mTORC1?',\n",
       "     'answers': [{'text': 'TBC1D7', 'answer_start': 1001}]}],\n",
       "   'context': 'TBC1D7 is a third subunit of the TSC1-TSC2 complex upstream of mTORC1. The tuberous sclerosis complex (TSC) tumor suppressors form the TSC1-TSC2 complex, which limits cell growth in response to poor growth conditions. Through its GTPase-activating protein (GAP) activity toward Rheb, this complex inhibits the mechanistic target of rapamycin (mTOR) complex 1 (mTORC1), a key promoter of cell growth. Here, we identify and biochemically characterize TBC1D7 as a stably associated and ubiquitous third core subunit of the TSC1-TSC2 complex. We demonstrate that the TSC1-TSC2-TBC1D7 (TSC-TBC) complex is the functional complex that senses specific cellular growth conditions and possesses Rheb-GAP activity. Sequencing analyses of samples from TSC patients suggest that TBC1D7 is unlikely to represent TSC3. TBC1D7 knockdown decreases the association of TSC1 and TSC2 leading to decreased Rheb-GAP activity, without effects on the localization of TSC2 to the lysosome. Like the other TSC-TBC components, TBC1D7 knockdown results in increased mTORC1 signaling, delayed induction of autophagy, and enhanced cell growth under poor growth conditions.'},\n",
       "  {'qas': [{'id': '54f89e1a06d9727f76000001_001',\n",
       "     'question': 'Which kinase is inhibited by the small molecule KN-93?',\n",
       "     'answers': [{'text': 'CaM kinase II', 'answer_start': 1156}]}],\n",
       "   'context': 'Calcium/calmodulin-dependent protein kinase IIalpha mediates activation of mitogen-activated protein kinase and cytosolic phospholipase A2 in norepinephrine-induced arachidonic acid release in rabbit aortic smooth muscle cells. We have investigated the contribution of Ca2+/calmodulin-dependent protein kinase II (CaM kinase II) and mitogen-activated protein kinase (MAP kinase) in norepinephrine (NE)-induced arachidonic acid (AA) release in rabbit aortic vascular smooth muscle cells (VSMC). NE enhanced release of AA via activation of cytosolic phospholipase A2 (cPLA2) but not secretory PLA2 in VSMC prelabeled with [3H]AA. NE (10 microM) enhanced CaM kinase II and MAP kinase activity. In cells transiently transfected with antisense oligonucleotides complementary to the translation initiation sites of CaM kinase II and MAP kinase, NE-induced AA release was inhibited by 100 and 35% respectively. Treatment of cells with PD-098059, a MAP kinase kinase inhibitor, or with MAP kinase antisense oligonucleotide reduced NE-induced activation of MAP kinase and cPLA2. NE-induced MAP kinase and cPLA2 activation was also inhibited in cells treated with a CaM kinase II inhibitor, KN-93, or with CaM kinase II antisense oligonucleotide. On the other hand, inhibition of MAP kinase kinase with PD-098059 or of MAP kinase with antisense oligonucleotides did not alter the NE-induced increase in CaM kinase II activity. Phosphorylation of MAP kinase and CaM kinase II by NE, studied by 32P incorporation and immune complex kinase assays, was inhibited by KN-93. Collectively, these data suggest that CaM kinase II can activate MAP kinase, which in turn activates cPLA2 to release AA for prostacyclin synthesis in the rabbit VSMC. This novel pathway for activation of MAP kinase by CaM kinase II appears to be mediated through stimulation of MAP kinase kinase. Activation of adrenergic receptors with NE in VSMC caused translocation of CaM kinase II, MAP kinase, and cPLA2 to the nuclear envelope only in the presence of extracellular Ca2+. Okadaic acid, which increased phosphorylation and activity, did not translocate these enzymes. Therefore, it appears that in rabbit VSMC, NE, by promoting extracellular Ca2+ influx, increases CaM kinase II activity, leading to activation of MAP kinase and cPLA2 and translocation to the nuclear envelope, resulting in release of AA from the nuclear envelope for prostacyclin synthesis.'},\n",
       "  {'qas': [{'id': '54f89e1a06d9727f76000001_002',\n",
       "     'question': 'Which kinase is inhibited by the small molecule KN-93?',\n",
       "     'answers': [{'text': 'CaM kinase II', 'answer_start': 1196}]}],\n",
       "   'context': 'Calcium/calmodulin-dependent protein kinase IIalpha mediates activation of mitogen-activated protein kinase and cytosolic phospholipase A2 in norepinephrine-induced arachidonic acid release in rabbit aortic smooth muscle cells. We have investigated the contribution of Ca2+/calmodulin-dependent protein kinase II (CaM kinase II) and mitogen-activated protein kinase (MAP kinase) in norepinephrine (NE)-induced arachidonic acid (AA) release in rabbit aortic vascular smooth muscle cells (VSMC). NE enhanced release of AA via activation of cytosolic phospholipase A2 (cPLA2) but not secretory PLA2 in VSMC prelabeled with [3H]AA. NE (10 microM) enhanced CaM kinase II and MAP kinase activity. In cells transiently transfected with antisense oligonucleotides complementary to the translation initiation sites of CaM kinase II and MAP kinase, NE-induced AA release was inhibited by 100 and 35% respectively. Treatment of cells with PD-098059, a MAP kinase kinase inhibitor, or with MAP kinase antisense oligonucleotide reduced NE-induced activation of MAP kinase and cPLA2. NE-induced MAP kinase and cPLA2 activation was also inhibited in cells treated with a CaM kinase II inhibitor, KN-93, or with CaM kinase II antisense oligonucleotide. On the other hand, inhibition of MAP kinase kinase with PD-098059 or of MAP kinase with antisense oligonucleotides did not alter the NE-induced increase in CaM kinase II activity. Phosphorylation of MAP kinase and CaM kinase II by NE, studied by 32P incorporation and immune complex kinase assays, was inhibited by KN-93. Collectively, these data suggest that CaM kinase II can activate MAP kinase, which in turn activates cPLA2 to release AA for prostacyclin synthesis in the rabbit VSMC. This novel pathway for activation of MAP kinase by CaM kinase II appears to be mediated through stimulation of MAP kinase kinase. Activation of adrenergic receptors with NE in VSMC caused translocation of CaM kinase II, MAP kinase, and cPLA2 to the nuclear envelope only in the presence of extracellular Ca2+. Okadaic acid, which increased phosphorylation and activity, did not translocate these enzymes. Therefore, it appears that in rabbit VSMC, NE, by promoting extracellular Ca2+ influx, increases CaM kinase II activity, leading to activation of MAP kinase and cPLA2 and translocation to the nuclear envelope, resulting in release of AA from the nuclear envelope for prostacyclin synthesis.'},\n",
       "  {'qas': [{'id': '54f89e1a06d9727f76000001_003',\n",
       "     'question': 'Which kinase is inhibited by the small molecule KN-93?',\n",
       "     'answers': [{'text': 'CaM Kinase II', 'answer_start': 176}]}],\n",
       "   'context': 'Inhibition of acid secretion in gastric parietal cells by the Ca2+/calmodulin-dependent protein kinase II inhibitor KN-93. A novel Ca2+/calmodulin-dependent protein kinase II (CaM Kinase II) inhibitor, KN-93 potently inhibits gastric acid secretion from parietal cells. As previously reported (1), treatment of parietal cells with a selective inhibitor of CaM kinase II, KN-62 resulted in the inhibition of cholinergic-stimulated rabbit parietal cell secretion, whereas it failed to inhibit the histamine and forskolin response. In contrast effects of carbachol, histamine and forskolin were significantly inhibited by KN-93 with an IC50 of 0.15, 0.3 and 1 microM, respectively; these effects occurred without any changes in intracellular cyclic AMP and Ca2+ levels. In the present study we investigated the mechanism by which KN-93 acts upon the acid-secreting machinery of gastric parietal cells. Neither redistribution of the proton pump activity nor the morphological transformation were affected by KN-93. The drug only weakly inhibited the H+, K(+)-ATPase activity but strongly dissipated the proton gradient formed in the gastric membrane vesicles and reduced the volume of luminal space. Thus KN-93 acts at pH gradient formation whereas KN-62 acts only at CaM Kinase II.'},\n",
       "  {'qas': [{'id': '54f89e1a06d9727f76000001_004',\n",
       "     'question': 'Which kinase is inhibited by the small molecule KN-93?',\n",
       "     'answers': [{'text': 'CaM kinase II', 'answer_start': 739}]}],\n",
       "   'context': 'Inhibition of CaM kinase II activation and force maintenance by KN-93 in arterial smooth muscle. Ca(+)/calmodulin-dependent protein kinase II (CaM kinase II) has been implicated in the regulation of smooth muscle contractility. The goals of this study were to determine: 1) to what extent CaM kinase II is activated by contractile stimuli in intact arterial smooth muscle, and 2) the effect of a CaM kinase II inhibitor (KN-93) on CaM kinase II activation, phosphorylation of myosin regulatory light chains (MLC(20)), and force. Both histamine (1 microM) and KCl depolarization activated CaM kinase II with a time course preceding maximal force development, and suprabasal CaM kinase II activation was sustained during tonic contractions. CaM kinase II activation was inhibited by KN-93 pretreatment (IC(50) approximately 1 microM). KN-93 inhibited histamine-induced tonic force maintenance, whereas early force development and MLC(20) phosphorylation responses during the entire time course were unaffected. Both force development and maintenance in response to KCl were inhibited by KN-93. Rapid increases in KCl-induced MLC(20) phosphorylation were also inhibited by KN-93, whereas steady-state MLC(20) phosphorylation responses were unaffected. In contrast, phorbol 12,13-dibutyrate (PDBu) did not activate CaM kinase II and PDBu-stimulated force development was unaffected by KN-93. Thus KN-93 appears to target a step(s) essential for force maintenance in response to physiological stimuli, suggesting a role for CaM kinase II in regulating tonic contractile responses in arterial smooth muscle. Pharmacological activation of protein kinase C bypasses the KN-93 sensitive step.'},\n",
       "  {'qas': [{'id': '54f89e1a06d9727f76000001_005',\n",
       "     'question': 'Which kinase is inhibited by the small molecule KN-93?',\n",
       "     'answers': [{'text': 'CaM kinase II', 'answer_start': 739}]}],\n",
       "   'context': 'Inhibition of CaM kinase II activation and force maintenance by KN-93 in arterial smooth muscle. Ca(+)/calmodulin-dependent protein kinase II (CaM kinase II) has been implicated in the regulation of smooth muscle contractility. The goals of this study were to determine: 1) to what extent CaM kinase II is activated by contractile stimuli in intact arterial smooth muscle, and 2) the effect of a CaM kinase II inhibitor (KN-93) on CaM kinase II activation, phosphorylation of myosin regulatory light chains (MLC(20)), and force. Both histamine (1 microM) and KCl depolarization activated CaM kinase II with a time course preceding maximal force development, and suprabasal CaM kinase II activation was sustained during tonic contractions. CaM kinase II activation was inhibited by KN-93 pretreatment (IC(50) approximately 1 microM). KN-93 inhibited histamine-induced tonic force maintenance, whereas early force development and MLC(20) phosphorylation responses during the entire time course were unaffected. Both force development and maintenance in response to KCl were inhibited by KN-93. Rapid increases in KCl-induced MLC(20) phosphorylation were also inhibited by KN-93, whereas steady-state MLC(20) phosphorylation responses were unaffected. In contrast, phorbol 12,13-dibutyrate (PDBu) did not activate CaM kinase II and PDBu-stimulated force development was unaffected by KN-93. Thus KN-93 appears to target a step(s) essential for force maintenance in response to physiological stimuli, suggesting a role for CaM kinase II in regulating tonic contractile responses in arterial smooth muscle. Pharmacological activation of protein kinase C bypasses the KN-93 sensitive step.'},\n",
       "  {'qas': [{'id': '571e275dbb137a4b0c000005_001',\n",
       "     'question': \"What are 'vildagliptin', 'sitagliptin', 'saxagliptin', 'alogliptin', 'linagliptin', and 'dutogliptin'?\",\n",
       "     'answers': [{'text': 'dipeptidyl peptidase-4 (DPP-4) inhibitors',\n",
       "       'answer_start': 78}]}],\n",
       "   'context': 'Linagliptin and newer DPP-4 inhibitors: newer uses and newer indications. The dipeptidyl peptidase-4 (DPP-4) inhibitors linagliptin, sitagliptin, saxagliptin, vildagliptin and alogliptin are being developed and have been approved for the treatment of type-2 diabetes. These agents may be used either as monotherapy for the treatment of type-2 diabetes or in combination with other anti-diabetic drugs. The present review highlights the use of linagliptin and other new (DPP-4) inhibitors in the management of type-2 diabetes. The review also highlights advantages, comparative pharmacokinetic, safety profile and other potential uses including potential newer indications of DPP-4 inhibitors and relevant patents. The other potential uses that are not restricted to diabetes include obesity, cardiovascular disease, neurological disease, hepatobiliary disease, wound healing, and other inflammatory illnesses.'},\n",
       "  {'qas': [{'id': '571e275dbb137a4b0c000005_002',\n",
       "     'question': \"What are 'vildagliptin', 'sitagliptin', 'saxagliptin', 'alogliptin', 'linagliptin', and 'dutogliptin'?\",\n",
       "     'answers': [{'text': 'dipeptidyl peptidase-4 (DPP-4) inhibitors',\n",
       "       'answer_start': 158}]}],\n",
       "   'context': 'Dipeptidyl peptidase-4 inhibitors and HbA1c target of <7% in type 2 diabetes: meta-analysis of randomized controlled trials. AIM: We assessed the efficacy of dipeptidyl peptidase-4 (DPP-4) inhibitors vildagliptin, sitagliptin, saxagliptin and alogliptin to reach the haemoglobin HbA1c target of <7% in people with type 2 diabetes. METHODS: We conducted an electronic search for randomized controlled trials (RCTs) involving DPP-4 inhibitors through September 2010. RCTs were included if they lasted at least 12 weeks, included 30 patients or more and reported the proportion of patients reaching the HbA1c target of <7%. RESULTS: A total of 43 RCTs reporting 52 comparisons met the selection criteria, which included 19 101 study participants evaluated for the primary endpoint, 10 467 treated with a DPP-4 inhibitor and 8634 treated with placebo or a comparator drug. DPP-4 inhibitors showed a statistically significant reduction in HbA1c compared to placebo and approximately 40% of participants achieved the HbA1c goal of <7%: this was associated with weight neutrality and no greater hypoglycaemia. The reduction of the HbA1c level and the rate of HbA1c goal attainment was not different from comparator drugs, with similar hypoglycaemia, and different effect on weight owing to the nature of comparator (metformin, sulfonylurea or glitazones). Baseline HbA1c was the best predictor for achievement of A1C target (overall weighted r(2) value = 0.410, p < 0.001). CONCLUSIONS: A greater proportion of type 2 diabetic patients can achieve the HbA1c goal <7% with DPP-4 inhibitors compared to placebo, with no weight gain, and no hypoglycaemic risk when used alone; DPP-4 inhibitors were not different from comparator drugs.'},\n",
       "  {'qas': [{'id': '56e857ae42442bac75000004_001',\n",
       "     'question': 'What is needed for MMP proteins to be functional?',\n",
       "     'answers': [{'text': 'zinc', 'answer_start': 113}]}],\n",
       "   'context': 'MMP-2 selectivity in hydroxamate-type inhibitors. Extracellular matrix metalloproteinases (MMPs) are a family of zinc-dependent neutral endopeptidases involved in physiological and pathological processes, through the cleavage of extracellular matrix. MMPs are capable of degrading essentially all matrix components, which is crucial for malignant tumor growth, invasion, metastasis and angiogenesis. The vertebrates MMP family includes at least 26 enzymes (23 have been known in humans) with only MMP-1, 2, and 7 experimentally validated as targets for antitumoral drug design. However, inhibition of MMP-1 has been hypothesized to be the cause of the clinically observed musculoskeletal syndrome when broad spectrum inhibitors are used. On the other hand, MMP-9 is a tricky enzyme, since its inhibition might be useful in treating patients with early-stage cancers, but MMP-9 is an anti-target in patients with advanced disease. So, MMP-9 inhibition should also be prevented. Therefore, selective MMP-2 inhibition arises as a pursued profile for MMP binders. Among them, hydroxamates have been extensively studied as small molecule drug candidates characterized by an effective zinc-binding group plus additional side chains responsible for the selectivity. This article pays particular attention to MMP-2 selectivity on hydroxamate-type inhibitors, especially against MMP-9, and their chemical structure, SAR, general synthetic methods, and molecular modelling studies are here reviewed in order to inspire further design of new effective anticancer agents.'},\n",
       "  {'qas': [{'id': '56e857ae42442bac75000004_002',\n",
       "     'question': 'What is needed for MMP proteins to be functional?',\n",
       "     'answers': [{'text': 'zinc', 'answer_start': 223}]}],\n",
       "   'context': '[The role of metalloproteinases in modification of extracellular matrix in invasive tumor growth, metastasis and angiogenesis]. Extracellular matrix metalloproteinases (MMPs) are a family of endopeptydases which recquire a zinc ion at their active site, for proteolityc activity. There are six members of the MMP family: matrilysins, collagenases, stromelysins, gelatinases, membrane MMPs and other MMPs. Activity of MMPs is regulated at the level of gene transcription, mRNA stability, zymogene proteolitic activation, inhibition of an active enzyme and MMP degradation. Tissue inhibitors of metalloproteinases (TIMPs) are main intracellular inhibitors of MMPs. Host cells can be stimulated by tumor cells to produce MMPs by secreted interleukins, interferons, growth factors and an extracellular matrix metalloproteinase inducer (EMMPRIN). MMPs are produced by tumor cells, fibroblasts, macrophages, mast cells, polimorphonuclear neutrophiles (PMNs) and endothelial cells (ECs). MMPs affect many stages of tumor development, facilitating its growth through promoting tumor cells proliferation, invasion and migration, new blood vessels formation and blocking tumor cells apoptosis. MMPs can promote tumor development in several ways. ECM degradation results in release of peptide growth factors. Growth factors linked with cell surface or binding proteins can also be liberated by MMPs. MMPs can indirectly regulate integrin signalling or cleave E-cadherins, facilitating cell migration. MMPs support metastasis inducing an epithelial to mesenchymal transition (EMT). MMP also support transendothelial migration. MMPs support angiogenesis by releasing pro-angiogenic factors and degrading ECM to support ECs migration. Cell surface growth factor receptors are also cleaved by MMPs, which results in inhibition of tumor development, so is release of anti-angiogenic factors from ECM.'},\n",
       "  {'qas': [{'id': '5509c52f1180f13250000004_001',\n",
       "     'question': 'What is hyperosmia',\n",
       "     'answers': [{'text': 'increased olfactory acuity', 'answer_start': 52}]}],\n",
       "   'context': 'Cranial Nerve I: The Olfactory Nerve. Hyperosmia is increased olfactory acuity, and hypoosmia is diminished olfactory acuity. Anosmia, the inability to recognize odors, may be unilateral or bilateral. Dysosmia is an abnormal sense of smell.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_001',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 235}]}],\n",
       "   'context': \"A monoclonal antibody based capture ELISA for botulinum neurotoxin serotype B: toxin detection in food. Botulism is a serious foodborne neuroparalytic disease, caused by botulinum neurotoxin (BoNT), produced by the anaerobic bacterium Clostridium botulinum. Seven toxin serotypes (A-H) have been described. The majority of human cases of botulism are caused by serotypes A and B followed by E and F. We report here a group of serotype B specific monoclonal antibodies (mAbs) capable of binding toxin under physiological conditions. Thus, they serve as capture antibodies for a sandwich (capture) ELISA. The antibodies were generated using recombinant peptide fragments corresponding to the receptor-binding domain of the toxin heavy chain as immunogen. Their binding properties suggest that they bind a complex epitope with dissociation constants (KD's) for individual antibodies ranging from 10 to 48 × 10-11 M. Assay performance for all possible combinations of capture-detector antibody pairs was evaluated and the antibody pair resulting in the lowest level of detection (L.O.D.), ~20 pg/mL was determined. Toxin was detected in spiked dairy samples with good recoveries at concentrations as low as 0.5 pg/mL and in ground beef samples at levels as low as 2 ng/g. Thus, the sandwich ELISA described here uses mAb for both the capture and detector antibodies (binding different epitopes on the toxin molecule) and readily detects toxin in those food samples tested.\"},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_002',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 185}]}],\n",
       "   'context': 'Vaccination of cattle with a recombinant bivalent toxoid against botulism serotypes C and D. Cattle botulism is a fatal intoxication caused by botulinum neurotoxins (BoNTs) produced by Clostridium botulinum serotypes C and D resulting in economic losses. Vaccination is the most effective way to control botulism. However, the commercially available vaccines are difficult and hazardous to produce. Neutralizing antibodies against the C-terminal fragment of the BoNT heavy chain (HC) are known to protect against lethal doses of BoNTs. We report the vaccination of cattle with a previously tested recombinant chimera consisting of Escherichia coli heat-labile enterotoxin B subunit and the HC of BoNTs C and D. Vaccinated animals produced neutralizing antibodies against serotypes C and D averaging 5±0 and 6.14±1.06IU/mL, respectively. For BoNT D, the titers were greater than those measured for the commercial vaccine, which induced titers of 5±0 and 2.85±1.35 against the respective serotypes, suggesting that this chimera is effective against cattle botulism.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_003',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 169}]}],\n",
       "   'context': 'Quantitative real-time PCR for detection of neurotoxin genes of Clostridium botulinum types A, B and C in equine samples. Botulism in horses in the USA is attributed to Clostridium botulinum types A, B or C. In this study, a duplex quantitative real-time PCR (qPCR) for detection of the neurotoxin genes of C. botulinum types A and B, and a singleplex qPCR for detection of the neurotoxin gene of C. botulinum type C, were optimized and validated for equine gastrointestinal, faecal and feed samples. The performance of these assays was evaluated and compared to the standard mouse bioassay (MBA) using 148 well-characterized samples, most of which were acquired from a repository of veterinary diagnostic samples from cases of botulism: 106 samples positive for C. botulinum (25 type A, 27 type B, 28 type C, 1 type D and 25 type E) and 42 negative samples. The sensitivities of the qPCR assays were 89%, 86% and 96% for C. botulinum types A, B and C, respectively. The overall sensitivity of the mouse bioassay for types A, B and C was 81%. The specificities of the qPCR assays were 99-100% and the specificity of the mouse bioassay was 95%.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_004',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 139}]}],\n",
       "   'context': 'Analysis of the genetic distribution among members of Clostridium botulinum group I using a novel multilocus sequence typing (MLST) assay. Clostridium botulinum is the etiological agent of botulism. Due to food-borne poisoning and the potential use of the extremely toxic botulinum neurotoxin (BoNT) from C. botulinum in bioterror or biocrime related actions, reliable high resolution typing methods for discriminating C. botulinum strains are needed. Partial sequencing of the adk, atpH, gyrB, proC, rpoD and spo0A genes from 51 various C. botulinum/sporogenes isolates was performed, resulting in 37 different sequence types (STs). Analysis of the sequence data revealed a genetic distribution in five larger clusters with a loose correlation to the BoNT serotypes. The developed MLST assay had a slightly lower resolution ability when compared to the MLVA (multilocus variable number of tandem repeat analysis), but the two methods resulted in similar subclusters of the strains possessing the BoNT serotypes A, B and F. The current work presents the development of a novel MLST assay useful for genotyping C. botulinum related to basic phylogenetic research and trace-back analysis in microbial forensic studies.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_005',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 155}]}],\n",
       "   'context': 'Identification of botulinum toxin type in clinical samples and foods in Iran. BACKGROUND: Botulism is a serious neuroparalytic disease caused by toxins of Clostridium botulinum. Botulinum toxin is produced under anaerobic conditions and is one of the most dangerous toxin in the world. Rapid diagnosis of botulism is very essential for successful therapy. In this study, we reviewed data of cases of botulism in Iran from April 2004 through March 2010. MATHERIALS AND METHODS: From a total of 1140 samples of suspected botulism samples, 477 serum, 294 stool, 111 gastric secretions, and 258 food samples were collected from 21 provinces. These samples belonged to 432 distinct patients. All samples were tested for botulism by mouse bioassay, a gold standard method for detection of botulism. RESULTS: From 1140 received samples, 64 (5.6 %) positive samples of botulism were identified. Of these, 14 (21.8 %) cases had toxin type A, seven (11 %) cases had toxin type B, 22 (34.3 %) cases had toxin type E, and seven (11 %) cases had toxin type AB. The toxin type could not been identified in 14 (21.8 %) cases. The highest positive results were in Gilan, Tehran, Golestan, and Hamedan provinces. Seafoods and locally- made cheese were the most implicated foods in type E and type A botulism, respectively. CONCLUSION: Accurate and rapid diagnosis of botulism is very important because every case of botulism can be a public health emergency. During the study period, the median number of positive cases per year was 2.7 (range: one to18). Therefore, it is suggested that all clinicians are required to submit the collected samples from patients with botulism symptoms to the botulism reference laboratory for specific diagnosis and confirmation of botulism.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_006',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 280}]}],\n",
       "   'context': 'Multiplex real-time PCR for detecting and typing Clostridium botulinum group III organisms and their mosaic variants. Botulism is a neuroparalytic disease that can occur in all warm-blooded animals, birds, and fishes. The disease in animals is mainly caused by toxins produced by Clostridium botulinum strains belonging to group III, although outbreaks due to toxins produced by group I and II organisms have been recognized. Group III strains are capable of producing botulinum toxins of type C, D, and C/D and D/C mosaic variants. Definitive diagnosis of animal botulism is made by combining clinical findings with laboratory investigations. Detection of toxins in clinical specimens and feed is the gold standard for laboratory diagnosis. Since toxins may be degraded by organisms contained in the gastrointestinal tract or may be present at levels below the detection limit, the recovery of C. botulinum from sick animal specimens is consistent for laboratory confirmation. In this article we report the development and in-house validation of a new multiplex real-time PCR for detecting and typing the neurotoxin genes found in C. botulinum group III organisms. Validation procedures have been carried out according to ISO 16140, using strains and samples recovered from cases of animal botulism in Italy and France.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_007',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 292}]}],\n",
       "   'context': \"Management of animal botulism outbreaks: from clinical suspicion to practical countermeasures to prevent or minimize outbreaks. Botulism is a severe neuroparalytic disease that affects humans, all warm-blooded animals, and some fishes. The disease is caused by exposure to toxins produced by Clostridium botulinum and other botulinum toxin-producing clostridia. Botulism in animals represents a severe environmental and economic concern because of its high mortality rate. Moreover, meat or other products from affected animals entering the food chain may result in a public health problem. To this end, early diagnosis is crucial to define and apply appropriate veterinary public health measures. Clinical diagnosis is based on clinical findings eliminating other causes of neuromuscular disorders and on the absence of internal lesions observed during postmortem examination. Since clinical signs alone are often insufficient to make a definitive diagnosis, laboratory confirmation is required. Botulinum antitoxin administration and supportive therapies are used to treat sick animals. Once the diagnosis has been made, euthanasia is frequently advisable. Vaccine administration is subject to health authorities' permission, and it is restricted to a small number of animal species. Several measures can be adopted to prevent or minimize outbreaks. In this article we outline all phases of management of animal botulism outbreaks occurring in wet wild birds, poultry, cattle, horses, and fur farm animals.\"},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_008',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 244}]}],\n",
       "   'context': 'Crystal structure of the receptor binding domain of the botulinum C-D mosaic neurotoxin reveals potential roles of lysines 1118 and 1136 in membrane interactions. The botulinum neurotoxins (BoNTs) produced by different strains of the bacterium Clostridium botulinum are responsible for the disease botulism and include a group of immunologically distinct serotypes (A, B, E, and F) that are considered to be the most lethal natural proteins known for humans. Two BoNT serotypes, C and D, while rarely associated with human infection, are responsible for deadly botulism outbreaks afflicting animals. Also associated with animal infections is the BoNT C-D mosaic protein (BoNT/CD), a BoNT subtype that is essentially a hybrid of the BoNT/C (∼two-third) and BoNT/D (∼one-third) serotypes. While the amino acid sequence of the heavy chain receptor binding (HCR) domain of BoNT/CD (BoNT/CD-HCR) is very similar to the corresponding amino acid sequence of BoNT/D, BoNT/CD-HCR binds synaptosome membranes better than BoNT/D-HCR. To obtain structural insights for the different membrane binding properties, the crystal structure of BoNT/CD-HCR (S867-E1280) was determined at 1.56 Å resolution and compared to previously reported structures for BoNT/D-HCR. Overall, the BoNT/CD-HCR structure is similar to the two sub-domain organization observed for other BoNT HCRs: an N-terminal jellyroll barrel motif and a C-terminal β-trefoil fold. Comparison of the structure of BoNT/CD-HCR with BoNT/D-HCR indicates that K1118 has a similar structural role as the equivalent residue, E1114, in BoNT/D-HCR, while K1136 has a structurally different role than the equivalent residue, G1132, in BoNT/D-HCR. Lysine-1118 forms a salt bridge with E1247 and may enhance membrane interactions by stabilizing the putative membrane binding loop (K1240-N1248). Lysine-1136 is observed on the surface of the protein. A sulfate ion bound to K1136 may mimic a natural interaction with the negatively changed phospholipid membrane surface. Liposome-binding experiments demonstrate that BoNT/CD-HCR binds phosphatidylethanolamine liposomes more tightly than BoNT/D-HCR.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_009',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 112}]}],\n",
       "   'context': '[Botulinum toxin: from poison to drug. A historical review]. Botulinumtoxin (BTX) is a neurotoxin produced from Clostridium botulinum under anaerobic conditions and is responsible for botulism, a notifiable, bacterial form of food poisoning. The first case of botulism is believed to have occurred in 1735. An epidemic in Southern Germany in 1793 claimed the death of over the half of those patients who had become ill through eating uncooked blood sausages. The term \"pharmakon\" is Greek and implicates that a drug originates from poison (potion, remedy). Theophrastus Bombast von Hohenheim known as Paracelsus (1493/94-1541) first described this duality with his dictum \"alle ding sind gift und nichts on gift; alein die dosis macht das ein ding kein gift ist\" (only the dose makes a remedy poisonous). In Baden-Württemberg in 1817, the poet and physician Dr. Justinus Christian Kerner described the symptoms of botulism, so that at this time botulism was also called Kerner disease. Until the turn of the century the reason for poisoning was not known. Van Ermengem succeeded in isolating the anaerobic bacterium causing botulism, but the specific mechanism of BTX was only established after the second World War. In the late seventies the ophthalmologist Dr. Alan Scott used BTX the first time in the treatment of strabismus. The drug was then used in the treatment of several muscle spasticities such as, for example, torticollis or hemifacial spasm. Only recently BTX has been successfully used for focal hyperhidrosis. We review the history of botulinum toxin from its discovery in the nineteenth century and the research into its effect in the middle of the 20th century up to its clinical use at the present time.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_010',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 297}]}],\n",
       "   'context': 'Formulating a new basis for the treatment against botulinum neurotoxin intoxication: 3,4-Diaminopyridine prodrug design and characterization. Botulism is a disease characterized by neuromuscular paralysis and is produced from botulinum neurotoxins (BoNTs) found within the Gram positive bacterium Clostridium botulinum. This bacteria produces the most deadliest toxin known, with lethal doses as low as 1 ng/kg. Due to the relative ease of production and transport, the use of these agents as potential bioterrorist weapons has become of utmost concern. No small molecule therapies against BoNT intoxication have been approved to date. However, 3,4-diaminopyridine (3,4-DAP), a potent reversible inhibitor of voltage-gated potassium channels, is an effective cholinergic agonist used in the treatment of neuromuscular degenerative disorders that require cholinergic enhancement. 3,4-DAP has also been shown to facilitate recovery of neuromuscular action potential post botulinum intoxication by blocking K(+) channels. Unfortunately, 3,4-DAP displays toxicity largely due to blood-brain-barrier (BBB) penetration. As a dual-action prodrug approach to cholinergic enhancement we have designed carbamate and amide conjugates of 3,4-DAP. The carbamate prodrug is intended to be a slowly reversible inhibitor of acetylcholinesterase (AChE) along the lines of the stigmines thereby allowing increased persistence of released acetylcholine within the synaptic cleft. As a secondary activity, cleavage of the carbamate prodrug by AChE will afford the localized release of 3,4-DAP, which in turn, will enhance the pre-synaptic release of additional acetylcholine. Being a competitive inhibitor with respect to acetylcholine, the activity of the prodrug will be greatest at the synaptic junctions most depleted of acetylcholine. Here we report upon the synthesis and biochemical characterization of three new classes of prodrugs intended to limit previously reported stability and toxicity issues. Of the prodrugs examined, compound 32, demonstrated the most clinically relevant half-life of 2.76 h, while selectively inhibiting AChE over butyrylcholinesterase--a plasma-based high activity esterase. Future in vivo studies could provide validation of prodrug 32 as a potential treatment against BoNT intoxication as well as other neuromuscular disorders.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_011',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 286}]}],\n",
       "   'context': 'Multilocus outbreak of foodborne botulism linked to contaminated sausage in Hebei province, China. In 2007, an outbreak of foodborne botulism occurred in Hebei province, China. An epidemiological investigation and laboratory detection studies showed that sausage contaminated by type A Clostridium botulinum caused this outbreak of food poisoning. Its clinical and epidemiological features were different from previous reports of food poisoning.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_012',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 111}]}],\n",
       "   'context': '[Botulinum toxin as a biological weapon]. Botulism is caused by botulinum neurotoxin produced by the bacterium Clostridium botulinum. It is a flaccid paralysis in which consciousness and nociception are preserved. Natural botulism typically results from ingestion of inadequately heated or unheated vacuum-packed foods. In addition, botulinum toxin is one of the most feared biological weapons. In the diagnosis and treatment of botulism early suspicion is essential. Several coinciding or local clusters without a typical connecting source, or an uncommon type of toxin may indicate an intentionally caused epidemic.'},\n",
       "  {'qas': [{'id': '55475dc2f35db75526000001_013',\n",
       "     'question': 'Which is the most known bacterium responsible for botulism (sausage-poisoning)?',\n",
       "     'answers': [{'text': 'Clostridium botulinum', 'answer_start': 112}]}],\n",
       "   'context': '[Botulinum toxin: from poison to drug. A historical review]. Botulinumtoxin (BTX) is a neurotoxin produced from Clostridium botulinum under anaerobic conditions and is responsible for botulism, a notifiable, bacterial form of food poisoning. The first case of botulism is believed to have occurred in 1735. An epidemic in Southern Germany in 1793 claimed the death of over the half of those patients who had become ill through eating uncooked blood sausages. The term \"pharmakon\" is Greek and implicates that a drug originates from poison (potion, remedy). Theophrastus Bombast von Hohenheim known as Paracelsus (1493/94-1541) first described this duality with his dictum \"alle ding sind gift und nichts on gift; alein die dosis macht das ein ding kein gift ist\" (only the dose makes a remedy poisonous). In Baden-Württemberg in 1817, the poet and physician Dr. Justinus Christian Kerner described the symptoms of botulism, so that at this time botulism was also called Kerner disease. Until the turn of the century the reason for poisoning was not known. Van Ermengem succeeded in isolating the anaerobic bacterium causing botulism, but the specific mechanism of BTX was only established after the second World War. In the late seventies the ophthalmologist Dr. Alan Scott used BTX the first time in the treatment of strabismus. The drug was then used in the treatment of several muscle spasticities such as, for example, torticollis or hemifacial spasm. Only recently BTX has been successfully used for focal hyperhidrosis. We review the history of botulinum toxin from its discovery in the nineteenth century and the research into its effect in the middle of the 20th century up to its clinical use at the present time.'},\n",
       "  {'qas': [{'id': '56c073fcef6e394741000020_001',\n",
       "     'question': 'What is the association of spermidine with α-synuclein neurotoxicity?',\n",
       "     'answers': [{'text': 'Spermidine protects against α-synuclein neurotoxicity',\n",
       "       'answer_start': 0}]}],\n",
       "   'context': 'Spermidine protects against α-synuclein neurotoxicity. As our society ages, neurodegenerative disorders like Parkinson`s disease (PD) are increasing in pandemic proportions. While mechanistic understanding of PD is advancing, a treatment with well tolerable drugs is still elusive. Here, we show that administration of the naturally occurring polyamine spermidine, which declines continuously during aging in various species, alleviates a series of PD-related degenerative processes in the fruit fly Drosophila melanogaster and the nematode Caenorhabditis elegans, two established model systems for PD pathology. In the fruit fly, simple feeding with spermidine inhibited loss of climbing activity and early organismal death upon heterologous expression of human α-synuclein, which is thought to be the principal toxic trigger of PD. In this line, administration of spermidine rescued α-synuclein-induced loss of dopaminergic neurons, a hallmark of PD, in nematodes. Alleviation of PD-related neurodegeneration by spermidine was accompanied by induction of autophagy, suggesting that this cytoprotective process may be responsible for the beneficial effects of spermidine administration.'},\n",
       "  {'qas': [{'id': '56c073fcef6e394741000020_002',\n",
       "     'question': 'What is the association of spermidine with α-synuclein neurotoxicity?',\n",
       "     'answers': [{'text': 'Spermidine protects against α-synuclein neurotoxicity',\n",
       "       'answer_start': 0}]}],\n",
       "   'context': 'Spermidine protects against α-synuclein neurotoxicity. As our society ages, neurodegenerative disorders like Parkinson`s disease (PD) are increasing in pandemic proportions. While mechanistic understanding of PD is advancing, a treatment with well tolerable drugs is still elusive. Here, we show that administration of the naturally occurring polyamine spermidine, which declines continuously during aging in various species, alleviates a series of PD-related degenerative processes in the fruit fly Drosophila melanogaster and the nematode Caenorhabditis elegans, two established model systems for PD pathology. In the fruit fly, simple feeding with spermidine inhibited loss of climbing activity and early organismal death upon heterologous expression of human α-synuclein, which is thought to be the principal toxic trigger of PD. In this line, administration of spermidine rescued α-synuclein-induced loss of dopaminergic neurons, a hallmark of PD, in nematodes. Alleviation of PD-related neurodegeneration by spermidine was accompanied by induction of autophagy, suggesting that this cytoprotective process may be responsible for the beneficial effects of spermidine administration.'},\n",
       "  {'qas': [{'id': '52fe52702059c6d71c000078_001',\n",
       "     'question': 'Where is the histone variant CENPA preferentially localized?',\n",
       "     'answers': [{'text': 'centromeres', 'answer_start': 476}]}],\n",
       "   'context': 'Partially functional Cenpa-GFP fusion protein causes increased chromosome missegregation and apoptosis during mouse embryogenesis. CENP-A is an essential histone H3-like protein that localizes to the centromeric region of eukaryotic chromosomes. Heterozygous and homozygous Cenpa-GFP fusion-protein mouse mutants, generated through targeted insertion of the green fluorescent protein (GFP) gene into the mouse Cenpa gene locus, show specific localized fluorescence at all the centromeres. Heterozygous mice are healthy and fertile. Cenpa-GFP homozygotes (Cenpag/g) undergo many cell divisions, giving rise to up to one million cells that show relatively accurate differentiation into distinct mouse embryonic tissues until day 10.5 when significant levels of chromosome missegregation, aneuploidy and apoptosis result in death. Cenpag/g embryos assemble functional kinetochores that bind to a host of centromere-specific structural and mitotic spindle checkpoint proteins (Cenpc, BubR1, Mad2 and Zw10). Examination of the nucleosomal phasing of centromeric minor and pericentromeric major satellite sequences indicates that the formation of Cenpag/g homotypic nucleosomes is not accompanied by any overt alteration to the overall size of the monomeric nucleosomal structure or the spacing of these structures. This study provides the first example of an essential centromeric protein gene variant in which subtle perturbation at the centromeric nucleosomal/chromatin level manifests in a significantly delayed lethality when compared with Cenpa null mice.'},\n",
       "  {'qas': [{'id': '52fe52702059c6d71c000078_002',\n",
       "     'question': 'Where is the histone variant CENPA preferentially localized?',\n",
       "     'answers': [{'text': 'centromeres', 'answer_start': 382}]}],\n",
       "   'context': \"Gene structure and sequence analysis of mouse centromere proteins A and C. We have determined the genomic structure and organization of the mouse Cenpa and Cenpc genes. CENPA is a member of the histone H3-like proteins and is thought to replace histone H3 in centromeric nucleosomes. CENPC is a DNA-binding protein that is located at the inner kinetochore plate of active mammalian centromeres. The Cenpa cDNA encodes a 134-amino-acid product that is 70% identical and 84% similar to its human homolog. The mouse Cenpa gene is approximately 8 kb in length and contains five exons. Sequence analysis of the 5' DNA sequence of the gene revealed two consensus CAAT boxes, a putative TFIID-binding site, an Sp1-binding domain, and two cell cycle regulatory motifs, but no consensus TATA element. The mouse Cenpc gene spans 60 kb and contains 19 exons that range in size from 44 to 602 bp. Sequence analysis of the C+G-rich promoter region showed the presence of known promoter elements, including a CpG island, a CAAT box, and several GC boxes, but the absence of a consensus TATA element.\"},\n",
       "  {'qas': [{'id': '5717d64f29809bbe7a000001_001',\n",
       "     'question': 'Which is the cellular localization of the protein Opa1?',\n",
       "     'answers': [{'text': 'mitochondrial intermembrane space',\n",
       "       'answer_start': 136}]}],\n",
       "   'context': 'Regulation of the mitochondrial dynamin-like protein Opa1 by proteolytic cleavage. The dynamin-related protein Opa1 is localized to the mitochondrial intermembrane space, where it facilitates fusion between mitochondria. Apoptosis causes Opa1 release into the cytosol and causes mitochondria to fragment. Loss of mitochondrial membrane potential also causes mitochondrial fragmentation but not Opa1 release into the cytosol. Both conditions induce the proteolytic cleavage of Opa1, suggesting that mitochondrial fragmentation is triggered by Opa1 inactivation. The opposite effect was observed with knockdown of the mitochondrial intermembrane space protease Yme1. Knockdown of Yme1 prevents the constitutive cleavage of a subset of Opa1 splice variants but does not affect carbonyl cyanide m-chlorophenyl hydrazone or apoptosis-induced cleavage. Knockdown of Yme1 also increases mitochondrial connectivity, but this effect is independent of Opa1 because it also occurs in Opa1 knockdown cells. We conclude that Yme1 constitutively regulates a subset of Opa1 isoforms and an unknown mitochondrial morphology protein, whereas the loss of membrane potential induces the further proteolysis of Opa1.'},\n",
       "  {'qas': [{'id': '5717d64f29809bbe7a000001_002',\n",
       "     'question': 'Which is the cellular localization of the protein Opa1?',\n",
       "     'answers': [{'text': 'mitochondrial intermembrane space',\n",
       "       'answer_start': 136}]}],\n",
       "   'context': 'Regulation of the mitochondrial dynamin-like protein Opa1 by proteolytic cleavage. The dynamin-related protein Opa1 is localized to the mitochondrial intermembrane space, where it facilitates fusion between mitochondria. Apoptosis causes Opa1 release into the cytosol and causes mitochondria to fragment. Loss of mitochondrial membrane potential also causes mitochondrial fragmentation but not Opa1 release into the cytosol. Both conditions induce the proteolytic cleavage of Opa1, suggesting that mitochondrial fragmentation is triggered by Opa1 inactivation. The opposite effect was observed with knockdown of the mitochondrial intermembrane space protease Yme1. Knockdown of Yme1 prevents the constitutive cleavage of a subset of Opa1 splice variants but does not affect carbonyl cyanide m-chlorophenyl hydrazone or apoptosis-induced cleavage. Knockdown of Yme1 also increases mitochondrial connectivity, but this effect is independent of Opa1 because it also occurs in Opa1 knockdown cells. We conclude that Yme1 constitutively regulates a subset of Opa1 isoforms and an unknown mitochondrial morphology protein, whereas the loss of membrane potential induces the further proteolysis of Opa1.'},\n",
       "  {'qas': [{'id': '5717d64f29809bbe7a000001_003',\n",
       "     'question': 'Which is the cellular localization of the protein Opa1?',\n",
       "     'answers': [{'text': 'mitochondrial intermembrane space',\n",
       "       'answer_start': 136}]}],\n",
       "   'context': 'Regulation of the mitochondrial dynamin-like protein Opa1 by proteolytic cleavage. The dynamin-related protein Opa1 is localized to the mitochondrial intermembrane space, where it facilitates fusion between mitochondria. Apoptosis causes Opa1 release into the cytosol and causes mitochondria to fragment. Loss of mitochondrial membrane potential also causes mitochondrial fragmentation but not Opa1 release into the cytosol. Both conditions induce the proteolytic cleavage of Opa1, suggesting that mitochondrial fragmentation is triggered by Opa1 inactivation. The opposite effect was observed with knockdown of the mitochondrial intermembrane space protease Yme1. Knockdown of Yme1 prevents the constitutive cleavage of a subset of Opa1 splice variants but does not affect carbonyl cyanide m-chlorophenyl hydrazone or apoptosis-induced cleavage. Knockdown of Yme1 also increases mitochondrial connectivity, but this effect is independent of Opa1 because it also occurs in Opa1 knockdown cells. We conclude that Yme1 constitutively regulates a subset of Opa1 isoforms and an unknown mitochondrial morphology protein, whereas the loss of membrane potential induces the further proteolysis of Opa1.'},\n",
       "  {'qas': [{'id': '5717d64f29809bbe7a000001_004',\n",
       "     'question': 'Which is the cellular localization of the protein Opa1?',\n",
       "     'answers': [{'text': 'mitochondrial intermembrane space',\n",
       "       'answer_start': 616}]}],\n",
       "   'context': 'Loss of the intermembrane space protein Mgm1/OPA1 induces swelling and localized constrictions along the lengths of mitochondria. Mgm1 is a member of the dynamin family of GTP-binding proteins. Mgm1 was first identified in yeast, where it affects mitochondrial morphology. The human homologue of Mgm1 is called OPA1. Mutations in the OPA1 gene are the prevailing cause of dominant optic atrophy, a hereditary disease in which progressive degeneration of the optic nerve can lead to blindness. Here we investigate the properties of the Mgm1/OPA1 protein in mammalian cells. We find that Mgm1/OPA1 is localized to the mitochondrial intermembrane space, where it is tightly bound to the outer surface of the inner membrane. Overexpression of wild type or mutant forms of the Mgm1/OPA1 protein cause mitochondria to fragment and, in some cases, cluster near the nucleus, whereas the loss of protein caused by small interfering RNA (siRNA) leads to dispersal of mitochondrial fragments throughout the cytosol. The cristae of these fragmented mitochondria are disorganized. At early time points after transfection with Mgm1/OPA1 siRNA, the mitochondria are not yet fragmented. Instead, the mitochondria swell and stretch, after which they form localized constrictions similar to the mitochondrial abnormalities observed during the early stages of apoptosis. These abnormalities might be the earliest effects of losing Mgm1/OPA1 protein.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_001',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 0}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_002',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 639}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_003',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 695}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_004',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 900}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_005',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 919}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_006',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 919}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_007',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 639}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_008',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 695}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_009',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 919}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_010',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 919}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_011',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 639}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_012',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 695}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_013',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 900}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_014',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 919}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_015',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 639}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_016',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 695}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_017',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 900}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_018',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 919}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_019',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 639}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '56c8f4615795f9a73e00001a_020',\n",
       "     'question': 'Which genome browser database for DNA shape annotations is available?',\n",
       "     'answers': [{'text': 'GBshape', 'answer_start': 695}]}],\n",
       "   'context': 'GBshape: a genome browser database for DNA shape annotations. Many regulatory mechanisms require a high degree of specificity in protein-DNA binding. Nucleotide sequence does not provide an answer to the question of why a protein binds only to a small subset of the many putative binding sites in the genome that share the same core motif. Whereas higher-order effects, such as chromatin accessibility, cooperativity and cofactors, have been described, DNA shape recently gained attention as another feature that fine-tunes the DNA binding specificities of some transcription factor families. Our Genome Browser for DNA shape annotations (GBshape; freely available at http://rohslab.cmb.usc.edu/GBshape/) provides minor groove width, propeller twist, roll, helix twist and hydroxyl radical cleavage predictions for the entire genomes of 94 organisms. Additional genomes can easily be added using the GBshape framework. GBshape can be used to visualize DNA shape annotations qualitatively in a genome browser track format, and to download quantitative values of DNA shape features as a function of genomic position at nucleotide resolution. As biological applications, we illustrate the periodicity of DNA shape features that are present in nucleosome-occupied sequences from human, fly and worm, and we demonstrate structural similarities between transcription start sites in the genomes of four Drosophila species.'},\n",
       "  {'qas': [{'id': '552421082c8b63434a000005_001',\n",
       "     'question': 'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
       "     'answers': [{'text': 'E3 ubiquitin ligase activity',\n",
       "       'answer_start': 235}]}],\n",
       "   'context': 'BRCA1 tumor suppression depends on BRCT phosphoprotein binding, but not its E3 ligase activity. Germline mutations of the breast cancer 1 (BRCA1) gene are a major cause of familial breast and ovarian cancer. The BRCA1 protein displays E3 ubiquitin ligase activity, and this enzymatic function is thought to be required for tumor suppression. To test this hypothesis, we generated mice that express an enzymatically defective Brca1. We found that this mutant Brca1 prevents tumor formation to the same degree as does wild-type Brca1 in three different genetically engineered mouse (GEM) models of cancer. In contrast, a mutation that ablates phosphoprotein recognition by the BRCA C terminus (BRCT) domains of BRCA1 elicits tumors in each of the three GEM models. Thus, BRCT phosphoprotein recognition, but not the E3 ligase activity, is required for BRCA1 tumor suppression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_001',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 270}]}],\n",
       "   'context': 'Transcriptome analysis identifies Bacillus anthracis genes that respond to CO2 through an AtxA-dependent mechanism. BACKGROUND: Upon infection of a mammalian host, Bacillus anthracis responds to host cues, and particularly to elevated temperature (37°C) and bicarbonate/CO2 concentrations, with increased expression of virulence factors that include the anthrax toxins and extracellular capsular layer. This response requires the presence of the pXO1 virulence plasmid-encoded pleiotropic regulator AtxA. To better understand the genetic basis of this response, we utilized a controlled in vitro system and Next Generation sequencing to determine and compare RNA expression profiles of the parental strain and an isogenic AtxA-deficient strain in a 2 × 2 factorial design with growth environments containing or lacking carbon dioxide. RESULTS: We found 15 pXO1-encoded genes and 3 chromosomal genes that were strongly regulated by the separate or synergistic actions of AtxA and carbon dioxide. The majority of the regulated genes responded to both AtxA and carbon dioxide rather than to just one of these factors. Interestingly, we identified two previously unrecognized small RNAs that are highly expressed under physiological carbon dioxide concentrations in an AtxA-dependent manner. Expression levels of the two small RNAs were found to be higher than that of any other gene differentially expressed in response to these conditions. Secondary structure and small RNA-mRNA binding predictions for the two small RNAs suggest that they may perform important functions in regulating B. anthracis virulence. CONCLUSIONS: A majority of genes on the virulence plasmid pXO1 that are regulated by the presence of either CO2 or AtxA separately are also regulated synergistically in the presence of both. These results also elucidate novel pXO1-encoded small RNAs that are associated with virulence conditions.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_002',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 258}]}],\n",
       "   'context': 'Transcriptome analysis identifies Bacillus anthracis genes that respond to CO2 through an AtxA-dependent mechanism. BACKGROUND: Upon infection of a mammalian host, Bacillus anthracis responds to host cues, and particularly to elevated temperature (37°C) and bicarbonate/CO2 concentrations, with increased expression of virulence factors that include the anthrax toxins and extracellular capsular layer. This response requires the presence of the pXO1 virulence plasmid-encoded pleiotropic regulator AtxA. To better understand the genetic basis of this response, we utilized a controlled in vitro system and Next Generation sequencing to determine and compare RNA expression profiles of the parental strain and an isogenic AtxA-deficient strain in a 2 × 2 factorial design with growth environments containing or lacking carbon dioxide. RESULTS: We found 15 pXO1-encoded genes and 3 chromosomal genes that were strongly regulated by the separate or synergistic actions of AtxA and carbon dioxide. The majority of the regulated genes responded to both AtxA and carbon dioxide rather than to just one of these factors. Interestingly, we identified two previously unrecognized small RNAs that are highly expressed under physiological carbon dioxide concentrations in an AtxA-dependent manner. Expression levels of the two small RNAs were found to be higher than that of any other gene differentially expressed in response to these conditions. Secondary structure and small RNA-mRNA binding predictions for the two small RNAs suggest that they may perform important functions in regulating B. anthracis virulence. CONCLUSIONS: A majority of genes on the virulence plasmid pXO1 that are regulated by the presence of either CO2 or AtxA separately are also regulated synergistically in the presence of both. These results also elucidate novel pXO1-encoded small RNAs that are associated with virulence conditions.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_003',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1389}]}],\n",
       "   'context': 'Bacillus anthracis virulence regulator AtxA: oligomeric state, function and CO(2) -signalling. AtxA, a unique regulatory protein of unknown molecular function, positively controls expression of the major virulence genes of Bacillus anthracis. The 475 amino acid sequence of AtxA reveals DNA binding motifs and regions similar to proteins associated with the phosphoenolpyruvate: carbohydrate phosphotransferase system (PTS). We used strains producing native and functional epitope-tagged AtxA proteins to examine protein-protein interactions in cell lysates and in solutions of purified protein. Co-affinity purification, non-denaturing polyacrylamide gel electrophoresis and bis(maleimido)hexane (BMH) cross-linking experiments revealed AtxA homo-multimers. Dimers were the most abundant species. BMH cross-links available cysteines within 13 Å. To localize interaction sites, six AtxA mutants containing distinct Cys→Ser substitutions were tested for multimerization and cross-linking. All mutants multimerized, but one mutation, C402S, prevented cross-linking. Thus, BMH uses C402 to make the inter-molecular bond between AtxA proteins, but C402 is not required for protein-protein interaction. C402 is in a region bearing amino acid similarity to Enzyme IIB proteins of the PTS. The AtxA EIIB motif may function in protein oligomerization. Finally, cultures grown with elevated CO(2) /bicarbonate exhibited increased AtxA dimer/monomer ratios and increased AtxA activity, relative to cultures grown without added CO(2) /bicarbonate, suggesting that this host-associated signal enhances AtxA function by shifting the dimer/monomer equilibrium towards the dimeric state.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_004',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1524}]}],\n",
       "   'context': 'Bacillus anthracis virulence regulator AtxA: oligomeric state, function and CO(2) -signalling. AtxA, a unique regulatory protein of unknown molecular function, positively controls expression of the major virulence genes of Bacillus anthracis. The 475 amino acid sequence of AtxA reveals DNA binding motifs and regions similar to proteins associated with the phosphoenolpyruvate: carbohydrate phosphotransferase system (PTS). We used strains producing native and functional epitope-tagged AtxA proteins to examine protein-protein interactions in cell lysates and in solutions of purified protein. Co-affinity purification, non-denaturing polyacrylamide gel electrophoresis and bis(maleimido)hexane (BMH) cross-linking experiments revealed AtxA homo-multimers. Dimers were the most abundant species. BMH cross-links available cysteines within 13 Å. To localize interaction sites, six AtxA mutants containing distinct Cys→Ser substitutions were tested for multimerization and cross-linking. All mutants multimerized, but one mutation, C402S, prevented cross-linking. Thus, BMH uses C402 to make the inter-molecular bond between AtxA proteins, but C402 is not required for protein-protein interaction. C402 is in a region bearing amino acid similarity to Enzyme IIB proteins of the PTS. The AtxA EIIB motif may function in protein oligomerization. Finally, cultures grown with elevated CO(2) /bicarbonate exhibited increased AtxA dimer/monomer ratios and increased AtxA activity, relative to cultures grown without added CO(2) /bicarbonate, suggesting that this host-associated signal enhances AtxA function by shifting the dimer/monomer equilibrium towards the dimeric state.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_005',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 195}]}],\n",
       "   'context': 'In vivo Bacillus anthracis gene expression requires PagR as an intermediate effector of the AtxA signalling cascade. Transcription of the major Bacillus anthracis virulence genes is triggered by CO2, a signal mimicking the host environment. A 182-kb plasmid, pXO1, carries the anthrax toxin genes and the genes responsible for their regulation of transcription, namely atxA and, pagR, the second gene of the pag operon. AtxA has major effects on the physiology of B. anthracis. It coordinates the transcription activation of the toxin genes with that of the capsule biosynthetic enzyme operon, located on the second virulence plasmid, pXO2. In rich medium, B. anthracis synthesises alternatively two S-layer proteins (Sap and EA1). An exponential phase \"Sap-layer\" is subsequently replaced by a stationary phase \"EA1-layer\". S-layer gene transcription is controlled by alternative sigma factors and by Sap acting as a transcriptional repressor of eag. Furthermore, in vitro in presence of CO2 and in vivo, AtxA is part of the sap and eag regulatory network. Only eag is significantly expressed in these conditions and this is due to AtxA activating eag and repressing sap transcription. PagR, and not AtxA itself, is the direct effector of this regulation by binding to sap and eag promoter regions. Therefore, PagR mediates the effect of AtxA on eag and sap and is the most downstream element of a signalling cascade initiated by AtxA. Taken together, these results indicate that the B. anthracis transcriptional regulator AtxA is controlling the synthesis of the three toxin components and of the surface elements (capsule and S-layer). Thus, AtxA is a master regulator that coordinates the response to host signals by orchestrating positive and negative controls over genes located on all genetic elements.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_006',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 989}]}],\n",
       "   'context': 'In vivo Bacillus anthracis gene expression requires PagR as an intermediate effector of the AtxA signalling cascade. Transcription of the major Bacillus anthracis virulence genes is triggered by CO2, a signal mimicking the host environment. A 182-kb plasmid, pXO1, carries the anthrax toxin genes and the genes responsible for their regulation of transcription, namely atxA and, pagR, the second gene of the pag operon. AtxA has major effects on the physiology of B. anthracis. It coordinates the transcription activation of the toxin genes with that of the capsule biosynthetic enzyme operon, located on the second virulence plasmid, pXO2. In rich medium, B. anthracis synthesises alternatively two S-layer proteins (Sap and EA1). An exponential phase \"Sap-layer\" is subsequently replaced by a stationary phase \"EA1-layer\". S-layer gene transcription is controlled by alternative sigma factors and by Sap acting as a transcriptional repressor of eag. Furthermore, in vitro in presence of CO2 and in vivo, AtxA is part of the sap and eag regulatory network. Only eag is significantly expressed in these conditions and this is due to AtxA activating eag and repressing sap transcription. PagR, and not AtxA itself, is the direct effector of this regulation by binding to sap and eag promoter regions. Therefore, PagR mediates the effect of AtxA on eag and sap and is the most downstream element of a signalling cascade initiated by AtxA. Taken together, these results indicate that the B. anthracis transcriptional regulator AtxA is controlling the synthesis of the three toxin components and of the surface elements (capsule and S-layer). Thus, AtxA is a master regulator that coordinates the response to host signals by orchestrating positive and negative controls over genes located on all genetic elements.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_007',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 101}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_008',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 105}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_009',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 441}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_010',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 445}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_011',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 1043}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_012',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 1139}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_013',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1047}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_014',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1143}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_015',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 333}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_016',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 462}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_017',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 916}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_018',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 1125}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_019',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 1257}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_020',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 1612}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_021',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1708}]}],\n",
       "   'context': 'Cross-talk to the genes for Bacillus anthracis capsule synthesis by atxA, the gene encoding the trans-activator of anthrax toxin synthesis. The two major virulence factors of Bacillus anthracis are the tripartite toxin and the polyglutamate capsule, which are encoded by genes on the large plasmids, pXO1 and pXO2, respectively. The genes atxA, located on pXO1, and acpA, located on pXO2, encode positive trans-acting proteins that are involved in bicarbonate-mediated regulation of toxin and capsule production, respectively. A derivative strain cured of pXO1 produced less capsular substance than the parent strain harbouring both pXO1 and pXO2, and electroporation of the strain cured of pXO1 with a plasmid containing the cloned atxA gene resulted in an increased level of capsule production. An acpA-null mutant was complemented by not only acpA but also the atxA gene. The cap region, which is essential for encapsulation, contains three genes capB, capC, and capA, arranged in that order. The atxA gene stimulated capsule synthesis from the cloned cap region. Transcriptional analysis of cap by RNA slot-blot hybridization and primer-extension analysis revealed that atxA activated expression of cap in trans at the transcriptional level. These results indicate that cross-talk occurs, in which the pXO1-located gene, atxA, activates transcription of the cap region genes located on pXO2. We identified two major apparent transcriptional start sites, designated P1 and P2, located at positions 731 bp and 625 bp, respectively, upstream of the translation-initiation codon of capB. Transcription initiated from P1 and P2 was activated by both atxA and acpA, and activation appeared to be stimulated by bicarbonate. Deletion analysis of the upstream region of the cap promoter revealed that activation by both atxA and acpA required a DNA segment of 70 bp extending upstream of the P1 site. These results suggest that cross-talk by atxA to the genes encoding capsule synthesis is caused by the interaction of the atxA gene product with a regulatory sequence upstream of cap.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_022',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 57}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_023',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 75}]}],\n",
       "   'context': 'Transcriptome analysis identifies Bacillus anthracis genes that respond to CO2 through an AtxA-dependent mechanism. BACKGROUND: Upon infection of a mammalian host, Bacillus anthracis responds to host cues, and particularly to elevated temperature (37°C) and bicarbonate/CO2 concentrations, with increased expression of virulence factors that include the anthrax toxins and extracellular capsular layer. This response requires the presence of the pXO1 virulence plasmid-encoded pleiotropic regulator AtxA. To better understand the genetic basis of this response, we utilized a controlled in vitro system and Next Generation sequencing to determine and compare RNA expression profiles of the parental strain and an isogenic AtxA-deficient strain in a 2 × 2 factorial design with growth environments containing or lacking carbon dioxide. RESULTS: We found 15 pXO1-encoded genes and 3 chromosomal genes that were strongly regulated by the separate or synergistic actions of AtxA and carbon dioxide. The majority of the regulated genes responded to both AtxA and carbon dioxide rather than to just one of these factors. Interestingly, we identified two previously unrecognized small RNAs that are highly expressed under physiological carbon dioxide concentrations in an AtxA-dependent manner. Expression levels of the two small RNAs were found to be higher than that of any other gene differentially expressed in response to these conditions. Secondary structure and small RNA-mRNA binding predictions for the two small RNAs suggest that they may perform important functions in regulating B. anthracis virulence. CONCLUSIONS: A majority of genes on the virulence plasmid pXO1 that are regulated by the presence of either CO2 or AtxA separately are also regulated synergistically in the presence of both. These results also elucidate novel pXO1-encoded small RNAs that are associated with virulence conditions.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_024',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1708}]}],\n",
       "   'context': 'Cross-talk to the genes for Bacillus anthracis capsule synthesis by atxA, the gene encoding the trans-activator of anthrax toxin synthesis. The two major virulence factors of Bacillus anthracis are the tripartite toxin and the polyglutamate capsule, which are encoded by genes on the large plasmids, pXO1 and pXO2, respectively. The genes atxA, located on pXO1, and acpA, located on pXO2, encode positive trans-acting proteins that are involved in bicarbonate-mediated regulation of toxin and capsule production, respectively. A derivative strain cured of pXO1 produced less capsular substance than the parent strain harbouring both pXO1 and pXO2, and electroporation of the strain cured of pXO1 with a plasmid containing the cloned atxA gene resulted in an increased level of capsule production. An acpA-null mutant was complemented by not only acpA but also the atxA gene. The cap region, which is essential for encapsulation, contains three genes capB, capC, and capA, arranged in that order. The atxA gene stimulated capsule synthesis from the cloned cap region. Transcriptional analysis of cap by RNA slot-blot hybridization and primer-extension analysis revealed that atxA activated expression of cap in trans at the transcriptional level. These results indicate that cross-talk occurs, in which the pXO1-located gene, atxA, activates transcription of the cap region genes located on pXO2. We identified two major apparent transcriptional start sites, designated P1 and P2, located at positions 731 bp and 625 bp, respectively, upstream of the translation-initiation codon of capB. Transcription initiated from P1 and P2 was activated by both atxA and acpA, and activation appeared to be stimulated by bicarbonate. Deletion analysis of the upstream region of the cap promoter revealed that activation by both atxA and acpA required a DNA segment of 70 bp extending upstream of the P1 site. These results suggest that cross-talk by atxA to the genes encoding capsule synthesis is caused by the interaction of the atxA gene product with a regulatory sequence upstream of cap.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_025',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 75}]}],\n",
       "   'context': 'Transcriptome analysis identifies Bacillus anthracis genes that respond to CO2 through an AtxA-dependent mechanism. BACKGROUND: Upon infection of a mammalian host, Bacillus anthracis responds to host cues, and particularly to elevated temperature (37°C) and bicarbonate/CO2 concentrations, with increased expression of virulence factors that include the anthrax toxins and extracellular capsular layer. This response requires the presence of the pXO1 virulence plasmid-encoded pleiotropic regulator AtxA. To better understand the genetic basis of this response, we utilized a controlled in vitro system and Next Generation sequencing to determine and compare RNA expression profiles of the parental strain and an isogenic AtxA-deficient strain in a 2 × 2 factorial design with growth environments containing or lacking carbon dioxide. RESULTS: We found 15 pXO1-encoded genes and 3 chromosomal genes that were strongly regulated by the separate or synergistic actions of AtxA and carbon dioxide. The majority of the regulated genes responded to both AtxA and carbon dioxide rather than to just one of these factors. Interestingly, we identified two previously unrecognized small RNAs that are highly expressed under physiological carbon dioxide concentrations in an AtxA-dependent manner. Expression levels of the two small RNAs were found to be higher than that of any other gene differentially expressed in response to these conditions. Secondary structure and small RNA-mRNA binding predictions for the two small RNAs suggest that they may perform important functions in regulating B. anthracis virulence. CONCLUSIONS: A majority of genes on the virulence plasmid pXO1 that are regulated by the presence of either CO2 or AtxA separately are also regulated synergistically in the presence of both. These results also elucidate novel pXO1-encoded small RNAs that are associated with virulence conditions.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_026',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 372}]}],\n",
       "   'context': \"The atxA gene product activates transcription of the anthrax toxin genes and is essential for virulence. Bacillus anthracis plasmid pXO1 carries the structural genes for the three anthrax toxin proteins, cya (edema factor), lef (lethal factor), and pag (protective antigen). Expression of the toxin genes by B. anthracis is enhanced during growth under elevated levels of CO2. This CO2 effect is observed only in the presence of another pXO1 gene, atxA, which encodes a transactivator of anthrax toxin synthesis. Here we show that transcription of atxA does not appear to differ in cells grown in 5% CO2 compared with cells grown in air. Using a new efficient method for gene replacement in B. anthracis, we constructed an atxA-null mutant in which the atxA-coding sequence on pXO1 is replaced with an omega km-2 cassette. Transcription of all three toxin genes is decreased in the absence of atxA. The pag gene possesses two apparent transcription start sites, P1 and P2; only transcripts with 5' ends mapping to P1 are decreased in the atxA-null mutant. Deletion analysis of the pag promoter region indicates that the 111 bp region upstream of the P1 site is sufficient for atxA-mediated activation of this transcript. The cya and lef genes each have one apparent start site for transcription. Transcripts with 5' ends mapping to these sites are not detected in the atxA-null mutant. The atxA-null mutant is avirulent in mice. Moreover, the antibody response to all three toxin proteins is decreased significantly in atxA-null mutant-infected mice. These data suggest that the atxA gene product also regulates toxin gene expression during infection.\"},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_027',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 382}]}],\n",
       "   'context': \"The atxA gene product activates transcription of the anthrax toxin genes and is essential for virulence. Bacillus anthracis plasmid pXO1 carries the structural genes for the three anthrax toxin proteins, cya (edema factor), lef (lethal factor), and pag (protective antigen). Expression of the toxin genes by B. anthracis is enhanced during growth under elevated levels of CO2. This CO2 effect is observed only in the presence of another pXO1 gene, atxA, which encodes a transactivator of anthrax toxin synthesis. Here we show that transcription of atxA does not appear to differ in cells grown in 5% CO2 compared with cells grown in air. Using a new efficient method for gene replacement in B. anthracis, we constructed an atxA-null mutant in which the atxA-coding sequence on pXO1 is replaced with an omega km-2 cassette. Transcription of all three toxin genes is decreased in the absence of atxA. The pag gene possesses two apparent transcription start sites, P1 and P2; only transcripts with 5' ends mapping to P1 are decreased in the atxA-null mutant. Deletion analysis of the pag promoter region indicates that the 111 bp region upstream of the P1 site is sufficient for atxA-mediated activation of this transcript. The cya and lef genes each have one apparent start site for transcription. Transcripts with 5' ends mapping to these sites are not detected in the atxA-null mutant. The atxA-null mutant is avirulent in mice. Moreover, the antibody response to all three toxin proteins is decreased significantly in atxA-null mutant-infected mice. These data suggest that the atxA gene product also regulates toxin gene expression during infection.\"},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_028',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 600}]}],\n",
       "   'context': \"The atxA gene product activates transcription of the anthrax toxin genes and is essential for virulence. Bacillus anthracis plasmid pXO1 carries the structural genes for the three anthrax toxin proteins, cya (edema factor), lef (lethal factor), and pag (protective antigen). Expression of the toxin genes by B. anthracis is enhanced during growth under elevated levels of CO2. This CO2 effect is observed only in the presence of another pXO1 gene, atxA, which encodes a transactivator of anthrax toxin synthesis. Here we show that transcription of atxA does not appear to differ in cells grown in 5% CO2 compared with cells grown in air. Using a new efficient method for gene replacement in B. anthracis, we constructed an atxA-null mutant in which the atxA-coding sequence on pXO1 is replaced with an omega km-2 cassette. Transcription of all three toxin genes is decreased in the absence of atxA. The pag gene possesses two apparent transcription start sites, P1 and P2; only transcripts with 5' ends mapping to P1 are decreased in the atxA-null mutant. Deletion analysis of the pag promoter region indicates that the 111 bp region upstream of the P1 site is sufficient for atxA-mediated activation of this transcript. The cya and lef genes each have one apparent start site for transcription. Transcripts with 5' ends mapping to these sites are not detected in the atxA-null mutant. The atxA-null mutant is avirulent in mice. Moreover, the antibody response to all three toxin proteins is decreased significantly in atxA-null mutant-infected mice. These data suggest that the atxA gene product also regulates toxin gene expression during infection.\"},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_029',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1708}]}],\n",
       "   'context': 'Cross-talk to the genes for Bacillus anthracis capsule synthesis by atxA, the gene encoding the trans-activator of anthrax toxin synthesis. The two major virulence factors of Bacillus anthracis are the tripartite toxin and the polyglutamate capsule, which are encoded by genes on the large plasmids, pXO1 and pXO2, respectively. The genes atxA, located on pXO1, and acpA, located on pXO2, encode positive trans-acting proteins that are involved in bicarbonate-mediated regulation of toxin and capsule production, respectively. A derivative strain cured of pXO1 produced less capsular substance than the parent strain harbouring both pXO1 and pXO2, and electroporation of the strain cured of pXO1 with a plasmid containing the cloned atxA gene resulted in an increased level of capsule production. An acpA-null mutant was complemented by not only acpA but also the atxA gene. The cap region, which is essential for encapsulation, contains three genes capB, capC, and capA, arranged in that order. The atxA gene stimulated capsule synthesis from the cloned cap region. Transcriptional analysis of cap by RNA slot-blot hybridization and primer-extension analysis revealed that atxA activated expression of cap in trans at the transcriptional level. These results indicate that cross-talk occurs, in which the pXO1-located gene, atxA, activates transcription of the cap region genes located on pXO2. We identified two major apparent transcriptional start sites, designated P1 and P2, located at positions 731 bp and 625 bp, respectively, upstream of the translation-initiation codon of capB. Transcription initiated from P1 and P2 was activated by both atxA and acpA, and activation appeared to be stimulated by bicarbonate. Deletion analysis of the upstream region of the cap promoter revealed that activation by both atxA and acpA required a DNA segment of 70 bp extending upstream of the P1 site. These results suggest that cross-talk by atxA to the genes encoding capsule synthesis is caused by the interaction of the atxA gene product with a regulatory sequence upstream of cap.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_030',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1708}]}],\n",
       "   'context': 'Cross-talk to the genes for Bacillus anthracis capsule synthesis by atxA, the gene encoding the trans-activator of anthrax toxin synthesis. The two major virulence factors of Bacillus anthracis are the tripartite toxin and the polyglutamate capsule, which are encoded by genes on the large plasmids, pXO1 and pXO2, respectively. The genes atxA, located on pXO1, and acpA, located on pXO2, encode positive trans-acting proteins that are involved in bicarbonate-mediated regulation of toxin and capsule production, respectively. A derivative strain cured of pXO1 produced less capsular substance than the parent strain harbouring both pXO1 and pXO2, and electroporation of the strain cured of pXO1 with a plasmid containing the cloned atxA gene resulted in an increased level of capsule production. An acpA-null mutant was complemented by not only acpA but also the atxA gene. The cap region, which is essential for encapsulation, contains three genes capB, capC, and capA, arranged in that order. The atxA gene stimulated capsule synthesis from the cloned cap region. Transcriptional analysis of cap by RNA slot-blot hybridization and primer-extension analysis revealed that atxA activated expression of cap in trans at the transcriptional level. These results indicate that cross-talk occurs, in which the pXO1-located gene, atxA, activates transcription of the cap region genes located on pXO2. We identified two major apparent transcriptional start sites, designated P1 and P2, located at positions 731 bp and 625 bp, respectively, upstream of the translation-initiation codon of capB. Transcription initiated from P1 and P2 was activated by both atxA and acpA, and activation appeared to be stimulated by bicarbonate. Deletion analysis of the upstream region of the cap promoter revealed that activation by both atxA and acpA required a DNA segment of 70 bp extending upstream of the P1 site. These results suggest that cross-talk by atxA to the genes encoding capsule synthesis is caused by the interaction of the atxA gene product with a regulatory sequence upstream of cap.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_031',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 1708}]}],\n",
       "   'context': 'Cross-talk to the genes for Bacillus anthracis capsule synthesis by atxA, the gene encoding the trans-activator of anthrax toxin synthesis. The two major virulence factors of Bacillus anthracis are the tripartite toxin and the polyglutamate capsule, which are encoded by genes on the large plasmids, pXO1 and pXO2, respectively. The genes atxA, located on pXO1, and acpA, located on pXO2, encode positive trans-acting proteins that are involved in bicarbonate-mediated regulation of toxin and capsule production, respectively. A derivative strain cured of pXO1 produced less capsular substance than the parent strain harbouring both pXO1 and pXO2, and electroporation of the strain cured of pXO1 with a plasmid containing the cloned atxA gene resulted in an increased level of capsule production. An acpA-null mutant was complemented by not only acpA but also the atxA gene. The cap region, which is essential for encapsulation, contains three genes capB, capC, and capA, arranged in that order. The atxA gene stimulated capsule synthesis from the cloned cap region. Transcriptional analysis of cap by RNA slot-blot hybridization and primer-extension analysis revealed that atxA activated expression of cap in trans at the transcriptional level. These results indicate that cross-talk occurs, in which the pXO1-located gene, atxA, activates transcription of the cap region genes located on pXO2. We identified two major apparent transcriptional start sites, designated P1 and P2, located at positions 731 bp and 625 bp, respectively, upstream of the translation-initiation codon of capB. Transcription initiated from P1 and P2 was activated by both atxA and acpA, and activation appeared to be stimulated by bicarbonate. Deletion analysis of the upstream region of the cap promoter revealed that activation by both atxA and acpA required a DNA segment of 70 bp extending upstream of the P1 site. These results suggest that cross-talk by atxA to the genes encoding capsule synthesis is caused by the interaction of the atxA gene product with a regulatory sequence upstream of cap.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_032',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 101}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_033',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'bicarbonate', 'answer_start': 105}]}],\n",
       "   'context': 'Regulation of anthrax toxin activator gene (atxA) expression in Bacillus anthracis: temperature, not CO2/bicarbonate, affects AtxA synthesis. Anthrax toxin gene expression in Bacillus anthracis is dependent on the presence of atxA, a trans-acting regulatory gene located on the resident 185-kb plasmid pXO1. In atxA+ strains, expression of the toxin genes (pag, lef, and cya) is enhanced by two physiologically significant signals: elevated CO2/bicarbonate and temperature. To determine whether increased toxin gene expression in response to these signals is associated with increased atxA expression, we monitored steady-state levels of atxA mRNA and AtxA protein in cells cultured in different conditions. We purified histidine-tagged AtxA [AtxA(His)] from Escherichia coli and used anti-AtxA(His) serum to detect AtxA in protein preparations from B. anthracis cells. AtxA was identified as a protein with an apparent size of 56 kDa in cytoplasmic fractions of B. anthracis cells. Our data indicate that atxA expression is not influenced by CO2/bicarbonate levels. However, the steady-state level of atxA mRNA in cells grown in elevated CO2/bicarbonate at 37 degrees C is five- to sixfold higher than that observed in cells grown in the same conditions at 28 degrees C. A corresponding difference in AtxA protein was also seen at the different growth temperatures. When atxA was cloned on a multicopy plasmid in B. anthracis, AtxA levels corresponding to the atxA gene copy number were observed. However, this strain produced significantly less pag mRNA and protective antigen protein than the parental strain harboring atxA in single copy on pXO1. These results indicate that increased AtxA expression does not lead to a corresponding increase in pag expression. Our data strongly suggest that an additional factor(s) is involved in regulation of pag and that the relative amounts of such a factor(s) and AtxA are important for optimal toxin gene expression.'},\n",
       "  {'qas': [{'id': '5710a592cf1c32585100002a_034',\n",
       "     'question': 'Which metabolite activates AtxA?',\n",
       "     'answers': [{'text': 'CO2', 'answer_start': 57}]}],\n",
       "   'context': 'The anthrax toxin activator gene atxA is associated with CO2-enhanced non-toxin gene expression in Bacillus anthracis. The Bacillus anthracis toxin genes, cya, lef, and pag, can be viewed as a regulon, in which transcription of all three genes is activated in trans by the same regulatory gene, atxA, in response to the same signal, CO2. In atxA+ strains, toxin gene expression is increased 5- to 20-fold in cells grown in 5% CO2 relative to cells grown in air. CO2-enhanced toxin gene transcription is not observed in atx4-null mutants. Here, we used two independent techniques to obtain evidence for additional CO2-induced atxA-regulated genes. First, total protein preparations from atxA4+ and atxA isolates grown in 5% CO2 and in air were examined by two-dimensional electrophoresis. Comparison of the resulting protein patterns indicated that synthesis of non-toxin proteins is influenced by growth in elevated CO2 and the toxin gene regulator, atxA. Second, we generated random transcriptional lacZ fusions in B. anthracis with transposon Tn917-LTV3. Transposon-insertion libraries were screened for mutants expressing CO2-enhanced atxA-dependent beta-galactosidase activity. DNA sequence analysis of transposon insertion sites in 17 mutants carrying CO2- and atxA-regulated fusions revealed 10 mutants carrying independent insertions on the 185-kb toxin plasmid pXO1 which did not map to the toxin genes. The tcr-lacZ fusion mutants (tcr for toxin coregulated) were Tox+, indicating that these genes may not be involved in anthrax toxin gene activation. Our data indicate a clear association of atxA with CO2-enhanced gene expression in B. anthracis and provide evidence that atxA regulates genes other than the structural genes for the anthrax toxin proteins.'},\n",
       "  {'qas': [{'id': '532366f09b2d7acc7e000015_001',\n",
       "     'question': 'Which amino acid residue appears mutated in most of the cases reported with  cadasil syndrome?',\n",
       "     'answers': [{'text': 'cysteine', 'answer_start': 205}]}],\n",
       "   'context': 'Considerations on a mutation in the NOTCH3 gene sparing a cysteine residue: a rare polymorphism rather than a CADASIL variant. Some missense mutations and small deletions in the NOTCH3 gene, not involving cysteine residues, have been described in patients considered to be affected by paucisymptomatic CADASIL. However, the significance of such molecular variants is still unclear. We describe a 49-year-old woman with a CADASIL-like phenotype, carrying a novel cysteine-sparing mutation in exon 29 of the NOTCH3 gene, and discuss the possible pathogenetic role of this molecular variant. Even though atypical clinical and MRI findings make a diagnosis of CADASIL unlikely in this patient, our report nevertheless underlines the intriguing genotype-phenotype relationship in NOTCH3 mutations and the importance of functional investigation to ascertain the role of new NOTCH3 mutations in CADASIL pathogenesis.'},\n",
       "  {'qas': [{'id': '532366f09b2d7acc7e000015_002',\n",
       "     'question': 'Which amino acid residue appears mutated in most of the cases reported with  cadasil syndrome?',\n",
       "     'answers': [{'text': 'cysteine', 'answer_start': 442}]}],\n",
       "   'context': 'First deep intronic mutation in the NOTCH3 gene in a family with late-onset CADASIL. CADASIL is the most prominent inherited form of vascular dementia. The main clinical features include migraine with aura, stroke, mood disturbances, and cognitive decline, with a mid-life (30s-60s) adult onset. Genetic testing is the gold standard for the diagnosis. CADASIL is caused mostly by missense mutations in the NOTCH3 gene, invariably involving a cysteine residue. Only a couple of splice site mutations have been reported. In a few pathologically defined patients, genetic mutations remain unidentified. We report a family with late-onset CADASIL phenotype carrying a novel intronic deletion in the NOTCH3 gene (c.341-26_24delAAC). Transcript analysis revealed a splicing alteration, with the complete intron 3 retention. The insertion was in-frame and encoded an extra 25 amino acids, including 1 cysteine. This is the first report of an aberrant splicing event of the NOTCH3 gene associated with a mutation far away from the canonical splice site. Our finding suggests that the assays used to evaluate splicing should be mandatory in the diagnostic setting of genetically undefined CADASIL cases.'},\n",
       "  {'qas': [{'id': '532366f09b2d7acc7e000015_003',\n",
       "     'question': 'Which amino acid residue appears mutated in most of the cases reported with  cadasil syndrome?',\n",
       "     'answers': [{'text': 'cysteine', 'answer_start': 265}]}],\n",
       "   'context': 'NOTCH3 gene mutations in subjects clinically suspected of CADASIL. BACKGROUND: Cerebral autosomal dominant arteriopathy with subcortical infarcts and leukoencephalopathy (CADASIL) is an inherited cerebrovascular disease due to mutations involving loss or gain of a cysteine residue in the NOTCH3 gene. A cluster of mutations around exons 3 and 4 was originally reported. Identification of pathogenic mutation is important for diagnostic confirmation of the disease, however genetic counselling and testing of relatives at risk is critical in mutation carriers. METHODS: Mutation analysis of the NOTCH3 gene was performed through direct sequencing in 140 patients with clinical suspicion of CADASIL. Patients underwent genetic counselling pre and post testing. The 2-23 exons containing all EGF-like domains were screened. RESULTS: 14 familial forms of the disease have been identified with 14 different causative mutations in exons 2, 3, 4, 5, 7, 10, 14, 19, 20 and 22 of the NOTCH3 gene; no pathogenetic mutations have been identified in exons 6 and 8; several genetic variations both in coding as well as in intronic regions were identified too. CONCLUSIONS: Our data confirm the importance of screening the whole EGF-like domains region of NOTCH3 gene for the molecular diagnosis of CADASIL among the Italian population too. Moreover genetic variants different from loss or gain of a cysteine residue are identified and presented.'},\n",
       "  {'qas': [{'id': '532366f09b2d7acc7e000015_004',\n",
       "     'question': 'Which amino acid residue appears mutated in most of the cases reported with  cadasil syndrome?',\n",
       "     'answers': [{'text': 'cysteine', 'answer_start': 602}]}],\n",
       "   'context': 'A novel Notch3 gene mutation not involving a cysteine residue in an Italian family with CADASIL. Cerebral autosomal dominant arteriopathy with subcortical infarcts and leukoencephalopathy (CADASIL) is a hereditary cerebrovascular disease leading to accumulating neurologic deficits and dementia. CADASIL has been linked to nucleotide substitutions and deletions in the Notch3 gene. All the mutations described until now lead to unpaired cysteine residue in the epidermal growth factor-like repeats. The authors report a family with CADASIL carrying a deletion in the Notch3 gene that did not involve a cysteine residue.'},\n",
       "  {'qas': [{'id': '54ede28094afd61504000003_001',\n",
       "     'question': 'What is the disease in which patients are sensitive to DNA crosslinking agents, presenting with a high frequency of chromosomal aberrations?',\n",
       "     'answers': [{'text': 'Fanconi anemia', 'answer_start': 49}]}],\n",
       "   'context': 'Fanconi anemia: at the crossroads of DNA repair. Fanconi anemia (FA) is an autosomal disorder that causes genome instability. FA patients suffer developmental abnormalities, early-onset bone marrow failure, and a predisposition to cancer. The disease is manifested by defects in DNA repair, hypersensitivity to DNA crosslinking agents, and a high degree of chromosomal aberrations. The FA pathway comprises 13 disease-causing genes involved in maintaining genomic stability. The fast pace of study of the novel DNA damage network has led to the constant discovery of new FA-like genes involved in the pathway that when mutated lead to similar disorders. A majority of the FA proteins act as signal transducers and scaffolding proteins to employ other pathways to repair DNA. This review discusses what is known about the FA proteins and other recently linked FA-like proteins. The goal is to clarify how the proteins work together to carry out interstrand crosslink repair and homologous recombination-mediated repair of damaged DNA.'},\n",
       "  {'qas': [{'id': '54ede28094afd61504000003_002',\n",
       "     'question': 'What is the disease in which patients are sensitive to DNA crosslinking agents, presenting with a high frequency of chromosomal aberrations?',\n",
       "     'answers': [{'text': 'Fanconi anemia', 'answer_start': 102}]}],\n",
       "   'context': 'Effect of hydroxyurea and normal plasma on DNA synthesis in lymphocytes from Fanconi anemia patients. Fanconi anemia (FA) is characterized at the cellular level by a high frequency of spontaneous chromosomal aberrations; crosslinking agents cause an abnormal increase in the frequency of chromosomal damage, and semiconservative DNA synthesis is severely inhibited. Deoxyribonucleotides are needed in both semiconservative and repair DNA synthesis. To investigate the involvement of deoxyribonucleotide pools in the inhibition of DNA synthesis in FA, we evaluated the effect on FA lymphocytes of hydroxyurea (HU), an inhibitor of ribonucleotide reductase which is known to alter the intracellular levels of deoxyribonucleotides. To achieve this goal, lymphocyte cultures of 4 FA patients and 4 normal individuals were used. Cultures were treated with HU and/or mitomycin C and normal human plasma. All cultures were processed to detect the number of DNA synthesizing nuclei by autoradiography. Scoring of 2000 nuclei for each kind of culture every 6 h in the last 24 h of incubation showed that, in long incubation periods, DNA synthesis in FA is largely inhibited by HU and this hypersensitivity may be partially decreased by addition of normal human plasma. It is known that recovery from damage induced by HU involves several enzymes such as flavin oxido-reductase, superoxide dismutase and catalase which are involved in the production or scavenging of O2 radicals; FA cells are deficient in the detoxification of oxygen and this could explain the response of FA cells to HU.'},\n",
       "  {'qas': [{'id': '54ede28094afd61504000003_003',\n",
       "     'question': 'What is the disease in which patients are sensitive to DNA crosslinking agents, presenting with a high frequency of chromosomal aberrations?',\n",
       "     'answers': [{'text': 'Fanconi anemia', 'answer_start': 247}]}],\n",
       "   'context': 'Localization of Fanconi anemia C protein to the cytoplasm of mammalian cells. Features of chromosomal aberrations, hypersensitivity to DNA crosslinking agents, and predisposition to malignancy have suggested a fundamental anomaly of DNA repair in Fanconi anemia. The function of the recently isolated FACC (Fanconi anemia group C complementing) gene for a subset of this disorder is not yet known. The notion that FACC plays a direct role in DNA repair would predict that the polypeptide should reside in the nucleus. In this study, a polyclonal antiserum raised against FACC was used to determine the subcellular location of the polypeptide. Immunofluorescence and subcellular fractionation studies of human cell lines as well as COS-7 cells transiently expressing human FACC showed that the protein was localized primarily to the cytoplasm under steady-state conditions, transit through the cell cycle, and exposure to crosslinking or cytotoxic agents. However, placement of a nuclear localization signal from the simian virus 40 large tumor antigen at the amino terminus of FACC directed the hybrid protein to the nuclei of transfected COS-7 cells. These observations suggest an indirect role for FACC in regulating DNA repair in this group of Fanconi anemia.'},\n",
       "  {'qas': [{'id': '54ede28094afd61504000003_004',\n",
       "     'question': 'What is the disease in which patients are sensitive to DNA crosslinking agents, presenting with a high frequency of chromosomal aberrations?',\n",
       "     'answers': [{'text': 'Fanconi anemia', 'answer_start': 102}]}],\n",
       "   'context': 'Effect of hydroxyurea and normal plasma on DNA synthesis in lymphocytes from Fanconi anemia patients. Fanconi anemia (FA) is characterized at the cellular level by a high frequency of spontaneous chromosomal aberrations; crosslinking agents cause an abnormal increase in the frequency of chromosomal damage, and semiconservative DNA synthesis is severely inhibited. Deoxyribonucleotides are needed in both semiconservative and repair DNA synthesis. To investigate the involvement of deoxyribonucleotide pools in the inhibition of DNA synthesis in FA, we evaluated the effect on FA lymphocytes of hydroxyurea (HU), an inhibitor of ribonucleotide reductase which is known to alter the intracellular levels of deoxyribonucleotides. To achieve this goal, lymphocyte cultures of 4 FA patients and 4 normal individuals were used. Cultures were treated with HU and/or mitomycin C and normal human plasma. All cultures were processed to detect the number of DNA synthesizing nuclei by autoradiography. Scoring of 2000 nuclei for each kind of culture every 6 h in the last 24 h of incubation showed that, in long incubation periods, DNA synthesis in FA is largely inhibited by HU and this hypersensitivity may be partially decreased by addition of normal human plasma. It is known that recovery from damage induced by HU involves several enzymes such as flavin oxido-reductase, superoxide dismutase and catalase which are involved in the production or scavenging of O2 radicals; FA cells are deficient in the detoxification of oxygen and this could explain the response of FA cells to HU.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_001',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 814}]}],\n",
       "   'context': 'Proteasome inhibitors - molecular basis and current perspectives in multiple myeloma. Inhibition of proteasome, a proteolytic complex responsible for the degradation of ubiquitinated proteins, has emerged as a powerful strategy for treatment of multiple myeloma (MM), a plasma cell malignancy. First-in-class agent, bortezomib, has demonstrated great positive therapeutic efficacy in MM, both in pre-clinical and in clinical studies. However, despite its high efficiency, a large proportion of patients do not achieve sufficient clinical response. Therefore, the development of a second-generation of proteasome inhibitors (PIs) with improved pharmacological properties was needed. Recently, several of these new agents have been introduced into clinics including carfilzomib, marizomib and ixazomib. Further, new orally administered second-generation PI oprozomib is being investigated. This review provides an overview of main mechanisms of action of PIs in MM, focusing on the ongoing development and progress of novel anti-proteasome therapeutics.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_002',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 312}]}],\n",
       "   'context': 'Carfilzomib and oprozomib synergize with histone deacetylase inhibitors in head and neck squamous cell carcinoma models of acquired resistance to proteasome inhibitors. Acquired resistance to proteasome inhibitors represents a considerable impediment to their effective clinical application. Carfilzomib and its orally bioavailable structural analog oprozomib are second-generation, highly-selective, proteasome inhibitors. However, the mechanisms of acquired resistance to carfilzomib and oprozomib are incompletely understood, and effective strategies for overcoming this resistance are needed. Here, we developed models of acquired resistance to carfilzomib in two head and neck squamous cell carcinoma cell lines, UMSCC-1 and Cal33, through gradual exposure to increasing drug concentrations. The resistant lines R-UMSCC-1 and R-Cal33 demonstrated 205- and 64-fold resistance, respectively, relative to the parental lines. Similarly, a high level of cross-resistance to oprozomib, as well as paclitaxel, was observed, whereas only moderate resistance to bortezomib (8- to 29-fold), and low level resistance to cisplatin (1.5- to 5-fold) was seen. Synergistic induction of apoptosis signaling and cell death, and inhibition of colony formation followed co-treatment of acquired resistance models with carfilzomib and the histone deacetylase inhibitor (HDACi) vorinostat. Synergism was also seen with other combinations, including oprozomib plus vorinostat, or carfilzomib plus the HDACi entinostat. Synergism was accompanied by upregulation of proapoptotic Bik, and suppression of Bik attenuated the synergy. The acquired resistance models also exhibited elevated levels of MDR-1/P-gp. Inhibition of MDR-1/P-gp with reversin 121 partially overcame carfilzomib resistance in R-UMSCC-1 and R-Cal33 cells. Collectively, these studies indicate that combining carfilzomib or oprozomib with HDAC or MDR-1/P-gp inhibitors may be a useful strategy for overcoming acquired resistance to these proteasome inhibitors.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_003',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 885}]}],\n",
       "   'context': 'Current strategies for treatment of relapsed/refractory multiple myeloma. In spite of significant advances in the management of multiple myeloma (MM), the disease remains incurable and nearly all patients ultimately relapse and require salvage chemotherapy. As such, relapsed and relapsed-refractory MM remains a critical area of research pertaining to biological mechanisms of progression and chemotherapy resistance, as well as to the development of new pharmacologic agents and immunologic approaches for the disease. The immunomodulatory agents and proteasome inhibitors represent the cornerstone of treatment in this setting, with combination regimens incorporating these drugs demonstrating encouraging rates and duration of response, including the newer agents, pomalidomide and carfilzomib. In addition, novel drug classes have shown promising activity in RR MM, including the orally-administered proteasome inhibitors ixazomib and oprozomib; monoclonal antibodies such as the anti-CS1 monoclonal antibody elotuzumab and anti-CD38 monoclonal antibody daratumumab; and histone deacetylase inhibitors such as panobinostat and rocilinostat.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_004',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 814}]}],\n",
       "   'context': 'Proteasome inhibitors - molecular basis and current perspectives in multiple myeloma. Inhibition of proteasome, a proteolytic complex responsible for the degradation of ubiquitinated proteins, has emerged as a powerful strategy for treatment of multiple myeloma (MM), a plasma cell malignancy. First-in-class agent, bortezomib, has demonstrated great positive therapeutic efficacy in MM, both in pre-clinical and in clinical studies. However, despite its high efficiency, a large proportion of patients do not achieve sufficient clinical response. Therefore, the development of a second-generation of proteasome inhibitors (PIs) with improved pharmacological properties was needed. Recently, several of these new agents have been introduced into clinics including carfilzomib, marizomib and ixazomib. Further, new orally administered second-generation PI oprozomib is being investigated. This review provides an overview of main mechanisms of action of PIs in MM, focusing on the ongoing development and progress of novel anti-proteasome therapeutics.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_005',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 885}]}],\n",
       "   'context': 'Current strategies for treatment of relapsed/refractory multiple myeloma. In spite of significant advances in the management of multiple myeloma (MM), the disease remains incurable and nearly all patients ultimately relapse and require salvage chemotherapy. As such, relapsed and relapsed-refractory MM remains a critical area of research pertaining to biological mechanisms of progression and chemotherapy resistance, as well as to the development of new pharmacologic agents and immunologic approaches for the disease. The immunomodulatory agents and proteasome inhibitors represent the cornerstone of treatment in this setting, with combination regimens incorporating these drugs demonstrating encouraging rates and duration of response, including the newer agents, pomalidomide and carfilzomib. In addition, novel drug classes have shown promising activity in RR MM, including the orally-administered proteasome inhibitors ixazomib and oprozomib; monoclonal antibodies such as the anti-CS1 monoclonal antibody elotuzumab and anti-CD38 monoclonal antibody daratumumab; and histone deacetylase inhibitors such as panobinostat and rocilinostat.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_006',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 561}]}],\n",
       "   'context': 'The epoxyketone-based proteasome inhibitors carfilzomib and orally bioavailable oprozomib have anti-resorptive and bone-anabolic activity in addition to anti-myeloma effects. Proteasome inhibitors (PIs), namely bortezomib, have become a cornerstone therapy for multiple myeloma (MM), potently reducing tumor burden and inhibiting pathologic bone destruction. In clinical trials, carfilzomib, a next generation epoxyketone-based irreversible PI, has exhibited potent anti-myeloma efficacy and decreased side effects compared with bortezomib. Carfilzomib and its orally bioavailable analog oprozomib, effectively decreased MM cell viability following continual or transient treatment mimicking in vivo pharmacokinetics. Interactions between myeloma cells and the bone marrow (BM) microenvironment augment the number and activity of bone-resorbing osteoclasts (OCs) while inhibiting bone-forming osteoblasts (OBs), resulting in increased tumor growth and osteolytic lesions. At clinically relevant concentrations, carfilzomib and oprozomib directly inhibited OC formation and bone resorption in vitro, while enhancing osteogenic differentiation and matrix mineralization. Accordingly, carfilzomib and oprozomib increased trabecular bone volume, decreased bone resorption and enhanced bone formation in non-tumor bearing mice. Finally, in mouse models of disseminated MM, the epoxyketone-based PIs decreased murine 5TGM1 and human RPMI-8226 tumor burden and prevented bone loss. These data demonstrate that, in addition to anti-myeloma properties, carfilzomib and oprozomib effectively shift the bone microenvironment from a catabolic to an anabolic state and, similar to bortezomib, may decrease skeletal complications of MM.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_007',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 312}]}],\n",
       "   'context': 'Carfilzomib and oprozomib synergize with histone deacetylase inhibitors in head and neck squamous cell carcinoma models of acquired resistance to proteasome inhibitors. Acquired resistance to proteasome inhibitors represents a considerable impediment to their effective clinical application. Carfilzomib and its orally bioavailable structural analog oprozomib are second-generation, highly-selective, proteasome inhibitors. However, the mechanisms of acquired resistance to carfilzomib and oprozomib are incompletely understood, and effective strategies for overcoming this resistance are needed. Here, we developed models of acquired resistance to carfilzomib in two head and neck squamous cell carcinoma cell lines, UMSCC-1 and Cal33, through gradual exposure to increasing drug concentrations. The resistant lines R-UMSCC-1 and R-Cal33 demonstrated 205- and 64-fold resistance, respectively, relative to the parental lines. Similarly, a high level of cross-resistance to oprozomib, as well as paclitaxel, was observed, whereas only moderate resistance to bortezomib (8- to 29-fold), and low level resistance to cisplatin (1.5- to 5-fold) was seen. Synergistic induction of apoptosis signaling and cell death, and inhibition of colony formation followed co-treatment of acquired resistance models with carfilzomib and the histone deacetylase inhibitor (HDACi) vorinostat. Synergism was also seen with other combinations, including oprozomib plus vorinostat, or carfilzomib plus the HDACi entinostat. Synergism was accompanied by upregulation of proapoptotic Bik, and suppression of Bik attenuated the synergy. The acquired resistance models also exhibited elevated levels of MDR-1/P-gp. Inhibition of MDR-1/P-gp with reversin 121 partially overcame carfilzomib resistance in R-UMSCC-1 and R-Cal33 cells. Collectively, these studies indicate that combining carfilzomib or oprozomib with HDAC or MDR-1/P-gp inhibitors may be a useful strategy for overcoming acquired resistance to these proteasome inhibitors.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_008',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 814}]}],\n",
       "   'context': 'Proteasome inhibitors - molecular basis and current perspectives in multiple myeloma. Inhibition of proteasome, a proteolytic complex responsible for the degradation of ubiquitinated proteins, has emerged as a powerful strategy for treatment of multiple myeloma (MM), a plasma cell malignancy. First-in-class agent, bortezomib, has demonstrated great positive therapeutic efficacy in MM, both in pre-clinical and in clinical studies. However, despite its high efficiency, a large proportion of patients do not achieve sufficient clinical response. Therefore, the development of a second-generation of proteasome inhibitors (PIs) with improved pharmacological properties was needed. Recently, several of these new agents have been introduced into clinics including carfilzomib, marizomib and ixazomib. Further, new orally administered second-generation PI oprozomib is being investigated. This review provides an overview of main mechanisms of action of PIs in MM, focusing on the ongoing development and progress of novel anti-proteasome therapeutics.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_009',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 885}]}],\n",
       "   'context': 'Current strategies for treatment of relapsed/refractory multiple myeloma. In spite of significant advances in the management of multiple myeloma (MM), the disease remains incurable and nearly all patients ultimately relapse and require salvage chemotherapy. As such, relapsed and relapsed-refractory MM remains a critical area of research pertaining to biological mechanisms of progression and chemotherapy resistance, as well as to the development of new pharmacologic agents and immunologic approaches for the disease. The immunomodulatory agents and proteasome inhibitors represent the cornerstone of treatment in this setting, with combination regimens incorporating these drugs demonstrating encouraging rates and duration of response, including the newer agents, pomalidomide and carfilzomib. In addition, novel drug classes have shown promising activity in RR MM, including the orally-administered proteasome inhibitors ixazomib and oprozomib; monoclonal antibodies such as the anti-CS1 monoclonal antibody elotuzumab and anti-CD38 monoclonal antibody daratumumab; and histone deacetylase inhibitors such as panobinostat and rocilinostat.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_010',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 60}]}],\n",
       "   'context': 'The epoxyketone-based proteasome inhibitors carfilzomib and orally bioavailable oprozomib have anti-resorptive and bone-anabolic activity in addition to anti-myeloma effects. Proteasome inhibitors (PIs), namely bortezomib, have become a cornerstone therapy for multiple myeloma (MM), potently reducing tumor burden and inhibiting pathologic bone destruction. In clinical trials, carfilzomib, a next generation epoxyketone-based irreversible PI, has exhibited potent anti-myeloma efficacy and decreased side effects compared with bortezomib. Carfilzomib and its orally bioavailable analog oprozomib, effectively decreased MM cell viability following continual or transient treatment mimicking in vivo pharmacokinetics. Interactions between myeloma cells and the bone marrow (BM) microenvironment augment the number and activity of bone-resorbing osteoclasts (OCs) while inhibiting bone-forming osteoblasts (OBs), resulting in increased tumor growth and osteolytic lesions. At clinically relevant concentrations, carfilzomib and oprozomib directly inhibited OC formation and bone resorption in vitro, while enhancing osteogenic differentiation and matrix mineralization. Accordingly, carfilzomib and oprozomib increased trabecular bone volume, decreased bone resorption and enhanced bone formation in non-tumor bearing mice. Finally, in mouse models of disseminated MM, the epoxyketone-based PIs decreased murine 5TGM1 and human RPMI-8226 tumor burden and prevented bone loss. These data demonstrate that, in addition to anti-myeloma properties, carfilzomib and oprozomib effectively shift the bone microenvironment from a catabolic to an anabolic state and, similar to bortezomib, may decrease skeletal complications of MM.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_011',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 312}]}],\n",
       "   'context': 'Carfilzomib and oprozomib synergize with histone deacetylase inhibitors in head and neck squamous cell carcinoma models of acquired resistance to proteasome inhibitors. Acquired resistance to proteasome inhibitors represents a considerable impediment to their effective clinical application. Carfilzomib and its orally bioavailable structural analog oprozomib are second-generation, highly-selective, proteasome inhibitors. However, the mechanisms of acquired resistance to carfilzomib and oprozomib are incompletely understood, and effective strategies for overcoming this resistance are needed. Here, we developed models of acquired resistance to carfilzomib in two head and neck squamous cell carcinoma cell lines, UMSCC-1 and Cal33, through gradual exposure to increasing drug concentrations. The resistant lines R-UMSCC-1 and R-Cal33 demonstrated 205- and 64-fold resistance, respectively, relative to the parental lines. Similarly, a high level of cross-resistance to oprozomib, as well as paclitaxel, was observed, whereas only moderate resistance to bortezomib (8- to 29-fold), and low level resistance to cisplatin (1.5- to 5-fold) was seen. Synergistic induction of apoptosis signaling and cell death, and inhibition of colony formation followed co-treatment of acquired resistance models with carfilzomib and the histone deacetylase inhibitor (HDACi) vorinostat. Synergism was also seen with other combinations, including oprozomib plus vorinostat, or carfilzomib plus the HDACi entinostat. Synergism was accompanied by upregulation of proapoptotic Bik, and suppression of Bik attenuated the synergy. The acquired resistance models also exhibited elevated levels of MDR-1/P-gp. Inhibition of MDR-1/P-gp with reversin 121 partially overcame carfilzomib resistance in R-UMSCC-1 and R-Cal33 cells. Collectively, these studies indicate that combining carfilzomib or oprozomib with HDAC or MDR-1/P-gp inhibitors may be a useful strategy for overcoming acquired resistance to these proteasome inhibitors.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_012',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 561}]}],\n",
       "   'context': 'The epoxyketone-based proteasome inhibitors carfilzomib and orally bioavailable oprozomib have anti-resorptive and bone-anabolic activity in addition to anti-myeloma effects. Proteasome inhibitors (PIs), namely bortezomib, have become a cornerstone therapy for multiple myeloma (MM), potently reducing tumor burden and inhibiting pathologic bone destruction. In clinical trials, carfilzomib, a next generation epoxyketone-based irreversible PI, has exhibited potent anti-myeloma efficacy and decreased side effects compared with bortezomib. Carfilzomib and its orally bioavailable analog oprozomib, effectively decreased MM cell viability following continual or transient treatment mimicking in vivo pharmacokinetics. Interactions between myeloma cells and the bone marrow (BM) microenvironment augment the number and activity of bone-resorbing osteoclasts (OCs) while inhibiting bone-forming osteoblasts (OBs), resulting in increased tumor growth and osteolytic lesions. At clinically relevant concentrations, carfilzomib and oprozomib directly inhibited OC formation and bone resorption in vitro, while enhancing osteogenic differentiation and matrix mineralization. Accordingly, carfilzomib and oprozomib increased trabecular bone volume, decreased bone resorption and enhanced bone formation in non-tumor bearing mice. Finally, in mouse models of disseminated MM, the epoxyketone-based PIs decreased murine 5TGM1 and human RPMI-8226 tumor burden and prevented bone loss. These data demonstrate that, in addition to anti-myeloma properties, carfilzomib and oprozomib effectively shift the bone microenvironment from a catabolic to an anabolic state and, similar to bortezomib, may decrease skeletal complications of MM.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_013',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 312}]}],\n",
       "   'context': 'Carfilzomib and oprozomib synergize with histone deacetylase inhibitors in head and neck squamous cell carcinoma models of acquired resistance to proteasome inhibitors. Acquired resistance to proteasome inhibitors represents a considerable impediment to their effective clinical application. Carfilzomib and its orally bioavailable structural analog oprozomib are second-generation, highly-selective, proteasome inhibitors. However, the mechanisms of acquired resistance to carfilzomib and oprozomib are incompletely understood, and effective strategies for overcoming this resistance are needed. Here, we developed models of acquired resistance to carfilzomib in two head and neck squamous cell carcinoma cell lines, UMSCC-1 and Cal33, through gradual exposure to increasing drug concentrations. The resistant lines R-UMSCC-1 and R-Cal33 demonstrated 205- and 64-fold resistance, respectively, relative to the parental lines. Similarly, a high level of cross-resistance to oprozomib, as well as paclitaxel, was observed, whereas only moderate resistance to bortezomib (8- to 29-fold), and low level resistance to cisplatin (1.5- to 5-fold) was seen. Synergistic induction of apoptosis signaling and cell death, and inhibition of colony formation followed co-treatment of acquired resistance models with carfilzomib and the histone deacetylase inhibitor (HDACi) vorinostat. Synergism was also seen with other combinations, including oprozomib plus vorinostat, or carfilzomib plus the HDACi entinostat. Synergism was accompanied by upregulation of proapoptotic Bik, and suppression of Bik attenuated the synergy. The acquired resistance models also exhibited elevated levels of MDR-1/P-gp. Inhibition of MDR-1/P-gp with reversin 121 partially overcame carfilzomib resistance in R-UMSCC-1 and R-Cal33 cells. Collectively, these studies indicate that combining carfilzomib or oprozomib with HDAC or MDR-1/P-gp inhibitors may be a useful strategy for overcoming acquired resistance to these proteasome inhibitors.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_014',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 814}]}],\n",
       "   'context': 'Proteasome inhibitors - molecular basis and current perspectives in multiple myeloma. Inhibition of proteasome, a proteolytic complex responsible for the degradation of ubiquitinated proteins, has emerged as a powerful strategy for treatment of multiple myeloma (MM), a plasma cell malignancy. First-in-class agent, bortezomib, has demonstrated great positive therapeutic efficacy in MM, both in pre-clinical and in clinical studies. However, despite its high efficiency, a large proportion of patients do not achieve sufficient clinical response. Therefore, the development of a second-generation of proteasome inhibitors (PIs) with improved pharmacological properties was needed. Recently, several of these new agents have been introduced into clinics including carfilzomib, marizomib and ixazomib. Further, new orally administered second-generation PI oprozomib is being investigated. This review provides an overview of main mechanisms of action of PIs in MM, focusing on the ongoing development and progress of novel anti-proteasome therapeutics.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_015',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 885}]}],\n",
       "   'context': 'Current strategies for treatment of relapsed/refractory multiple myeloma. In spite of significant advances in the management of multiple myeloma (MM), the disease remains incurable and nearly all patients ultimately relapse and require salvage chemotherapy. As such, relapsed and relapsed-refractory MM remains a critical area of research pertaining to biological mechanisms of progression and chemotherapy resistance, as well as to the development of new pharmacologic agents and immunologic approaches for the disease. The immunomodulatory agents and proteasome inhibitors represent the cornerstone of treatment in this setting, with combination regimens incorporating these drugs demonstrating encouraging rates and duration of response, including the newer agents, pomalidomide and carfilzomib. In addition, novel drug classes have shown promising activity in RR MM, including the orally-administered proteasome inhibitors ixazomib and oprozomib; monoclonal antibodies such as the anti-CS1 monoclonal antibody elotuzumab and anti-CD38 monoclonal antibody daratumumab; and histone deacetylase inhibitors such as panobinostat and rocilinostat.'},\n",
       "  {'qas': [{'id': '56ecfd572ac5ed1459000002_016',\n",
       "     'question': 'How is oprozomib administered?',\n",
       "     'answers': [{'text': 'orally', 'answer_start': 312}]}],\n",
       "   'context': 'Carfilzomib and oprozomib synergize with histone deacetylase inhibitors in head and neck squamous cell carcinoma models of acquired resistance to proteasome inhibitors. Acquired resistance to proteasome inhibitors represents a considerable impediment to their effective clinical application. Carfilzomib and its orally bioavailable structural analog oprozomib are second-generation, highly-selective, proteasome inhibitors. However, the mechanisms of acquired resistance to carfilzomib and oprozomib are incompletely understood, and effective strategies for overcoming this resistance are needed. Here, we developed models of acquired resistance to carfilzomib in two head and neck squamous cell carcinoma cell lines, UMSCC-1 and Cal33, through gradual exposure to increasing drug concentrations. The resistant lines R-UMSCC-1 and R-Cal33 demonstrated 205- and 64-fold resistance, respectively, relative to the parental lines. Similarly, a high level of cross-resistance to oprozomib, as well as paclitaxel, was observed, whereas only moderate resistance to bortezomib (8- to 29-fold), and low level resistance to cisplatin (1.5- to 5-fold) was seen. Synergistic induction of apoptosis signaling and cell death, and inhibition of colony formation followed co-treatment of acquired resistance models with carfilzomib and the histone deacetylase inhibitor (HDACi) vorinostat. Synergism was also seen with other combinations, including oprozomib plus vorinostat, or carfilzomib plus the HDACi entinostat. Synergism was accompanied by upregulation of proapoptotic Bik, and suppression of Bik attenuated the synergy. The acquired resistance models also exhibited elevated levels of MDR-1/P-gp. Inhibition of MDR-1/P-gp with reversin 121 partially overcame carfilzomib resistance in R-UMSCC-1 and R-Cal33 cells. Collectively, these studies indicate that combining carfilzomib or oprozomib with HDAC or MDR-1/P-gp inhibitors may be a useful strategy for overcoming acquired resistance to these proteasome inhibitors.'},\n",
       "  {'qas': [{'id': '5709ee36cf1c32585100001e_001',\n",
       "     'question': 'Which syndrome is associated with mutant DVL1?',\n",
       "     'answers': [{'text': 'Robinow syndrome', 'answer_start': 50}]}],\n",
       "   'context': 'Mutations in DVL1 cause an osteosclerotic form of Robinow syndrome. Robinow syndrome (RS) is a phenotypically and genetically heterogeneous condition that can be caused by mutations in genes encoding components of the non-canonical Wnt signaling pathway. In contrast, germline mutations that act to increase canonical Wnt signaling lead to distinctive osteosclerotic phenotypes. Here, we identified de novo frameshift mutations in DVL1, a mediator of both canonical and non-canonical Wnt signaling, as the cause of RS-OS, an RS subtype involving osteosclerosis, in three unrelated individuals. The mutations all delete the DVL1 C terminus and replace it, in each instance, with a novel, highly basic sequence. We showed the presence of mutant transcript in fibroblasts from one individual with RS-OS and demonstrated unimpaired protein stability with transfected GFP-tagged constructs bearing a frameshift mutation. In\\xa0vitro TOPFlash assays, in apparent contradiction to the osteosclerotic phenotype, revealed that the mutant allele was less active than the wild-type allele in the canonical Wnt signaling pathway. However, when the mutant and wild-type alleles were co-expressed, canonical Wnt activity was 2-fold higher than that in the wild-type construct alone. This work establishes that DVL1 mutations cause a specific RS subtype, RS-OS, and that the osteosclerosis associated with this subtype might be the result of an interaction between the wild-type and mutant alleles and thus lead to elevated canonical Wnt signaling.'},\n",
       "  {'qas': [{'id': '5709ee36cf1c32585100001e_002',\n",
       "     'question': 'Which syndrome is associated with mutant DVL1?',\n",
       "     'answers': [{'text': 'Robinow syndrome', 'answer_start': 86}]}],\n",
       "   'context': \"DVL1 frameshift mutations clustering in the penultimate exon cause autosomal-dominant Robinow syndrome. Robinow syndrome is a genetically heterogeneous disorder characterized by mesomelic limb shortening, genital hypoplasia, and distinctive facial features and for which both autosomal-recessive and autosomal-dominant inheritance patterns have been described. Causative variants in the non-canonical signaling gene WNT5A underlie a subset of autosomal-dominant Robinow syndrome (DRS) cases, but most individuals with DRS remain without a molecular diagnosis. We performed whole-exome sequencing in four unrelated DRS-affected individuals without coding mutations in WNT5A and found heterozygous DVL1 exon 14 mutations in three of them. Targeted Sanger sequencing in additional subjects with DRS uncovered DVL1 exon 14 mutations in five individuals, including a pair of monozygotic twins. In total, six distinct frameshift mutations were found in eight subjects, and all were heterozygous truncating variants within the penultimate exon of DVL1. In five families in which samples from unaffected parents were available, the variants were demonstrated to represent de novo mutations. All variant alleles are predicted to result in a premature termination codon within the last exon, escape nonsense-mediated decay (NMD), and most likely generate a C-terminally truncated protein with a distinct -1 reading-frame terminus. Study of the transcripts extracted from affected subjects' leukocytes confirmed expression of both wild-type and variant alleles, supporting the hypothesis that mutant mRNA escapes NMD. Genomic variants identified in our study suggest that truncation of the C-terminal domain of DVL1, a protein hypothesized to have a downstream role in the Wnt-5a non-canonical pathway, is a common cause of DRS.\"},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_001',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 194}]}],\n",
       "   'context': 'PARP inhibition restores extrinsic apoptotic sensitivity in glioblastoma. BACKGROUND: Resistance to apoptosis is a paramount issue in the treatment of Glioblastoma (GBM). We show that targeting PARP by the small molecule inhibitors, Olaparib (AZD-2281) or PJ34, reduces proliferation and lowers the apoptotic threshold of GBM cells in vitro and in vivo. METHODS: The sensitizing effects of PARP inhibition on TRAIL-mediated apoptosis and potential toxicity were analyzed using viability assays and flow cytometry in established GBM cell lines, low-passage neurospheres and astrocytes in vitro. Molecular analyses included western blots and gene silencing. In vivo, effects on tumor growth were examined in a murine subcutaneous xenograft model. RESULTS: The combination treatment of PARP inhibitors and TRAIL led to an increased cell death with activation of caspases and inhibition of formation of neurospheres when compared to single-agent treatment. Mechanistically, pharmacological PARP inhibition elicited a nuclear stress response with up-regulation of down-stream DNA-stress response proteins, e.g., CCAAT enhancer binding protein (C/EBP) homology protein (CHOP). Furthermore, Olaparib and PJ34 increased protein levels of DR5 in a concentration and time-dependent manner. In turn, siRNA-mediated suppression of DR5 mitigated the effects of TRAIL/PARP inhibitor-mediated apoptosis. In addition, suppression of PARP-1 levels enhanced TRAIL-mediated apoptosis in malignant glioma cells. Treatment of human astrocytes with the combination of TRAIL/PARP inhibitors did not cause toxicity. Finally, the combination treatment of TRAIL and PJ34 significantly reduced tumor growth in vivo when compared to treatment with each agent alone. CONCLUSIONS: PARP inhibition represents a promising avenue to overcome apoptotic resistance in GBM.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_002',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 843}]}],\n",
       "   'context': \"XRCC2 promotes colorectal cancer cell growth, regulates cell cycle progression, and apoptosis. X-ray repair complementing defective repair in Chinese hamster cells 2 (XRCC2) and poly(ADP-ribose) polymerase 1 (PARP1) both play important roles in homologous recombination DNA repair. According to the theory of synthetic lethality, XRCC2-deficient cells are more sensitive to PARP1 inhibitors compared to XRCC2-expressing cells. We investigated XRCC2 expression and function in colorectal cancer (CRC), and the characteristics of sensitivity to PARP1 inhibitor in CRC cells with different XRCC2 levels. We enrolled 153 patients with CRC who had undergone surgery in this study. XRCC2 expression was assessed using immunohistochemistry. Stable CRC SW480 cell lines with low or high XRCC2 expression were constructed. Following treatment with the PARP1 inhibitor olaparib, the viability of cells with different XRCC2 levels was determined; cell cycle distribution and apoptosis were analyzed using flow cytometry. B-cell lymphoma-2 (Bcl-2) protein expression was measured by Western blotting. The positive rates of XRCC2 in primary CRC tissue were significantly higher than that in the matched adjacent noncancerous tissue, and XRCC2 expression status in primary CRC was related to tumor site, Dukes' stage, and tumor-nodes-metastasis (TNM) stage. XRCC2 overexpression inhibited CRC cell apoptosis and promoted proliferation by enriching cells in the G0/G1 phase. Moreover, olaparib suppressed proliferation, and olaparib sensitivity in CRC cells with high XRCC2 expression was greater. High XRCC2 expression promotes CRC cell proliferation and enriches cells in the G0/G1 phase but inhibits apoptosis. High XRCC2 expression cells are more sensitive to olaparib, which inhibits their viability.\"},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_003',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 88}]}],\n",
       "   'context': 'BRCA1, PARP1 and γH2AX in acute myeloid leukemia: Role as biomarkers of response to the PARP inhibitor olaparib. Olaparib (AZD-2281, Ku-0059436) is an orally bioavailable and well-tolerated poly(ADP-ribose) polymerase (PARP) inhibitor currently under investigation in patients with solid tumors. To study the clinical potential of olaparib as a single-agent for the treatment of acute myeloid leukemia (AML) patients, we analyzed the in vitro sensitivity of AML cell lines and primary blasts. Clinically achievable concentrations of olaparib were able to induce cell death in the majority of primary AML case samples (88%) and tested cell lines. At these concentrations, olaparib preferentially killed leukemic blasts sparing normal lymphocytes derived from the same patient and did not substantially affect the viability of normal bone marrow and CD34-enriched peripheral blood cells obtained from healthy donors. Most primary AML analyzed were characterized by low BRCA1 mRNA level and undetectable protein expression that likely contributed to explain their sensitivity to olaparib. Noteworthy, while PARP1 over-expression was detected in blasts not responsive to olaparib, phosphorylation of the histone H2AFX (γH2AX) was associated with drug sensitivity. As to genetic features of tested cases the highest sensitivity was shown by a patient carrying a 11q23 deletion. The high sensitivity of AML blasts and the identification of biomarkers potentially able to predict response and/or resistance may foster further investigation of olaparib monotherapy for AML patients unfit to conventional chemotherapy.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_004',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'poly(ADP-ribose) polymerase',\n",
       "       'answer_start': 190}]}],\n",
       "   'context': 'BRCA1, PARP1 and γH2AX in acute myeloid leukemia: Role as biomarkers of response to the PARP inhibitor olaparib. Olaparib (AZD-2281, Ku-0059436) is an orally bioavailable and well-tolerated poly(ADP-ribose) polymerase (PARP) inhibitor currently under investigation in patients with solid tumors. To study the clinical potential of olaparib as a single-agent for the treatment of acute myeloid leukemia (AML) patients, we analyzed the in vitro sensitivity of AML cell lines and primary blasts. Clinically achievable concentrations of olaparib were able to induce cell death in the majority of primary AML case samples (88%) and tested cell lines. At these concentrations, olaparib preferentially killed leukemic blasts sparing normal lymphocytes derived from the same patient and did not substantially affect the viability of normal bone marrow and CD34-enriched peripheral blood cells obtained from healthy donors. Most primary AML analyzed were characterized by low BRCA1 mRNA level and undetectable protein expression that likely contributed to explain their sensitivity to olaparib. Noteworthy, while PARP1 over-expression was detected in blasts not responsive to olaparib, phosphorylation of the histone H2AFX (γH2AX) was associated with drug sensitivity. As to genetic features of tested cases the highest sensitivity was shown by a patient carrying a 11q23 deletion. The high sensitivity of AML blasts and the identification of biomarkers potentially able to predict response and/or resistance may foster further investigation of olaparib monotherapy for AML patients unfit to conventional chemotherapy.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_005',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 219}]}],\n",
       "   'context': 'BRCA1, PARP1 and γH2AX in acute myeloid leukemia: Role as biomarkers of response to the PARP inhibitor olaparib. Olaparib (AZD-2281, Ku-0059436) is an orally bioavailable and well-tolerated poly(ADP-ribose) polymerase (PARP) inhibitor currently under investigation in patients with solid tumors. To study the clinical potential of olaparib as a single-agent for the treatment of acute myeloid leukemia (AML) patients, we analyzed the in vitro sensitivity of AML cell lines and primary blasts. Clinically achievable concentrations of olaparib were able to induce cell death in the majority of primary AML case samples (88%) and tested cell lines. At these concentrations, olaparib preferentially killed leukemic blasts sparing normal lymphocytes derived from the same patient and did not substantially affect the viability of normal bone marrow and CD34-enriched peripheral blood cells obtained from healthy donors. Most primary AML analyzed were characterized by low BRCA1 mRNA level and undetectable protein expression that likely contributed to explain their sensitivity to olaparib. Noteworthy, while PARP1 over-expression was detected in blasts not responsive to olaparib, phosphorylation of the histone H2AFX (γH2AX) was associated with drug sensitivity. As to genetic features of tested cases the highest sensitivity was shown by a patient carrying a 11q23 deletion. The high sensitivity of AML blasts and the identification of biomarkers potentially able to predict response and/or resistance may foster further investigation of olaparib monotherapy for AML patients unfit to conventional chemotherapy.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_006',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'poly(ADP-ribose) polymerase',\n",
       "       'answer_start': 129}]}],\n",
       "   'context': 'Olaparib combined with chemotherapy for recurrent platinum-sensitive ovarian cancer: a randomised phase 2 trial. BACKGROUND: The poly(ADP-ribose) polymerase inhibitor olaparib has shown antitumour activity in patients with platinum-sensitive, recurrent, high-grade serous ovarian cancer with or without BRCA1 or BRCA2 mutations. The aim of this study was to assess the efficacy and tolerability of olaparib in combination with chemotherapy, followed by olaparib maintenance monotherapy, versus chemotherapy alone in patients with platinum-sensitive, recurrent, high-grade serous ovarian cancer. METHODS: In this randomised, open-label, phase 2 study, adult patients with platinum-sensitive, recurrent, high-grade serous ovarian cancer who had received up to three previous courses of platinum-based chemotherapy and who were progression free for at least 6 months before randomisation received either olaparib (200 mg capsules twice daily, administered orally on days 1-10 of each 21-day cycle) plus paclitaxel (175 mg/m(2), administered intravenously on day 1) and carboplatin (area under the curve [AUC] 4 mg/mL per min, according to the Calvert formula, administered intravenously on day 1), then olaparib monotherapy (400 mg capsules twice daily, given continuously) until progression (the olaparib plus chemotherapy group), or paclitaxel (175 mg/m(2) on day 1) and carboplatin (AUC 6 mg/mL per min on day 1) then no further treatment (the chemotherapy alone group). Randomisation was done by an interactive voice response system, stratified by number of previous platinum-containing regimens received and time to disease progression after the previous platinum regimen. The primary endpoint was progression-free survival according to Response Evaluation Criteria in Solid Tumors version 1.1, analysed by intention to treat. Prespecified exploratory analyses included efficacy by BRCA mutation status, assessed retrospectively. This study is registered with ClinicalTrials.gov, number NCT01081951, and has been completed. FINDINGS: Between Feb 12 and July 30, 2010, 173 patients at 43 investigational sites in 12 countries were enrolled into the study, of whom 162 were eligible and were randomly assigned to the two treatment groups (81 to the olaparib plus chemotherapy group and 81 to the chemotherapy alone group). Of these randomised patients, 156 were treated in the combination phase (81 in the olaparib plus chemotherapy group and 75 in the chemotherapy alone group) and 121 continued to the maintenance or no further treatment phase (66 in the olaparib plus chemotherapy group and 55 in the chemotherapy alone group). BRCA mutation status was known for 107 patients (either at baseline or determined retrospectively): 41 (38%) of 107 had a BRCA mutation (20 in the olaparib plus chemotherapy group and 21 in the chemotherapy alone group). Progression-free survival was significantly longer in the olaparib plus chemotherapy group (median 12.2 months [95% CI 9.7-15.0]) than in the chemotherapy alone group (median 9.6 months [95% CI 9.1-9.7) (HR 0.51 [95% CI 0.34-0.77]; p=0.0012), especially in patients with BRCA mutations (HR 0.21 [0.08-0.55]; p=0.0015). In the combination phase, adverse events that were reported at least 10% more frequently with olaparib plus chemotherapy than with chemotherapy alone were alopecia (60 [74%] of 81 vs 44 [59%] of 75), nausea (56 [69%] vs 43 [57%]), neutropenia (40 [49%] vs 29 [39%]), diarrhoea (34 [42%] vs 20 [27%]), headache (27 [33%] vs seven [9%]), peripheral neuropathy (25 [31%] vs 14 [19%]), and dyspepsia (21 [26%] vs 9 [12%]); most were of mild-to-moderate intensity. The most common grade 3 or higher adverse events during the combination phase were neutropenia (in 35 [43%] of 81 patients in the olaparib plus chemotherapy group vs 26 [35%] of 75 in the chemotherapy alone group) and anaemia (seven [9%] vs five [7%]). Serious adverse events were reported in 12 (15%) of 81 patients in the olaparib plus chemotherapy group and 16 of 75 (21%) patients in the chemotherapy alone group. INTERPRETATION: Olaparib plus paclitaxel and carboplatin followed by maintenance monotherapy significantly improved progression-free survival versus paclitaxel plus carboplatin alone, with the greatest clinical benefit in BRCA-mutated patients, and had an acceptable and manageable tolerability profile. FUNDING: AstraZeneca.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_007',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 379}]}],\n",
       "   'context': 'Phase II study of olaparib in patients with refractory Ewing sarcoma following failure of standard chemotherapy. BACKGROUND: Preclinical studies have documented antitumor activity of PARP inhibition both in vitro and in vivo, against Ewing sarcoma cells. This study aimed to translate that observation into a clinical trial to assess the efficacy and tolerability of olaparib, a PARP inhibitor, in patients with advanced Ewing sarcoma (EWS) progressing after prior chemotherapy. METHODS: In this nonrandomized phase II trial, adult participants with radiographically measureable metastatic EWS received olaparib tablets, 400 mg orally twice daily, until disease progression or drug intolerance. Tumor measurements were determined by CT or MRI at 6 and 12 weeks after starting olaparib administration, and then every 8 weeks thereafter. Tumor response determinations were made according to RECIST 1.1, and adverse event determinations were made according to CTCAE, version 4.0. A total of 22 participants were planned to be enrolled using a conventional 2-step phase II study design. If no objective responses were observed after 12 participants had been followed for at least 3 months, further accrual would be stopped. RESULTS: 12 participants were enrolled, and all were evaluable. There were no objective responses (PR/CR), 4 SD (duration 10.9, 11.4, 11.9, and 17.9 wks), and 8 PD as best response. Of the SD, 2 had minor responses (-9% and -11.7% by RECIST 1.1). The median time to disease progression was 5.7 weeks. Further enrollment was therefore discontinued. No significant or unexpected toxicities were observed with olaparib, with only a single case each of grade 3 anemia and grade 3 thrombocytopenia observed. CONCLUSIONS: This study is the first report of a prospective phase II trial to evaluate the safety and efficacy of a PARP inhibitor in patients with advanced Ewing sarcoma after failure of standard chemotherapy. Olaparib administration was safe and well tolerated when administered to this small heavily pre-treated cohort at the 400 mg BID dose, although the median duration of dosing was for only 5.7 weeks. No significant responses or durable disease control was seen, and the short average interval to disease progression underscores the aggressiveness of this disease. Other studies to combine cytotoxic chemotherapy with PARP inhibition in EWS are actively ongoing. TRIAL REGISTRATION: ClinicalTrials.gov Identifier: NCT01583543.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_008',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 884}]}],\n",
       "   'context': 'New therapeutic perspectives in CCDC6 deficient lung cancer cells. Non-small cell lung cancer (NSCLC) is the main cause of cancer-related death worldwide and new therapeutic strategies are urgently needed. In this study, we have characterized a panel of NSC lung cancer cell lines for the expression of coiled-coil-domain containing 6 (CCDC6), a tumor suppressor gene involved in apoptosis and DNA damage response. We show that low CCDC6 protein levels are associated with a weak response to DNA damage and a low number of Rad51 positive foci. Moreover, CCDC6 deficient lung cancer cells show defects in DNA repair via homologous recombination. In accordance with its role in the DNA damage response, CCDC6 attenuation confers resistance to cisplatinum, the current treatment of choice for NSCLC, but sensitizes the cells to olaparib, a small molecule inhibitor of the repair enzymes PARP1/2. Remarkably, the combination of the two drugs is more effective than each agent individually, as demonstrated by a combination index <1. Finally, CCDC6 is expressed at low levels in about 30% of the NSCL tumors we analyzed by TMA immunostaining. The weak CCDC6 protein staining is significatively correlated with the presence of lymph node metastasis (p < 0.02) and negatively correlated to the disease free survival (p < 0.01) and the overall survival (p < 0.05). Collectively, the data indicate that CCDC6 levels provide valuable insight for OS. CCDC6 could represent a predictive biomarker of resistance to conventional single mode therapy and yield insight on tumor sensitivity to PARP inhibitors in NSCLC.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_009',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 1540}]}],\n",
       "   'context': 'Impact of DNA repair pathways on the cytotoxicity of piperlongumine in chicken DT40 cell-lines. Piperlongumine is a naturally-occurring small molecule with various biological activities. Recent studies demonstrate that piperlongumine selectively kills various types of transformed cells with minimal toxicity to non-transformed cells by inducing a high level of reactive oxygen species (ROS). ROS generates various types of DNA lesions, including base modifications and single strand breaks. In order to examine the contribution of ROS-induced DNA damage to the cytotoxicity by piperlongumine, various DNA repair-deficient chicken DT40 cell-lines with a single DNA repair gene deletion were tested for cellular sensitivity to piperlongumine. The results showed that cell lines defective in homologous recombination (HR) display hyper-sensitivity to piperlongumine, while other cell lines with a deficiency in non-homologous end joining (NHEJ), base excision repair (BER), nucleotide excision repair (NER), Fanconi anemia (FA) pathway, or translesion DNA synthesis (TLS) polymerases, show no sensitivity to piperlongumine. The results strongly implicate that double strand breaks (DSBs) generated by piperlongumine are major cytotoxic DNA lesions. Furthermore, a deletion of 53BP1 or Ku70 in the BRCA1-deficient cell line restored cellular resistance to piperlongumine. This strongly supports the idea that piperlongumine induces DSB- mediated cell death. Interestingly, piperlongumine makes the wild type DT40 cell line hypersensitive to a PARP-inhibitor, Olaparib. The results implicate that piperlongumine inhibits HR. Further analysis with cell-based HR assay and the kinetic study of Rad51 foci formation confirmed that piperlongumine suppresses HR activity. Altogether, we revealed novel mechanisms of piperlongumine-induced cytotoxicity.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_010',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'poly(ADP-ribose) polymerase',\n",
       "       'answer_start': 171}]}],\n",
       "   'context': 'Combination cediranib and olaparib versus olaparib alone for women with recurrent platinum-sensitive ovarian cancer: a randomised phase 2 study. BACKGROUND: Olaparib is a poly(ADP-ribose) polymerase inhibitor and cediranib is an anti-angiogenic agent with activity against VEGF receptor (VEGFR) 1, VEGFR2, and VEGFR3. Both oral agents have antitumour activity in women with recurrent ovarian cancer, and their combination was active and had manageable toxicities in a phase 1 trial. We investigated whether this combination could improve progression-free survival (PFS) compared with olaparib monotherapy in women with recurrent platinum-sensitive ovarian cancer. METHODS: In our randomised, open-label, phase 2 study, we recruited women (aged > 18 years) who had measurable platinum-sensitive, relapsed, high-grade serous or endometrioid ovarian, fallopian tube, or primary peritoneal cancer, or those with deleterious germline BRCA1/2 mutations from nine participating US academic medical centres. We randomly allocated participants (1:1) according to permuted blocks, stratified by germline BRCA status and previous anti-angiogenic therapy, to receive olaparib capsules 400 mg twice daily or the combination at the recommended phase 2 dose of cediranib 30 mg daily and olaparib capsules 200 mg twice daily. The primary endpoint was progression-free survival analysed in the intention-to-treat population. The phase 2 trial is no longer accruing patients. An interim analysis was conducted in November, 2013, after 50% of expected events had occurred and efficacy results were unmasked. The primary analysis was performed on March 31, 2014, after 47 events (66% of those expected). The trial is registered with ClinicalTrials.gov, number NCT01116648. FINDINGS: Between Oct 26, 2011, and June 3, 2013, we randomly allocated 46 women to receive olaparib alone and 44 to receive the combination of olaparib and cediranib. Median PFS was 17·7 months (95% CI 14·7-not reached) for the women treated with cediranib plus olaparib compared with 9·0 months (95% CI 5·7-16·5) for those treated with olaparib monotherapy (hazard ratio 0·42, 95% CI 0·23-0·76; p=0·005). Grade 3 and 4 adverse events were more common with combination therapy than with monotherapy, including fatigue (12 patients in the cediranib plus olaparib group vs five patients in the olaparib monotherapy group), diarrhoea (ten vs none), and hypertension (18 vs none). INTERPRETATION: Cediranib plus olaparib seems to improve PFS in women with recurrent platinum-sensitive high-grade serous or endometrioid ovarian cancer, and warrants study in a phase 3 trial. The side-effect profile suggests such investigations should include assessments of quality of life and patient-reported outcomes to understand the effects of a continuing oral regimen with that of intermittent chemotherapy. FUNDING: American Recovery and Reinvestment Act grant from the National Institutes of Health (NIH) (3 U01 CA062490-16S2); Intramural Program of the Center for Cancer Research; and the Division of Cancer Treatment and Diagnosis, National Cancer Institute, NIH.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_011',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 1144}]}],\n",
       "   'context': 'Poly(ADP-ribose) polymerase 1 (PARP1) overexpression in human breast cancer stem cells and resistance to olaparib. BACKGROUND: Breast cancer stem cells (BCSCs) have been recognized as playing a major role in various aspects of breast cancer biology. To identify specific biomarkers of BCSCs, we have performed comparative proteomics of BCSC-enriched and mature cancer cell populations from the human breast cancer cell line (BCL), BrCA-MZ-01. METHODS: ALDEFLUOR assay was used to sort BCSC-enriched (ALDH+) and mature cancer (ALDH-) cell populations. Total proteins were extracted from both fractions and subjected to 2-Dimensional Difference In-Gel Electrophoresis (2-D DIGE). Differentially-expressed spots were excised and proteins were gel-extracted, digested and identified using MALDI-TOF MS. RESULTS: 2-D DIGE identified poly(ADP-ribose) polymerase 1 (PARP1) as overexpressed in ALDH+ cells from BrCA-MZ-01. This observation was confirmed by western blot and extended to four additional human BCLs. ALDH+ cells from BRCA1-mutated HCC1937, which had the highest level of PARP1 overexpression, displayed resistance to olaparib, a specific PARP1 inhibitor. CONCLUSION: An unbiased proteomic approach identified PARP1 as upregulated in ALDH+, BCSC-enriched cells from various human BCLs, which may contribute to clinical resistance to PARP inhibitors.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_012',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 0}]}],\n",
       "   'context': 'PARP inhibitor olaparib increases the oncolytic activity of dl922-947 in in vitro and in vivo model of anaplastic thyroid carcinoma. PARP inhibitors are mostly effective as anticancer drugs in association with DNA damaging agents. We have previously shown that the oncolytic adenovirus dl922-947 induces extensive DNA damage, therefore we hypothesized a synergistic antitumoral effect of the PARP inhibitor olaparib in association with dl922-947. Anaplastic thyroid carcinoma was chosen as model since it is a particularly aggressive tumor and, because of its localized growth, it is suitable for intratumoral treatment with oncolytic viruses. Here, we show that dl922-947 infection induces PARP activation, and we confirm in vitro and in vivo that PARP inhibition increases dl922-947 replication and oncolytic activity. In vitro, the combination with olaparib exacerbates the appearance of cell death markers, such as Annexin V positivity, caspase 3 cleavage, cytochrome C release and propidium iodide permeability. In vivo, we also observed a better viral distribution upon PARP inhibition. Changes in CD31 levels suggest a direct effect of olaparib on tumor vascularization and on the viral distribution within the tumor mass. The observation that PARP inhibition enhances the effects of dl922-947 is highly promising not only for the treatment of anaplastic thyroid carcinoma but, in general, for the treatment of other tumors that could benefit from the use of oncolytic viruses.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_013',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 659}]}],\n",
       "   'context': 'Poly (ADP-ribose) polymerase inhibition enhances trastuzumab antitumour activity in HER2 overexpressing breast cancer. AIM: Poly (ADP-ribose) polymerase (PARP) inhibitors have shown promising results in Breast Cancer (BRCA) deficient breast cancer, but not in molecularly unselected patient populations. Two lines of research in this field are needed: the identification of novel subsets of patients that could potentially benefit from PARP inhibitors and the discovery of suitable targeted therapies for combination strategies. METHODS: We tested PARP inhibition, alone or combined with the anti-HER2 antibody trastuzumab on HER2+ breast cancer. We used two PARP inhibitors in clinical development, olaparib and rucaparib, as well as genetic downmodulation of PARP-1 for in vitro studies. DNA damage was studied by the formation of γH2AX foci and comet assay. Finally, the in vivo anti-tumour effect of olaparib and trastuzumab was examined in nude mice subcutaneously implanted with BT474 cells. RESULTS: In a panel of four HER2 overexpressing breast cancer cell lines, both olaparib and rucaparib significantly decreased cell growth and enhanced anti-tumour effects of trastuzumab. Cells exposed to olaparib and trastuzumab had greater DNA damage than cells exposed to each agent alone. Mechanistic exploratory assays showed that trastuzumab downmodulated the homologous recombination protein proliferating cell nuclear antigen (PCNA). Combination treatment in the BT474 xenograft model resulted in enhanced growth inhibition, reduced tumour cell proliferation, and increased DNA damage and apoptosis. CONCLUSION: Taken together, our results show that PARP inhibition has antitumour effects and increases trastuzumab activity in HER2 overexpressing breast cancer. These findings make this novel combination a promising strategy for clinical development.'},\n",
       "  {'qas': [{'id': '54d649843706e89528000009_014',\n",
       "     'question': 'What is the target of the drug Olaparib?',\n",
       "     'answers': [{'text': 'PARP', 'answer_start': 741}]}],\n",
       "   'context': 'Synergistic loss of prostate cancer cell viability by coinhibition of HDAC and PARP. UNLABELLED: Tumors with BRCA germline mutations are defective in repairing DNA double-strand breaks (DSB) through homologous recombination (HR) pathways, making them sensitive to PARP inhibitors (PARPi). However, BRCA germline mutations are rare in prostate cancer limiting the ability to therapeutically target these pathways. This study investigates whether histone deacetylase (HDAC) inhibitors (HDACi), reported to modulate DSB repair pathways in sporadic cancers, can downregulate DSB repair pathways and sensitize prostate cancer cells to PARPi. Prostate cancer cells cotreated with the HDAC inhibitor, suberoylanilide hydroxamic acid (SAHA) and the PARPi, olaparib, demonstrated a synergistic decrease in cell viability compared with single-agent treatment (combination index < 0.9), whereas normal prostatic cells did not. Similarly, clonogenicity was significantly decreased after cotreatment. Flow cytometric cell-cycle analysis and Annexin-V staining revealed significant apoptosis upon treatment with SAHA+olaparib. This coincided with increased DNA damage observed by immunofluorescence microscopy analysis of γH2AX foci, a marker of DSBs. In addition, immunoblot analysis showed a significant and persistent increase in nuclear γH2AX levels. Both SAHA and olaparib downregulated the expression of HR-related proteins, BRCA1 and RAD51, whereas SAHA + olaparib had an additive effect on RAD51. Silencing RAD51 sensitized prostate cancer cells to SAHA and olaparib alone. Collectively, cotreatment with HDACi and PARPi downregulated HR-related protein expression and concomitantly increased DNA damage, resulting in prostate cancer cell death. IMPLICATIONS: These findings provide a strong rationale for supporting the use of combined HDAC and PARP inhibition in treating advanced prostate cancer.'},\n",
       "  {'qas': [{'id': '5335c7f2d6d3ac6a34000051_001',\n",
       "     'question': 'Inhibition of which transporter is the mechanism of action of drug Canagliflozin?',\n",
       "     'answers': [{'text': 'Sodium glucose co-transporter 2',\n",
       "       'answer_start': 0}]}],\n",
       "   'context': 'Sodium glucose co-transporter 2 (SGLT2) inhibition with canagliflozin in type 2 diabetes mellitus. The sodium glucose cotransporter 2 (SGLT2) is expressed primarily in the kidneys and is involved in the reabsorption of filtered glucose in the renal tubule. Clinical trials of SGLT2 inhibitors in patients with type 2 diabetes mellitus demonstrate a significant clinical effect in decreasing serum glucose, hemoglobin A1C, body weight, systolic blood pressure, improving β-cell function, and minimizing the risk of hypoglycemia. This report reviews the potentially beneficial effects of SGLT2 inhibitors in type 2 diabetes mellitus, specifically focusing on canagliflozin, the only SGLT2 inhibitor approved for use in the United States.'},\n",
       "  {'qas': [{'id': '5335c7f2d6d3ac6a34000051_002',\n",
       "     'question': 'Inhibition of which transporter is the mechanism of action of drug Canagliflozin?',\n",
       "     'answers': [{'text': 'sodium glucose co-transporter 2',\n",
       "       'answer_start': 263}]}],\n",
       "   'context': 'Rationale, design, and baseline characteristics of the Canagliflozin Cardiovascular Assessment Study (CANVAS)--a randomized placebo-controlled trial. Sodium glucose co-transporter 2 inhibition is a novel mode of treatment for type 2 diabetes mellitus (T2DM). The sodium glucose co-transporter 2 inhibitor canagliflozin lowered blood glucose, blood pressure, and body weight, with increased risk of urogenital infections in Phase 2 studies. Effects on macrovascular complications of diabetes remain to be determined. CANVAS is a double-blind, placebo-controlled trial designed to evaluate the effects of canagliflozin on the risk of cardiovascular disease and to assess safety and tolerability in patients with inadequately controlled T2DM and increased cardiovascular risk. The first of 2 planned phases randomized 4,330 individuals to placebo, canagliflozin 100 or 300 mg (1:1:1) with planned follow-up of about 2 years to substantiate potential cardiovascular protection by assessing key biomarkers and to achieve initial safety objectives. By the end of mid-September 2012, a total of 7174 patient-years of follow-up were accrued. Mean baseline age was 62 years, duration of diabetes 13 years; hemoglobin A1c 8.2%, fasting plasma glucose 9.3 mmol/L, and body mass index 32 kg/m(2). Of the participants, 34% are female and 57% had a history of atherosclerotic vascular disease. Participants will be followed up to achieve primary safety and tolerability objectives and to investigate secondary outcomes. The planned second phase will not be undertaken. CANVAS will define the effects of canagliflozin on biomarkers and provide data on cardiovascular safety against established regulatory parameters.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_001',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 255}]}],\n",
       "   'context': 'KIAA1530 protein is recruited by Cockayne syndrome complementation group protein A (CSA) to participate in transcription-coupled repair (TCR). Transcription-coupled repair (TCR) is the major pathway involved in the removal of UV-induced photolesions from the transcribed strand of active genes. Two Cockayne syndrome (CS) complementation group proteins, CSA and CSB, are important for TCR repair. The molecular mechanisms by which CS proteins regulate TCR remain elusive. Here, we report the characterization of KIAA1530, an evolutionarily conserved protein that participates in this pathway through its interaction with CSA and the TFIIH complex. We found that UV irradiation led to the recruitment of KIAA1530 onto chromatin in a CSA-dependent manner. Cells lacking KIAA1530 were highly sensitive to UV irradiation and displayed deficiency in TCR. In addition, KIAA1530 depletion abrogated stability of the CSB protein following UV irradiation. More excitingly, we found that a unique CSA mutant (W361C), which was previously identified in a patient with UV(s)S syndrome, showed defective KIAA1530 binding and resulted in a failure of recruiting KIAA1530 and stabilizing CSB after UV treatment. Together, our data not only reveal that KIAA1530 is an important player in TCR but also lead to a better understanding of the molecular mechanism underlying UV(s)S syndrome.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_002',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 681}]}],\n",
       "   'context': 'Multiple roles of ubiquitination in the control of nucleotide excision repair. Nucleotide excision repair (NER) is a remarkably versatile DNA repair system, essential for maintenance of genomic stability. Hereditary alterations in NER enzymes can result in increased cancer propensity, but also in developmental, neurodegenerative, and progeroid syndromes. NER can be operationally divided in three subtypes, which share many common steps: global genomic repair (GGR) operates on the whole genome level, transcription domain-associated repair (DAR) is a concentration of NER activity within transcription factories, and transcription-coupled repair (TCR) provides faster repair of the transcribed strand of active genes. Interestingly, ubiquitination plays an important role in all three classes of NER, as well as in associated phenomena, such as damage signalling by histone ubiquitination, and degradation of stalled RNA polymerase II when repair does not occur in a timely manner.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_003',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 363}]}],\n",
       "   'context': 'Strand-specific PCR of UV radiation-damaged genomic DNA revealed an essential role of DNA-PKcs in the transcription-coupled repair. BACKGROUND: In eukaryotic cells, there are two sub-pathways of nucleotide excision repair (NER), the global genome (gg) NER and the transcription-coupled repair (TCR). TCR can preferentially remove the bulky DNA lesions located at the transcribed strand of a transcriptional active gene more rapidly than those at the untranscribed strand or overall genomic DNA. This strand-specific repair in a suitable restriction fragment is usually determined by alkaline gel electrophoresis followed by Southern blotting transfer and hybridization with an indirect end-labeled single-stranded probe. Here we describe a new method of TCR assay based on strand-specific-PCR (SS-PCR). Using this method, we have investigated the role of DNA-dependent protein kinase catalytic subunit (DNA-PKcs), a member of the phosphatidylinositol 3-kinase-related protein kinases (PIKK) family, in the TCR pathway of UV-induced DNA damage. RESULTS: Although depletion of DNA-PKcs sensitized HeLa cells to UV radiation, it did not affect the ggNER efficiency of UV-induced cyclobutane pyrimidine dimers (CPD) damage. We postulated that DNA-PKcs may involve in the TCR process. To test this hypothesis, we have firstly developed a novel method of TCR assay based on the strand-specific PCR technology with a set of smart primers, which allows the strand-specific amplification of a restricted gene fragment of UV radiation-damaged genomic DNA in mammalian cells. Using this new method, we confirmed that siRNA-mediated downregulation of Cockayne syndrome B resulted in a deficiency of TCR of the UV-damaged dihydrofolate reductase (DHFR) gene. In addition, DMSO-induced silencing of the c-myc gene led to a decreased TCR efficiency of UV radiation-damaged c-myc gene in HL60 cells. On the basis of the above methodology verification, we found that the depletion of DNA-PKcs mediated by siRNA significantly decreased the TCR capacity of repairing the UV-induced CPDs damage in DHFR gene in HeLa cells, indicating that DNA-PKcs may also be involved in the TCR pathway of DNA damage repair. By means of immunoprecipitation and MALDI-TOF-Mass spectrometric analysis, we have revealed the interaction of DNA-PKcs and cyclin T2, which is a subunit of the human transcription elongation factor (P-TEFb). While the P-TEFb complex can phosphorylate the serine 2 of the carboxyl-terminal domain (CTD) of RNA polymerase II and promote transcription elongation. CONCLUSION: A new method of TCR assay was developed based the strand-specific-PCR (SS-PCR). Our data suggest that DNA-PKcs plays a role in the TCR pathway of UV-damaged DNA. One possible mechanistic hypothesis is that DNA-PKcs may function through associating with CyclinT2/CDK9 (P-TEFb) to modulate the activity of RNA Pol II, which has already been identified as a key molecule recognizing and initializing TCR.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_004',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 181}]}],\n",
       "   'context': 'Methods to study transcription-coupled repair in chromatin. Transcription-coupled repair (TCR) is a sub-pathway of nucleotide excision repair that allows for the enhanced repair of the transcribed strand of active genes. A classical method to study DNA repair in vivo consists in the molecular analysis of UV-induced DNA damages at specific loci. Cells are irradiated with a defined dose of UV light leading to the formation of DNA lesions and incubated in the dark to allow repair. About 90% of the photoproducts consist of cyclobutane pyrimidine dimers, which can be cleaved by the DNA nicking activity of the T4 endonuclease V (T4endoV) repair enzyme. Strand-specific repair in a suitable restriction fragment is determined by alkaline gel electrophoresis followed by Southern blot transfer and indirect end-labeling using a single-stranded probe. Recent approaches have assessed the role of transcription factors in TCR by analyzing RNA polymerase II occupancy on a damaged template by chromatin immunoprecipitation (ChIP). Cells are treated with formaldehyde in vivo to cross-link proteins to DNA and enrichment of a protein of interest is done by subsequent immunoprecipitation. Upon reversal of the protein-DNA cross-links, the amount of coprecipitated DNA fragments can be detected by quantitative PCR. To perform ChIP on UV-damaged templates, we included an in vitro photoreactivation step prior to PCR analysis to ensure that all precipitated DNA fragments serve as substrates for the PCR reaction. Here, we provide a detailed protocol for both the DNA repair analysis and the ChIP approaches to study TCR in chromatin.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_005',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 213}]}],\n",
       "   'context': 'Yeast Elc1 plays an important role in global genomic repair but not in transcription coupled repair. Transcription coupled repair (TCR) is a nucleotide excision repair (NER) pathway that is dedicated to repair in the transcribed strand of an active gene. The genome overall NER is called global genomic repair (GGR). Elc1, the yeast homolog of the mammalian elongation factor elongin C, has been shown to be a component of a ubiquitin ligase complex that contains Rad7 and Rad16, two factors that are specifically required for GGR. Elc1 has also been suggested to be present in another ubiquitin ligase complex that lacks Rad7 and Rad16 and is involved in UV-induced ubiquitylation and subsequent degradation of RNA polymerase II. Here we show that elc1 deletion increases UV sensitivity of TCR-deficient cells but does not affect the UV sensitivity of otherwise wild type and GGR-deficient cells. Cells deleted for elc1 show normal NER in the transcribed strand of an active gene but have no detectable NER in the non-transcribed strand. Elc1 does not affect UV-induced mutagenesis when TCR is operative, but plays an important role in preventing the mutagenesis if TCR is defective. Furthermore, the levels of Rad7 and Rad16 proteins are not significantly decreased in elc1 cells, and overexpression of Rad7 and Rad16 individually or simultaneously in elc1 cells does not restore repair in the non-transcribed strand of an active gene. Our results suggest that Elc1 has no function in TCR but plays an important role in GGR. Furthermore, the role of Elc1 in GGR may not be subsidiary to that of Rad7 and Rad16.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_006',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 400}]}],\n",
       "   'context': 'Base excision repair and nucleotide excision repair contribute to the removal of N-methylpurines from active genes. Many different cellular pathways have evolved to protect the genome from the deleterious effects of DNA damage that result from exposure to chemical and physical agents. Among these is a process called transcription-coupled repair (TCR) that catalyzes the removal of DNA lesions from the transcribed strand of expressed genes, often resulting in a preferential bias of damage clearance from this strand relative to its non-transcribed counterpart. Lesions subject to this type of repair include cyclobutane pyrimidine dimers that are normally repaired by nucleotide excision repair (NER) and thymine glycols (TGs) that are removed primarily by base excision repair (BER). While the mechanism underlying TCR is not completely clear, it is known that its facilitation requires proteins used by other repair pathways like NER. It is also believed that the signal for TCR is the stalled RNA polymerase that results when DNA damage prevents its translocation during transcription elongation. While there is a clear role for some NER proteins in TCR, the involvement of BER proteins is less clear. To explore this further, we studied the removal of 7-methylguanine (7MeG) and 3-methyladenine (3MeA) from the dihydrofolate reductase (dhfr) gene of murine cell lines that vary in their repair phenotypes. 7MeG and 3MeA constitute the two principal N-methylpurines formed in DNA following exposure to methylating agents. In mammalian cells, alkyladenine DNA alkyladenine glycosylase (Aag) is the major enzyme required for the repair of these lesions via BER, and their removal from the total genome is quite rapid. There is no observable TCR of these lesions in specific genes in DNA repair proficient cells; however, it is possible that the rapid repair of these adducts by BER masks any TCR. The repair of 3MeA and 7MeG was examined in cells lacking Aag, NER, or both Aag and NER to determine if rapid overall repair masks TCR. The results show that both 3MeA and 7MeG are removed without strand bias from the dhfr gene of BER deficient (Aag deficient) and NER deficient murine cell lines. Furthermore, repair of 3MeA in this region is highly dependent on Aag, but repair of 7MeG is equally efficient in the repair proficient, BER deficient, and NER deficient cell lines. Strikingly, in the absence of both BER and NER, neither 7MeG nor 3MeA is repaired. These results demonstrate that NER, but not TCR, contributes to the repair of 7MeG, and to a lesser extent 3MeA.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_007',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 1652}]}],\n",
       "   'context': 'UV-enhanced reactivation of a UV-damaged reporter gene suggests transcription-coupled repair is UV-inducible in human cells. The genetic disorders xeroderma pigmentosum (XP) and Cockayne syndrome (CS) exhibit deficiencies in the repair of UV-induced DNA damage. CS fibroblasts retain proficient nucleotide excision repair (NER) of inactive (or bulk) DNA, but are deficient in the transcription-coupled repair (TCR) of active genes. In contrast, XP complementation group C (XP-C) fibroblasts retain proficient TCR, but are deficient in bulk DNA repair. The remaining NER-deficient XP groups exhibit deficiencies in both repair pathways. Ad5HCMVsp1lacZ is a recombinant adenovirus vector that is unable to replicate in human fibroblasts, but can efficiently infect and express the beta-galactosidase reporter gene in these cells. We have examined the host cell reactivation (HCR) of beta-galactosidase activity for UV-irradiated Ad5HCMVsp1lacZ in non-irradiated and UV-irradiated normal, XP-B, XP-C, XP-D, XP-F, XP-G, CS-A and CS-B fibroblasts. HCR of beta-galactosidase activity for UV-irradiated Ad5HCMVsp1lacZ was reduced in non-irradiated cells from each of the repair-deficient groups examined (including XP-C) relative to that in non-irradiated normal cells. Prior irradiation of cells with low UV fluences resulted in an enhancement of HCR for normal and XP-C strains, but not for the remaining XP and CS strains. HCR of the UV-damaged reporter gene in UV-irradiated XP and CS strains was similar to measurements of TCR reported previously for these cells. These results suggest that UV treatment results in an induced repair of UV-damaged DNA in the transcribed strand of an active gene in XP-C and normal cells through an enhancement of TCR or a mechanism which involves the TCR pathway.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_008',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 363}]}],\n",
       "   'context': 'Strand-specific PCR of UV radiation-damaged genomic DNA revealed an essential role of DNA-PKcs in the transcription-coupled repair. BACKGROUND: In eukaryotic cells, there are two sub-pathways of nucleotide excision repair (NER), the global genome (gg) NER and the transcription-coupled repair (TCR). TCR can preferentially remove the bulky DNA lesions located at the transcribed strand of a transcriptional active gene more rapidly than those at the untranscribed strand or overall genomic DNA. This strand-specific repair in a suitable restriction fragment is usually determined by alkaline gel electrophoresis followed by Southern blotting transfer and hybridization with an indirect end-labeled single-stranded probe. Here we describe a new method of TCR assay based on strand-specific-PCR (SS-PCR). Using this method, we have investigated the role of DNA-dependent protein kinase catalytic subunit (DNA-PKcs), a member of the phosphatidylinositol 3-kinase-related protein kinases (PIKK) family, in the TCR pathway of UV-induced DNA damage. RESULTS: Although depletion of DNA-PKcs sensitized HeLa cells to UV radiation, it did not affect the ggNER efficiency of UV-induced cyclobutane pyrimidine dimers (CPD) damage. We postulated that DNA-PKcs may involve in the TCR process. To test this hypothesis, we have firstly developed a novel method of TCR assay based on the strand-specific PCR technology with a set of smart primers, which allows the strand-specific amplification of a restricted gene fragment of UV radiation-damaged genomic DNA in mammalian cells. Using this new method, we confirmed that siRNA-mediated downregulation of Cockayne syndrome B resulted in a deficiency of TCR of the UV-damaged dihydrofolate reductase (DHFR) gene. In addition, DMSO-induced silencing of the c-myc gene led to a decreased TCR efficiency of UV radiation-damaged c-myc gene in HL60 cells. On the basis of the above methodology verification, we found that the depletion of DNA-PKcs mediated by siRNA significantly decreased the TCR capacity of repairing the UV-induced CPDs damage in DHFR gene in HeLa cells, indicating that DNA-PKcs may also be involved in the TCR pathway of DNA damage repair. By means of immunoprecipitation and MALDI-TOF-Mass spectrometric analysis, we have revealed the interaction of DNA-PKcs and cyclin T2, which is a subunit of the human transcription elongation factor (P-TEFb). While the P-TEFb complex can phosphorylate the serine 2 of the carboxyl-terminal domain (CTD) of RNA polymerase II and promote transcription elongation. CONCLUSION: A new method of TCR assay was developed based the strand-specific-PCR (SS-PCR). Our data suggest that DNA-PKcs plays a role in the TCR pathway of UV-damaged DNA. One possible mechanistic hypothesis is that DNA-PKcs may function through associating with CyclinT2/CDK9 (P-TEFb) to modulate the activity of RNA Pol II, which has already been identified as a key molecule recognizing and initializing TCR.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_009',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 494}]}],\n",
       "   'context': 'Transcription domain-associated repair in human cells. Nucleotide excision repair (NER), which is arguably the most versatile DNA repair system, is strongly attenuated in human cells of the monocytic lineage when they differentiate into macrophages. Within active genes, however, both DNA strands continue to be proficiently repaired. The proficient repair of the nontranscribed strand cannot be explained by the dedicated subpathway of transcription-coupled repair (TCR), which is targeted to the transcribed strand in expressed genes. We now report that the previously termed differentiation-associated repair (DAR) depends upon transcription, but not simply upon RNA polymerase II (RNAPII) encountering a lesion: proficient repair of both DNA strands can occur in a part of a gene that the polymerase never reaches, and even if the translocation of RNAPII is blocked with transcription inhibitors. This suggests that DAR may be a subset of global NER, restricted to the subnuclear compartments or chromatin domains within which transcription occurs. Downregulation of selected NER genes with small interfering RNA has confirmed that DAR relies upon the same genes as global genome repair, rather than upon TCR-specific genes. Our findings support the general view that the genomic domains within which transcription is active are more accessible than the bulk of the genome to the recognition and repair of lesions through the global pathway and that TCR is superimposed upon that pathway of NER.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_010',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 211}]}],\n",
       "   'context': 'RAD26, the functional S. cerevisiae homolog of the Cockayne syndrome B gene ERCC6. Transcription-coupled repair (TCR) is a universal sub-pathway of the nucleotide excision repair (NER) system that is limited to the transcribed strand of active structural genes. It accomplishes the preferential elimination of transcription-blocking DNA lesions and permits rapid resumption of the vital process of transcription. A defect in TCR is responsible for the rare hereditary disorder Cockayne syndrome (CS). Recently we found that mutations in the ERCC6 repair gene, encoding a putative helicase, underly the repair defect of CS complementation group B. Here we report the cloning and characterization of the Saccharomyces cerevisiae homolog of CSB/ERCC6, which we designate RAD26. A rad26 disruption mutant appears viable and grows normally, indicating that the gene does not have an essential function. In analogy with CS, preferential repair of UV-induced cyclobutane pyrimidine dimers in the transcribed strand of the active RBP2 gene is severely impaired. Surprisingly, in contrast to the human CS mutant, yeast RAD26 disruption does not induce any UV-, cisPt- or X-ray sensitivity, explaining why it was not isolated as a mutant before. Recovery of growth after UV exposure was somewhat delayed in rad26. These findings suggest that TCR in lower eukaryotes is not very important for cell survival and that the global genome repair pathway of NER is the major determinant of cellular resistance to genotoxicity.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_011',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 643}]}],\n",
       "   'context': 'Nucleotide excision repair and its interplay with transcription. Nucleotide excision repair (NER) is a multistep process capable to remove a variety of DNA distorting lesions from prokaryotic and eukaryotic genomes. In eukaryotic cells, the process requires more than 30 proteins to perform the different steps, i.e. recognition of DNA damage, single strand incisions and excision of the lesion-containing DNA fragment and DNA repair synthesis/ligation. NER can operate via two subpathways: global genome repair (GGR) and a specialized pathway coupled to active transcription (transcription-coupled repair, TCR) and directed to DNA lesions in the transcribed strand of active genes. Both in vivo as well as in cultured cells the fast removal of transcription blocking lesions by TCR is crucial to escape from lethal effects of inhibited transcription inhibition The most delicate step in NER is the recognition of the DNA lesions in their different chromatin context and the mechanism of damage recognition in GGR and TCR is principally different and requires specific proteins. In GGR, the XPC-HR23B is essential for the formation of the incision complex. In TCR the Cockayne syndrome (CS) gene products are key players in the recognition of a stalled RNA polymerase the presumed signaling structure for repair of transcribed strands. In this study, we show that the extent of recovery of UV-inhibited transcription and TCR strictly depends on the amount of CSB protein as well as the amount of DNA damage present in the cell. This indicates that the ratio between DNA damage frequency and CSB protein concentration in the cell is rather critical for acute cellular response, i.e. recovery of inhibited transcription upon DNA damage infliction, and hence cellular survival.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_012',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 457}]}],\n",
       "   'context': 'Non-transcribed strand repair revealed in quiescent cells. Stem cells, one of the progenitors of cancer, exist predominately in a quiescent state. Thus, understanding the mechanisms of DNA repair and mutagenesis in such arrested cells may help unravel the complex process of tumorigenesis. Two major nucleotide excision repair (NER) pathways are known to remove bulky physical or chemical lesions from DNA. Transcription-coupled repair (TCR) acts solely on the transcribed strand of expressed genes, while global genomic repair (GGR) is responsible for the ubiquitous repair of the genome. Indirectly, it has been shown that while TCR functions in quiescent cells GGR does not. To explicitly elucidate this phenomenon, we adapted a quantitative PCR (QPCR) assay to study UV-damage repair via TCR and GGR in quiescent and proliferating cells. We present evidence that repair of untranscribed silent regions of the genome and repair of the non-transcribed strand of active genes proceeds by two discrete mechanisms in quiescent cells; rather than by GGR, which was believed to encompass both. Thus, our findings suggest the existence of an alternate NER pathway in quiescent cells. The proposed subcategories of NER are as follows: (i) TCR, responsible for maintenance of transcribed strands; (ii) GGR, responsible for ubiquitous genome repair; and (iii) non-transcribed strand repair (NTSR), predominantly responsible for the repair of the NTS in arrested cells. In quiescent cells, it is evident that TCR and NTSR function and GGR are arrested. As a consequence, mutation accumulation at temporally silent genes and incomplete or imperfect repair of transcribed genes, in quiescent stem cells, may provide a source of cancer causing mutations.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_013',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 590}]}],\n",
       "   'context': 'In UV-irradiated Saccharomyces cerevisiae, overexpression of Swi2/Snf2 family member Rad26 increases transcription-coupled repair and repair of the non-transcribed strand. Nucleotide excision repair (NER) in eukaryotes is a pathway conserved from yeast to humans that removes many bulky chemical adducts and UV-induced photoproducts from DNA in a relatively error-free manner. In addition to the recognition and excision of DNA damage throughout the genome (GGR), there exists a mechanism, transcription-coupled nucleotide excision repair (TCR), for recognizing some types of DNA damage in the transcribed strand of genes in Escherichia coli, yeast and mammalian cells. An obstacle in the repair of the transcribed strand of active genes is the RNA polymerase complex stalled at sites of DNA damage. The stalled RNA polymerase complex may then mediate recruitment of repair proteins to damage in the transcribed strand. Proteins enabling TCR are the Cockayne syndrome B (CSB) protein in humans and its yeast homologue Rad26. Both CSB and Rad26 belong to the Swi2/Snf2 family of DNA-dependent ATPases, which change DNA accessibility to proteins by altering chromatin structure. To address how Rad26 functions in yeast repair, we used the genetic approach of overexpressing Rad26 and examined phenotypic changes, i.e. changes in NER. We found that repair of both the transcribed and the non-transcribed strands is increased. In addition, overexpression of Rad26 partially bypasses the requirement for Rad7 in GGR, specifically in the repair of non-transcribed sequences. As TCR takes place in very localized regions of DNA (i.e. within genes) in wild-type cells, we propose that overexpression of recombinant Rad26 increases accessibility of the damaged DNA in chromatin for interaction with repair proteins.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_014',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 494}]}],\n",
       "   'context': 'Transcription domain-associated repair in human cells. Nucleotide excision repair (NER), which is arguably the most versatile DNA repair system, is strongly attenuated in human cells of the monocytic lineage when they differentiate into macrophages. Within active genes, however, both DNA strands continue to be proficiently repaired. The proficient repair of the nontranscribed strand cannot be explained by the dedicated subpathway of transcription-coupled repair (TCR), which is targeted to the transcribed strand in expressed genes. We now report that the previously termed differentiation-associated repair (DAR) depends upon transcription, but not simply upon RNA polymerase II (RNAPII) encountering a lesion: proficient repair of both DNA strands can occur in a part of a gene that the polymerase never reaches, and even if the translocation of RNAPII is blocked with transcription inhibitors. This suggests that DAR may be a subset of global NER, restricted to the subnuclear compartments or chromatin domains within which transcription occurs. Downregulation of selected NER genes with small interfering RNA has confirmed that DAR relies upon the same genes as global genome repair, rather than upon TCR-specific genes. Our findings support the general view that the genomic domains within which transcription is active are more accessible than the bulk of the genome to the recognition and repair of lesions through the global pathway and that TCR is superimposed upon that pathway of NER.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_015',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 363}]}],\n",
       "   'context': 'Strand-specific PCR of UV radiation-damaged genomic DNA revealed an essential role of DNA-PKcs in the transcription-coupled repair. BACKGROUND: In eukaryotic cells, there are two sub-pathways of nucleotide excision repair (NER), the global genome (gg) NER and the transcription-coupled repair (TCR). TCR can preferentially remove the bulky DNA lesions located at the transcribed strand of a transcriptional active gene more rapidly than those at the untranscribed strand or overall genomic DNA. This strand-specific repair in a suitable restriction fragment is usually determined by alkaline gel electrophoresis followed by Southern blotting transfer and hybridization with an indirect end-labeled single-stranded probe. Here we describe a new method of TCR assay based on strand-specific-PCR (SS-PCR). Using this method, we have investigated the role of DNA-dependent protein kinase catalytic subunit (DNA-PKcs), a member of the phosphatidylinositol 3-kinase-related protein kinases (PIKK) family, in the TCR pathway of UV-induced DNA damage. RESULTS: Although depletion of DNA-PKcs sensitized HeLa cells to UV radiation, it did not affect the ggNER efficiency of UV-induced cyclobutane pyrimidine dimers (CPD) damage. We postulated that DNA-PKcs may involve in the TCR process. To test this hypothesis, we have firstly developed a novel method of TCR assay based on the strand-specific PCR technology with a set of smart primers, which allows the strand-specific amplification of a restricted gene fragment of UV radiation-damaged genomic DNA in mammalian cells. Using this new method, we confirmed that siRNA-mediated downregulation of Cockayne syndrome B resulted in a deficiency of TCR of the UV-damaged dihydrofolate reductase (DHFR) gene. In addition, DMSO-induced silencing of the c-myc gene led to a decreased TCR efficiency of UV radiation-damaged c-myc gene in HL60 cells. On the basis of the above methodology verification, we found that the depletion of DNA-PKcs mediated by siRNA significantly decreased the TCR capacity of repairing the UV-induced CPDs damage in DHFR gene in HeLa cells, indicating that DNA-PKcs may also be involved in the TCR pathway of DNA damage repair. By means of immunoprecipitation and MALDI-TOF-Mass spectrometric analysis, we have revealed the interaction of DNA-PKcs and cyclin T2, which is a subunit of the human transcription elongation factor (P-TEFb). While the P-TEFb complex can phosphorylate the serine 2 of the carboxyl-terminal domain (CTD) of RNA polymerase II and promote transcription elongation. CONCLUSION: A new method of TCR assay was developed based the strand-specific-PCR (SS-PCR). Our data suggest that DNA-PKcs plays a role in the TCR pathway of UV-damaged DNA. One possible mechanistic hypothesis is that DNA-PKcs may function through associating with CyclinT2/CDK9 (P-TEFb) to modulate the activity of RNA Pol II, which has already been identified as a key molecule recognizing and initializing TCR.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_016',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 173}]}],\n",
       "   'context': 'Transcription-coupled DNA repair in prokaryotes. Transcription-coupled repair (TCR) is a subpathway of nucleotide excision repair (NER) that acts specifically on lesions in the transcribed strand of expressed genes. First reported in mammalian cells, TCR was then documented in Escherichia coli. In this organism, an RNA polymerase arrested at a lesion is displaced by the transcription repair coupling factor, Mfd. This protein recruits the NER lesion-recognition factor UvrA, and then dissociates from the DNA. UvrA binds UvrB, and the assembled UvrAB* complex initiates repair. In mutants lacking active Mfd, TCR is absent. A gene transcribed by the bacteriophage T7 RNA polymerase in E. coli also requires Mfd for TCR. The CSB protein (missing or defective in cells of patients with Cockayne syndrome, complementation group B) is essential for TCR in humans. CSB and its homologs in higher eukaryotes are likely functional equivalents of Mfd.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_017',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 181}]}],\n",
       "   'context': 'Blockage of RNA polymerase II at a cyclobutane pyrimidine dimer and 6-4 photoproduct. The blockage of transcription elongation by RNA polymerase II (pol II) at a DNA damage site on the transcribed strand triggers a transcription-coupled DNA repair (TCR), which rapidly removes DNA damage on the transcribed strand of the expressed gene and allows the resumption of transcription. To analyze the effect of UV-induced DNA damage on transcription elongation, an in vitro transcription elongation system using pol II and oligo(dC)-tailed templates containing a cyclobutane pyrimidine dimer (CPD) or 6-4 photoproduct (6-4PP) at a specific site was employed. The results showed that pol II incorporated nucleotides opposite the CPD and 6-4PP and then stalled. Pol II formed a stable ternary complex consisting of pol II, the DNA damage template, and the nascent transcript. Furthermore, atomic force microscopy imaging revealed that pol II stalled at the damaged region. These findings may provide the basis for analysis of the initiation step of TCR.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_018',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 291}]}],\n",
       "   'context': 'Blockage of RNA polymerase II at a cyclobutane pyrimidine dimer and 6-4 photoproduct. The blockage of transcription elongation by RNA polymerase II (pol II) at a DNA damage site on the transcribed strand triggers a transcription-coupled DNA repair (TCR), which rapidly removes DNA damage on the transcribed strand of the expressed gene and allows the resumption of transcription. To analyze the effect of UV-induced DNA damage on transcription elongation, an in vitro transcription elongation system using pol II and oligo(dC)-tailed templates containing a cyclobutane pyrimidine dimer (CPD) or 6-4 photoproduct (6-4PP) at a specific site was employed. The results showed that pol II incorporated nucleotides opposite the CPD and 6-4PP and then stalled. Pol II formed a stable ternary complex consisting of pol II, the DNA damage template, and the nascent transcript. Furthermore, atomic force microscopy imaging revealed that pol II stalled at the damaged region. These findings may provide the basis for analysis of the initiation step of TCR.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_019',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 286}]}],\n",
       "   'context': 'Deletion of the CSB homolog, RAD26, yields Spt(-) strains with proficient transcription-coupled repair. It has been previously shown that disruption of RAD26 in yeast strain W303-1B results in a strain that is deficient in transcription-coupled repair (TCR), the preferential repair of the transcribed strand of an expressed gene over the non-transcribed strand and the rest of the genome. RAD26 encodes a protein that is homologous to Cockayne syndrome group B protein (CSB) and is a member of the SWI2/SNF2 family of DNA-dependent ATPases involved in chromatin remodeling. Like the rad26 mutant, cells from Cockayne syndrome patients are defective in TCR. We examined the role of Rad26 in TCR by disrupting RAD26 in two repair-proficient laboratory strains and, remarkably, observed no effect upon TCR. Our results indicate that disruption of RAD26 alone is insufficient to impair TCR. Thus, W303-1B must already possess a mutation that, together with disruption of RAD26, causes a deficiency in TCR. We suggest that other genes are mutated in Cockayne syndrome cells that contribute to the deficiency in TCR. Surprisingly, deletion of RAD26 results in expression of genes that are repressed by flanking transposon delta elements, an Spt(-) phenotype. The delta elements appear to perturb local chromatin structure. Expression of genes flanked by delta elements in rad26Delta mutants is consistent with a role for Rad26 in chromatin remodeling.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_020',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 494}]}],\n",
       "   'context': 'Transcription domain-associated repair in human cells. Nucleotide excision repair (NER), which is arguably the most versatile DNA repair system, is strongly attenuated in human cells of the monocytic lineage when they differentiate into macrophages. Within active genes, however, both DNA strands continue to be proficiently repaired. The proficient repair of the nontranscribed strand cannot be explained by the dedicated subpathway of transcription-coupled repair (TCR), which is targeted to the transcribed strand in expressed genes. We now report that the previously termed differentiation-associated repair (DAR) depends upon transcription, but not simply upon RNA polymerase II (RNAPII) encountering a lesion: proficient repair of both DNA strands can occur in a part of a gene that the polymerase never reaches, and even if the translocation of RNAPII is blocked with transcription inhibitors. This suggests that DAR may be a subset of global NER, restricted to the subnuclear compartments or chromatin domains within which transcription occurs. Downregulation of selected NER genes with small interfering RNA has confirmed that DAR relies upon the same genes as global genome repair, rather than upon TCR-specific genes. Our findings support the general view that the genomic domains within which transcription is active are more accessible than the bulk of the genome to the recognition and repair of lesions through the global pathway and that TCR is superimposed upon that pathway of NER.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_021',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 643}]}],\n",
       "   'context': 'Nucleotide excision repair and its interplay with transcription. Nucleotide excision repair (NER) is a multistep process capable to remove a variety of DNA distorting lesions from prokaryotic and eukaryotic genomes. In eukaryotic cells, the process requires more than 30 proteins to perform the different steps, i.e. recognition of DNA damage, single strand incisions and excision of the lesion-containing DNA fragment and DNA repair synthesis/ligation. NER can operate via two subpathways: global genome repair (GGR) and a specialized pathway coupled to active transcription (transcription-coupled repair, TCR) and directed to DNA lesions in the transcribed strand of active genes. Both in vivo as well as in cultured cells the fast removal of transcription blocking lesions by TCR is crucial to escape from lethal effects of inhibited transcription inhibition The most delicate step in NER is the recognition of the DNA lesions in their different chromatin context and the mechanism of damage recognition in GGR and TCR is principally different and requires specific proteins. In GGR, the XPC-HR23B is essential for the formation of the incision complex. In TCR the Cockayne syndrome (CS) gene products are key players in the recognition of a stalled RNA polymerase the presumed signaling structure for repair of transcribed strands. In this study, we show that the extent of recovery of UV-inhibited transcription and TCR strictly depends on the amount of CSB protein as well as the amount of DNA damage present in the cell. This indicates that the ratio between DNA damage frequency and CSB protein concentration in the cell is rather critical for acute cellular response, i.e. recovery of inhibited transcription upon DNA damage infliction, and hence cellular survival.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_022',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 213}]}],\n",
       "   'context': 'Yeast Elc1 plays an important role in global genomic repair but not in transcription coupled repair. Transcription coupled repair (TCR) is a nucleotide excision repair (NER) pathway that is dedicated to repair in the transcribed strand of an active gene. The genome overall NER is called global genomic repair (GGR). Elc1, the yeast homolog of the mammalian elongation factor elongin C, has been shown to be a component of a ubiquitin ligase complex that contains Rad7 and Rad16, two factors that are specifically required for GGR. Elc1 has also been suggested to be present in another ubiquitin ligase complex that lacks Rad7 and Rad16 and is involved in UV-induced ubiquitylation and subsequent degradation of RNA polymerase II. Here we show that elc1 deletion increases UV sensitivity of TCR-deficient cells but does not affect the UV sensitivity of otherwise wild type and GGR-deficient cells. Cells deleted for elc1 show normal NER in the transcribed strand of an active gene but have no detectable NER in the non-transcribed strand. Elc1 does not affect UV-induced mutagenesis when TCR is operative, but plays an important role in preventing the mutagenesis if TCR is defective. Furthermore, the levels of Rad7 and Rad16 proteins are not significantly decreased in elc1 cells, and overexpression of Rad7 and Rad16 individually or simultaneously in elc1 cells does not restore repair in the non-transcribed strand of an active gene. Our results suggest that Elc1 has no function in TCR but plays an important role in GGR. Furthermore, the role of Elc1 in GGR may not be subsidiary to that of Rad7 and Rad16.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_023',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 181}]}],\n",
       "   'context': 'Transcription-coupled repair: a complex affair. Transcription-coupled repair (TCR) is generally observed as more rapid or more efficient removal of certain types of DNA damage from the transcribed strands of expressed genes compared with the nontranscribed strands. It has been clearly demonstrated to be a subpathway of nucleotide excision repair (NER) in E. coli, yeast and mammalian cells. Genetic and biochemical studies indicate that it is a highly complex process and requires the participation of the NER pathway, the RNA polymerase complex and additional factors. An early event in TCR is likely the blocking of RNA polymerase complex elongation by damage present in the transcribed strands of expressed genes. Whether TCR is involved in base excision repair pathways or the repair of common forms of oxidative damage is less clear. This review is focused on the description of possible mechanisms of TCR in E. coli and mammalian cells.'},\n",
       "  {'qas': [{'id': '5545186cbf90a13052000002_024',\n",
       "     'question': 'Which gene strand is targeted by transcription-coupled repair (TCR)?',\n",
       "     'answers': [{'text': 'the transcribed strand', 'answer_start': 590}]}],\n",
       "   'context': 'In UV-irradiated Saccharomyces cerevisiae, overexpression of Swi2/Snf2 family member Rad26 increases transcription-coupled repair and repair of the non-transcribed strand. Nucleotide excision repair (NER) in eukaryotes is a pathway conserved from yeast to humans that removes many bulky chemical adducts and UV-induced photoproducts from DNA in a relatively error-free manner. In addition to the recognition and excision of DNA damage throughout the genome (GGR), there exists a mechanism, transcription-coupled nucleotide excision repair (TCR), for recognizing some types of DNA damage in the transcribed strand of genes in Escherichia coli, yeast and mammalian cells. An obstacle in the repair of the transcribed strand of active genes is the RNA polymerase complex stalled at sites of DNA damage. The stalled RNA polymerase complex may then mediate recruitment of repair proteins to damage in the transcribed strand. Proteins enabling TCR are the Cockayne syndrome B (CSB) protein in humans and its yeast homologue Rad26. Both CSB and Rad26 belong to the Swi2/Snf2 family of DNA-dependent ATPases, which change DNA accessibility to proteins by altering chromatin structure. To address how Rad26 functions in yeast repair, we used the genetic approach of overexpressing Rad26 and examined phenotypic changes, i.e. changes in NER. We found that repair of both the transcribed and the non-transcribed strands is increased. In addition, overexpression of Rad26 partially bypasses the requirement for Rad7 in GGR, specifically in the repair of non-transcribed sequences. As TCR takes place in very localized regions of DNA (i.e. within genes) in wild-type cells, we propose that overexpression of recombinant Rad26 increases accessibility of the damaged DNA in chromatin for interaction with repair proteins.'},\n",
       "  {'qas': [{'id': '5357a6d0f1005d6b58000004_001',\n",
       "     'question': 'Which is the branch site consensus sequence in U12-dependent introns?',\n",
       "     'answers': [{'text': 'UUCCUUAAC', 'answer_start': 464}]}],\n",
       "   'context': \"Mutational analysis of the U12-dependent branch site consensus sequence. Highly conserved sequences at the 5' splice site and branch site of U12-dependent introns are important determinants for splicing by U12-dependent spliceosomes. This study investigates the in vivo splicing phenotypes of mutations in the branch site consensus sequence of the U12-dependent intron F from a human NOL1 (P120) minigene. Intron F contains a fully consensus branch site sequence (UUCCUUAAC). Mutations at each position were analyzed for their effects on U12-dependent splicing in vivo. Mutations at most positions resulted in a significant reduction of correct U12-dependent splicing. Defects observed included increased unspliced RNA levels, the activation of cryptic U2-dependent 5' and 3' splice sites, and the activation of cryptic U12-dependent branch/3' splice sites. A strong correlation was observed between the predicted thermodynamic stability of the branch site: U12 snRNA interaction and correct U12-dependent splicing. The lack of a polypyrimidine tract between the branch site and 3' splice site of U12-dependent introns and the observed reliance on base-pairing interactions for correct U12-dependent splicing emphasize the importance of RNA/RNA interactions during U12-dependent intron recognition and proper splice site selection.\"},\n",
       "  {'qas': [{'id': '5357a6d0f1005d6b58000004_002',\n",
       "     'question': 'Which is the branch site consensus sequence in U12-dependent introns?',\n",
       "     'answers': [{'text': 'UUCCUUAAC', 'answer_start': 464}]}],\n",
       "   'context': \"Mutational analysis of the U12-dependent branch site consensus sequence. Highly conserved sequences at the 5' splice site and branch site of U12-dependent introns are important determinants for splicing by U12-dependent spliceosomes. This study investigates the in vivo splicing phenotypes of mutations in the branch site consensus sequence of the U12-dependent intron F from a human NOL1 (P120) minigene. Intron F contains a fully consensus branch site sequence (UUCCUUAAC). Mutations at each position were analyzed for their effects on U12-dependent splicing in vivo. Mutations at most positions resulted in a significant reduction of correct U12-dependent splicing. Defects observed included increased unspliced RNA levels, the activation of cryptic U2-dependent 5' and 3' splice sites, and the activation of cryptic U12-dependent branch/3' splice sites. A strong correlation was observed between the predicted thermodynamic stability of the branch site: U12 snRNA interaction and correct U12-dependent splicing. The lack of a polypyrimidine tract between the branch site and 3' splice site of U12-dependent introns and the observed reliance on base-pairing interactions for correct U12-dependent splicing emphasize the importance of RNA/RNA interactions during U12-dependent intron recognition and proper splice site selection.\"},\n",
       "  {'qas': [{'id': '5357a6d0f1005d6b58000004_003',\n",
       "     'question': 'Which is the branch site consensus sequence in U12-dependent introns?',\n",
       "     'answers': [{'text': 'UUCCUUAAC', 'answer_start': 464}]}],\n",
       "   'context': \"Mutational analysis of the U12-dependent branch site consensus sequence. Highly conserved sequences at the 5' splice site and branch site of U12-dependent introns are important determinants for splicing by U12-dependent spliceosomes. This study investigates the in vivo splicing phenotypes of mutations in the branch site consensus sequence of the U12-dependent intron F from a human NOL1 (P120) minigene. Intron F contains a fully consensus branch site sequence (UUCCUUAAC). Mutations at each position were analyzed for their effects on U12-dependent splicing in vivo. Mutations at most positions resulted in a significant reduction of correct U12-dependent splicing. Defects observed included increased unspliced RNA levels, the activation of cryptic U2-dependent 5' and 3' splice sites, and the activation of cryptic U12-dependent branch/3' splice sites. A strong correlation was observed between the predicted thermodynamic stability of the branch site: U12 snRNA interaction and correct U12-dependent splicing. The lack of a polypyrimidine tract between the branch site and 3' splice site of U12-dependent introns and the observed reliance on base-pairing interactions for correct U12-dependent splicing emphasize the importance of RNA/RNA interactions during U12-dependent intron recognition and proper splice site selection.\"},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_001',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 1061}]}],\n",
       "   'context': 'Endothelial deletion of Sag/Rbx2/Roc2 E3 ubiquitin ligase causes embryonic lethality and blocks tumor angiogenesis. SAG (Sensitive to Apoptosis Gene), also known as RBX2 or ROC2, is a RING protein required for the activity of Cullin-RING ligase (CRL). Our recent study showed that Sag total knockout caused embryonic lethality at E11.5-12.5 days with associated defects in vasculogenesis. Whether Sag is required for de novo vasculogenesis in embryos and angiogenesis in tumors is totally unknown. Here, we report that Sag endothelial deletion also causes embryonic lethality at E15.5 with poor vasculogenesis. Sag deletion in primary endothelial cells (ECs) or knockdown in MS-1 ECs inhibits migration, proliferation and tube formation, with p27 accumulation being responsible for the suppression of migration and proliferation. Furthermore, Sag deletion significantly inhibits angiogenesis in an in vivo Matrigel plug assay, and tumor angiogenesis and tumorigenesis in a B16F10 melanoma model. Finally, MLN4924, an investigational small molecule inhibitor of NEDD8-activating enzyme (NAE) that inhibits CRL, suppresses in vitro migration, proliferation and tube formation, as well as in vivo angiogenesis and tumorigenesis. Taken together, our study, using both genetic and pharmaceutical approaches, demonstrates that Sag is essential for embryonic vasculogenesis and tumor angiogenesis, and provides the proof-of-concept evidence that targeting Sag E3 ubiquitin ligase may have clinical value for anti-angiogenesis therapy of human cancer.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_002',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 408}]}],\n",
       "   'context': 'Molecular and cellular effects of NEDD8-activating enzyme inhibition in myeloma. The NEDD8-activating enzyme is upstream of the 20S proteasome in the ubiquitin/proteasome pathway and catalyzes the first step in the neddylation pathway. NEDD8 modification of cullins is required for ubiquitination of cullin-ring ligases that regulate degradation of a distinct subset of proteins. The more targeted impact of NEDD8-activating enzyme on protein degradation prompted us to study MLN4924, an investigational NEDD8-activating enzyme inhibitor, in preclinical multiple myeloma models. In vitro treatment with MLN4924 led to dose-dependent decrease of viability (EC(50) = 25-150 nmol/L) in a panel of human multiple myeloma cell lines. MLN4924 was similarly active against a bortezomib-resistant ANBL-6 subline and its bortezomib-sensitive parental cells. MLN4924 had submicromolar activity (EC(50) values <500 nmol/L) against primary CD138(+) multiple myeloma patient cells and exhibited at least additive effect when combined with dexamethasone, doxorubicin, and bortezomib against MM.1S cells. The bortezomib-induced compensatory upregulation of transcripts for ubiquitin/proteasome was not observed with MLN4924 treatment, suggesting distinct functional roles of NEDD8-activating enzyme versus 20S proteasome. MLN4924 was well tolerated at doses up to 60 mg/kg 2× daily and significantly reduced tumor burden in both a subcutaneous and an orthotopic mouse model of multiple myeloma. These studies provide the framework for the clinical investigation of MLN4924 in multiple myeloma.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_003',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 504}]}],\n",
       "   'context': 'Molecular and cellular effects of NEDD8-activating enzyme inhibition in myeloma. The NEDD8-activating enzyme is upstream of the 20S proteasome in the ubiquitin/proteasome pathway and catalyzes the first step in the neddylation pathway. NEDD8 modification of cullins is required for ubiquitination of cullin-ring ligases that regulate degradation of a distinct subset of proteins. The more targeted impact of NEDD8-activating enzyme on protein degradation prompted us to study MLN4924, an investigational NEDD8-activating enzyme inhibitor, in preclinical multiple myeloma models. In vitro treatment with MLN4924 led to dose-dependent decrease of viability (EC(50) = 25-150 nmol/L) in a panel of human multiple myeloma cell lines. MLN4924 was similarly active against a bortezomib-resistant ANBL-6 subline and its bortezomib-sensitive parental cells. MLN4924 had submicromolar activity (EC(50) values <500 nmol/L) against primary CD138(+) multiple myeloma patient cells and exhibited at least additive effect when combined with dexamethasone, doxorubicin, and bortezomib against MM.1S cells. The bortezomib-induced compensatory upregulation of transcripts for ubiquitin/proteasome was not observed with MLN4924 treatment, suggesting distinct functional roles of NEDD8-activating enzyme versus 20S proteasome. MLN4924 was well tolerated at doses up to 60 mg/kg 2× daily and significantly reduced tumor burden in both a subcutaneous and an orthotopic mouse model of multiple myeloma. These studies provide the framework for the clinical investigation of MLN4924 in multiple myeloma.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_004',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 25}]}],\n",
       "   'context': \"A gatekeeper residue for NEDD8-activating enzyme inhibition by MLN4924. Inhibition of NEDD8-activating enzyme (NAE) has emerged as a highly promising approach to treat cancer through the adenosine sulfamate analog MLN4924. Here, we show that selective pressure results in HCT116 colorectal carcinoma cells with decreased MLN4924 sensitivity and identify a single-nucleotide transition that changes alanine 171 to threonine (A171T) of the NAE subunit UBA3. This reduces the enzyme's affinity for MLN4924 and ATP while increasing NEDD8 activation at physiological ATP concentrations. Expression of UBA3 A171T is sufficient to decrease MLN4924 sensitivity of naive HCT116 cells, indicating that it is a dominant suppressor of MLN4924-mediated cell death. Our data suggest that the on-target potency of MLN4924 selects for a point mutation in NAE that overcomes the molecule's inhibitory effects, allowing cancer cell survival.\"},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_005',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 62}]}],\n",
       "   'context': 'Treatment-emergent mutations in NAEβ confer resistance to the NEDD8-activating enzyme inhibitor MLN4924. MLN4924 is an investigational small-molecule inhibitor of NEDD8-activating enzyme (NAE) in clinical trials for the treatment of cancer. MLN4924 is a mechanism-based inhibitor, with enzyme inhibition occurring through the formation of a tight-binding NEDD8-MLN4924 adduct. In cell and xenograft models of cancer, we identified treatment-emergent heterozygous mutations in the adenosine triphosphate binding pocket and NEDD8-binding cleft of NAEβ as the primary mechanism of resistance to MLN4924. Biochemical analyses of NAEβ mutants revealed slower rates of adduct formation and reduced adduct affinity for the mutant enzymes. A compound with tighter binding properties was able to potently inhibit mutant enzymes in cells. These data provide rationales for patient selection and the development of next-generation NAE inhibitors designed to overcome treatment-emergent NAEβ mutations.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_006',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 58}]}],\n",
       "   'context': 'Quantifiable analysis of cellular pathway inhibition of a Nedd8-activating enzyme inhibitor, MLN4924, using AlphaScreen. Cellular effects of a Nedd8-activating enzyme (NAE) inhibitor, MLN4924, using the AlphaScreen format were explored. MLN4924 acts as a substrate-assisted inhibitor of NAE by forming a tight binding Nedd8-MLN4924 adduct. The inhibited enzyme can no longer transfer Nedd8 downstream to modify and activate the E3 cullin-RING ligases. This results in the stabilization of proteins regulated by the proteasome, leading to cell death. These studies monitored the endogenous cellular changes to NAE∼Nedd8 thioester, the formation of the Nedd8-MLN4924 adduct, and the reduction in the Cul1-Nedd8. Lysates derived from MLN4924-treated HCT116 cells showed that whereas the β-subunit of NAE remained constant, reductions of both NAE∼Nedd8 thioester and Cul1-Nedd8 levels occurred with a concomitant rise of the adduct. Moreover, the formation of the Nedd8-MLN4924 adduct was approximately stoichiometric with the concentration of NAEβ. Higher density 384-well cell-based assays illustrated the kinetics of enzyme inactivation across a wider range of MLN4924 concentrations, showing a rapid loss of NAE∼Nedd8 thioester and Cul1-Nedd8. The reduction of NAE∼Nedd8 thioester precedes the loss of Cul1-Nedd8 at twice the rate. Finally, these results clearly demonstrate the utility of the homogeneous assay for quantitative assessment of these endogenous cellular components in a 384-well plate in response to inhibition of NAE by MLN4924.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_007',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 143}]}],\n",
       "   'context': 'Quantifiable analysis of cellular pathway inhibition of a Nedd8-activating enzyme inhibitor, MLN4924, using AlphaScreen. Cellular effects of a Nedd8-activating enzyme (NAE) inhibitor, MLN4924, using the AlphaScreen format were explored. MLN4924 acts as a substrate-assisted inhibitor of NAE by forming a tight binding Nedd8-MLN4924 adduct. The inhibited enzyme can no longer transfer Nedd8 downstream to modify and activate the E3 cullin-RING ligases. This results in the stabilization of proteins regulated by the proteasome, leading to cell death. These studies monitored the endogenous cellular changes to NAE∼Nedd8 thioester, the formation of the Nedd8-MLN4924 adduct, and the reduction in the Cul1-Nedd8. Lysates derived from MLN4924-treated HCT116 cells showed that whereas the β-subunit of NAE remained constant, reductions of both NAE∼Nedd8 thioester and Cul1-Nedd8 levels occurred with a concomitant rise of the adduct. Moreover, the formation of the Nedd8-MLN4924 adduct was approximately stoichiometric with the concentration of NAEβ. Higher density 384-well cell-based assays illustrated the kinetics of enzyme inactivation across a wider range of MLN4924 concentrations, showing a rapid loss of NAE∼Nedd8 thioester and Cul1-Nedd8. The reduction of NAE∼Nedd8 thioester precedes the loss of Cul1-Nedd8 at twice the rate. Finally, these results clearly demonstrate the utility of the homogeneous assay for quantitative assessment of these endogenous cellular components in a 384-well plate in response to inhibition of NAE by MLN4924.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_008',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 86}]}],\n",
       "   'context': 'Quantitative proteomic analysis of cellular protein modulation upon inhibition of the NEDD8-activating enzyme by MLN4924. Cullin-RING ubiquitin ligases (CRLs) are responsible for the ubiquitination of many cellular proteins, thereby targeting them for proteasomal degradation. In most cases the substrates of the CRLs have not been identified, although many of those that are known have cancer relevance. MLN4924, an investigational small molecule that is a potent and selective inhibitor of the Nedd8-activating enzyme (NAE), is currently being explored in Phase I clinical trials. Inhibition of Nedd8-activating enzyme by MLN4924 prevents the conjugation of cullin proteins with NEDD8, resulting in inactivation of the entire family of CRLs. We have performed stable isotope labeling with amino acids in cell culture analysis of A375 melanoma cells treated with MLN4924 to identify new CRL substrates, confidently identifying and quantitating 5122-6012 proteins per time point. Proteins such as MLX, EID1, KLF5, ORC6L, MAGEA6, MORF4L2, MRFAP1, MORF4L1, and TAX1BP1 are rapidly stabilized by MLN4924, suggesting that they are novel CRL substrates. Proteins up-regulated at later times were also identified and siRNA against their corresponding genes were used to evaluate their influence on MLN4924-induced cell death. Thirty-eight proteins were identified as being particularly important for the cytotoxicity of MLN4924. Strikingly, these proteins had roles in cell cycle, DNA damage repair, and ubiquitin transfer. Therefore, the combination of RNAi with stable isotope labeling with amino acids in cell culture provides a paradigm for understanding the mechanism of action of novel agents affecting the ubiquitin proteasome system and a path to identifying mechanistic biomarkers.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_009',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 55}]}],\n",
       "   'context': 'Inactivation of the Cullin (CUL)-RING E3 ligase by the NEDD8-activating enzyme inhibitor MLN4924 triggers protective autophagy in cancer cells. The multiunit Cullin (CUL)-RING E3 ligase (CRL) controls diverse biological processes by targeting a mass of substrates for ubiquitination and degradation, whereas its dysfunction causes carcinogenesis. Post-translational neddylation of CUL, a process triggered by the NEDD8-activating enzyme E1 subunit 1 (NAE1), is required for CRL activation. Recently, MLN4924 was discovered via a high-throughput screen as a specific NAE1 inhibitor and first-in-class anticancer drug. By blocking CUL neddylation, MLN4924 inactivates CRL and causes the accumulation of CRL substrates that trigger cell cycle arrest, senescence and/or apoptosis to suppress the growth of cancer cells in vitro and in vivo. Recently, we found that MLN4924 also triggers protective autophagy in response to CRL inactivation. MLN4924-induced autophagy is attributed partially to the inhibition of mechanistic target of rapamycin (also known as mammalian target of rapamycin, MTOR) activity by the accumulation of the MTOR inhibitory protein DEPTOR, as well as reactive oxygen species (ROS)-induced stress. Moreover, the blockage of autophagy response enhances apoptosis in MLN4924-treated cells. Together, our findings not only reveal autophagy as a novel cellular response to CRL inactivation by MLN4924, but also provide a piece of proof-of-concept evidence for the combination of MLN4924 with autophagy inhibitors to enhance therapeutic efficacy.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_010',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 4}]}],\n",
       "   'context': 'The Nedd8-activating enzyme inhibitor MLN4924 thwarts microenvironment-driven NF-κB activation and induces apoptosis in chronic lymphocytic leukemia B cells. BACKGROUND: Stromal-mediated signaling enhances NF-κB pathway activity in chronic lymphocytic leukemia (CLL) B cells, leading to cell survival and chemoresistance. Ubiquitination of IκBα may partially account for constitutive activation of NF-κB. MLN4924 is an investigational agent that inhibits the Nedd8-activating enzyme, thereby neutralizing Cullin-RING ubiquitin ligases and preventing degradation of their substrates. EXPERIMENTAL DESIGN: We conducted a preclinical assessment of MLN4924 in CLL. Primary CLL cells were cocultured in vitro with CD40L-expressing stroma to mimic the prosurvival conditions present in lymphoid tissue. The effect of MLN4924 on CLL cell apoptosis, NF-κB pathway activity, Bcl-2 family members, and cell cycle was assessed by flow cytometry, Western blotting, PCR, and immunocytochemistry. RESULTS: CD40L-expressing stroma protected CLL cells from spontaneous apoptosis and induced resistance to multiple drugs, accompanied by NF-κB activation and Bim repression. Treatment with MLN4924 induced CLL cell apoptosis and circumvented stroma-mediated resistance. This was accompanied by accumulation of phospho-IκBα, decreased nuclear translocation of p65 and p52 leading to inhibition of both the canonical and noncanonical NF-κB pathways, and reduced transcription of their target genes, notably chemokines. MLN4924 promoted induction of Bim and Noxa in the CLL cells leading to rebalancing of Bcl-2 family members toward the proapoptotic BH3-only proteins. siRNA-mediated knockdown of Bim or Noxa decreased sensitivity to MLN4924. MLN4924 enhanced the antitumor activity of the inhibitors of B-cell receptor (BCR)-associated kinases. CONCLUSIONS: MLN4924 disrupts NF-κB activation and induces Bim expression in CLL cells, thereby preventing stroma-mediated resistance. Our data provide rationale for further evaluation of MLN4924 in CLL.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_011',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 597}]}],\n",
       "   'context': 'Quantitative proteomic analysis of cellular protein modulation upon inhibition of the NEDD8-activating enzyme by MLN4924. Cullin-RING ubiquitin ligases (CRLs) are responsible for the ubiquitination of many cellular proteins, thereby targeting them for proteasomal degradation. In most cases the substrates of the CRLs have not been identified, although many of those that are known have cancer relevance. MLN4924, an investigational small molecule that is a potent and selective inhibitor of the Nedd8-activating enzyme (NAE), is currently being explored in Phase I clinical trials. Inhibition of Nedd8-activating enzyme by MLN4924 prevents the conjugation of cullin proteins with NEDD8, resulting in inactivation of the entire family of CRLs. We have performed stable isotope labeling with amino acids in cell culture analysis of A375 melanoma cells treated with MLN4924 to identify new CRL substrates, confidently identifying and quantitating 5122-6012 proteins per time point. Proteins such as MLX, EID1, KLF5, ORC6L, MAGEA6, MORF4L2, MRFAP1, MORF4L1, and TAX1BP1 are rapidly stabilized by MLN4924, suggesting that they are novel CRL substrates. Proteins up-regulated at later times were also identified and siRNA against their corresponding genes were used to evaluate their influence on MLN4924-induced cell death. Thirty-eight proteins were identified as being particularly important for the cytotoxicity of MLN4924. Strikingly, these proteins had roles in cell cycle, DNA damage repair, and ubiquitin transfer. Therefore, the combination of RNAi with stable isotope labeling with amino acids in cell culture provides a paradigm for understanding the mechanism of action of novel agents affecting the ubiquitin proteasome system and a path to identifying mechanistic biomarkers.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_012',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 235}]}],\n",
       "   'context': 'NEDD8-targeting drug MLN4924 elicits DNA rereplication by stabilizing Cdt1 in S phase, triggering checkpoint activation, apoptosis, and senescence in cancer cells. MLN4924 is a first-in-class experimental cancer drug that inhibits the NEDD8-activating enzyme, thereby inhibiting cullin-RING E3 ubiquitin ligases and stabilizing many cullin substrates. The mechanism by which MLN4924 inhibits cancer cell proliferation has not been defined, although it is accompanied by DNA rereplication and attendant DNA damage. Here we show that stabilization of the DNA replication factor Cdt1, a substrate of cullins 1 and 4, is critical for MLN4924 to trigger DNA rereplication and inhibit cell proliferation. Even only 1 hour of exposure to MLN4924, which was sufficient to elevate Cdt1 for 4-5 hours, was found to be sufficient to induce DNA rereplication and to activate apoptosis and senescence pathways. Cells in S phase were most susceptible, suggesting that MLN4924 will be most toxic on highly proliferating cancers. Although MLN4924-induced cell senescence seems to be dependent on induction of p53 and its downstream effector p21(Waf1), we found that p53(-/-) and p21(-/-) cells were even more susceptible than wild-type cells to MLN4924. Our results suggested that apoptosis, not senescence, might be more important for the antiproliferative effect of MLN4924. Furthermore, our findings show that transient exposure to this new investigational drug should be useful for controlling p53-negative cancer cells, which often pose significant clinical challenge.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_013',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 213}]}],\n",
       "   'context': 'A first-in-class NAE inhibitor, MLN4924, blocks lentiviral infection in myeloid cells by disrupting neddylation-dependent Vpx-mediated SAMHD1 degradation. MLN4924 is a first-in-class cancer drug that inhibits the Nedd8-activating enzyme (NAE). Herein, we report that MLN4924 inhibits Vpx/Vpr-induced SAMHD1 degradation by inhibiting the neddylation of E3 ubiquitin ligase and blocks macaque simian immunodeficiency virus (SIVmac) replication in myeloid cells. SAMHD1 is required for MLN4924-mediated SIVmac inhibition. Our findings indicate the potential efficacy of inhibiting neddylation as an antiretroviral strategy and identify the readily available anticancer drug MLN4924 as a candidate agent for that purpose.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_014',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 459}]}],\n",
       "   'context': 'The Nedd8-activating enzyme inhibitor MLN4924 thwarts microenvironment-driven NF-κB activation and induces apoptosis in chronic lymphocytic leukemia B cells. BACKGROUND: Stromal-mediated signaling enhances NF-κB pathway activity in chronic lymphocytic leukemia (CLL) B cells, leading to cell survival and chemoresistance. Ubiquitination of IκBα may partially account for constitutive activation of NF-κB. MLN4924 is an investigational agent that inhibits the Nedd8-activating enzyme, thereby neutralizing Cullin-RING ubiquitin ligases and preventing degradation of their substrates. EXPERIMENTAL DESIGN: We conducted a preclinical assessment of MLN4924 in CLL. Primary CLL cells were cocultured in vitro with CD40L-expressing stroma to mimic the prosurvival conditions present in lymphoid tissue. The effect of MLN4924 on CLL cell apoptosis, NF-κB pathway activity, Bcl-2 family members, and cell cycle was assessed by flow cytometry, Western blotting, PCR, and immunocytochemistry. RESULTS: CD40L-expressing stroma protected CLL cells from spontaneous apoptosis and induced resistance to multiple drugs, accompanied by NF-κB activation and Bim repression. Treatment with MLN4924 induced CLL cell apoptosis and circumvented stroma-mediated resistance. This was accompanied by accumulation of phospho-IκBα, decreased nuclear translocation of p65 and p52 leading to inhibition of both the canonical and noncanonical NF-κB pathways, and reduced transcription of their target genes, notably chemokines. MLN4924 promoted induction of Bim and Noxa in the CLL cells leading to rebalancing of Bcl-2 family members toward the proapoptotic BH3-only proteins. siRNA-mediated knockdown of Bim or Noxa decreased sensitivity to MLN4924. MLN4924 enhanced the antitumor activity of the inhibitors of B-cell receptor (BCR)-associated kinases. CONCLUSIONS: MLN4924 disrupts NF-κB activation and induces Bim expression in CLL cells, thereby preventing stroma-mediated resistance. Our data provide rationale for further evaluation of MLN4924 in CLL.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_015',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 684}]}],\n",
       "   'context': 'Multimodal imaging with (18)F-FDG PET and Cerenkov luminescence imaging after MLN4924 treatment in a human lymphoma xenograft model. UNLABELLED: Cerenkov luminescence imaging (CLI) is an emerging imaging technique that combines aspects of both optical and nuclear imaging fields. The ability to fully evaluate the correlation and sensitivity of CLI to PET is critical to progress this technique further for use in high-throughput screening of pharmaceutical compounds. To achieve this milestone, it must first be established that CLI data correlate to PET data in an in vivo preclinical antitumor study. We used MLN4924, a phase 2 oncology therapeutic, which targets and inhibits the NEDD8-activating enzyme pathway involved in the ubiquitin-proteasome system. We compared the efficacious effects of MLN4924 using PET and Cerenkov luminescence image values in the same animals. METHODS: Imaging of (18)F-FDG uptake was performed at 5 time points after drug treatment in the subcutaneously implanted diffuse large B-cell lymphoma tumor line OCI-Ly10. Data were acquired with both modalities on the same day, with a 15-min delay between CLI and PET. PET data analysis was performed using percentage injected dose per cubic centimeter of tissue (%ID/cm(3)), average standardized uptake values, and total glycolytic volume. CLI measurements were radiance, radiance per injected dose (radiance/ID), and total radiant volume. RESULTS: A strong correlation was found between PET total glycolytic volume and CLI total radiant volume (r(2) = 0.99) and various PET and CLI analysis methods, with strong correlations found between PET %ID/cm(3) and CLI radiance (r(2) = 0.83) and CLI radiance/ID (r(2) = 0.82). MLN4924 demonstrated a significant reduction in tumor volume after treatment (volume ratio of treated vs. control, 0.114 at day 29). CONCLUSION: The PET and CLI data presented confirm the correlation and dynamic sensitivity of this new imaging modality. CLI provides a preclinical alternative to expensive PET instrumentation. Future high-throughput studies should provide for quicker turnaround and higher cost-to-return benefits in the drug discovery process.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_016',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 235}]}],\n",
       "   'context': 'NEDD8-targeting drug MLN4924 elicits DNA rereplication by stabilizing Cdt1 in S phase, triggering checkpoint activation, apoptosis, and senescence in cancer cells. MLN4924 is a first-in-class experimental cancer drug that inhibits the NEDD8-activating enzyme, thereby inhibiting cullin-RING E3 ubiquitin ligases and stabilizing many cullin substrates. The mechanism by which MLN4924 inhibits cancer cell proliferation has not been defined, although it is accompanied by DNA rereplication and attendant DNA damage. Here we show that stabilization of the DNA replication factor Cdt1, a substrate of cullins 1 and 4, is critical for MLN4924 to trigger DNA rereplication and inhibit cell proliferation. Even only 1 hour of exposure to MLN4924, which was sufficient to elevate Cdt1 for 4-5 hours, was found to be sufficient to induce DNA rereplication and to activate apoptosis and senescence pathways. Cells in S phase were most susceptible, suggesting that MLN4924 will be most toxic on highly proliferating cancers. Although MLN4924-induced cell senescence seems to be dependent on induction of p53 and its downstream effector p21(Waf1), we found that p53(-/-) and p21(-/-) cells were even more susceptible than wild-type cells to MLN4924. Our results suggested that apoptosis, not senescence, might be more important for the antiproliferative effect of MLN4924. Furthermore, our findings show that transient exposure to this new investigational drug should be useful for controlling p53-negative cancer cells, which often pose significant clinical challenge.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_017',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 4}]}],\n",
       "   'context': 'The NEDD8-activating enzyme inhibitor, MLN4924, cooperates with TRAIL to augment apoptosis through facilitating c-FLIP degradation in head and neck cancer cells. TNF-related apoptosis-inducing ligand (TRAIL) is a tumor-selective cytokine with potential anticancer activity and is currently under clinical testing. Head and neck squamous cell carcinoma (HNSCC), like other cancer types, exhibits varied sensitivity to TRAIL. MLN4924 is a newly developed investigational small molecule inhibitor of NEDD8-activating enzyme with potent anticancer activity. This study reveals a novel function of MLN4924 in synergizing with TRAIL to induce apoptosis in HNSCC cells. MLN4924 alone effectively inhibited the growth of HNSCC cells and induced apoptosis. When combined with TRAIL, synergistic effects on decreasing the survival and inducing apoptosis of HNSCC cells occurred. MLN4924 decreased c-FLIP levels without modulating death receptor 4 and death receptor 5 expression. Enforced expression of c-FLIP substantially attenuated MLN4924/TRAIL-induced apoptosis. Thus c-FLIP reduction plays an important role in mediating MLN4924/TRAIL-induced apoptosis. Moreover, MLN4924 decreased c-FLIP stability, increased c-FLIP ubiquitination, and facilitated c-FLIP degradation, suggesting that MLN4924 decreases c-FLIP levels through promoting its degradation. MLN4924 activated c-jun-NH(2)-kinase (JNK) signaling, evidenced by increased levels of phospho-c-Jun in MLN4924-treated cells. Chemical inhibition of JNK activation not only prevented MLN4924-induced c-FLIP reduction, but also inhibited MLN4924/TRAIL-induced apoptosis, suggesting that JNK activation mediates c-FLIP downregulation and subsequent enhancement of TRAIL-induced apoptosis by MLN4924. Because knockdown of NEDD8 failed to activate JNK signaling and downregulate c-FLIP, it is likely that MLN4924 reduces c-FLIP levels and enhances TRAIL-induced apoptosis independent of NEDD8 inhibition.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_018',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 39}]}],\n",
       "   'context': \"Initial testing of the investigational NEDD8-activating enzyme inhibitor MLN4924 by the pediatric preclinical testing program. BACKGROUND: MLN4924 is an investigational first-in-class small molecule inhibitor of NEDD8-activating enzyme (NAE). NAE is an essential component of the NEDD8 conjugation pathway, controlling the activity of a subset of ubiquitin-proteasome system (UPS) E3 ligases, multiprotein complexes that transfer ubiquitin molecules to substrate proteins. PROCEDURES: MLN4924 was tested against the PPTP in vitro panel using 96-hour exposure time at concentrations ranging from 1.0 nM to 10 µM. It was tested in vivo at a dose of 100 mg/kg [66 mg/kg for the acute lymphoblastic leukemia (ALL) xenografts] administered orally twice daily × 5 days. Treatment duration was 3 weeks. RESULTS: The median relative IC(50) for MLN4924 against the PPTP cell lines was 143 nM, (range: 15-678 nM) with that for the Ewing panel being significantly lower (31 nM). MLN4924 induced significant differences in EFS distribution compared to control in 20 of 34 (59%) evaluable solid tumor xenografts. MLN4924 induced intermediate activity (EFS T/C values >2) in 9 of the 33 evaluable xenografts (27%), including 4 of 4 glioblastoma xenografts, 2 of 3 Wilm's tumor xenografts, 2 of 5 rhabdomyosarcoma xenografts, and 1 of 4 neuroblastoma xenografts. For the ALL panel, 5 of 8 evaluable xenografts showed intermediate activity for the EFS T/C measure. MLN4924 did not induce objective responses in the PPTP solid tumor or ALL panels. CONCLUSIONS: MLN4924 showed potent activity in vitro and in vivo showed tumor growth inhibitory activity against a subset of the PPTP solid tumor and ALL xenografts.\"},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_019',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 11}]}],\n",
       "   'context': 'MLN4924, a NEDD8-activating enzyme inhibitor, is active in diffuse large B-cell lymphoma models: rationale for treatment of NF-{kappa}B-dependent lymphoma. MLN4924 is a potent and selective small molecule NEDD8-activating enzyme (NAE) inhibitor. In most cancer cells tested, inhibition of NAE leads to induction of DNA rereplication, resulting in DNA damage and cell death. However, in preclinical models of activated B cell-like (ABC) diffuse large B-cell lymphoma (DLBCL), we show that MLN4924 induces an alternative mechanism of action. Treatment of ABC DLBCL cells with MLN4924 resulted in rapid accumulation of pIkappaBalpha, decrease in nuclear p65 content, reduction of nuclear factor-kappaB (NF-kappaB) transcriptional activity, and G(1) arrest, ultimately resulting in apoptosis induction, events consistent with potent NF-kappaB pathway inhibition. Treatment of germinal-center B cell-like (GCB) DLBCL cells resulted in an increase in cellular Cdt-1 and accumulation of cells in S-phase, consistent with cells undergoing DNA rereplication. In vivo administration of MLN4924 to mice bearing human xenograft tumors of ABC- and GCB-DLBCL blocked NAE pathway biomarkers and resulted in complete tumor growth inhibition. In primary human tumor models of ABC-DLBCL, MLN4924 treatment resulted in NF-kappaB pathway inhibition accompanied by tumor regressions. This work describes a novel mechanism of targeted NF-kappaB pathway modulation in DLBCL and provides strong rationale for clinical development of MLN4924 against NF-kappaB-dependent lymphomas.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_020',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 213}]}],\n",
       "   'context': 'A first-in-class NAE inhibitor, MLN4924, blocks lentiviral infection in myeloid cells by disrupting neddylation-dependent Vpx-mediated SAMHD1 degradation. MLN4924 is a first-in-class cancer drug that inhibits the Nedd8-activating enzyme (NAE). Herein, we report that MLN4924 inhibits Vpx/Vpr-induced SAMHD1 degradation by inhibiting the neddylation of E3 ubiquitin ligase and blocks macaque simian immunodeficiency virus (SIVmac) replication in myeloid cells. SAMHD1 is required for MLN4924-mediated SIVmac inhibition. Our findings indicate the potential efficacy of inhibiting neddylation as an antiretroviral strategy and identify the readily available anticancer drug MLN4924 as a candidate agent for that purpose.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_021',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'nedd8-activating enzyme', 'answer_start': 928}]}],\n",
       "   'context': 'Inhibition of CUL4A Neddylation causes a reversible block to SAMHD1-mediated restriction of HIV-1. The deoxynucleoside triphosphohydrolase SAMHD1 restricts retroviral replication in myeloid cells. Human immunodeficiency virus type 2 (HIV-2) and a simian immunodeficiency virus from rhesus macaques (SIVmac) encode Vpx, a virion-packaged accessory protein that counteracts SAMHD1 by inducing its degradation. SAMHD1 is thought to work by depleting the pool of intracellular deoxynucleoside triphosphates but has also been reported to have exonuclease activity that could allow it to degrade the viral genomic RNA or viral reverse-transcribed DNA. To induce the degradation of SAMHD1, Vpx co-opts the cullin4a-based E3 ubiquitin ligase, CRL4. E3 ubiquitin ligases are regulated by the covalent attachment of the ubiquitin-like protein Nedd8 to the cullin subunit. Neddylation can be prevented by MLN4924, a drug that inhibits the nedd8-activating enzyme. We report that MLN4924 inhibits the neddylation of CRL4, blocking Vpx-induced degradation of SAMHD1 and maintaining the restriction. Removal of the drug several hours postinfection released the block. Similarly, Vpx-containing virus-like particles and deoxynucleosides added to the cells more than 24 h postinfection released the SAMHD1-mediated block. Taken together, these findings support deoxynucleoside triphosphate pool depletion as the primary mechanism of SAMHD1 restriction and argue against a nucleolytic mechanism, which would not be reversible.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_022',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 235}]}],\n",
       "   'context': 'NEDD8-targeting drug MLN4924 elicits DNA rereplication by stabilizing Cdt1 in S phase, triggering checkpoint activation, apoptosis, and senescence in cancer cells. MLN4924 is a first-in-class experimental cancer drug that inhibits the NEDD8-activating enzyme, thereby inhibiting cullin-RING E3 ubiquitin ligases and stabilizing many cullin substrates. The mechanism by which MLN4924 inhibits cancer cell proliferation has not been defined, although it is accompanied by DNA rereplication and attendant DNA damage. Here we show that stabilization of the DNA replication factor Cdt1, a substrate of cullins 1 and 4, is critical for MLN4924 to trigger DNA rereplication and inhibit cell proliferation. Even only 1 hour of exposure to MLN4924, which was sufficient to elevate Cdt1 for 4-5 hours, was found to be sufficient to induce DNA rereplication and to activate apoptosis and senescence pathways. Cells in S phase were most susceptible, suggesting that MLN4924 will be most toxic on highly proliferating cancers. Although MLN4924-induced cell senescence seems to be dependent on induction of p53 and its downstream effector p21(Waf1), we found that p53(-/-) and p21(-/-) cells were even more susceptible than wild-type cells to MLN4924. Our results suggested that apoptosis, not senescence, might be more important for the antiproliferative effect of MLN4924. Furthermore, our findings show that transient exposure to this new investigational drug should be useful for controlling p53-negative cancer cells, which often pose significant clinical challenge.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_023',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 163}]}],\n",
       "   'context': 'Treatment-emergent mutations in NAEβ confer resistance to the NEDD8-activating enzyme inhibitor MLN4924. MLN4924 is an investigational small-molecule inhibitor of NEDD8-activating enzyme (NAE) in clinical trials for the treatment of cancer. MLN4924 is a mechanism-based inhibitor, with enzyme inhibition occurring through the formation of a tight-binding NEDD8-MLN4924 adduct. In cell and xenograft models of cancer, we identified treatment-emergent heterozygous mutations in the adenosine triphosphate binding pocket and NEDD8-binding cleft of NAEβ as the primary mechanism of resistance to MLN4924. Biochemical analyses of NAEβ mutants revealed slower rates of adduct formation and reduced adduct affinity for the mutant enzymes. A compound with tighter binding properties was able to potently inhibit mutant enzymes in cells. These data provide rationales for patient selection and the development of next-generation NAE inhibitors designed to overcome treatment-emergent NAEβ mutations.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_024',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 383}]}],\n",
       "   'context': \"Mechanistic studies of substrate-assisted inhibition of ubiquitin-activating enzyme by adenosine sulfamate analogues. Ubiquitin-activating enzyme (UAE or E1) activates ubiquitin via an adenylate intermediate and catalyzes its transfer to a ubiquitin-conjugating enzyme (E2). MLN4924 is an adenosine sulfamate analogue that was identified as a selective, mechanism-based inhibitor of NEDD8-activating enzyme (NAE), another E1 enzyme, by forming a NEDD8-MLN4924 adduct that tightly binds at the active site of NAE, a novel mechanism termed substrate-assisted inhibition (Brownell, J. E., Sintchak, M. D., Gavin, J. M., Liao, H., Bruzzese, F. J., Bump, N. J., Soucy, T. A., Milhollen, M. A., Yang, X., Burkhardt, A. L., Ma, J., Loke, H. K., Lingaraj, T., Wu, D., Hamman, K. B., Spelman, J. J., Cullis, C. A., Langston, S. P., Vyskocil, S., Sells, T. B., Mallender, W. D., Visiers, I., Li, P., Claiborne, C. F., Rolfe, M., Bolen, J. B., and Dick, L. R. (2010) Mol. Cell 37, 102-111). In the present study, substrate-assisted inhibition of human UAE (Ube1) by another adenosine sulfamate analogue, 5'-O-sulfamoyl-N(6)-[(1S)-2,3-dihydro-1H-inden-1-yl]-adenosine (Compound I), a nonselective E1 inhibitor, was characterized. Compound I inhibited UAE-dependent ATP-PP(i) exchange activity, caused loss of UAE thioester, and inhibited E1-E2 transthiolation in a dose-dependent manner. Mechanistic studies on Compound I and its purified ubiquitin adduct demonstrate that the proposed substrate-assisted inhibition via covalent adduct formation is entirely consistent with the three-step ubiquitin activation process and that the adduct is formed via nucleophilic attack of UAE thioester by the sulfamate group of Compound I after completion of step 2. Kinetic and affinity analysis of Compound I, MLN4924, and their purified ubiquitin adducts suggest that both the rate of adduct formation and the affinity between the adduct and E1 contribute to the overall potency. Because all E1s are thought to use a similar mechanism to activate their cognate ubiquitin-like proteins, the substrate-assisted inhibition by adenosine sulfamate analogues represents a promising strategy to develop potent and selective E1 inhibitors that can modulate diverse biological pathways.\"},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_025',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 110}]}],\n",
       "   'context': 'Mechanism-based neddylation inhibitor. Brownell et al. (2010) elucidate the mechanism of action of MLN4924, a NEDD8-activating enzyme inhibitor. MLN4924 requires the activity of the enzyme to generate a NEDD8-adenylate analog that potently and selectively shuts down this posttranslational modification system.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_026',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 143}]}],\n",
       "   'context': 'Quantifiable analysis of cellular pathway inhibition of a Nedd8-activating enzyme inhibitor, MLN4924, using AlphaScreen. Cellular effects of a Nedd8-activating enzyme (NAE) inhibitor, MLN4924, using the AlphaScreen format were explored. MLN4924 acts as a substrate-assisted inhibitor of NAE by forming a tight binding Nedd8-MLN4924 adduct. The inhibited enzyme can no longer transfer Nedd8 downstream to modify and activate the E3 cullin-RING ligases. This results in the stabilization of proteins regulated by the proteasome, leading to cell death. These studies monitored the endogenous cellular changes to NAE∼Nedd8 thioester, the formation of the Nedd8-MLN4924 adduct, and the reduction in the Cul1-Nedd8. Lysates derived from MLN4924-treated HCT116 cells showed that whereas the β-subunit of NAE remained constant, reductions of both NAE∼Nedd8 thioester and Cul1-Nedd8 levels occurred with a concomitant rise of the adduct. Moreover, the formation of the Nedd8-MLN4924 adduct was approximately stoichiometric with the concentration of NAEβ. Higher density 384-well cell-based assays illustrated the kinetics of enzyme inactivation across a wider range of MLN4924 concentrations, showing a rapid loss of NAE∼Nedd8 thioester and Cul1-Nedd8. The reduction of NAE∼Nedd8 thioester precedes the loss of Cul1-Nedd8 at twice the rate. Finally, these results clearly demonstrate the utility of the homogeneous assay for quantitative assessment of these endogenous cellular components in a 384-well plate in response to inhibition of NAE by MLN4924.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_027',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 213}]}],\n",
       "   'context': 'A first-in-class NAE inhibitor, MLN4924, blocks lentiviral infection in myeloid cells by disrupting neddylation-dependent Vpx-mediated SAMHD1 degradation. MLN4924 is a first-in-class cancer drug that inhibits the Nedd8-activating enzyme (NAE). Herein, we report that MLN4924 inhibits Vpx/Vpr-induced SAMHD1 degradation by inhibiting the neddylation of E3 ubiquitin ligase and blocks macaque simian immunodeficiency virus (SIVmac) replication in myeloid cells. SAMHD1 is required for MLN4924-mediated SIVmac inhibition. Our findings indicate the potential efficacy of inhibiting neddylation as an antiretroviral strategy and identify the readily available anticancer drug MLN4924 as a candidate agent for that purpose.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_028',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 235}]}],\n",
       "   'context': 'NEDD8-targeting drug MLN4924 elicits DNA rereplication by stabilizing Cdt1 in S phase, triggering checkpoint activation, apoptosis, and senescence in cancer cells. MLN4924 is a first-in-class experimental cancer drug that inhibits the NEDD8-activating enzyme, thereby inhibiting cullin-RING E3 ubiquitin ligases and stabilizing many cullin substrates. The mechanism by which MLN4924 inhibits cancer cell proliferation has not been defined, although it is accompanied by DNA rereplication and attendant DNA damage. Here we show that stabilization of the DNA replication factor Cdt1, a substrate of cullins 1 and 4, is critical for MLN4924 to trigger DNA rereplication and inhibit cell proliferation. Even only 1 hour of exposure to MLN4924, which was sufficient to elevate Cdt1 for 4-5 hours, was found to be sufficient to induce DNA rereplication and to activate apoptosis and senescence pathways. Cells in S phase were most susceptible, suggesting that MLN4924 will be most toxic on highly proliferating cancers. Although MLN4924-induced cell senescence seems to be dependent on induction of p53 and its downstream effector p21(Waf1), we found that p53(-/-) and p21(-/-) cells were even more susceptible than wild-type cells to MLN4924. Our results suggested that apoptosis, not senescence, might be more important for the antiproliferative effect of MLN4924. Furthermore, our findings show that transient exposure to this new investigational drug should be useful for controlling p53-negative cancer cells, which often pose significant clinical challenge.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_029',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'NEDD8-activating enzyme', 'answer_start': 163}]}],\n",
       "   'context': 'Treatment-emergent mutations in NAEβ confer resistance to the NEDD8-activating enzyme inhibitor MLN4924. MLN4924 is an investigational small-molecule inhibitor of NEDD8-activating enzyme (NAE) in clinical trials for the treatment of cancer. MLN4924 is a mechanism-based inhibitor, with enzyme inhibition occurring through the formation of a tight-binding NEDD8-MLN4924 adduct. In cell and xenograft models of cancer, we identified treatment-emergent heterozygous mutations in the adenosine triphosphate binding pocket and NEDD8-binding cleft of NAEβ as the primary mechanism of resistance to MLN4924. Biochemical analyses of NAEβ mutants revealed slower rates of adduct formation and reduced adduct affinity for the mutant enzymes. A compound with tighter binding properties was able to potently inhibit mutant enzymes in cells. These data provide rationales for patient selection and the development of next-generation NAE inhibitors designed to overcome treatment-emergent NAEβ mutations.'},\n",
       "  {'qas': [{'id': '56ed03862ac5ed1459000004_030',\n",
       "     'question': 'Which enzyme does MLN4924 inhibit?',\n",
       "     'answers': [{'text': 'Nedd8-activating enzyme', 'answer_start': 143}]}],\n",
       "   'context': 'Quantifiable analysis of cellular pathway inhibition of a Nedd8-activating enzyme inhibitor, MLN4924, using AlphaScreen. Cellular effects of a Nedd8-activating enzyme (NAE) inhibitor, MLN4924, using the AlphaScreen format were explored. MLN4924 acts as a substrate-assisted inhibitor of NAE by forming a tight binding Nedd8-MLN4924 adduct. The inhibited enzyme can no longer transfer Nedd8 downstream to modify and activate the E3 cullin-RING ligases. This results in the stabilization of proteins regulated by the proteasome, leading to cell death. These studies monitored the endogenous cellular changes to NAE∼Nedd8 thioester, the formation of the Nedd8-MLN4924 adduct, and the reduction in the Cul1-Nedd8. Lysates derived from MLN4924-treated HCT116 cells showed that whereas the β-subunit of NAE remained constant, reductions of both NAE∼Nedd8 thioester and Cul1-Nedd8 levels occurred with a concomitant rise of the adduct. Moreover, the formation of the Nedd8-MLN4924 adduct was approximately stoichiometric with the concentration of NAEβ. Higher density 384-well cell-based assays illustrated the kinetics of enzyme inactivation across a wider range of MLN4924 concentrations, showing a rapid loss of NAE∼Nedd8 thioester and Cul1-Nedd8. The reduction of NAE∼Nedd8 thioester precedes the loss of Cul1-Nedd8 at twice the rate. Finally, these results clearly demonstrate the utility of the homogeneous assay for quantitative assessment of these endogenous cellular components in a 384-well plate in response to inhibition of NAE by MLN4924.'},\n",
       "  {'qas': [{'id': '54f9cb34dd3fc62544000002_001',\n",
       "     'question': 'Which protein has been found to interact with phospholamban (PLN) and is also an anti-apoptotic protein?',\n",
       "     'answers': [{'text': '(HAX-1)', 'answer_start': 521}]}],\n",
       "   'context': 'Phospholamban interacts with HAX-1, a mitochondrial protein with anti-apoptotic function. Phospholamban (PLN) is a key regulator of Ca(2+) homeostasis and contractility in the heart. Its regulatory effects are mediated through its interaction with the sarcoplasmic reticulum Ca(2+)-ATPase, (SERCA2a), resulting in alterations of its Ca(2+)-affinity. To identify additional proteins that may interact with PLN, we used the yeast-two-hybrid system to screen an adult human cardiac cDNA library. HS-1 associated protein X-1 (HAX-1) was identified as a PLN-binding partner. The minimal binding regions were mapped to amino acid residues 203-245 for HAX-1 and residues 16-22 for PLN. The interaction between the two proteins was confirmed using GST-HAX-1, bound to the glutathione-matrix, which specifically adsorbed native PLN from human or mouse cardiac homogenates, while in reciprocal binding studies, recombinant His-HAX-1 bound GST-PLN. Kinetic studies using surface plasmon resonance yielded a K(D) of approximately 1 muM as the binding affinity for the PLN/HAX-1 complex. Phosphorylation of PLN by cAMP-dependent protein kinase reduced binding to HAX-1, while increasing concentrations of Ca(2+) diminished the PLN/HAX-1 interaction in a dose-dependent manner. HAX-1 concentrated to mitochondria, but upon transient co-transfection of HEK 293 cells with PLN, HAX-1 redistributed and co-localized with PLN at the endoplasmic reticulum. Analysis of the anti-apoptotic function of HAX-1 revealed that the presence of PLN enhanced the HAX-1 protective effects from hypoxia/reoxygenation-induced cell death. These findings suggest a possible link between the Ca(2+) handling by the sarcoplasmic reticulum and cell survival mediated by the PLN/HAX-1 interaction.'},\n",
       "  {'qas': [{'id': '54f9cb34dd3fc62544000002_002',\n",
       "     'question': 'Which protein has been found to interact with phospholamban (PLN) and is also an anti-apoptotic protein?',\n",
       "     'answers': [{'text': '(HAX-1)', 'answer_start': 812}]}],\n",
       "   'context': 'The role of SERCA2a/PLN complex, Ca(2+) homeostasis, and anti-apoptotic proteins in determining cell fate. Intracellular calcium is a major coordinator of numerous aspects of cellular physiology, including muscle contractility and cell survival. In cardiac muscle, aberrant Ca(2+) cycling has been implicated in a range of pathological conditions including cardiomyopathies and heart failure. The sarco(endo)plasmic reticulum Ca(2+) transport adenosine triphosphatase (SERCA2a) and its regulator phospholamban (PLN) have a central role in modulating Ca(2+) homeostasis and, therefore, cardiac function. Herein, we discuss the mechanisms through which SERCA2a and PLN control cardiomyocyte function in health and disease. Emphasis is placed on our newly identified PLN-binding partner HS-1-associated protein X-1 (HAX-1), which has an anti-apoptotic function and presents with numerous similarities to Bcl-2. Recent evidence indicates that proteins of the Bcl-2 family can influence ER Ca(2+) content, a critical determinant of cellular sensitivity to apoptosis. The discovery of the PLN/HAX-1 interaction therefore unveils an important new link between Ca(2+) homeostasis and cell survival, with significant therapeutic potential.'},\n",
       "  {'qas': [{'id': '54f9cb34dd3fc62544000002_003',\n",
       "     'question': 'Which protein has been found to interact with phospholamban (PLN) and is also an anti-apoptotic protein?',\n",
       "     'answers': [{'text': '(HAX-1)', 'answer_start': 441}]}],\n",
       "   'context': \"The anti-apoptotic protein HAX-1 interacts with SERCA2 and regulates its protein levels to promote cell survival. Cardiac contractility is regulated through the activity of various key Ca(2+)-handling proteins. The sarco(endo)plasmic reticulum (SR) Ca(2+) transport ATPase (SERCA2a) and its inhibitor phospholamban (PLN) control the uptake of Ca(2+) by SR membranes during relaxation. Recently, the antiapoptotic HS-1-associated protein X-1 (HAX-1) was identified as a binding partner of PLN, and this interaction was postulated to regulate cell apoptosis. In the current study, we determined that HAX-1 can also bind to SERCA2. Deletion mapping analysis demonstrated that amino acid residues 575-594 of SERCA2's nucleotide binding domain are required for its interaction with the C-terminal domain of HAX-1, containing amino acids 203-245. In transiently cotransfected human embryonic kidney 293 cells, recombinant SERCA2 was specifically targeted to the ER, whereas HAX-1 selectively concentrated at mitochondria. On triple transfections with PLN, however, HAX-1 massively translocated to the ER membranes, where it codistributed with PLN and SERCA2. Overexpression of SERCA2 abrogated the protective effects of HAX-1 on cell survival, after hypoxia/reoxygenation or thapsigargin treatment. Importantly, HAX-1 overexpression was associated with down-regulation of SERCA2 expression levels, resulting in significant reduction of apparent ER Ca(2+) levels. These findings suggest that HAX-1 may promote cell survival through modulation of SERCA2 protein levels and thus ER Ca(2+) stores.\"},\n",
       "  {'qas': [{'id': '54f9cb34dd3fc62544000002_004',\n",
       "     'question': 'Which protein has been found to interact with phospholamban (PLN) and is also an anti-apoptotic protein?',\n",
       "     'answers': [{'text': 'The HS-1 associated protein X-1',\n",
       "       'answer_start': 69}]}],\n",
       "   'context': 'The anti-apoptotic protein HAX-1 is a regulator of cardiac function. The HS-1 associated protein X-1 (HAX-1) is a ubiquitously expressed protein that protects cardiomyocytes from programmed cell death. Here we identify HAX-1 as a regulator of contractility and calcium cycling in the heart. HAX-1 overexpression reduced sarcoplasmic reticulum Ca-ATPase (SERCA2) pump activity in isolated cardiomyocytes and in vivo, leading to depressed myocyte calcium kinetics and mechanics. Conversely, downregulation of HAX-1 enhanced calcium cycling and contractility. The inhibitory effects of HAX-1 were abolished upon phosphorylation of phospholamban, which plays a fundamental role in controlling basal contractility and constitutes a key downstream effector of the beta-adrenergic signaling cascade. Mechanistically, HAX-1 promoted formation of phospholamban monomers, the active/inhibitory units of the calcium pump. Indeed, ablation of PLN rescued HAX-1 inhibition of contractility in vivo. Thus, HAX-1 represents a regulatory mechanism in cardiac calcium cycling and its responses to sympathetic stimulation, implicating its importance in calcium homeostasis and cell survival.'},\n",
       "  {'qas': [{'id': '54f9cb34dd3fc62544000002_005',\n",
       "     'question': 'Which protein has been found to interact with phospholamban (PLN) and is also an anti-apoptotic protein?',\n",
       "     'answers': [{'text': '(HAX-1)', 'answer_start': 101}]}],\n",
       "   'context': 'The anti-apoptotic protein HAX-1 is a regulator of cardiac function. The HS-1 associated protein X-1 (HAX-1) is a ubiquitously expressed protein that protects cardiomyocytes from programmed cell death. Here we identify HAX-1 as a regulator of contractility and calcium cycling in the heart. HAX-1 overexpression reduced sarcoplasmic reticulum Ca-ATPase (SERCA2) pump activity in isolated cardiomyocytes and in vivo, leading to depressed myocyte calcium kinetics and mechanics. Conversely, downregulation of HAX-1 enhanced calcium cycling and contractility. The inhibitory effects of HAX-1 were abolished upon phosphorylation of phospholamban, which plays a fundamental role in controlling basal contractility and constitutes a key downstream effector of the beta-adrenergic signaling cascade. Mechanistically, HAX-1 promoted formation of phospholamban monomers, the active/inhibitory units of the calcium pump. Indeed, ablation of PLN rescued HAX-1 inhibition of contractility in vivo. Thus, HAX-1 represents a regulatory mechanism in cardiac calcium cycling and its responses to sympathetic stimulation, implicating its importance in calcium homeostasis and cell survival.'},\n",
       "  {'qas': [{'id': '54f9cb34dd3fc62544000002_006',\n",
       "     'question': 'Which protein has been found to interact with phospholamban (PLN) and is also an anti-apoptotic protein?',\n",
       "     'answers': [{'text': '(HAX-1)', 'answer_start': 812}]}],\n",
       "   'context': 'The role of SERCA2a/PLN complex, Ca(2+) homeostasis, and anti-apoptotic proteins in determining cell fate. Intracellular calcium is a major coordinator of numerous aspects of cellular physiology, including muscle contractility and cell survival. In cardiac muscle, aberrant Ca(2+) cycling has been implicated in a range of pathological conditions including cardiomyopathies and heart failure. The sarco(endo)plasmic reticulum Ca(2+) transport adenosine triphosphatase (SERCA2a) and its regulator phospholamban (PLN) have a central role in modulating Ca(2+) homeostasis and, therefore, cardiac function. Herein, we discuss the mechanisms through which SERCA2a and PLN control cardiomyocyte function in health and disease. Emphasis is placed on our newly identified PLN-binding partner HS-1-associated protein X-1 (HAX-1), which has an anti-apoptotic function and presents with numerous similarities to Bcl-2. Recent evidence indicates that proteins of the Bcl-2 family can influence ER Ca(2+) content, a critical determinant of cellular sensitivity to apoptosis. The discovery of the PLN/HAX-1 interaction therefore unveils an important new link between Ca(2+) homeostasis and cell survival, with significant therapeutic potential.'},\n",
       "  {'qas': [{'id': '54d73e223706e89528000010_001',\n",
       "     'question': 'What is the generic name of Gliolan?',\n",
       "     'answers': [{'text': '5-aminolevulinic acid', 'answer_start': 152}]}],\n",
       "   'context': 'Cost-effectiveness of 5-aminolevulinic acid-induced fluorescence in malignant glioma surgery. OBJECTIVE: This study evaluates the cost-effectiveness of 5-aminolevulinic acid (5-ALA, Gliolan®) in patients undergoing surgery for malignant glioma, in standard clinical practice conditions in Spain. MATERIAL AND METHODS: Cost-effectiveness ratios were determined in terms of incremental cost per complete resection (CR) and incremental cost per additional quality-adjusted life year (QALY), based on data collected in the VISIONA observational study. RESULTS: Incremental cost with 5-ALA versus conventional surgery using white light only amounts to € 4550 per additional CR achieved and € 9021 per QALY gained. A sensitivity analysis shows these results to be robust. CONCLUSION: Malignant glioma surgery guided by 5-ALA fluorescence entails a moderate increase in hospital costs compared to current surgical practice and can be considered a cost-effective innovation.'},\n",
       "  {'qas': [{'id': '54d73e223706e89528000010_002',\n",
       "     'question': 'What is the generic name of Gliolan?',\n",
       "     'answers': [{'text': '5-aminolevulinic acid', 'answer_start': 177}]}],\n",
       "   'context': 'Observational, retrospective study of the effectiveness of 5-aminolevulinic acid in malignant glioma surgery in Spain (The VISIONA study). OBJECTIVE: To assess effectiveness of 5-aminolevulinic acid (5-ALA, Gliolan(®)) in patients treated for malignant glioma under typical daily practice conditions in Spain, using complete resection rate (CR) and progression free survival at 6 months (PFS6). MATERIAL AND METHODS: Retrospective review of data from 18 neurosurgery departments that were categorised as either using or not using 5-ALA. The study included adult patients with suspected malignant gliomas for whom the intended treatment plan included complete resection followed by radiotherapy and chemotherapy with temozolomide. Postoperative MRI and clinical data representing at least 6 months were required for inclusion. Rates of CR and PFS6 were compared between patients with 5-ALA treatment and those without. RESULTS: The study included 251 evaluable cases. CR and PFS6 rates were significantly higher in the group of patients treated surgically with 5-ALA: CR, 67% versus 45%, p=.000; PFS6 for patients with grade IV tumours, 69% versus 48%; p=.002. The differences retained their significance and magnitude after adjusting for all covariates including age, functional status, and whether gliomas were located in eloquent areas. CONCLUSIONS: In this retrospective series, use of 5-ALA during habitual surgical procedures in Spain was associated with a higher complete resection rate for malignant glioma and increased PFS6 for grade iv glioma.'},\n",
       "  {'qas': [{'id': '55058af6f73303d458000004_001',\n",
       "     'question': 'Which protein is the E3-ubiquitin ligase that targets the tumor suppressor p53 for proteasomal degradation?',\n",
       "     'answers': [{'text': 'the human homologue of Mdm2 (Hdm2)',\n",
       "       'answer_start': 206}]}],\n",
       "   'context': 'Interactions of the Hdm2/p53 and proteasome pathways may enhance the antitumor activity of bortezomib. PURPOSE: p53 is inactivated in many human malignancies through missense mutations or overexpression of the human homologue of Mdm2 (Hdm2), an E3 ubiquitin ligase that ubiquitinates p53, thereby promoting its proteasomal degradation. The cis-imidazoline nutlin-3 can disrupt the p53-Hdm2 interaction and activate p53, inducing apoptosis in vitro in many malignancies, including multiple myeloma (MM). EXPERIMENTAL DESIGN: We hypothesized that suppression of Hdm2-mediated p53 ubiquitination may augment sequelae of p53 accumulation caused by proteasomal inhibition. We compared the response of MM cells versus several epithelial cancer models to the proteasome inhibitor bortezomib in combination with nutlin-3. RESULTS: The combination of sublethal concentrations of bortezomib plus nutlin-3 induced additive cytotoxicity against bortezomib-sensitive MM cell lines. Importantly, however, in breast, prostate, colon, and thyroid (papillary, follicular, anaplastic, and medullary) carcinoma cell lines, this combination triggered synergistic cytotoxicity, and increased expression of p53, p21, Hdm2, Bax, Noxa, PUMA, and cleavage of caspase-3 and poly ADP ribose polymerase. Coculture with bone marrow stromal cells attenuated MM cell sensitivity to nutlin-3 monotherapy and was associated with evidence of suppression of p53 activity in MM cells, whereas combined bortezomib-nutlin-3 treatment maintained cytotoxicity even in the presence of bone marrow stromal cells. CONCLUSIONS: This differential response of MM versus epithelial carcinomas to combination of nutlin-3 with bortezomib sheds new light on the role of p53 in bortezomib-induced apoptosis. Concurrent Hdm2 inhibition with bortezomib may extend the spectrum of bortezomib applications to malignancies with currently limited sensitivity to single-agent bortezomib or, in the future, to MM patients with decreased clinical responsiveness to bortezomib-based therapy.'},\n",
       "  {'qas': [{'id': '56b7083376d8bf8d13000001_001',\n",
       "     'question': 'What is the causative agent of the \"Panama disease\" affecting bananas?',\n",
       "     'answers': [{'text': 'Fusarium oxysporum f. sp. cubense',\n",
       "       'answer_start': 184}]}],\n",
       "   'context': 'Development of a real-time fluorescence loop-mediated isothermal amplification assay for rapid and quantitative detection of Fusarium oxysporum f. sp. cubense tropical race 4 in soil. Fusarium oxysporum f. sp. cubense (Foc), the causal agent of Fusarium wilt (Panama disease), is one of the most devastating diseases of banana (Musa spp.). The Foc tropical race 4 (TR4) is currently known as a major concern in global banana production. No effective resistance is known in Musa to Foc, and no effective measures for controlling Foc once banana plants have been infected in place. Early and accurate detection of Foc TR4 is essential to protect banana industry and guide banana planting. A real-time fluorescence loop-mediated isothermal amplification assay (RealAmp) was developed for the rapid and quantitative detection of Foc TR4 in soil. The detection limit of the RealAmp assay was approximately 0.4 pg/µl plasmid DNA when mixed with extracted soil DNA or 10(3) spores/g of artificial infested soil, and no cross-reaction with other relative pathogens were observed. The RealAmp assay for quantifying genomic DNA of TR4 was confirmed by testing both artificially and naturally infested samples. Quantification of the soil-borne pathogen DNA of Foc TR4 in naturally infested samples was no significant difference compared to classic real-time PCR (P>0.05). Additionally, RealAmp assay was visual with an improved closed-tube visual detection system by adding SYBR Green I fluorescent dye to the inside of the lid prior to amplification, which avoided the inhibitory effects of the stain on DNA amplification and makes the assay more convenient in the field and could thus become a simple, rapid and effective technique that has potential as an alternative tool for the detection and monitoring of Foc TR4 in field, which would be a routine DNA-based testing service for the soil-borne pathogen in South China.'},\n",
       "  {'qas': [{'id': '56b7083376d8bf8d13000001_002',\n",
       "     'question': 'What is the causative agent of the \"Panama disease\" affecting bananas?',\n",
       "     'answers': [{'text': 'Fusarium oxysporum f. sp. cubense',\n",
       "       'answer_start': 277}]}],\n",
       "   'context': 'Analysis of banana transcriptome and global gene expression profiles in banana roots in response to infection by race 1 and tropical race 4 of Fusarium oxysporum f. sp. cubense. BACKGROUND: Cavendish, the most widely grown banana cultivar, is relatively resistant to Race 1 of Fusarium oxysporum f. sp. cubense (Foc1) which caused widespread Panama disease during the first half of the 20th century but is susceptible to Tropical Race 4 of Foc (Foc TR4) which is threatening world banana production. The genome of the diploid species Musa acuminata which is the ancestor of a majority of triploid banana cultivars has recently been sequenced. Availability of banana transcriptomes will be highly useful for improving banana genome annotation and for biological research. The knowledge of global gene expression patterns influenced by infection of different Foc races will help to understand the host responses to the infection. RESULTS: RNA samples from different organs of the Cavendish cultivar were pooled for deep sequencing using the Illumina technology. Analysis of the banana transcriptome led to identification of over 842 genes that were not annotated by the Musa genome project. A large number of simple nucleotide polymorphisms (SNPs) and short insertions and deletion (indels) were identified from the transcriptome data. GFP-expressing Foc1 and Foc TR4 were used to monitor the infection process. Both Foc1 and Foc TR4 were found to be able to invade banana roots and spread to root vascular tissues in the first two days following inoculation. Digital gene expression (DGE) profiling analysis reveal that the infection by Foc1 and Foc TR4 caused very similar changes in the global gene expression profiles in the banana roots during the first two days of infection. The Foc infection led to induction of many well-known defense-related genes. Two genes encoding the ethylene biosynthetic enzyme ACC oxidase and several ethylene-responsive transcription factors (ERF) were among the strongly induced genes by both Foc1 and Foc TR4. CONCLUSIONS: Both Foc1 and Foc TR4 are able to spread into the vascular system of banana roots during the early infection process and their infection led to similar gene expression profiles in banana roots. The transcriptome profiling analysis indicates that the ethylene synthetic and signalling pathways were activated in response to the Foc infection.'},\n",
       "  {'qas': [{'id': '56b7083376d8bf8d13000001_003',\n",
       "     'question': 'What is the causative agent of the \"Panama disease\" affecting bananas?',\n",
       "     'answers': [{'text': 'Fusarium oxysporum f. sp. cubense',\n",
       "       'answer_start': 499}]}],\n",
       "   'context': 'Control of Panama disease of banana by rotating and intercropping with Chinese chive (Allium tuberosum Rottler): role of plant volatiles. Intercropping and rotating banana (Musa spp.) with Chinese chive (Allium tuberosum Rottler) has been used as an effective method to control Panama disease (Fusarium wilt) of banana in South China. However, the underlying mechanism is unknown. In this study, we used aqueous leachates and volatiles from Chinese chive to evaluate their antimicrobial activity on Fusarium oxysporum f. sp. cubense race 4 (FOC), the causal agent of Panama disease in banana, and identified the antifungal compounds. Both leaf and root leachates of Chinese chive displayed strong inhibition against FOC, but the concentrated leachates showed lower inhibition than the original leachates. In a sealed system volatiles emitted from the leaves and roots of Chinese chive inhibited mycelial growth of FOC. Volatile compounds emitted from the intact growing roots mimicking natural environment inhibited spore germination of FOC. We identified five volatiles including 2-methyl-2-pentenal and four organosulfur compounds (dimethyl trisulfide, dimethyl disulfide, dipropyl disulfide, and dipropyl trisulfide) from the leaves and roots of Chinese chive. All these compounds exhibited inhibitory effects on FOC, but 2-methyl-2-pentenal and dimethyl trisulfide showed stronger inhibition than the other three compounds. 2-Methyl-2-pentenal at 50-100 μl/l completely inhibited the mycelial growth of FOC. Our results demonstrate that antifungal volatiles released from Chinese chive help control Panama disease in banana. We conclude that intercropping and rotating banana with Chinese chive can control Panama disease and increase cropland biodiversity.'},\n",
       "  {'qas': [{'id': '56b7083376d8bf8d13000001_004',\n",
       "     'question': 'What is the causative agent of the \"Panama disease\" affecting bananas?',\n",
       "     'answers': [{'text': 'Fusarium oxysporum f. sp. cubense',\n",
       "       'answer_start': 182}]}],\n",
       "   'context': 'Fusarium Wilt of Banana Is Caused by Several Pathogens Referred to as Fusarium oxysporum f. sp. cubense. ABSTRACT Fusarium wilt of banana (also known as Panama disease) is caused by Fusarium oxysporum f. sp. cubense. Where susceptible cultivars are grown, management is limited to the use of pathogen-free planting stock and clean soils. Resistant genotypes exist for some applications, but resistance is still needed in other situations. Progress has been made with this recalcitrant crop by traditional and nontraditional improvement programs. The disease was first reported in Australia in 1876, but did the greatest damage in export plantations in the western tropics before 1960. A new variant, tropical race 4, threatens the trades that are now based on Cavendish cultivars, and other locally important types such as the plantains. Phylogenetic studies indicate that F. oxysporum f. sp. cubense had several independent evolutionary origins. The significance of these results and the future impact of this disease are discussed.'},\n",
       "  {'qas': [{'id': '56b7083376d8bf8d13000001_005',\n",
       "     'question': 'What is the causative agent of the \"Panama disease\" affecting bananas?',\n",
       "     'answers': [{'text': 'Fusarium oxysporum f. sp. cubense',\n",
       "       'answer_start': 194}]}],\n",
       "   'context': 'Multiple evolutionary origins of the fungus causing Panama disease of banana: concordant evidence from nuclear and mitochondrial gene genealogies. Panama disease of banana, caused by the fungus Fusarium oxysporum f. sp. cubense, is a serious constraint both to the commercial production of banana and cultivation for subsistence agriculture. Previous work has indicated that F. oxysporum f. sp. cubense consists of several clonal lineages that may be genetically distant. In this study we tested whether lineages of the Panama disease pathogen have a monophyletic origin by comparing DNA sequences of nuclear and mitochondrial genes. DNA sequences were obtained for translation elongation factor 1alpha and the mitochondrial small subunit ribosomal RNA genes for F. oxysporum strains from banana, pathogenic strains from other hosts and putatively nonpathogenic isolates of F. oxysporum. Cladograms for the two genes were highly concordant and a partition-homogeneity test indicated the two datasets could be combined. The tree inferred from the combined dataset resolved five lineages corresponding to \"F. oxysporum f. sp. cubense\" with a large dichotomy between two taxa represented by strains most commonly isolated from bananas with Panama disease. The results also demonstrate that the latter two taxa have significantly different chromosome numbers. F. oxysporum isolates collected as nonpathogenic or pathogenic to other hosts that have very similar or identical elongation factor 1alpha and mitochondrial small subunit genotypes as banana pathogens were shown to cause little or no disease on banana. Taken together, these results indicate Panama disease of banana is caused by fungi with independent evolutionary origins.'},\n",
       "  {'qas': [{'id': '55376663bc4f83e82800000a_001',\n",
       "     'question': 'Which biomarker is widely used in the diagnosis of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'CD99', 'answer_start': 506}]}],\n",
       "   'context': \"Primary cutaneous pre-B lymphoblastic lymphoma immunohistologically mimics Ewing's sarcoma/primitive neuroectodermal tumor. Precursor B-cell lymphoblastic lymphomas (B-LBLs) are rare and most often involve the skin in the head and neck region. Histologically, cutaneous B-LBLs may be confused with other small round-cell neoplasms. Moreover, half of B-LBL patients are negative for CD45 (leucocyte common antigen, LCA), a widely used marker for the diagnosis of lymphoma, and a significant portion express CD99, a marker for Ewing's sarcoma (ES) or primitive neuroectodermal tumor (PNET). Therefore, an extranodal B-LBL may be misinterpreted as PNET or ES. Here, we report on 2 boys, aged 10 and 5 years, with primary cutaneous B-LBL of the scalp. PNET was initially misdiagnosed because the tumor cells were negative for CD45 but strongly positive for CD99. Advanced stage of acute lymphoblastic leukemia (ALL) developed later and both patients died during the course of treatment for ALL. In retrospective analyses, tumor cells in the initial biopsy specimens of both patients were found to be reactive to terminal deoxynucleotidyl transferase (TdT), CD43 and CD10. Thus, the diagnosis of B-LBL was confirmed. These cases illustrate the possibility that primary cutaneous B-LBL may mimic ES or PNET immunophenotypically, and that correct diagnosis in doubtful cases may be facilitated by analysis using a complete panel of antibodies, particularly including TdT and CD43.\"},\n",
       "  {'qas': [{'id': '55376663bc4f83e82800000a_002',\n",
       "     'question': 'Which biomarker is widely used in the diagnosis of Ewing sarcoma?',\n",
       "     'answers': [{'text': 'CD99', 'answer_start': 506}]}],\n",
       "   'context': \"Primary cutaneous pre-B lymphoblastic lymphoma immunohistologically mimics Ewing's sarcoma/primitive neuroectodermal tumor. Precursor B-cell lymphoblastic lymphomas (B-LBLs) are rare and most often involve the skin in the head and neck region. Histologically, cutaneous B-LBLs may be confused with other small round-cell neoplasms. Moreover, half of B-LBL patients are negative for CD45 (leucocyte common antigen, LCA), a widely used marker for the diagnosis of lymphoma, and a significant portion express CD99, a marker for Ewing's sarcoma (ES) or primitive neuroectodermal tumor (PNET). Therefore, an extranodal B-LBL may be misinterpreted as PNET or ES. Here, we report on 2 boys, aged 10 and 5 years, with primary cutaneous B-LBL of the scalp. PNET was initially misdiagnosed because the tumor cells were negative for CD45 but strongly positive for CD99. Advanced stage of acute lymphoblastic leukemia (ALL) developed later and both patients died during the course of treatment for ALL. In retrospective analyses, tumor cells in the initial biopsy specimens of both patients were found to be reactive to terminal deoxynucleotidyl transferase (TdT), CD43 and CD10. Thus, the diagnosis of B-LBL was confirmed. These cases illustrate the possibility that primary cutaneous B-LBL may mimic ES or PNET immunophenotypically, and that correct diagnosis in doubtful cases may be facilitated by analysis using a complete panel of antibodies, particularly including TdT and CD43.\"},\n",
       "  {'qas': [{'id': '56c077e9ef6e394741000021_001',\n",
       "     'question': 'Name monoclonal antibody against SLAMF7.',\n",
       "     'answers': [{'text': 'signaling lymphocytic activation molecule-F7',\n",
       "       'answer_start': 177}]}],\n",
       "   'context': 'Elotuzumab enhances natural killer cell activation and myeloma cell killing through interleukin-2 and TNF-α pathways. Elotuzumab is a humanized monoclonal antibody specific for signaling lymphocytic activation molecule-F7 (SLAMF7, also known as CS1, CD319, or CRACC) that enhances natural killer (NK) cell-mediated antibody-dependent cellular cytotoxicity (ADCC) of SLAMF7-expressing myeloma cells. This study explored the mechanisms underlying enhanced myeloma cell killing with elotuzumab as a single agent and in combination with lenalidomide, to support ongoing phase III trials in patients with relapsed/refractory or newly-diagnosed multiple myeloma (MM). An in vitro peripheral blood lymphocyte (PBL)/myeloma cell co-culture model was developed to evaluate the combination of elotuzumab and lenalidomide. Expression of activation markers and adhesion receptors was evaluated by flow cytometry, cytokine expression by Luminex and ELISPOT assays, and cytotoxicity by myeloma cell counts. Elotuzumab activated NK cells and promoted myeloma cell death in PBL/myeloma cell co-cultures. The combination of elotuzumab plus lenalidomide demonstrated superior anti-myeloma activity on established MM xenografts in vivo and in PBL/myeloma cell co-cultures in vitro than either agent alone. The combination enhanced myeloma cell killing by modulating NK cell function that coincided with the upregulation of adhesion and activation markers, including interleukin (IL)-2Rα expression, IL-2 production by CD3(+)CD56(+) lymphocytes, and tumor necrosis factor (TNF)-α production. In co-culture assays, TNF-α directly increased NK cell activation and myeloma cell death with elotuzumab or elotuzumab plus lenalidomide, and neutralizing TNF-α decreased NK cell activation and myeloma cell death with elotuzumab. These results demonstrate that elotuzumab activates NK cells and induces myeloma cell death via NK cell-mediated ADCC, which is further enhanced when combined with lenalidomide.'},\n",
       "  {'qas': [{'id': '56c077e9ef6e394741000021_002',\n",
       "     'question': 'Name monoclonal antibody against SLAMF7.',\n",
       "     'answers': [{'text': 'signaling lymphocytic activation molecule-F7',\n",
       "       'answer_start': 177}]}],\n",
       "   'context': 'Elotuzumab enhances natural killer cell activation and myeloma cell killing through interleukin-2 and TNF-α pathways. Elotuzumab is a humanized monoclonal antibody specific for signaling lymphocytic activation molecule-F7 (SLAMF7, also known as CS1, CD319, or CRACC) that enhances natural killer (NK) cell-mediated antibody-dependent cellular cytotoxicity (ADCC) of SLAMF7-expressing myeloma cells. This study explored the mechanisms underlying enhanced myeloma cell killing with elotuzumab as a single agent and in combination with lenalidomide, to support ongoing phase III trials in patients with relapsed/refractory or newly-diagnosed multiple myeloma (MM). An in vitro peripheral blood lymphocyte (PBL)/myeloma cell co-culture model was developed to evaluate the combination of elotuzumab and lenalidomide. Expression of activation markers and adhesion receptors was evaluated by flow cytometry, cytokine expression by Luminex and ELISPOT assays, and cytotoxicity by myeloma cell counts. Elotuzumab activated NK cells and promoted myeloma cell death in PBL/myeloma cell co-cultures. The combination of elotuzumab plus lenalidomide demonstrated superior anti-myeloma activity on established MM xenografts in vivo and in PBL/myeloma cell co-cultures in vitro than either agent alone. The combination enhanced myeloma cell killing by modulating NK cell function that coincided with the upregulation of adhesion and activation markers, including interleukin (IL)-2Rα expression, IL-2 production by CD3(+)CD56(+) lymphocytes, and tumor necrosis factor (TNF)-α production. In co-culture assays, TNF-α directly increased NK cell activation and myeloma cell death with elotuzumab or elotuzumab plus lenalidomide, and neutralizing TNF-α decreased NK cell activation and myeloma cell death with elotuzumab. These results demonstrate that elotuzumab activates NK cells and induces myeloma cell death via NK cell-mediated ADCC, which is further enhanced when combined with lenalidomide.'},\n",
       "  {'qas': [{'id': '571f609c0fd6f91b6800000c_001',\n",
       "     'question': 'Which enzyme deficiency can cause GM1 gangliosidoses?',\n",
       "     'answers': [{'text': 'β-galactosidase', 'answer_start': 349}]}],\n",
       "   'context': 'Gangliosidoses. The gangliosidoses comprise a family of lysosomal storage diseases characterized by the accumulation of complex glycosphingolipids in the nervous system and other tissues, secondary to the deficient activity of lysosomal hydrolases or their associated activator proteins. GM1 and GM2 gangliosidosis are associated with deficiency of β-galactosidase and β-hexosaminidase respectively. All gangliosidoses are characterized by progressive neurodegeneration, the severity of which is proportional to the residual enzyme activity. The GM1 gangliosidoses are characterized by dysostosis, organomegaly and coarsening in their most severe forms, whereas children with classic infantile GM2 gangliosidosis (Tay-Sachs disease) are usually spared systemic involvement, except in the case of the Sandhoff variant, in which organomegaly may occur. Cherry-red macular spots occur in the early onset forms of the gangliosidoses, but are less frequently seen in the less severe, later onset phenotypes. Macrocephaly, an exaggerated startle response, cognitive decline, seizures, ataxia, and progressive muscular atrophy may occur in different forms of gangliosidosis. The diagnosis is made by assay of enzyme activity, and can be confirmed by mutation analysis. Carrier screening for Tay-Sachs disease has been remarkably successful in reducing the incidence of this disease in the at-risk Ashkenazi population. There are no proven disease-modifying therapies for the gangliosidoses.'},\n",
       "  {'qas': [{'id': '571f609c0fd6f91b6800000c_002',\n",
       "     'question': 'Which enzyme deficiency can cause GM1 gangliosidoses?',\n",
       "     'answers': [{'text': 'β-galactosidase', 'answer_start': 349}]}],\n",
       "   'context': 'Gangliosidoses. The gangliosidoses comprise a family of lysosomal storage diseases characterized by the accumulation of complex glycosphingolipids in the nervous system and other tissues, secondary to the deficient activity of lysosomal hydrolases or their associated activator proteins. GM1 and GM2 gangliosidosis are associated with deficiency of β-galactosidase and β-hexosaminidase respectively. All gangliosidoses are characterized by progressive neurodegeneration, the severity of which is proportional to the residual enzyme activity. The GM1 gangliosidoses are characterized by dysostosis, organomegaly and coarsening in their most severe forms, whereas children with classic infantile GM2 gangliosidosis (Tay-Sachs disease) are usually spared systemic involvement, except in the case of the Sandhoff variant, in which organomegaly may occur. Cherry-red macular spots occur in the early onset forms of the gangliosidoses, but are less frequently seen in the less severe, later onset phenotypes. Macrocephaly, an exaggerated startle response, cognitive decline, seizures, ataxia, and progressive muscular atrophy may occur in different forms of gangliosidosis. The diagnosis is made by assay of enzyme activity, and can be confirmed by mutation analysis. Carrier screening for Tay-Sachs disease has been remarkably successful in reducing the incidence of this disease in the at-risk Ashkenazi population. There are no proven disease-modifying therapies for the gangliosidoses.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_001',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 394}]}],\n",
       "   'context': 'Dyke-Davidoff-Masson syndrome: case report of fetal unilateral ventriculomegaly and hypoplastic left middle cerebral artery. Prenatal ultrasonographic detection of unilateral cerebral ventriculomegaly arises suspicion of pathological condition related to cerebrospinal fluid flow obstruction or cerebral parenchimal pathology. Dyke-Davidoff-Masson syndrome is a rare condition characterized by cerebral hemiatrophy, calvarial thickening, skull and facial asymmetry, contralateral hemiparesis, cognitive impairment and seizures. Congenital and acquired types are recognized and have been described, mainly in late childhood, adolescence and adult ages. We describe a female infant with prenatal diagnosis of unilateral left ventriculomegaly in which early brain MRI and contrast enhanced-MRI angiography, showed cerebral left hemiatrophy associated with reduced caliber of the left middle cerebral artery revealing the characteristic findings of the Dyke-Davidoff-Masson syndrome. Prenatal imaging, cerebral vascular anomaly responsible for the cerebral hemiatrophy and the early clinical evolution have never been described before in such a young child and complete the acquired clinical descriptions in older children. Differential diagnosis, genetic investigations, neurophysiologic assessments, short term clinical and developmental follow up are described. Dyke-Davidoff-Masson syndrome must be ruled out in differential diagnosis of fetal unilateral ventriculomegaly. Early clinical assessment, differential diagnosis and cerebral imaging including cerebral MRI angiography allow the clinicians to diagnose also in early infancy this rare condition.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_002',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 9}]}],\n",
       "   'context': 'Acquired cerebral hemiatrophy: Dyke-Davidoff-Masson Syndrome - a case report. A rare syndrome, Dyke-Davidoff-Masson Syndrome (DDMS), with a diagnostic conundrum, and the way it was solved is presented. A 13-year-old boy presented with recurrent seizures for the past 10 years. He had been treated with anticonvulsant medication which was satisfactory at first but later the seizures recurred. Recently, the frequency of the seizures increased with preictal dizziness and postictal drowsiness. Physical examination revealed mild left hemiparesis and left deviated gait irregularity. He was mentally alert but had not achieved all the developmental milestones as compared to normal child of his age. CT and MRI scan of the head showed hemiatrophic cerebral parenchyma with prominent sulci and encephalomalacia. 24-hour intensive video EEG monitoring revealed suppression of alpha rhythm and local slow wave activity on the side of the atrophic hemisphere. PET-CT showed highly functional left cerebral hemisphere and less functional right cerebral hemisphere. The patient underwent functional hemispherectomy under neurophysiological monitoring and the nonfunctional brain tissues were resected while selectively preserving the functional areas detected by fMRI and PET-CT scan. During follow up, the patient was seizure free as well as without difficulties in performing his daily activities and communications. Functional hemispherectomy for DDMS patient has a good prognosis.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_003',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 275}]}],\n",
       "   'context': 'Dyke-Davidoff-Masson syndrome. A 14-month-old male child presented with recurrent generalized seizures, spastic hemiplegia, microcephaly and had developmental delay in motor and speech domains. CT of the brain revealed characteristic features diagnostic of infantile type of cerebral hemiatrophy or Dyke-Davidoff-Masson syndrome.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_004',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 1367}]}],\n",
       "   'context': 'Spontaneous resolution of invasive cerebral aspergillosis following partial resection in a medically untreated infant. Invasive craniocerebral aspergillosis, often encountered in an immunocompromised setting, is almost uniformly fatal despite radical surgical and medical management, and is frequently a necropsy finding. The authors report a unique, self-resolving clinical course of this aggressive infection in a 10-month-old infant. The infant was brought to the emergency services in altered sensorium with a 1-week history of left-sided hemiparesis, excessive irritability, and vomiting. An MRI study of the brain revealed multiple, heterogeneously enhancing lesions in the right cerebral hemisphere with mass effect. The largest lesion in the frontotemporal cortical and subcortical regions was decompressed on an emergent basis. Histopathological findings were suggestive of invasive aspergillosis, although there was no evidence of the infection in the lungs or paranasal sinuses. Computed tomography-guided aspiration of the remaining lesions and follow-up antifungal therapy were recommended. The parents, however, requested discharge without further treatment. The child was seen at a follow-up visit 3 years later without having received any antifungal treatment. Imaging showed resolution of the infection and features of Dyke-Davidoff-Masson syndrome (cerebral hemiatrophy). This report of invasive cerebral aspergillosis resolving without medical therapy is the first of its kind. Its clinicoradiological aspects are discussed in light of previously reported cases.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_005',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 201}]}],\n",
       "   'context': 'Temporo-spatial analyses define epileptogenic and functional zones in a case of Dyke-Davidoff-Masson syndrome. Dyke-Davidoff-Masson syndrome (DDMS) is a rare epilepsy syndrome that is characterized by cerebral hemiatrophy, homolateral skull hyperplasia, hyperpneumatization of the paranasal sinuses, seizures with or without mental retardation, and contralateral hemiparesis. We describe a case of DDMS in a 40-year-old female who had complex partial seizures with occasional secondary generalization since the age of 4 years. Her seizure frequency was 10-20 seizures/month even though she took four antiepileptic drugs. We applied magnetic resonance imaging (MRI), positron emission tomography (PET), functional MRI, and invasive electroencephalography (EEG) to define her epileptogenic and functional zones. Brain MRI showed prominent atrophy in the left frontal dorsal and lateral regions and mild atrophy of the left superior temporal gyrus and left parietal gyri. Interictal PET revealed decreased glucose metabolism in the atrophic regions. Functional MRI demonstrated that the inferior frontal and inferior parieto-occipital regions of the right hemisphere were activated by language testing. Invasive EEG revealed that the left lateral temporal lobe was the sole source of her seizures. Our results imply that the \"metabolic border zone\" rather than the atrophic region plays an important role in seizure activity, and that reorganization of functional zones occur after cerebral damage early in life.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_006',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 190}]}],\n",
       "   'context': 'Right cerebral hemiatrophy: neurocognitive and electroclinical features. The purpose of this study was to retrospectively evaluate the cognitive and electroclinical characteristics of right cerebral hemiatrophy (Dyke-Davidoff-Masson syndrome [DDMS]). Cognitive assessments with a particular emphasis on visuospatial functions, electroclinical features, and neuroimaging characteristics were analyzed for five patients with a clinically and neuroradiologically confirmed diagnosis of right-sided DDMS. Intelligence tests revealed mental retardation in all but one. Neuropsychological assessments demonstrated consistent impairments in tasks that have a spatial component (spatial processing and orientation discrimination), whereas attention, executive functions and verbal memory domains were variably impaired. Electroclinically, the main seizure types were simple partial motor, complex partial, and secondarily generalized seizures. Interictal EEG delineated lower amplitudes and slow background activity in the affected hemisphere. Overall, the cognitive performance of patients with DDMS encompasses a broad spectrum of impairments affecting multiple domains. Our findings support the concept that dorsal visual pathways responsible for spatial processing may be lateralized to the right hemisphere.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_007',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 129}]}],\n",
       "   'context': 'Treatment-refractory schizoaffective disorder in a patient with dyke-davidoff-masson syndrome. Dyke-Davidoff-Masson syndrome, or cerebral hemiatrophy, is a pre- or perinatally acquired entity characterized by predominantly neurologic symptoms, such as seizures, facial asymmetry, contralateral hemiplegia, and mental retardation. Psychiatric symptoms are rarely reported. We report the first case of left cerebral hemiatrophy and a late onset of treatment-resistant schizoaffective disorder after a stressful life event. The patient finally responded well to clozapine. The clinical history and results from structural neuroimaging are highlighted to discuss the possible developmental bias for psychotic disorders.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_008',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 208}]}],\n",
       "   'context': 'Dyke-Davidoff-Masson syndrome. Dyke Davidoff Masson syndrome (DDMS) is characterized by seizures, facial asymmetry, contralateral hemiplegia and mental retardation. The characteristic radiologic features are cerebral hemiatrophy with homolateral hypertrophy of the skull and sinuses. We report a case of DDMS in an 18-month-old girl who presented with right sided focal seizures, hemiparesis of the same side, and delayed milestones.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_009',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'Cerebral hemiatrophy', 'answer_start': 0}]}],\n",
       "   'context': 'Cerebral hemiatrophy (Dyke-Davidoff-Masson syndrome) in childhood: clinicoradiological analysis of 19 cases. BACKGROUND: The purpose of this study was to emphasize the clinical and imaging findings of 19 child cases of cerebral hemiatrophy. METHODS: A total of 11 male and eight female patients underwent assessment with computed tomography and magnetic resonance imaging. The patients ranged from 1 to 17 years in age. The evaluated parameters were: location of the lesions, midline structural shift effect, ipsilateral calvarial and parenchymal changes. RESULTS: Left cerebral hemiatrophy was seen in 14 of the cases while right cerebral hemiatrophy was observed in five cases. Unilateral calvarial thickening was seen in 11 cases, hyperpneumatization of paranasal sinuses in five, and hypoplasia of the middle frontal cranial fossa in three patients. Cerebral peduncle atrophy was noted in seven cases. In total, 11 patients had thalamic atrophy and lentiform nucleus hypoplasia. In one case, cerebral hemiatrophy was associated with ipsilateral large schizencephalic cleft and absence of the septum pellucidum, whereas in another case, there was diffuse cerebellar atrophy associated with cerebral hemiatrophy. CONCLUSION: Computed tomography and, in particular, magnetic resonance imaging are the procedures of choice with respect to assessment of the etiology and extent of cerebral parenchymal involvement in cerebral hemiatrophy.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_010',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 146}]}],\n",
       "   'context': 'Anesthetic approach to the Dyke-Davidoff-Masson syndrome--a case report. The so-called Dyke-Davidoff-Masson syndrome (DDMS) is a rare disorder of cerebral hemiatrophy. The clinical presentation may consist of facial asymmetry, contralateral atrophy (including the trunk, and the extremities) and hemiparesis, speech difficulties, mental retardation, and epilepsy. Because it involves multiple systems, especially problem of the airway, occult myopathy, and seizure disorder, anesthesia for such a patient is a challenge to any anesthesiologist. However, there are no clinical reports which concern the anesthetic management of such patients in the literature. We herein report a 5-year-old girl, a sufferer of DDMS, scheduled for burr trephination. The successful anesthetic management is brought forward with highlights of inference from the experience.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_011',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 130}]}],\n",
       "   'context': 'Left hemisphere and male sex dominance of cerebral hemiatrophy (Dyke-Davidoff-Masson Syndrome). Although radiological findings of cerebral hemiatrophy (Dyke-Davidoff-Masson Syndrome) are well known, there is no systematic study about the gender and the affected side in this syndrome. Brain images in 26 patients (mean aged 11) with cerebral hemiatrophy were retrospectively reviewed. Nineteen patients (73.5%) were male and seven patients (26.5%) were female. Left hemisphere involvement was seen in 18 patients (69.2%) and right hemisphere involvement was seen in eight patients (30.8%). We conclude that male gender and left side involvement are frequent in cerebral hemiatrophy disease.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_012',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 415}]}],\n",
       "   'context': 'Dyke-Davidoff-Masson syndrome manifested by seizure in late childhood: a case report. The patient was a 19-year-old woman who presented with hemiatrophy and diminished superficial sensation on the left side of her body including her face. She had a past history of tonic-clonic seizures accompanied by left hemiparesis in late childhood. Brain CT demonstrated dilatation of the frontal sinus, calvarial thickening, cerebral hemiatrophy and dilatation of the lateral ventricle on the right side. Brain MRI showed atrophy of the right cerebrum and midbrain and dilatation of the lateral ventricle on T1-weighted images, as well as a high signal intensity area from the parietal to the occipital lobe on T2-weighted images. These findings are suggestive of an episode that may have caused a transient ischemia through the right cerebral hemisphere after the intrauterine period.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_013',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'Cerebral hemiatrophy', 'answer_start': 139}]}],\n",
       "   'context': 'MR and CT imaging in the Dyke-Davidoff-Masson syndrome. Report of three cases and contribution to pathogenesis and differential diagnosis. Cerebral hemiatrophy or Dyke-Davidoff-Masson syndrome is a condition characterized by seizures, facial asymmetry, contralateral hemiplegia or hemiparesis, and mental retardation. These findings are due to cerebral injury that may occur early in life or in utero. The radiological features are unilateral loss of cerebral volume and associated compensatory bone alterations in the calvarium, like thickening, hyperpneumatization of the paranasal sinuses and mastoid cells and elevation of the petrous ridge. The authors describe three cases. Classical findings of the syndrome are present in variable degrees according to the extent of the brain injury. Pathogenesis is commented.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_014',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 367}]}],\n",
       "   'context': 'Growing skull fracture in a patient with cerebral hemiatrophy. A growing skull fracture or leptomeningeal cyst most commonly occurs in children under the age of 3 years, and is extremely rare in adults. The reason for a growing skull fracture is usually a dural tear in association with the fracture. This paper presents an 18-year-old mentally retarded patient with cerebral hemiatrophy (Dyke-Davidoff-Masson syndrome) associated with a growing skull fracture in the ipsilateral hemicranium, in whom not only a dural tear but also the ipsilaterally displaced and dilated lateral ventricle due to the original disease apparently contributed to the development of growing skull fracture.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_015',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 94}]}],\n",
       "   'context': 'MR of craniocerebral hemiatrophy. The magnetic resonance (MR) findings of three patients with cerebral hemiatrophy, the so-called Dyke-Davidoff-Masson syndrome, which is characterized by variable degrees of unilateral loss of cerebral volume and compensatory changes of the calvarium are presented. The condition was due to middle cerebral artery stroke in all patients. The pathologic alterations of cerebral tissue and the brainstem were reflected in detail on the MR studies. MR findings in addition to the primary vascular insult included prominence of the cortical sulci and perimesencephalic cistern in one subject with acquired infarction, but an absence of such generalized sulcal prominence in two cases of congenital infantile paralysis. Otherwise the secondary ipsilateral pathologic observations were quite similar in the patients with congenital and acquired ischemic disease and encompassed a unilaterally small cerebral hemisphere together with ipsilateral diploic calvarial expansion, elevation of the petrous bone and orbital roof, and hypoplasia/atrophy of the cerebral peduncle. Although computed tomography (CT) and MR are complimentary, it is felt that MR represents the imaging procedure of choice with respect to the assessment of the etiology and extent of cerebral parenchymal involvement in patients presenting with a clinical combination of congenital or early onset of seizures, hemiparesis/plegia, and/or craniofacial asymmetry.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_016',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 272}]}],\n",
       "   'context': 'Dyke-Davidoff-Masson syndrome: Classical imaging findings. A 15-year-old female presented with seizures, right-sided hemiparesis, hemiatrophy of the right side of the body and mental retardation. MRI brain revealed characteristic features diagnostic of congenital type of cerebral hemiatrophy or Dyke-Davidoff-Masson syndrome.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_017',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 1367}]}],\n",
       "   'context': 'Spontaneous resolution of invasive cerebral aspergillosis following partial resection in a medically untreated infant. Invasive craniocerebral aspergillosis, often encountered in an immunocompromised setting, is almost uniformly fatal despite radical surgical and medical management, and is frequently a necropsy finding. The authors report a unique, self-resolving clinical course of this aggressive infection in a 10-month-old infant. The infant was brought to the emergency services in altered sensorium with a 1-week history of left-sided hemiparesis, excessive irritability, and vomiting. An MRI study of the brain revealed multiple, heterogeneously enhancing lesions in the right cerebral hemisphere with mass effect. The largest lesion in the frontotemporal cortical and subcortical regions was decompressed on an emergent basis. Histopathological findings were suggestive of invasive aspergillosis, although there was no evidence of the infection in the lungs or paranasal sinuses. Computed tomography-guided aspiration of the remaining lesions and follow-up antifungal therapy were recommended. The parents, however, requested discharge without further treatment. The child was seen at a follow-up visit 3 years later without having received any antifungal treatment. Imaging showed resolution of the infection and features of Dyke-Davidoff-Masson syndrome (cerebral hemiatrophy). This report of invasive cerebral aspergillosis resolving without medical therapy is the first of its kind. Its clinicoradiological aspects are discussed in light of previously reported cases.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_018',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 275}]}],\n",
       "   'context': 'Dyke-Davidoff-Masson syndrome. A 14-month-old male child presented with recurrent generalized seizures, spastic hemiplegia, microcephaly and had developmental delay in motor and speech domains. CT of the brain revealed characteristic features diagnostic of infantile type of cerebral hemiatrophy or Dyke-Davidoff-Masson syndrome.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_019',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 214}]}],\n",
       "   'context': 'Dyke-Davidoff-Masson syndrome: a clinicoradiological amalgam. Dyke-Davidoff-Masson syndrome is a relatively rare syndrome with its typical clinical and radiological features including facial asymmetry, hemiplegia, cerebral hemiatrophy, mental retardation with calvarial thickening, hypertrophy of sinuses and elevated petrous ridge on imaging. We present here a case of congenital type Dyke-Davidoff-Masson syndrome with some additional features in the form of microcephaly, hypospadias and pachygyria.'},\n",
       "  {'qas': [{'id': '55032e65e9bde69634000034_020',\n",
       "     'question': 'What is the characteristic feature of the Dyke-Davidoff-Masson syndrome.',\n",
       "     'answers': [{'text': 'cerebral hemiatrophy', 'answer_start': 190}]}],\n",
       "   'context': 'Right cerebral hemiatrophy: neurocognitive and electroclinical features. The purpose of this study was to retrospectively evaluate the cognitive and electroclinical characteristics of right cerebral hemiatrophy (Dyke-Davidoff-Masson syndrome [DDMS]). Cognitive assessments with a particular emphasis on visuospatial functions, electroclinical features, and neuroimaging characteristics were analyzed for five patients with a clinically and neuroradiologically confirmed diagnosis of right-sided DDMS. Intelligence tests revealed mental retardation in all but one. Neuropsychological assessments demonstrated consistent impairments in tasks that have a spatial component (spatial processing and orientation discrimination), whereas attention, executive functions and verbal memory domains were variably impaired. Electroclinically, the main seizure types were simple partial motor, complex partial, and secondarily generalized seizures. Interictal EEG delineated lower amplitudes and slow background activity in the affected hemisphere. Overall, the cognitive performance of patients with DDMS encompasses a broad spectrum of impairments affecting multiple domains. Our findings support the concept that dorsal visual pathways responsible for spatial processing may be lateralized to the right hemisphere.'},\n",
       "  {'qas': [{'id': '5717d86029809bbe7a000003_001',\n",
       "     'question': 'Which gene is involved in the development of Barth syndrome?',\n",
       "     'answers': [{'text': 'tafazzin (TAZ) gene', 'answer_start': 143}]}],\n",
       "   'context': \"Barth syndrome: cellular compensation of mitochondrial dysfunction and apoptosis inhibition due to changes in cardiolipin remodeling linked to tafazzin (TAZ) gene mutation. Cardiolipin is a mitochondrion-specific phospholipid that stabilizes the assembly of respiratory chain complexes, favoring full-yield operation. It also mediates key steps in apoptosis. In Barth syndrome, an X chromosome-linked cardiomyopathy caused by tafazzin mutations, cardiolipins display acyl chain modifications and are present at abnormally low concentrations, whereas monolysocardiolipin accumulates. Using immortalized lymphoblasts from Barth syndrome patients, we showed that the production of abnormal cardiolipin led to mitochondrial alterations. Indeed, the lack of normal cardiolipin led to changes in electron transport chain stability, resulting in cellular defects. We found a destabilization of the supercomplex (respirasome) I+III2+IVn but also decreased amounts of individual complexes I and IV and supercomplexes I+III and III+IV. No changes were observed in the amounts of individual complex III and complex II. We also found decreased levels of complex V. This complex is not part of the supercomplex suggesting that cardiolipin is required not only for the association/stabilization of the complexes into supercomplexes but also for the modulation of the amount of individual respiratory chain complexes. However, these alterations were compensated by an increase in mitochondrial mass, as demonstrated by electron microscopy and measurements of citrate synthase activity. We suggest that this compensatory increase in mitochondrial content prevents a decrease in mitochondrial respiration and ATP synthesis in the cells. We also show, by extensive flow cytometry analysis, that the type II apoptosis pathway was blocked at the mitochondrial level and that the mitochondria of patients with Barth syndrome cannot bind active caspase-8. Signal transduction is thus blocked before any mitochondrial event can occur. Remarkably, basal levels of superoxide anion production were slightly higher in patients' cells than in control cells as previously evidenced via an increased protein carbonylation in the taz1Δ mutant in the yeast. This may be deleterious to cells in the long term. The consequences of mitochondrial dysfunction and alterations to apoptosis signal transduction are considered in light of the potential for the development of future treatments.\"},\n",
       "  {'qas': [{'id': '5717d86029809bbe7a000003_002',\n",
       "     'question': 'Which gene is involved in the development of Barth syndrome?',\n",
       "     'answers': [{'text': 'tafazzin (TAZ) gene', 'answer_start': 143}]}],\n",
       "   'context': \"Barth syndrome: cellular compensation of mitochondrial dysfunction and apoptosis inhibition due to changes in cardiolipin remodeling linked to tafazzin (TAZ) gene mutation. Cardiolipin is a mitochondrion-specific phospholipid that stabilizes the assembly of respiratory chain complexes, favoring full-yield operation. It also mediates key steps in apoptosis. In Barth syndrome, an X chromosome-linked cardiomyopathy caused by tafazzin mutations, cardiolipins display acyl chain modifications and are present at abnormally low concentrations, whereas monolysocardiolipin accumulates. Using immortalized lymphoblasts from Barth syndrome patients, we showed that the production of abnormal cardiolipin led to mitochondrial alterations. Indeed, the lack of normal cardiolipin led to changes in electron transport chain stability, resulting in cellular defects. We found a destabilization of the supercomplex (respirasome) I+III2+IVn but also decreased amounts of individual complexes I and IV and supercomplexes I+III and III+IV. No changes were observed in the amounts of individual complex III and complex II. We also found decreased levels of complex V. This complex is not part of the supercomplex suggesting that cardiolipin is required not only for the association/stabilization of the complexes into supercomplexes but also for the modulation of the amount of individual respiratory chain complexes. However, these alterations were compensated by an increase in mitochondrial mass, as demonstrated by electron microscopy and measurements of citrate synthase activity. We suggest that this compensatory increase in mitochondrial content prevents a decrease in mitochondrial respiration and ATP synthesis in the cells. We also show, by extensive flow cytometry analysis, that the type II apoptosis pathway was blocked at the mitochondrial level and that the mitochondria of patients with Barth syndrome cannot bind active caspase-8. Signal transduction is thus blocked before any mitochondrial event can occur. Remarkably, basal levels of superoxide anion production were slightly higher in patients' cells than in control cells as previously evidenced via an increased protein carbonylation in the taz1Δ mutant in the yeast. This may be deleterious to cells in the long term. The consequences of mitochondrial dysfunction and alterations to apoptosis signal transduction are considered in light of the potential for the development of future treatments.\"},\n",
       "  {'qas': [{'id': '5717d86029809bbe7a000003_003',\n",
       "     'question': 'Which gene is involved in the development of Barth syndrome?',\n",
       "     'answers': [{'text': 'tafazzin (TAZ) gene', 'answer_start': 237}]}],\n",
       "   'context': 'The cellular and molecular mechanisms for neutropenia in Barth syndrome. Barth syndrome (BTHS), a rare, X-linked, recessive disease, is characterized by neutropenia and cardiomyopathy. BTHS is caused by loss-of-function mutations of the tafazzin (TAZ) gene. We developed a model of BTHS by transfecting human HL60 myeloid progenitor cells with TAZ-specific shRNAs. Results demonstrate a significant downregulation in TAZ expression, mimicking the effects of naturally occurring truncation mutations in TAZ. Flow cytometry analyses of cells with TAZ-specific, but not scrambled, shRNAs demonstrate nearly twofold increase in the proportion of annexin V-positive cells and significantly increased dissipation of mitochondrial membrane potential as determined by DIOC6 staining. Transfection of TAZ-specific shRNA had similar effects in U937 myeloid cells but not in lymphoid cell lines. Further studies in HL60 myeloid progenitor cells revealed aberrant release of cytochrome c from mitochondria and significantly elevated levels of activated caspase-3 in response to TAZ knockdown. Treatment with caspase-specific inhibitor zVAD-fmk resulted in substantially reduced apoptosis to near-normal levels. These data suggest that neutropenia in BTHS is attributable to increased dissipation of mitochondrial membrane potential, aberrant release of cytochrome c, activation of caspase-3, and accelerated apoptosis of myeloid progenitor cells, and that this defect can be partially restored in vitro by treatment with caspase-specific inhibitors.'},\n",
       "  {'qas': [{'id': '51651e24298dcd4e51000054_001',\n",
       "     'question': 'How many genera comprise the Flaviviridae family?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 306}]}],\n",
       "   'context': '5-acetyl-2-arylbenzimidazoles as antiviral agents. Part 4. Within a project aimed at discovering new Flaviviridae inhibitors, new variously substituted 2-phenylbenzimidazoles were synthesized and evaluated in cell-based assays for cytotoxicity and antiviral activity against viruses representatives of the three genera of the Flaviviridae family, i.e.: Pestivirus (BVDV), Flavivirus (YFV) and Hepacivirus (HCV). Title compounds were also tested against RNA viruses representative of other single-stranded, positive-sense (ssRNA(+)) negative-sense (RNA(-)), or double-stranded (dsRNA) genomes, as well as against representatives of two DNA virus families. Nine compounds showed activity against BVDV (EC(50) = 0.8-8.0 μM), compound 31 being the most potent (EC(50) = 0.80 μM) and selective (SI = CC(50)/EC(50) = >100). When tested in an HCV replicon assay, compound 31 resulted again the most potent, displaying an EC(50) value of 1.11 μM and an SI of 100. Besides inhibiting BVDV, two compounds (35 and 38) showed a moderate activity also against YFV (EC(50) = 13 μM). Interestingly, 35 was moderately active also against RSV (EC(50) = 25 μM).'},\n",
       "  {'qas': [{'id': '51651e24298dcd4e51000054_002',\n",
       "     'question': 'How many genera comprise the Flaviviridae family?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 330}]}],\n",
       "   'context': 'Styrylbenzimidazoles. Synthesis and biological activity - part 3. As a follow up of an anti-Flaviviridae project, a new series of variously substituted 2-styryl-benzimidazoles were synthesized and tested in vitro for biological activity. Compounds were tested in cell-based assays against viruses representative of: i) two of the three genera of the Flaviviridae family, i.e. Pestiviruses and Flaviviruses; ii) other RNA virus families, such as Retroviridae, Picornaviridae, Paramyxoviridae, Rhabdoviridae and Reoviridae; iii) two DNA virus families (Herpesviridae and Poxviridae) as well as for cytotoxicity tests, run in parallel with antiviral assays,against MDBK, BHK and Vero 76 cells. In the series examined, new leads emerged against BVDV, CVB-2 and RSV. Compounds 11, 12, 17, 18, 24, 31 exhibited anti-BVDV activity in the concentration range 1.7-16 microM; among them, compound 17 was the most active, with an EC(50) = 1.7 microM. Compounds 18 and 21 were equally active against CVB-2, with EC(50) values of 7 - 8 microM, while the derivative 30 was active against RSV with EC(50)= 1 microM and represents a new lead compound.'},\n",
       "  {'qas': [{'id': '51651e24298dcd4e51000054_003',\n",
       "     'question': 'How many genera comprise the Flaviviridae family?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 360}]}],\n",
       "   'context': \"2-Arylbenzimidazoles as antiviral and antiproliferative agents--Part 2. In prosecution of an anti-Flaviviridae project a new series of variously substituted 2-diphenyl-benzimidazoles were synthesized and tested in vitro for antiviral and antiproliferative activities. Compounds were tested in cell-based assays against viruses representative of: i) two of the three genera of the Flaviviridae family, i.e. Flaviviruses and Pestiviruses; ii) other RNA virus families, such as Retroviridae, Picornaviridae, Paramyxoviridae, Rhabdoviridae and Reoviridae; iii) two DNA virus families (Herpesviridae and Poxviridae). The 5-Acetyl-2-(4'-nitrobiphenyl-4-yl)-1H-benzimidazole (24) emerged as potent active lead compound against Yellow Fever Virus (a Flavivirus) (EC(50) = 0.5 microM) and CVB-2 at 1 microM and was not cytotoxic, whereas the other title benzimidazoles showed no antiviral activity at concentrations not cytotoxic for the resting cell monolayers. Among the examined series, the most cytotoxic derivatives (11,12,14,16,18,19,20,21,23,25-30) against mock-infected MT-4 cells (CC50 < 8.0 microM) were evaluated against a panel of human cell lines derived from haematological and solid tumours,using 6-mercaptopurine (6-MP) and etoposide as reference drugs. In particular, compounds 26 and 28 showed a similar potency of 6-MP and etoposide.\"},\n",
       "  {'qas': [{'id': '51651e24298dcd4e51000054_004',\n",
       "     'question': 'How many genera comprise the Flaviviridae family?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 141}]}],\n",
       "   'context': 'Antiviral activity of benzimidazole derivatives. I. Antiviral activity of 1-substituted-2-[(benzotriazol-1/2-yl)methyl]benzimidazoles. Forty-three 2-[(benzotriazol-1/2-yl)methyl]benzimidazoles, bearing either linear (dialkylamino)alkyl- or bulkier (quinolizidin-1-yl)alkyl moieties at position 1, were evaluated in cell-based assays for cytotoxicity and antiviral activity against viruses representative of two of the three genera of the Flaviviridae family, i.e. Flaviviruses (Yellow Fever Virus (YFV)) and Pestiviruses (Bovine Viral Diarrhoea Virus (BVDV)), as Hepaciviruses can hardly be used in routine cell-based assays. Compounds were also tested against representatives of other virus families. Among ssRNA+ viruses were a retrovirus (Human Immunodeficiency Virus type 1 (HIV-1)), two picornaviruses (Coxsackie Virus type B2 (CVB2), and Poliovirus type-1, Sabin strain (Sb-1)); among ssRNA- viruses were a Paramyxoviridae (Respiratory Syncytial Virus (RSV)) and a Rhabdoviridae (Vesicular Stomatitis Virus (VSV)) representative. Among double-stranded RNA (dsRNA) viruses was a Reoviridae representative (Reo-1). Two representatives of DNA virus families were also included: Herpes Simplex type 1, (HSV-1; Herpesviridae) and Vaccinia Virus (VV; Poxviridae). Most compounds exhibited potent activity against RSV, with EC(50) values as low as 20 nM. Moreover, some compounds, in particular when bearing a (quinolizidin-1-yl)alkyl residue, were also moderately active against BVDV, YFV, and CVB2.'},\n",
       "  {'qas': [{'id': '51651e24298dcd4e51000054_005',\n",
       "     'question': 'How many genera comprise the Flaviviridae family?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 418}]}],\n",
       "   'context': 'Antiviral activity of benzimidazole derivatives. I. Antiviral activity of 1-substituted-2-[(benzotriazol-1/2-yl)methyl]benzimidazoles. Forty-three 2-[(benzotriazol-1/2-yl)methyl]benzimidazoles, bearing either linear (dialkylamino)alkyl- or bulkier (quinolizidin-1-yl)alkyl moieties at position 1, were evaluated in cell-based assays for cytotoxicity and antiviral activity against viruses representative of two of the three genera of the Flaviviridae family, i.e. Flaviviruses (Yellow Fever Virus (YFV)) and Pestiviruses (Bovine Viral Diarrhoea Virus (BVDV)), as Hepaciviruses can hardly be used in routine cell-based assays. Compounds were also tested against representatives of other virus families. Among ssRNA+ viruses were a retrovirus (Human Immunodeficiency Virus type 1 (HIV-1)), two picornaviruses (Coxsackie Virus type B2 (CVB2), and Poliovirus type-1, Sabin strain (Sb-1)); among ssRNA- viruses were a Paramyxoviridae (Respiratory Syncytial Virus (RSV)) and a Rhabdoviridae (Vesicular Stomatitis Virus (VSV)) representative. Among double-stranded RNA (dsRNA) viruses was a Reoviridae representative (Reo-1). Two representatives of DNA virus families were also included: Herpes Simplex type 1, (HSV-1; Herpesviridae) and Vaccinia Virus (VV; Poxviridae). Most compounds exhibited potent activity against RSV, with EC(50) values as low as 20 nM. Moreover, some compounds, in particular when bearing a (quinolizidin-1-yl)alkyl residue, were also moderately active against BVDV, YFV, and CVB2.'},\n",
       "  {'qas': [{'id': '51651e24298dcd4e51000054_006',\n",
       "     'question': 'How many genera comprise the Flaviviridae family?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 524}]}],\n",
       "   'context': '2-Arylbenzimidazoles as antiviral and antiproliferative agents. Part 1. Being involved in an anti-Flaviviridae Project, and because of the role played by benzimidazole derivatives as promising inhibitors of the HCV helicase and RNA polymerase, as well as of the Zn finger transcription factor, we synthesized a new series of 2-arylbenzimidazoles and evaluated them for antiviral activity, as well as for antiproliferative activity. Compounds were tested in cell-based assays against viruses representative of: i) two of the three genera of the Flaviviridae family, i.e. Flaviviruses and Pestiviruses; ii) other RNA virus families, such as Retroviridae, Picornaviridae, Paramyxoviridae, Rhabdoviridae and Reoviridae; iii) two DNA virus families (Herpesviridae and Poxviridae). Compounds 15, 28 and 29 resulted moderately active only against Yellow Fever Virus (a Flavivirus) (range 6-27 microM), whereas none of the title benzimidazoles showed any antiviral activity at concentrations not cytotoxic for the resting cell monolayers. Compounds were also tested for antiproliferative activity against a panel of exponentially growing cell lines derived from human haematological and solid tumors. Several new benzimidazoles turned out active. Among them, compound 27 was the most potent against human haematologic and solid tumor cells and turned out to be as potent as Etoposide and more potent than 6-mercaptopurine (6-MP), used as reference antitumor agents.'},\n",
       "  {'qas': [{'id': '51651e24298dcd4e51000054_007',\n",
       "     'question': 'How many genera comprise the Flaviviridae family?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 120}]}],\n",
       "   'context': 'Pestivirus virion morphogenesis in the absence of uncleaved nonstructural protein 2-3. The family Flaviviridae contains three genera of positive-strand RNA viruses, namely, Flavivirus, Hepacivirus (e.g., hepatitis C virus [HCV]), and Pestivirus. Pestiviruses, like bovine viral diarrhea virus (BVDV), bear a striking degree of similarity to HCV concerning polyprotein organization, processing, and function. Along this line, in both systems, release of nonstructural protein 3 (NS3) is essential for viral RNA replication. However, both viruses differ significantly with respect to processing efficiency at the NS2/3 cleavage site and abundance as well as functional relevance of uncleaved NS2-3. In BVDV-infected cells, significant amounts of NS2-3 accumulate at late time points postinfection and play an essential but ill-defined role in the production of infectious virions. In contrast, complete cleavage of the HCV NS2-3 counterpart has been reported, and unprocessed NS2-3 is not required throughout the life cycle of HCV, at least in cell culture. Here we describe the selection and characterization of the first pestiviral genome with the capability to complete productive infection in the absence of uncleaved NS2-3. Despite the insertion of a ubiquitin gene or an internal ribosomal entry site between the NS2 and NS3 coding sequences, the selected chimeric BVDV-1 genomes gave rise to infectious virus progeny. In this context, a mutation in the N-terminal third of NS2 was identified as a critical determinant for efficient production of infectious virions in the absence of uncleaved NS2-3. These findings challenge a previously accepted dogma for pestivirus replication and provide new implications for virion morphogenesis of pestiviruses and HCV.'},\n",
       "  {'qas': [{'id': '52b2e498f828ad283c000010_001',\n",
       "     'question': 'Mutation of which gene is associated with Achondroplasia?',\n",
       "     'answers': [{'text': 'fibroblast growth factor receptor 3 (FGFR3)',\n",
       "       'answer_start': 750}]}],\n",
       "   'context': 'Hypochondroplasia due to FGFR3 gene mutation (N540K) and mosaic form of Down syndrome in the same patient. The simultaneous presence of Down syndrome and achondroplasia has rarely been reported in the literature, and our search revealed only six patients with such an association. We are reporting the first case of a patient with Down syndrome and hypochondroplasia. In this patient, Down syndrome was clinically recognised and confirmed by the cytogenetic finding of mosaic karyotype (47,XX,+21/46,XX) shortly after birth. She was subsequently diagnosed with hypochondroplasia at the age of 6 years when disproportional short stature, stocky habitus and macrocephaly were observed. These phenotypic findings were later confirmed by the presence of fibroblast growth factor receptor 3 (FGFR3) gene mutation N540K. The overlapping common clinical features of Down syndrome and hypochondroplasia resulted in delayed diagnosis of hypochondroplasia in our patient and the associated deleterious effect on her linear growth. Her final height is 126.5 cm, which is -3.76 standard deviations (SD) lower than the median height in patients with Down syndrome, and is under the lower borderline of the adult height range for women with hypochondroplasia.'},\n",
       "  {'qas': [{'id': '52b2e498f828ad283c000010_002',\n",
       "     'question': 'Mutation of which gene is associated with Achondroplasia?',\n",
       "     'answers': [{'text': 'fibroblast growth factor receptor 3 (FGFR3)',\n",
       "       'answer_start': 209}]}],\n",
       "   'context': '[Detection of fibroblast growth factor receptor 3 gene mutation at nucleotide 1138 site in congenita achondroplasia patients]. OBJECTIVE: [corrected] To investigate the mutation at the transmembrane domain of fibroblast growth factor receptor 3 (FGFR3) nucleotide 1138 site for identifying the major pathologic mechanism of achondroplasia (ACH) and to evaluate the efficacy of denaturing gradient gel electrophoresis(DGGE) method for screening the point mutations. METHODS: The genomic DNA from 17 clinically diagnosed ACH patients where analysed by polymerase chain reaction-restriction fragment length polymorphism(PCR-RFLP) with Sfc I and Msp I restriction endonucleases and by PCR-DGGE technique for screening. RESULTS: G to A transition mutation at nucleotide 1138 was detected in 14/17 of the ACH patients as heterozygotes by PCR-RFLP with Sfc I digestion. No 1138 G to C transition was detected by Msp I digestion. All of the 14 samples with G to A mutation were also found to be positive for point mutation by PCR-DGGE. No mutation was detected in 3 negative samples by PCR-RFLP, implying that there was actually no point mutation in this amplified region. CONCLUSION: Nucleotide 1138 in transmembrane domain of FGFR3 gene is the hot point for mutation in ACH and hence its major pathologic cause. PCR-DGGE is a sensitive and reliable technique for point mutation screening, especially for the heterozygotes.'},\n",
       "  {'qas': [{'id': '52b2e498f828ad283c000010_003',\n",
       "     'question': 'Mutation of which gene is associated with Achondroplasia?',\n",
       "     'answers': [{'text': 'fibroblast growth factor receptor 3 (FGFR3)',\n",
       "       'answer_start': 463}]}],\n",
       "   'context': 'A common FGFR3 gene mutation is present in achondroplasia but not in hypochondroplasia. Achondroplasia is the most common type of genetic dwarfism. It is characterized by disproportionate short stature and other skeletal anomalies resulting from a defect in the maturation of the chondrocytes in the growth plate of the cartilage. Recent studies mapped the achondroplasia gene on chromosome region 4p16.3 and identified a common mutation in the gene encoding the fibroblast growth factor receptor 3 (FGFR3). In an analysis of 19 achondroplasia families from a variety of ethnic backgrounds we confirmed the presence of the G380R mutation in 21 of 23 achondroplasia chromosomes studied. In contrast, the G380R mutation was not found in any of the 8 hypochondroplasia chromosomes studied. Furthermore, linkage studies in a 3-generation family with hypochondroplasia show discordant segregation with markers in the 4p16.3 region suggesting that at least some cases of hypochondroplasia are caused by mutations in a gene other than FGFR3.'},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_001',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'small-cell lung cancer', 'answer_start': 166}]}],\n",
       "   'context': '[Lambert-Eaton syndrome. Apropos of 2 cases]. Lambert-Eaton syndrome is a myasthenia-like syndrome of paraneoplastic origin which is often associated with anaplastic small-cell lung cancer. It seems to be an autoimmune disease responsible for a deficit of acetylcholine ejection in the motor end plate. On the occasion of two recent cases, we review the clinical, physiopathological and diagnostic aspects of this paraneoplastic syndrome.'},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_002',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'Small-cell lung cancer', 'answer_start': 262}]}],\n",
       "   'context': 'Lambert-Eaton syndrome with large-cell neuroendocrine carcinoma of the lung. Lambert-Eaton syndrome (LES) is an immune-mediated disorder of the presynaptic neuromuscular junction due to the blocking effect of the voltage-gated calcium channel (VGCC) antibodies. Small-cell lung cancer (SCLC) is the most common cause of LES. We report an unusual case of LES associated with large-cell neuroendocrine carcinoma (LCNEC) of the lung. In this case, clinical symptoms of LES predated the diagnosis of LCNEC by 6 years. After tumor resection, the patient experienced clinical and electrophysiological improvement. In addition, he had a decrease in VGCC antibody titer from 130 to 80 pmol/L. The onset of LES can be prolonged, and tumor surveillance should continue in these cases.'},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_003',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'small-cell lung cancer', 'answer_start': 1160}]}],\n",
       "   'context': '[Difficulties associated with Lambert-Eaton syndrome]. INTRODUCTION: The diagnosis and treatment of the neurological paraneoplastic syndromes associated with lung cancer can pose a challenge both to general physicians and neurologists as well as pulmonologists. CASE REPORT: A 53 year-old heavy smoker presented with a Lambert-Eaton myasthenic syndrome (LEMS). Bronchoscopy was normal but radiological examinations revealed a lymph node in site 4R. The pathological diagnosis after mediastinoscopy was negative. Twenty-five months later, an opacity on chest X-ray led to a biopsy which revealed a squamous cell carcinoma. A lobectomy was performed for a pT2N0M0 lesion. A significant improvement of neurological symptoms was seen. The myasthenic syndrome reappeared 21 months later. A local and general relapse was diagnosed. The patient died 10 months later despite chemotherapy. CONCLUSION: LEMS occurs because of an immunological reaction against voltage-dependent calcium channels. LEMS is generally associated with small cell lung cancer occurring in three percent of cases. However, the case that we report shows the unusual association of LEMS with non small-cell lung cancer and highlights the difficulties associated in the management of this condition.'},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_004',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'small-cell lung cancer', 'answer_start': 949}]}],\n",
       "   'context': '[Atypical paraneoplastic myasthenic syndrome: Lambert-Eaton syndrome or myasthenia?]. We report a case of paraneoplastic myasthenic syndrome with clinical features suggesting Lambert Eaton syndrome but without the electromyographic elements required for diagnosis. Anti-calcium channel antibodies were also lacking. The electromyogram evidenced a block and the Tensilon test was positive. The efficacy of anticholinesterases argued in favor of myasthenia but anti-acetylcholine receptor antibodies were negative. The block was more of a mixed nature, involving both presynaptic transmission as in Lambert Eaton syndrome and post-synaptic transmission as in paraneoplastic myasthenia. The primary tumor was identified as a small-cell neuroendocrine lung carcinoma on mediastinal biopsies obtained directly on CT-scan guided puncture of a mediastinal node. Thoracotomy was thus avoided. The Lambert Eaton syndrome is a paraneoplastic manifestation of small-cell lung cancer in 50% of the cases unlike generalized myasthenia which apparently is never associated with small-cell lung cancer. A mixed paraneoplastic neuro-muscle junction disorder with aspects of each can be exceptionally observed.'},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_005',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'small-cell lung cancer', 'answer_start': 1064}]}],\n",
       "   'context': '[Atypical paraneoplastic myasthenic syndrome: Lambert-Eaton syndrome or myasthenia?]. We report a case of paraneoplastic myasthenic syndrome with clinical features suggesting Lambert Eaton syndrome but without the electromyographic elements required for diagnosis. Anti-calcium channel antibodies were also lacking. The electromyogram evidenced a block and the Tensilon test was positive. The efficacy of anticholinesterases argued in favor of myasthenia but anti-acetylcholine receptor antibodies were negative. The block was more of a mixed nature, involving both presynaptic transmission as in Lambert Eaton syndrome and post-synaptic transmission as in paraneoplastic myasthenia. The primary tumor was identified as a small-cell neuroendocrine lung carcinoma on mediastinal biopsies obtained directly on CT-scan guided puncture of a mediastinal node. Thoracotomy was thus avoided. The Lambert Eaton syndrome is a paraneoplastic manifestation of small-cell lung cancer in 50% of the cases unlike generalized myasthenia which apparently is never associated with small-cell lung cancer. A mixed paraneoplastic neuro-muscle junction disorder with aspects of each can be exceptionally observed.'},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_006',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'small-cell lung cancer', 'answer_start': 120}]}],\n",
       "   'context': 'Inhibition of calcium currents and exocytosis by Lambert-Eaton syndrome antibodies in human lung cancer cells. 1. Human small-cell lung cancer (SCLC) cells are believed to express the antigens responsible for the production of pathological antibodies in the Lambert-Eaton syndrome (LES), a Ca2+ channel disorder in which quantal transmitter release from the motor nerve terminal is impaired. Whole-cell patch-clamp techniques were used to study the voltage-dependent Ca2+ channels expressed by H146 SCLC cells and the effects of LES antibodies on these channels. The types of Ca2+ channels were determined using biophysical properties and pharmacological sensitivity to several antagonists. 2. Whole-cell Ca2+ currents (ICa) in SCLC cells are sensitive to the dihydropyridine (DHP) nicardipine, omega-conotoxin GVIA (omega-CgTX GVIA) and omega-agatoxin IVA (omega-AgTX IVA). Nicardipine at 100 nM and 10 microM reduced ICa by 35 and 45% (n = 38 cells), respectively, while omega-CgTX GVIA (1 microM) inhibited ICa by 32% (n = 31). Application of omega-AgTX IVA at 50 and 100 nM to the cancer cells decreased ICa by 41 and 42%, respectively (n = 22). 3. Measurement of cell membrane capacitance (Cm) revealed that Ca(2+)-dependent exocytosis underlies the secretory activity of SCLC cells. Exocytosis, when induced by step depolarizing pulses and measured by increases in Cm, was markedly inhibited by nicardipine (10 microM) and omega-AgTX IVA (100 nM). In contrast, omega-CgTX GVIA (1 microM) was not as effective in altering increases in Cm. 4. From negative (-80 mV) and depolarized (-40 mV) holding potentials, both peak and plateau ICa were inhibited by the presence of LES antibodies (1 mg ml-1 IgG). LES serum also reduced depolarization-induced increases in Cm by 48% (n = 15). 5. To determine whether the LES antibodies are downregulating a specific type(s) of Ca2+ channel, nicardipine (10 microM), omega-CgTX GVIA (1 microM) or omega-AgTX IVA (100 nM) was applied to tumour cells that had been previously exposed to LES serum for 24 h. The most pronounced change was that omega-AgTX IVA was 38-84% less effective at reducing ICa after the IgG treatment. The effectiveness of nicardipine was diminished by 18% after incubation with the LES antibodies, whereas the omega-CgTX GVIA was seen to be more effective. These results suggest that LES IgG downregulates P-type Ca2+ channels and, possibly, to a lesser extent L-type channels. 6. In view of recent evidence that P-type Ca2+ channels mediate cholinergic transmitter release at the mammalian neuromuscular junction (NMJ), the expression of P-type Ca2+ channels in the SCLC cells and the reactivity of LES IgG with these channels support the hypothesis that P-type Ca2+ channels in these cancer cells may trigger the autoantibody production in this disorder. The antibodies so produced are implicated in the functional impairment of the Ca2+ channels characteristic of LES.'},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_007',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'small-cell lung cancer', 'answer_start': 837}]}],\n",
       "   'context': \"Sequential fluctuating paraneoplastic ocular flutter-opsoclonus-myoclonus syndrome and Lambert-Eaton myasthenic syndrome in small-cell lung cancer. Paraneoplastic cerebellar degeneration may occur in association with Lambert-Eaton myasthenic syndrome (LEMS), but to our knowledge, the co-occurrence of paraneoplastic opsoclonus-myoclonus syndrome and LEMS has not been previously reported. A 67-year-old woman presented with a complex partial seizure and evolving ocular flutter, opsoclonus, myoclonus and 'cerebellar' signs, all of which improved spontaneously within 6 weeks. Approximately 8 weeks after symptom onset, the patient became encephalopathic, she had a further complex partial seizure, and she became areflexic with potentiation of deep tendon reflexes. Radiological, bronchoscopic and histological investigations revealed small-cell lung cancer, and neurophysiological investigations confirmed a diagnosis of LEMS. High-titre anti-P/Q-type voltage-gated calcium-channel antibodies were identified in the serum, which increased as the signs of opsoclonus and myoclonus resolved. The encephalopathy and clinical features of LEMS responded dramatically to chemotherapy and radiotherapy. Spontaneous improvement of paraneoplastic opsoclonus-myoclonus syndrome may occur, and this syndrome may occur in association with LEMS. Antivoltage-gated calcium-channel antibodies are not implicated in the pathogenesis of paraneoplastic opsoclonus-myoclonus syndrome.\"},\n",
       "  {'qas': [{'id': '5147c088d24251bc05000026_008',\n",
       "     'question': 'Which type of lung cancer is the most strongly associated with Lambert-Eaton syndrome?',\n",
       "     'answers': [{'text': 'small-cell lung cancer', 'answer_start': 406}]}],\n",
       "   'context': 'Small-cell lung cancer presenting with Lambert-Eaton myasthenic syndrome and respiratory failure. Lambert-Eaton myasthenic syndrome (LEMS) is a neuromuscular disorder characterized by defective neurotransmitter release at presynaptic terminals. It is caused by an IgG autoantibody reacting against voltage-gated calcium channels. Severe LEMS complicated by ventilatory failure is rare. We report a case of small-cell lung cancer (SCLC) presenting with LEMS and ventilatory failure in a 67-year-old man who initially presented with progressive limb weakness for 6 months and tachypnea with shallow breathing for 1 week. LEMS was diagnosed through electrophysiologic studies. Chest radiography and computerized tomography showed a huge mass lesion over the left anterior and middle mediastinum with an encasement of the left pulmonary artery. Cytologic examination of ultrasound-guided fine needle aspiration disclosed SCLC. Successful treatment in combination with plasma exchange and chemotherapy resulted in dramatic tumor regression and LEMS remission, which were confirmed by chest radiography and electrophysiologic studies. This case suggests that plasma exchange and chemotherapy can be effective in treating SCLC with severe LEMS that produces ventilatory failure.'},\n",
       "  {'qas': [{'id': '517901bc8ed59a060a00003b_001',\n",
       "     'question': 'The antibodies MK-3475 and CT-011 have shown promising results in treating malignancies. Which protein are they targeting?',\n",
       "     'answers': [{'text': 'PD-1', 'answer_start': 433}]}],\n",
       "   'context': 'Anti-PD-1 synergizes with cyclophosphamide to induce potent anti-tumor vaccine effects through novel mechanisms. Programmed death-1 receptor (PD-1) is expressed on T cells following TCR activation. Binding of this receptor to its cognate ligands, programmed death ligand (PDL)-1 and PDL-2, down-regulates signals by the TCR, promoting T-cell anergy and apoptosis, thus leading to immune suppression. Here, we find that using an anti-PD-1 antibody (CT-011) with Treg-cell depletion by low-dose cyclophosphamide (CPM), combined with a tumor vaccine, induces synergistic antigen-specific immune responses and reveals novel activities of each agent in this combination. This strategy led to complete regression of established tumors in a significant percentage of treated animals, with survival prolongation. We show for the first time that combining CT-011 and CPM significantly increases the number of vaccine-induced tumor-infiltrating CD8(+) T cells, with simultaneous decrease in infiltrating Treg cells. Interestingly, we find that CT-011 prolongs Treg-cell inhibition induced by CPM, leading to a sustainable significant synergistic decrease of splenic and tumor-infiltrated Treg cells. Surprisingly, we find that the anti-tumor effect elicited by the combination of CT-011 and CPM is dependent on both CD8(+) and CD4(+) T-cell responses, although the antigen we used is a class I MHC-restricted peptide. Thus, we describe a novel and effective therapeutic approach by combining multiple strategies to target several tumor-mediated immune inhibitory mechanisms.'},\n",
       "  {'qas': [{'id': '517901bc8ed59a060a00003b_002',\n",
       "     'question': 'The antibodies MK-3475 and CT-011 have shown promising results in treating malignancies. Which protein are they targeting?',\n",
       "     'answers': [{'text': 'PD-1', 'answer_start': 0}]}],\n",
       "   'context': 'PD-1 blockade by CT-011, anti-PD-1 antibody, enhances ex vivo T-cell responses to autologous dendritic cell/myeloma fusion vaccine. We have developed a cancer vaccine in which autologous tumor is fused with dendritic cells (DCs) resulting in the presentation of tumor antigens in the context of DC-mediated costimulation. In clinical trials, immunologic responses have been observed, however responses may be muted by inhibitory pathways. The PD1/PDL1 pathway is an important element contributing to tumor-mediated immune suppression. In this study, we demonstrate that myeloma cells and DC/tumor fusions strongly express PD-L1. Compared with a control population of normal volunteers, increased PD-1 expression was observed on T cells isolated from patients with myeloma. It is interesting to note that after autologous transplantation, T-cell expression of PD-1 returned to levels seen in normal controls. We examined the effect of PD-1 blockade on T-cell response to DC/tumor fusions ex vivo. Presence of CT-011, an anti-PD1 antibody, promoted the vaccine-induced T-cell polarization towards an activated phenotype expressing Th1 compared with Th2 cytokines. A concomitant decrease in regulatory T cells and enhanced killing in a cytotoxicity assay was observed. In summary, we demonstrate that PD-1 expression is increased in T cells of patients with active myeloma, and that CT-011 enhances activated T-cell responses after DC/tumor fusion stimulation.'},\n",
       "  {'qas': [{'id': '517901bc8ed59a060a00003b_003',\n",
       "     'question': 'The antibodies MK-3475 and CT-011 have shown promising results in treating malignancies. Which protein are they targeting?',\n",
       "     'answers': [{'text': 'PD-1', 'answer_start': 30}]}],\n",
       "   'context': 'PD-1 blockade by CT-011, anti-PD-1 antibody, enhances ex vivo T-cell responses to autologous dendritic cell/myeloma fusion vaccine. We have developed a cancer vaccine in which autologous tumor is fused with dendritic cells (DCs) resulting in the presentation of tumor antigens in the context of DC-mediated costimulation. In clinical trials, immunologic responses have been observed, however responses may be muted by inhibitory pathways. The PD1/PDL1 pathway is an important element contributing to tumor-mediated immune suppression. In this study, we demonstrate that myeloma cells and DC/tumor fusions strongly express PD-L1. Compared with a control population of normal volunteers, increased PD-1 expression was observed on T cells isolated from patients with myeloma. It is interesting to note that after autologous transplantation, T-cell expression of PD-1 returned to levels seen in normal controls. We examined the effect of PD-1 blockade on T-cell response to DC/tumor fusions ex vivo. Presence of CT-011, an anti-PD1 antibody, promoted the vaccine-induced T-cell polarization towards an activated phenotype expressing Th1 compared with Th2 cytokines. A concomitant decrease in regulatory T cells and enhanced killing in a cytotoxicity assay was observed. In summary, we demonstrate that PD-1 expression is increased in T cells of patients with active myeloma, and that CT-011 enhances activated T-cell responses after DC/tumor fusion stimulation.'},\n",
       "  {'qas': [{'id': '517901bc8ed59a060a00003b_004',\n",
       "     'question': 'The antibodies MK-3475 and CT-011 have shown promising results in treating malignancies. Which protein are they targeting?',\n",
       "     'answers': [{'text': 'PD-1', 'answer_start': 143}]}],\n",
       "   'context': \"The PD-1/PD-L1 axis modulates the natural killer cell versus multiple myeloma effect: a therapeutic target for CT-011, a novel monoclonal anti-PD-1 antibody. T-cell expression of programmed death receptor-1 (PD-1) down-regulates the immune response against malignancy by interacting with cognate ligands (eg, PD-L1) on tumor cells; however, little is known regarding PD-1 and natural killer (NK) cells. NK cells exert cytotoxicity against multiple myeloma (MM), an effect enhanced through novel therapies. We show that NK cells from MM patients express PD-1 whereas normal NK cells do not and confirm PD-L1 on primary MM cells. Engagement of PD-1 with PD-L1 should down-modulate the NK-cell versus MM effect. We demonstrate that CT-011, a novel anti-PD-1 antibody, enhances human NK-cell function against autologous, primary MM cells, seemingly through effects on NK-cell trafficking, immune complex formation with MM cells, and cytotoxicity specifically toward PD-L1(+) MM tumor cells but not normal cells. We show that lenalidomide down-regulates PD-L1 on primary MM cells and may augment CT-011's enhancement of NK-cell function against MM. We demonstrate a role for the PD-1/PD-L1 signaling axis in the NK-cell immune response against MM and a role for CT-011 in enhancing the NK-cell versus MM effect. A phase 2 clinical trial of CT-011 in combination with lenalidomide for patients with MM should be considered.\"},\n",
       "  {'qas': [{'id': '517901bc8ed59a060a00003b_005',\n",
       "     'question': 'The antibodies MK-3475 and CT-011 have shown promising results in treating malignancies. Which protein are they targeting?',\n",
       "     'answers': [{'text': 'PD-1', 'answer_start': 90}]}],\n",
       "   'context': 'Phase I safety and pharmacokinetic study of CT-011, a humanized antibody interacting with PD-1, in patients with advanced hematologic malignancies. PURPOSE: CT-011 is a humanized IgG1 monoclonal antibody that modulates the immune response through interaction with PD-1, a protein belonging to the B7 receptor family present on lymphocytes. The objectives of this phase I study were to assess the dose-limiting toxicities, to determine the maximum tolerated dose, and to study the pharmacokinetics of CT-011 administered once to patients with advanced hematologic malignancies. EXPERIMENTAL DESIGN: Seventeen patients were treated with escalating doses of CT-011 ranging from 0.2 to 6 mg/kg. For pharmacokinetic analysis, blood samples were withdrawn from the patients before and immediately after treatment and at 24 hours, 48 hours, and on days 7, 14, and 21. CT-011 blood levels were assessed with a specific ELISA and derived concentrations were used to calculate pharmacokinetic parameters. Activation of the immune system was assessed by measuring peripheral blood CD4+, CD8+, and CD69+ lymphocytes. RESULTS: The study showed the antibody to be safe and well tolerated in this patient population. No single maximum tolerated dose was defined in this study. Clinical benefit was observed in 33% of the patients with one complete remission. Pharmacokinetic analyses show that serum Cmax and the AUC of CT-011 increased proportionally with dose. The median t1/2 of CT-011 ranged from 217 to 410 hours. Sustained elevation in the percentage of peripheral blood CD4+ lymphocytes was observed up to 21 days following CT-011 treatment. CONCLUSIONS: A single administration of 0.2 to 6.0 mg/kg of CT-011 is safe and well tolerated in patients with advanced hematologic malignancies.'},\n",
       "  {'qas': [{'id': '517901bc8ed59a060a00003b_006',\n",
       "     'question': 'The antibodies MK-3475 and CT-011 have shown promising results in treating malignancies. Which protein are they targeting?',\n",
       "     'answers': [{'text': 'PD-1', 'answer_start': 577}]}],\n",
       "   'context': 'Programmed death 1 pathway inhibition in metastatic renal cell cancer and prostate cancer. Programmed death 1 (PD-1) is a T cell co-inhibitory receptor with two ligands, PD-L1 and PD-L2. In cancer, this pathway plays a major role in immune resistance in the tumor environment. Blockade of this pathway can enhance antitumor immune responses. This review discusses the preclinical rationale for PD-1 pathway inhibition in advanced renal cell carcinoma and prostate cancer, in addition to the clinical activity and toxicity of the anti-PD-L1 antibody BMS-936559, as well as anti-PD-1 antibodies MK-3475 and BMS-936558.'},\n",
       "  {'qas': [{'id': '54e0e902ae9738404b000001_001',\n",
       "     'question': 'What family do mDia proteins belong in?',\n",
       "     'answers': [{'text': 'mDia proteins are members of the formin family',\n",
       "       'answer_start': 32}]}],\n",
       "   'context': 'mDia1-3 in mammalian filopodia. mDia proteins are members of the formin family of actin nucleating proteins that polymerize linear actin filaments. Such filaments form the core of thin, tubular, membrane-bound cell surface protrusions known as filopodia, which are a major feature of mammalian cell morphology. Filopodia are dynamic structures that help cells sense environmental cues, and play a role in cell migration, axon guidance, angiogenesis and other processes. RhoGTPases bind to and control the activity of mDia proteins, and several other binding partners of the three mDia1 isoforms-mDia1, mDia2 and mDia3-have been documented. Two independent pathways controlling mammalian filopodium formation have emerged, with one driven by the RhoGTPase Cdc42, and the other by Rif. While mDia2 has been the main formin implicated in forming filopodia, mDia1 has recently surfaced as the key formin utilized by both the Cdc42 and Rif pathways to drive filopodial protrusion.'},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_001',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 767}]}],\n",
       "   'context': \"Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes. Plasmids are ubiquitous mobile elements that serve as a pool of many host beneficial traits such as antibiotic resistance in bacterial communities. To understand the importance of plasmids in horizontal gene transfer, we need to gain insight into the 'evolutionary history' of these plasmids, i.e. the range of hosts in which they have evolved. Since extensive data support the proposal that foreign DNA acquires the host's nucleotide composition during long-term residence, comparison of nucleotide composition of plasmids and chromosomes could shed light on a plasmid's evolutionary history. The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids. Here, we introduce the Mahalanobis distance, which takes into account the variance-covariance structure of the chromosome signatures. We demonstrate that the Mahalanobis distance is better than the delta-distance at measuring genomic signature differences between plasmids and chromosomes of potential hosts. We illustrate the usefulness of this metric for proposing candidate long-term hosts for plasmids, focusing on the virulence plasmids pXO1 from Bacillus anthracis, and pO157 from Escherichia coli O157:H7, as well as the broad host range multi-drug resistance plasmid pB10 from an unknown host.\"},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_002',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 184}]}],\n",
       "   'context': 'Heterogeneity of genomes: measures and values. Genomic homogeneity is investigated for a broad base of DNA sequences in terms of dinucleotide relative abundance distances (abbreviated delta-distances) and of oligonucleotide compositional extremes. It is shown that delta-distances between different genomic sequences in the same species are low, only about 2 or 3 times the distance found in random DNA, and are generally smaller than the between-species delta-distances. Extremes in short oligonucleotides include underrepresentation of TpA and overrepresentation of GpC in most temperate bacteriophage sequences; underrepresentation of CTAG in most eubacterial genomes; underrepresentation of GATC in most bacteriophage; CpG suppression in vertebrates, in all animal mitochondrial genomes, and in many thermophilic bacterial sequences; and overrepresentation of GpG/CpC in all animal mitochondrial sets and chloroplast genomes. Interpretations center on DNA structures (dinucleotide stacking energies, DNA curvature and superhelicity, nucleosome organization), context-dependent mutational events, methylation effects, and processes of replication and repair.'},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_003',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 265}]}],\n",
       "   'context': 'Heterogeneity of genomes: measures and values. Genomic homogeneity is investigated for a broad base of DNA sequences in terms of dinucleotide relative abundance distances (abbreviated delta-distances) and of oligonucleotide compositional extremes. It is shown that delta-distances between different genomic sequences in the same species are low, only about 2 or 3 times the distance found in random DNA, and are generally smaller than the between-species delta-distances. Extremes in short oligonucleotides include underrepresentation of TpA and overrepresentation of GpC in most temperate bacteriophage sequences; underrepresentation of CTAG in most eubacterial genomes; underrepresentation of GATC in most bacteriophage; CpG suppression in vertebrates, in all animal mitochondrial genomes, and in many thermophilic bacterial sequences; and overrepresentation of GpG/CpC in all animal mitochondrial sets and chloroplast genomes. Interpretations center on DNA structures (dinucleotide stacking energies, DNA curvature and superhelicity, nucleosome organization), context-dependent mutational events, methylation effects, and processes of replication and repair.'},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_004',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 455}]}],\n",
       "   'context': 'Heterogeneity of genomes: measures and values. Genomic homogeneity is investigated for a broad base of DNA sequences in terms of dinucleotide relative abundance distances (abbreviated delta-distances) and of oligonucleotide compositional extremes. It is shown that delta-distances between different genomic sequences in the same species are low, only about 2 or 3 times the distance found in random DNA, and are generally smaller than the between-species delta-distances. Extremes in short oligonucleotides include underrepresentation of TpA and overrepresentation of GpC in most temperate bacteriophage sequences; underrepresentation of CTAG in most eubacterial genomes; underrepresentation of GATC in most bacteriophage; CpG suppression in vertebrates, in all animal mitochondrial genomes, and in many thermophilic bacterial sequences; and overrepresentation of GpG/CpC in all animal mitochondrial sets and chloroplast genomes. Interpretations center on DNA structures (dinucleotide stacking energies, DNA curvature and superhelicity, nucleosome organization), context-dependent mutational events, methylation effects, and processes of replication and repair.'},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_005',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 767}]}],\n",
       "   'context': \"Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes. Plasmids are ubiquitous mobile elements that serve as a pool of many host beneficial traits such as antibiotic resistance in bacterial communities. To understand the importance of plasmids in horizontal gene transfer, we need to gain insight into the 'evolutionary history' of these plasmids, i.e. the range of hosts in which they have evolved. Since extensive data support the proposal that foreign DNA acquires the host's nucleotide composition during long-term residence, comparison of nucleotide composition of plasmids and chromosomes could shed light on a plasmid's evolutionary history. The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids. Here, we introduce the Mahalanobis distance, which takes into account the variance-covariance structure of the chromosome signatures. We demonstrate that the Mahalanobis distance is better than the delta-distance at measuring genomic signature differences between plasmids and chromosomes of potential hosts. We illustrate the usefulness of this metric for proposing candidate long-term hosts for plasmids, focusing on the virulence plasmids pXO1 from Bacillus anthracis, and pO157 from Escherichia coli O157:H7, as well as the broad host range multi-drug resistance plasmid pB10 from an unknown host.\"},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_006',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 1124}]}],\n",
       "   'context': \"Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes. Plasmids are ubiquitous mobile elements that serve as a pool of many host beneficial traits such as antibiotic resistance in bacterial communities. To understand the importance of plasmids in horizontal gene transfer, we need to gain insight into the 'evolutionary history' of these plasmids, i.e. the range of hosts in which they have evolved. Since extensive data support the proposal that foreign DNA acquires the host's nucleotide composition during long-term residence, comparison of nucleotide composition of plasmids and chromosomes could shed light on a plasmid's evolutionary history. The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids. Here, we introduce the Mahalanobis distance, which takes into account the variance-covariance structure of the chromosome signatures. We demonstrate that the Mahalanobis distance is better than the delta-distance at measuring genomic signature differences between plasmids and chromosomes of potential hosts. We illustrate the usefulness of this metric for proposing candidate long-term hosts for plasmids, focusing on the virulence plasmids pXO1 from Bacillus anthracis, and pO157 from Escherichia coli O157:H7, as well as the broad host range multi-drug resistance plasmid pB10 from an unknown host.\"},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_007',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 767}]}],\n",
       "   'context': \"Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes. Plasmids are ubiquitous mobile elements that serve as a pool of many host beneficial traits such as antibiotic resistance in bacterial communities. To understand the importance of plasmids in horizontal gene transfer, we need to gain insight into the 'evolutionary history' of these plasmids, i.e. the range of hosts in which they have evolved. Since extensive data support the proposal that foreign DNA acquires the host's nucleotide composition during long-term residence, comparison of nucleotide composition of plasmids and chromosomes could shed light on a plasmid's evolutionary history. The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids. Here, we introduce the Mahalanobis distance, which takes into account the variance-covariance structure of the chromosome signatures. We demonstrate that the Mahalanobis distance is better than the delta-distance at measuring genomic signature differences between plasmids and chromosomes of potential hosts. We illustrate the usefulness of this metric for proposing candidate long-term hosts for plasmids, focusing on the virulence plasmids pXO1 from Bacillus anthracis, and pO157 from Escherichia coli O157:H7, as well as the broad host range multi-drug resistance plasmid pB10 from an unknown host.\"},\n",
       "  {'qas': [{'id': '554356d0ed966d112c000005_008',\n",
       "     'question': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "     'answers': [{'text': 'delta-distance', 'answer_start': 767}]}],\n",
       "   'context': \"Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes. Plasmids are ubiquitous mobile elements that serve as a pool of many host beneficial traits such as antibiotic resistance in bacterial communities. To understand the importance of plasmids in horizontal gene transfer, we need to gain insight into the 'evolutionary history' of these plasmids, i.e. the range of hosts in which they have evolved. Since extensive data support the proposal that foreign DNA acquires the host's nucleotide composition during long-term residence, comparison of nucleotide composition of plasmids and chromosomes could shed light on a plasmid's evolutionary history. The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids. Here, we introduce the Mahalanobis distance, which takes into account the variance-covariance structure of the chromosome signatures. We demonstrate that the Mahalanobis distance is better than the delta-distance at measuring genomic signature differences between plasmids and chromosomes of potential hosts. We illustrate the usefulness of this metric for proposing candidate long-term hosts for plasmids, focusing on the virulence plasmids pXO1 from Bacillus anthracis, and pO157 from Escherichia coli O157:H7, as well as the broad host range multi-drug resistance plasmid pB10 from an unknown host.\"},\n",
       "  {'qas': [{'id': '532206819b2d7acc7e00000f_001',\n",
       "     'question': 'Name a method for enrichment of arginine-methylated peptides.',\n",
       "     'answers': [{'text': 'immunoaffinity purification',\n",
       "       'answer_start': 710}]}],\n",
       "   'context': 'Immunoaffinity enrichment and mass spectrometry analysis of protein methylation. Protein methylation is a common posttranslational modification that mostly occurs on arginine and lysine residues. Arginine methylation has been reported to regulate RNA processing, gene transcription, DNA damage repair, protein translocation, and signal transduction. Lysine methylation is best known to regulate histone function and is involved in epigenetic regulation of gene transcription. To better study protein methylation, we have developed highly specific antibodies against monomethyl arginine; asymmetric dimethyl arginine; and monomethyl, dimethyl, and trimethyl lysine motifs. These antibodies were used to perform immunoaffinity purification of methyl peptides followed by LC-MS/MS analysis to identify and quantify arginine and lysine methylation sites in several model studies. Overall, we identified over 1000 arginine methylation sites in human cell line and mouse tissues, and ∼160 lysine methylation sites in human cell line HCT116. The number of methylation sites identified in this study exceeds those found in the literature to date. Detailed analysis of arginine-methylated proteins observed in mouse brain compared with those found in mouse embryo shows a tissue-specific distribution of arginine methylation, and extends the types of proteins that are known to be arginine methylated to include many new protein types. Many arginine-methylated proteins that we identified from the brain, including receptors, ion channels, transporters, and vesicle proteins, are involved in synaptic transmission, whereas the most abundant methylated proteins identified from mouse embryo are transcriptional regulators and RNA processing proteins.'},\n",
       "  {'qas': [{'id': '54d8fd334b1fd0d33c000005_001',\n",
       "     'question': 'Rindopepimut is an analog of which growth factor?',\n",
       "     'answers': [{'text': 'EGFRvIII', 'answer_start': 708}]}],\n",
       "   'context': 'Rindopepimut: an evidence-based review of its therapeutic potential in the treatment of EGFRvIII-positive glioblastoma. Glioblastoma multiforme (GBM) is the most common primary brain tumor in adults and is universally fatal. Despite surgical resection, radiotherapy, and systemic chemotherapy, the median overall survival is less than 15 months. As current therapies are not tumor-specific, treatment commonly results in toxicity. The epidermal growth factor receptor variant III (EGFRvIII) is a naturally occurring mutant of EGFR and is expressed on approximately 20% to 30% of GBMs. As it is not expressed on normal cells, it is an ideal therapeutic target. Rindopepimut is a peptide vaccine which elicits EGFRvIII-specific humoral and cellular immune responses. Phase I and II clinical trials have demonstrated significantly higher progression-free and overall survival times in vaccinated patients with EGFRvIII-expressing GBM tumors. Side effects are minimal and mainly consist of hypersensitivity reactions. Due to the efficacy and safety of rindopepimut, it is a promising therapy for patients with GBM. Currently, rindopepimut is undergoing clinical testing in an international Phase III trial for newly diagnosed GBM and a Phase II trial for relapsed GBM.'},\n",
       "  {'qas': [{'id': '54d8fd334b1fd0d33c000005_002',\n",
       "     'question': 'Rindopepimut is an analog of which growth factor?',\n",
       "     'answers': [{'text': 'EGFRvIII', 'answer_start': 752}]}],\n",
       "   'context': 'Targeting EGF receptor variant III: tumor-specific peptide vaccination for malignant gliomas. Glioblastoma multiforme (GBM) is the most common and deadly of the human brain cancers. The EGF receptor is often amplified in GBM and provides a potential therapeutic target. However, targeting the normal receptor is complicated by its nearly ubiquitous and high level of expression in certain tissues. A naturally occurring deletion mutant of the EGF receptor, EGFRvIII, is a constitutively active variant originally identified in a high percentage of brain cancer cases, and more importantly is rarely found in normal tissue. A peptide vaccine, rindopepimut (CDX-110, Celldex Therapeutics), is directed against the novel exon 1-8 junction produced by the EGFRvIII deletion, and it has shown high efficacy in preclinical models. Recent Phase II clinical trials in patients with newly diagnosed GBM have shown EGFRvIII-specific immune responses and significantly increased time to progression and overall survival in those receiving vaccine therapy, as compared with published results for standard of care. Rindopepimut therefore represents a very promising therapy for patients with GBM.'},\n",
       "  {'qas': [{'id': '54d8fd334b1fd0d33c000005_003',\n",
       "     'question': 'Rindopepimut is an analog of which growth factor?',\n",
       "     'answers': [{'text': 'EGFRvIII', 'answer_start': 58}]}],\n",
       "   'context': 'Rindopepimut, a 14-mer injectable peptide vaccine against EGFRvIII for the potential treatment of glioblastoma multiforme. Celldex Therapeutics is developing rindopepimut (CDX-110), a 14-mer injectable peptide vaccine for the potential treatment of glioblastoma multiforme (GBM). Rindopepimut specifically targets a novel junctional epitope of the EGFR deletion mutant EGFRvIII, which is a constitutively active receptor that is expressed in approximately 60 to 70% of patients with GBM. EGFRvIII expression is correlated with worse prognosis and reduced overall survival. Importantly, EGFRvIII is not expressed in normal brain tissue, making it an excellent therapeutic target. Preclinical studies demonstrated lasting tumor regression and increased survival times, as well as efficient generation of EGFRvIII-specific humoral and cellular immune responses, in animals expressing EGFRvIII and vaccinated with rindopepimut. Phase I and II clinical trials in patients with GBM demonstrated significantly increased median time to progression and overall survival time in those vaccinated with rindopepimut compared with matched historical controls. Only limited side effects have been observed in patients. Given these results, rindopepimut is an extremely promising therapy for patients with GBM. Phase I and II clinical trials in patients with GBM were ongoing at the time of publication. In the future, it may be beneficial to combine rindopepimut with other treatment modalities to further prolong survival.'},\n",
       "  {'qas': [{'id': '54d8fd334b1fd0d33c000005_004',\n",
       "     'question': 'Rindopepimut is an analog of which growth factor?',\n",
       "     'answers': [{'text': 'EGFRvIII', 'answer_start': 369}]}],\n",
       "   'context': 'Rindopepimut, a 14-mer injectable peptide vaccine against EGFRvIII for the potential treatment of glioblastoma multiforme. Celldex Therapeutics is developing rindopepimut (CDX-110), a 14-mer injectable peptide vaccine for the potential treatment of glioblastoma multiforme (GBM). Rindopepimut specifically targets a novel junctional epitope of the EGFR deletion mutant EGFRvIII, which is a constitutively active receptor that is expressed in approximately 60 to 70% of patients with GBM. EGFRvIII expression is correlated with worse prognosis and reduced overall survival. Importantly, EGFRvIII is not expressed in normal brain tissue, making it an excellent therapeutic target. Preclinical studies demonstrated lasting tumor regression and increased survival times, as well as efficient generation of EGFRvIII-specific humoral and cellular immune responses, in animals expressing EGFRvIII and vaccinated with rindopepimut. Phase I and II clinical trials in patients with GBM demonstrated significantly increased median time to progression and overall survival time in those vaccinated with rindopepimut compared with matched historical controls. Only limited side effects have been observed in patients. Given these results, rindopepimut is an extremely promising therapy for patients with GBM. Phase I and II clinical trials in patients with GBM were ongoing at the time of publication. In the future, it may be beneficial to combine rindopepimut with other treatment modalities to further prolong survival.'},\n",
       "  {'qas': [{'id': '54d8fd334b1fd0d33c000005_005',\n",
       "     'question': 'Rindopepimut is an analog of which growth factor?',\n",
       "     'answers': [{'text': 'EGFRvIII', 'answer_start': 752}]}],\n",
       "   'context': 'Targeting EGF receptor variant III: tumor-specific peptide vaccination for malignant gliomas. Glioblastoma multiforme (GBM) is the most common and deadly of the human brain cancers. The EGF receptor is often amplified in GBM and provides a potential therapeutic target. However, targeting the normal receptor is complicated by its nearly ubiquitous and high level of expression in certain tissues. A naturally occurring deletion mutant of the EGF receptor, EGFRvIII, is a constitutively active variant originally identified in a high percentage of brain cancer cases, and more importantly is rarely found in normal tissue. A peptide vaccine, rindopepimut (CDX-110, Celldex Therapeutics), is directed against the novel exon 1-8 junction produced by the EGFRvIII deletion, and it has shown high efficacy in preclinical models. Recent Phase II clinical trials in patients with newly diagnosed GBM have shown EGFRvIII-specific immune responses and significantly increased time to progression and overall survival in those receiving vaccine therapy, as compared with published results for standard of care. Rindopepimut therefore represents a very promising therapy for patients with GBM.'},\n",
       "  {'qas': [{'id': '54d8fd334b1fd0d33c000005_006',\n",
       "     'question': 'Rindopepimut is an analog of which growth factor?',\n",
       "     'answers': [{'text': 'EGFRvIII', 'answer_start': 708}]}],\n",
       "   'context': 'Rindopepimut: an evidence-based review of its therapeutic potential in the treatment of EGFRvIII-positive glioblastoma. Glioblastoma multiforme (GBM) is the most common primary brain tumor in adults and is universally fatal. Despite surgical resection, radiotherapy, and systemic chemotherapy, the median overall survival is less than 15 months. As current therapies are not tumor-specific, treatment commonly results in toxicity. The epidermal growth factor receptor variant III (EGFRvIII) is a naturally occurring mutant of EGFR and is expressed on approximately 20% to 30% of GBMs. As it is not expressed on normal cells, it is an ideal therapeutic target. Rindopepimut is a peptide vaccine which elicits EGFRvIII-specific humoral and cellular immune responses. Phase I and II clinical trials have demonstrated significantly higher progression-free and overall survival times in vaccinated patients with EGFRvIII-expressing GBM tumors. Side effects are minimal and mainly consist of hypersensitivity reactions. Due to the efficacy and safety of rindopepimut, it is a promising therapy for patients with GBM. Currently, rindopepimut is undergoing clinical testing in an international Phase III trial for newly diagnosed GBM and a Phase II trial for relapsed GBM.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_001',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'lung', 'answer_start': 923}]}],\n",
       "   'context': 'High specific detection and near-infrared photothermal therapy of lung cancer cells with high SERS active aptamer-silver-gold shell-core nanostructures. Lung cancer is the leading cause of cancer death worldwide. Its early detection is of paramount importance for diagnosis, classification, treatment, and improvement of survivorship. However, current methods are not sensitive enough to detect lung cancer in its nascent stage. We reported an aptamer-Ag-Au shell-core nanostructure-based surface-enhanced Raman scattering (SERS) assay for sensitive and specific detection, and near-infrared (NIR) photothermal therapy of lung adenocarcinoma cells (A549 cells). The nanostructures target the cells with high affinity and specificity via the specific interaction between the aptamer (a 45-base oligonucleotide) and the cell, and distinguish A549 cells from other types of cancer cells (HeLa and MCF-7 cells) and subtypes of lung cancer cells (NCI-H157, NCI-H520, NCI-H1299, and NCI-H446 cells). The nanostructures have a high capability to absorb NIR irradiation and are able to perform photothermal therapy of the cells at a very low irradiation power density (0.20 W cm(-2)) without destroying the healthy cells and the surrounding normal tissues. In addition, the nanostructures exhibit a high SERS activity. Based on the SERS signal of the labeled Raman reporter (Rh6G molecules), we can specifically detect A549 cells at a very low abundance (~10 cells per mL) and monitor the therapy process of the cancer cells. Therefore, this nanostructure-based SERS assay has great potential in specific recognition, sensitive detection, and effective photothermal therapy of lung cancer.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_002',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'squamous cell carcinoma', 'answer_start': 558}]}],\n",
       "   'context': 'Musashi1 as a potential therapeutic target and diagnostic marker for lung cancer. Lung cancer remains one of the leading causes of cancer-related deaths worldwide with a 5-year survival rate of less than 20%. One approach to improving survival is the identification of biomarkers to detect early stage disease. In this study, we investigated the potential of the stem cell and progenitor cell marker, Musashi1 (Msi1), as a diagnostic marker and potential therapeutic target for lung cancer. Functional studies in A549 bronchioalveolar carcinoma and NCI-H520 squamous cell carcinoma cells revealed that Msi1 was enriched in spheroid cultures of tumor cells and in the CD133+ cell population. Downregulation of Msi1 by lentivirus-mediated expression of an Msi1 shRNA reduced spheroid colony proliferation. Growth inhibition was associated with reduced nuclear localization of β-catenin and inhibition of the processing of intracellular Notch. In primary lung cancer, Msi1 protein expression was elevated in 86% of 202 tissue microarray specimens, and Msi1 mRNA was increased in 80% of 118 bronchoscopic biopsies, including metastatic disease, but was rarely detected in adjacent normal lung tissue and in non-malignant diseased tissue. Msi1 was expressed in a diffuse pattern in most tumor subtypes, except in squamous cell carcinomas, where it appeared in a focal pattern in 50% of specimens. Thus, Msi1 is a sensitive and specific diagnostic marker for all lung cancer subtypes.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_003',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'lung', 'answer_start': 417}]}],\n",
       "   'context': 'BHLHB3: a candidate tumor suppressor in lung cancer. BHLHB3 is a basic helix-loop-helix (bHLH) domain-containing protein that acts as a transcriptional repressor. We found that BHLHB3 transcript levels were low in three human lung cancer cell lines and downregulated in human lung adenocarcinomas as compared to normal lung tissue. BHLHB3 gene overexpression inhibited colony formation of A549, NCI-H520 and NCI-H596 lung cancer cells. The reduced colony growth was likely due to inhibition of cell proliferation as suggested by the downregulation of cyclin D1 (CCND1) expression in NCI-H520 cells transfected to overexpress the BHLHB3 gene; no evidence of apoptosis was observed. These results point to the potential role of the BHLHB3 protein as a tumor suppressor for lung cancer.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_004',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'non-small cell lung cancer',\n",
       "       'answer_start': 758}]}],\n",
       "   'context': 'Suppression of cell proliferation and signaling transduction by connective tissue growth factor in non-small cell lung cancer cells. Connective tissue growth factor (CTGF) is a secreted protein that belongs to CCN family. The proteins in this family are implicated in various biological processes, such as angiogenesis, adhesion, migration, and apoptosis. In this study, we explored the roles of CTGF in lung tumorigenesis. The expression levels of CTGF in 58 lung cancer samples were reduced by >2 fold in 57% of the samples compared with matched normal samples using real-time reverse transcription-PCR. These results were confirmed by immunohistochemical staining for CTGF in normal lung epithelia and lung cancer. Cellular proliferation was inhibited in non-small cell lung cancer (NSCLC) cell lines NCI-H460, NCI-H520, NCI-H1299, and SK-MES-1 by CTGF overexpression. Partially purified CTGF suppressed lung cancer cell growth. The growth inhibition caused by CTGF overexpression was associated with growth arrest at G(0)-G(1) and prominent induction of p53 and ADP ribosylation factor. Most interestingly, overexpression of CTGF suppressed insulin-like growth factor-I-dependent Akt phosphorylation and epidermal growth factor-dependent extracellular signal-regulated kinase 1/2 phosphorylation. In summary, NSCLC cells expressed decreased levels of CTGF compared with normal lung cells; this lower expression has an effect on lung cancer cell proliferation and its cellular response to growth factors. Our data suggest that CTGF may behave as a secreted tumor suppressor protein in the normal lung, and its expression is suppressed in many NSCLCs.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_005',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'lung', 'answer_start': 773}]}],\n",
       "   'context': 'Suppression of cell proliferation and signaling transduction by connective tissue growth factor in non-small cell lung cancer cells. Connective tissue growth factor (CTGF) is a secreted protein that belongs to CCN family. The proteins in this family are implicated in various biological processes, such as angiogenesis, adhesion, migration, and apoptosis. In this study, we explored the roles of CTGF in lung tumorigenesis. The expression levels of CTGF in 58 lung cancer samples were reduced by >2 fold in 57% of the samples compared with matched normal samples using real-time reverse transcription-PCR. These results were confirmed by immunohistochemical staining for CTGF in normal lung epithelia and lung cancer. Cellular proliferation was inhibited in non-small cell lung cancer (NSCLC) cell lines NCI-H460, NCI-H520, NCI-H1299, and SK-MES-1 by CTGF overexpression. Partially purified CTGF suppressed lung cancer cell growth. The growth inhibition caused by CTGF overexpression was associated with growth arrest at G(0)-G(1) and prominent induction of p53 and ADP ribosylation factor. Most interestingly, overexpression of CTGF suppressed insulin-like growth factor-I-dependent Akt phosphorylation and epidermal growth factor-dependent extracellular signal-regulated kinase 1/2 phosphorylation. In summary, NSCLC cells expressed decreased levels of CTGF compared with normal lung cells; this lower expression has an effect on lung cancer cell proliferation and its cellular response to growth factors. Our data suggest that CTGF may behave as a secreted tumor suppressor protein in the normal lung, and its expression is suppressed in many NSCLCs.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_006',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'lung', 'answer_start': 859}]}],\n",
       "   'context': '[Expression and structure of BNIP3L in lung cancer]. BACKGROUND & OBJECTIVE: Bcl-2/E1B 19kDa interacting protein3-like (BNIP3L) gene is a tumor suppressor gene cloned from a human fetal liver cDNA library, which is located at 8p21, one of the high frequent regions of loss of heterozygosity (LOH) in lung carcinoma. BNIP3L protein can interact with antiapoptotic proteins, such as Bcl-2, Bcl-x(L), E1B19K, which promotes apoptosis. This study was designed to explore the correlation of alteration of expression and structure of BNIP3L gene with the progression of lung cancer. METHODS: The expression and structure of BNIP3L gene in 4 lung cancer cell strains and 30 tissues were determined by SP immunohistochemistry, immunoblot, semi-quantitative reverse transcription-PCR (RT-PCR), PCR-single strain conformation polymorphism (PCR-SSCP). RESULTS: (1) In 4 lung cancer cell strains, BNIP3L protein was not detected in A549, NCI-H460, NCI-H446, except for NCI-H520, in which the protein expression level was slightly lower than that in immortal bronchial epithelial cell strain HBE4-E6/E7. BNIP3L protein was observed in 46.7% (14/30) lung cancer tissues, while 100% (12/12) in normal lung tissues. The difference was significant in statistics (P< 0.05). (2) BNIP3L mRNA was detected in 4 lung cancer cell strains; and there existed no obvious discrepancy of the amount between these cell strains and HBE4-E6/E7. Absence or decrease of BNIP3L mRNA was observed in 26.7%(8/30) of lung cancer tissues. The average quantity of BNIP3L mRNA was 0.404+/-0.070 in lung cancer tissues, while 0.575+/-0.065 in paired normal lung tissues. The difference was significant in statistics (P< 0.05). In all the cancerous cell strains and tissues with BNIP3L mRNA, the products of RT-PCR were as long as those from their control samples in size, including the entire coding region, and no variation of BNIP3L gene structure such as absence, rearrangement, aberrant splicing were detected.(3) No point mutation was detected in all 6 exons of BNIP3L gene in 4 lung cancer cell strains and 30 tissues. CONCLUSION: BNIP3L protein expression was down-regulated in lung cancer, which might be involved in the occurrence and/or development of lung cancer. The down-regulation of BNIP3L protein expression in lung cancer was partly caused by the down-regulation of its transcription. The variation of gene structure may be not the reason of BNIP3L inactivity in lung cancer.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_007',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'lung', 'answer_start': 1164}]}],\n",
       "   'context': 'The CLN3 gene is a novel molecular target for cancer drug discovery. Juvenile Batten disease is a neurodegenerative disease caused by accelerated apoptotic death of photoreceptors and neurons attributable to defects in the CLN3 gene. CLN3 is antiapoptotic when overexpressed in NT2 neuronal precursor cells. CLN3 negatively modulates endogenous ceramide levels in NT2 cells and acts upstream of ceramide generation. Because defects in regulation of apoptosis are involved in the development of cancer, we evaluated the expression of CLN3 on both mRNA and protein levels in a variety of cancer cell lines and solid colon cancer tissue. We also observed the effect of the blocking of CLN3 protein expression on cancer cell growth, survival, ceramide production, and apoptosis by using an adenovirus-bearing antisense CLN3 construct. We show that CLN3 mRNA and protein are overexpressed in glioblastoma (U-373G and T98g), neuroblastoma (IMR-32 and SK-N-MC), prostate (Du145, PC-3, and LNCaP), ovarian (SK-OV-3, SW626, and PA-1), breast (BT-20, BT-549, and BT-474), and colon (SW1116, SW480, and HCT 116) cancer cell lines but not in pancreatic (CAPAN and As-PC-1) or lung (A-549 and NCI-H520) cancer cell lines. CLN3 is also up-regulated in mouse melanoma and breast carcinoma cancer cell lines. We found CLN3 expression is 22-330% higher than in corresponding normal colon control tissue in 8 of 10 solid colon tumors. An adenovirus-expressing antisense CLN3 (Ad-AS-CLN3) blocks CLN3 protein expression in DU-145, BT-20, SW1116, and T98g cancer cell lines as seen by Western blot. Blocking of CLN3 expression using Ad-AS-CLN3 inhibits growth and viability of cancer cells. It also causes elevation in endogenous ceramide production through de novo ceramide synthesis and results in increased apoptosis as shown by propidium iodide and JC-1 staining. This suggests that Ad-AS-CLN3 may be an option for therapy in some cancers. More importantly these results suggest that CLN3 is a novel molecular target for cancer drug discovery.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_008',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'lung', 'answer_start': 1221}]}],\n",
       "   'context': 'Expression of the aromatic L-amino acid decarboxylase mRNA in human tumour cell lines of neuroendocrine and neuroectodermal origin. Neuroendocrine differentiation of lung tumours is characterised by the expression of several neuroendocrine markers and is confined mostly to specific histological subtypes, i.e. small cell carcinomas and carcinoids. One of the markers seen in neuroendocrine tumours, high activity of the aromatic L-amino acid decarboxylase (AADC), is helpful in distinguishing the classic and variant small cell lung tumour subtypes. Here, we have analysed the expression and quantified the level of mRNA coding for AADC in human tumour cell lines by use of the reverse transcription and polymerase chain reaction (RT-PCR). High amounts of mRNA were detected in classic small cell lung carcinomas and a neuroblastoma cell line. Other cell lines (melanomas, non-small cell lung carcinomas and osteosarcoma) also showed AADC expression, but the levels were 2-3 orders lower. Also, the tissue-specific (neuronal versus liver-specific) mRNA type has been estimated. Small cell lung carcinomas, neuroblastoma and melanoma expressed messenger RNA specific for neuronal tissues. Importantly, the non-small cell lung carcinoma cell lines expressed either liver-specific (non-neuronal) mRNA (cell line A549) or predominantly the neuronal (cell line NCI-H520) AADC message. These data indicate that a range of tumour cell lines transcribe the AADC gene and that two distinct types of AADC mRNA which reflect the embryonal (neuronal or non-neuronal) origin of the tumour may be produced in non-small cell lung cancer cells.'},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_009',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'squamous cell carcinoma', 'answer_start': 333}]}],\n",
       "   'context': \"Antitumor effect of CGP41251, a new selective protein kinase C inhibitor, on human non-small cell lung cancer cells. The antitumor effect of CGP41251 (4'-N-benzoyl staurosporine), a selective protein kinase C (PKC) inhibitor, was examined on two kinds of human non-small cell lung cancer (NSCLC) cell lines (adenocarcinoma: A549 and squamous cell carcinoma: NCI-H520). CGP41251 at 0.5 or 1.0 microM inhibited the proliferation of these tumor cell lines significantly; However, at 0.1 microM, it did not show any significant inhibition. Cell cycle analysis indicated that CGP41251 at 0.5 or 1.0 microM arrested the cell cycle progression at the G2/M phase up to 24 hr, but 0.1 microM did not. It seems that the antiproliferative action of CGP41251 against human NSCLC is related to G2/M accumulation. In NCI-H520, CGP41251 caused DNA re-replication without mitosis. In a nude mice xenograft, CGP41251 at a dose of 200 mg/kg showed antitumor activity against these cell lines. Histopathologically, expansion of central necrosis was observed, although no destruction of tumor nests was seen by CGP41251 administration. In both tumor tissues, the PKC activity of the particulate fraction was significantly decreased by CGP41251 treatment. From these results, it is thought that the antitumor activity of CGP41251 against human NSCLS is accompanied by the decrease of PKC activity in the particulate fraction. Moreover, the G2/M arrest of the cell cycle induced by CGP41251 might be important for the growth inhibitory action of this compound.\"},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_010',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'non-small cell lung cancer',\n",
       "       'answer_start': 261}]}],\n",
       "   'context': \"Antitumor effect of CGP41251, a new selective protein kinase C inhibitor, on human non-small cell lung cancer cells. The antitumor effect of CGP41251 (4'-N-benzoyl staurosporine), a selective protein kinase C (PKC) inhibitor, was examined on two kinds of human non-small cell lung cancer (NSCLC) cell lines (adenocarcinoma: A549 and squamous cell carcinoma: NCI-H520). CGP41251 at 0.5 or 1.0 microM inhibited the proliferation of these tumor cell lines significantly; However, at 0.1 microM, it did not show any significant inhibition. Cell cycle analysis indicated that CGP41251 at 0.5 or 1.0 microM arrested the cell cycle progression at the G2/M phase up to 24 hr, but 0.1 microM did not. It seems that the antiproliferative action of CGP41251 against human NSCLC is related to G2/M accumulation. In NCI-H520, CGP41251 caused DNA re-replication without mitosis. In a nude mice xenograft, CGP41251 at a dose of 200 mg/kg showed antitumor activity against these cell lines. Histopathologically, expansion of central necrosis was observed, although no destruction of tumor nests was seen by CGP41251 administration. In both tumor tissues, the PKC activity of the particulate fraction was significantly decreased by CGP41251 treatment. From these results, it is thought that the antitumor activity of CGP41251 against human NSCLS is accompanied by the decrease of PKC activity in the particulate fraction. Moreover, the G2/M arrest of the cell cycle induced by CGP41251 might be important for the growth inhibitory action of this compound.\"},\n",
       "  {'qas': [{'id': '52f89fc62059c6d71c000050_011',\n",
       "     'question': 'From which tissue was the NCI-H520 cell-line derived?',\n",
       "     'answers': [{'text': 'lung', 'answer_start': 276}]}],\n",
       "   'context': \"Antitumor effect of CGP41251, a new selective protein kinase C inhibitor, on human non-small cell lung cancer cells. The antitumor effect of CGP41251 (4'-N-benzoyl staurosporine), a selective protein kinase C (PKC) inhibitor, was examined on two kinds of human non-small cell lung cancer (NSCLC) cell lines (adenocarcinoma: A549 and squamous cell carcinoma: NCI-H520). CGP41251 at 0.5 or 1.0 microM inhibited the proliferation of these tumor cell lines significantly; However, at 0.1 microM, it did not show any significant inhibition. Cell cycle analysis indicated that CGP41251 at 0.5 or 1.0 microM arrested the cell cycle progression at the G2/M phase up to 24 hr, but 0.1 microM did not. It seems that the antiproliferative action of CGP41251 against human NSCLC is related to G2/M accumulation. In NCI-H520, CGP41251 caused DNA re-replication without mitosis. In a nude mice xenograft, CGP41251 at a dose of 200 mg/kg showed antitumor activity against these cell lines. Histopathologically, expansion of central necrosis was observed, although no destruction of tumor nests was seen by CGP41251 administration. In both tumor tissues, the PKC activity of the particulate fraction was significantly decreased by CGP41251 treatment. From these results, it is thought that the antitumor activity of CGP41251 against human NSCLS is accompanied by the decrease of PKC activity in the particulate fraction. Moreover, the G2/M arrest of the cell cycle induced by CGP41251 might be important for the growth inhibitory action of this compound.\"},\n",
       "  {'qas': [{'id': '52f350042059c6d71c000010_001',\n",
       "     'question': 'What is the incidence of Edwards syndrom in the european population?',\n",
       "     'answers': [{'text': '1:5000', 'answer_start': 583}]}],\n",
       "   'context': '[Difficult therapeutic decision making in treatment of children with oesophageal atresia and trisomy of chromosome 18 - comments by geneticist, surgeon, neonatologist, paediatrician and anaesthesiologist]. Oesophageal atresia is a congenital defect of alimentary tract concerning the interruption of oesophagus with or without connection with the trachea. Its incidence is 1:3000-3500 of live-born. Associated anomalies including genetic disorders occur in 50% of patients. Edwards syndrome which is trisomy of chromosome 18 with poor prognosis. The incidence of Edwards syndrome is 1:5000 of live-born. About 5% of these children live more than 1 year. The aim of this article is a retrospective analysis of the course of treatment of newborn with oesophageal atresia and Edwards syndrome and making of therapeutic decision. The authors from different medical specializations: clinical genetics, paediatric surgery, paediatrics and neonatology, paediatric intensive care and palliative medicine, have undertaken a discussion regarding surgical treatment of children with oesophageal atresia and chromosomal, lethal syndrome.'},\n",
       "  {'qas': [{'id': '54f60ae05f206a0c06000008_001',\n",
       "     'question': 'What is the substrate of the microbial enzyme inulinase?',\n",
       "     'answers': [{'text': 'The inulinase acts on the beta-(2,1)-D-fructoside links in inulin releasing D-fructose.',\n",
       "       'answer_start': 635}]}],\n",
       "   'context': 'Trends in inulinase production--a review. This article highlights the research work carried out in the production of inulinases from various inulin substrates using strains of bacteria, yeast and fungi. Inulin is one of the numerous polysaccharides of plant origin that contains glucose or fructose. It is used as a substrate in industrial fermentation processes and in food industries due to its relatively cheap and abundant source for the microbiological production of high-fructose syrups, ethanol and acetone-butanol. The various oligosaccharides derived from inulin also find their application in the medical and dietary sector. The inulinase acts on the beta-(2,1)-D-fructoside links in inulin releasing D-fructose. Hence, this article illustrates the capability of various microbes in hydrolyzing the carbon at its optimum nutrient concentration and operating condition towards inulinase production.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_001',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 287}]}],\n",
       "   'context': 'The development of dasatinib as a treatment for chronic myeloid leukemia (CML): from initial studies to application in newly diagnosed patients. PURPOSE: Dasatinib is a dual Abl/Src tyrosine kinase inhibitor (TKI) designed as a prototypic short-acting BCR-ABL-targeted TKI that inhibits BCR-ABL with greater potency compared with imatinib, nilotinib, bosutinib, and ponatinib and has been shown to have potential immunomodulatory effects. Dasatinib is approved for the treatment of all phases of chronic myeloid leukemia (CML) and Philadelphia chromosome-positive acute lymphoblastic leukemia resistant or intolerant to prior imatinib treatment and first-line treatment for CML in chronic phase. In this article, the development of dasatinib as a treatment for patients with CML is reviewed. METHODS: This is a review of the relevant literature regarding dasatinib development in CML (2003-2013). RESULTS: Dasatinib demonstrates efficacy against most BCR-ABL mutations arising during imatinib therapy and is effective in treating patients with imatinib resistance due to other mechanisms. Randomized trial data show that first-line dasatinib provides superior responses compared with imatinib and enables patients to achieve early, deep responses correlated with improved longer-term outcomes. Dasatinib has a generally acceptable safety profile, with most adverse events (AEs) proving manageable and reversible. Cytopenias are commonly observed with dasatinib, and some nonhematologic AEs including pleural effusion have been consistently reported. CONCLUSION: Dasatinib is an effective treatment option for patients with CML.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_002',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 219}]}],\n",
       "   'context': 'Detection of BCR-ABL1 kinase domain mutations causing imatinib resistance in chronic myelogenous leukemia. The reciprocal translocation between chromosomes 9 and 22 [t(9;22)(q34;q11), Philadelphia chromosome] creates a BCR-ABL1 fusion protein that occurs in approximately 95% of cases of chronic myelogenous leukemia (CML), 15% of cases of adult acute lymphoblastic leukemia, and 5% of adult cases of acute myeloid leukemia. The BCR-ABL1 protein is a constitutively activated tyrosine kinase that induces and maintains the neoplastic phenotype in these leukemias. PCR-based methods to identify and quantitate the tumor-specific BCR-ABL1 RNA have been shown to be an ultrasensitive diagnostic, prognostic, and monitoring tool for Philadelphia-positive leukemias. A novel tyrosine kinase inhibitor (TKI), imatinib, has been confirmed as an effective targeted treatment in most CML patients. However, a significant minority of patients being treated with imatinib develop resistance to the drug as evidenced by rising BCR-ABL1 levels. The most common mechanism of resistance in these patients is the development of mutations in the BCR-ABL1 kinase domain (KD) that abrogate binding of imatinib. Although KD mutations are quite heterogeneous, the identification of the exact mutation site is clinically important, as some mutations, but not others, can be effectively treated with second-generation TKIs. One mutation, T315I, for example, renders the leukemia resistant to all first- and second-line TKIs. Thus, DNA sequencing of the BCR-ABL1 kinase domain in resistant patients helps identify those who may benefit from a change in TKI agents, or those who should be considered for other therapeutic measures, such as stem cell transplantation. We describe here a method for sequencing the BCR-ABL1 kinase domain in peripheral blood or bone marrow of CML patients.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_003',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 313}]}],\n",
       "   'context': 'Imatinib and nilotinib inhibit hematopoietic progenitor cell growth, but do not prevent adhesion, migration and engraftment of human cord blood CD34+ cells. BACKGROUND: The availability of tyrosine kinase inhibitors (TKIs) has considerably changed the management of Philadelphia chromosome positive leukemia. The BCR-ABL inhibitor imatinib is also known to inhibit the tyrosine kinase of the stem cell factor receptor, c-Kit. Nilotinib is 30 times more potent than imatinib towards BCR-ABL in vitro. Studies in healthy volunteers and patients with chronic myelogenous leukemia or gastrointestinal stromal tumors have shown that therapeutic doses of nilotinib deliver drug levels similar to those of imatinib. The aim of this study was to compare the inhibitory effects of imatinib and nilotinib on proliferation, differentiation, adhesion, migration and engraftment capacities of human cord blood CD34(+) cells. DESIGN AND METHODS: After a 48-hour cell culture with or without TKIs, CFC, LTC-IC, migration, adhesion and cell cycle analysis were performed. In a second time, the impact of these TKIs on engraftment was assessed in a xenotransplantation model using NOD/SCID/IL-2Rγ (null) mice. RESULTS: TKIs did not affect LTC-IC frequencies despite in vitro inhibition of CFC formation due to inhibition of CD34(+) cell cycle entry. Adhesion of CD34(+) cells to retronectin was reduced in the presence of either imatinib or nilotinib but only at high concentrations. Migration through a SDF-1α gradient was not changed by cell culture in the presence of TKIs. Finally, bone marrow cellularity and human chimerism were not affected by daily doses of imatinib and nilotinib in a xenogenic transplantation model. No significant difference was seen between TKIs given the equivalent affinity of imatinib and nilotinib for KIT. CONCLUSIONS: These data suggest that combining non-myeloablative conditioning regimen with TKIs starting the day of the transplantation could be safe.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_004',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 640}]}],\n",
       "   'context': \"[Chronic myeloid leukemia and imatinib: Experience at the Lome Campus teaching hospital (Togo)]. BACKGROUND: Chronic myeloid leukemia (CML) is a clonal malignant myeloproliferative disorder characterized by the expansion of hematopoietic cells carrying the Philadelphia chromosome (t 9.22). Our main objective was to assess the efficacy of imatinib in CML patients, measured by their survival. METHODS: Over a six-year period (June 2003 through May 2009), 25 patients were seen regularly for CML at the Lomé Campus teaching hospital. Patients received imatinib after diagnosis and underwent regular laboratory monitoring (quantification of BCR-ABL ratio by RT-PCR). Patients' survival and treatment response were evaluated. RESULTS: Patients' mean age at diagnosis was 40 years (range: 9 to 72 years). Men predominated (17 compared with 7 women). Splenomegaly was found in 80% of cases. The mean leukocyte level was 188.71 g/L (24.4-350). Six patients (24%) had thrombocytosis with a mean platelet count of 491.15 g/L (108-2000). Six patients (24%) died after developing accelerated-phase CML or blast crisis. Estimated overall survival of patients at 6 years was 60%. Molecular biology monitoring detected a secondary G250E mutation with resistance to imatinib in one patient. Standard hematological side effects led to reduction in imatinib doses. The principal nonhematological side effects were weight gain and transient digestible disorders. CONCLUSIONS: At six years after diagnosis, imatinib was effective in treating patients with CML, even in sub-Saharan Africa. Mutation-induced resistance required regular molecular biological monitoring to determine the need to switch to later-generation tyrosine kinase inhibitors.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_005',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 401}]}],\n",
       "   'context': 'New and established tyrosine kinase inhibitors for chronic myeloid leukemia. Chronic myeloid leukemia (CML) is an uncommon malignancy, the treatment and prognosis of which have dramatically shifted over the last decade. Characterized by a translocation between chromosomes 9 and 22, known as the Philadelphia chromosome, small-molecule tyrosine kinase inhibitors (TKIs) targeted against the oncogenic BCR-ABL fusion protein have changed this once fatal disease into the model of targeted therapy. This article will review the pharmacological and clinical data supporting the use of imatinib and the second-generation TKIs dasatinib and nilotinib, and the novel TKIs bosutinib and ponatinib.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_006',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 297}]}],\n",
       "   'context': 'Oridonin in combination with imatinib exerts synergetic anti-leukemia effect in Ph+ acute lymphoblastic leukemia cells in vitro by inhibiting activation of LYN/mTOR signaling pathway. Philadelphia chromosome-positive acute lymphoblastic leukemia (Ph+ ALL) is triggered by constitutively activated BCR-ABL and SRC family tyrosine kinases.They account for the activations of multiple growth-signaling pathways, including Raf/MEK/ERK, Akt/mTOR and STAT5 pathways. The BCR-ABL tyrosine kinase inhibitor imatinib is the standard treatment for Ph+ leukemia and plays efficacious role in CML. However, imatinib has few inhibitory effects on SRC tyrosine kinase with response rate of Ph+ ALL lower and relapse more frequent and quicker compared with CML. Previous studies showed that oridonin inhibits proliferation and induces apoptosis in many tumor cells. However, the anticancer activity and mechanism of oridonin in Ph+ ALL is unknown. To investigate the anticancer activity of oridonin, we examined its role in constitutively activated Akt/mTOR, Raf/MEK/ERK, STAT5 and SRC pathway, mRNA level of bcr/abl gene, cell viability and apoptosis in Ph+ ALL SUP-B15 cells. Furthermore, we detected synergetic effect of oridonin plus imatinib. Our results showed that oridonin inhibiting activations of LYN (one of SRC family kinases) and ABL and their downstream Akt/mTOR, Raf/MEK/ERK and STAT5 pathways, downregulated Bcl-2 but upregulated Bax protein and then induced apoptosis in Ph+ ALL cells. Oridonin plus imatinib exerted synergetic effects by overcoming imatinib defect of upregulating Akt/mTOR and LYN signaling. Additionally, we examined the effect of oridonin on the signaling pathways in the primary specimens from Ph+ ALL patients. Our data showed that oridonin remarkably suppressed activations of Akt/mTOR, Raf/MEK and STAT5 pathway in these primary specimens and oridonin with imatinib exerted synergetic suppressive effects on mTOR, STAT5 and LYN signaling in one imatinib resistant patient specimen. Additional evaluation of oridonin as a potential therapeutic agent for Ph+ ALL seems warranted.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_007',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 465}]}],\n",
       "   'context': 'Oridonin in combination with imatinib exerts synergetic anti-leukemia effect in Ph+ acute lymphoblastic leukemia cells in vitro by inhibiting activation of LYN/mTOR signaling pathway. Philadelphia chromosome-positive acute lymphoblastic leukemia (Ph+ ALL) is triggered by constitutively activated BCR-ABL and SRC family tyrosine kinases.They account for the activations of multiple growth-signaling pathways, including Raf/MEK/ERK, Akt/mTOR and STAT5 pathways. The BCR-ABL tyrosine kinase inhibitor imatinib is the standard treatment for Ph+ leukemia and plays efficacious role in CML. However, imatinib has few inhibitory effects on SRC tyrosine kinase with response rate of Ph+ ALL lower and relapse more frequent and quicker compared with CML. Previous studies showed that oridonin inhibits proliferation and induces apoptosis in many tumor cells. However, the anticancer activity and mechanism of oridonin in Ph+ ALL is unknown. To investigate the anticancer activity of oridonin, we examined its role in constitutively activated Akt/mTOR, Raf/MEK/ERK, STAT5 and SRC pathway, mRNA level of bcr/abl gene, cell viability and apoptosis in Ph+ ALL SUP-B15 cells. Furthermore, we detected synergetic effect of oridonin plus imatinib. Our results showed that oridonin inhibiting activations of LYN (one of SRC family kinases) and ABL and their downstream Akt/mTOR, Raf/MEK/ERK and STAT5 pathways, downregulated Bcl-2 but upregulated Bax protein and then induced apoptosis in Ph+ ALL cells. Oridonin plus imatinib exerted synergetic effects by overcoming imatinib defect of upregulating Akt/mTOR and LYN signaling. Additionally, we examined the effect of oridonin on the signaling pathways in the primary specimens from Ph+ ALL patients. Our data showed that oridonin remarkably suppressed activations of Akt/mTOR, Raf/MEK and STAT5 pathway in these primary specimens and oridonin with imatinib exerted synergetic suppressive effects on mTOR, STAT5 and LYN signaling in one imatinib resistant patient specimen. Additional evaluation of oridonin as a potential therapeutic agent for Ph+ ALL seems warranted.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_008',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 115}]}],\n",
       "   'context': 'Late response to low-dose imatinib in patients with chronic phase chronic myeloid leukemia. Imatinib was the first BCR-ABL tyrosine kinase inhibitor to become clinically available. In this study, we retrospectively evaluated the long-term efficacy of low-dose imatinib (final maintenance dose <300\\xa0mg per day) due to intolerance, in comparison to optimal-dose imatinib ( > 300\\xa0mg per day) in patients with Philadelphia chromosome-positive chronic myeloid leukemia in the chronic phase. The Kaplan-Meier estimates of the median time to complete cytogenetic response, major molecular response, and complete molecular response were longer for 31 patients receiving low-dose imatinib (360, 1360, and 1420\\xa0days, respectively) than 74 patients receiving optimal-dose imatinib (170, 420, and 720\\xa0days, respectively). However, the differences in response shrank over time and progression-free survival were comparable between the two groups. These findings suggest that long-term treatment with low-dose imatinib is an acceptable alternative for patients with intolerance to the optimal dose.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_009',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 307}]}],\n",
       "   'context': 'Label-free MSE proteomic analysis of chronic myeloid leukemia bone marrow plasma: disclosing new insights from therapy resistance. Chronic myeloid leukemia (CML) is a pluripotent hematopoietic disorder that is currently considered incurable. The tyrosine kinase product of the Philadelphia chromosome, P210 BCR-ABL, provided a pathogenetic explanation for the initiation of the CML chronic phase and is the molecular therapeutic target for the disease. Imatinib mesylate, an orally available BCR-ABL kinase inhibitor, can induce haematologic and cytogenetic remission of CML. However, imatinib resistance occurs frequently, resulting in relapse. New treatment strategies are focusing on resistant CML stem cells and the bone marrow stroma. The identification of novel pathways and mechanisms in the bone marrow microenvironment could significantly contribute to the development of such strategies. In this work, we used a high-resolution label-free MS(E) proteomic approach to identify differential protein expression in the CML bone marrow plasma of responsive and resistant patients. Oxidative lipid metabolism and regulation of the switch from canonical to noncanonical WNT signaling may contribute to CML resistance in the bone marrow compartment.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_010',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 492}]}],\n",
       "   'context': 'Label-free MSE proteomic analysis of chronic myeloid leukemia bone marrow plasma: disclosing new insights from therapy resistance. Chronic myeloid leukemia (CML) is a pluripotent hematopoietic disorder that is currently considered incurable. The tyrosine kinase product of the Philadelphia chromosome, P210 BCR-ABL, provided a pathogenetic explanation for the initiation of the CML chronic phase and is the molecular therapeutic target for the disease. Imatinib mesylate, an orally available BCR-ABL kinase inhibitor, can induce haematologic and cytogenetic remission of CML. However, imatinib resistance occurs frequently, resulting in relapse. New treatment strategies are focusing on resistant CML stem cells and the bone marrow stroma. The identification of novel pathways and mechanisms in the bone marrow microenvironment could significantly contribute to the development of such strategies. In this work, we used a high-resolution label-free MS(E) proteomic approach to identify differential protein expression in the CML bone marrow plasma of responsive and resistant patients. Oxidative lipid metabolism and regulation of the switch from canonical to noncanonical WNT signaling may contribute to CML resistance in the bone marrow compartment.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_011',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 210}]}],\n",
       "   'context': 'Tyrosine kinase inhibitors in acute and chronic leukemias. INTRODUCTION: Since the initial approval of imatinib much has been learned about its resistance mechanisms, and efforts have continued to improve upon BCR-ABL tyrosine kinase inhibitor therapy. Targeted therapy with TKIs has continued to be an area of active research and development in the care of acute and chronic leukemia patients. AREAS COVERED: This article reviews current approved and investigational TKI treatments for chronic myelogenous leukemia (CML), Philadelphia-chromosome positive acute lymphoblastic leukemia (Ph + ALL) and acute myelogenous leukemia (AML). EXPERT OPINION: There are now more potent BCR-ABL TKIs approved, which allow for additional options when determining front-line and second-line CML and Ph + ALL treatments. The T315I mutation is an ever-present challenge. Ponatinib, a pan BCR-ABL TKI, while still under investigation, is very hopeful with its ability to overcome T315I mutations in resistant CML and Ph + ALL patients. Because nilotinib and dasatinib have not been directly compared, at present we recommend selecting one or the other based on the side-effect profile, drug interactions, patient comorbidities, and mutational status. FLT-3 inhibition is of particular interest in AML patients with FLT-3 internal tandem duplication mutations; this type of targeted therapy continues to be studied.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_012',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 176}]}],\n",
       "   'context': '[Current therapy of chronic myeloid leukemia]. Chronic myeloid leukemia (CML) originates from a hematopoietic stem cell carrying the Philadelphia (Ph) chromosome and oncogenic BCR-ABL1 fusion gene. The first tyrosine-kinase inhibitor (TKI) imatinib was introduced to clinical practice 10 years ago, and it radically improved the outcome of CML patients. The rare patients that are imatinib resistant or intolerant can be treated with second generation TKIs such as dasatinib or nilotinib. As second generation TKIs appear to be more effective than imatinib and well tolerated, they may become standard first-line treatment for CML. The major future aim in CML is curative drug therapy.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_013',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 506}]}],\n",
       "   'context': 'Patterns and prognostic indicators of response to CML treatment in a multi-country medical record review study. We conducted a review of patient medical records to assess treatment response patterns and prognostic indicators of response among chronic myeloid leukemia (CML) patients in the United States, the United Kingdom, Germany, and Japan. All 1,063 patients selected met the following inclusion criteria: aged 18 or older and in chronic phase at the time of diagnosis, Philadelphia chromosome and/or BCR-ABL positive, received first-line treatment with imatinib, and not enrolled in a randomized clinical trial during the period of retrospective review. Multivariable logistic regression models were used to evaluate prognostic indicators of complete hematological response (CHR), complete cytogenetic response (CCyR), and complete or major molecular response (C/MMR). Among patients treated with first-line imatinib, CHR at three months, CCyR at 12 months, and C/MMR at 18 months were observed in 53, 53.1, and 57.8 % of patients, respectively. Among patients treated with second-line dasatinib or nilotinib, CHR was achieved at three months in 49 and 42 %, CCyR at 12 months in 32 and 23 %, and MMR at 18 months in 30.5 and 26.1 % of patients, respectively. Prognostic indicators of first-line response included age, race, and Sokal score. For second-line treatment, duration of first-line hematological response and choice of drug used were also significant.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_014',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 136}]}],\n",
       "   'context': 'First-line imatinib mesylate in patients with newly diagnosed accelerated phase-chronic myeloid leukemia. Imatinib mesylate is the sole BCR-ABL tyrosine kinase inhibitor approved as first-line treatment of accelerated-phase (AP) chronic myeloid leukemia (CML). Indication was based on the STI571 0109 study, in which imatinib favorably compared to historical treatments in patients failing prior therapies. The relevance of these results to currently newly diagnosed AP-CML patients remains unknown. We evaluated the benefit of imatinib in 42 newly diagnosed AP-CML patients. In all, 16 patients had hematological acceleration without chromosomal abnormalities in addition to the Philadelphia chromosome (ACAs; HEM-AP), 16 solely had ACAs (ACA-AP) and 10 had hematological acceleration plus ACAs (HEM-AP + ACA). Major cytogenetic responses were achieved in 93.7% of HEM-AP patients, 75% of patients with ACA-AP (P=NS) and 40% of patients with HEM-AP + ACA (P=0.0053). The 24-month failure-free survival rate was 87.5% in HEM-AP patients, 43.8% in ACA-AP patients and 15% in HEM-AP + ACA patients (P=0.022). The 24-month estimate of progression-free survival was 100% in HEM-AP patients, 92.8% in ACA-AP patients and 58.3% in HEM-AP + ACA patients (P=0.0052). In conclusion, frontline imatinib allows favorable outcomes in HEM-AP and ACA-AP patients but appears insufficient for patients with HEM-AP + ACA. Broader-target and/or more potent BCR-ABL tyrosine kinase inhibitors alone or in combination may be considered in this setting.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_015',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 166}]}],\n",
       "   'context': 'Phase I trial of the combination of flavopiridol and imatinib mesylate in patients with Bcr-Abl+ hematological malignancies. PURPOSE: Imatinib is an inhibitor of the Bcr-Abl tyrosine kinase; however, resistance is common. Flavopiridol, a cyclin-dependent kinase (CDK) inhibitor, down-regulates short-lived anti-apoptotic proteins via inhibition of transcription. In preclinical studies, flavopiridol synergizes with imatinib to induce apoptosis. We investigated this novel combination regimen in patients with Bcr-Abl(+) malignancies. METHODS: In a phase I dose-escalation study, imatinib was administered orally daily, and flavopiridol by 1\\xa0h intravenous infusion weekly for 3\\xa0weeks every 4\\xa0weeks. Adults with chronic myelogenous leukemia or Philadelphia chromosome-positive acute leukemia were eligible. Patients were divided into two strata based on peripheral blood and bone marrow blast counts. The primary objective was to identify the recommended phase II doses for the combination. Correlative pharmacokinetic and pharmacodynamic studies were also performed. RESULTS: A total of 21 patients received study treatment. Four dose levels were evaluated before the study was closed following the approval of the second-generation Bcr-Abl tyrosine kinase inhibitors (TKIs). Five patients responded, including four sustained responses. Four patients had stable disease. All but one responder, and all patients with stable disease had previously been treated with imatinib. One patient had a complete response sustained for 30\\xa0months. Changes in expression of phospho-Bcr/Abl, -Stat5, and Mcl-1 were monitored. No major pharmacokinetic interaction was observed. CONCLUSIONS: This is the first study to evaluate the combination of a CDK inhibitor and a TKI in humans. The combination of flavopiridol and imatinib is tolerable and produces encouraging responses, including in some patients with imatinib-resistant disease.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_016',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 189}]}],\n",
       "   'context': '[Factors associated with achievement and durability of cytogenetic response in patients with chronic myeloid leukemia treated with imatinib]. BACKGROUND/AIM: Imatinib mesylate, a selective Bcr-Abl tyrosine kinase inhibitor, has revolutionized the treatment of Bcr-Abl positive chronic myeloid leukemia and become the standard of care for this disease. The aim of this study was evaluation and analysis of cytogenetic response in different intervals and risk groups as well as finding association between pre-treatment characteristics and later probability of achievement of major cytogenetic response. METHODS: We analyzed a total of 22 adult patients with newly diagnosed Philadelphia positive early chronic phase chronic myeloid leukemia treated at our institution from June 2006 to December 2009. RESULTS: The median follow-up time for patients during treatment with imatinib was 25.7 months (range, 12-42 months). A complete hematologic response was achieved in all of the analyzed patients within 6 months from the start of the treatment. The major cytogenetic response rate was 81.8%, and the complete cytogenetic response rate was 72.7%. The patients with low or moderate relative risk had the rate of complementary achieving major and complete cytogenetic response of 75-90%. A multivariate analysis identified the following independent prognostic factors for achieving major cytogenetic response: the absence of splenomegaly, white blood cell count less than 10 x 10(9)/L, the platelet count less than 450 x 10(9)/L, the presence of less than 5% of bone marrow blasts and basophils, the absence of blasts in peripheral blood, the presence of less than 7% of basophils in peripheral blood. CONCLUSION: Patients who early achieve complete and major cytogenetic response as well as those with low and moderate relative risk have a higher rate of achieving and maintaining complete cytogenetic response. There are also characteristics of patients before treatment that may indicate the treatment outcome.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_017',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 260}]}],\n",
       "   'context': '[Factors associated with achievement and durability of cytogenetic response in patients with chronic myeloid leukemia treated with imatinib]. BACKGROUND/AIM: Imatinib mesylate, a selective Bcr-Abl tyrosine kinase inhibitor, has revolutionized the treatment of Bcr-Abl positive chronic myeloid leukemia and become the standard of care for this disease. The aim of this study was evaluation and analysis of cytogenetic response in different intervals and risk groups as well as finding association between pre-treatment characteristics and later probability of achievement of major cytogenetic response. METHODS: We analyzed a total of 22 adult patients with newly diagnosed Philadelphia positive early chronic phase chronic myeloid leukemia treated at our institution from June 2006 to December 2009. RESULTS: The median follow-up time for patients during treatment with imatinib was 25.7 months (range, 12-42 months). A complete hematologic response was achieved in all of the analyzed patients within 6 months from the start of the treatment. The major cytogenetic response rate was 81.8%, and the complete cytogenetic response rate was 72.7%. The patients with low or moderate relative risk had the rate of complementary achieving major and complete cytogenetic response of 75-90%. A multivariate analysis identified the following independent prognostic factors for achieving major cytogenetic response: the absence of splenomegaly, white blood cell count less than 10 x 10(9)/L, the platelet count less than 450 x 10(9)/L, the presence of less than 5% of bone marrow blasts and basophils, the absence of blasts in peripheral blood, the presence of less than 7% of basophils in peripheral blood. CONCLUSION: Patients who early achieve complete and major cytogenetic response as well as those with low and moderate relative risk have a higher rate of achieving and maintaining complete cytogenetic response. There are also characteristics of patients before treatment that may indicate the treatment outcome.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_018',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 558}]}],\n",
       "   'context': \"Tyrosine kinase inhibitor use in pediatric Philadelphia chromosome-positive acute lymphoblastic anemia. Until recently, pediatric Philadelphia chromosome-positive (Ph(+)) acute lymphoblastic leukemia (ALL) was associated with an extremely poor outcome when treated with chemotherapy alone, and only modest survival benefits were obtained with the widespread use of hematopoietic stem cell transplantation (HSCT). The development of first-generation (imatinib) and second-generation (dasatinib and nilotinib) tyrosine kinase inhibitors (TKIs) that target the BCR-ABL1 fusion protein produced by the Ph chromosome revolutionized the treatment of chronic myelogenous leukemia (CML). The Children's Oncology Group (COG) AALL0031 trial showed that the addition of imatinib to intensive chemotherapy did not cause increased toxicity and resulted in 3-year event-free survival rates that were more than double those of historical control data from the pre-imatinib era. These findings create a new paradigm for integrating molecularly targeted agents with conventional chemotherapy and call for a reassessment of the routine use of HSCT for children and adolescents with Ph(+) ALL. Second-generation TKIs have theoretical advantages over imatinib, and are now being tested in Ph(+) ALL. The focus of contemporary trials is to define the optimal use of chemotherapy, HSCT, and TKI in Ph(+) ALL. In the coming years, it is anticipated that additional agents will become available to potentiate TKI therapy and/or circumvent TKI resistance in Ph(+) ALL. Recent genomic studies have identified a subtype of high-risk pediatric B-cell-precursor ALL with a gene-expression profile similar to that of Ph(+) ALL, suggestive of active kinase signaling. Many of these Ph-like ALL cases harbor chromosome rearrangements and mutations that dysregulate cytokine receptor and kinase signaling, and these leukemias may also be candidates for TKI therapy.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_019',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 204}]}],\n",
       "   'context': 'Regulation of hTERT by BCR-ABL at multiple levels in K562 cells. BACKGROUND: The cytogenetic characteristic of Chronic Myeloid Leukemia (CML) is the formation of the Philadelphia chromosome gene product, BCR-ABL. Given that BCR-ABL is the specific target of Gleevec in CML treatment, we investigated the regulation of the catalytic component of telomerase, hTERT, by BCR-ABL at multiple levels in K562 cells. METHODS: Molecular techniques such as over expression, knockdown, real-time PCR, immunoprecipitation, western blotting, reporter assay, confocal microscopy, telomerase assays and microarray were used to suggest that hTERT expression and activity is modulated by BCR-ABL at multiple levels. RESULTS: Our results suggest that BCR-ABL plays an important role in regulating hTERT in K562 (BCR-ABL positive human leukemia) cells. When Gleevec inhibited the tyrosine kinase activity of BCR-ABL, phosphorylation of hTERT was downregulated, therefore suggesting a positive correlation between BCR-ABL and hTERT. Gleevec treatment inhibited hTERT at mRNA level and significantly reduced telomerase activity (TA) in K562 cells, but not in HL60 or Jurkat cells (BCR-ABL negative cells). We also demonstrated that the transcription factor STAT5a plays a critical role in hTERT gene regulation in K562 cells. Knockdown of STAT5a, but not STAT5b, resulted in a marked downregulation of hTERT mRNA level, TA and hTERT protein level in K562 cells. Furthermore, translocation of hTERT from nucleoli to nucleoplasm was observed in K562 cells induced by Gleevec. CONCLUSIONS: Our data reveal that BCR-ABL can regulate TA at multiple levels, including transcription, post-translational level, and proper localization. Thus, suppression of cell growth and induction of apoptosis by Gleevec treatment may be partially due to TA inhibition. Additionally, we have identified STAT5a as critical mediator of the hTERT gene expression in BCR-ABL positive CML cells, suggesting that targeting STAT5a may be a promising therapeutic strategy for BCR-ABL positive CML patients.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_020',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 224}]}],\n",
       "   'context': 'Regulation of hTERT by BCR-ABL at multiple levels in K562 cells. BACKGROUND: The cytogenetic characteristic of Chronic Myeloid Leukemia (CML) is the formation of the Philadelphia chromosome gene product, BCR-ABL. Given that BCR-ABL is the specific target of Gleevec in CML treatment, we investigated the regulation of the catalytic component of telomerase, hTERT, by BCR-ABL at multiple levels in K562 cells. METHODS: Molecular techniques such as over expression, knockdown, real-time PCR, immunoprecipitation, western blotting, reporter assay, confocal microscopy, telomerase assays and microarray were used to suggest that hTERT expression and activity is modulated by BCR-ABL at multiple levels. RESULTS: Our results suggest that BCR-ABL plays an important role in regulating hTERT in K562 (BCR-ABL positive human leukemia) cells. When Gleevec inhibited the tyrosine kinase activity of BCR-ABL, phosphorylation of hTERT was downregulated, therefore suggesting a positive correlation between BCR-ABL and hTERT. Gleevec treatment inhibited hTERT at mRNA level and significantly reduced telomerase activity (TA) in K562 cells, but not in HL60 or Jurkat cells (BCR-ABL negative cells). We also demonstrated that the transcription factor STAT5a plays a critical role in hTERT gene regulation in K562 cells. Knockdown of STAT5a, but not STAT5b, resulted in a marked downregulation of hTERT mRNA level, TA and hTERT protein level in K562 cells. Furthermore, translocation of hTERT from nucleoli to nucleoplasm was observed in K562 cells induced by Gleevec. CONCLUSIONS: Our data reveal that BCR-ABL can regulate TA at multiple levels, including transcription, post-translational level, and proper localization. Thus, suppression of cell growth and induction of apoptosis by Gleevec treatment may be partially due to TA inhibition. Additionally, we have identified STAT5a as critical mediator of the hTERT gene expression in BCR-ABL positive CML cells, suggesting that targeting STAT5a may be a promising therapeutic strategy for BCR-ABL positive CML patients.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_021',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 89}]}],\n",
       "   'context': 'Effects of BCR-ABL inhibitors on anti-tumor immunity. In chronic myeloid leukemia (CML), BCR-ABL-mediated oncogenic signaling can be successfully targeted with the BCRABL- inhibitors imatinib, nilotinib, and dasatinib leading to complete cytogenetic (Philadelphia chromosome not detectable upon cytogenetic testing of bone marrow) and even complete molecular (BCR-ABL not detectable by PCR in peripheral blood) responses. However, CML apparently can not be cured by BCR-ABL inhibitors alone, likely due to treatment-resistance of CML stem/progenitor cells, which provokes a relapse of disease after cessation of therapy. Evidence from patients treated with allogenic stem cell transplantation or IFN-α points to an important role of anti-tumor immunity for durable control of CML disease. Data from multiple in vitro and ex vivo studies indicate that BCR-ABL inhibitors may also influence anti-tumor immunity. Varying effects on different immune effector cell subsets and of the different compounds have been reported, the latter being due to their particular and diverse potency and spectrum of target kinases. As multiple approaches presently aim to combine BCR-ABL inhibition with immunotherapeutic strategies to improve disease control in CML, immunomodulatory effects of the available BCR-ABL inhibitors may be of direct clinical relevance. Here we review the available data regarding the effects of imatinib, nilotinib, and dasatinib on dendritic cells, T cells and natural killer cells as important cellular components of anti-tumor immunity.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_022',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 227}]}],\n",
       "   'context': 'Pigmentary changes in a patient treated with imatinib. Imatinib mesylate (STI 571; Gleevec; Novartis Pharmaceuticals, Basel, Switzerland) is an orally available tyrosine kinase inhibitor that targets a constitutively activated BCR-ABL tyrosine kinase with additional inhibitory effects on platelet derived growth factor (PDGF) receptors alpha and beta, and KIT. It has revolutionized the treatment of adult and pediatric patients with Philadelphia chromosome positive chronic myelogenous leukemia (CML) and is also FDA-approved for KIT-positive advanced gastrointestinal tumor (GIST) and dermatofibrosarcoma protuberans. A wide spectrum of dermatologic toxicities has been associated with this agent, among which a maculopapular rash is the most common event. In addition, a variety of pigmentary abnormalities of the skin and mucosal surfaces have been reported. Hypopigmentation is a well-recognized adverse effect. In contrast, paradoxical hyperpigmentation has only rarely been documented. In this case report we describe imatinib-induced cutaneous hyperpigmentation and graying of hair occurring in the same patient with dermatofibrosarcoma protuberans treated with imatinib.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_023',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 171}]}],\n",
       "   'context': 'Monitoring twenty-six chronic myeloid leukemia patients by BCR-ABL mRNA level in bone marrow:a single hospital experience. Chronic myeloid leukemia (CML) is caused by the BCR-ABL oncogene. The Philadelphia chromosome (Ph) from a reciprocal translocation, t(9;22) (q34;q11) causes a fusion gene, BCR-ABL, that encodes a constitutively active tyrosine kinase. Treatment of CML by imatinib is effective to control the tyrosyl phosphorylation of the protein related to the cell signaling. BCR-ABL mRNA is overexpressed in the minimal residual disease (MRD), known as an early sign of relapse. Between December 2005 and June 2008, we measured BCR-ABL mRNA levels in the bone marrow (BM) from patients by quantitative real-time polymerase chain reaction (RQ-PCR) in Aomori Prefectural Central Hospital. Eighty-six samples from 26 patients were collected. Among the 26 CML patients, 11 patients (42%) were in the pretreatment group. Seven (64%) of the 11 patients achieved complete molecular response (CMR). In the post-treatment group consisting of the remaining 15 patients, 9 (60%) patients achieved CMR. The patients receiving imatinib at a dose over 300 mg per day required 13 (6-77) months [median (range)] to achieve CMR. On the other hand, the patients receiving a dose below 300 mg per day required 29.5 (11-84) months [median (range)]. When BCR-ABL mRNA was detected during the treatment course of patients with CMR, careful observation of BCR-ABL mRNA was useful for tracking the clinical course of patients. In conclusion, the BCR-ABL mRNA level was useful for monitoring the clinical course in 26 patients with CML.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_024',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 295}]}],\n",
       "   'context': 'Monitoring twenty-six chronic myeloid leukemia patients by BCR-ABL mRNA level in bone marrow:a single hospital experience. Chronic myeloid leukemia (CML) is caused by the BCR-ABL oncogene. The Philadelphia chromosome (Ph) from a reciprocal translocation, t(9;22) (q34;q11) causes a fusion gene, BCR-ABL, that encodes a constitutively active tyrosine kinase. Treatment of CML by imatinib is effective to control the tyrosyl phosphorylation of the protein related to the cell signaling. BCR-ABL mRNA is overexpressed in the minimal residual disease (MRD), known as an early sign of relapse. Between December 2005 and June 2008, we measured BCR-ABL mRNA levels in the bone marrow (BM) from patients by quantitative real-time polymerase chain reaction (RQ-PCR) in Aomori Prefectural Central Hospital. Eighty-six samples from 26 patients were collected. Among the 26 CML patients, 11 patients (42%) were in the pretreatment group. Seven (64%) of the 11 patients achieved complete molecular response (CMR). In the post-treatment group consisting of the remaining 15 patients, 9 (60%) patients achieved CMR. The patients receiving imatinib at a dose over 300 mg per day required 13 (6-77) months [median (range)] to achieve CMR. On the other hand, the patients receiving a dose below 300 mg per day required 29.5 (11-84) months [median (range)]. When BCR-ABL mRNA was detected during the treatment course of patients with CMR, careful observation of BCR-ABL mRNA was useful for tracking the clinical course of patients. In conclusion, the BCR-ABL mRNA level was useful for monitoring the clinical course in 26 patients with CML.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_025',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 458}]}],\n",
       "   'context': 'BCR-ABL transcript dynamics support the hypothesis that leukemic stem cells are reduced during imatinib treatment. PURPOSE: Imatinib induces a durable response in most patients with Philadelphia chromosome-positive chronic myeloid leukemia, but it is currently unclear whether imatinib reduces the leukemic stem cell (LSC) burden, which may be an important step toward enabling safe discontinuation of therapy. In this article, we use mathematical models of BCR-ABL levels to make inferences on the dynamics of LSCs. EXPERIMENTAL DESIGN: Patients with at least 1 BCR-ABL transcript measurement on imatinib were included (N = 477). Maximum likelihood methods were used to test 3 potential hypotheses of the dynamics of BCR-ABL transcripts on imatinib therapy: (i) monoexponential, in which there is little, if any, decline in BCR-ABL transcripts; (ii) biexponential, in which patients have a rapid initial decrease in BCR-ABL transcripts followed by a more gradual response; and (iii) triexponential, in which patients first exhibit a biphasic decline but then have a third phase when BCR-ABL transcripts increase rapidly. RESULTS: We found that most patients treated with imatinib exhibit a biphasic decrease in BCR-ABL transcript levels, with a rapid decrease during the first few months of treatment, followed by a more gradual decrease that often continues over many years. CONCLUSIONS: We show that the only hypothesis consistent with current data on progenitor cell turnover and with the long-term, gradual decrease in the BCR-ABL levels seen in most patients is that these patients exhibit a continual, gradual reduction of the LSCs. This observation may explain the ability to discontinue imatinib therapy without relapse in some cases.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_026',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 408}]}],\n",
       "   'context': 'Follow-up of childhood chronic myelogenous leukemia with monitoring the BCR-ABL fusion gene expression in peripheral blood. UNLABELLED: Chronic myelogenous leukemia (CML) accounts for 15-20% of adult leukemias but is very rare in children (2%). Fewer than 10% of CML patients are younger than 20 years. CML is a myeloproliferative disorder characterized by the presence of the Philadelphia chromosome or the BCR-ABL fusion oncogene. The objective of this paper is to present the monitoring of imatinib therapy in two children with CML by the BCR-ABL fusion gene expression assessment from peripheral blood with quantitative real-time polymerase chain reaction (PCR) method. PATIENTS AND METHODS: The 18 and six months follow-up of the patients included clinical examination, routine laboratory tests, bone marrow aspirate investigation including cytogenetic tests and the major BCR-ABL fusion gene expression measurement with qRT-PCR method from the peripheral blood. RESULTS: Patient No. 1 diagnosed with chronic phase CML showed excellent adherence to daily 400 mg imatinib treatment and achieved complete hematologic (CHR) and cytogenetic response (CCR) by three months and major molecular response (MMR) by 12 months, with lack of side effects due to imatinib. Patient No. 2 experienced severe hematologic toxicity, which necessitated temporary withdrawal of the drug. Transient non-compliance together with imatinib dose reduction has driven to treatment failure. In this case, mutational analysis is warranted. CONCLUSIONS: BCR-ABL fusion gene expression level measurement from peripheral blood with qRT-PCR method is an excellent tool in the follow-up of CML patients.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_027',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 1169}]}],\n",
       "   'context': \"Use of tyrosine kinase inhibitors for chronic myeloid leukemia: management of patients and practical applications for pharmacy practitioners. OBJECTIVE: To summarize the use of tyrosine kinase inhibitors (TKIs) for treatment of patients with chronic myeloid leukemia (CML) and provide practical information for patient management. DATA SOURCES: Literature was retrieved from PubMed (2000-January 2011), using the search terms chronic myeloid leukemia and tyrosine kinase inhibitor. Abstracts presented at the 2008-2010 annual meetings of the American Society of Hematology and the American Society of Clinical Oncology, reference citations from identified publications, as well as the manufacturers' full prescribing information for cited drugs, also were reviewed. STUDY SELECTION AND DATA EXTRACTION: Articles evaluating the efficacy and safety of the TKIs imatinib, nilotinib, and dasatinib were evaluated. Focus was placed on publications supporting management of patients with CML in the chronic phase. Reports presenting clinical trial information of TKIs in development also were included. DATA SYNTHESIS: The discovery of targeted tyrosine kinase inhibition of BCR-ABL kinase dramatically changed the treatment of CML. Imatinib, the first TKI approved for treatment of patients with Philadelphia chromosome--positive CML, demonstrated significant superiority over the previous standard of care: interferon plus cytarabine. The newer, more potent TKIs, nilotinib and dasatinib, have demonstrated improved efficacy over imatinib as first-line therapy and provide an effective option for patients with resistance or intolerance to imatinib. CONCLUSIONS: To maximize efficacy of TKI therapy, close patient management, involving frequent monitoring of patient response, is essential. Given the importance of continuing TKI therapy, early recognition and management of adverse events are critical to optimizing outcomes in patients with CML. In addition to the safety profile and considerations of comorbidities, additional factors can affect therapeutic selection, including drug-drug and drug-food interactions. Research investigating new therapies, particularly for patients harboring the T315I mutation-which remains refractory to current TKIs-continues in the quest to improve outcomes in patients with CML.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_028',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 312}]}],\n",
       "   'context': 'Dasatinib treatment for imatinib resistant or intolerant patients with chronic myeloid leukaemia. Chronic myeloid leukaemia (CML) is a genetically associated malignancy of haematopoietic stem cells, characterized by a t(9;22) translocation that forms the Philadelphia chromosome and creates a novel fusion gene, BCR-ABL. Treatment with molecular-targeted therapy is usually initiated with imatinib, an inhibitor of BCR-ABL tyrosine kinase. Imatinib resistance is, however, observed in some CML patients, especially in those with advanced disease. Through computerized literature searches, a systematic analysis was conducted to examine the efficacy and benefits of dasatinib therapy for imatinib resistant or intolerant CML patients in the chronic phase (CP), accelerated phase (AP) and fatal blast crisis phase (BC). In terms of major haematological and cytogenetic responses, this meta-analysis showed no significant differences in dasatinib treatment between myeloid BC-CML and lymphoid BC-CML patients with imatinib resistance or intolerance. Dasatinib therapy was, however, significantly more effective in improving major haematological and cytogenetic responses for CP-CML patients than for AP-CML patients with imatinib resistance or intolerance.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_029',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 671}]}],\n",
       "   'context': 'Successful treatment with nilotinib after imatinib failure in a CML patient with a four-way Ph chromosome translocation and point mutations in BCR/ABL gene. Chronic myelogenous leukemia (CML) is characterized by Philadelphia (Ph) chromosome with a chimeric gene BCR-ABL created by reciprocal t(9:22) (q34;q11) translocation. Variant Ph chromosome translocations involving chromosomes other than 9 and 22 are found in 5-10% of CML cases. We here report a CML patient who carries a four-way Ph chromosome translocation, t(9;22;15;19) (q34;q11;q15;q13). The patient was diagnosed in 1997 and initially treated with hydroxyurea. In 2002, treatment with imatinib, a selective BCR-ABL tyrosine kinase inhibitor (TKI), was started but Ph-positive chromosomes remained at the levels of 42-65%, indicating imatinib failure. In 2006, the point mutations of F359I and L387M were detected in BCR/ABL gene, which may be related to imatinib failure. Treatment with nilotinib, a TKI with high target specificity, was then started which resulted in durable major molecular response. Administration of nilotinib offered an effective treatment in a CML patient with variant Ph chromosome translocations and BCR-ABL point mutations after imatinib failure.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_030',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 731}]}],\n",
       "   'context': 'Molecular and cellular bases of chronic myeloid leukemia. Chronic myeloid leukemia (CML) is a myeloproliferative disease characterized by the overproduction of granulocytes, which leads to high white blood cell counts and splenomegaly in patients. Based on clinical symptoms and laboratory findings, CML is classified into three clinical phases, often starting with a chronic phase, progressing to an accelerated phase and ultimately ending in a terminal phase called blast crisis. Blast crisis phase of CML is clinically similar to an acute leukemia; in particular, B-cell acute lymphoblastic leukemia (B-ALL) is a severe form of acute leukemia in blast crisis, and there is no effective therapy for it yet. CML is induced by the BCR-ABL oncogene, whose gene product is a BCR-ABL tyrosine kinase. Currently, inhibition of BCR-ABL kinase activity by its kinase inhibitor such as imatinib mesylate (Gleevec) is a major therapeutic strategy for CML. However, the inability of BCR-ABL kinase inhibitors to completely kill leukemia stem cells (LSCs) indicates that these kinase inhibitors are unlikely to cure CML. In addition, drug resistance due to the development of BCRABL mutations occurs before and during treatment of CML with kinase inhibitors. A critical issue to resolve this problem is to fully understand the biology of LSCs, and to identify key genes that play significant roles in survival and self-renewal of LSCs. In this review, we will focus on LSCs in CML by summarizing and discussing available experimental results, including the original studies from our own laboratory.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_031',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 773}]}],\n",
       "   'context': 'Molecular and cellular bases of chronic myeloid leukemia. Chronic myeloid leukemia (CML) is a myeloproliferative disease characterized by the overproduction of granulocytes, which leads to high white blood cell counts and splenomegaly in patients. Based on clinical symptoms and laboratory findings, CML is classified into three clinical phases, often starting with a chronic phase, progressing to an accelerated phase and ultimately ending in a terminal phase called blast crisis. Blast crisis phase of CML is clinically similar to an acute leukemia; in particular, B-cell acute lymphoblastic leukemia (B-ALL) is a severe form of acute leukemia in blast crisis, and there is no effective therapy for it yet. CML is induced by the BCR-ABL oncogene, whose gene product is a BCR-ABL tyrosine kinase. Currently, inhibition of BCR-ABL kinase activity by its kinase inhibitor such as imatinib mesylate (Gleevec) is a major therapeutic strategy for CML. However, the inability of BCR-ABL kinase inhibitors to completely kill leukemia stem cells (LSCs) indicates that these kinase inhibitors are unlikely to cure CML. In addition, drug resistance due to the development of BCRABL mutations occurs before and during treatment of CML with kinase inhibitors. A critical issue to resolve this problem is to fully understand the biology of LSCs, and to identify key genes that play significant roles in survival and self-renewal of LSCs. In this review, we will focus on LSCs in CML by summarizing and discussing available experimental results, including the original studies from our own laboratory.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_032',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 823}]}],\n",
       "   'context': 'Molecular and cellular bases of chronic myeloid leukemia. Chronic myeloid leukemia (CML) is a myeloproliferative disease characterized by the overproduction of granulocytes, which leads to high white blood cell counts and splenomegaly in patients. Based on clinical symptoms and laboratory findings, CML is classified into three clinical phases, often starting with a chronic phase, progressing to an accelerated phase and ultimately ending in a terminal phase called blast crisis. Blast crisis phase of CML is clinically similar to an acute leukemia; in particular, B-cell acute lymphoblastic leukemia (B-ALL) is a severe form of acute leukemia in blast crisis, and there is no effective therapy for it yet. CML is induced by the BCR-ABL oncogene, whose gene product is a BCR-ABL tyrosine kinase. Currently, inhibition of BCR-ABL kinase activity by its kinase inhibitor such as imatinib mesylate (Gleevec) is a major therapeutic strategy for CML. However, the inability of BCR-ABL kinase inhibitors to completely kill leukemia stem cells (LSCs) indicates that these kinase inhibitors are unlikely to cure CML. In addition, drug resistance due to the development of BCRABL mutations occurs before and during treatment of CML with kinase inhibitors. A critical issue to resolve this problem is to fully understand the biology of LSCs, and to identify key genes that play significant roles in survival and self-renewal of LSCs. In this review, we will focus on LSCs in CML by summarizing and discussing available experimental results, including the original studies from our own laboratory.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_033',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 518}]}],\n",
       "   'context': 'Management of patients with newly diagnosed chronic myeloid leukemia: opportunities and challenges. Chronic myelogenous leukemia (CML) is a progressive and often fatal hematopoietic neoplasm characterized by the presence of the Philadelphia chromosome. This arises from a balanced translocation between chromosomes 9 and 22, creating the bcr-abl fusion gene. It is often stated that the only proven curative option is allogeneic stem cell transplantation, which is indicated for only a limited subset of patients. The Bcr-Abl tyrosine kinase inhibitor imatinib represented a major advance over conventional CML therapy. After imatinib treatment, > 90% of patients had a complete hematologic response, and 70%-80% had a complete cytogenetic response. With 5 years of follow-up, the data are very encouraging and exhibit a major change in the natural history of the disease. The understanding of some of the mechanisms of resistance to imatinib has led to a rapid development of new agents that might overcome this resistance. The outlook today for patients with CML is much brighter than that of a few years ago.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_034',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 346}]}],\n",
       "   'context': 'Optimizing therapy of chronic myeloid leukemia. Chronic myeloid leukemia (CML) is caused by Bcr-Abl, a constitutively active tyrosine kinase that is the result of a reciprocal translocation between chromosomes 9 and 22 and cytogenetically evident as the Philadelphia chromosome. Imatinib (Glivec, Gleevec), a specific small molecule inhibitor of Bcr-Abl, has become the standard drug therapy for CML, and has dramatically diminished the use of allogeneic stem cell transplantation. Despite unprecedented rates of complete cytogenetic response, residual disease remains detectable in the majority of patients, suggesting that imatinib fails to eradicate leukemic stem cells. In this publication, the current perspectives for CML patients treated with imatinib are reviewed, focusing on the results of both standard and high-dose therapy. Monitoring of time-dependent prognostic factors is reviewed. The reasons imatinib may not be able to eradicate the disease are discussed, and potential strategies to achieve disease elimination are presented. Lastly, resistance to imatinib and the potential of second-generation Abl kinase inhibitors in the setting of clinical resistance are considered.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_035',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 461}]}],\n",
       "   'context': 'Imatinib-induced immune hepatitis: case report and literature review. Imatinib is one of the most recent medications used for the treatment of chronic myeloid leukemia (CML) and gastrointestinal stromal tumor (GIST). It is an orally administered protein-tyrosine kinase inhibitor, an enzyme which is produced by BCR-ABL fusion which results from translocation of 9:22 chromosome (Philadelphia chromosome). Imatinib blocks proliferation and induces apoptosis of BCR-ABL-expression in CML. Many side effects produced by imatinib have been documented but its induction of hepatotoxcity has been rarely reported. Only a few cases so far have been reported in the literature and almost all were in females. We describe another case of hepatotoxicity due to imatinib in a 17-year old female with clinical, laboratory and histopathological changes. The case described here suggests that imatinib may also induce immune hepatitis, in some patients.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_036',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'bcr-abl', 'answer_start': 489}]}],\n",
       "   'context': 'The biology of chronic myelogenous leukemia: implications for imatinib therapy. Chronic myelogenous leukemia (CML) results from the neoplastic transformation of primitive hematopoietic stem cells, and has been classified as a myeloproliferative disorder. The hallmark of CML is the presence of a balanced translocation between the long arms of chromosomes 9 and 22, t(9;22)(q34;q11.2), which is known as the Philadelphia (Ph) chromosome. This translocation results in the formation of the bcr-abl fusion gene, which, in turn, is translated into a chimeric Bcr-Abl protein with deregulated tyrosine kinase activity. Constitutive Bcr-Abl expression has been shown to be necessary and sufficient for the transformed phenotype of CML cells. CML is unique among human cancers in that a single genetic defect, the Ph chromosome, is responsible for the transformed phenotype. Since this discovery more than 40 years ago, our understanding of the clinical course, therapy, and prognosis of patients with CML has changed significantly. These changes have culminated in the emergence of imatinib, the first rationally designed, molecularly targeted therapy for human malignancy. In this review, the authors describe the molecular biology of CML and the development of imatinib as a therapeutic agent for the treatment of CML.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_037',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 227}]}],\n",
       "   'context': 'Monitoring minimal residual disease and controlling drug resistance in chronic myeloid leukaemia patients in treatment with imatinib as a guide to clinical management. Imatinib mesylate, binding to the inactive conformation of Bcr-Abl tyrosine kinase and suppressing the Ph chromosome positive clone, has revolutionized the treatment of chronic myeloid leukaemia (CML) patients. Given the high rates of clinical and cytogenetic remission achieved, the molecular monitoring of BCR-ABL transcript levels by RT-qPCR has become always more important to assess minimal residual disease. Recently, recommendations for harmonizing current methodologies for detecting and measuring BCR-ABL transcripts in CML patients have been suggested. Studies of imatinib-treated patients have determined that the BCR-ABL levels measured early in therapy may predict durable cytogenetic remission and in turn prolonged progression free-survival or acquisition of resistance. The major mechanism of imatinib resistance is clonal expansion of leukaemia cells with mutations in the Bcr-Abl fusion tyrosine kinase. The early reduction of such mutations may allow timely treatment intervention to prevent or overcome resistance. We review current trends in the management of chronic myeloid leukaemia patients undergoing treatment with tyrosine kinase inhibitors.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_038',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'bcr-abl', 'answer_start': 195}]}],\n",
       "   'context': 'Treatment of chronic myeloid leukemia with imatinib mesylate. Philadelphia (Ph) chromosome is the cytogenetic hallmark of chronic myeloid leukemia (CML). The translocation forms a chimeric gene, bcr-abl, which generates BCR-ABL. This fusion protein constitutively activate ABL tyrosine kinase and causes CML. Imatinib mesylate is a selective tyrosine kinase inhibitor on ABL, c-Kit and PGDF-receptor, and functions through competitive inhibition at the ATP-binding site of the enzyme, which leads to growth arrest or apoptosis in cells that express BCR-ABL. Imatinib has revolutionized the management of patients with CML, and at a dose of 400 mg daily has become the current standard therapy for newly diagnosed patients with CML even when they have HLA-matched family donors. Although imatinib therapy has only a 5-year history, it is hoped that CML will be cured with this drug and with forthcoming second-generation tyrosine kinase inhibitors as well as by allogeneic stem cell transplantation in patients who have become resistant to these drugs.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_039',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 220}]}],\n",
       "   'context': 'Treatment of chronic myeloid leukemia with imatinib mesylate. Philadelphia (Ph) chromosome is the cytogenetic hallmark of chronic myeloid leukemia (CML). The translocation forms a chimeric gene, bcr-abl, which generates BCR-ABL. This fusion protein constitutively activate ABL tyrosine kinase and causes CML. Imatinib mesylate is a selective tyrosine kinase inhibitor on ABL, c-Kit and PGDF-receptor, and functions through competitive inhibition at the ATP-binding site of the enzyme, which leads to growth arrest or apoptosis in cells that express BCR-ABL. Imatinib has revolutionized the management of patients with CML, and at a dose of 400 mg daily has become the current standard therapy for newly diagnosed patients with CML even when they have HLA-matched family donors. Although imatinib therapy has only a 5-year history, it is hoped that CML will be cured with this drug and with forthcoming second-generation tyrosine kinase inhibitors as well as by allogeneic stem cell transplantation in patients who have become resistant to these drugs.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_040',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 146}]}],\n",
       "   'context': 'Random aneuploidy in CML patients at diagnosis and under imatinib treatment. Chronic myeloid leukemia (CML) is characterized by the presence of a BCR-ABL fusion gene, which is the result of a reciprocal translocation between chromosomes 9 and 22, and is cytogenetically visible as a shortened chromosome 22 (Philadelphia). Research during the past two decades has established that BCR-ABL is probably the pathogenetic pathway leading to CML, and that constitutive tyrosine kinase activity is central to BCR-ABL capacity to transform hematopoietic cells in vitro and in vivo. The tyrosine kinase inhibitor imatinib mesylate was introduced into the treatment regimen for CML in 1998. During the last few years, reports on chromosomal changes during imatinib treatment have been described. In this study, we evaluated the random aneuploidy rate with chromosomes 9 and 18 in bone marrow from treated and untreated patients. We found higher aneuploidy rates in both treated and untreated patients compared to the control group. In three patients who were treated with imatinib mesylate for more than 1.5 years, triploidy also appeared in some nuclei. To our knowledge, this is the first report on new chromosomal changes such as random aneuploidy and triploidy under imatinib treatment, but more studies are needed to investigate the long-term effect of the imatinib treatment on genetic instability.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_041',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 154}]}],\n",
       "   'context': 'The second generation of BCR-ABL tyrosine kinase inhibitors. Imatinib was developed as the first molecularly targeted therapy to specifically inhibit the BCR-ABL kinase in Philadelphia chromosome (Ph)-positive chronic myeloid leukemia (CML). Because of the excellent hematologic and cytogenetic responses, imatinib has moved toward first-line treatment for newly diagnosed CML. However, the emergence of resistance to imatinib remains a major problem in the treatment of Ph-positive leukemia. Several mechanisms of imatinib resistance have been identified, including BCR-ABL gene amplification that leads to overexpression of the BCR-ABL protein, point mutations in the BCR-ABL kinase domain that interfere with imatinib binding, and point mutations outside of the kinase domain that allosterically inhibit imatinib binding to BCR-ABL. The need for alternative or additional treatment for imatinib-resistant BCR-ABL-positive leukemia has guided the way to the design of a second generation of targeted therapies, which has resulted mainly in the development of novel small-molecule inhibitors such as AMN107, dasatinib, NS-187, and ON012380. The major goal of these efforts is to create new compounds that are more potent than imatinib and/or more effective against imatinib-resistant BCR-ABL clones. In this review, we discuss the next generation of BCR-ABL kinase inhibitors for overcoming imatinib resistance.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_042',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 264}]}],\n",
       "   'context': '[Tyrosine kinase inhibitors for the treatment of CML]. Chronic myelogenous leukemia is characterized by the Philadelphia-chromosome, a shortened chromosome 22 which is the result of a reciprocal translocation between chromosome 9 and 22. The fusion gene is called BCR-ABL. After transcription and translation the constitutively activated p210 BCR-ABL oncoprotein is formed. This leads to uncontrolled activation of the ABL tyrosin kinase. Deregulated cellular proliferation and diminished apoptosis of BCR-ABL transformed cells is the result. Expression of the BCR-ABL oncoprotein is sufficient and necessary for the development of a CML phenotype. Imatinib mesylate (Glivec) is a small molecule that binds to the ATP pocket of ABL and blocks downstream signalling events. Imatinib is very effective in the treatment of CML in all stages of the disease. Patients with newly diagnosed chronic phase CML were randomized to imatinib or to interferon plus cytarabine in the IRIS trial. Imatinib showed significantly superior tolerability, hematologic and cytogenetic resposes and increased time to progression. In patients with advanced phase CML, imatinib is less effective and response duration is short. Median overall survival of blast crisis patients is 6.9 months only. Additional BCR-ABL independent chromosomal abnormalities are common in advanced phase CML and result in resistance to imatinib. BCR-ABL kinase-domaine mutations are frequently found in imatinib resistant patients and confer diminished sensitivity to imatinib. Second generation, more powerful ABL kinase inhibitors, which are effective against most of the known mutations are currently tested in clinical trials.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_043',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 603}]}],\n",
       "   'context': 'Differential molecular response of the transcripts B2A2 and B3A2 to imatinib mesylate in chronic myeloid leukemia. Chronic myeloid leukemia (CML) originates from the hematopoietic stem cell and is characterized by the reciprocal translocation t(9;22)(q34;q11), which results in the BCR-ABL fusion gene on chromosome 22q-, also known as the Philadelphia chromosome. This chimeric gene codes for a cytoplasmic protein with constitutive tyrosine-kinase activity, responsible for cellular transformation and leukemogenesis in CML. The aim of this observational cohort study was to discriminate and quantify BCR-ABL transcripts in the peripheral blood of patients with CML who were treated with imatinib mesylate (Glivec, Novartis). Twenty-two patients were followed for six months during treatment. Quantitative real time polymerase chain reaction was performed before treatment and after 3 and 6 months from treatment initiation. As compared with the third month, there was a significant decrease in BCR-ABL expression in the sixth month of treatment (P = 0.0002). At the sixth month, there was a significant difference in the levels of the two major transcripts of BCR-ABL, B2A2 and B3A2 (P = 0.0347), indicating that B2A2 may be more sensitive to imatinib. The results of our study indicate that imatinib is able to modify the natural history of CML, and raise the hypothesis that patients who express the B2A2 transcript may have a better prognosis.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_044',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 511}]}],\n",
       "   'context': \"Molecularly targeted treatment of chronic myeloid leukemia: beyond the imatinib era. Chronic myeloid leukemia cells contain a BCR-ABL oncoprotein with an enhanced tyrosine kinase activity, which is considered to be the principal 'cause' of the leukemia. Though the precise mechanisms underlying the leukemogenesis remains enigmatic, the use of imatinib to inhibit the dysregulated kinase activity has proved remarkably successful in clinical practice. Imatinib was the first small molecule developed to inhibit BCR-ABL tyrosine kinase activity and its success introduced the current era of molecularly targeted therapies for a number of other malignancies. In patients with chronic myeloid leukaemia who develop resistance to imatinib, the Bcr-Abl signaling pathway is often re-established. This has led to the emergence of a number of alternative treatment strategies designed to target the leukemic cell which are resistant to imatinib.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_045',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 126}]}],\n",
       "   'context': \"Molecularly targeted treatment of chronic myeloid leukemia: beyond the imatinib era. Chronic myeloid leukemia cells contain a BCR-ABL oncoprotein with an enhanced tyrosine kinase activity, which is considered to be the principal 'cause' of the leukemia. Though the precise mechanisms underlying the leukemogenesis remains enigmatic, the use of imatinib to inhibit the dysregulated kinase activity has proved remarkably successful in clinical practice. Imatinib was the first small molecule developed to inhibit BCR-ABL tyrosine kinase activity and its success introduced the current era of molecularly targeted therapies for a number of other malignancies. In patients with chronic myeloid leukaemia who develop resistance to imatinib, the Bcr-Abl signaling pathway is often re-established. This has led to the emergence of a number of alternative treatment strategies designed to target the leukemic cell which are resistant to imatinib.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_046',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 283}]}],\n",
       "   'context': \"Hematopathologic and cytogenetic findings in imatinib mesylate treated chronic myelogenous leukemia patients: 2.5 years' experience. BACKGROUND/AIM: Imatinib mesylate, a tyrosine kinase inhibitor with specific activity against the breakpoint cluster region--Abelson murine leukemia (BCR-ABL) tyrosine kinase has been developed for treatment of chronic myelogenous leukemia (CML). Its hematologic and cytogenetic effects have been evaluated in a series of clinical trials. The aim of this study was to report hematologic and cytogenetic response in CML patients during the treatment with imatinib mesylate. METHODS: A total of 21 patients were treated and observed from July 2006 to December 2008. The median time from CML diagnosis was no more than 12 months, so all the patients received previous treatment with hydroxyurea for which the median time was 3 months. The patients received imatinib mesylate in an effective oral dose of 400 to 800 mg daily, which was followed with peripheral blood counts, bone marrow examination, and cytogenetic studies at 6, 12, 18 and 24 months. RESULTS: Complete hematologic responses were reported for 19 (90.48%) of 21 patients studied. Among 19 patients who had a response, 16 (86%) did so within 3 months. The best cytogenetic response rate at any time during the study treatment with imatinib mesylate, among 14 patients in which cytogenetic response evaluated was: complete cytogenetic response in 7 (50%) patients, partial cytogenetic response in 6 (42.9%) patients and minor cytogenetic response in 1 (7.1%) patient. No patients had progressed to accelerated or blastic phase. The most frequent adverse effects that seemed to be related to treatment with imatinib mesylate were edema and musculoskeletal pain; overall, most were mild. Only one patient discontinued treatment because of hematologic toxic effects. CONCLUSION: The results obtained in this study confirm that imatinib mesylate induces a complete hematological and cytogenetic response in a high percentage of patients with chronic-phase CML.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_047',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 403}]}],\n",
       "   'context': \"Long-term outcomes in the second-line treatment of chronic myeloid leukemia: a review of tyrosine kinase inhibitors. Chronic myeloid leukemia (CML) is a progressive and often fatal myeloproliferative neoplasm. The hallmark of CML is an acquired chromosomal translocation known as the Philadelphia chromosome (Ph), which results in the synthesis of the breakpoint cluster region-Abelson murine leukemia (BCR-ABL) fusion oncoprotein, a constitutively active tyrosine kinase. The introduction of imatinib, a tyrosine kinase inhibitor (TKI) that is specific for BCR-ABL, was a major breakthrough in CML therapy. Although most patients respond to first-line imatinib therapy, some experience loss of response (resistance) or require treatment discontinuation because of toxicity (intolerance). For patients with CML, failure on standard-dose imatinib therapy (400 mg daily), imatinib dose escalation (600-800 mg daily) is a second-line option. However, high-dose imatinib is not an appropriate approach for patients who experience drug toxicity, and there remain questions over the durability of responses achieved with this strategy. Alternative second-line options include the TKIs dasatinib and nilotinib. A substantial amount of long-term data for these agents is available. Although both are potent and specific BCR-ABL TKIs, dasatinib and nilotinib exhibit unique pharmacologic profiles and response patterns relative to different patient characteristics, such as disease stage and BCR-ABL mutation status. To optimize therapeutic benefit, clinicians should select treatment based on each patient's historic response, adverse-event tolerance, and risk factors.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_048',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 558}]}],\n",
       "   'context': \"Long-term outcomes in the second-line treatment of chronic myeloid leukemia: a review of tyrosine kinase inhibitors. Chronic myeloid leukemia (CML) is a progressive and often fatal myeloproliferative neoplasm. The hallmark of CML is an acquired chromosomal translocation known as the Philadelphia chromosome (Ph), which results in the synthesis of the breakpoint cluster region-Abelson murine leukemia (BCR-ABL) fusion oncoprotein, a constitutively active tyrosine kinase. The introduction of imatinib, a tyrosine kinase inhibitor (TKI) that is specific for BCR-ABL, was a major breakthrough in CML therapy. Although most patients respond to first-line imatinib therapy, some experience loss of response (resistance) or require treatment discontinuation because of toxicity (intolerance). For patients with CML, failure on standard-dose imatinib therapy (400 mg daily), imatinib dose escalation (600-800 mg daily) is a second-line option. However, high-dose imatinib is not an appropriate approach for patients who experience drug toxicity, and there remain questions over the durability of responses achieved with this strategy. Alternative second-line options include the TKIs dasatinib and nilotinib. A substantial amount of long-term data for these agents is available. Although both are potent and specific BCR-ABL TKIs, dasatinib and nilotinib exhibit unique pharmacologic profiles and response patterns relative to different patient characteristics, such as disease stage and BCR-ABL mutation status. To optimize therapeutic benefit, clinicians should select treatment based on each patient's historic response, adverse-event tolerance, and risk factors.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_049',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 787}]}],\n",
       "   'context': 'Chronic myeloid leukemia: a historical perspective. Patients with splenomegaly and abnormally high leukocyte counts were first recognized in France, Germany, and Scotland in the 1840s. The only well-documented therapy in the 19th century was use of arsenic in one or other form, which did undoubtedly reduce the leukocyte count but probably did little or nothing to prolong life. These early cases were probably examples of chronic myeloid leukemia (CML) (then called chronic granulocytic leukemia). In the 20th century important steps in unraveling the pathogenesis of CML were the discovery of the Philadelphia chromosome in 1960, and of the (9;22) translocation in 1973. There followed definition of the breakpoint cluster region on chromosome 22 in 1984 and the demonstration of the BCR-ABL transcript in CML in 1985. In the first half of the 20th century patients were treated predominantly with radiotherapy, and later on with busulfan, hydroxycarbamide, or interferon-alfa (IFN-α). From 1980 onwards allogeneic stem cell transplantation (SCT) became the treatment of choice for eligible patients. The era of tyrosine kinase inhibitors (TKI) began in 1998 and today the use of the original TKI, imatinib, has replaced SCT as initial therapy for patients who present with CML in chronic phase.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_050',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 186}]}],\n",
       "   'context': \"Prognostic factors associated with complete cytogenetic response in patients with chronic myelogenous leukemia on imatinib mesylate therapy. INTRODUCTION: Imatinib mesylate, a selective Bcr-Abl tyrosine kinase inhibitor, has proved to be most effective therapy of Philadelphia chromosome-positive chronic myelogenous leukemia. Imatinib induces complete haematological and cytogenetic response in high percentage of patients. OBJECTIVE: The aim of this study was to identify potential prognostic factors before beginning treatment with imatinib associated with complete cytogenetic response. METHODS: We analyzed 20 patients with newly diagnosed Philadelphia positive chronic myelogenous leukemia treated at our institution from June 2006 until May 2009. These patients were treated with imatinib mesylate in oral dose of 400 to 800 mg daily. Complete blood counts were performed every month, while serum chemistry evaluations and bone marrow evaluations including morphology and cytogenetics were performed every 6 months. RESULTS: Of the 20 patients analyzed in this study, 19 (95%) achieved complete haematologic response within three months. In all patients cytogenetic analyses were done and all have achieved absolute cytogenetic response. The best cytogenetic response rate at any time during study treatment among 20 patients was: complete cytogenetic response in 15, partial cytogenetic response in three and minor cytogenetic response in two patients. Among 11 observed base-line patients' characteristics five were independent predictors of a high rate of complete cytogenetic response; the absence of blasts and basophils in peripheral blood, the presence of less than 5 percent of bone marrow blasts, white blood cell count less than 10 x 10(9)/L and the absence of splenomegaly (p < 0.01). CONCLUSION: Our results showed that some pre-treatment characteristics of patients might be the cause of differences in treatment outcome. On the basis of this analysis, we identified several pre-treatment patients' characteristics to be independent prognostic factors for achievement of complete cytogenetic response.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_051',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 336}]}],\n",
       "   'context': \"Clinical algorithms for the treatment of patients with chronic myeloid leukemia: the 2010 perspective. Chronic myeloid leukemia (CML) is a progressive and often fatal myeloproliferative disorder. The hallmark of CML is an acquired chromosomal translocation known as the Philadelphia chromosome (Ph) that results in the synthesis of the BCR-ABL fusion protein, a constitutively active tyrosine kinase (TK). The introduction of imatinib, a TK inhibitor (TKI) specific for BCR-ABL, was a major breakthrough in CML therapy. Although most patients respond to first-line imatinib therapy, some experience a loss of response (resistance) or require treatment discontinuation because of toxicity (intolerance). In patients for whom standard-dose imatinib therapy (400 mg/day) fails, imatinib dose escalation (600-800 mg/day) is a second-line option. However, high-dose imatinib is not an appropriate approach for patients experiencing drug toxicity, and there remain questions over the durability of responses achieved with this strategy. Alternative second-line options include the newer TKIs dasatinib and nilotinib. A substantial amount of long-term data for these agents is available. Although both are potent and specific BCR-ABL TKIs, dasatinib and nilotinib exhibit unique pharmacologic profiles and response patterns relative to different patient characteristics, such as disease stage and BCR-ABL mutational status. To optimize therapeutic benefit, clinicians should select treatment based on each patient's historical response, adverse event tolerance level, and risk factors.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_052',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 470}]}],\n",
       "   'context': \"Clinical algorithms for the treatment of patients with chronic myeloid leukemia: the 2010 perspective. Chronic myeloid leukemia (CML) is a progressive and often fatal myeloproliferative disorder. The hallmark of CML is an acquired chromosomal translocation known as the Philadelphia chromosome (Ph) that results in the synthesis of the BCR-ABL fusion protein, a constitutively active tyrosine kinase (TK). The introduction of imatinib, a TK inhibitor (TKI) specific for BCR-ABL, was a major breakthrough in CML therapy. Although most patients respond to first-line imatinib therapy, some experience a loss of response (resistance) or require treatment discontinuation because of toxicity (intolerance). In patients for whom standard-dose imatinib therapy (400 mg/day) fails, imatinib dose escalation (600-800 mg/day) is a second-line option. However, high-dose imatinib is not an appropriate approach for patients experiencing drug toxicity, and there remain questions over the durability of responses achieved with this strategy. Alternative second-line options include the newer TKIs dasatinib and nilotinib. A substantial amount of long-term data for these agents is available. Although both are potent and specific BCR-ABL TKIs, dasatinib and nilotinib exhibit unique pharmacologic profiles and response patterns relative to different patient characteristics, such as disease stage and BCR-ABL mutational status. To optimize therapeutic benefit, clinicians should select treatment based on each patient's historical response, adverse event tolerance level, and risk factors.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_053',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 180}]}],\n",
       "   'context': 'Targeting chronic myeloid leukemia stem cells. Chronic myeloid leukemia (CML) arises as a consequence of a chromosomal translocation giving rise to the Philadelphia chromosome and Bcr-Abl oncogene. CML is a clonal disease of stem cell origin and an excellent example of a malignancy in which tumor-initiating cells may hold the key to disease eradication. The known molecular basis of CML has enabled the development of Abl-specific tyrosine kinase inhibitors, such as imatinib mesylate. However, the success of tyrosine kinase inhibitors, as rationally designed first-line therapies, has been tempered by problems of disease persistence and resistance. Residual disease has been shown to be enriched within the stem cell compartment and to persist at stable levels for up to 5 years of complete cytogenetic response. This finding has led to further searches for novel strategies aimed at eliminating these cells; such strategies may be essential in achieving cure. The most significant recent findings are discussed in this review.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_054',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 167}]}],\n",
       "   'context': '[Molecular pathogenesis of chronic myeloid leukemia]. Chronic myeloid leukemia (CML) is a paradigm for neoplasias that are defined by a unique genetic aberration, the BCR-ABL1 fusion gene. CML is also the best example for molecular target therapy. The development of protein tyrosine kinase inhibitor, imatinib, has entirely changed the strategy of therapy for CML. Nonetheless, many fields of pathogenesis for CML have not been elucidated, such as the mechanisms of blastic crisis, the causes of genetic instability including the inactivation of tumor suppressor genes, and oncogenic signaling pathways downstreams of the BCR-ABL1 fusion gene product. Herein, we review current knowledge on the molecular pathogenesis of CML.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_055',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 113}]}],\n",
       "   'context': 'Chronic myelogenous leukemia with the e6a2 BCR-ABL and lacking imatinib response: presentation of two cases. The BCR-ABL fusion gene represents the hallmark of chronic myelogenous leukemia (CML) and is derived from a translocation between chromosome 9 and 22. The majority of CML patients have a breakpoint in the major BCR region of the BCR gene giving rise to e13a2 or e14a2 BCR-ABL transcripts. Occasionally, other BCR breakpoints occur. The current report describes two e6a2 CML patients with imatinib treatment failure and unusual disease progression. One patient was Philadelphia chromosome positive and one was Philadelphia chromosome negative with an atypical BCR-ABL rearrangement, ins (22;9).'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_056',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 301}]}],\n",
       "   'context': 'Targeted drugs in chronic myeloid leukemia. Chronic myeloid leukemia (CML) is characterized by the presence of the Philadelphia (Ph) chromosome, which results from a reciprocal translocation between the long arms of the chromosomes 9 and 22 t(9;22)(q34;q11). This translocation creates two new genes, BCR-ABL on the 22q- (Ph chromosome) and the reciprocal ABL-BCR on 9q-. The BCR-ABL gene encodes for a 210-kD protein with deregulated tyrosine kinase (TK) activity, which is crucial for malignant transformation in CML. The recognition of the BCR-ABL gene and corresponding protein led to the synthesis of small-molecule drugs, designed to interfere with BCR-ABL tyrosine kinase activation by competitive binding at the ATP-binding site. The first tyrosine kinase inhibitor (TKI), introduced into clinical practice in 1998, was imatinib mesylate. Imatinib became the first choice drug in chronic phase CML, because of its high efficacy, low toxicity and ability to maintain durable hematological and cytogenetic responses. However, approximately 20-25% of patients initially treated with imatinib will need alternative therapy, due to drug resistance, which is often caused by the appearance of clones expressing mutant forms of BCR-ABL. Second-generation TKIs have provided new therapeutic option for the patients resistant to imatinib. Dasatinib is the first, second-generation TKI, approved in the US and European Union for the treatment of CML patients with imatinib resistance or intolerance. This drug is a dual SRC-ABL kinase inhibitor, active in most clinically relevant BCR-ABL mutations, except highly resistant T315I. Other second-generation TKIs include nilotinib, bosutinib and INNO 406. Apart from TKIs, the promising group of molecules is inhibitors of Aurora family of serine-threonine kinases. One of these molecules, MK0457, has entered clinical trials, and initial reports indicate that this compound could be active in disease associated with T315I mutation. Thus, wide spectrum of new agents, with different mode of action, is currently in clinical development for CML. It is likely that combination therapy will be the best therapeutic strategy in the future.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_057',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 530}]}],\n",
       "   'context': 'Translation of the Philadelphia chromosome into therapy for CML. Throughout its history, chronic myeloid leukemia (CML) has set precedents for cancer research and therapy. These range from the identification of the first specific chromosomal abnormality associated with cancer to the development of imatinib as a specific, targeted therapy for the disease. The successful development of imatinib as a therapeutic agent for CML can be attributed directly to decades of scientific discoveries. These discoveries determined that the BCR-ABL tyrosine kinase is the critical pathogenetic event in CML and an ideal target for therapy. This was confirmed in clinical trials of imatinib, with imatinib significantly improving the long-term survival of patients with CML. Continuing in this tradition of scientific discoveries leading to improved therapies, the understanding of resistance to imatinib has rapidly led to strategies to circumvent resistance. Continued studies of hematologic malignancies will allow this paradigm of targeting molecular pathogenetic events to be applied to many additional hematologic cancers.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_058',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 237}]}],\n",
       "   'context': \"Growth arrest of BCR-ABL positive cells with a sequence-specific polyamide-chlorambucil conjugate. Chronic myeloid leukemia (CML) is characterized by the presence of a constitutively active Abl kinase, which is the product of a chimeric BCR-ABL gene, caused by the genetic translocation known as the Philadelphia chromosome. Imatinib, a selective inhibitor of the Bcr-Abl tyrosine kinase, has significantly improved the clinical outcome of patients with CML. However, subsets of patients lose their response to treatment through the emergence of imatinib-resistant cells, and imatinib treatment is less durable for patients with late stage CML. Although alternative Bcr-Abl tyrosine kinase inhibitors have been developed to overcome drug resistance, a cocktail therapy of different kinase inhibitors and additional chemotherapeutics may be needed for complete remission of CML in some cases. Chlorambucil has been used for treatment of B cell chronic lymphocytic leukemia, non-Hodgkin's and Hodgkin's disease. Here we report that a DNA sequence-specific pyrrole-imidazole polyamide-chlorambucil conjugate, 1R-Chl, causes growth arrest of cells harboring both unmutated BCR-ABL and three imatinib resistant strains. 1R-Chl also displays selective toxicities against activated lymphocytes and a high dose tolerance in a murine model.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_059',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 364}]}],\n",
       "   'context': \"Growth arrest of BCR-ABL positive cells with a sequence-specific polyamide-chlorambucil conjugate. Chronic myeloid leukemia (CML) is characterized by the presence of a constitutively active Abl kinase, which is the product of a chimeric BCR-ABL gene, caused by the genetic translocation known as the Philadelphia chromosome. Imatinib, a selective inhibitor of the Bcr-Abl tyrosine kinase, has significantly improved the clinical outcome of patients with CML. However, subsets of patients lose their response to treatment through the emergence of imatinib-resistant cells, and imatinib treatment is less durable for patients with late stage CML. Although alternative Bcr-Abl tyrosine kinase inhibitors have been developed to overcome drug resistance, a cocktail therapy of different kinase inhibitors and additional chemotherapeutics may be needed for complete remission of CML in some cases. Chlorambucil has been used for treatment of B cell chronic lymphocytic leukemia, non-Hodgkin's and Hodgkin's disease. Here we report that a DNA sequence-specific pyrrole-imidazole polyamide-chlorambucil conjugate, 1R-Chl, causes growth arrest of cells harboring both unmutated BCR-ABL and three imatinib resistant strains. 1R-Chl also displays selective toxicities against activated lymphocytes and a high dose tolerance in a murine model.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_060',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 325}]}],\n",
       "   'context': \"Imatinib mesylate for the treatment of chronic myeloid leukemia. Chronic myeloid leukemia (CML) is the first human malignancy for which the promise of targeted therapy has come true. CML is invariably associated with a specific genetic lesion--the t(9;22) chromosomal translocation. As a consequence of this translocation, a BCR-ABL fusion gene is formed on the 22q- derivative (traditionally known as the Philadelphia chromosome) and the deregulated tyrosine kinase activity of the protein encoded by this gene has been shown to be both necessary and sufficient for initiation and maintenance of the disease. Imatinib mesylate, an orally available tyrosine kinase inhibitor that targets Bcr-Abl, entered clinical evaluation in 1998. Its efficacy surpassed almost everyone's predictions, and the observation of high response rates and favorable toxicity profile associated with imatinib therapy led to its approval as first-line treatment for all newly diagnosed CML patients over an exceptionally short period of time. The 6-year results of the Phase III trial have recently been reported and confirm durability of responses and declining incidence of adverse events over time, although, at present, occurrence of unexpected side effects in the long term cannot be excluded. Although imatinib does not 'cure' CML and has to be administered chronically to patients, it has revolutionized both outcome and quality of life of CML patients.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_061',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 688}]}],\n",
       "   'context': \"Imatinib mesylate for the treatment of chronic myeloid leukemia. Chronic myeloid leukemia (CML) is the first human malignancy for which the promise of targeted therapy has come true. CML is invariably associated with a specific genetic lesion--the t(9;22) chromosomal translocation. As a consequence of this translocation, a BCR-ABL fusion gene is formed on the 22q- derivative (traditionally known as the Philadelphia chromosome) and the deregulated tyrosine kinase activity of the protein encoded by this gene has been shown to be both necessary and sufficient for initiation and maintenance of the disease. Imatinib mesylate, an orally available tyrosine kinase inhibitor that targets Bcr-Abl, entered clinical evaluation in 1998. Its efficacy surpassed almost everyone's predictions, and the observation of high response rates and favorable toxicity profile associated with imatinib therapy led to its approval as first-line treatment for all newly diagnosed CML patients over an exceptionally short period of time. The 6-year results of the Phase III trial have recently been reported and confirm durability of responses and declining incidence of adverse events over time, although, at present, occurrence of unexpected side effects in the long term cannot be excluded. Although imatinib does not 'cure' CML and has to be administered chronically to patients, it has revolutionized both outcome and quality of life of CML patients.\"},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_062',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 152}]}],\n",
       "   'context': 'Targeted chronic myeloid leukemia therapy: seeking a cure. BACKGROUND: Chronic myeloid leukemia (CML) is a hematopoietic stem cell cancer driven by the BCR-ABL fusion protein that arises from the translocation of chromosomes 9 and 22. The disease begins with an indolent chronic phase (CP) that can last for 3 to 5 years. If untreated, it progresses into accelerated phase (AP) and within a year, blast phase (BP). Survival at this point is less than 1 year. during disease progression, mutations and the Philadelphia chromosome (Ph) appear (a process called clonal evolution). The only known curative therapy for CML is allogeneic bone marrow transplant (BMT). However, toxicity is formidable, with treatment- related mortality reported in the 30% range. Thus, effective therapy that maintains the patient with CML in CP with minimal toxicity is the goal for treatment of modern therapies. Because the preeminent mutation driving CML is BCr-ABL, therapies targeting BCR-ABL are the logical choice for disease-specific therapy. BCR-ABL inhibitors, such as imatinib, are proof that targeting specific genetic mutations associated with cancer yields a high degree of efficacy with minimal toxicity. OBJECTIVE: This review will outline the evolution of therapy in CML. Preimatinib and imatinib-based treatment strategies, clinical efficacy, and the mechanism of imatinib resistance will be discussed. SUMMARY: The discovery of the Ph and, subsequently, the identification of BCr-ABL revolutionized the treatment of CML. Cytoreductive chemotherapy, such as busulfan and hydroxyurea, was a mainstay of therapy to control white blood cell (WBC) counts; however, it did not modify the progression of the disease to AP and BP. The overall survival with CML ranges from 45 to 58 months in patients treated with cytoreductive therapy only. Treatment was advanced with the introduction of interferon (IFn ) immunotherapy in the 1980s. In some studies, IFn produced a complete hematologic response (CHR) in more than 50% of patients; however, its nonspecific immunostimulatory mechanism also produced severe flulike symptoms that limited tolerability. despite the significant toxicity, cost, and inconvenience of injecting IFN thrice weekly, median survival ranged from 60 to 89 months. Allogeneic BMT is the only known curative therapy for CML; however, treatment-related mortality from infection, bleeding, and graft versus host disease, age, and the availability of suitable donors limits its widespread use. Imatinib functions by competing with adenosine triphosphate (ATP) for binding to the BCr-ABL tyrosine kinase. In the absence of ATP, BCR-ABL is not able to activate downstream effector tyrosine kinase molecules that drive wBC proliferation. The International randomized Interferon versus STI571 clinical trial was the first to document the efficacy of imatinib as a first-line therapy for patients in CP. More than 90% of these patients had a CHr. Toxicities associated with this therapy are low. response in patients with advanced CML is less pronounced than in CP and is shorter lived, with less than 30% of patients achieving a CHR. For patients with CML in BP, the only viable therapy is to attempt a temporary reduction in disease burden with a salvage chemotherapy regimen, such as VAC (etoposide, cytarabine, and carboplatin). The goal of this induction chemotherapy is to induce a second remission; then the patient may be considered for allogeneic BMT. The main toxicities seen with imatinib therapy are myelosuppression, edema, and myalgia/arthralgia. In many cases, imatinib dosage can be briefly halted or lowered while the toxicity is managed. Imatinib resistance may develop at any time and inevitably leads to disease progression. resistance is usually caused by mutations within BCr-ABL, decreasing the affinity of imatinib binding. next-generation kinase inhibitors are focused on the ability to inhibit these mutated forms of BCR-ABL. CONCLUSION: For the majority of patients with CML in CP, the standard of care is to maintain the patient in CP with imatinib therapy. Clinical trials have been extraordinarily successful, with 5-year survival rates greater than 90%. Allogeneic BMT continues to be an option for those who cannot tolerate imatinib or when CML progresses on imatinib therapy.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_063',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 1028}]}],\n",
       "   'context': 'Targeted chronic myeloid leukemia therapy: seeking a cure. BACKGROUND: Chronic myeloid leukemia (CML) is a hematopoietic stem cell cancer driven by the BCR-ABL fusion protein that arises from the translocation of chromosomes 9 and 22. The disease begins with an indolent chronic phase (CP) that can last for 3 to 5 years. If untreated, it progresses into accelerated phase (AP) and within a year, blast phase (BP). Survival at this point is less than 1 year. during disease progression, mutations and the Philadelphia chromosome (Ph) appear (a process called clonal evolution). The only known curative therapy for CML is allogeneic bone marrow transplant (BMT). However, toxicity is formidable, with treatment- related mortality reported in the 30% range. Thus, effective therapy that maintains the patient with CML in CP with minimal toxicity is the goal for treatment of modern therapies. Because the preeminent mutation driving CML is BCr-ABL, therapies targeting BCR-ABL are the logical choice for disease-specific therapy. BCR-ABL inhibitors, such as imatinib, are proof that targeting specific genetic mutations associated with cancer yields a high degree of efficacy with minimal toxicity. OBJECTIVE: This review will outline the evolution of therapy in CML. Preimatinib and imatinib-based treatment strategies, clinical efficacy, and the mechanism of imatinib resistance will be discussed. SUMMARY: The discovery of the Ph and, subsequently, the identification of BCr-ABL revolutionized the treatment of CML. Cytoreductive chemotherapy, such as busulfan and hydroxyurea, was a mainstay of therapy to control white blood cell (WBC) counts; however, it did not modify the progression of the disease to AP and BP. The overall survival with CML ranges from 45 to 58 months in patients treated with cytoreductive therapy only. Treatment was advanced with the introduction of interferon (IFn ) immunotherapy in the 1980s. In some studies, IFn produced a complete hematologic response (CHR) in more than 50% of patients; however, its nonspecific immunostimulatory mechanism also produced severe flulike symptoms that limited tolerability. despite the significant toxicity, cost, and inconvenience of injecting IFN thrice weekly, median survival ranged from 60 to 89 months. Allogeneic BMT is the only known curative therapy for CML; however, treatment-related mortality from infection, bleeding, and graft versus host disease, age, and the availability of suitable donors limits its widespread use. Imatinib functions by competing with adenosine triphosphate (ATP) for binding to the BCr-ABL tyrosine kinase. In the absence of ATP, BCR-ABL is not able to activate downstream effector tyrosine kinase molecules that drive wBC proliferation. The International randomized Interferon versus STI571 clinical trial was the first to document the efficacy of imatinib as a first-line therapy for patients in CP. More than 90% of these patients had a CHr. Toxicities associated with this therapy are low. response in patients with advanced CML is less pronounced than in CP and is shorter lived, with less than 30% of patients achieving a CHR. For patients with CML in BP, the only viable therapy is to attempt a temporary reduction in disease burden with a salvage chemotherapy regimen, such as VAC (etoposide, cytarabine, and carboplatin). The goal of this induction chemotherapy is to induce a second remission; then the patient may be considered for allogeneic BMT. The main toxicities seen with imatinib therapy are myelosuppression, edema, and myalgia/arthralgia. In many cases, imatinib dosage can be briefly halted or lowered while the toxicity is managed. Imatinib resistance may develop at any time and inevitably leads to disease progression. resistance is usually caused by mutations within BCr-ABL, decreasing the affinity of imatinib binding. next-generation kinase inhibitors are focused on the ability to inhibit these mutated forms of BCR-ABL. CONCLUSION: For the majority of patients with CML in CP, the standard of care is to maintain the patient in CP with imatinib therapy. Clinical trials have been extraordinarily successful, with 5-year survival rates greater than 90%. Allogeneic BMT continues to be an option for those who cannot tolerate imatinib or when CML progresses on imatinib therapy.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_064',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'bcr-abl', 'answer_start': 277}]}],\n",
       "   'context': 'Characterization of cancer stem cells in chronic myeloid leukaemia. CML (chronic myeloid leukaemia) is a myeloproliferative disease that originates in an HSC (haemopoietic stem cell) as a result of the t(9;22) translocation, giving rise to the Ph (Philadelphia chromosome) and bcr-abl oncoprotein. The disease starts in CP (chronic phase), but as a result of genomic instability, it progresses over time to accelerated phase and then to BC (blast crisis), becoming increasingly resistant to therapy. bcr-abl is a constitutively active tyrosine kinase that has been targeted by TKIs (tyrosine kinase inhibitors), including IM (imatinib mesylate), nilotinib and dasatinib. We have developed various flow cytometry techniques to enable us to isolate candidate CML stem cells from CP patients at diagnosis that efflux Hoechst dye, express CD34, lack CD38 and are cytokine-non-responsive in culture over periods of up to 12 days in growth factors. These stem cells have been shown to regenerate bcr-abl-positive haemopoiesis in immunocompromised mice upon transplantation. We previously demonstrated that IM was antiproliferative for CML stem cells but did not induce apoptosis. Clinical experience now confirms that IM may not target CML stem cells in vivo with few patients achieving complete molecular remission and relapse occurring rapidly upon drug withdrawal. Our recent efforts have focused on understanding why CML stem cells are resistant to IM and on trying to find novel ways to induce apoptosis of this population. We have shown that CML stem cells express very high levels of functional wild-type bcr-abl; no kinase domain mutations have been detected in the stem cell population. Dasatinib, a more potent multitargeted TKI than IM, inhibits bcr-abl activity more efficiently than IM but still does not induce apoptosis of the stem cell population. Most recently, we have tested a number of novel drug combinations and found that FTIs (farnesyl transferase inhibitors) have activity against CML. BMS-214662 is the most effective of these and induces apoptosis of phenotypically and functionally defined CML stem cells in vitro, as a single agent and in combination with IM or dasatinib. The effect against CML stem cells is selective with little effect on normal stem cells. The drug is also effective against BC CML stem cells and equally effective against wild-type and mutant bcr-abl, including the most resistant mutant T315I. In association with apoptosis, there is activation of caspase 8 and caspase 3, inhibition of the MAPK pathway, IAP-1 (inhibitor of apoptosis protein-1), NF-kappaB (nuclear factor kappaB) and iNOS (inducible nitric oxide synthase). Furthermore, BMS-214662 synergizes with MEK1/2 [MAPK (mitogen-activated protein kinase)/ERK (extracellular-signal-regulated kinase) kinase 1/2] inhibitors, suggesting a second mechanism other that RAS inhibition for induction of apoptosis. Our intentions are now to explore the activity of BMS-214662 in other cancer stem cell disorders and to move this preclinical work to a clinical trial combining dasatinib with BMS-214662 in CML.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_065',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 271}]}],\n",
       "   'context': 'The Bcr-Abl tyrosine kinase inhibitor imatinib and promising new agents against Philadelphia chromosome-positive leukemias. Chronic myeloid leukemia (CML) was the first human malignant disease to be linked to a single, acquired genetic abnormality. Identification of the Bcr-Abl kinase fusion protein and its pivotal role in the pathogenesis of CML provided new opportunities to develop molecular-targeted therapies. Imatinib mesylate (IM, Gleevec, Novartis Pharmaceuticals, Basel, Switzerland), which specifically inhibits the autophosphorylation of the Abl TK, has improved the treatment of CML. However, resistance is often reported in patients with advanced-stage disease. Several novel TK inhibitors have been developed that override IM resistance mechanisms caused by point mutations within the Abl kinase domain. Inhibitors of Abl TK are divided into two main groups, namely, ATP-competitive and ATP noncompetitive inhibitors. The ATP-competitive inhibitors fall into two subclasses, the Src/Abl inhibitors, and the 2-phenylaminopyrimidine-based compounds. Dasatinib (formerly BMS-354825), AP23464, SKI-606, and PD166326 are classified as Src/Abl inhibitors, while nilotinib (AMN107) and INNO-406 (NS-187) belong to the latter subclass of inhibitors. Of these agents, dasatinib and nilotinib underwent clinical trials earlier than the others and favorable results are now accumulating. Clinical studies of the other compounds, including SKI-606 and INNO-406, have been performed in rapid succession. Because of their strong affinities for the ATP-binding site compared to IM, most ATP-competitive inhibitors may be effective in IM-resistant patients. However, an ATP-competitive inhibitor that can inhibit the phosphorylation of T315I Bcr-Abl has not yet been developed. Instead, ATP noncompetitive inhibitors, such as ON012380, Aurora kinase inhibitor MK0457 (VX-680), and p38 MAP kinase inhibitor BIRB-796, have been developed to address this problem. This review provides an update on the underlying pathophysiologies of disease progression and IM resistance, and discusses the development of new targeted TK inhibitors for managing CML and the importance of future strategies targeting CML stem cells.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_066',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 203}]}],\n",
       "   'context': 'Applying the discovery of the Philadelphia chromosome. The identification of the Philadelphia chromosome in cells from individuals with chronic myelogenous leukemia (CML) led to the recognition that the BCR-ABL tyrosine kinase causes CML. This in turn led to the development of imatinib mesylate, a clinically successful inhibitor of the BCR-ABL kinase. Incorporating the use of markers of BCR-ABL kinase inhibition into clinical trials led to the realization that imatinib-resistant kinase domain mutations are the major cause of relapse during imatinib therapy and the subsequent development of new inhibitors to treat CML patients. The development of imatinib validates an emerging paradigm in cancer, in which a tumor is defined by genetic abnormalities and effective therapies are developed that target events critical to the growth and survival of a specific tumor.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_067',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 338}]}],\n",
       "   'context': 'Applying the discovery of the Philadelphia chromosome. The identification of the Philadelphia chromosome in cells from individuals with chronic myelogenous leukemia (CML) led to the recognition that the BCR-ABL tyrosine kinase causes CML. This in turn led to the development of imatinib mesylate, a clinically successful inhibitor of the BCR-ABL kinase. Incorporating the use of markers of BCR-ABL kinase inhibition into clinical trials led to the realization that imatinib-resistant kinase domain mutations are the major cause of relapse during imatinib therapy and the subsequent development of new inhibitors to treat CML patients. The development of imatinib validates an emerging paradigm in cancer, in which a tumor is defined by genetic abnormalities and effective therapies are developed that target events critical to the growth and survival of a specific tumor.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_068',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 336}]}],\n",
       "   'context': 'Second-generation tyrosine kinase inhibitors as therapy for chronic myeloid leukemia. Chronic myeloid leukemia (CML) was the first human malignancy to be associated with a single genetic abnormality, characterized by a reciprocal translocation involving chromosomes 9 and 22 (the Philadelphia chromosome). The fusion gene that results (BCR-ABL) produces a constitutively activated tyrosine kinase that exists in different isoforms depending on BCR break-points. Imatinib mesylate is a highly selective inhibitor of this kinase, producing normal blood-counts in 98% of patients in chronic phase CML and disappearance of the Philadelphia chromosome in 86%. However, 17% of patients in the chronic phase will either relapse or develop resistance resulting mainly from one or more point mutations affecting at least 30 amino acids within the Abl kinase protein. This review focuses on the relevant biology of CML, imatinib mesylate resistance mechanisms, and the current status of the next generation of Bcr-Abl inhibitors.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_069',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'bcr-abl', 'answer_start': 343}]}],\n",
       "   'context': 'Novel Abl kinase inhibitors in chronic myeloid leukemia in blastic phase and Philadelphia chromosome-positive acute lymphoblastic leukemia. Chronic myeloid leukemia (CML) is characterized by the presence of the Philadelphia chromosome, which is associated with a balanced translocation involving chromosomes 9 and 22 to produce a fusion gene (bcr-abl) that gives rise to a constitutively activated Abl tyrosine kinase. This kinase led to the discovery of several small-molecule inhibitors, imatinib being the first and most successful of these. Resistance to imatinib results in some patients from Abl kinase point mutations. Overcoming imatinib resistance represents one of the biggest challenges facing clinicians in the modern management of CML. In this review, we discuss the current understanding of CML pathophysiology and mechanisms of imatinib resistance and how advancing this knowledge has led to the design of novel therapies in the area of blastic phase CML and Philadelphia chromosome-positive acute lymphoblastic leukemia with previous imatinib failure.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_070',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 173}]}],\n",
       "   'context': 'BCR-ABL gene amplification and overexpression in a patient with chronic myeloid leukemia treated with imatinib. Imatinib mesylate was designed as an inhibitor targeting the BCR-ABL tyrosine kinase, the molecular counterpart of the Philadelphia translocation t(9;22)(q34;q11). We report on a patient with chronic myeloid leukemia (CML) undergoing acceleration during imatinib treatment. Cytogenetic analysis revealed four different cell populations: 46,XX,t(9;22)(q34;q11),der(18)t(2;18)(p11;p11)[1]/47,idem,i(17)(q10),-der(18)t(2;18),+der(22)t(9;22)[1]/46,idem,-t(9;22),der(9)t(9;22),ider(22)t(9;22)[12]/ 47,idem,-t(9;22),der(9)t(9;22),+22,ider(22)t(9;22)x2[1]. FISH analysis confirmed the presence of these four clones. Moreover, 49% of the interphase nuclei contained either one or two clustered fusion signals, indicating a low-level amplification of the BCR-ABL fusion gene. With quantitative real-time RT-PCR, a BCR-ABL/G6PDH ratio of 0.8 was determined, which is comparable to that measured in the K562 cell line with a known BCR-ABL amplification and which is increased by more than about 60-fold compared to a CML at diagnosis with >80% Philadelphia-positive cells. We give further evidence that the genomic BCR-ABL amplification results in an increased level of BCR-ABL transcript linking two potent mechanisms of resistance against imatinib treatment.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_071',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 261}]}],\n",
       "   'context': '[Imatinib therapy for patients with chronic myelogenous leukemia]. Chronic myelogenous leukemia (CML) is a clonal hematopoietic disorder caused by the reciprocal translocation between chromosome 9 and 22. As a result of this translocation, a novel fusion gene, BCR-ABL, is created on Philadelphia (Ph) chromosome, and the constitutive activity of the BCR-ABL protein tyrosine kinase plays a critical role in the disease pathogenesis. Imatinib mesylate, a selective BCR-ABL tyrosine kinase inhibitor, was first given to a patient with CML in June 1998. Since then, it has continued to demonstrate remarkable efficacy in treating patients with CML. Based upon the results of early phase I and II studies, a phase III study (IRIS Study) that was randomized to first-line imatinib (400 mg/day) or to standard treatment with interferon+low-dose Ara-C, was conducted on 1,106 patients newly diagnosed (within 6 months) with chronic-phase CML. After median follow-up of 30 months, imatinib showed significantly superior tolerability, hematologic and cytogenetic responses (major cytogenetic response, 90%; complete cytogenetic response, 82%), and overall survival (95% without censoring allo-HSCT). Although imatinib is the first-line therapy and has changed the paradigm of CML treatment strategy, questions remain as to the meaning of cytogenetic and molecular response, curability, optimal dose, and relation with allo-HSCT.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_072',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 351}]}],\n",
       "   'context': '[Imatinib therapy for patients with chronic myelogenous leukemia]. Chronic myelogenous leukemia (CML) is a clonal hematopoietic disorder caused by the reciprocal translocation between chromosome 9 and 22. As a result of this translocation, a novel fusion gene, BCR-ABL, is created on Philadelphia (Ph) chromosome, and the constitutive activity of the BCR-ABL protein tyrosine kinase plays a critical role in the disease pathogenesis. Imatinib mesylate, a selective BCR-ABL tyrosine kinase inhibitor, was first given to a patient with CML in June 1998. Since then, it has continued to demonstrate remarkable efficacy in treating patients with CML. Based upon the results of early phase I and II studies, a phase III study (IRIS Study) that was randomized to first-line imatinib (400 mg/day) or to standard treatment with interferon+low-dose Ara-C, was conducted on 1,106 patients newly diagnosed (within 6 months) with chronic-phase CML. After median follow-up of 30 months, imatinib showed significantly superior tolerability, hematologic and cytogenetic responses (major cytogenetic response, 90%; complete cytogenetic response, 82%), and overall survival (95% without censoring allo-HSCT). Although imatinib is the first-line therapy and has changed the paradigm of CML treatment strategy, questions remain as to the meaning of cytogenetic and molecular response, curability, optimal dose, and relation with allo-HSCT.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_073',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 465}]}],\n",
       "   'context': '[Imatinib therapy for patients with chronic myelogenous leukemia]. Chronic myelogenous leukemia (CML) is a clonal hematopoietic disorder caused by the reciprocal translocation between chromosome 9 and 22. As a result of this translocation, a novel fusion gene, BCR-ABL, is created on Philadelphia (Ph) chromosome, and the constitutive activity of the BCR-ABL protein tyrosine kinase plays a critical role in the disease pathogenesis. Imatinib mesylate, a selective BCR-ABL tyrosine kinase inhibitor, was first given to a patient with CML in June 1998. Since then, it has continued to demonstrate remarkable efficacy in treating patients with CML. Based upon the results of early phase I and II studies, a phase III study (IRIS Study) that was randomized to first-line imatinib (400 mg/day) or to standard treatment with interferon+low-dose Ara-C, was conducted on 1,106 patients newly diagnosed (within 6 months) with chronic-phase CML. After median follow-up of 30 months, imatinib showed significantly superior tolerability, hematologic and cytogenetic responses (major cytogenetic response, 90%; complete cytogenetic response, 82%), and overall survival (95% without censoring allo-HSCT). Although imatinib is the first-line therapy and has changed the paradigm of CML treatment strategy, questions remain as to the meaning of cytogenetic and molecular response, curability, optimal dose, and relation with allo-HSCT.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_074',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 475}]}],\n",
       "   'context': 'Glivec and CML: a lucky date. Chronic Myeloid Leukemia (CML) has always been an ideal model to understand the molecular pathogenesis of human leukaemias and the way to cure them. This can be ascribed to the fact that CML was the first human cancer demonstrated to be strongly associated to the presence of a recurrent chromosomal translocation (the t(9;22)(q34;q11) that creates the Philadelphia (Ph)-chromosome) and to a specific molecular defect, the formation of a hybrid BCR-ABL gene that generates new fusion proteins endowed with a constitutive tyrosine-kinase (TK) activity, strongly implicated in the pathogenesis of the disease. The introduction into clinical practice of imatinib, (Glivec, Gleevec, Novartis), a potent tyrosine kinase inhibitor of the Bcr-Abl protein as well as of a restricted number of other TKs, has not only produced a substantial improvement in the treatment of CML, but represents a major break-through in the perspective of opening a new era, that of molecularly targeted therapy, in the management of other types of leukemia, lymphoma and cancer in general.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_075',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 762}]}],\n",
       "   'context': 'Glivec and CML: a lucky date. Chronic Myeloid Leukemia (CML) has always been an ideal model to understand the molecular pathogenesis of human leukaemias and the way to cure them. This can be ascribed to the fact that CML was the first human cancer demonstrated to be strongly associated to the presence of a recurrent chromosomal translocation (the t(9;22)(q34;q11) that creates the Philadelphia (Ph)-chromosome) and to a specific molecular defect, the formation of a hybrid BCR-ABL gene that generates new fusion proteins endowed with a constitutive tyrosine-kinase (TK) activity, strongly implicated in the pathogenesis of the disease. The introduction into clinical practice of imatinib, (Glivec, Gleevec, Novartis), a potent tyrosine kinase inhibitor of the Bcr-Abl protein as well as of a restricted number of other TKs, has not only produced a substantial improvement in the treatment of CML, but represents a major break-through in the perspective of opening a new era, that of molecularly targeted therapy, in the management of other types of leukemia, lymphoma and cancer in general.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_076',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 180}]}],\n",
       "   'context': '[Molecular targeted treatment--new treatment strategy for patients with chronic myeloid leukemia]. Imatinib mesylate is a new drug that can inhibit the tyrosine kinase activity of Bcr-Abl, the receptors for platelet-derived growth factor receptor(PDGF) and stem cell factor, or c-kit. Chronic myeloid leukemia (CML) is distinguished by the presence of a reciprocal translocation between chromosomes 9 and 22 that results in a shortened chromosome 22, termed the Philadelphia(Ph) chromosome. As a result of the translocation, a fusion gene called the Bcr-Abl gene is created from two normal cellular genes, encoding a chimeric Bcr-Abl protein with a deregulated tyrosine kinase activity. The expression of Bcr-Abl tyrosine kinase has been shown to be necessary and sufficient for the transformed phenotype of CML cells. Imatinib can block the kinase activity of Bcr-Abl, thus inhibiting the proliferation of Ph-positive progenitors, and has shown activity against all phases of CML, though responses are most substantial and durable in patients in the chronic phase. An international phase III study which compared the efficacy of imatinib with that of interferon alpha combined with low-dose cytarabine in newly diagnosed chronic-phase CML showed the rate of major cytogenetic response at 24 months was 90%, including 82% of complete cytogenetic response. These results indicated that imatinib was superior to interferon-containing treatment as a first-line therapy. More than 10,000 patients worldwide, including those in Japan, have been treated with imatinib in clinical trials, and a lot of information has been accumulated on the use of this drug. The aim of this article is to review the use of this drug and the practical management of patients with chronic myeloid leukemia.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_077',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 127}]}],\n",
       "   'context': 'Imatinib mesylate resistance through BCR-ABL independence in chronic myelogenous leukemia. Imatinib mesylate (IM) binds to the BCR-ABL protein, inhibiting its kinase activity and effectively controlling diseases driven by this kinase. IM resistance has been associated with kinase mutations or increased BCR-ABL expression. However, disease progression may be mediated by other mechanisms that render tumor cells independent of BCR-ABL. To demonstrate this potential, IM-resistant cells were found in chronic myelogenous leukemia patients with continuous BCR-ABL gene expression but undetectable BCR-ABL protein expression. These cells were unresponsive to IM and acquired BCR-ABL-independent signaling characteristics. IM resistance in some patients may be mediated through loss of kinase target dependence.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_078',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 446}]}],\n",
       "   'context': 'Specific targeted therapy of chronic myelogenous leukemia with imatinib. Chronic myeloid leukemia (CML) is characterized by the Philadelphia translocation that fuses BCR sequences from chromosome 22 upstream of the ABL gene on chromosome 9. The chimerical Bcr-Abl protein expressed by CML cells has constitutive tyrosine kinase activity, which is essential for the pathogenesis of the disease. Imatinib, an ATP-competitive selective inhibitor of Bcr-Abl, has unprecedented efficacy for the treatment of CML. Most patients with early stage disease achieve durable complete hematological and complete cytogenetic remissions, with minimal toxicity. In contrast, responses are less stable in patients with advanced CML. This review highlights the pathogenesis of CML, its clinical features, and the development of imatinib as a specific molecularly targeted therapy. Aspects of disease monitoring and side effects are covered as well as resistance to imatinib and strategies to overcome resistance, such as alternative signal transduction inhibitors and drug combinations. Perspectives for further development are also discussed.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_079',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 283}]}],\n",
       "   'context': 'The effect of dose increase of imatinib mesylate in patients with chronic or accelerated phase chronic myelogenous leukemia with inadequate hematologic or cytogenetic response to initial treatment. PURPOSE: Imatinib mesylate is a tyrosine kinase inhibitor with high affinity for the BCR-ABL fusion protein expressed by the hematopoietic cells in chronic myelogenous leukemia (CML). Some patients with chronic-phase or accelerated-phase CML either relapse after an initial response or are refractory to imatinib, prompting us to evaluate the efficacy of dose increase in such patients. EXPERIMENTAL DESIGN: Twelve chronic-phase patients initially receiving 400 mg/day and 4 patients with accelerated phase initially receiving either 400 mg/day (two patients) or 600 mg/day (two patients) had their dose increased (14 to 800 mg/day and 2 to 600 mg/day) because of progressive disease (usually clonal evolution) or inadequate cytogenetic response after at least 1 year of therapy. RESULTS: Six patients had major cytogenetic responses after dose increase (3 complete and 3 partial). Two others had minor cytogenetic responses. Two patients with clonal evolution transiently lost the additional clonal aberrations. Almost all of the responses occurred within 6 months, and were typically 3-6 months in duration. However, 3 patients have continuing major cytogenetic responses of >18 months duration. Dose increase was well tolerated, with thrombocytopenia, mild leukopenia, and exacerbation of prior edema being the most common adverse events. CONCLUSIONS: Although increasing the dose of imatinib can benefit a subgroup of patients with CML with either an inadequate cytogenetic response or disease progression, our results suggest the majority will not have a sustained meaningful response, and that other options, such as allogeneic stem cell transplant or investigational therapies, also need to be considered at the time of dose increase.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_080',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 231}]}],\n",
       "   'context': 'Philadelphia chromosome-positive leukemias: from basic mechanisms to molecular therapeutics. The Philadelphia chromosome translocation (t(9;22)) results in the molecular juxtaposition of two genes, BCR and ABL, to form an aberrant BCR-ABL gene on chromosome 22. BCR-ABL is critical to the pathogenesis of chronic myelogenous leukemia and a subset of acute leukemias. The chimeric Bcr-Abl protein has constitutively elevated tyrosine phosphokinase activity. This abnormal enzymatic activation is critical to the oncogenic potential of Bcr-Abl. Initially, protein kinases were thought to be poor therapeutic targets because of their ubiquitous nature and crucial role in many normal physiologic processes. However, the advent of imatinib mesylate (Gleevec, Novartis Pharmaceuticals, Basel, Switzerland), formerly known as STI571 and CGP57148B, demonstrated that designer kinase inhibitors could be specific. This agent has shown striking activity in chronic myelogenous leukemia. It also inhibits phosphorylation of Kit (stem-cell factor receptor) and platelet-derived growth factor receptor. In addition, it has shown similar impressive responses, with little host toxicity, in gastrointestinal stromal tumors, which harbor activating Kit mutations, and in tumors with activated platelet-derived growth factor receptor. The studies of imatinib mesylate provide proof-of-principle for using aberrant kinases as a therapeutic target and are a model for the promise of molecular therapeutics. This paper reviews the current knowledge on the function of Bcr-Abl and its normal counterparts (Bcr and Abl), as well as the impact of this knowledge on the development of a remarkably successful targeted therapy approach.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_081',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 262}]}],\n",
       "   'context': 'Philadelphia chromosome-positive leukemias: from basic mechanisms to molecular therapeutics. The Philadelphia chromosome translocation (t(9;22)) results in the molecular juxtaposition of two genes, BCR and ABL, to form an aberrant BCR-ABL gene on chromosome 22. BCR-ABL is critical to the pathogenesis of chronic myelogenous leukemia and a subset of acute leukemias. The chimeric Bcr-Abl protein has constitutively elevated tyrosine phosphokinase activity. This abnormal enzymatic activation is critical to the oncogenic potential of Bcr-Abl. Initially, protein kinases were thought to be poor therapeutic targets because of their ubiquitous nature and crucial role in many normal physiologic processes. However, the advent of imatinib mesylate (Gleevec, Novartis Pharmaceuticals, Basel, Switzerland), formerly known as STI571 and CGP57148B, demonstrated that designer kinase inhibitors could be specific. This agent has shown striking activity in chronic myelogenous leukemia. It also inhibits phosphorylation of Kit (stem-cell factor receptor) and platelet-derived growth factor receptor. In addition, it has shown similar impressive responses, with little host toxicity, in gastrointestinal stromal tumors, which harbor activating Kit mutations, and in tumors with activated platelet-derived growth factor receptor. The studies of imatinib mesylate provide proof-of-principle for using aberrant kinases as a therapeutic target and are a model for the promise of molecular therapeutics. This paper reviews the current knowledge on the function of Bcr-Abl and its normal counterparts (Bcr and Abl), as well as the impact of this knowledge on the development of a remarkably successful targeted therapy approach.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_082',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'bcr-abl', 'answer_start': 297}]}],\n",
       "   'context': 'Successful peripheral blood stem cell mobilisation with filgrastim in patients with chronic myeloid leukaemia achieving complete cytogenetic response with imatinib, without increasing disease burden as measured by quantitative real-time PCR. Imatinib mesylate (Glivec) is a selective inhibitor of bcr-abl tyrosine kinase, the product of the Philadelphia chromosome, which is the hallmark of chronic myeloid leukaemia (CML). With imatinib, complete cytogenetic response (CCR) can be achieved in over 70% of newly diagnosed patients with CML. However, the optimal long-term management of patients who achieve CCR after imatinib is unknown. With longer follow-up, it is anticipated that some patients are likely to progress and become candidates for autologous transplantation. We studied filgrastim (r-metHuG-CSF) mobilisation of peripheral blood stem cells (PBSC) in 32 patients who have achieved CCR with imatinib. Our data demonstrate that (1) the target CD34(+) cell yields of >/=2.0 x 10(6)/kg were attained with filgrastim 10 microg/kg/day, in 9/18 (50%) of patients during uninterrupted imatinib therapy, and in 10/14 (70%) when imatinib was temporarily withheld. The median CD34(+) cell yield per aphaeresis was 0.70 x 10(6)/kg (range 0.14-2.18) and 2.90 x 10(6)/kg (range 0.15-8.71) in the two groups, respectively (P&<0.005). (2) The cell yields did not correlate with the duration of imatinib administration. (3) There was no impact of the mobilisation procedure on the level of leukaemia as measured by serial blood bcr-abl levels using real-time quantitative PCR with either protocol. (4) bcr-abl remained detectable at low levels in the harvests in most but not all patients. In conclusion, filgrastim can safely be used to mobilise PBSC in patients who have achieved CCR with imatinib, but CD34(+) cell yields are significantly improved when imatinib is temporarily withheld.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_083',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 213}]}],\n",
       "   'context': 'Drug responses of imatinib mesylate-resistant cells: synergism of imatinib with other chemotherapeutic drugs. Imatinib mesylate (STI571, Glivec, Gleevec) is a powerful inhibitor of the tyrosine kinase activity of Bcr-Abl, the oncoprotein responsible for chronic myeloid leukemia (CML). The drug shows great efficacy in chronic phase, but is less effective in maintaining hematologic remissions in blast crisis patients. Our group has previously described several cell lines made resistant to imatinib. We now examine the question of cross-resistance to other chemotherapeutic drugs used in CML. Four paired imatinib-sensitive/resistant CML cell lines were assessed by caspase-3 and MTS assays for their proliferative response to cytosine arabinoside (Ara-C), daunorubicin (DNR), homoharringtonine (HHT) and hydroxyurea (HU), either alone or in combination with imatinib. Primary blasts from advanced-stage CML patients refractory to imatinib therapy were studied by semi-solid media clonogenic assays. We found that these drugs are generally capable of major inhibition of proliferation of the CML cell lines, although differential responses to DNR and HHT were noted between some sensitive and resistant cell line pairs, implying that resistance to imatinib may confer a growth advantage under such conditions. The four drugs were also effective in preventing the formation of progenitor cell colonies from CML patients both before treatment with imatinib, and after relapse on the drug. Isobolographic analysis implied that these drugs will generally combine well with imatinib, and in some cases will be synergistic. We conclude that Ara-C, DNR or HHT, either alone or in combination with imatinib, are likely to be the best therapeutic alternatives in the management of patients who become resistant to imatinib monotherapy.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_084',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 251}]}],\n",
       "   'context': 'Emergence of clonal cytogenetic abnormalities in Ph- cells in some CML patients in cytogenetic remission to imatinib but restoration of polyclonal hematopoiesis in the majority. Chronic myelogenous leukemia (CML) is characterized by the presence of a Bcr-Abl fusion protein with deregulated tyrosine kinase activity that is required for maintaining the malignant phenotype. Imatinib, a selective inhibitor of Bcr-Abl, induces major cytogenetic remission (MCR) or complete cytogenetic remission (CCR) in the majority of patients with CML in first chronic phase. However, thorough re-evaluation of cytogenetics in a cohort of patients in MCR or CCR demonstrated clonal karyotypic abnormalities in more than 10% of cases, some of which were clinically associated with a myelodysplastic syndrome (MDS). Further analysis identified previous exposure to cytarabine and idarubicin as significant risk factors for the subsequent occurrence of abnormalities in Philadelphia chromosome-negative (Ph-) cells. To investigate if cytogenetically normal but clonal hematopoiesis might be present in other patients in cytogenetic remission, we studied X-chromosome inactivation as a marker of clonality by polymerase chain reaction analysis of the human androgen receptor (HUMARA). We find that imatinib restores a polyclonal pattern in most patients in CCR and MCR. Nonetheless, our results are consistent with the notion that targeted therapy of CML with imatinib favors the manifestation of Ph- clonal disorders in some patients. They indicate that patients on imatinib should be followed with conventional cytogenetics, even after induction of CCR.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_085',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 409}]}],\n",
       "   'context': 'Emergence of clonal cytogenetic abnormalities in Ph- cells in some CML patients in cytogenetic remission to imatinib but restoration of polyclonal hematopoiesis in the majority. Chronic myelogenous leukemia (CML) is characterized by the presence of a Bcr-Abl fusion protein with deregulated tyrosine kinase activity that is required for maintaining the malignant phenotype. Imatinib, a selective inhibitor of Bcr-Abl, induces major cytogenetic remission (MCR) or complete cytogenetic remission (CCR) in the majority of patients with CML in first chronic phase. However, thorough re-evaluation of cytogenetics in a cohort of patients in MCR or CCR demonstrated clonal karyotypic abnormalities in more than 10% of cases, some of which were clinically associated with a myelodysplastic syndrome (MDS). Further analysis identified previous exposure to cytarabine and idarubicin as significant risk factors for the subsequent occurrence of abnormalities in Philadelphia chromosome-negative (Ph-) cells. To investigate if cytogenetically normal but clonal hematopoiesis might be present in other patients in cytogenetic remission, we studied X-chromosome inactivation as a marker of clonality by polymerase chain reaction analysis of the human androgen receptor (HUMARA). We find that imatinib restores a polyclonal pattern in most patients in CCR and MCR. Nonetheless, our results are consistent with the notion that targeted therapy of CML with imatinib favors the manifestation of Ph- clonal disorders in some patients. They indicate that patients on imatinib should be followed with conventional cytogenetics, even after induction of CCR.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_086',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 501}]}],\n",
       "   'context': 'A phase 2 study of imatinib in patients with relapsed or refractory Philadelphia chromosome-positive acute lymphoid leukemias. The translocation (9;22) gives rise to the p190(Bcr-Abl) and p210(Bcr-Abl) tyrosine kinase proteins, considered sufficient for leukemic transformation. Philadelphia-positive (Ph(+)) acute leukemia patients failing to respond to initial induction therapy have a poor prognosis with few effective treatment options. Imatinib is an orally administered, potent inhibitor of the Bcr-Abl tyrosine kinase. We conducted a clinical trial in 56 patients with relapsed or refractory Ph(+) acute lymphoblastic leukemia (ALL; 48 patients) or chronic myelogenous leukemia in lymphoid blast crisis (LyBC; 8 patients). Imatinib was given once daily at 400 mg or 600 mg. Imatinib induced complete hematologic responses (CHRs) and complete marrow responses (marrow-CRs) in 29% of ALL patients (CHR, 19%; marrow-CR, 10%), which were sustained for at least 4 weeks in 6% of patients. Median estimated time to progression and overall survival for ALL patients were 2.2 and 4.9 months, respectively. CHRs were reported for 3 (38%) of the patients with LyBC (one sustained CHR). Grade 3 or 4 treatment-related nonhematologic toxicity was reported for 9% of patients; none of the patients discontinued therapy because of nonhematologic adverse reactions. Grade 4 neutropenia and thrombocytopenia occurred in 54% and 27% of patients, respectively. Imatinib therapy resulted in a clinically relevant hematologic response rate in relapsed or refractory Ph(+) acute lymphoid leukemia patients, but development of resistance and subsequent disease progression were rapid. Further studies are warranted to test the effects of imatinib in combination with other agents and to define the mechanisms of resistance to imatinib.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_087',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 367}]}],\n",
       "   'context': 'The impact of clonal evolution on response to imatinib mesylate (STI571) in accelerated phase CML. In chronic myelogenous leukemia (CML), the development of chromosomal abnormalities in addition to the Philadelphia chromosome (clonal evolution) is considered by many to be a feature of accelerated phase (AP). Imatinib mesylate (STI571), a selective inhibitor of the Bcr-Abl tyrosine kinase, has significant activity in AP CML. As clonal evolution could allow Bcr-Abl independent proliferation, we analyzed its impact on the outcome of 71 AP patients treated with 600 mg of imatinib mesylate. Fifteen patients had clonal evolution alone (AP-CE), 32 had AP features but no evidence of clonal evolution (HEM-AP), and 24 had AP features plus clonal evolution (HEM-AP + CE). Of the AP-CE patients, 73% had a major cytogenetic response, compared with 31% of the HEM-AP patients (P =.043) and 12.5% of the HEM-AP + CE patients (P =.007). Complete cytogenetic responses were seen in 60% of AP-CE patients, compared with 31% of HEM-AP patients (P =.19) and 8% of HEM-AP + CE patients (P <.001). With mean follow-up of 11.2 months, 35% of all patients failed treatment. The lowest estimated rate of treatment failure at 1 year, 0%, was seen in AP-CE patients, compared with rates of 31% for HEM-AP patients and 69% for HEM-AP + CE patients (P =.0004). After 1 year, 100% of AP-CE patients were still alive, compared with 85% of HEM-AP patients and 67.5% of HEM-AP + CE patients (P =.01). In conclusion, in patients with clonal evolution as the sole criterion of disease acceleration, good responses to imatinib are still possible. Once patients have other signs of acceleration, clonal evolution predicts lower response rates and a shorter time to treatment failure.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_088',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 232}]}],\n",
       "   'context': 'In vitro effects of STI 571-containing drug combinations on the growth of Philadelphia-positive chronic myelogenous leukemia cells. BACKGROUND: Chronic myelogenous leukemia (CML) is characterized by a molecular aberration, a fusion BCR-ABL gene encoding for aberrant tyrosine kinase activity, which is crucial in the pathogenesis of CML. In vitro, inhibition of BCR-ABL protein tyrosine kinase activity by a tyrosine kinase inhibitor, Imatinib mesylate (STI571; formerly CGP57148B), successfully suppressed proliferation/survival of the BCR-ABL positive clones. In clinical studies, hematologic and cytogenetic remissions have been achieved in most patients with chronic phase CML; in accelerated and blastic phases of CML, STI571 appeared less effective. In the current study, the authors tested combinations of STI571 and cytarabine and homoharringtonine (HHT), drugs with documented activity in CML. METHODS: The single agents and their combinations were studied for in vitro effect on proliferation of BCR-ABL positive cell lines KBM5 and KBM7 by 3(4,5-dimethylthiazol-2yl)-2,5 diphenyl-tetrazolium bromide assay and on primary patient-derived BCR-ABL cells by clonogenic assays. The in vitro additive, synergistic, or antagonistic effects of cytarabine and HHT with STI571 were then investigated by computer-assisted analysis using the CalcuSyn software. RESULTS: STI571 consistently suppressed BCR-ABL positive cell proliferation with a dose-effect correlation. In the model system used, STI571/cytarabine and STI571/HHT combinations were more effective in inhibiting KBM5 and KBM7 cell growth than each drug as single agent. These results were also verified in primary CML-derived clonogenic cells in semisolid cultures. CONCLUSIONS: In this experimental system, our studies documented additive or synergistic effects with STI571 plus cytarabine or HHT, supporting the future use of STI571 combinations in clinical trials in patients with Philadelphia chromosome-positive leukemias.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_089',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 362}]}],\n",
       "   'context': 'In vitro effects of STI 571-containing drug combinations on the growth of Philadelphia-positive chronic myelogenous leukemia cells. BACKGROUND: Chronic myelogenous leukemia (CML) is characterized by a molecular aberration, a fusion BCR-ABL gene encoding for aberrant tyrosine kinase activity, which is crucial in the pathogenesis of CML. In vitro, inhibition of BCR-ABL protein tyrosine kinase activity by a tyrosine kinase inhibitor, Imatinib mesylate (STI571; formerly CGP57148B), successfully suppressed proliferation/survival of the BCR-ABL positive clones. In clinical studies, hematologic and cytogenetic remissions have been achieved in most patients with chronic phase CML; in accelerated and blastic phases of CML, STI571 appeared less effective. In the current study, the authors tested combinations of STI571 and cytarabine and homoharringtonine (HHT), drugs with documented activity in CML. METHODS: The single agents and their combinations were studied for in vitro effect on proliferation of BCR-ABL positive cell lines KBM5 and KBM7 by 3(4,5-dimethylthiazol-2yl)-2,5 diphenyl-tetrazolium bromide assay and on primary patient-derived BCR-ABL cells by clonogenic assays. The in vitro additive, synergistic, or antagonistic effects of cytarabine and HHT with STI571 were then investigated by computer-assisted analysis using the CalcuSyn software. RESULTS: STI571 consistently suppressed BCR-ABL positive cell proliferation with a dose-effect correlation. In the model system used, STI571/cytarabine and STI571/HHT combinations were more effective in inhibiting KBM5 and KBM7 cell growth than each drug as single agent. These results were also verified in primary CML-derived clonogenic cells in semisolid cultures. CONCLUSIONS: In this experimental system, our studies documented additive or synergistic effects with STI571 plus cytarabine or HHT, supporting the future use of STI571 combinations in clinical trials in patients with Philadelphia chromosome-positive leukemias.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_090',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 244}]}],\n",
       "   'context': '[STI571 (Glivec)--a new drug for the treatment of chronic myeloid leukemia]. Chronic myeloid leukaemia (CML) is characterised by the occurrence of the Philadelphia (Ph) chromosome (9/22 translocation) and the formation of a fusion protein--the BCR-ABL transcript with constitutive activation of the BCR-ABL tyrosine kinase and consequent changes in the intracellular signal transduction, which is responsible for the deregulated myeloid cell proliferation. STI571 (signal transduction inhibition number 571) is a potent and selective inhibitor of the BCR-ABL tyrosine kinase. In the chronic phase of the disease, normal peripheral blood values are achieved within the first month of treatment in the large majority of patients and in many patients also a cytogenic response within the following months. The results in the advanced phase are far less favourable, which is explained by the development of resistance owing to reactivation of the BCR-ABL signal transduction. Side effects are primarily nausea, vomiting, various rashes, oedema, most often in the periorbital region, and musculoskeletal symptoms, including muscle cramps. Perspectives for treatment with STI571 are described, as are combinations with alpha-interferon and other cytostatics with a synergistic profile.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_091',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 299}]}],\n",
       "   'context': '[STI571 (Glivec)--a new drug for the treatment of chronic myeloid leukemia]. Chronic myeloid leukaemia (CML) is characterised by the occurrence of the Philadelphia (Ph) chromosome (9/22 translocation) and the formation of a fusion protein--the BCR-ABL transcript with constitutive activation of the BCR-ABL tyrosine kinase and consequent changes in the intracellular signal transduction, which is responsible for the deregulated myeloid cell proliferation. STI571 (signal transduction inhibition number 571) is a potent and selective inhibitor of the BCR-ABL tyrosine kinase. In the chronic phase of the disease, normal peripheral blood values are achieved within the first month of treatment in the large majority of patients and in many patients also a cytogenic response within the following months. The results in the advanced phase are far less favourable, which is explained by the development of resistance owing to reactivation of the BCR-ABL signal transduction. Side effects are primarily nausea, vomiting, various rashes, oedema, most often in the periorbital region, and musculoskeletal symptoms, including muscle cramps. Perspectives for treatment with STI571 are described, as are combinations with alpha-interferon and other cytostatics with a synergistic profile.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_092',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 551}]}],\n",
       "   'context': '[STI571 (Glivec)--a new drug for the treatment of chronic myeloid leukemia]. Chronic myeloid leukaemia (CML) is characterised by the occurrence of the Philadelphia (Ph) chromosome (9/22 translocation) and the formation of a fusion protein--the BCR-ABL transcript with constitutive activation of the BCR-ABL tyrosine kinase and consequent changes in the intracellular signal transduction, which is responsible for the deregulated myeloid cell proliferation. STI571 (signal transduction inhibition number 571) is a potent and selective inhibitor of the BCR-ABL tyrosine kinase. In the chronic phase of the disease, normal peripheral blood values are achieved within the first month of treatment in the large majority of patients and in many patients also a cytogenic response within the following months. The results in the advanced phase are far less favourable, which is explained by the development of resistance owing to reactivation of the BCR-ABL signal transduction. Side effects are primarily nausea, vomiting, various rashes, oedema, most often in the periorbital region, and musculoskeletal symptoms, including muscle cramps. Perspectives for treatment with STI571 are described, as are combinations with alpha-interferon and other cytostatics with a synergistic profile.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_093',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'Bcr-Abl', 'answer_start': 355}]}],\n",
       "   'context': 'Imatinib mesylate (STI571) therapy for Philadelphia chromosome-positive chronic myelogenous leukemia in blast phase. Molecular abnormalities caused by the hybrid Bcr-Abl gene are causally associated with the development and progression of Philadelphia chromosome-positive (Ph(+)) chronic myelogenous leukemia (CML). Imatinib mesylate (STI571), a specific Bcr-Abl tyrosine-kinase signal-transduction inhibitor, has shown encouraging activity in phase I and II studies of CML. Here, we describe the use of imatinib mesylate to treat 75 patients in blast-phase CML (median age, 53 years; 65 with nonlymphoid and 10 with lymphoid blasts), and compare the results with those of a historical control group treated with standard cytarabine-based therapy. Imatinib mesylate was given as oral doses at 300 to 1000 mg per day and was the first salvage therapy for 47 patients. The objective response rate was 52% (39 of 75 patients: 16 had complete and 3 had partial hematologic response; 12 had hematologic improvement; 7 returned to second chronic phase; and 1 had a complete response in extramedullary blastic disease). Response rates were not different between nonlymphoid and lymphoid groups. The cytogenetic response rate was 16% (12 patients: 5 complete, 3 partial [Ph(+) below 35%], and 4 minor [Ph(+), 34% to 90%]). The estimated median overall survival was 6.5 months; the estimated 1-year survival was 22%. Response to therapy (landmark analysis at 8 weeks) was associated with survival prolongation. Compared with standard cytarabine combinations, imatinib mesylate therapy was less toxic and produced a higher response rate (55% versus 29%, P =.001), longer median survival (7 versus 4 months, P =.04), and lower 4-week induction mortality (4% versus 15%, P =.07). Imatinib mesylate is currently being tested in combination with other drugs to improve the prognosis for blast-phase CML.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_094',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 156}]}],\n",
       "   'context': 'Hematologic and cytogenetic responses to imatinib mesylate in chronic myelogenous leukemia. BACKGROUND: Chronic myelogenous leukemia (CML) is caused by the BCR-ABL tyrosine kinase, the product of the Philadelphia chromosome. Imatinib mesylate, formerly STI571, is a selective inhibitor of this kinase. METHODS: A total of 532 patients with late--chronic-phase CML in whom previous therapy with interferon alfa had failed were treated with 400 mg of oral imatinib daily. Patients were evaluated for cytogenetic and hematologic responses. Time to progression, survival, and toxic effects were also evaluated. RESULTS: Imatinib induced major cytogenetic responses in 60 percent of the 454 patients with confirmed chronic-phase CML and complete hematologic responses in 95 percent. After a median follow-up of 18 months, CML had not progressed to the accelerated or blast phases in an estimated 89 percent of patients, and 95 percent of the patients were alive. Grade 3 or 4 nonhematologic toxic effects were infrequent, and hematologic toxic effects were manageable. Only 2 percent of patients discontinued treatment because of drug-related adverse events, and no treatment-related deaths occurred. CONCLUSIONS: Imatinib induced high rates of cytogenetic and hematologic responses in patients with chronic-phase CML in whom previous interferon therapy had failed.'},\n",
       "  {'qas': [{'id': '5324a8ac9b2d7acc7e000018_095',\n",
       "     'question': 'What tyrosine kinase, involved in a Philadelphia- chromosome positive chronic myelogenous leukemia, is the target of Imatinib (Gleevec)?',\n",
       "     'answers': [{'text': 'BCR-ABL', 'answer_start': 263}]}],\n",
       "   'context': '[Antileukemic drug--a selective inhibitor of BCR-ABL tyrosine kynase, imatinib(STI571)]. Chronic myeloid leukemia (CML) is a clonal hematopoietic stem cell disorder characterized by Philadelphia chromosome and resultant production of the constitutively activated BCR-ABL tyrosine kinase. Imatinib (STI571), selective inhibitor of the ABL-tyrosine kinase, inhibits the activity of BCR-ABL tyrosine kinase. A phase I and II study of STI571 showed remarkable cytogenetic effect in patients with interferon-refractory CML, offering new hope for therapy for CML. It will, however, require long-term follow-up data from phase II and III clinical studies to validate the effect of STI571 on survival. As therapy for CML improves, monitoring minimal residual disease will be important.'},\n",
       "  {'qas': [{'id': '571e172bbb137a4b0c000002_001',\n",
       "     'question': 'When was empagliflozin FDA approved?',\n",
       "     'answers': [{'text': '2014', 'answer_start': 228}]}],\n",
       "   'context': 'Empagliflozin, an SGLT2 inhibitor for the treatment of type 2 diabetes mellitus: a review of the evidence. OBJECTIVE: To review available studies of empagliflozin, a sodium glucose co-transporter-2 (SGLT2) inhibitor approved in 2014 by the European Commission and the United States Food and Drug Administration for the treatment of type 2 diabetes mellitus (T2DM). DATA SOURCES: PubMed was searched using the search terms empagliflozin, BI 10773, and BI10773, for entries between January 1, 2000, and December 1, 2014. Reference lists from retrieved articles were searched manually for additional peer-reviewed publications. STUDY SELECTION AND DATA EXTRACTION: All publications reporting clinical trials of empagliflozin were eligible for inclusion. DATA SYNTHESIS: Empagliflozin is a new once-daily oral SGLT2 inhibitor with a mechanism of action that is independent of β-cell function and the insulin pathway. Data from a comprehensive phase III clinical trial program have demonstrated its efficacy as monotherapy, as add-on to other glucose-lowering agents, and in different patient populations. In these studies, empagliflozin resulted in improvements in blood glucose levels as well as reductions in body weight and blood pressure. Empagliflozin was well tolerated and was not associated with an increased risk of hypoglycemia versus placebo. CONCLUSION: The oral antidiabetes agent, empagliflozin, can be used as monotherapy or alongside other glucose-lowering treatments, including insulin, to treat T2DM.'},\n",
       "  {'qas': [{'id': '571e172bbb137a4b0c000002_002',\n",
       "     'question': 'When was empagliflozin FDA approved?',\n",
       "     'answers': [{'text': '2014', 'answer_start': 228}]}],\n",
       "   'context': 'Empagliflozin, an SGLT2 inhibitor for the treatment of type 2 diabetes mellitus: a review of the evidence. OBJECTIVE: To review available studies of empagliflozin, a sodium glucose co-transporter-2 (SGLT2) inhibitor approved in 2014 by the European Commission and the United States Food and Drug Administration for the treatment of type 2 diabetes mellitus (T2DM). DATA SOURCES: PubMed was searched using the search terms empagliflozin, BI 10773, and BI10773, for entries between January 1, 2000, and December 1, 2014. Reference lists from retrieved articles were searched manually for additional peer-reviewed publications. STUDY SELECTION AND DATA EXTRACTION: All publications reporting clinical trials of empagliflozin were eligible for inclusion. DATA SYNTHESIS: Empagliflozin is a new once-daily oral SGLT2 inhibitor with a mechanism of action that is independent of β-cell function and the insulin pathway. Data from a comprehensive phase III clinical trial program have demonstrated its efficacy as monotherapy, as add-on to other glucose-lowering agents, and in different patient populations. In these studies, empagliflozin resulted in improvements in blood glucose levels as well as reductions in body weight and blood pressure. Empagliflozin was well tolerated and was not associated with an increased risk of hypoglycemia versus placebo. CONCLUSION: The oral antidiabetes agent, empagliflozin, can be used as monotherapy or alongside other glucose-lowering treatments, including insulin, to treat T2DM.'},\n",
       "  {'qas': [{'id': '571e172bbb137a4b0c000002_003',\n",
       "     'question': 'When was empagliflozin FDA approved?',\n",
       "     'answers': [{'text': '2014', 'answer_start': 228}]}],\n",
       "   'context': 'Empagliflozin, an SGLT2 inhibitor for the treatment of type 2 diabetes mellitus: a review of the evidence. OBJECTIVE: To review available studies of empagliflozin, a sodium glucose co-transporter-2 (SGLT2) inhibitor approved in 2014 by the European Commission and the United States Food and Drug Administration for the treatment of type 2 diabetes mellitus (T2DM). DATA SOURCES: PubMed was searched using the search terms empagliflozin, BI 10773, and BI10773, for entries between January 1, 2000, and December 1, 2014. Reference lists from retrieved articles were searched manually for additional peer-reviewed publications. STUDY SELECTION AND DATA EXTRACTION: All publications reporting clinical trials of empagliflozin were eligible for inclusion. DATA SYNTHESIS: Empagliflozin is a new once-daily oral SGLT2 inhibitor with a mechanism of action that is independent of β-cell function and the insulin pathway. Data from a comprehensive phase III clinical trial program have demonstrated its efficacy as monotherapy, as add-on to other glucose-lowering agents, and in different patient populations. In these studies, empagliflozin resulted in improvements in blood glucose levels as well as reductions in body weight and blood pressure. Empagliflozin was well tolerated and was not associated with an increased risk of hypoglycemia versus placebo. CONCLUSION: The oral antidiabetes agent, empagliflozin, can be used as monotherapy or alongside other glucose-lowering treatments, including insulin, to treat T2DM.'},\n",
       "  {'qas': [{'id': '56b330bb39c782df06000001_001',\n",
       "     'question': 'Which R/bioconductor package is used for integrative genomics visualizations?',\n",
       "     'answers': [{'text': 'Sushi.R', 'answer_start': 0}]}],\n",
       "   'context': 'Sushi.R: flexible, quantitative and integrative genomic visualizations for publication-quality multi-panel figures. Interpretation and communication of genomic data require flexible and quantitative tools to analyze and visualize diverse data types, and yet, a comprehensive tool to display all common genomic data types in publication quality figures does not exist to date. To address this shortcoming, we present Sushi.R, an R/Bioconductor package that allows flexible integration of genomic visualizations into highly customizable, publication-ready, multi-panel figures from common genomic data formats including Browser Extensible Data (BED), bedGraph and Browser Extensible Data Paired-End (BEDPE). Sushi.R is open source and made publicly available through GitHub (https://github.com/dphansti/Sushi) and Bioconductor (http://bioconductor.org/packages/release/bioc/html/Sushi.html).'},\n",
       "  {'qas': [{'id': '56b330bb39c782df06000001_002',\n",
       "     'question': 'Which R/bioconductor package is used for integrative genomics visualizations?',\n",
       "     'answers': [{'text': 'Sushi.R', 'answer_start': 416}]}],\n",
       "   'context': 'Sushi.R: flexible, quantitative and integrative genomic visualizations for publication-quality multi-panel figures. Interpretation and communication of genomic data require flexible and quantitative tools to analyze and visualize diverse data types, and yet, a comprehensive tool to display all common genomic data types in publication quality figures does not exist to date. To address this shortcoming, we present Sushi.R, an R/Bioconductor package that allows flexible integration of genomic visualizations into highly customizable, publication-ready, multi-panel figures from common genomic data formats including Browser Extensible Data (BED), bedGraph and Browser Extensible Data Paired-End (BEDPE). Sushi.R is open source and made publicly available through GitHub (https://github.com/dphansti/Sushi) and Bioconductor (http://bioconductor.org/packages/release/bioc/html/Sushi.html).'},\n",
       "  {'qas': [{'id': '56b330bb39c782df06000001_003',\n",
       "     'question': 'Which R/bioconductor package is used for integrative genomics visualizations?',\n",
       "     'answers': [{'text': 'Sushi.R', 'answer_start': 706}]}],\n",
       "   'context': 'Sushi.R: flexible, quantitative and integrative genomic visualizations for publication-quality multi-panel figures. Interpretation and communication of genomic data require flexible and quantitative tools to analyze and visualize diverse data types, and yet, a comprehensive tool to display all common genomic data types in publication quality figures does not exist to date. To address this shortcoming, we present Sushi.R, an R/Bioconductor package that allows flexible integration of genomic visualizations into highly customizable, publication-ready, multi-panel figures from common genomic data formats including Browser Extensible Data (BED), bedGraph and Browser Extensible Data Paired-End (BEDPE). Sushi.R is open source and made publicly available through GitHub (https://github.com/dphansti/Sushi) and Bioconductor (http://bioconductor.org/packages/release/bioc/html/Sushi.html).'},\n",
       "  {'qas': [{'id': '56b330bb39c782df06000001_004',\n",
       "     'question': 'Which R/bioconductor package is used for integrative genomics visualizations?',\n",
       "     'answers': [{'text': 'Sushi.R', 'answer_start': 416}]}],\n",
       "   'context': 'Sushi.R: flexible, quantitative and integrative genomic visualizations for publication-quality multi-panel figures. Interpretation and communication of genomic data require flexible and quantitative tools to analyze and visualize diverse data types, and yet, a comprehensive tool to display all common genomic data types in publication quality figures does not exist to date. To address this shortcoming, we present Sushi.R, an R/Bioconductor package that allows flexible integration of genomic visualizations into highly customizable, publication-ready, multi-panel figures from common genomic data formats including Browser Extensible Data (BED), bedGraph and Browser Extensible Data Paired-End (BEDPE). Sushi.R is open source and made publicly available through GitHub (https://github.com/dphansti/Sushi) and Bioconductor (http://bioconductor.org/packages/release/bioc/html/Sushi.html).'},\n",
       "  {'qas': [{'id': '57090c33cf1c325851000013_001',\n",
       "     'question': 'How many genes are imprinted in the human genome?',\n",
       "     'answers': [{'text': ' fewer than 100', 'answer_start': 304}]}],\n",
       "   'context': 'dsPIG: a tool to predict imprinted genes from the deep sequencing of whole transcriptomes. BACKGROUND: Dysregulation of imprinted genes, which are expressed in a parent-of-origin-specific manner, plays an important role in various human diseases, such as cancer and behavioral disorder. To date, however, fewer than 100 imprinted genes have been identified in the human genome. The recent availability of high-throughput technology makes it possible to have large-scale prediction of imprinted genes. Here we propose a Bayesian model (dsPIG) to predict imprinted genes on the basis of allelic expression observed in mRNA-Seq data of independent human tissues. RESULTS: Our model (dsPIG) was capable of identifying imprinted genes with high sensitivity and specificity and a low false discovery rate when the number of sequenced tissue samples was fairly large, according to simulations. By applying dsPIG to the mRNA-Seq data, we predicted 94 imprinted genes in 20 cerebellum samples and 57 imprinted genes in 9 diverse tissue samples with expected low false discovery rates. We also assessed dsPIG using previously validated imprinted and non-imprinted genes. With simulations, we further analyzed how imbalanced allelic expression of non-imprinted genes or different minor allele frequencies affected the predictions of dsPIG. Interestingly, we found that, among biallelically expressed genes, at least 18 genes expressed significantly more transcripts from one allele than the other among different individuals and tissues. CONCLUSION: With the prevalence of the mRNA-Seq technology, dsPIG has become a useful tool for analysis of allelic expression and large-scale prediction of imprinted genes. For ease of use, we have set up a web service and also provided an R package for dsPIG at http://www.shoudanliang.com/dsPIG/.'},\n",
       "  {'qas': [{'id': '57090c33cf1c325851000013_002',\n",
       "     'question': 'How many genes are imprinted in the human genome?',\n",
       "     'answers': [{'text': ' fewer than 100', 'answer_start': 304}]}],\n",
       "   'context': 'dsPIG: a tool to predict imprinted genes from the deep sequencing of whole transcriptomes. BACKGROUND: Dysregulation of imprinted genes, which are expressed in a parent-of-origin-specific manner, plays an important role in various human diseases, such as cancer and behavioral disorder. To date, however, fewer than 100 imprinted genes have been identified in the human genome. The recent availability of high-throughput technology makes it possible to have large-scale prediction of imprinted genes. Here we propose a Bayesian model (dsPIG) to predict imprinted genes on the basis of allelic expression observed in mRNA-Seq data of independent human tissues. RESULTS: Our model (dsPIG) was capable of identifying imprinted genes with high sensitivity and specificity and a low false discovery rate when the number of sequenced tissue samples was fairly large, according to simulations. By applying dsPIG to the mRNA-Seq data, we predicted 94 imprinted genes in 20 cerebellum samples and 57 imprinted genes in 9 diverse tissue samples with expected low false discovery rates. We also assessed dsPIG using previously validated imprinted and non-imprinted genes. With simulations, we further analyzed how imbalanced allelic expression of non-imprinted genes or different minor allele frequencies affected the predictions of dsPIG. Interestingly, we found that, among biallelically expressed genes, at least 18 genes expressed significantly more transcripts from one allele than the other among different individuals and tissues. CONCLUSION: With the prevalence of the mRNA-Seq technology, dsPIG has become a useful tool for analysis of allelic expression and large-scale prediction of imprinted genes. For ease of use, we have set up a web service and also provided an R package for dsPIG at http://www.shoudanliang.com/dsPIG/.'},\n",
       "  {'qas': [{'id': '5157539ed24251bc0500008a_001',\n",
       "     'question': 'Gene silencing can be achieved by RNA interference (RNAi) in eukaryotic organisms. What is the name of the analogous process in prokaryotic organisms?',\n",
       "     'answers': [{'text': 'CRISPR-Cas', 'answer_start': 83}]}],\n",
       "   'context': 'CRISPR-Cas: evolution of an RNA-based adaptive immunity system in prokaryotes. The CRISPR-Cas (clustered regularly interspaced short palindromic repeats, CRISPR-associated genes) is an adaptive immunity system in bacteria and archaea that functions via a distinct self-non-self recognition mechanism that is partially analogous to the mechanism of eukaryotic RNA interference (RNAi). The CRISPR-Cas system incorporates fragments of virus or plasmid DNA into the CRISPR repeat cassettes and employs the processed transcripts of these spacers as guide RNAs to cleave the cognate foreign DNA or RNA. The Cas proteins, however, are not homologous to the proteins involved in RNAi and comprise numerous, highly diverged families. The majority of the Cas proteins contain diverse variants of the RNA recognition motif (RRM), a widespread RNA-binding domain. Despite the fast evolution that is typical of the cas genes, the presence of diverse versions of the RRM in most Cas proteins provides for a simple scenario for the evolution of the three distinct types of CRISPR-cas systems. In addition to several proteins that are directly implicated in the immune response, the cas genes encode a variety of proteins that are homologous to prokaryotic toxins that typically possess nuclease activity. The predicted toxins associated with CRISPR-Cas systems include the essential Cas2 protein, proteins of COG1517 that, in addition to a ligand-binding domain and a helix-turn-helix domain, typically contain different nuclease domains and several other predicted nucleases. The tight association of the CRISPR-Cas immunity systems with predicted toxins that, upon activation, would induce dormancy or cell death suggests that adaptive immunity and dormancy/suicide response are functionally coupled. Such coupling could manifest in the persistence state being induced and potentially providing conditions for more effective action of the immune system or in cell death being triggered when immunity fails.'},\n",
       "  {'qas': [{'id': '5157539ed24251bc0500008a_002',\n",
       "     'question': 'Gene silencing can be achieved by RNA interference (RNAi) in eukaryotic organisms. What is the name of the analogous process in prokaryotic organisms?',\n",
       "     'answers': [{'text': 'CRISPR-Cas', 'answer_start': 100}]}],\n",
       "   'context': \"RNA-guided RNA cleavage by a CRISPR RNA-Cas protein complex. Compelling evidence indicates that the CRISPR-Cas system protects prokaryotes from viruses and other potential genome invaders. This adaptive prokaryotic immune system arises from the clustered regularly interspaced short palindromic repeats (CRISPRs) found in prokaryotic genomes, which harbor short invader-derived sequences, and the CRISPR-associated (Cas) protein-coding genes. Here, we have identified a CRISPR-Cas effector complex that is comprised of small invader-targeting RNAs from the CRISPR loci (termed prokaryotic silencing (psi)RNAs) and the RAMP module (or Cmr) Cas proteins. The psiRNA-Cmr protein complexes cleave complementary target RNAs at a fixed distance from the 3' end of the integral psiRNAs. In Pyrococcus furiosus, psiRNAs occur in two size forms that share a common 5' sequence tag but have distinct 3' ends that direct cleavage of a given target RNA at two distinct sites. Our results indicate that prokaryotes possess a unique RNA silencing system that functions by homology-dependent cleavage of invader RNAs.\"},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_001',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'hypertrophic cardiomyopathy',\n",
       "       'answer_start': 1205}]}],\n",
       "   'context': 'Active surveillance of sudden cardiac death in young athletes by periodic Internet searches. The authors hypothesized that prospective, systematic Internet searches could identify occurrences of sudden cardiac death (SCD) in athletes and would be useful for establishing a system of active surveillance. Weekly advanced Google searches of the Internet were conducted for cases of SCD in young athletes during a 12-month period (2007-2008). Athletes ages 11-30 years who collapsed during a game, practice, or within an hour of exercise were included in the study. Individuals with known histories of cardiac issues and events occurring outside the United States were excluded. Verification of SCD was by autopsy reports and death certificates from county coroner offices and vital record agencies. Initially, 71 events were identified. Verification for the cause of death by coroner reports was possible in 45 cases, 43 (96 %) of which were confirmed to be SCDs. A total of 69 individuals 11-30 years of age (mean 17 ± 5 years) died suddenly of cardiovascular causes while participating in 15 different organized sports and a variety of nonorganized physical activities. The most common cause of death was hypertrophic cardiomyopathy (30 %), followed by coronary artery anomalies (9 %), and myocarditis (9 %). The incidence of athlete SCD, the types of sports involved, and the cardiac causes of death in our study were comparable with those of previous reports. Readily available Internet searches have the potential to be a powerful tool for identifying occurrences of athlete SCD. An active surveillance system using Google searches followed by coroner report verification can provide important epidemiologic and clinical information.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_002',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'hypertrophic cardiomyopathy',\n",
       "       'answer_start': 285}]}],\n",
       "   'context': 'Can an athlete have too much ticker? Hypertrophic cardiomyopathy in young athletes. Sudden cardiac death (SCD) is an uncommon but devastating potential consequence of participation in competitive sport. It is seen in adolescent and young adult athletes. The most common cause of this, hypertrophic cardiomyopathy (HCM), is a genetic disorder responsible for more than a third of cases and is manageable. Screening is undertaken for HCM, using differing strategies in Europe and North America. Screening and early diagnosis have reduced the mortality rate but has come at a significant economic cost. The evidence and relevant arguments for and against screening are presented together with management strategies as reflected by an illustrative case.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_003',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'Hypertrophic cardiomyopathy',\n",
       "       'answer_start': 136}]}],\n",
       "   'context': 'Risk stratification and role of implantable defibrillators for prevention of sudden death in patients with hypertrophic cardiomyopathy. Hypertrophic cardiomyopathy (HCM) is the most common cause of sudden cardiac death (SCD) in young people, including trained athletes. It is now 30 years since the introduction of implantable cardioverter-defibrillators (ICDs) to clinical cardiovascular practice and coronary artery disease, and now device therapy represents the most significant therapeutic innovation and the only definitive strategy for prolonging the life of HCM patients. ICDs have proved effective in preventing SCD in young HCM patients with appropriate intervention rates of 11% for secondary and 4% for primary prevention, despite massive left ventricular (LV) hypertrophy, LV outflow obstruction, diastolic dysfunction or microvascular ischemia. Targeting candidates for prophylactic ICD therapy can be complex, compounded by the unpredictability of the arrhythmogenic substrate, the absence of a dominant risk factor, and difficulty in assembling randomized trials. However, a single major risk factor is often sufficient to justify an ICD, although additional markers and other disease features can resolve ambiguous decision-making. Nevertheless, the absence of all risk factors does not convey absolute immunity to SCD. The current risk factor algorithm, when combined with a measure of individual physician judgment (and patient autonomy considerations), is an effective guide to identifying high-risk HCM patients. ICDs have altered the natural history of HCM for many patients and provided an opportunity to achieve many decades of productive life, and the potential for normal or near-normal longevity. Indeed, prevention of SCD has now become a new paradigm in the management of HCM.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_004',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'Hypertrophic cardiomyopathy',\n",
       "       'answer_start': 45}]}],\n",
       "   'context': 'Sudden death in hypertrophic cardiomyopathy. Hypertrophic cardiomyopathy (HCM) is regarded as the most common cause of sudden cardiac death in young people (including trained athletes). However, assessing sudden death (SD) risk and identifying the most appropriate candidates for prophylactic device therapy is a complex process compounded by the unpredictability of the underlying arrhythmogenic substrate, absence of a single dominant and quantitative risk maker for this heterogeneous disease, and also the difficulty encountered in assembling sufficiently powered prospective and randomized trials in large patient populations. Patients with multiple risk factors and most young patients with one strong and unequivocal risk marker can be considered candidates for primary prevention defibrillators. Despite certain limitations, the current risk factor algorithm (when combined with a measure of individual physician judgment) has proved to be an effective strategy for targeting high-risk status. This approach has served the HCM patient population well, as evidenced by the significant appropriate defibrillator intervention rates, although a very small proportion of patients without conventional risk factors may also be at risk for SD. Indeed, the introduction of implantable defibrillators to the HCM patient population represents a new paradigm for clinical practice, offering the only proven protection against SD by virtue of effectively terminating ventricular tachycardia/fibrillation. In the process, implantable defibrillators have altered the natural history of HCM, potentially providing the opportunity of normal or near-normal longevity for many patients. Prevention of SD is now an integral, albeit challenging, component of HCM management.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_005',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'Hypertrophic cardiomyopathy',\n",
       "       'answer_start': 90}]}],\n",
       "   'context': 'Implantable defibrillators and prevention of sudden death in hypertrophic cardiomyopathy. Hypertrophic cardiomyopathy (HCM) is the most common cause of sudden cardiac death in young people, including trained athletes. The implantable cardioverter-defibrillator (ICD), although initially designed as a treatment for older patients with coronary artery disease, has more recently proved to be a safe and effective therapeutic intervention in young patients with HCM, both for primary or secondary prevention of sudden death. The largest such report of >500 patients showed that the ICD intervened appropriately to abort ventricular tachycardia/fibrillation (VT/VF) in 20% of patients over an average follow-up period of only 3.7 years, at a rate of about 4% per year in those patients implanted prophylactically, and often with considerable delays of up to 10 years. Extensive experience with high-risk HCM patients showed that appropriate device discharges for VT/VF occur with similar frequency in patients with 1, 2, or > or = 3 noninvasive risk markers. Despite the extreme morphology characteristic of HCM, often with massive degrees of left ventricular (LV) hypertrophy and/or LV outflow tract obstruction, ICDs performed in a highly effective fashion, with failure to convert life-threatening arrhythmias extraordinarily rare. In conclusion, in a large high-risk HCM cohort, ICD interventions for life-threatening ventricular tachyarrhythmias were frequent and highly effective in restoring normal rhythm. An important proportion of ICD discharges occurred in primary prevention patients with only one risk factor. Therefore, a single marker of high risk may represent sufficient evidence to justify the recommendation for a prophylactic ICD in selected patients with HCM.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_006',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'Hypertrophic cardiomyopathy',\n",
       "       'answer_start': 39}]}],\n",
       "   'context': 'Can sudden cardiac death be prevented? Hypertrophic cardiomyopathy is regarded as the most common cause of sudden cardiac death in young people (including trained athletes). Introduction of implantable cardioverter-defibrillators to the hypertrophic cardiomyopathy patient population represents a new paradigm for clinical practice and perhaps the most significant advance in the management of this disease to date. Implantable defibrillators offer the only proven protection against sudden death by virtue of effectively terminating ventricular tachycardia/fibrillation and, in the process, altering the natural history of hypertrophic cardiomyopathy and providing the potential opportunity of normal or near-normal longevity for many patients. However, targeting the most appropriate candidates for prophylactic device therapy can be complex, compounded by the unpredictability of the underlying arrhythmogenic substrate, absence of a single dominant and quantitative risk marker in this heterogeneous disease, and the historical difficulty in assembling sufficiently powered prospective and randomized trials in large patient populations. Nevertheless, the current risk factor algorithm, when combined with a measure of individual physician judgment, is an effective strategy for identifying high-risk patients. Indeed, prevention of sudden death has now become an integral, albeit challenging, component of overall hypertrophic cardiomyopathy management.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_007',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'hypertrophic cardiomyopathy',\n",
       "       'answer_start': 580}]}],\n",
       "   'context': '[Causes of sudden cardiac death in athletes]. INTRODUCTION: Sudden cardiac death in athletes is a growing problem, despite the huge existing knowledge in medicine and sports. EFFECTS OF VIGOROUS PHYSICAL ACTIVITY: In response to vigorous physical activity, the body undergoes profound morphologic and functional changes. These changes are usually healthy, but sometimes may gravitate to some cardiac diseases. But still, most saudden cardiac deaths are due to previous unknown diseases. CAUSES OF SUDDEN CARDIAC DEATH: The most common cause of sudden cardiac death in athletes is hypertrophic cardiomyopathy. Other reasons are congenital coronary artery anomalies, nivocarditis, dilatative cardiomyopathy, arrhythmogenic cardiomyopathy of the right ventricle, sarcoidosis, mitral valve prolapse, aortic valve stenosis, atherosclerosis, long QT syndrome, and blunt impact to the chest. CONCLUSION: Bearing in mind the above mentioned, more frequent physical examinations of athletes are recommended.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_008',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'Hypertrophic cardiomyopathy',\n",
       "       'answer_start': 129}]}],\n",
       "   'context': 'Hypertrophic cardiomyopathy: electrocardiographic manifestations and other important considerations for the emergency physician. Hypertrophic cardiomyopathy (HCM) is one of the most common inherited primary cardiac disorders and the most common cause of sudden cardiac death in young athletes. With advances in technology, it is now recognized that HCM affects individuals of all ages. Many patients with HCM will have a benign course with few symptoms. Some patients, however, possess risk factors that greatly increase the likelihood of sudden death if their disease remains undiagnosed. Therefore, it is imperative that emergency physicians be familiar with the symptoms and typical electrocardiogram manifestations of HCM. Three illustrative cases are presented with a review of the disease.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_009',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'hypertrophic cardiomyopathy',\n",
       "       'answer_start': 199}]}],\n",
       "   'context': 'Sudden death prevented in hypertrophic cardiomyopathy. The most common cause of sudden cardiac death in individuals aged less than 35 years, including competitive athletes, is the inherited disorder hypertrophic cardiomyopathy (HCM). Until recently, no therapeutic intervention had been identified to prevent sudden death in HCM. This case report highlights the role of the implantable cardioverter defibrillator (ICD) in preventing sudden death in patients with HCM. The report highlights the importance of risk stratification in patients with HCM in order to identify those individuals who would most likely benefit from ICD therapy. The ICD implantation is now the treatment of choice in preventing sudden death in selected (highest risk) populations with HCM. In the future, understanding the molecular basis of sudden death in HCM may identify potential new gene-based therapeutic strategies.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_010',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'Hypertrophic cardiomyopathy',\n",
       "       'answer_start': 126}]}],\n",
       "   'context': 'Sudden cardiac death in patients with hypertrophic cardiomyopathy: from bench to bedside with an emphasis on genetic markers. Hypertrophic cardiomyopathy (HCM) is the most common cause of death in the young, particularly in young competitive athletes. Death often occurs suddenly in asymptomatic, apparently healthy individuals. Several clinical parameters as well as genetic factors have been characterized that can identify those HCM patients who are at high risk for sudden cardiac death (SCD). The clinical parameters that have some predictive values for SCD in HCM patients are the following: a prior history of SCD, a family history of SCD, history of syncope, symptomatic ventricular tachycardia on Holter monitoring, inducible ventricular tachycardia during electrophysiologic studies, and myocardial ischemia in children with HCM. Recent identification of mutations in the beta myosin heavy chain gene and genotype-phenotype correlation in HCM patients have shown that the beta myosin heavy chain mutations are also prognosticators in HCM families. Several mutations such as Arg403Gln and Arg719Gln are associated with a high incidence of SCD, while Leu908Val mutation is associated with a benign course and a low incidence of SCD in HCM families. Additional genetic factors such as a polymorphism in angiotensin-converting enzyme I gene may also contribute to a high incidence of SCD in HCM families. Identification and characterization of HCM patients at high risk for SCD provide the opportunity to render prophylactic therapeutic interventions, such as implantation of defibrillators, in these individuals.'},\n",
       "  {'qas': [{'id': '530cf4e0c8a0b4a00c000006_011',\n",
       "     'question': 'Which is the most common cause of sudden cardiac death in young athletes?',\n",
       "     'answers': [{'text': 'Hypertrophic cardiomyopathy',\n",
       "       'answer_start': 66}]}],\n",
       "   'context': 'Surviving competitive athletics with hypertrophic cardiomyopathy. Hypertrophic cardiomyopathy (HC) is probably the most common cause of sudden cardiac death in youthful athletes, and this diagnosis has represented a contraindication to continued participation in competitive sports. Less well appreciated is the fact that within the clinical spectrum of HC are patients who, despite having this disease, have been able to undertake particularly intensive and often extraordinary levels of training for sports competition over many years without dying suddenly. Fourteen such patients (13 men and 1 woman, aged 30 to 66 years [mean 43]) form the present study group. HC was initially identified at 24 to 57 years of age (mean 34), usually under fortuitous circumstances. Patients most often competed in distance running (including the marathon, 7), but also in swimming, triathalon, basketball, and football. The duration of training ranged from 6 to 22 years (mean 15) and 5 continue to train and compete actively. The magnitude of training, competition, and achievement was considerable in most patients; 12 of the 14 performed either at the national, collegiate or professional level in their sport, completed numerous marathon and triathalon events, or sustained particularly rigorous training regimens of > or = 50 miles/week. Echocardiographic studies demonstrated a left ventricular wall thickness of 18 to 28 mm (mean 20) in most patients (12 of 14) having a relatively localized pattern of ventricular septal hypertrophy. It is possible for some patients with HC to tolerate particularly intense athletic training and competition for many years, and even maintain high levels of achievement without incurring symptoms and disease progression or dying suddenly.(ABSTRACT TRUNCATED AT 250 WORDS)'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_001',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 260}]}],\n",
       "   'context': 'Intermediate filament protein accumulation in motor neurons derived from giant axonal neuropathy iPSCs rescued by restoration of gigaxonin. Giant axonal neuropathy (GAN) is a progressive neurodegenerative disease caused by autosomal recessive mutations in the GAN gene resulting in a loss of a ubiquitously expressed protein, gigaxonin. Gene replacement therapy is a promising strategy for treatment of the disease; however, the effectiveness and safety of gigaxonin reintroduction have not been tested in human GAN nerve cells. Here we report the derivation of induced pluripotent stem cells (iPSCs) from three GAN patients with different GAN mutations. Motor neurons differentiated from GAN iPSCs exhibit accumulation of neurofilament (NF-L) and peripherin (PRPH) protein and formation of PRPH aggregates, the key pathological phenotypes observed in patients. Introduction of gigaxonin either using a lentiviral vector or as a stable transgene resulted in normalization of NEFL and PRPH levels in GAN neurons and disappearance of PRPH aggregates. Importantly, overexpression of gigaxonin had no adverse effect on survival of GAN neurons, supporting the feasibility of gene replacement therapy. Our findings demonstrate that GAN iPSCs provide a novel model for studying human GAN neuropathologies and for the development and testing of new therapies in relevant cell types.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_002',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 506}]}],\n",
       "   'context': 'Giant axonal neuropathy diagnosed on skin biopsy. Evaluation of hereditary axonal neuropathy in childhood is complex. Often, the child has to be subjected to general anaesthesia for a nerve biopsy to guide further genetic testing, which may or may not be readily available. We describe a toddler with clinical features suggesting giant axonal neuropathy (GAN), whose diagnosis was confirmed by minimally invasive skin biopsy and corroborated by the finding of compound heterozygous mutations involving the GAN gene, including a novel interstitial microdeletion at 16q23.2 detected by microarray and a point mutation detected by direct sequencing.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_003',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 205}]}],\n",
       "   'context': 'The absence of curly hair is associated with a milder phenotype in Giant Axonal Neuropathy. Giant Axonal Neuropathy is a pediatric neurodegenerative disorder caused by autosomal recessive mutations in the GAN gene on chromosome 16q24.1. Mutations in the GAN gene lead to functional impairment of the cytoskeletal protein gigaxonin and a generalized disorder of intermediate filaments, including neurofilaments in axons. Tightly curled hair is a common but not universal feature of Giant Axonal Neuropathy. The pathogenesis of curly hair is unknown, although disruption of keratin architecture is thought to play a role. As part of a broader natural history study of Giant Axonal Neuropathy, we found that the absence of curly hair is correlated with superior motor function (p=0.013) when controlling for age, as measured by the Gross Motor Function Measure. Theoretically, higher levels of functional gigaxonin protein or compensatory mechanisms could produce fewer abnormalities of neurofilaments and keratin, accounting for this phenotype. We suggest that straight-haired patients with Giant Axonal Neuropathy are potentially underdiagnosed due to their divergence from the classic phenotype of the disease. Due to their non-specific features of an axonal neuropathy, these patients may be misdiagnosed with Charcot-Marie-Tooth Disease type 2. Genetic testing for Giant Axonal Neuropathy should be considered in relevant cases of Charcot-Marie-Tooth Disease type 2.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_004',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 212}]}],\n",
       "   'context': 'Heterogeneity of axonal pathology in Chinese patients with giant axonal neuropathy. INTRODUCTION: Giant axonal neuropathy (GAN) is a rare autosomal recessive neurodegenerative disorder caused by mutations in the GAN gene. Herein we report ultrastructural changes in Chinese patients with GAN. METHODS: General clinical assessment, sural nerve biopsy, and genetic analysis were performed. RESULTS: Sural biopsy revealed giant axons in 3 patients, 2 with a mild phenotype and 1 with a classical phenotype. Ultrastructurally, all patients had giant axons filled with closely packed neurofilaments. In addition, the classical patient had some axons containing irregular tubular-like structures. GAN mutation analysis revealed novel compound heterozygous c.98A>C and c.158C>T mutations in the BTB domain in 1 mild patient, a novel homozygous c.371T>G mutation in the BACK domain in another mild patient, and a novel c.1342G>T homozygous mutation in the Kelch domain in the classical patient. CONCLUSION: Closely packed neurofilaments in giant axons are common pathological changes in Chinese patients with GAN, whereas irregular tubular-like structures appear in the classical type of this neuropathy.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_005',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 391}]}],\n",
       "   'context': 'Giant axonal neuropathy: An updated perspective on its pathology and pathogenesis. Giant axonal neuropathy (GAN) is a rare pediatric neurodegenerative disease. It is best known for the \"giant\" axons caused by accumulations of intermediate filaments. The disease is progressive, with onset around age 3 years and death by the third decade of life. GAN results from recessive mutations in the GAN gene encoding gigaxonin, and our analysis of all reported mutations shows that they are distributed throughout the protein structure. Precisely how these mutations cause the disease remains to be determined. In addition to changes in peripheral nerves that are similar to those seen in neuropathies such as Charcot-Marie-Tooth type 2, GAN patients exhibit a wide range of central nervous system signs. These features, corroborated by degeneration of central tracts apparent from postmortem pathology, indicate that GAN is also a progressive neurodegenerative disease. To reflect this phenotype more precisely, we therefore propose that the disease should be more appropriately referred to as \"giant axonal neurodegeneration.\"'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_006',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 24}]}],\n",
       "   'context': 'A novel mutation in the GAN gene causes an intermediate form of giant axonal neuropathy in an Arab-Israeli family. Giant axonal neuropathy is a severe autosomal recessive neurodegenerative disorder of childhood that affects both the peripheral and central nervous systems. It is caused by mutations in the GAN gene linked to chromosome 16q24.1 At least 45 distinct disease-causing mutations have been identified throughout the gene in families of various ethnic origins, with different symptomatologies and different clinical courses. To date, no characteristic mutation or phenotype-genotype correlation has been established. We describe a novel missense mutation in four siblings born to consanguineous parents of Arab original with clinical and molecular features compatible with giant axonal neuropathy. The phenotype was characterized by a predominant motor and sensory peripheral neuropathies and severe skeletal deformities.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_007',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 882}]}],\n",
       "   'context': 'Clinical and genetic studies in a Chinese family with giant axonal neuropathy. The objective of the study was to investigate a girl with giant axonal neuropathy and detect the mutation of GAN gene in her family. The encoding exons of GAN gene were amplified from genomic DNA of the proband and her parents by polymerase chain reaction and directly sequenced after purification. The proband manifested typical neurological symptoms and pathological abnormalities. The case had 2 heterozygous missense mutations in GAN gene: 1. c. 224 T>A in exon 2, her mother was a heterozygote of this mutation and had normal phenotype; 2. c.1634G>A in exon 10, and her father was a heterozygote of this mutation and had normal phenotype. Both of the mutations caused amino acid changes in the gigaxonin protein. In this family, missense mutation of c.224 T>A and missense mutation of c.1634G>A in GAN gene caused the phenotype of giant axonal neuropathy in the proband. Her parents are heterozygotes of the disease without symptoms.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_008',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 205}]}],\n",
       "   'context': 'Giant axonal neuropathy caused by compound heterozygosity for a maternally inherited microdeletion and a paternal mutation within the GAN gene. Different missense, nonsense and frameshift mutations in the GAN gene encoding gigaxonin have been described to cause giant axonal neuropathy, a severe early-onset progressive neurological disease with autosomal recessive inheritance. By oligonucleotide array CGH analysis, we identified a 57-131 kb microdeletion affecting this gene in a patient with developmental delay, ataxia, areflexia, macrocephaly, and strikingly frizzy hair. The microdeletion was inherited from the mother and mutation analysis revealed a paternally inherited missense mutation c.1456G>A in exon 9 on the other allele. Our findings illustrate the power of higher resolution array CGH studies and highlight the importance of considering copy number variations in autosomal recessive diseases.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_009',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 212}]}],\n",
       "   'context': 'Heterogeneity of axonal pathology in Chinese patients with giant axonal neuropathy. INTRODUCTION: Giant axonal neuropathy (GAN) is a rare autosomal recessive neurodegenerative disorder caused by mutations in the GAN gene. Herein we report ultrastructural changes in Chinese patients with GAN. METHODS: General clinical assessment, sural nerve biopsy, and genetic analysis were performed. RESULTS: Sural biopsy revealed giant axons in 3 patients, 2 with a mild phenotype and 1 with a classical phenotype. Ultrastructurally, all patients had giant axons filled with closely packed neurofilaments. In addition, the classical patient had some axons containing irregular tubular-like structures. GAN mutation analysis revealed novel compound heterozygous c.98A>C and c.158C>T mutations in the BTB domain in 1 mild patient, a novel homozygous c.371T>G mutation in the BACK domain in another mild patient, and a novel c.1342G>T homozygous mutation in the Kelch domain in the classical patient. CONCLUSION: Closely packed neurofilaments in giant axons are common pathological changes in Chinese patients with GAN, whereas irregular tubular-like structures appear in the classical type of this neuropathy.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_010',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 506}]}],\n",
       "   'context': 'Giant axonal neuropathy diagnosed on skin biopsy. Evaluation of hereditary axonal neuropathy in childhood is complex. Often, the child has to be subjected to general anaesthesia for a nerve biopsy to guide further genetic testing, which may or may not be readily available. We describe a toddler with clinical features suggesting giant axonal neuropathy (GAN), whose diagnosis was confirmed by minimally invasive skin biopsy and corroborated by the finding of compound heterozygous mutations involving the GAN gene, including a novel interstitial microdeletion at 16q23.2 detected by microarray and a point mutation detected by direct sequencing.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_011',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 24}]}],\n",
       "   'context': 'A novel mutation in the GAN gene causes an intermediate form of giant axonal neuropathy in an Arab-Israeli family. Giant axonal neuropathy is a severe autosomal recessive neurodegenerative disorder of childhood that affects both the peripheral and central nervous systems. It is caused by mutations in the GAN gene linked to chromosome 16q24.1 At least 45 distinct disease-causing mutations have been identified throughout the gene in families of various ethnic origins, with different symptomatologies and different clinical courses. To date, no characteristic mutation or phenotype-genotype correlation has been established. We describe a novel missense mutation in four siblings born to consanguineous parents of Arab original with clinical and molecular features compatible with giant axonal neuropathy. The phenotype was characterized by a predominant motor and sensory peripheral neuropathies and severe skeletal deformities.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_012',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 186}]}],\n",
       "   'context': 'Explaining intermediate filament accumulation in giant axonal neuropathy. Giant axonal neuropathy (GAN)(1) is a rare autosomal recessive neurological disorder caused by mutations in the GAN gene that encodes gigaxonin, a member of the BTB/Kelch family of E3 ligase adaptor proteins.(1) This disease is characterized by the aggregation of Intermediate Filaments (IF)-cytoskeletal elements that play important roles in cell physiology including the regulation of cell shape, motility, mechanics and intra-cellular signaling. Although a range of cell types are affected in GAN, neurons display the most severe pathology, with neuronal intermediate filament accumulation and aggregation; this in turn causes axonal swellings or \"giant axons.\" A mechanistic understanding of GAN IF pathology has eluded researchers for many years. In a recent study(1) we demonstrate that the normal function of gigaxonin is to regulate the degradation of IF proteins via the proteasome. Our findings present the first direct link between GAN mutations and IF pathology; moreover, given the importance of IF aggregations in a wide range of disease conditions, our findings could have wider ramifications.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_013',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 205}]}],\n",
       "   'context': 'The absence of curly hair is associated with a milder phenotype in Giant Axonal Neuropathy. Giant Axonal Neuropathy is a pediatric neurodegenerative disorder caused by autosomal recessive mutations in the GAN gene on chromosome 16q24.1. Mutations in the GAN gene lead to functional impairment of the cytoskeletal protein gigaxonin and a generalized disorder of intermediate filaments, including neurofilaments in axons. Tightly curled hair is a common but not universal feature of Giant Axonal Neuropathy. The pathogenesis of curly hair is unknown, although disruption of keratin architecture is thought to play a role. As part of a broader natural history study of Giant Axonal Neuropathy, we found that the absence of curly hair is correlated with superior motor function (p=0.013) when controlling for age, as measured by the Gross Motor Function Measure. Theoretically, higher levels of functional gigaxonin protein or compensatory mechanisms could produce fewer abnormalities of neurofilaments and keratin, accounting for this phenotype. We suggest that straight-haired patients with Giant Axonal Neuropathy are potentially underdiagnosed due to their divergence from the classic phenotype of the disease. Due to their non-specific features of an axonal neuropathy, these patients may be misdiagnosed with Charcot-Marie-Tooth Disease type 2. Genetic testing for Giant Axonal Neuropathy should be considered in relevant cases of Charcot-Marie-Tooth Disease type 2.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_014',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 506}]}],\n",
       "   'context': 'Giant axonal neuropathy diagnosed on skin biopsy. Evaluation of hereditary axonal neuropathy in childhood is complex. Often, the child has to be subjected to general anaesthesia for a nerve biopsy to guide further genetic testing, which may or may not be readily available. We describe a toddler with clinical features suggesting giant axonal neuropathy (GAN), whose diagnosis was confirmed by minimally invasive skin biopsy and corroborated by the finding of compound heterozygous mutations involving the GAN gene, including a novel interstitial microdeletion at 16q23.2 detected by microarray and a point mutation detected by direct sequencing.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_015',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 212}]}],\n",
       "   'context': 'Heterogeneity of axonal pathology in Chinese patients with giant axonal neuropathy. INTRODUCTION: Giant axonal neuropathy (GAN) is a rare autosomal recessive neurodegenerative disorder caused by mutations in the GAN gene. Herein we report ultrastructural changes in Chinese patients with GAN. METHODS: General clinical assessment, sural nerve biopsy, and genetic analysis were performed. RESULTS: Sural biopsy revealed giant axons in 3 patients, 2 with a mild phenotype and 1 with a classical phenotype. Ultrastructurally, all patients had giant axons filled with closely packed neurofilaments. In addition, the classical patient had some axons containing irregular tubular-like structures. GAN mutation analysis revealed novel compound heterozygous c.98A>C and c.158C>T mutations in the BTB domain in 1 mild patient, a novel homozygous c.371T>G mutation in the BACK domain in another mild patient, and a novel c.1342G>T homozygous mutation in the Kelch domain in the classical patient. CONCLUSION: Closely packed neurofilaments in giant axons are common pathological changes in Chinese patients with GAN, whereas irregular tubular-like structures appear in the classical type of this neuropathy.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_016',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 24}]}],\n",
       "   'context': 'A novel mutation in the GAN gene causes an intermediate form of giant axonal neuropathy in an Arab-Israeli family. Giant axonal neuropathy is a severe autosomal recessive neurodegenerative disorder of childhood that affects both the peripheral and central nervous systems. It is caused by mutations in the GAN gene linked to chromosome 16q24.1 At least 45 distinct disease-causing mutations have been identified throughout the gene in families of various ethnic origins, with different symptomatologies and different clinical courses. To date, no characteristic mutation or phenotype-genotype correlation has been established. We describe a novel missense mutation in four siblings born to consanguineous parents of Arab original with clinical and molecular features compatible with giant axonal neuropathy. The phenotype was characterized by a predominant motor and sensory peripheral neuropathies and severe skeletal deformities.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_017',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 205}]}],\n",
       "   'context': 'The absence of curly hair is associated with a milder phenotype in Giant Axonal Neuropathy. Giant Axonal Neuropathy is a pediatric neurodegenerative disorder caused by autosomal recessive mutations in the GAN gene on chromosome 16q24.1. Mutations in the GAN gene lead to functional impairment of the cytoskeletal protein gigaxonin and a generalized disorder of intermediate filaments, including neurofilaments in axons. Tightly curled hair is a common but not universal feature of Giant Axonal Neuropathy. The pathogenesis of curly hair is unknown, although disruption of keratin architecture is thought to play a role. As part of a broader natural history study of Giant Axonal Neuropathy, we found that the absence of curly hair is correlated with superior motor function (p=0.013) when controlling for age, as measured by the Gross Motor Function Measure. Theoretically, higher levels of functional gigaxonin protein or compensatory mechanisms could produce fewer abnormalities of neurofilaments and keratin, accounting for this phenotype. We suggest that straight-haired patients with Giant Axonal Neuropathy are potentially underdiagnosed due to their divergence from the classic phenotype of the disease. Due to their non-specific features of an axonal neuropathy, these patients may be misdiagnosed with Charcot-Marie-Tooth Disease type 2. Genetic testing for Giant Axonal Neuropathy should be considered in relevant cases of Charcot-Marie-Tooth Disease type 2.'},\n",
       "  {'qas': [{'id': '572096c90fd6f91b6800000e_018',\n",
       "     'question': 'Which gene is involved in Giant Axonal Neuropathy?',\n",
       "     'answers': [{'text': 'GAN gene', 'answer_start': 212}]}],\n",
       "   'context': 'Heterogeneity of axonal pathology in Chinese patients with giant axonal neuropathy. INTRODUCTION: Giant axonal neuropathy (GAN) is a rare autosomal recessive neurodegenerative disorder caused by mutations in the GAN gene. Herein we report ultrastructural changes in Chinese patients with GAN. METHODS: General clinical assessment, sural nerve biopsy, and genetic analysis were performed. RESULTS: Sural biopsy revealed giant axons in 3 patients, 2 with a mild phenotype and 1 with a classical phenotype. Ultrastructurally, all patients had giant axons filled with closely packed neurofilaments. In addition, the classical patient had some axons containing irregular tubular-like structures. GAN mutation analysis revealed novel compound heterozygous c.98A>C and c.158C>T mutations in the BTB domain in 1 mild patient, a novel homozygous c.371T>G mutation in the BACK domain in another mild patient, and a novel c.1342G>T homozygous mutation in the Kelch domain in the classical patient. CONCLUSION: Closely packed neurofilaments in giant axons are common pathological changes in Chinese patients with GAN, whereas irregular tubular-like structures appear in the classical type of this neuropathy.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_001',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 137}]}],\n",
       "   'context': 'Specific isoforms of p73 control the induction of cell death induced by the viral proteins, E1A or apoptin. A member of the p53 family, p73, has several isoforms and differentially regulates transcription of genes involved in the control of the cell cycle and apoptosis. We have previously shown efficient and p53-independent, tumor-specific cell death induced by the viral proteins E1A and Apoptin. Here, we demonstrate that the induction of apoptosis by these viral proteins involves activation of TAp73. Both E1A and Apoptin induced expression of endogenous TAp73 and the p53/p73 BH3-only pro-apoptotic target, PUMA, independently of the p53 function. Furthermore, exogenous expression of TAp73 isoforms, particularly TAp73beta, sensitized cells to killing by both E1A and Apoptin, while expression of DeltaNp73alpha blocked this activity. Besides, knockout of the p73 regulator, c-Abl, attenuated E1A-induced apoptosis. In accordance with the role of p73 in apoptosis induced by these viral proteins, overexpression of TAp73beta strongly induced apoptosis in p53-deficient cancer cells in vitro and in HNSCC xenografts. Using a doxycycline-inducible system, we provide evidence for target selectivity and significant differences in protein stability for specific p73 isoforms, suggesting a diverse and pivotal role for p73 in response to various genotoxic agents. Collectively, our data show that in the absence of the p53 function, viral proteins E1A and Apoptin utilize the p73 pathway to induce efficient tumor cell death.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_002',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 125}]}],\n",
       "   'context': 'The p73 tumor suppressor is targeted by Pirh2 RING finger E3 ubiquitin ligase for the proteasome-dependent degradation. The p73 gene, a homologue of the p53 tumor suppressor, is expressed as TA and ΔN isoforms. TAp73 has similar activity as p53 and functions as a tumor suppressor whereas ΔNp73 has both pro- and anti-survival functions. While p73 is rarely mutated in spontaneous tumors, the expression status of p73 is linked to the sensitivity of tumor cells to chemotherapy and prognosis for many types of human cancer. Thus, uncovering its regulators in tumors is of great interest. Here, we found that Pirh2, a RING finger E3 ubiquitin ligase, promotes the proteasome-dependent degradation of p73. Specifically, we showed that knockdown of Pirh2 up-regulates, whereas ectopic expression of Pirh2 down-regulates, expression of endogenous and exogenous p73. In addition, Pirh2 physically associates with and promotes TAp73 polyubiquitination both in vivo and in vitro. Moreover, we found that p73 can be degraded by both 20 S and 26 S proteasomes. Finally, we showed that Pirh2 knockdown leads to growth suppression in a TAp73-dependent manner. Taken together, our findings indicate that Pirh2 promotes the proteasomal turnover of TAp73, and thus targeting Pirh2 to restore TAp73-mediated growth suppression in p53-deficient tumors may be developed as a novel anti-cancer strategy.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_003',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 214}]}],\n",
       "   'context': 'The p73 tumor suppressor is targeted by Pirh2 RING finger E3 ubiquitin ligase for the proteasome-dependent degradation. The p73 gene, a homologue of the p53 tumor suppressor, is expressed as TA and ΔN isoforms. TAp73 has similar activity as p53 and functions as a tumor suppressor whereas ΔNp73 has both pro- and anti-survival functions. While p73 is rarely mutated in spontaneous tumors, the expression status of p73 is linked to the sensitivity of tumor cells to chemotherapy and prognosis for many types of human cancer. Thus, uncovering its regulators in tumors is of great interest. Here, we found that Pirh2, a RING finger E3 ubiquitin ligase, promotes the proteasome-dependent degradation of p73. Specifically, we showed that knockdown of Pirh2 up-regulates, whereas ectopic expression of Pirh2 down-regulates, expression of endogenous and exogenous p73. In addition, Pirh2 physically associates with and promotes TAp73 polyubiquitination both in vivo and in vitro. Moreover, we found that p73 can be degraded by both 20 S and 26 S proteasomes. Finally, we showed that Pirh2 knockdown leads to growth suppression in a TAp73-dependent manner. Taken together, our findings indicate that Pirh2 promotes the proteasomal turnover of TAp73, and thus targeting Pirh2 to restore TAp73-mediated growth suppression in p53-deficient tumors may be developed as a novel anti-cancer strategy.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_004',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 292}]}],\n",
       "   'context': 'The p73 tumor suppressor is targeted by Pirh2 RING finger E3 ubiquitin ligase for the proteasome-dependent degradation. The p73 gene, a homologue of the p53 tumor suppressor, is expressed as TA and ΔN isoforms. TAp73 has similar activity as p53 and functions as a tumor suppressor whereas ΔNp73 has both pro- and anti-survival functions. While p73 is rarely mutated in spontaneous tumors, the expression status of p73 is linked to the sensitivity of tumor cells to chemotherapy and prognosis for many types of human cancer. Thus, uncovering its regulators in tumors is of great interest. Here, we found that Pirh2, a RING finger E3 ubiquitin ligase, promotes the proteasome-dependent degradation of p73. Specifically, we showed that knockdown of Pirh2 up-regulates, whereas ectopic expression of Pirh2 down-regulates, expression of endogenous and exogenous p73. In addition, Pirh2 physically associates with and promotes TAp73 polyubiquitination both in vivo and in vitro. Moreover, we found that p73 can be degraded by both 20 S and 26 S proteasomes. Finally, we showed that Pirh2 knockdown leads to growth suppression in a TAp73-dependent manner. Taken together, our findings indicate that Pirh2 promotes the proteasomal turnover of TAp73, and thus targeting Pirh2 to restore TAp73-mediated growth suppression in p53-deficient tumors may be developed as a novel anti-cancer strategy.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_005',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 258}]}],\n",
       "   'context': 'Differential control of TAp73 and DeltaNp73 protein stability by the ring finger ubiquitin ligase PIR2. p73 is a p53-related transcription factor with fundamental roles in development and tumor suppression. Transcription from two different promoters on the p73 gene results in generation of transcriptionally active TAp73 isoforms and dominant negative DeltaNp73 isoforms with opposing pro- and anti-apoptotic functions. Therefore, the relative ratio of each isoform is an important determinant of the cell fate. Proteasomal degradation of p73 is mediated by polyubiquitination-dependent and -independent processes both of which appear, thus far, to lack selectivity for the TAp73 and DeltaNp73 isoforms. Here, we describe the characterization of another transcriptional target of TAp73; a ring finger domain ubiquitin ligase p73 Induced RING 2 protein (PIR2). Although PIR2 was initially identified a p53-induced gene (p53RFP), low abundance of PIR2 transcript in mouse embryonic fibroblasts of TAp73 KO mice compared with WT mice and comparison of PIR2 mRNA and protein levels following TAp73 or p53 overexpression substantiate TAp73 isoforms as strong inducers of PIR2. Although PIR2 expression was induced by DNA damage, its expression did not alter apoptotic response or cell cycle profile per se. However, coexpression of PIR2 with TAp73 or DeltaNp73 resulted in an increase of the TA/DeltaNp73 ratio, due to preferential degradation of DeltaNp73. Finally, PIR2 was able to relieve the inhibitory effect of DeltaNp73 on TAp73 induced apoptosis following DNA damage. These results suggest that PIR2, by being induced by TAp73 and degrading DeltaNp73, differentially regulates TAp73/DeltaNp73 stability, and, hence, it may offer a therapeutic approach to enhance the chemosensitivity of tumor cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_006',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 319}]}],\n",
       "   'context': 'Differential control of TAp73 and DeltaNp73 protein stability by the ring finger ubiquitin ligase PIR2. p73 is a p53-related transcription factor with fundamental roles in development and tumor suppression. Transcription from two different promoters on the p73 gene results in generation of transcriptionally active TAp73 isoforms and dominant negative DeltaNp73 isoforms with opposing pro- and anti-apoptotic functions. Therefore, the relative ratio of each isoform is an important determinant of the cell fate. Proteasomal degradation of p73 is mediated by polyubiquitination-dependent and -independent processes both of which appear, thus far, to lack selectivity for the TAp73 and DeltaNp73 isoforms. Here, we describe the characterization of another transcriptional target of TAp73; a ring finger domain ubiquitin ligase p73 Induced RING 2 protein (PIR2). Although PIR2 was initially identified a p53-induced gene (p53RFP), low abundance of PIR2 transcript in mouse embryonic fibroblasts of TAp73 KO mice compared with WT mice and comparison of PIR2 mRNA and protein levels following TAp73 or p53 overexpression substantiate TAp73 isoforms as strong inducers of PIR2. Although PIR2 expression was induced by DNA damage, its expression did not alter apoptotic response or cell cycle profile per se. However, coexpression of PIR2 with TAp73 or DeltaNp73 resulted in an increase of the TA/DeltaNp73 ratio, due to preferential degradation of DeltaNp73. Finally, PIR2 was able to relieve the inhibitory effect of DeltaNp73 on TAp73 induced apoptosis following DNA damage. These results suggest that PIR2, by being induced by TAp73 and degrading DeltaNp73, differentially regulates TAp73/DeltaNp73 stability, and, hence, it may offer a therapeutic approach to enhance the chemosensitivity of tumor cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_007',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 360}]}],\n",
       "   'context': 'Differential control of TAp73 and DeltaNp73 protein stability by the ring finger ubiquitin ligase PIR2. p73 is a p53-related transcription factor with fundamental roles in development and tumor suppression. Transcription from two different promoters on the p73 gene results in generation of transcriptionally active TAp73 isoforms and dominant negative DeltaNp73 isoforms with opposing pro- and anti-apoptotic functions. Therefore, the relative ratio of each isoform is an important determinant of the cell fate. Proteasomal degradation of p73 is mediated by polyubiquitination-dependent and -independent processes both of which appear, thus far, to lack selectivity for the TAp73 and DeltaNp73 isoforms. Here, we describe the characterization of another transcriptional target of TAp73; a ring finger domain ubiquitin ligase p73 Induced RING 2 protein (PIR2). Although PIR2 was initially identified a p53-induced gene (p53RFP), low abundance of PIR2 transcript in mouse embryonic fibroblasts of TAp73 KO mice compared with WT mice and comparison of PIR2 mRNA and protein levels following TAp73 or p53 overexpression substantiate TAp73 isoforms as strong inducers of PIR2. Although PIR2 expression was induced by DNA damage, its expression did not alter apoptotic response or cell cycle profile per se. However, coexpression of PIR2 with TAp73 or DeltaNp73 resulted in an increase of the TA/DeltaNp73 ratio, due to preferential degradation of DeltaNp73. Finally, PIR2 was able to relieve the inhibitory effect of DeltaNp73 on TAp73 induced apoptosis following DNA damage. These results suggest that PIR2, by being induced by TAp73 and degrading DeltaNp73, differentially regulates TAp73/DeltaNp73 stability, and, hence, it may offer a therapeutic approach to enhance the chemosensitivity of tumor cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_008',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 334}]}],\n",
       "   'context': \"Different intracellular compartmentalization of TA and DeltaNp73 in non-small cell lung cancer. The p53 homologue p73 is overexpressed in many tumors, including lung cancer. We have evaluated the differential expression and subcellular localization of the functionally distinct apoptotic (TA) and anti-apoptotic (DeltaN) isoforms of p73 in non-small cell lung cancer (NSCLC), their possible association with p53 expression and determined the methylation status of the two p73 gene promoters (P1 and P2) in this tumor type. Immunohistochemical analysis showed that both isoforms are expressed in the majority of cases. However, the oncogenic DeltaN variant, derived from the transcripts DeltaN'p73 (from P1) and/or DeltaNp73 (from P2), is localized mainly in the nucleus, while the anti-oncogenic TAp73 isoform (derived from a P1 transcript) is sequestered in the cytoplasm in almost all cases analyzed. Significant correlation was found between p53 and DeltaNp73 expression (p=0.041). Methylation analysis conducted on 41 tumor samples showed that the P1 promoter is almost invariably unmethylated (39/41 cases) whereas P2 was found completely methylated in 17 cases and partially or totally unmethylated in 24 samples. No correlation was found between the methylation status of P1 and P2 and p73 expression. Our results demonstrate that both isoforms contribute to p73 overexpression in NSCLC and suggest that their different intracellular localization may reflect an alteration of the functional p53-p73 network that might contribute to lung cancer development.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_009',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 473}]}],\n",
       "   'context': \"Different intracellular compartmentalization of TA and DeltaNp73 in non-small cell lung cancer. The p53 homologue p73 is overexpressed in many tumors, including lung cancer. We have evaluated the differential expression and subcellular localization of the functionally distinct apoptotic (TA) and anti-apoptotic (DeltaN) isoforms of p73 in non-small cell lung cancer (NSCLC), their possible association with p53 expression and determined the methylation status of the two p73 gene promoters (P1 and P2) in this tumor type. Immunohistochemical analysis showed that both isoforms are expressed in the majority of cases. However, the oncogenic DeltaN variant, derived from the transcripts DeltaN'p73 (from P1) and/or DeltaNp73 (from P2), is localized mainly in the nucleus, while the anti-oncogenic TAp73 isoform (derived from a P1 transcript) is sequestered in the cytoplasm in almost all cases analyzed. Significant correlation was found between p53 and DeltaNp73 expression (p=0.041). Methylation analysis conducted on 41 tumor samples showed that the P1 promoter is almost invariably unmethylated (39/41 cases) whereas P2 was found completely methylated in 17 cases and partially or totally unmethylated in 24 samples. No correlation was found between the methylation status of P1 and P2 and p73 expression. Our results demonstrate that both isoforms contribute to p73 overexpression in NSCLC and suggest that their different intracellular localization may reflect an alteration of the functional p53-p73 network that might contribute to lung cancer development.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_010',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 99}]}],\n",
       "   'context': 'TAp73 regulates the spindle assembly checkpoint by modulating BubR1 activity. The role of various p73 isoforms in tumorigenesis has been controversial. However, as we have recently shown, the generation of TAp73-deficient (TAp73(-/-)) mice reveals that TAp73 isoforms exert tumor-suppressive functions, indicating an emerging role for Trp-73 in the maintenance of genomic stability. Unlike mice lacking all p73 isoforms, TAp73(-/-) mice show a high incidence of spontaneous tumors. Moreover, TAp73(-/-) mice are infertile and produce oocytes exhibiting spindle abnormalities. These data suggest a link between TAp73 activities and the common molecular machinery underlying meiosis and mitosis. Previous studies have indicated that the spindle assembly checkpoint (SAC) complex, whose activation leads to mitotic arrest, also regulates meiosis. In this study, we demonstrate in murine and human cells that TAp73 is able to interact directly with several partners of the SAC complex (Bub1, Bub3, and BubR1). We also show that TAp73 is involved in SAC protein localization and activities. Moreover, we show that decreased TAp73 expression correlates with increases of SAC protein expression in patients with lung cancer. Our results establish TAp73 as a regulator of SAC responses and indicate that TAp73 loss can lead to mitotic arrest defects. Our data suggest that SAC impairment in the absence of functional TAp73 could explain the genomic instability and increased aneuploidy observed in TAp73-deficient cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_011',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 209}]}],\n",
       "   'context': 'TAp73 regulates the spindle assembly checkpoint by modulating BubR1 activity. The role of various p73 isoforms in tumorigenesis has been controversial. However, as we have recently shown, the generation of TAp73-deficient (TAp73(-/-)) mice reveals that TAp73 isoforms exert tumor-suppressive functions, indicating an emerging role for Trp-73 in the maintenance of genomic stability. Unlike mice lacking all p73 isoforms, TAp73(-/-) mice show a high incidence of spontaneous tumors. Moreover, TAp73(-/-) mice are infertile and produce oocytes exhibiting spindle abnormalities. These data suggest a link between TAp73 activities and the common molecular machinery underlying meiosis and mitosis. Previous studies have indicated that the spindle assembly checkpoint (SAC) complex, whose activation leads to mitotic arrest, also regulates meiosis. In this study, we demonstrate in murine and human cells that TAp73 is able to interact directly with several partners of the SAC complex (Bub1, Bub3, and BubR1). We also show that TAp73 is involved in SAC protein localization and activities. Moreover, we show that decreased TAp73 expression correlates with increases of SAC protein expression in patients with lung cancer. Our results establish TAp73 as a regulator of SAC responses and indicate that TAp73 loss can lead to mitotic arrest defects. Our data suggest that SAC impairment in the absence of functional TAp73 could explain the genomic instability and increased aneuploidy observed in TAp73-deficient cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_012',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 226}]}],\n",
       "   'context': 'TAp73 regulates the spindle assembly checkpoint by modulating BubR1 activity. The role of various p73 isoforms in tumorigenesis has been controversial. However, as we have recently shown, the generation of TAp73-deficient (TAp73(-/-)) mice reveals that TAp73 isoforms exert tumor-suppressive functions, indicating an emerging role for Trp-73 in the maintenance of genomic stability. Unlike mice lacking all p73 isoforms, TAp73(-/-) mice show a high incidence of spontaneous tumors. Moreover, TAp73(-/-) mice are infertile and produce oocytes exhibiting spindle abnormalities. These data suggest a link between TAp73 activities and the common molecular machinery underlying meiosis and mitosis. Previous studies have indicated that the spindle assembly checkpoint (SAC) complex, whose activation leads to mitotic arrest, also regulates meiosis. In this study, we demonstrate in murine and human cells that TAp73 is able to interact directly with several partners of the SAC complex (Bub1, Bub3, and BubR1). We also show that TAp73 is involved in SAC protein localization and activities. Moreover, we show that decreased TAp73 expression correlates with increases of SAC protein expression in patients with lung cancer. Our results establish TAp73 as a regulator of SAC responses and indicate that TAp73 loss can lead to mitotic arrest defects. Our data suggest that SAC impairment in the absence of functional TAp73 could explain the genomic instability and increased aneuploidy observed in TAp73-deficient cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_013',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 256}]}],\n",
       "   'context': 'TAp73 regulates the spindle assembly checkpoint by modulating BubR1 activity. The role of various p73 isoforms in tumorigenesis has been controversial. However, as we have recently shown, the generation of TAp73-deficient (TAp73(-/-)) mice reveals that TAp73 isoforms exert tumor-suppressive functions, indicating an emerging role for Trp-73 in the maintenance of genomic stability. Unlike mice lacking all p73 isoforms, TAp73(-/-) mice show a high incidence of spontaneous tumors. Moreover, TAp73(-/-) mice are infertile and produce oocytes exhibiting spindle abnormalities. These data suggest a link between TAp73 activities and the common molecular machinery underlying meiosis and mitosis. Previous studies have indicated that the spindle assembly checkpoint (SAC) complex, whose activation leads to mitotic arrest, also regulates meiosis. In this study, we demonstrate in murine and human cells that TAp73 is able to interact directly with several partners of the SAC complex (Bub1, Bub3, and BubR1). We also show that TAp73 is involved in SAC protein localization and activities. Moreover, we show that decreased TAp73 expression correlates with increases of SAC protein expression in patients with lung cancer. Our results establish TAp73 as a regulator of SAC responses and indicate that TAp73 loss can lead to mitotic arrest defects. Our data suggest that SAC impairment in the absence of functional TAp73 could explain the genomic instability and increased aneuploidy observed in TAp73-deficient cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_014',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 339}]}],\n",
       "   'context': 'TAp73 regulates the spindle assembly checkpoint by modulating BubR1 activity. The role of various p73 isoforms in tumorigenesis has been controversial. However, as we have recently shown, the generation of TAp73-deficient (TAp73(-/-)) mice reveals that TAp73 isoforms exert tumor-suppressive functions, indicating an emerging role for Trp-73 in the maintenance of genomic stability. Unlike mice lacking all p73 isoforms, TAp73(-/-) mice show a high incidence of spontaneous tumors. Moreover, TAp73(-/-) mice are infertile and produce oocytes exhibiting spindle abnormalities. These data suggest a link between TAp73 activities and the common molecular machinery underlying meiosis and mitosis. Previous studies have indicated that the spindle assembly checkpoint (SAC) complex, whose activation leads to mitotic arrest, also regulates meiosis. In this study, we demonstrate in murine and human cells that TAp73 is able to interact directly with several partners of the SAC complex (Bub1, Bub3, and BubR1). We also show that TAp73 is involved in SAC protein localization and activities. Moreover, we show that decreased TAp73 expression correlates with increases of SAC protein expression in patients with lung cancer. Our results establish TAp73 as a regulator of SAC responses and indicate that TAp73 loss can lead to mitotic arrest defects. Our data suggest that SAC impairment in the absence of functional TAp73 could explain the genomic instability and increased aneuploidy observed in TAp73-deficient cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_015',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 408}]}],\n",
       "   'context': 'TAp73 regulates the spindle assembly checkpoint by modulating BubR1 activity. The role of various p73 isoforms in tumorigenesis has been controversial. However, as we have recently shown, the generation of TAp73-deficient (TAp73(-/-)) mice reveals that TAp73 isoforms exert tumor-suppressive functions, indicating an emerging role for Trp-73 in the maintenance of genomic stability. Unlike mice lacking all p73 isoforms, TAp73(-/-) mice show a high incidence of spontaneous tumors. Moreover, TAp73(-/-) mice are infertile and produce oocytes exhibiting spindle abnormalities. These data suggest a link between TAp73 activities and the common molecular machinery underlying meiosis and mitosis. Previous studies have indicated that the spindle assembly checkpoint (SAC) complex, whose activation leads to mitotic arrest, also regulates meiosis. In this study, we demonstrate in murine and human cells that TAp73 is able to interact directly with several partners of the SAC complex (Bub1, Bub3, and BubR1). We also show that TAp73 is involved in SAC protein localization and activities. Moreover, we show that decreased TAp73 expression correlates with increases of SAC protein expression in patients with lung cancer. Our results establish TAp73 as a regulator of SAC responses and indicate that TAp73 loss can lead to mitotic arrest defects. Our data suggest that SAC impairment in the absence of functional TAp73 could explain the genomic instability and increased aneuploidy observed in TAp73-deficient cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_016',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 424}]}],\n",
       "   'context': 'TAp73 regulates the spindle assembly checkpoint by modulating BubR1 activity. The role of various p73 isoforms in tumorigenesis has been controversial. However, as we have recently shown, the generation of TAp73-deficient (TAp73(-/-)) mice reveals that TAp73 isoforms exert tumor-suppressive functions, indicating an emerging role for Trp-73 in the maintenance of genomic stability. Unlike mice lacking all p73 isoforms, TAp73(-/-) mice show a high incidence of spontaneous tumors. Moreover, TAp73(-/-) mice are infertile and produce oocytes exhibiting spindle abnormalities. These data suggest a link between TAp73 activities and the common molecular machinery underlying meiosis and mitosis. Previous studies have indicated that the spindle assembly checkpoint (SAC) complex, whose activation leads to mitotic arrest, also regulates meiosis. In this study, we demonstrate in murine and human cells that TAp73 is able to interact directly with several partners of the SAC complex (Bub1, Bub3, and BubR1). We also show that TAp73 is involved in SAC protein localization and activities. Moreover, we show that decreased TAp73 expression correlates with increases of SAC protein expression in patients with lung cancer. Our results establish TAp73 as a regulator of SAC responses and indicate that TAp73 loss can lead to mitotic arrest defects. Our data suggest that SAC impairment in the absence of functional TAp73 could explain the genomic instability and increased aneuploidy observed in TAp73-deficient cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_017',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 384}]}],\n",
       "   'context': 'Mdm2-mediated NEDD8 modification of TAp73 regulates its transactivation function. Mutations in p73 are rare in cancer. Emerging evidence suggests that the relative expression of various p73 isoforms may contribute to tumorigenesis. Alternative promoters and N-terminal splicing result in the transcription and processing of either full-length (TA) or N-terminally truncated (deltaN) p73 isoforms. TAp73 possesses pro-apoptotic functions, while deltaNp73 has anti-apoptotic properties via functional inhibition of TAp73 and p53. Here, we report that TAp73, but not deltaNp73, is covalently modified by NEDD8 under physiologic conditions in an Mdm2-dependent manner. Co-expression of NEDP1, a cysteine protease that specifically cleaves NEDD8 conjugates, was shown to deneddylate TAp73. In addition, blockage of the endogenous NEDD8 pathway increased TAp73-mediated transactivation of p53- and p73-responsive promoter-driven reporter activity, and in conjunction, neddylated TAp73 species were found preferentially in the cytoplasm. These results suggest that Mdm2 attenuates TAp73 transactivation function, at least in part, by promoting NEDD8-dependent TAp73 cytoplasmic localization and provide the first evidence of a covalent post-translational modification exclusively targeting the TA isoforms of p73.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_018',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 400}]}],\n",
       "   'context': 'Mdm2-mediated NEDD8 modification of TAp73 regulates its transactivation function. Mutations in p73 are rare in cancer. Emerging evidence suggests that the relative expression of various p73 isoforms may contribute to tumorigenesis. Alternative promoters and N-terminal splicing result in the transcription and processing of either full-length (TA) or N-terminally truncated (deltaN) p73 isoforms. TAp73 possesses pro-apoptotic functions, while deltaNp73 has anti-apoptotic properties via functional inhibition of TAp73 and p53. Here, we report that TAp73, but not deltaNp73, is covalently modified by NEDD8 under physiologic conditions in an Mdm2-dependent manner. Co-expression of NEDP1, a cysteine protease that specifically cleaves NEDD8 conjugates, was shown to deneddylate TAp73. In addition, blockage of the endogenous NEDD8 pathway increased TAp73-mediated transactivation of p53- and p73-responsive promoter-driven reporter activity, and in conjunction, neddylated TAp73 species were found preferentially in the cytoplasm. These results suggest that Mdm2 attenuates TAp73 transactivation function, at least in part, by promoting NEDD8-dependent TAp73 cytoplasmic localization and provide the first evidence of a covalent post-translational modification exclusively targeting the TA isoforms of p73.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_019',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 451}]}],\n",
       "   'context': 'Mdm2-mediated NEDD8 modification of TAp73 regulates its transactivation function. Mutations in p73 are rare in cancer. Emerging evidence suggests that the relative expression of various p73 isoforms may contribute to tumorigenesis. Alternative promoters and N-terminal splicing result in the transcription and processing of either full-length (TA) or N-terminally truncated (deltaN) p73 isoforms. TAp73 possesses pro-apoptotic functions, while deltaNp73 has anti-apoptotic properties via functional inhibition of TAp73 and p53. Here, we report that TAp73, but not deltaNp73, is covalently modified by NEDD8 under physiologic conditions in an Mdm2-dependent manner. Co-expression of NEDP1, a cysteine protease that specifically cleaves NEDD8 conjugates, was shown to deneddylate TAp73. In addition, blockage of the endogenous NEDD8 pathway increased TAp73-mediated transactivation of p53- and p73-responsive promoter-driven reporter activity, and in conjunction, neddylated TAp73 species were found preferentially in the cytoplasm. These results suggest that Mdm2 attenuates TAp73 transactivation function, at least in part, by promoting NEDD8-dependent TAp73 cytoplasmic localization and provide the first evidence of a covalent post-translational modification exclusively targeting the TA isoforms of p73.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_020',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 516}]}],\n",
       "   'context': 'Mdm2-mediated NEDD8 modification of TAp73 regulates its transactivation function. Mutations in p73 are rare in cancer. Emerging evidence suggests that the relative expression of various p73 isoforms may contribute to tumorigenesis. Alternative promoters and N-terminal splicing result in the transcription and processing of either full-length (TA) or N-terminally truncated (deltaN) p73 isoforms. TAp73 possesses pro-apoptotic functions, while deltaNp73 has anti-apoptotic properties via functional inhibition of TAp73 and p53. Here, we report that TAp73, but not deltaNp73, is covalently modified by NEDD8 under physiologic conditions in an Mdm2-dependent manner. Co-expression of NEDP1, a cysteine protease that specifically cleaves NEDD8 conjugates, was shown to deneddylate TAp73. In addition, blockage of the endogenous NEDD8 pathway increased TAp73-mediated transactivation of p53- and p73-responsive promoter-driven reporter activity, and in conjunction, neddylated TAp73 species were found preferentially in the cytoplasm. These results suggest that Mdm2 attenuates TAp73 transactivation function, at least in part, by promoting NEDD8-dependent TAp73 cytoplasmic localization and provide the first evidence of a covalent post-translational modification exclusively targeting the TA isoforms of p73.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_021',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 61}]}],\n",
       "   'context': 'Expression of TAP73 and DeltaNP73 in malignant gliomas. The p73 gene is able to encode transcriptionaly active TAp73, as well as a dominant-negatively acting DeltaNp73 transcript isoforms. We studied differential expression of these forms in normal brain as well as glial tumors, by semiquantitative RT-PCR. The expression of p73 was low or undetectable in normal brain tissues. Most of the tumors and non-tumor brain tissues also lacked significant expression of p73 in patients with low-grade astrocytomas. In contrast, most high-grade glial tumors displayed strong up-regulation of TAp73, whereas only a few displayed DeltaNp73 expression. These aberrations may reflect the inactivation of retinoblastoma pathway in these tumors which result in the activation of E2F transcription factors, since TAp73 is a known target of E2F1 gene. The study of TAp73 expression in brain tumors may serve as a means to evaluate the retinoblastoma pathway-dependent tumor progression.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_022',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 114}]}],\n",
       "   'context': 'Expression of TAP73 and DeltaNP73 in malignant gliomas. The p73 gene is able to encode transcriptionaly active TAp73, as well as a dominant-negatively acting DeltaNp73 transcript isoforms. We studied differential expression of these forms in normal brain as well as glial tumors, by semiquantitative RT-PCR. The expression of p73 was low or undetectable in normal brain tissues. Most of the tumors and non-tumor brain tissues also lacked significant expression of p73 in patients with low-grade astrocytomas. In contrast, most high-grade glial tumors displayed strong up-regulation of TAp73, whereas only a few displayed DeltaNp73 expression. These aberrations may reflect the inactivation of retinoblastoma pathway in these tumors which result in the activation of E2F transcription factors, since TAp73 is a known target of E2F1 gene. The study of TAp73 expression in brain tumors may serve as a means to evaluate the retinoblastoma pathway-dependent tumor progression.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_023',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 165}]}],\n",
       "   'context': 'Expression of TAP73 and DeltaNP73 in malignant gliomas. The p73 gene is able to encode transcriptionaly active TAp73, as well as a dominant-negatively acting DeltaNp73 transcript isoforms. We studied differential expression of these forms in normal brain as well as glial tumors, by semiquantitative RT-PCR. The expression of p73 was low or undetectable in normal brain tissues. Most of the tumors and non-tumor brain tissues also lacked significant expression of p73 in patients with low-grade astrocytomas. In contrast, most high-grade glial tumors displayed strong up-regulation of TAp73, whereas only a few displayed DeltaNp73 expression. These aberrations may reflect the inactivation of retinoblastoma pathway in these tumors which result in the activation of E2F transcription factors, since TAp73 is a known target of E2F1 gene. The study of TAp73 expression in brain tumors may serve as a means to evaluate the retinoblastoma pathway-dependent tumor progression.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_024',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 275}]}],\n",
       "   'context': 'Differential effects of diverse p53 isoforms on TAp73 transcriptional activity and apoptosis. The p53 activities are due, at least in part, to its ability to form oligomers that bind to specific DNA sequences and activate transcription. Since some mutant p53 proteins and ΔNp73 isoforms form heterocomplexes with TAp73, we asked whether p53 isoforms can do the same and potentially act as dominant-negative inhibitors of TAp73. Moreover, it has already been found that some isoforms form complex with wtp53 and some of them inhibit p53 tumor-suppressor functions. Therefore, we studied the complex formation and co-immunoprecipitation assays show that all six p53 isoforms examined can form complexes with TAp73β, whereas only Δ133p53α/β/γ isoforms form complex with TAp73α. All p53 isoforms counteract TAp73β transactivation function but with different efficiency and in a promoter-dependent manner. Furthermore, apoptotic activity of TAp73β was augmented by coexpression of p53β, whereas Δ133p53α and β inhibit its apoptotic activity most efficiently. We have determined the half-life of different p53 isoforms: p53γ isoform has the shortest half-life, whereas Δ133p53γ has the longest half-life. Inhibitory interactions of two proteins in complex often lead to their stabilization. However, only three isoforms (Δ133p53α, Δ133p53β and Δ40p53α) stabilize TAp73β. We are convinced that defining the interactions between p53/p73 would give a new insight into how the p53 isoforms modulate the p73 functions in tumorigenesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_025',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 316}]}],\n",
       "   'context': 'Differential effects of diverse p53 isoforms on TAp73 transcriptional activity and apoptosis. The p53 activities are due, at least in part, to its ability to form oligomers that bind to specific DNA sequences and activate transcription. Since some mutant p53 proteins and ΔNp73 isoforms form heterocomplexes with TAp73, we asked whether p53 isoforms can do the same and potentially act as dominant-negative inhibitors of TAp73. Moreover, it has already been found that some isoforms form complex with wtp53 and some of them inhibit p53 tumor-suppressor functions. Therefore, we studied the complex formation and co-immunoprecipitation assays show that all six p53 isoforms examined can form complexes with TAp73β, whereas only Δ133p53α/β/γ isoforms form complex with TAp73α. All p53 isoforms counteract TAp73β transactivation function but with different efficiency and in a promoter-dependent manner. Furthermore, apoptotic activity of TAp73β was augmented by coexpression of p53β, whereas Δ133p53α and β inhibit its apoptotic activity most efficiently. We have determined the half-life of different p53 isoforms: p53γ isoform has the shortest half-life, whereas Δ133p53γ has the longest half-life. Inhibitory interactions of two proteins in complex often lead to their stabilization. However, only three isoforms (Δ133p53α, Δ133p53β and Δ40p53α) stabilize TAp73β. We are convinced that defining the interactions between p53/p73 would give a new insight into how the p53 isoforms modulate the p73 functions in tumorigenesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_026',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 424}]}],\n",
       "   'context': 'Differential effects of diverse p53 isoforms on TAp73 transcriptional activity and apoptosis. The p53 activities are due, at least in part, to its ability to form oligomers that bind to specific DNA sequences and activate transcription. Since some mutant p53 proteins and ΔNp73 isoforms form heterocomplexes with TAp73, we asked whether p53 isoforms can do the same and potentially act as dominant-negative inhibitors of TAp73. Moreover, it has already been found that some isoforms form complex with wtp53 and some of them inhibit p53 tumor-suppressor functions. Therefore, we studied the complex formation and co-immunoprecipitation assays show that all six p53 isoforms examined can form complexes with TAp73β, whereas only Δ133p53α/β/γ isoforms form complex with TAp73α. All p53 isoforms counteract TAp73β transactivation function but with different efficiency and in a promoter-dependent manner. Furthermore, apoptotic activity of TAp73β was augmented by coexpression of p53β, whereas Δ133p53α and β inhibit its apoptotic activity most efficiently. We have determined the half-life of different p53 isoforms: p53γ isoform has the shortest half-life, whereas Δ133p53γ has the longest half-life. Inhibitory interactions of two proteins in complex often lead to their stabilization. However, only three isoforms (Δ133p53α, Δ133p53β and Δ40p53α) stabilize TAp73β. We are convinced that defining the interactions between p53/p73 would give a new insight into how the p53 isoforms modulate the p73 functions in tumorigenesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_027',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 87}]}],\n",
       "   'context': \"p73 expression is regulated by RNPC1, a target of the p53 family, via mRNA stability. p73, a p53 family tumor suppressor, is expressed as TA and ΔN isoforms. Due to the role of p73 in tumor suppression and neural development, its expression and activity are tightly regulated by multiple mechanisms, including transcription and posttranslational modifications. Here, we found that p73 mRNA stability is regulated by RNPC1, an RNA binding protein and a target of the p53 family. We also showed that a CU-rich element in the 3' untranslated region of p73 is recognized by and responsive to RNPC1. To explore the physiological significance of RNPC1-regulated p73 expression, we showed that the loss of RNPC1 in p53-null mouse embryonic fibroblasts leads to reduced expression of p73, along with decreased expression of p21, p130, and γ-H2A.X, and consequently a decreased number of senescent cells. Furthermore, we observed that knockdown of TAp73 or p21, another target of RNPC1, attenuates the inhibitory effect of RNPC1 on cell proliferation and premature senescence, whereas combined knockdown of TAp73 and p21 completely abolishes it. Due to the fact that RNPC1 is a target of p73, the mutual regulation between p73 and RNPC1 constitutes a novel feed-forward loop, which might be explored as a target for tumors without a functional p53.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_028',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 120}]}],\n",
       "   'context': 'Mammary epithelial cell polarity is regulated differentially by p73 isoforms via epithelial-to-mesenchymal transition. p73 is expressed as TA and ΔN isoforms, both of which are implicated in tumor suppression and/or promotion. To address how p73 possesses these opposing functions, we developed three-dimensional culture of MCF10A cells, which undergo cell morphogenesis to form polarized spheroids with hollow lumen similar to normal mammary acini in vivo. Here, we showed that upon knockdown of p73, particularly TAp73 but not ΔNp73, MCF10A cells formed irregular and near-normal acini without hollow lumen in three-dimensional culture. We also found that upon knockdown of p73 or TAp73, but not ΔNp73, MCF10A cells underwent epithelial-to-mesenchymal transition (EMT) via down-regulation of E-cadherin coupled with up-regulation of β-catenin and laminin V. In addition, we found that Snail-1, Slug, and Twist, all of which are known to act as EMT inducers by repressing E-cadherin expression, were increased markedly upon knockdown of p73 and TAp73 but little if any by ΔNp73. Furthermore, we showed that knockdown of p73 or TAp73 in MCF10A cells led to a marked increase in cell proliferation and migration. Together, our data suggest that TAp73 is necessary for maintaining normal cell polarity by suppressing EMT.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_029',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 973}]}],\n",
       "   'context': 'TAp73 is downregulated in oocytes from women of advanced reproductive age. Studies on oocyte transcriptome are important to understand the biological pathways involved in oogenesis, totipotence and early embryonic development. Moreover, genes regulating physiological pathways in gametes could represent potential candidates for reproductive disorders. In addition to oocyte specific transcription factors, also the members of the p53 family could be etiologically involved due to their biological functions. In fact, their role in the control of cell cycle, apoptosis, and germ-line genome stability is well known. Female reproductive aging is one of the causes of fertility reduction and it is often associated with egg aneuploidy increase. In order to verify the potential involvement of p73 in reproductive aging, we determined its expression in single mature MII oocytes from two groups of women, younger than 35 or older than 38 years, respectively. We found that TAp73 isoforms are down regulated in oocytes from women older than 38 years. We confirmed these data in pools of mouse oocytes. TAp73 down regulation in oocytes from women of advanced reproductive age could explain both the reduction of fertility and the increase of newborns with chromosomal abnormalities.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_030',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 100}]}],\n",
       "   'context': 'Global DNA hypomethylation-induced ΔNp73 transcriptional activation in non-small cell lung cancer. p73 possesses an extrinsic P1 promoter and an intrinsic P2 promoter controlling the expression of the pro-apoptotic TAp73 isoforms and the anti-apoptotic ΔΝp73 isoforms respectively. In this study, we investigated the DNA methylation status of both promoters as a means of epigenetic transcriptional control of their corresponding isoforms in 102 primary non-small cell lung carcinomas (NSCLCs). We demonstrated that while P1 hypermethylation-associated reduction of TAp73 mRNA levels is relatively infrequent, the P2 hypomethylation-associated over-expression of ΔΝp73 mRNA is a frequent event, particularly among squamous cell carcinomas. P2 hypomethylation strongly correlated with LINE-1 element hypomethylation, indicating that ΔΝp73 over-expression may be a passive consequence of global DNA hypomethylation.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_031',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 218}]}],\n",
       "   'context': 'Global DNA hypomethylation-induced ΔNp73 transcriptional activation in non-small cell lung cancer. p73 possesses an extrinsic P1 promoter and an intrinsic P2 promoter controlling the expression of the pro-apoptotic TAp73 isoforms and the anti-apoptotic ΔΝp73 isoforms respectively. In this study, we investigated the DNA methylation status of both promoters as a means of epigenetic transcriptional control of their corresponding isoforms in 102 primary non-small cell lung carcinomas (NSCLCs). We demonstrated that while P1 hypermethylation-associated reduction of TAp73 mRNA levels is relatively infrequent, the P2 hypomethylation-associated over-expression of ΔΝp73 mRNA is a frequent event, particularly among squamous cell carcinomas. P2 hypomethylation strongly correlated with LINE-1 element hypomethylation, indicating that ΔΝp73 over-expression may be a passive consequence of global DNA hypomethylation.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_032',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 256}]}],\n",
       "   'context': 'Global DNA hypomethylation-induced ΔNp73 transcriptional activation in non-small cell lung cancer. p73 possesses an extrinsic P1 promoter and an intrinsic P2 promoter controlling the expression of the pro-apoptotic TAp73 isoforms and the anti-apoptotic ΔΝp73 isoforms respectively. In this study, we investigated the DNA methylation status of both promoters as a means of epigenetic transcriptional control of their corresponding isoforms in 102 primary non-small cell lung carcinomas (NSCLCs). We demonstrated that while P1 hypermethylation-associated reduction of TAp73 mRNA levels is relatively infrequent, the P2 hypomethylation-associated over-expression of ΔΝp73 mRNA is a frequent event, particularly among squamous cell carcinomas. P2 hypomethylation strongly correlated with LINE-1 element hypomethylation, indicating that ΔΝp73 over-expression may be a passive consequence of global DNA hypomethylation.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_033',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1063}]}],\n",
       "   'context': 'BRCA1-associated epigenetic regulation of p73 mediates an effector pathway for chemosensitivity in ovarian carcinoma. The majority of tumors arising in BRCA1 mutation carriers exhibit inactivation of p53, a key effector of cell death after DNA damage. Despite the loss of p53, BRCA1-deficient tumor cells exhibit increased sensitivity to cisplatin, and patients with BRCA1-associated ovarian carcinomas experience improved outcomes with platinum-based chemotherapy compared with sporadic cases. Although it is known that chemosensitivity in BRCA1-associated cancers is associated with unrepaired DNA damage, the specific effector pathway mediating the cellular response to platinum-induced damage in these tumors is poorly understood. Here, we show that the p53-related gene p73, encoding a proapoptotic protein that is linked to chemosensitivity in many settings, is upregulated through a novel epigenetic mechanism in both human and murine models of BRCA1-associated ovarian carcinoma. BRCA1-deficient ovarian carcinoma cells exhibit hypermethylation within a p73 regulatory region, which includes the binding site for the p73 transcriptional repressor ZEB1, leading to the abrogation of ZEB1 binding and increased expression of transactivating p73 isoforms (TAp73). Cisplatin chemotherapy induces TAp73 target genes specifically in BRCA1-deficient cells, and knockdown of TAp73 in these cells causes chemoresistance while having little or no effect on BRCA1-expressing tumor cells. In primary ovarian carcinomas, ZEB1 binding site methylation and TAp73 expression correlate with BRCA1 status and with clinical response. Together, these findings uncover a novel regulatory mechanism that supports the contribution of TAp73 as an important mediator of the response to platinum chemotherapy in a subset of ovarian carcinomas. TAp73 might represent a response predictor and potential therapeutic target for enhancing chemosensitivity in this disease.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_034',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1126}]}],\n",
       "   'context': 'BRCA1-associated epigenetic regulation of p73 mediates an effector pathway for chemosensitivity in ovarian carcinoma. The majority of tumors arising in BRCA1 mutation carriers exhibit inactivation of p53, a key effector of cell death after DNA damage. Despite the loss of p53, BRCA1-deficient tumor cells exhibit increased sensitivity to cisplatin, and patients with BRCA1-associated ovarian carcinomas experience improved outcomes with platinum-based chemotherapy compared with sporadic cases. Although it is known that chemosensitivity in BRCA1-associated cancers is associated with unrepaired DNA damage, the specific effector pathway mediating the cellular response to platinum-induced damage in these tumors is poorly understood. Here, we show that the p53-related gene p73, encoding a proapoptotic protein that is linked to chemosensitivity in many settings, is upregulated through a novel epigenetic mechanism in both human and murine models of BRCA1-associated ovarian carcinoma. BRCA1-deficient ovarian carcinoma cells exhibit hypermethylation within a p73 regulatory region, which includes the binding site for the p73 transcriptional repressor ZEB1, leading to the abrogation of ZEB1 binding and increased expression of transactivating p73 isoforms (TAp73). Cisplatin chemotherapy induces TAp73 target genes specifically in BRCA1-deficient cells, and knockdown of TAp73 in these cells causes chemoresistance while having little or no effect on BRCA1-expressing tumor cells. In primary ovarian carcinomas, ZEB1 binding site methylation and TAp73 expression correlate with BRCA1 status and with clinical response. Together, these findings uncover a novel regulatory mechanism that supports the contribution of TAp73 as an important mediator of the response to platinum chemotherapy in a subset of ovarian carcinomas. TAp73 might represent a response predictor and potential therapeutic target for enhancing chemosensitivity in this disease.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_035',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1248}]}],\n",
       "   'context': 'BRCA1-associated epigenetic regulation of p73 mediates an effector pathway for chemosensitivity in ovarian carcinoma. The majority of tumors arising in BRCA1 mutation carriers exhibit inactivation of p53, a key effector of cell death after DNA damage. Despite the loss of p53, BRCA1-deficient tumor cells exhibit increased sensitivity to cisplatin, and patients with BRCA1-associated ovarian carcinomas experience improved outcomes with platinum-based chemotherapy compared with sporadic cases. Although it is known that chemosensitivity in BRCA1-associated cancers is associated with unrepaired DNA damage, the specific effector pathway mediating the cellular response to platinum-induced damage in these tumors is poorly understood. Here, we show that the p53-related gene p73, encoding a proapoptotic protein that is linked to chemosensitivity in many settings, is upregulated through a novel epigenetic mechanism in both human and murine models of BRCA1-associated ovarian carcinoma. BRCA1-deficient ovarian carcinoma cells exhibit hypermethylation within a p73 regulatory region, which includes the binding site for the p73 transcriptional repressor ZEB1, leading to the abrogation of ZEB1 binding and increased expression of transactivating p73 isoforms (TAp73). Cisplatin chemotherapy induces TAp73 target genes specifically in BRCA1-deficient cells, and knockdown of TAp73 in these cells causes chemoresistance while having little or no effect on BRCA1-expressing tumor cells. In primary ovarian carcinomas, ZEB1 binding site methylation and TAp73 expression correlate with BRCA1 status and with clinical response. Together, these findings uncover a novel regulatory mechanism that supports the contribution of TAp73 as an important mediator of the response to platinum chemotherapy in a subset of ovarian carcinomas. TAp73 might represent a response predictor and potential therapeutic target for enhancing chemosensitivity in this disease.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_036',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1264}]}],\n",
       "   'context': 'BRCA1-associated epigenetic regulation of p73 mediates an effector pathway for chemosensitivity in ovarian carcinoma. The majority of tumors arising in BRCA1 mutation carriers exhibit inactivation of p53, a key effector of cell death after DNA damage. Despite the loss of p53, BRCA1-deficient tumor cells exhibit increased sensitivity to cisplatin, and patients with BRCA1-associated ovarian carcinomas experience improved outcomes with platinum-based chemotherapy compared with sporadic cases. Although it is known that chemosensitivity in BRCA1-associated cancers is associated with unrepaired DNA damage, the specific effector pathway mediating the cellular response to platinum-induced damage in these tumors is poorly understood. Here, we show that the p53-related gene p73, encoding a proapoptotic protein that is linked to chemosensitivity in many settings, is upregulated through a novel epigenetic mechanism in both human and murine models of BRCA1-associated ovarian carcinoma. BRCA1-deficient ovarian carcinoma cells exhibit hypermethylation within a p73 regulatory region, which includes the binding site for the p73 transcriptional repressor ZEB1, leading to the abrogation of ZEB1 binding and increased expression of transactivating p73 isoforms (TAp73). Cisplatin chemotherapy induces TAp73 target genes specifically in BRCA1-deficient cells, and knockdown of TAp73 in these cells causes chemoresistance while having little or no effect on BRCA1-expressing tumor cells. In primary ovarian carcinomas, ZEB1 binding site methylation and TAp73 expression correlate with BRCA1 status and with clinical response. Together, these findings uncover a novel regulatory mechanism that supports the contribution of TAp73 as an important mediator of the response to platinum chemotherapy in a subset of ovarian carcinomas. TAp73 might represent a response predictor and potential therapeutic target for enhancing chemosensitivity in this disease.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_037',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 590}]}],\n",
       "   'context': 'Analysis of the intracellular localization of p73 N-terminal protein isoforms TAp73 and ∆Np73 in medulloblastoma cell lines. The protein homologous to the tumor suppressor p53, p73, has essential roles in development and tumorigenesis. This protein exists in a wide range of isoforms with different, even antagonistic, functions. However, there are virtually no detailed morphological studies analyzing the endogenous expression of p73 isoforms at the cellular level in cancer cells. In this study, we investigated the expression and subcellular distribution of two N-terminal isoforms, TAp73 and ΔNp73, in medulloblastoma cells using immunofluorescence microscopy. Both proteins were observed in all cell lines examined, but differences were noted in their intracellular localization between the reference Daoy cell line and four newly established medulloblastoma cell lines (MBL-03, MBL-06, MBL-07 and MBL-10). In the new cell lines, TAp73 and ΔNp73 were located predominantly in cell nuclei. However, there was heterogeneity in TAp73 distribution in the cells of all MBL cell lines, with the protein located in the nucleus and also in a limited non-random area in the cytoplasm. In a small percentage of cells, we detected cytoplasmic localization of TAp73 only, i.e., nuclear exclusion was observed. Our results provide a basis for future studies on the causes and function of distinct intracellular localization of p73 protein isoforms with respect to different protein-protein interactions in medulloblastoma cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_038',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 600}]}],\n",
       "   'context': 'Analysis of the intracellular localization of p73 N-terminal protein isoforms TAp73 and ∆Np73 in medulloblastoma cell lines. The protein homologous to the tumor suppressor p53, p73, has essential roles in development and tumorigenesis. This protein exists in a wide range of isoforms with different, even antagonistic, functions. However, there are virtually no detailed morphological studies analyzing the endogenous expression of p73 isoforms at the cellular level in cancer cells. In this study, we investigated the expression and subcellular distribution of two N-terminal isoforms, TAp73 and ΔNp73, in medulloblastoma cells using immunofluorescence microscopy. Both proteins were observed in all cell lines examined, but differences were noted in their intracellular localization between the reference Daoy cell line and four newly established medulloblastoma cell lines (MBL-03, MBL-06, MBL-07 and MBL-10). In the new cell lines, TAp73 and ΔNp73 were located predominantly in cell nuclei. However, there was heterogeneity in TAp73 distribution in the cells of all MBL cell lines, with the protein located in the nucleus and also in a limited non-random area in the cytoplasm. In a small percentage of cells, we detected cytoplasmic localization of TAp73 only, i.e., nuclear exclusion was observed. Our results provide a basis for future studies on the causes and function of distinct intracellular localization of p73 protein isoforms with respect to different protein-protein interactions in medulloblastoma cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_039',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 541}]}],\n",
       "   'context': 'Differential control of TAp73 and DeltaNp73 protein stability by the ring finger ubiquitin ligase PIR2. p73 is a p53-related transcription factor with fundamental roles in development and tumor suppression. Transcription from two different promoters on the p73 gene results in generation of transcriptionally active TAp73 isoforms and dominant negative DeltaNp73 isoforms with opposing pro- and anti-apoptotic functions. Therefore, the relative ratio of each isoform is an important determinant of the cell fate. Proteasomal degradation of p73 is mediated by polyubiquitination-dependent and -independent processes both of which appear, thus far, to lack selectivity for the TAp73 and DeltaNp73 isoforms. Here, we describe the characterization of another transcriptional target of TAp73; a ring finger domain ubiquitin ligase p73 Induced RING 2 protein (PIR2). Although PIR2 was initially identified a p53-induced gene (p53RFP), low abundance of PIR2 transcript in mouse embryonic fibroblasts of TAp73 KO mice compared with WT mice and comparison of PIR2 mRNA and protein levels following TAp73 or p53 overexpression substantiate TAp73 isoforms as strong inducers of PIR2. Although PIR2 expression was induced by DNA damage, its expression did not alter apoptotic response or cell cycle profile per se. However, coexpression of PIR2 with TAp73 or DeltaNp73 resulted in an increase of the TA/DeltaNp73 ratio, due to preferential degradation of DeltaNp73. Finally, PIR2 was able to relieve the inhibitory effect of DeltaNp73 on TAp73 induced apoptosis following DNA damage. These results suggest that PIR2, by being induced by TAp73 and degrading DeltaNp73, differentially regulates TAp73/DeltaNp73 stability, and, hence, it may offer a therapeutic approach to enhance the chemosensitivity of tumor cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_040',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 678}]}],\n",
       "   'context': 'Differential control of TAp73 and DeltaNp73 protein stability by the ring finger ubiquitin ligase PIR2. p73 is a p53-related transcription factor with fundamental roles in development and tumor suppression. Transcription from two different promoters on the p73 gene results in generation of transcriptionally active TAp73 isoforms and dominant negative DeltaNp73 isoforms with opposing pro- and anti-apoptotic functions. Therefore, the relative ratio of each isoform is an important determinant of the cell fate. Proteasomal degradation of p73 is mediated by polyubiquitination-dependent and -independent processes both of which appear, thus far, to lack selectivity for the TAp73 and DeltaNp73 isoforms. Here, we describe the characterization of another transcriptional target of TAp73; a ring finger domain ubiquitin ligase p73 Induced RING 2 protein (PIR2). Although PIR2 was initially identified a p53-induced gene (p53RFP), low abundance of PIR2 transcript in mouse embryonic fibroblasts of TAp73 KO mice compared with WT mice and comparison of PIR2 mRNA and protein levels following TAp73 or p53 overexpression substantiate TAp73 isoforms as strong inducers of PIR2. Although PIR2 expression was induced by DNA damage, its expression did not alter apoptotic response or cell cycle profile per se. However, coexpression of PIR2 with TAp73 or DeltaNp73 resulted in an increase of the TA/DeltaNp73 ratio, due to preferential degradation of DeltaNp73. Finally, PIR2 was able to relieve the inhibitory effect of DeltaNp73 on TAp73 induced apoptosis following DNA damage. These results suggest that PIR2, by being induced by TAp73 and degrading DeltaNp73, differentially regulates TAp73/DeltaNp73 stability, and, hence, it may offer a therapeutic approach to enhance the chemosensitivity of tumor cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_041',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 692}]}],\n",
       "   'context': 'Differential control of TAp73 and DeltaNp73 protein stability by the ring finger ubiquitin ligase PIR2. p73 is a p53-related transcription factor with fundamental roles in development and tumor suppression. Transcription from two different promoters on the p73 gene results in generation of transcriptionally active TAp73 isoforms and dominant negative DeltaNp73 isoforms with opposing pro- and anti-apoptotic functions. Therefore, the relative ratio of each isoform is an important determinant of the cell fate. Proteasomal degradation of p73 is mediated by polyubiquitination-dependent and -independent processes both of which appear, thus far, to lack selectivity for the TAp73 and DeltaNp73 isoforms. Here, we describe the characterization of another transcriptional target of TAp73; a ring finger domain ubiquitin ligase p73 Induced RING 2 protein (PIR2). Although PIR2 was initially identified a p53-induced gene (p53RFP), low abundance of PIR2 transcript in mouse embryonic fibroblasts of TAp73 KO mice compared with WT mice and comparison of PIR2 mRNA and protein levels following TAp73 or p53 overexpression substantiate TAp73 isoforms as strong inducers of PIR2. Although PIR2 expression was induced by DNA damage, its expression did not alter apoptotic response or cell cycle profile per se. However, coexpression of PIR2 with TAp73 or DeltaNp73 resulted in an increase of the TA/DeltaNp73 ratio, due to preferential degradation of DeltaNp73. Finally, PIR2 was able to relieve the inhibitory effect of DeltaNp73 on TAp73 induced apoptosis following DNA damage. These results suggest that PIR2, by being induced by TAp73 and degrading DeltaNp73, differentially regulates TAp73/DeltaNp73 stability, and, hence, it may offer a therapeutic approach to enhance the chemosensitivity of tumor cells.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_042',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 359}]}],\n",
       "   'context': 'ΔNp73β is oncogenic in hepatocellular carcinoma by blocking apoptosis signaling via death receptors and mitochondria. BACKGROUND & AIMS: p73 belongs to the p53 family of transcription factors known to regulate cell cycle and apoptosis. The Trp73 gene has two promoters that drive the expression of two major p73 isoform subfamilies: TA and ΔN. In general, TAp73 isoforms show proapoptotic activities, whereas members of the N-terminally truncated (ΔN) p73 subfamily that lack the transactivation domain show antiapoptotic functions. We found that upregulation of ΔNp73 in hepatocellular carcinoma (HCC) correlated with reduced survival. Here, we investigated the molecular mechanisms accounting for the oncogenic role of ΔNp73 in HCC. RESULTS: ΔNp73β can directly interfere with the transcriptional activation function of the TA (containing the transactivation domain) isoforms of the p53 family and consequently inhibit transactivation of proapoptotic target genes. Interference of ΔNp73β with apoptosis-/chemosensitivity takes place at several levels of apoptosis signaling. ΔNp73β negatively regulates the genes encoding for the death receptors CD95, TNF-R1, TRAIL-R2 and TNFRSF18. Furthermore, ΔNp73β represses the genes encoding caspase-2, -3, -6, -8 and -9. Concomitantly, ΔNp73β inhibits apoptosis emanating from mitochondria. CONCLUSIONS: Thus, ΔNp73 expression in HCC selects against both the death receptor and the mitochondrial apoptosis activity of the TA isoforms. Our data suggest that ΔNp73 isoforms repress apoptosis-related genes of the extrinsic and intrinsic apoptosis signaling pathways thereby contributing to chemoresistance. The clinical importance of these data is evidenced by our finding that the ΔNp73ß target gene signature can predict the prognosis of patients suffering from HCC.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_043',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 453}]}],\n",
       "   'context': 'ΔNp73β is oncogenic in hepatocellular carcinoma by blocking apoptosis signaling via death receptors and mitochondria. BACKGROUND & AIMS: p73 belongs to the p53 family of transcription factors known to regulate cell cycle and apoptosis. The Trp73 gene has two promoters that drive the expression of two major p73 isoform subfamilies: TA and ΔN. In general, TAp73 isoforms show proapoptotic activities, whereas members of the N-terminally truncated (ΔN) p73 subfamily that lack the transactivation domain show antiapoptotic functions. We found that upregulation of ΔNp73 in hepatocellular carcinoma (HCC) correlated with reduced survival. Here, we investigated the molecular mechanisms accounting for the oncogenic role of ΔNp73 in HCC. RESULTS: ΔNp73β can directly interfere with the transcriptional activation function of the TA (containing the transactivation domain) isoforms of the p53 family and consequently inhibit transactivation of proapoptotic target genes. Interference of ΔNp73β with apoptosis-/chemosensitivity takes place at several levels of apoptosis signaling. ΔNp73β negatively regulates the genes encoding for the death receptors CD95, TNF-R1, TRAIL-R2 and TNFRSF18. Furthermore, ΔNp73β represses the genes encoding caspase-2, -3, -6, -8 and -9. Concomitantly, ΔNp73β inhibits apoptosis emanating from mitochondria. CONCLUSIONS: Thus, ΔNp73 expression in HCC selects against both the death receptor and the mitochondrial apoptosis activity of the TA isoforms. Our data suggest that ΔNp73 isoforms repress apoptosis-related genes of the extrinsic and intrinsic apoptosis signaling pathways thereby contributing to chemoresistance. The clinical importance of these data is evidenced by our finding that the ΔNp73ß target gene signature can predict the prognosis of patients suffering from HCC.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_044',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 113}]}],\n",
       "   'context': \"Sp1 binds to the external promoter of the p73 gene and induces the expression of TAp73gamma in lung cancer. The p73 gene possesses an extrinsic P1 promoter and an intrinsic P2 promoter, resulting in TAp73 and DeltaNup73 isoforms, respectively. The ultimate effect of p73 in oncogenesis is thought to depend on the apoptotic TA to antiapoptotic DeltaN isoforms' ratio. This study was aimed at identifying novel transcription factors that affect TA isoform synthesis. With the use of bioinformatics tools, in vitro binding assays, and chromatin immunoprecipitation analysis, a region extending -233 to -204 bp upstream of the transcription start site of the human p73 P1 promoter, containing conserved Sp1-binding sites, was characterized. Treatment of cells with Sp1 RNAi and Sp1 inhibitor functionally suppress TAp73 expression, indicating positive regulation of P1 by the Sp1 protein. Notably Sp1 inhibition or knockdown also reduces DeltaNup73 protein levels. Therefore, Sp1 directly regulates TAp73 transcription and affects DeltaNup73 levels in lung cancer. TAp73gamma was shown to be the only TA isoform overexpressed in several lung cancer cell lines and in 26 non-small cell lung cancers, consistent with Sp1 overexpression, thereby questioning the apoptotic role of this specific p73 isoform in lung cancer.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_045',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 202}]}],\n",
       "   'context': \"Sp1 binds to the external promoter of the p73 gene and induces the expression of TAp73gamma in lung cancer. The p73 gene possesses an extrinsic P1 promoter and an intrinsic P2 promoter, resulting in TAp73 and DeltaNup73 isoforms, respectively. The ultimate effect of p73 in oncogenesis is thought to depend on the apoptotic TA to antiapoptotic DeltaN isoforms' ratio. This study was aimed at identifying novel transcription factors that affect TA isoform synthesis. With the use of bioinformatics tools, in vitro binding assays, and chromatin immunoprecipitation analysis, a region extending -233 to -204 bp upstream of the transcription start site of the human p73 P1 promoter, containing conserved Sp1-binding sites, was characterized. Treatment of cells with Sp1 RNAi and Sp1 inhibitor functionally suppress TAp73 expression, indicating positive regulation of P1 by the Sp1 protein. Notably Sp1 inhibition or knockdown also reduces DeltaNup73 protein levels. Therefore, Sp1 directly regulates TAp73 transcription and affects DeltaNup73 levels in lung cancer. TAp73gamma was shown to be the only TA isoform overexpressed in several lung cancer cell lines and in 26 non-small cell lung cancers, consistent with Sp1 overexpression, thereby questioning the apoptotic role of this specific p73 isoform in lung cancer.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_046',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 217}]}],\n",
       "   'context': \"Sp1 binds to the external promoter of the p73 gene and induces the expression of TAp73gamma in lung cancer. The p73 gene possesses an extrinsic P1 promoter and an intrinsic P2 promoter, resulting in TAp73 and DeltaNup73 isoforms, respectively. The ultimate effect of p73 in oncogenesis is thought to depend on the apoptotic TA to antiapoptotic DeltaN isoforms' ratio. This study was aimed at identifying novel transcription factors that affect TA isoform synthesis. With the use of bioinformatics tools, in vitro binding assays, and chromatin immunoprecipitation analysis, a region extending -233 to -204 bp upstream of the transcription start site of the human p73 P1 promoter, containing conserved Sp1-binding sites, was characterized. Treatment of cells with Sp1 RNAi and Sp1 inhibitor functionally suppress TAp73 expression, indicating positive regulation of P1 by the Sp1 protein. Notably Sp1 inhibition or knockdown also reduces DeltaNup73 protein levels. Therefore, Sp1 directly regulates TAp73 transcription and affects DeltaNup73 levels in lung cancer. TAp73gamma was shown to be the only TA isoform overexpressed in several lung cancer cell lines and in 26 non-small cell lung cancers, consistent with Sp1 overexpression, thereby questioning the apoptotic role of this specific p73 isoform in lung cancer.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_047',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 142}]}],\n",
       "   'context': 'Isoform-specific p73 knockout mice reveal a novel role for delta Np73 in the DNA damage response pathway. Mice with a complete deficiency of p73 have severe neurological and immunological defects due to the absence of all TAp73 and DeltaNp73 isoforms. As part of our ongoing program to distinguish the biological functions of these isoforms, we generated mice that are selectively deficient for the DeltaNp73 isoform. Mice lacking DeltaNp73 (DeltaNp73(-/-) mice) are viable and fertile but display signs of neurodegeneration. Cells from DeltaNp73(-/-) mice are sensitized to DNA-damaging agents and show an increase in p53-dependent apoptosis. When analyzing the DNA damage response (DDR) in DeltaNp73(-/-) cells, we discovered a completely new role for DeltaNp73 in inhibiting the molecular signal emanating from a DNA break to the DDR pathway. We found that DeltaNp73 localizes directly to the site of DNA damage, can interact with the DNA damage sensor protein 53BP1, and inhibits ATM activation and subsequent p53 phosphorylation. This novel finding may explain why human tumors with high levels of DeltaNp73 expression show enhanced resistance to chemotherapy.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_048',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 225}]}],\n",
       "   'context': 'Isoform-specific p73 knockout mice reveal a novel role for delta Np73 in the DNA damage response pathway. Mice with a complete deficiency of p73 have severe neurological and immunological defects due to the absence of all TAp73 and DeltaNp73 isoforms. As part of our ongoing program to distinguish the biological functions of these isoforms, we generated mice that are selectively deficient for the DeltaNp73 isoform. Mice lacking DeltaNp73 (DeltaNp73(-/-) mice) are viable and fertile but display signs of neurodegeneration. Cells from DeltaNp73(-/-) mice are sensitized to DNA-damaging agents and show an increase in p53-dependent apoptosis. When analyzing the DNA damage response (DDR) in DeltaNp73(-/-) cells, we discovered a completely new role for DeltaNp73 in inhibiting the molecular signal emanating from a DNA break to the DDR pathway. We found that DeltaNp73 localizes directly to the site of DNA damage, can interact with the DNA damage sensor protein 53BP1, and inhibits ATM activation and subsequent p53 phosphorylation. This novel finding may explain why human tumors with high levels of DeltaNp73 expression show enhanced resistance to chemotherapy.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_049',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 239}]}],\n",
       "   'context': 'Isoform-specific p73 knockout mice reveal a novel role for delta Np73 in the DNA damage response pathway. Mice with a complete deficiency of p73 have severe neurological and immunological defects due to the absence of all TAp73 and DeltaNp73 isoforms. As part of our ongoing program to distinguish the biological functions of these isoforms, we generated mice that are selectively deficient for the DeltaNp73 isoform. Mice lacking DeltaNp73 (DeltaNp73(-/-) mice) are viable and fertile but display signs of neurodegeneration. Cells from DeltaNp73(-/-) mice are sensitized to DNA-damaging agents and show an increase in p53-dependent apoptosis. When analyzing the DNA damage response (DDR) in DeltaNp73(-/-) cells, we discovered a completely new role for DeltaNp73 in inhibiting the molecular signal emanating from a DNA break to the DDR pathway. We found that DeltaNp73 localizes directly to the site of DNA damage, can interact with the DNA damage sensor protein 53BP1, and inhibits ATM activation and subsequent p53 phosphorylation. This novel finding may explain why human tumors with high levels of DeltaNp73 expression show enhanced resistance to chemotherapy.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_050',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 832}]}],\n",
       "   'context': 'Cell density-dependent regulation of p73 in breast cancer cells. Molecular regulation of p73, a p53 family member, remains unclear. Here we report that p73 expression is significantly regulated by cell densities. In particular, we found that p73alpha and p73beta are differentially regulated. While p73beta protein levels were inversely correlated with cell densities, p73alpha protein levels behaved oppositely. We further showed that density-dependent changes of p73alpha follow the same patterns as E2F-1 and TAp73 mRNA levels, suggesting transcriptional regulation. Our data also suggest that high levels of p73beta at lower densities may be due to increased protein stability. However, AIP-4/Itch appeared not to be involved in downregulation of p73beta at high densities. Moreover, we also found that subcellular location of p73 isoforms changes with the culture density increases. While high level of p73beta at low density was mainly presented in the nucleus, low levels of this protein at high densities were mainly in the cytosol. Taken together, these findings reveal a novel mechanism that differentially regulates p73 isoforms and underscores the role of cell-cell interaction in p73 regulation, which may advance our understanding of p73 expression and function in human cancers.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_051',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1280}]}],\n",
       "   'context': 'Induction of the small heat shock protein alphaB-crystallin by genotoxic stress is mediated by p53 and p73. The small heat shock protein alphaB-crystallin is a molecular chaperone that is induced by stress and protects cells by inhibiting protein aggregation and apoptosis. To identify novel transcriptional regulators of the alphaB-crystallin gene, we examined the alphaB-crystallin promoter for conserved transcription factor DNA-binding elements and identified a putative response element for the p53 tumor suppressor protein. Ectopic expression of wild-type p53 induced alphaB-crystallin mRNA and protein with delayed kinetics compared to p21. Additionally, the induction of alphaB-crystallin by genotoxic stress was inhibited by siRNAs targeting p53. Although the p53-dependent transactivation of an alphaB-crystallin promoter luciferase reporter required the putative p53RE, chromatin immunoprecipitation failed to detect p53 binding to the alphaB-crystallin promoter. These results suggested an indirect mechanism of transactivation involving p53 family members p63 or p73. DeltaNp73 was dramatically induced by p53 in a TAp73-dependent manner, and silencing p73 suppressed the transcriptional activation of alphaB-crystallin by p53. Moreover, ectopic expression of DeltaNp73alpha (but not other p73 isoforms) increased alphaB-crystallin mRNA levels in the absence of p53. Collectively, our results link the molecular chaperone alphaB-crystallin to the cellular genotoxic stress response via a novel mechanism of transcriptional regulation by p53 and p73.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_052',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1304}]}],\n",
       "   'context': 'Induction of the small heat shock protein alphaB-crystallin by genotoxic stress is mediated by p53 and p73. The small heat shock protein alphaB-crystallin is a molecular chaperone that is induced by stress and protects cells by inhibiting protein aggregation and apoptosis. To identify novel transcriptional regulators of the alphaB-crystallin gene, we examined the alphaB-crystallin promoter for conserved transcription factor DNA-binding elements and identified a putative response element for the p53 tumor suppressor protein. Ectopic expression of wild-type p53 induced alphaB-crystallin mRNA and protein with delayed kinetics compared to p21. Additionally, the induction of alphaB-crystallin by genotoxic stress was inhibited by siRNAs targeting p53. Although the p53-dependent transactivation of an alphaB-crystallin promoter luciferase reporter required the putative p53RE, chromatin immunoprecipitation failed to detect p53 binding to the alphaB-crystallin promoter. These results suggested an indirect mechanism of transactivation involving p53 family members p63 or p73. DeltaNp73 was dramatically induced by p53 in a TAp73-dependent manner, and silencing p73 suppressed the transcriptional activation of alphaB-crystallin by p53. Moreover, ectopic expression of DeltaNp73alpha (but not other p73 isoforms) increased alphaB-crystallin mRNA levels in the absence of p53. Collectively, our results link the molecular chaperone alphaB-crystallin to the cellular genotoxic stress response via a novel mechanism of transcriptional regulation by p53 and p73.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_053',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 436}]}],\n",
       "   'context': 'Active transcription of the human FAS/CD95/TNFRSF6 gene involves the p53 family. p63 and p73 express two main classes of isoforms: isoforms which contain the transactivation domain (TAp73 and TAp63) executing transcriptional activity and dominant-negative isoforms which are truncated at the NH2-terminus acting as operant inhibitors of TAp73, TAp63 and wild-type p53, and thus possessing oncogenic potential. Like wt p53, TAp63 and TAp73 isoforms transactivate target genes that activate apoptosis signaling pathways. In an attempt to understand how the CD95 gene is regulated by the p53 family, we investigated the contributions of a p53-responsive element (RE) within the first intron of the CD95 gene as well as three elements within the promoter. The intronic element conferred transcriptional activation by p53, TAp63 and TAp73 and cooperated with the p53-REs in the promoter of the CD95 gene. Cooperation between the p53-REs in the promoter and the intronic p53-binding site resulted in maximal transcriptional activation of the CD95 gene by the p53 family.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_054',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 77}]}],\n",
       "   'context': 'p73 plays a role in erythroid differentiation through GATA1 induction. The TP73 gene gives rise to transactivation domain-p73 isoforms (TAp73) as well as DeltaNp73 variants with a truncated N terminus. Although TAp73alpha and -beta proteins are capable of inducing cell cycle arrest, apoptosis, and differentiation, DeltaNp73 acts in many cell types as a dominant-negative repressor of p53 and TAp73. It has been proposed that p73 is involved in myeloid differentiation, and its altered expression is involved in leukemic degeneration. However, there is little evidence as to which p73 variants (TA or DeltaN) are expressed during differentiation and whether specific p73 isoforms have the capacity to induce, or hinder, this differentiation in leukemia cells. In this study we identify GATA1 as a direct transcriptional target of TAp73alpha. Furthermore, TAp73alpha induces GATA1 activity, and it is required for erythroid differentiation. Additionally, we describe a functional cooperation between TAp73 and DeltaNp73 in the context of erythroid differentiation in human myeloid cells, K562 and UT-7. Moreover, the impaired expression of GATA1 and other erythroid genes in the liver of p73KO embryos, together with the moderated anemia observed in p73KO young mice, suggests a physiological role for TP73 in erythropoiesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_055',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 123}]}],\n",
       "   'context': 'p73 plays a role in erythroid differentiation through GATA1 induction. The TP73 gene gives rise to transactivation domain-p73 isoforms (TAp73) as well as DeltaNp73 variants with a truncated N terminus. Although TAp73alpha and -beta proteins are capable of inducing cell cycle arrest, apoptosis, and differentiation, DeltaNp73 acts in many cell types as a dominant-negative repressor of p53 and TAp73. It has been proposed that p73 is involved in myeloid differentiation, and its altered expression is involved in leukemic degeneration. However, there is little evidence as to which p73 variants (TA or DeltaN) are expressed during differentiation and whether specific p73 isoforms have the capacity to induce, or hinder, this differentiation in leukemia cells. In this study we identify GATA1 as a direct transcriptional target of TAp73alpha. Furthermore, TAp73alpha induces GATA1 activity, and it is required for erythroid differentiation. Additionally, we describe a functional cooperation between TAp73 and DeltaNp73 in the context of erythroid differentiation in human myeloid cells, K562 and UT-7. Moreover, the impaired expression of GATA1 and other erythroid genes in the liver of p73KO embryos, together with the moderated anemia observed in p73KO young mice, suggests a physiological role for TP73 in erythropoiesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_056',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 139}]}],\n",
       "   'context': 'p73 plays a role in erythroid differentiation through GATA1 induction. The TP73 gene gives rise to transactivation domain-p73 isoforms (TAp73) as well as DeltaNp73 variants with a truncated N terminus. Although TAp73alpha and -beta proteins are capable of inducing cell cycle arrest, apoptosis, and differentiation, DeltaNp73 acts in many cell types as a dominant-negative repressor of p53 and TAp73. It has been proposed that p73 is involved in myeloid differentiation, and its altered expression is involved in leukemic degeneration. However, there is little evidence as to which p73 variants (TA or DeltaN) are expressed during differentiation and whether specific p73 isoforms have the capacity to induce, or hinder, this differentiation in leukemia cells. In this study we identify GATA1 as a direct transcriptional target of TAp73alpha. Furthermore, TAp73alpha induces GATA1 activity, and it is required for erythroid differentiation. Additionally, we describe a functional cooperation between TAp73 and DeltaNp73 in the context of erythroid differentiation in human myeloid cells, K562 and UT-7. Moreover, the impaired expression of GATA1 and other erythroid genes in the liver of p73KO embryos, together with the moderated anemia observed in p73KO young mice, suggests a physiological role for TP73 in erythropoiesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_057',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 161}]}],\n",
       "   'context': 'p73 plays a role in erythroid differentiation through GATA1 induction. The TP73 gene gives rise to transactivation domain-p73 isoforms (TAp73) as well as DeltaNp73 variants with a truncated N terminus. Although TAp73alpha and -beta proteins are capable of inducing cell cycle arrest, apoptosis, and differentiation, DeltaNp73 acts in many cell types as a dominant-negative repressor of p53 and TAp73. It has been proposed that p73 is involved in myeloid differentiation, and its altered expression is involved in leukemic degeneration. However, there is little evidence as to which p73 variants (TA or DeltaN) are expressed during differentiation and whether specific p73 isoforms have the capacity to induce, or hinder, this differentiation in leukemia cells. In this study we identify GATA1 as a direct transcriptional target of TAp73alpha. Furthermore, TAp73alpha induces GATA1 activity, and it is required for erythroid differentiation. Additionally, we describe a functional cooperation between TAp73 and DeltaNp73 in the context of erythroid differentiation in human myeloid cells, K562 and UT-7. Moreover, the impaired expression of GATA1 and other erythroid genes in the liver of p73KO embryos, together with the moderated anemia observed in p73KO young mice, suggests a physiological role for TP73 in erythropoiesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_058',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 340}]}],\n",
       "   'context': 'Cyclooxygenase inhibitors differentially modulate p73 isoforms in neuroblastoma. p73 encodes multiple functionally distinct isoforms. Proapoptotic TAp73 isoforms contain a transactivation (TA) domain, and like p53, have tumor suppressor properties and are activated by chemotherapies to induce cell death. In contrast, antiapoptotic DeltaNp73 isoforms lack the TA domain and are dominant-negative inhibitors of p53 and TAp73. DeltaNp73 proteins are overexpressed in a variety of tumors including neuroblastoma. Thus, identification of drugs that upregulate TAp73 and/or downregulate DeltaNp73 represents a potential therapeutic strategy. Here, we report that cyclooxygenase (COX) inhibitors induce apoptosis independent of p53, and differentially modulate endogenous p73 isoforms in neuroblastoma and other tumors. COX inhibitor-mediated apoptosis is associated with the induction of TAp73beta and its target genes. COX inhibitors also downregulate the alternative-spliced DeltaNp73(AS) isoforms, Deltaexon2 and Deltaexon2/3. Furthermore, forced expression of DeltaNp73(AS) results in diminished apoptosis in response to the selective COX-2 inhibitor celecoxib. Celecoxib-mediated downregulation of DeltaNp73(AS) is associated with decreased E2F1 levels and diminished E2F1 activation of the p73 promoter. These results provide the first evidence that COX inhibitors differentially modulate p73 isoforms leading to enhanced apoptosis, and support the potential use of COX inhibitors as novel regulators of p73 to enhance chemosensitivity in tumors with deregulated E2F1 and in those with wild-type (wt) or mutant p53.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_059',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 422}]}],\n",
       "   'context': 'Cyclooxygenase inhibitors differentially modulate p73 isoforms in neuroblastoma. p73 encodes multiple functionally distinct isoforms. Proapoptotic TAp73 isoforms contain a transactivation (TA) domain, and like p53, have tumor suppressor properties and are activated by chemotherapies to induce cell death. In contrast, antiapoptotic DeltaNp73 isoforms lack the TA domain and are dominant-negative inhibitors of p53 and TAp73. DeltaNp73 proteins are overexpressed in a variety of tumors including neuroblastoma. Thus, identification of drugs that upregulate TAp73 and/or downregulate DeltaNp73 represents a potential therapeutic strategy. Here, we report that cyclooxygenase (COX) inhibitors induce apoptosis independent of p53, and differentially modulate endogenous p73 isoforms in neuroblastoma and other tumors. COX inhibitor-mediated apoptosis is associated with the induction of TAp73beta and its target genes. COX inhibitors also downregulate the alternative-spliced DeltaNp73(AS) isoforms, Deltaexon2 and Deltaexon2/3. Furthermore, forced expression of DeltaNp73(AS) results in diminished apoptosis in response to the selective COX-2 inhibitor celecoxib. Celecoxib-mediated downregulation of DeltaNp73(AS) is associated with decreased E2F1 levels and diminished E2F1 activation of the p73 promoter. These results provide the first evidence that COX inhibitors differentially modulate p73 isoforms leading to enhanced apoptosis, and support the potential use of COX inhibitors as novel regulators of p73 to enhance chemosensitivity in tumors with deregulated E2F1 and in those with wild-type (wt) or mutant p53.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_060',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 218}]}],\n",
       "   'context': 'p63 protein expression in high risk diffuse large B-cell lymphoma. BACKGROUND: p63 gene is a p53 homologue that encodes proteins with transactivation, DNA-binding and tetramerisation domains. The isoforms TAp63 and TAp73 transactivate p53 target genes and induce apoptosis, whereas the isoforms DeltaNp63 and DeltaNp73 lack transactivation and might have dominant-negative effects in p53 family members. p63 is expressed in germinal centre lymphocytes and can be related to the development of the lymphoma, but the prognostic significance of its expression in the survival of patients with diffuse large B-cell lymphoma (DLBCL) remains unclear. AIMS: To determine whether quantitative immunohistochemical (IHC) analysis of p63 protein expression correlates with CD10 antigen, Bcl-6 antigen and IRF4 antigen expression and to determine whether p63 is a surrogate predictor of overall survival in high-intermediate and high risk DLBCL populations. METHODS: CD10, Bcl-6 and IRF4 expression were retrospectively evaluated by IHC in 73 samples of high-intermediate and high risk DLBCL and were used to divide the lymphomas into subgroups of germinal centre B-cell-like (GCB) and activate B-cell-like (ABC) DLBCL. Similarly, p63 expression was evaluated by IHC and the results were compared with subgroups of DLBCL origin and with the survival rates for these patients. RESULTS: p63 was expressed in more than 50% of malignant cells in 11 patients and did not show correlation with subgroups of GCB-like DLBCL or ABC-like DLBCL, but p63(+) patients had better disease-free survival (DFS) than those who were negative (p = 0.01). CONCLUSIONS: p63(+) high-intermediate and high risk DLBCL patients have a better DFS than negative cases.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_061',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 316}]}],\n",
       "   'context': 'p63 protein expression in high risk diffuse large B-cell lymphoma. BACKGROUND: p63 gene is a p53 homologue that encodes proteins with transactivation, DNA-binding and tetramerisation domains. The isoforms TAp63 and TAp73 transactivate p53 target genes and induce apoptosis, whereas the isoforms DeltaNp63 and DeltaNp73 lack transactivation and might have dominant-negative effects in p53 family members. p63 is expressed in germinal centre lymphocytes and can be related to the development of the lymphoma, but the prognostic significance of its expression in the survival of patients with diffuse large B-cell lymphoma (DLBCL) remains unclear. AIMS: To determine whether quantitative immunohistochemical (IHC) analysis of p63 protein expression correlates with CD10 antigen, Bcl-6 antigen and IRF4 antigen expression and to determine whether p63 is a surrogate predictor of overall survival in high-intermediate and high risk DLBCL populations. METHODS: CD10, Bcl-6 and IRF4 expression were retrospectively evaluated by IHC in 73 samples of high-intermediate and high risk DLBCL and were used to divide the lymphomas into subgroups of germinal centre B-cell-like (GCB) and activate B-cell-like (ABC) DLBCL. Similarly, p63 expression was evaluated by IHC and the results were compared with subgroups of DLBCL origin and with the survival rates for these patients. RESULTS: p63 was expressed in more than 50% of malignant cells in 11 patients and did not show correlation with subgroups of GCB-like DLBCL or ABC-like DLBCL, but p63(+) patients had better disease-free survival (DFS) than those who were negative (p = 0.01). CONCLUSIONS: p63(+) high-intermediate and high risk DLBCL patients have a better DFS than negative cases.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_062',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 504}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_063',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 515}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_064',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 527}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_065',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 535}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_066',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 548}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_067',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 641}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_068',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 652}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_069',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 667}]}],\n",
       "   'context': \"Patterns of p73 N-terminal isoform expression and p53 status have prognostic value in gynecological cancers. The goal of this study was to determine whether patterns of expression profiles of p73 isoforms and of p53 mutational status are useful combinatorial biomarkers for predicting outcome in a gynecological cancer cohort. This is the first such study using matched tumor/normal tissue pairs from each patient. The median follow-up was over two years. The expression of all 5 N-terminal isoforms (TAp73, DeltaNp73, DeltaN'p73, Ex2p73 and Ex2/3p73) was measured by real-time RT-PCR and p53 status was analyzed by immunohistochemistry. TAp73, DeltaNp73 and DeltaN'p73 were significantly upregulated in tumors. Surprisingly, their range of overexpression was age-dependent, with the highest differences delta (tumor-normal) in the youngest age group. Correction of this age effect was important in further survival correlations. We used all 6 variables (five p73 isoform levels plus p53 status) as input into a principal component analysis with Varimax rotation (VrPCA) to filter out noise from non-disease related individual variability of p73 levels. Rationally selected and individually weighted principal components from each patient were then used to train a support vector machine (SVM) algorithm to predict clinical outcome. This SVM algorithm was able to predict correct outcome in 30 of the 35 patients. We use here a mathematical tool for pattern recognition that has been commonly used in e.g. microarray data mining and apply it for the first time in a prognostic model. We find that PCA/SVM is able to test a clinical hypothesis with robust statistics and show that p73 expression profiles and p53 status are useful prognostic biomarkers that differentiate patients with good vs. poor prognosis with gynecological cancers.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_070',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 634}]}],\n",
       "   'context': 'Calpain cleavage regulates the protein stability of p73. The function of p73, a transcription factor belonging to the p53 family, is finely regulated by its steady-state protein stability. p73 protein degradation/stabilization can be regulated by mechanisms in part dependent on the ubiquitin proteasome system (UPS): (i) Itch/NEDD4-like UPS degradation, (ii) NEDD8 UPS degradation, and (iii) NQO1 20S proteasome-dependent (but ubiquitin-independent) breakdown. Here, we show that, in vitro, Calpain I can cleave p73 at two distinct sites: the first proline-rich region and within the oligomerization domain. Consequently, different p73 isoforms can be degraded by calpains, i.e., both N-terminal isoforms (TAp73 and DeltaNp73) as well as the C-terminal isoforms (alpha, beta, gamma, delta). Moreover, overexpression of the specific endogenous calpain inhibitor, calpastatin, in cultured cells increased the steady-state p73 level. This suggests that calpains may play a physiological role in the regulation of p73 protein stability.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_071',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 710}]}],\n",
       "   'context': 'Calpain cleavage regulates the protein stability of p73. The function of p73, a transcription factor belonging to the p53 family, is finely regulated by its steady-state protein stability. p73 protein degradation/stabilization can be regulated by mechanisms in part dependent on the ubiquitin proteasome system (UPS): (i) Itch/NEDD4-like UPS degradation, (ii) NEDD8 UPS degradation, and (iii) NQO1 20S proteasome-dependent (but ubiquitin-independent) breakdown. Here, we show that, in vitro, Calpain I can cleave p73 at two distinct sites: the first proline-rich region and within the oligomerization domain. Consequently, different p73 isoforms can be degraded by calpains, i.e., both N-terminal isoforms (TAp73 and DeltaNp73) as well as the C-terminal isoforms (alpha, beta, gamma, delta). Moreover, overexpression of the specific endogenous calpain inhibitor, calpastatin, in cultured cells increased the steady-state p73 level. This suggests that calpains may play a physiological role in the regulation of p73 protein stability.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_072',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 724}]}],\n",
       "   'context': 'Calpain cleavage regulates the protein stability of p73. The function of p73, a transcription factor belonging to the p53 family, is finely regulated by its steady-state protein stability. p73 protein degradation/stabilization can be regulated by mechanisms in part dependent on the ubiquitin proteasome system (UPS): (i) Itch/NEDD4-like UPS degradation, (ii) NEDD8 UPS degradation, and (iii) NQO1 20S proteasome-dependent (but ubiquitin-independent) breakdown. Here, we show that, in vitro, Calpain I can cleave p73 at two distinct sites: the first proline-rich region and within the oligomerization domain. Consequently, different p73 isoforms can be degraded by calpains, i.e., both N-terminal isoforms (TAp73 and DeltaNp73) as well as the C-terminal isoforms (alpha, beta, gamma, delta). Moreover, overexpression of the specific endogenous calpain inhibitor, calpastatin, in cultured cells increased the steady-state p73 level. This suggests that calpains may play a physiological role in the regulation of p73 protein stability.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_073',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 356}]}],\n",
       "   'context': \"New antibodies recognizing p73: comparison with commercial antibodies. p73, unlike p53, is expressed as a number of isomeric forms. Alternative splicing at the 3' end of p73 transcript, together with the usage of a second promoter downstream of exon 3, can generate up to 24 p73 isoforms. Variants lacking the TA domain (DeltaN isoforms) are induced by TAp73 and by p53, and inhibit their transcriptional activity. However, understanding the complex biology of p73 has been handicapped by the lack of high affinity specific antibodies for the different isoforms. Here, we report the characterization, by Western blotting and immunoprecipitation, of three new polyclonal antisera recognizing all p73 isoforms, only DeltaN isoforms or only p73alpha, and which have advantages of affinity and specificity over previously available antibodies.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_074',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 572}]}],\n",
       "   'context': 'The p73 gene is an anti-tumoral target of the RARbeta/gamma-selective retinoid tazarotene. Tazarotene, a member of the new class of acetylenic retinoids, has been shown to be effective in the treatment of several hyperproliferative skin diseases, including non-melanoma skin cancer. Its effectiveness is thought to rely on the ability to activate retinoic acid receptors beta and gamma and to induce a number of downstream anti-proliferative genes. Here, we show that the p53-related gene p73 is a target of tazarotene. Indeed, tazarotene modulates the expression of the p73 gene in immortalized keratinocyte cell lines by inducing the pro-apoptotic and anti-proliferative TAp73 isoforms and by repressing the anti-apoptotic and pro-proliferative DeltaNp73 isoforms. This occurs at the transcriptional level through a coordinated action on P1p73 and P2p73 promoters that control the expression of TA and DeltaN isoforms, respectively. The selective downregulation of DeltaNp73 expression by small interfering RNA led to an enhancement of tazarotene-induced bax activation and apoptosis, whereas the downregulation of both TA and DeltaN isoforms impairs tazarotene-mediated apoptosis. These results indicate the relevance of p73 gene products in tazarotene-induced growth inhibition and effectiveness in the treatment of skin tumors.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_075',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 676}]}],\n",
       "   'context': 'The p73 gene is an anti-tumoral target of the RARbeta/gamma-selective retinoid tazarotene. Tazarotene, a member of the new class of acetylenic retinoids, has been shown to be effective in the treatment of several hyperproliferative skin diseases, including non-melanoma skin cancer. Its effectiveness is thought to rely on the ability to activate retinoic acid receptors beta and gamma and to induce a number of downstream anti-proliferative genes. Here, we show that the p53-related gene p73 is a target of tazarotene. Indeed, tazarotene modulates the expression of the p73 gene in immortalized keratinocyte cell lines by inducing the pro-apoptotic and anti-proliferative TAp73 isoforms and by repressing the anti-apoptotic and pro-proliferative DeltaNp73 isoforms. This occurs at the transcriptional level through a coordinated action on P1p73 and P2p73 promoters that control the expression of TA and DeltaN isoforms, respectively. The selective downregulation of DeltaNp73 expression by small interfering RNA led to an enhancement of tazarotene-induced bax activation and apoptosis, whereas the downregulation of both TA and DeltaN isoforms impairs tazarotene-mediated apoptosis. These results indicate the relevance of p73 gene products in tazarotene-induced growth inhibition and effectiveness in the treatment of skin tumors.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_076',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 754}]}],\n",
       "   'context': 'The p73 gene is an anti-tumoral target of the RARbeta/gamma-selective retinoid tazarotene. Tazarotene, a member of the new class of acetylenic retinoids, has been shown to be effective in the treatment of several hyperproliferative skin diseases, including non-melanoma skin cancer. Its effectiveness is thought to rely on the ability to activate retinoic acid receptors beta and gamma and to induce a number of downstream anti-proliferative genes. Here, we show that the p53-related gene p73 is a target of tazarotene. Indeed, tazarotene modulates the expression of the p73 gene in immortalized keratinocyte cell lines by inducing the pro-apoptotic and anti-proliferative TAp73 isoforms and by repressing the anti-apoptotic and pro-proliferative DeltaNp73 isoforms. This occurs at the transcriptional level through a coordinated action on P1p73 and P2p73 promoters that control the expression of TA and DeltaN isoforms, respectively. The selective downregulation of DeltaNp73 expression by small interfering RNA led to an enhancement of tazarotene-induced bax activation and apoptosis, whereas the downregulation of both TA and DeltaN isoforms impairs tazarotene-mediated apoptosis. These results indicate the relevance of p73 gene products in tazarotene-induced growth inhibition and effectiveness in the treatment of skin tumors.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_077',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 101}]}],\n",
       "   'context': 'Differential response of p53 target genes to p73 overexpression in SH-SY5Y neuroblastoma cell line. p73, the first p53 gene homologue, encodes an array of p73 proteins including p73 alpha full-length (TAp73 alpha) and amino-truncated isoforms (Delta Np73 alpha), two proteins with opposite biological functions. TAp73 alpha can induce tumor suppressive properties, while Delta Np73 alpha antagonizes p53 as well as TAp73 in a dominant-negative manner. In human malignant neuroblasts, p53 protein is wild-type but known to be excluded from the nucleus, therefore disabling its function as a tumor suppressor. The present study investigates whether there is a functional link between p73 isoforms and p53 in neuroblastoma. Experiments were performed on two neuroblastoma cell lines differing in their p53 status, e.g. wild-type p53 SH-5Y5Y cells and mutated p53 IGR-N-91 cells. Data indicate that (i) both TA- and Delta N-p73 alpha enhance p53 protein level in SH-SY5Y cells, whereas level remains unchanged in IGR-N-91 cells; (ii) only in SH-SY5Y cells does forced TAp73 alpha overexpression markedly induce nuclear accumulation of p53 protein; (iii) p21 protein expression is increased in both cell lines infected with TAp73, suggesting that, in IGR-N-91 cells, p21 is induced by p73 through a p53-independent pathway; (iv) in the SHSY5Y cell line, Btg2 expression is strongly enhanced in cells overexpressing TA, and to a lesser extent in cells overexpressing Delta N. Taken together our results suggest that TAp73 may restore p53 function in NB with wild-type nonfunctional p53, but not in NB with mutated p53.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_078',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 156}]}],\n",
       "   'context': 'Differential response of p53 target genes to p73 overexpression in SH-SY5Y neuroblastoma cell line. p73, the first p53 gene homologue, encodes an array of p73 proteins including p73 alpha full-length (TAp73 alpha) and amino-truncated isoforms (Delta Np73 alpha), two proteins with opposite biological functions. TAp73 alpha can induce tumor suppressive properties, while Delta Np73 alpha antagonizes p53 as well as TAp73 in a dominant-negative manner. In human malignant neuroblasts, p53 protein is wild-type but known to be excluded from the nucleus, therefore disabling its function as a tumor suppressor. The present study investigates whether there is a functional link between p73 isoforms and p53 in neuroblastoma. Experiments were performed on two neuroblastoma cell lines differing in their p53 status, e.g. wild-type p53 SH-5Y5Y cells and mutated p53 IGR-N-91 cells. Data indicate that (i) both TA- and Delta N-p73 alpha enhance p53 protein level in SH-SY5Y cells, whereas level remains unchanged in IGR-N-91 cells; (ii) only in SH-SY5Y cells does forced TAp73 alpha overexpression markedly induce nuclear accumulation of p53 protein; (iii) p21 protein expression is increased in both cell lines infected with TAp73, suggesting that, in IGR-N-91 cells, p21 is induced by p73 through a p53-independent pathway; (iv) in the SHSY5Y cell line, Btg2 expression is strongly enhanced in cells overexpressing TA, and to a lesser extent in cells overexpressing Delta N. Taken together our results suggest that TAp73 may restore p53 function in NB with wild-type nonfunctional p53, but not in NB with mutated p53.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_079',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 179}]}],\n",
       "   'context': 'Differential response of p53 target genes to p73 overexpression in SH-SY5Y neuroblastoma cell line. p73, the first p53 gene homologue, encodes an array of p73 proteins including p73 alpha full-length (TAp73 alpha) and amino-truncated isoforms (Delta Np73 alpha), two proteins with opposite biological functions. TAp73 alpha can induce tumor suppressive properties, while Delta Np73 alpha antagonizes p53 as well as TAp73 in a dominant-negative manner. In human malignant neuroblasts, p53 protein is wild-type but known to be excluded from the nucleus, therefore disabling its function as a tumor suppressor. The present study investigates whether there is a functional link between p73 isoforms and p53 in neuroblastoma. Experiments were performed on two neuroblastoma cell lines differing in their p53 status, e.g. wild-type p53 SH-5Y5Y cells and mutated p53 IGR-N-91 cells. Data indicate that (i) both TA- and Delta N-p73 alpha enhance p53 protein level in SH-SY5Y cells, whereas level remains unchanged in IGR-N-91 cells; (ii) only in SH-SY5Y cells does forced TAp73 alpha overexpression markedly induce nuclear accumulation of p53 protein; (iii) p21 protein expression is increased in both cell lines infected with TAp73, suggesting that, in IGR-N-91 cells, p21 is induced by p73 through a p53-independent pathway; (iv) in the SHSY5Y cell line, Btg2 expression is strongly enhanced in cells overexpressing TA, and to a lesser extent in cells overexpressing Delta N. Taken together our results suggest that TAp73 may restore p53 function in NB with wild-type nonfunctional p53, but not in NB with mutated p53.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_080',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 204}]}],\n",
       "   'context': 'Differential response of p53 target genes to p73 overexpression in SH-SY5Y neuroblastoma cell line. p73, the first p53 gene homologue, encodes an array of p73 proteins including p73 alpha full-length (TAp73 alpha) and amino-truncated isoforms (Delta Np73 alpha), two proteins with opposite biological functions. TAp73 alpha can induce tumor suppressive properties, while Delta Np73 alpha antagonizes p53 as well as TAp73 in a dominant-negative manner. In human malignant neuroblasts, p53 protein is wild-type but known to be excluded from the nucleus, therefore disabling its function as a tumor suppressor. The present study investigates whether there is a functional link between p73 isoforms and p53 in neuroblastoma. Experiments were performed on two neuroblastoma cell lines differing in their p53 status, e.g. wild-type p53 SH-5Y5Y cells and mutated p53 IGR-N-91 cells. Data indicate that (i) both TA- and Delta N-p73 alpha enhance p53 protein level in SH-SY5Y cells, whereas level remains unchanged in IGR-N-91 cells; (ii) only in SH-SY5Y cells does forced TAp73 alpha overexpression markedly induce nuclear accumulation of p53 protein; (iii) p21 protein expression is increased in both cell lines infected with TAp73, suggesting that, in IGR-N-91 cells, p21 is induced by p73 through a p53-independent pathway; (iv) in the SHSY5Y cell line, Btg2 expression is strongly enhanced in cells overexpressing TA, and to a lesser extent in cells overexpressing Delta N. Taken together our results suggest that TAp73 may restore p53 function in NB with wild-type nonfunctional p53, but not in NB with mutated p53.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_081',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 252}]}],\n",
       "   'context': 'Differential response of p53 target genes to p73 overexpression in SH-SY5Y neuroblastoma cell line. p73, the first p53 gene homologue, encodes an array of p73 proteins including p73 alpha full-length (TAp73 alpha) and amino-truncated isoforms (Delta Np73 alpha), two proteins with opposite biological functions. TAp73 alpha can induce tumor suppressive properties, while Delta Np73 alpha antagonizes p53 as well as TAp73 in a dominant-negative manner. In human malignant neuroblasts, p53 protein is wild-type but known to be excluded from the nucleus, therefore disabling its function as a tumor suppressor. The present study investigates whether there is a functional link between p73 isoforms and p53 in neuroblastoma. Experiments were performed on two neuroblastoma cell lines differing in their p53 status, e.g. wild-type p53 SH-5Y5Y cells and mutated p53 IGR-N-91 cells. Data indicate that (i) both TA- and Delta N-p73 alpha enhance p53 protein level in SH-SY5Y cells, whereas level remains unchanged in IGR-N-91 cells; (ii) only in SH-SY5Y cells does forced TAp73 alpha overexpression markedly induce nuclear accumulation of p53 protein; (iii) p21 protein expression is increased in both cell lines infected with TAp73, suggesting that, in IGR-N-91 cells, p21 is induced by p73 through a p53-independent pathway; (iv) in the SHSY5Y cell line, Btg2 expression is strongly enhanced in cells overexpressing TA, and to a lesser extent in cells overexpressing Delta N. Taken together our results suggest that TAp73 may restore p53 function in NB with wild-type nonfunctional p53, but not in NB with mutated p53.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_082',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 472}]}],\n",
       "   'context': 'deltaNp73 facilitates cell immortalization and cooperates with oncogenic Ras in cellular transformation in vivo. TP73, despite significant homology to TP53, is not a classic tumor suppressor gene, since it exhibits upregulation of nonmutated products in human tumors and lacks a tumor phenotype in p73-deficient mice. We recently reported that an N-terminally truncated isoform, DeltaNp73, is upregulated in breast and gynecological cancers. We further showed that DeltaNp73 is a potent transdominant inhibitor of wild-type p53 and TAp73 in cultured human tumor cells by efficiently counteracting their target gene transactivations, apoptosis, and growth suppression functions (A. I. Zaika et al., J. Exp. Med. 6:765-780, 2002). Although these data strongly suggest oncogenic properties of DeltaNp73, this can only be directly shown in primary cells. We report here that DeltaNp73 confers resistance to spontaneous replicative senescence of primary mouse embryo fibroblasts (MEFs) and immortalizes MEFs at a 1,000-fold-higher frequency than occurs spontaneously. DeltaNp73 cooperates with cMyc and E1A in promoting primary cell proliferation and colony formation and compromises p53-dependent MEF apoptosis. Importantly, DeltaNp73 rescues Ras-induced senescence. Moreover, DeltaNp73 cooperates with oncogenic Ras in transforming primary fibroblasts in vitro and in inducing MEF-derived fibrosarcomas in vivo in nude mice. Wild-type p53 is likely a major target of DeltaNp73 inhibition in primary fibroblasts since deletion of p53 or its requisite upstream activator ARF abrogates the growth-promoting effect of DeltaNp73. Taken together, DeltaNp73 behaves as an oncogene that targets p53 that might explain why DeltaNp73 upregulation may be selected for during tumorigenesis of human cancers.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_083',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 535}]}],\n",
       "   'context': 'deltaNp73 facilitates cell immortalization and cooperates with oncogenic Ras in cellular transformation in vivo. TP73, despite significant homology to TP53, is not a classic tumor suppressor gene, since it exhibits upregulation of nonmutated products in human tumors and lacks a tumor phenotype in p73-deficient mice. We recently reported that an N-terminally truncated isoform, DeltaNp73, is upregulated in breast and gynecological cancers. We further showed that DeltaNp73 is a potent transdominant inhibitor of wild-type p53 and TAp73 in cultured human tumor cells by efficiently counteracting their target gene transactivations, apoptosis, and growth suppression functions (A. I. Zaika et al., J. Exp. Med. 6:765-780, 2002). Although these data strongly suggest oncogenic properties of DeltaNp73, this can only be directly shown in primary cells. We report here that DeltaNp73 confers resistance to spontaneous replicative senescence of primary mouse embryo fibroblasts (MEFs) and immortalizes MEFs at a 1,000-fold-higher frequency than occurs spontaneously. DeltaNp73 cooperates with cMyc and E1A in promoting primary cell proliferation and colony formation and compromises p53-dependent MEF apoptosis. Importantly, DeltaNp73 rescues Ras-induced senescence. Moreover, DeltaNp73 cooperates with oncogenic Ras in transforming primary fibroblasts in vitro and in inducing MEF-derived fibrosarcomas in vivo in nude mice. Wild-type p53 is likely a major target of DeltaNp73 inhibition in primary fibroblasts since deletion of p53 or its requisite upstream activator ARF abrogates the growth-promoting effect of DeltaNp73. Taken together, DeltaNp73 behaves as an oncogene that targets p53 that might explain why DeltaNp73 upregulation may be selected for during tumorigenesis of human cancers.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_084',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 353}]}],\n",
       "   'context': 'Expression of DeltaNp73 is a molecular marker for adverse outcome in neuroblastoma patients. The p73 gene is a p53 homologue which induces apoptosis and inhibits cell proliferation. Although p73 maps at 1p36.3 and is frequently deleted in neuroblastoma (NB), it does not act as a classic oncosuppressor gene. In developing sympathetic neurons of mice, p73 is predominantly expressed as a truncated anti-apoptotic isoform (DeltaNp73), which antagonizes both p53 and the full-length p73 protein (TAp73). This suggests that p73 may be part of a complex tumor-control mechanism. To determine the role of DeltaNp73 in NB we analyzed the pattern of expression of this gene in vivo and evaluated the prognostic significance of its expression. Our results indicate that DeltaNp73 expression is associated with reduced apoptosis in a NB tumor tissue. Expression of this variant in NB patients significantly correlates with age at diagnosis and VMA urinary excretion. Moreover it is strongly associated with reduced survival (HR=7.93; P<0.001) and progression-free survival (HR=5.3; P<0.001) and its role in predicting a poorer outcome is independent from age, primary tumor site, stage and MYCN amplification (OS: HR=5.24, P=0.012; PFS: HR=4.36, P=0.005). In conclusion our data seem to indicate that DeltaNp73 is a crucial gene in neuroblastoma pathogenesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_085',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 429}]}],\n",
       "   'context': 'Expression of DeltaNp73 is a molecular marker for adverse outcome in neuroblastoma patients. The p73 gene is a p53 homologue which induces apoptosis and inhibits cell proliferation. Although p73 maps at 1p36.3 and is frequently deleted in neuroblastoma (NB), it does not act as a classic oncosuppressor gene. In developing sympathetic neurons of mice, p73 is predominantly expressed as a truncated anti-apoptotic isoform (DeltaNp73), which antagonizes both p53 and the full-length p73 protein (TAp73). This suggests that p73 may be part of a complex tumor-control mechanism. To determine the role of DeltaNp73 in NB we analyzed the pattern of expression of this gene in vivo and evaluated the prognostic significance of its expression. Our results indicate that DeltaNp73 expression is associated with reduced apoptosis in a NB tumor tissue. Expression of this variant in NB patients significantly correlates with age at diagnosis and VMA urinary excretion. Moreover it is strongly associated with reduced survival (HR=7.93; P<0.001) and progression-free survival (HR=5.3; P<0.001) and its role in predicting a poorer outcome is independent from age, primary tumor site, stage and MYCN amplification (OS: HR=5.24, P=0.012; PFS: HR=4.36, P=0.005). In conclusion our data seem to indicate that DeltaNp73 is a crucial gene in neuroblastoma pathogenesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_086',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 482}]}],\n",
       "   'context': 'Expression of DeltaNp73 is a molecular marker for adverse outcome in neuroblastoma patients. The p73 gene is a p53 homologue which induces apoptosis and inhibits cell proliferation. Although p73 maps at 1p36.3 and is frequently deleted in neuroblastoma (NB), it does not act as a classic oncosuppressor gene. In developing sympathetic neurons of mice, p73 is predominantly expressed as a truncated anti-apoptotic isoform (DeltaNp73), which antagonizes both p53 and the full-length p73 protein (TAp73). This suggests that p73 may be part of a complex tumor-control mechanism. To determine the role of DeltaNp73 in NB we analyzed the pattern of expression of this gene in vivo and evaluated the prognostic significance of its expression. Our results indicate that DeltaNp73 expression is associated with reduced apoptosis in a NB tumor tissue. Expression of this variant in NB patients significantly correlates with age at diagnosis and VMA urinary excretion. Moreover it is strongly associated with reduced survival (HR=7.93; P<0.001) and progression-free survival (HR=5.3; P<0.001) and its role in predicting a poorer outcome is independent from age, primary tumor site, stage and MYCN amplification (OS: HR=5.24, P=0.012; PFS: HR=4.36, P=0.005). In conclusion our data seem to indicate that DeltaNp73 is a crucial gene in neuroblastoma pathogenesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_087',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 497}]}],\n",
       "   'context': 'Expression of DeltaNp73 is a molecular marker for adverse outcome in neuroblastoma patients. The p73 gene is a p53 homologue which induces apoptosis and inhibits cell proliferation. Although p73 maps at 1p36.3 and is frequently deleted in neuroblastoma (NB), it does not act as a classic oncosuppressor gene. In developing sympathetic neurons of mice, p73 is predominantly expressed as a truncated anti-apoptotic isoform (DeltaNp73), which antagonizes both p53 and the full-length p73 protein (TAp73). This suggests that p73 may be part of a complex tumor-control mechanism. To determine the role of DeltaNp73 in NB we analyzed the pattern of expression of this gene in vivo and evaluated the prognostic significance of its expression. Our results indicate that DeltaNp73 expression is associated with reduced apoptosis in a NB tumor tissue. Expression of this variant in NB patients significantly correlates with age at diagnosis and VMA urinary excretion. Moreover it is strongly associated with reduced survival (HR=7.93; P<0.001) and progression-free survival (HR=5.3; P<0.001) and its role in predicting a poorer outcome is independent from age, primary tumor site, stage and MYCN amplification (OS: HR=5.24, P=0.012; PFS: HR=4.36, P=0.005). In conclusion our data seem to indicate that DeltaNp73 is a crucial gene in neuroblastoma pathogenesis.'},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_088',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1479}]}],\n",
       "   'context': \"Human delta Np73 regulates a dominant negative feedback loop for TAp73 and p53. Inactivation of the tumour suppressor p53 is the most common defect in cancer cells. p53 is a sequence specific transcription factor that is activated in response to various forms of genotoxic stress to induce cell cycle arrest and apoptosis. Induction of p53 is subjected to complex and strict control through several pathways, as it will often determine cellular fate. The p73 protein shares strong structural and functional similarities with p53 such as the potential to activate p53 responsive genes and the ability to induce apoptosis. In addition to alternative splicing at the carboxyl terminus which yields several p73 isoforms, a p73 variant lacking the N-terminal transactivation domain (Delta Np73) was described in mice. In this study, we report the cloning and characterisation of the human Delta Np73 isoforms, their regulation by p53 and their possible role in carcinogenesis. As in mice, human Delta Np73 lacks the transactivation domain and starts with an alternative exon (exon 3'). Its expression is driven by a second promoter located in a genomic region upstream of this exon, supporting the idea of two independently regulated proteins, derived from the same gene. As anticipated, Delta Np73 is capable of regulating TAp73 and p53 function since it is able to block their transactivation activity and their ability to induce apoptosis. Interestingly, expression of the Delta Np73 is strongly up-regulated by the TA isoforms and by p53, thus creating a feedback loop that tightly regulates the function of TAp73 and more importantly of p53. The regulation of Delta Np73 is exerted through a p53 responsive element located on the Delta N promoter. Expression of Delta Np73 not only regulates the function of p53 and TAp73 but also shuts off its own expression, once again finely regulating the whole system. Our data also suggest that increased expression of Delta Np73, functionally inactivating p53, could be involved in tumorogenesis. An extensive analysis of the expression pattern of Delta Np73 in primary tumours would clarify this issue.\"},\n",
       "  {'qas': [{'id': '5173bdb38ed59a060a000020_089',\n",
       "     'question': 'How many TAp73 isoforms have been identified in humans?',\n",
       "     'answers': [{'text': '7', 'answer_start': 1610}]}],\n",
       "   'context': \"Human delta Np73 regulates a dominant negative feedback loop for TAp73 and p53. Inactivation of the tumour suppressor p53 is the most common defect in cancer cells. p53 is a sequence specific transcription factor that is activated in response to various forms of genotoxic stress to induce cell cycle arrest and apoptosis. Induction of p53 is subjected to complex and strict control through several pathways, as it will often determine cellular fate. The p73 protein shares strong structural and functional similarities with p53 such as the potential to activate p53 responsive genes and the ability to induce apoptosis. In addition to alternative splicing at the carboxyl terminus which yields several p73 isoforms, a p73 variant lacking the N-terminal transactivation domain (Delta Np73) was described in mice. In this study, we report the cloning and characterisation of the human Delta Np73 isoforms, their regulation by p53 and their possible role in carcinogenesis. As in mice, human Delta Np73 lacks the transactivation domain and starts with an alternative exon (exon 3'). Its expression is driven by a second promoter located in a genomic region upstream of this exon, supporting the idea of two independently regulated proteins, derived from the same gene. As anticipated, Delta Np73 is capable of regulating TAp73 and p53 function since it is able to block their transactivation activity and their ability to induce apoptosis. Interestingly, expression of the Delta Np73 is strongly up-regulated by the TA isoforms and by p53, thus creating a feedback loop that tightly regulates the function of TAp73 and more importantly of p53. The regulation of Delta Np73 is exerted through a p53 responsive element located on the Delta N promoter. Expression of Delta Np73 not only regulates the function of p53 and TAp73 but also shuts off its own expression, once again finely regulating the whole system. Our data also suggest that increased expression of Delta Np73, functionally inactivating p53, could be involved in tumorogenesis. An extensive analysis of the expression pattern of Delta Np73 in primary tumours would clarify this issue.\"},\n",
       "  {'qas': [{'id': '53442ca9aeec6fbd0700000b_001',\n",
       "     'question': 'Which is the  subcellular localization of ERAP2?',\n",
       "     'answers': [{'text': 'luminal side of the endoplasmic reticulum',\n",
       "       'answer_start': 365}]}],\n",
       "   'context': 'Endoplasmic reticulum aminopeptidases: biochemistry, physiology and pathology. The human endoplasmic reticulum aminopeptidase (ERAP) 1 and 2 proteins were initially identified as homologues of human placental leucine aminopeptidase/insulin-regulated aminopeptidase. They are categorized as a unique class of proteases based on their subcellular localization on the luminal side of the endoplasmic reticulum. ERAPs play an important role in the N-terminal processing of the antigenic precursors that are presented on the major histocompatibility complex (MHC) class I molecules. ERAPs are also implicated in the regulation of a wide variety of physiological phenomena and pathogenic conditions. In this review, the current knowledge on ERAPs is summarized.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_001',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1548}]}],\n",
       "   'context': 'The dopamine transporter is decreased in the striatum of subjects with restless legs syndrome. STUDY OBJECTIVES: Prior studies, all using SPECT techniques, failed to find any differences for dopamine transporter (DAT) in restless legs syndrome (RLS) subjects. The distinct pharmacokinetic properties associated with SPECT-determined DAT along with rapid biodynamic changes in DAT may, however, have missed membrane-bound DAT differences. The current studies assessed real-time DAT binding potentials (BP) in striatum of RLS patients using (11)C-methylphenidate and PET techniques. DESIGN: RLS medications were stopped at least 11 days prior to the PET study. Clinical severity of RLS was also assessed. PET scans were performed at 2 different times of day (starting at 08:30 and 19:30) in separate groups of subjects. The primary outcome measure was total striatal DAT BP. PARTICIPANTS: Thirty-six patients with primary RLS and 34 age- and gender-matched controls. RESULTS: RLS subjects had significantly lower DAT binding in the striatum compared to controls on both the Day and the Night scans. DAT was decreased in putamen and caudate but not the ventral striatum of RLS subjects. There were no diurnal differences in DAT for the total group or for control and RLS separately. DAT BP did not correlate with any clinical measures of RLS. CONCLUSION: The current study found a significant decrease in DAT BP in two independent studies. These results when viewed along with prior RLS SPECT and autopsy studies of DAT, and cell culture studies with iron deficiency and DAT, suggest that membrane-bound striatal DAT, but not total cellular DAT, may be decreased in RLS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_002',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1002}]}],\n",
       "   'context': \"[The clinical research of restless leg syndrome and Parkinson's disease]. OBJECTIVES: To investigate the clinical feature of Parkinson's disease (PD) with restless leg syndrome (RLS) and the pathogenesis of RLS. METHODS: We conducted a cross-sectional and control study. The case group concluded 31 PD with RLS patients, meanwhile 39 PD patients were selected as the control group. Clinical history, clinical manifestations, complications and laboratory examinations were compared respectively between the two groups. RESULTS: All the RLS symptoms did not appear in RLS patients until the PD symptoms came out. Significant differences were found in complications such as swallow disturbance, constipation and illusion, when we compared the two PD groups (P < 0.05). Compared with the PD or healthy group, the level of serum ferritin and the H-reflex latency of tibial nerve were significantly decreased in PD with RLS group (P < 0.05). CONCLUSIONS: Secondary RLS is a complication of PD. Deficiency of iron and decreased inhibition function of spinal cord may lead to the occurrence of RLS in PD patients. When their motor symptoms are serious and complications are more common, PD patients are more possible to have RLS symptoms.\"},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_003',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 585}]}],\n",
       "   'context': \"Iron deficiency: beyond anemia. Iron deficiency is the most common nutritional disorder affecting at least one third of world's population. Though anemia is common manifestation of iron deficiency, other effects of iron deficiency on various tissues, organs and systems are usually under recognized. Impaired brain development and cognitive, behavioural and psychomotor impairment are most worrisome manifestations of iron deficiency. Studies have demonstrated that some of these changes occurring during period of brain growth spurt (<2 years age) may be irreversible. Association of iron deficiency with febrile seizures, pica, breath holding spells, restless leg syndrome and thrombosis is increasingly being recognized. Impaired cell-mediated immunity and bactericidal function are generally noted in iron-deficient persons; however, the findings are inconsistent. Despite proven reversible functional immunological defects in vitro studies, a clinically important relationship between states of iron deficiency and susceptibility to infections remains controversial. Studies from malaria endemic regions have reported increased incidence of malaria in association with iron supplementation. These and some other aspects of iron deficiency are reviewed in this article.\"},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_004',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'Iron', 'answer_start': 889}]}],\n",
       "   'context': 'High incidence of iron depletion and restless leg syndrome (RLS) in regular blood donors: intravenous iron sucrose substitution more effective than oral iron. BACKGROUND AND OBJECTIVES: Iron depletion is common in regular blood donors. The objective of the study was to investigate the frequency and severity of iron depletion in regular blood donors and whether IV iron is more effective than oral to avoid iron depletion and symptoms thereof, especially restless legs syndrome (RLS). METHOD: One hundred and twenty blood donors with at least five previous whole blood donations were randomized to receive either IV iron sucrose (Venofer(®), RenaPharma/Vifor, Uppsala, Sweden), 200 mg, or to 20×100 mg of oral iron sulphate (Duroferon(®), GlaxoSmithKline, Stockholm, Sweden), after each blood donation during 1 year. Iron status and RLS incidence and severity were investigated. RESULTS: Iron status was generally poor among regular blood donors, especially in women, with a high incidence of iron depletion (>20%) and RLS (18%). The IV iron group increased storage iron to a greater extent than the oral iron group after 12 months (P=0·0043). Female donors were more responsive to IV iron sucrose compared to oral iron sulphate, particularly female donors below 50 years of age. RLS severity scores were significantly lower in the IV iron group. The two treatments were safe. CONCLUSION: Iron status is poor in regular blood donors, restless legs syndrome is common, and the routine iron supplementation is insufficient. IV iron sucrose substitutes iron loss in blood donors more efficiently compared with oral iron sulphate, especially in women. Iron substitution to blood donors should be individualized and based on P-ferritin monitoring.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_005',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 994}]}],\n",
       "   'context': 'High incidence of iron depletion and restless leg syndrome (RLS) in regular blood donors: intravenous iron sucrose substitution more effective than oral iron. BACKGROUND AND OBJECTIVES: Iron depletion is common in regular blood donors. The objective of the study was to investigate the frequency and severity of iron depletion in regular blood donors and whether IV iron is more effective than oral to avoid iron depletion and symptoms thereof, especially restless legs syndrome (RLS). METHOD: One hundred and twenty blood donors with at least five previous whole blood donations were randomized to receive either IV iron sucrose (Venofer(®), RenaPharma/Vifor, Uppsala, Sweden), 200 mg, or to 20×100 mg of oral iron sulphate (Duroferon(®), GlaxoSmithKline, Stockholm, Sweden), after each blood donation during 1 year. Iron status and RLS incidence and severity were investigated. RESULTS: Iron status was generally poor among regular blood donors, especially in women, with a high incidence of iron depletion (>20%) and RLS (18%). The IV iron group increased storage iron to a greater extent than the oral iron group after 12 months (P=0·0043). Female donors were more responsive to IV iron sucrose compared to oral iron sulphate, particularly female donors below 50 years of age. RLS severity scores were significantly lower in the IV iron group. The two treatments were safe. CONCLUSION: Iron status is poor in regular blood donors, restless legs syndrome is common, and the routine iron supplementation is insufficient. IV iron sucrose substitutes iron loss in blood donors more efficiently compared with oral iron sulphate, especially in women. Iron substitution to blood donors should be individualized and based on P-ferritin monitoring.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_006',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'Iron', 'answer_start': 1390}]}],\n",
       "   'context': 'High incidence of iron depletion and restless leg syndrome (RLS) in regular blood donors: intravenous iron sucrose substitution more effective than oral iron. BACKGROUND AND OBJECTIVES: Iron depletion is common in regular blood donors. The objective of the study was to investigate the frequency and severity of iron depletion in regular blood donors and whether IV iron is more effective than oral to avoid iron depletion and symptoms thereof, especially restless legs syndrome (RLS). METHOD: One hundred and twenty blood donors with at least five previous whole blood donations were randomized to receive either IV iron sucrose (Venofer(®), RenaPharma/Vifor, Uppsala, Sweden), 200 mg, or to 20×100 mg of oral iron sulphate (Duroferon(®), GlaxoSmithKline, Stockholm, Sweden), after each blood donation during 1 year. Iron status and RLS incidence and severity were investigated. RESULTS: Iron status was generally poor among regular blood donors, especially in women, with a high incidence of iron depletion (>20%) and RLS (18%). The IV iron group increased storage iron to a greater extent than the oral iron group after 12 months (P=0·0043). Female donors were more responsive to IV iron sucrose compared to oral iron sulphate, particularly female donors below 50 years of age. RLS severity scores were significantly lower in the IV iron group. The two treatments were safe. CONCLUSION: Iron status is poor in regular blood donors, restless legs syndrome is common, and the routine iron supplementation is insufficient. IV iron sucrose substitutes iron loss in blood donors more efficiently compared with oral iron sulphate, especially in women. Iron substitution to blood donors should be individualized and based on P-ferritin monitoring.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_007',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1485}]}],\n",
       "   'context': 'High incidence of iron depletion and restless leg syndrome (RLS) in regular blood donors: intravenous iron sucrose substitution more effective than oral iron. BACKGROUND AND OBJECTIVES: Iron depletion is common in regular blood donors. The objective of the study was to investigate the frequency and severity of iron depletion in regular blood donors and whether IV iron is more effective than oral to avoid iron depletion and symptoms thereof, especially restless legs syndrome (RLS). METHOD: One hundred and twenty blood donors with at least five previous whole blood donations were randomized to receive either IV iron sucrose (Venofer(®), RenaPharma/Vifor, Uppsala, Sweden), 200 mg, or to 20×100 mg of oral iron sulphate (Duroferon(®), GlaxoSmithKline, Stockholm, Sweden), after each blood donation during 1 year. Iron status and RLS incidence and severity were investigated. RESULTS: Iron status was generally poor among regular blood donors, especially in women, with a high incidence of iron depletion (>20%) and RLS (18%). The IV iron group increased storage iron to a greater extent than the oral iron group after 12 months (P=0·0043). Female donors were more responsive to IV iron sucrose compared to oral iron sulphate, particularly female donors below 50 years of age. RLS severity scores were significantly lower in the IV iron group. The two treatments were safe. CONCLUSION: Iron status is poor in regular blood donors, restless legs syndrome is common, and the routine iron supplementation is insufficient. IV iron sucrose substitutes iron loss in blood donors more efficiently compared with oral iron sulphate, especially in women. Iron substitution to blood donors should be individualized and based on P-ferritin monitoring.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_008',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1329}]}],\n",
       "   'context': 'Proton pump inhibitors and pain. There may be a relationship between proton pump inhibitors (PPIs) and iron absorption. PPIs may decrease the amount of iron absorbed gastrointestinally specifically due to alteration of the pH in the duodenum. Restless legs syndrome (RLS) is a sensorimotor disorder that includes an urge to move legs, accompanied or caused by uncomfortable and unpleasant sensations in the legs; the urge to move begins or worsens during periods of rest or inactivity, the urge to move is partially or totally relieved by movement, and the urge is worse or only occurs at night. In the majority of the restless leg syndrome population, the sensation is deep seated, often described as being in the shin bones, and most commonly felt between the knee and ankle. It may be described as a creepy, shock-like, tense, electric, buzzing, itchy, or even numb sensation. A subpopulation of this restless leg syndrome patient population experiences restless leg syndrome associated pain (RLSAP) that has been described as a deep \"achy pain.\" This pain has not been found to be relieved by many of the typical over the counter analgesics. Often, constant movement of the legs appears to be the only remedy, as these sensations usually appear during periods of rest. Furthermore, there appears to be an association between iron deficiency and those suffering from Restless Leg Syndrome (RLS). The authors theorize that there may be a possible correlation between PPIs and the symptoms (e.g. pain) associated with RLS. The authors propose that PPIs, such as omeprazole, may interfere with iron absorption in certain patients and that a subpopulation of patients who develop significant iron deficiency characterized by low serum ferritin levels while on PPIs may also develop RLS-like symptoms (including RLSAP). While there is no robust direct evidence to support any associations of PPIs and iron deficiency or PPIs associated with RLS-like symptoms (including RLSAP), it is hoped that this manuscript may spark research efforts on this issue.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_009',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1594}]}],\n",
       "   'context': 'Proton pump inhibitors and pain. There may be a relationship between proton pump inhibitors (PPIs) and iron absorption. PPIs may decrease the amount of iron absorbed gastrointestinally specifically due to alteration of the pH in the duodenum. Restless legs syndrome (RLS) is a sensorimotor disorder that includes an urge to move legs, accompanied or caused by uncomfortable and unpleasant sensations in the legs; the urge to move begins or worsens during periods of rest or inactivity, the urge to move is partially or totally relieved by movement, and the urge is worse or only occurs at night. In the majority of the restless leg syndrome population, the sensation is deep seated, often described as being in the shin bones, and most commonly felt between the knee and ankle. It may be described as a creepy, shock-like, tense, electric, buzzing, itchy, or even numb sensation. A subpopulation of this restless leg syndrome patient population experiences restless leg syndrome associated pain (RLSAP) that has been described as a deep \"achy pain.\" This pain has not been found to be relieved by many of the typical over the counter analgesics. Often, constant movement of the legs appears to be the only remedy, as these sensations usually appear during periods of rest. Furthermore, there appears to be an association between iron deficiency and those suffering from Restless Leg Syndrome (RLS). The authors theorize that there may be a possible correlation between PPIs and the symptoms (e.g. pain) associated with RLS. The authors propose that PPIs, such as omeprazole, may interfere with iron absorption in certain patients and that a subpopulation of patients who develop significant iron deficiency characterized by low serum ferritin levels while on PPIs may also develop RLS-like symptoms (including RLSAP). While there is no robust direct evidence to support any associations of PPIs and iron deficiency or PPIs associated with RLS-like symptoms (including RLSAP), it is hoped that this manuscript may spark research efforts on this issue.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_010',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1691}]}],\n",
       "   'context': 'Proton pump inhibitors and pain. There may be a relationship between proton pump inhibitors (PPIs) and iron absorption. PPIs may decrease the amount of iron absorbed gastrointestinally specifically due to alteration of the pH in the duodenum. Restless legs syndrome (RLS) is a sensorimotor disorder that includes an urge to move legs, accompanied or caused by uncomfortable and unpleasant sensations in the legs; the urge to move begins or worsens during periods of rest or inactivity, the urge to move is partially or totally relieved by movement, and the urge is worse or only occurs at night. In the majority of the restless leg syndrome population, the sensation is deep seated, often described as being in the shin bones, and most commonly felt between the knee and ankle. It may be described as a creepy, shock-like, tense, electric, buzzing, itchy, or even numb sensation. A subpopulation of this restless leg syndrome patient population experiences restless leg syndrome associated pain (RLSAP) that has been described as a deep \"achy pain.\" This pain has not been found to be relieved by many of the typical over the counter analgesics. Often, constant movement of the legs appears to be the only remedy, as these sensations usually appear during periods of rest. Furthermore, there appears to be an association between iron deficiency and those suffering from Restless Leg Syndrome (RLS). The authors theorize that there may be a possible correlation between PPIs and the symptoms (e.g. pain) associated with RLS. The authors propose that PPIs, such as omeprazole, may interfere with iron absorption in certain patients and that a subpopulation of patients who develop significant iron deficiency characterized by low serum ferritin levels while on PPIs may also develop RLS-like symptoms (including RLSAP). While there is no robust direct evidence to support any associations of PPIs and iron deficiency or PPIs associated with RLS-like symptoms (including RLSAP), it is hoped that this manuscript may spark research efforts on this issue.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_011',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 296}]}],\n",
       "   'context': 'Altered dopaminergic profile in the putamen and substantia nigra in restless leg syndrome. Restless leg syndrome (RLS) is a sensorimotor disorder. Clinical studies have implicated the dopaminergic system in RLS, while others have suggested that it is associated with insufficient levels of brain iron. To date, alterations in brain iron status have been demonstrated but, despite suggestions from the clinical literature, there have been no consistent findings documenting a dopaminergic abnormality in RLS brain tissue. In this study, the substantia nigra and putamen were obtained at autopsy from individuals with primary RLS and a neurologically normal control group. A quantitative profile of the dopaminergic system was obtained. Additional assays were performed on a catecholaminergic cell line and animal models of iron deficiency. RLS tissue, compared with controls, showed a significant decrease in D2R in the putamen that correlated with severity of the RLS. RLS also showed significant increases in tyrosine hydroxylase (TH) in the substantia nigra, compared with the controls, but not in the putamen. Both TH and phosphorylated (active) TH were significantly increased in both the substantia nigra and putamen. There were no significant differences in either the putamen or nigra for dopamine receptor 1, dopamine transporters or for VMAT. Significant increases in TH and phosphorylated TH were also seen in both the animal and cell models of iron insufficiency similar to that from the RLS autopsy data. For the first time, a clear indication of dopamine pathology in RLS is revealed in this autopsy study. The results suggest cellular regulation of dopamine production that closely matches the data from cellular and animal iron insufficiency models. The results are consistent with the hypothesis that a primary iron insufficiency produces a dopaminergic abnormality characterized as an overly activated dopaminergic system as part of the RLS pathology.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_012',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1827}]}],\n",
       "   'context': 'Altered dopaminergic profile in the putamen and substantia nigra in restless leg syndrome. Restless leg syndrome (RLS) is a sensorimotor disorder. Clinical studies have implicated the dopaminergic system in RLS, while others have suggested that it is associated with insufficient levels of brain iron. To date, alterations in brain iron status have been demonstrated but, despite suggestions from the clinical literature, there have been no consistent findings documenting a dopaminergic abnormality in RLS brain tissue. In this study, the substantia nigra and putamen were obtained at autopsy from individuals with primary RLS and a neurologically normal control group. A quantitative profile of the dopaminergic system was obtained. Additional assays were performed on a catecholaminergic cell line and animal models of iron deficiency. RLS tissue, compared with controls, showed a significant decrease in D2R in the putamen that correlated with severity of the RLS. RLS also showed significant increases in tyrosine hydroxylase (TH) in the substantia nigra, compared with the controls, but not in the putamen. Both TH and phosphorylated (active) TH were significantly increased in both the substantia nigra and putamen. There were no significant differences in either the putamen or nigra for dopamine receptor 1, dopamine transporters or for VMAT. Significant increases in TH and phosphorylated TH were also seen in both the animal and cell models of iron insufficiency similar to that from the RLS autopsy data. For the first time, a clear indication of dopamine pathology in RLS is revealed in this autopsy study. The results suggest cellular regulation of dopamine production that closely matches the data from cellular and animal iron insufficiency models. The results are consistent with the hypothesis that a primary iron insufficiency produces a dopaminergic abnormality characterized as an overly activated dopaminergic system as part of the RLS pathology.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_013',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 573}]}],\n",
       "   'context': 'Restless leg syndrome: is it a real problem? Restless legs syndrome (RLS) is a common condition that is frequently unrecognized, misdiagnosed and poorly managed. It is characterized by uncomfortable sensations deep in the legs developing at rest that compel the person to move; symptoms are worst at night and sleep disturbance is common. RLS occurs in 7%-11% of the population in Western countries, and many such people experience troublesome symptoms. Primary RLS is familial in up to two thirds of patients. RLS may also be secondary to a number of conditions including iron deficiency, pregnancy and end-stage renal failure and, perhaps, neuropathy. Secondary RLS is most common in those presenting for the first time in later life. The pathogenesis of RLS probably involves the interplay of systemic or brain iron deficiency and impaired dopaminergic neurotransmission in the subcortex of the brain. RLS is very responsive to dopaminergic therapies. Rebound of RLS symptoms during the early morning and development of severe symptoms earlier in the day (augmentation) are problematic in those treated for a prolonged period with levodopa. Consequently, dopamine agonists have become first line treatment. Anti-convulsant medications and opioids are helpful in some patients. Correction of underlying problem wherever possible is important in the management of secondary RLS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_014',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 814}]}],\n",
       "   'context': 'Restless leg syndrome: is it a real problem? Restless legs syndrome (RLS) is a common condition that is frequently unrecognized, misdiagnosed and poorly managed. It is characterized by uncomfortable sensations deep in the legs developing at rest that compel the person to move; symptoms are worst at night and sleep disturbance is common. RLS occurs in 7%-11% of the population in Western countries, and many such people experience troublesome symptoms. Primary RLS is familial in up to two thirds of patients. RLS may also be secondary to a number of conditions including iron deficiency, pregnancy and end-stage renal failure and, perhaps, neuropathy. Secondary RLS is most common in those presenting for the first time in later life. The pathogenesis of RLS probably involves the interplay of systemic or brain iron deficiency and impaired dopaminergic neurotransmission in the subcortex of the brain. RLS is very responsive to dopaminergic therapies. Rebound of RLS symptoms during the early morning and development of severe symptoms earlier in the day (augmentation) are problematic in those treated for a prolonged period with levodopa. Consequently, dopamine agonists have become first line treatment. Anti-convulsant medications and opioids are helpful in some patients. Correction of underlying problem wherever possible is important in the management of secondary RLS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_015',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1004}]}],\n",
       "   'context': '[Restless-leg syndrome--possible unrecognized cause for insomnia and irritability in children]. Restless legs syndrome (RLS) has gradually been recognized as a cause for insomnia in adults, but there have been few reports about children with RLS in Japan. Here we described seven pediatric RLS patients. All of the parents of our patients had difficult times to make their children sleep due to irritability, restlessness, and demanding bedtime routines. All patients had asked their parents to rub their feet in bed, and it took more than half an hour to soothe them until they fell asleep. Their mothers had been exhausted from this night-time routine. However, they did not consider the routine abnormal, as it had been their habitual behavior since infancy. Some parents were too distressed or embarrassed to describe the symptoms of their child properly. Five patients had clear family history and none had obvious periodic leg movements during sleep. All patients showed low levels of ferritin and iron supplementation was effective in five cases. In the severest two cases, pramipexole, but not iron, was dramatically effective. Both patients started to show RLS symptoms in the early days of infancy, which may suggest more severe hereditary dopaminergic dysfunction. RLS does occur in childhood and pediatricians should bear it in mind as one of the differential diagnoses when seeing children who are irritated and/or having difficulty in initiating their sleep.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_016',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 524}]}],\n",
       "   'context': 'Nonhematological benefits of iron. Iron deficiency anemia is common in people with chronic kidney disease (CKD) and its importance in supporting erythropoiesis is unquestioned especially in those patients treated with erythropoietin. Clinical symptomatology such as fatigability, cold intolerance, failure to concentrate and poor effort intolerance is often attributed to anemia or uremia. That iron deficiency, per se, can cause these symptoms is poorly recognized. Clinical and animal studies that support the benefits of iron supplementation, independent of increasing hemoglobin, such as those on immune function, physical performance, thermoregulation, cognition, and restless leg syndrome and aluminum absorption is the subject of this narrative review.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_017',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 258}]}],\n",
       "   'context': 'Restless leg syndrome manifested by iron deficiency from chronic hemoptysis in cystic fibrosis. Restless leg syndrome (RLS) and periodic limb movement disorder (PLMD) are considered to be a continuum of a neurological sleep disorder associated with abnormal iron metabolism or deficiency. I describe a case of RLS and PLMD in a cystic fibrosis patient with iron deficiency from chronic hemoptysis. This is the first case that reports RLS and PLMD manifesting from iron deficiency caused by chronic hemoptysis in advanced cystic fibrosis lung disease.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_018',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 357}]}],\n",
       "   'context': 'Restless leg syndrome manifested by iron deficiency from chronic hemoptysis in cystic fibrosis. Restless leg syndrome (RLS) and periodic limb movement disorder (PLMD) are considered to be a continuum of a neurological sleep disorder associated with abnormal iron metabolism or deficiency. I describe a case of RLS and PLMD in a cystic fibrosis patient with iron deficiency from chronic hemoptysis. This is the first case that reports RLS and PLMD manifesting from iron deficiency caused by chronic hemoptysis in advanced cystic fibrosis lung disease.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_019',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 464}]}],\n",
       "   'context': 'Restless leg syndrome manifested by iron deficiency from chronic hemoptysis in cystic fibrosis. Restless leg syndrome (RLS) and periodic limb movement disorder (PLMD) are considered to be a continuum of a neurological sleep disorder associated with abnormal iron metabolism or deficiency. I describe a case of RLS and PLMD in a cystic fibrosis patient with iron deficiency from chronic hemoptysis. This is the first case that reports RLS and PLMD manifesting from iron deficiency caused by chronic hemoptysis in advanced cystic fibrosis lung disease.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_020',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 36}]}],\n",
       "   'context': 'Restless leg syndrome manifested by iron deficiency from chronic hemoptysis in cystic fibrosis. Restless leg syndrome (RLS) and periodic limb movement disorder (PLMD) are considered to be a continuum of a neurological sleep disorder associated with abnormal iron metabolism or deficiency. I describe a case of RLS and PLMD in a cystic fibrosis patient with iron deficiency from chronic hemoptysis. This is the first case that reports RLS and PLMD manifesting from iron deficiency caused by chronic hemoptysis in advanced cystic fibrosis lung disease.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_021',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 202}]}],\n",
       "   'context': 'Acoustic startle response is disrupted in iron-deficient rats. Diurnal effects on motor control are evident in the human disease of Restless Leg Syndrome (RLS), which is purported to be linked to brain iron deficiency as well as alterations in dopaminergic systems. Thus, we explored the relationship between daily rhythms, the onset of motor dysregulation and brain iron deficiency in an animal model of iron deficiency. Male and female weanling Sprague-Dawley rats consuming control (CN) or iron-deficient (ID) diets were examined weekly for acoustic startle response (ASR) and prepulse inhibition (PPI) for a 5-week period. Iron deficiency reduced the magnitude, but not timing, of the ASR at specific time points. ASR was elevated 60% at the onset of the dark cycle relative to the median of the light cycle in male CN and ID rats. The respective elevation was 400% and 150% in female CN and ID rats during the first 2 weeks of testing. The diurnal cycle of ASR response was attenuated by 3 weeks of testing in both dietary treatment groups. PPI was not affected by iron deficiency, sex, diurnal cycle or the interaction between these factors. These results thus demonstrate that iron deficiency moderately alters ASR signaling although the inhibitory pathways of ASR do not appear to be affected.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_022',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'Iron', 'answer_start': 795}]}],\n",
       "   'context': 'Iron, brain and restless legs syndrome. Iron is the most important transitional metal in the body, as it is implicated in many metabolic processes, mostly related to its capacity as an electron donor/acceptor. Iron deficiency has been long been known to cause anaemia, iron excess to cause haemochromatosis. As excess free iron can cause oxidative damage, it is important that the levels of iron in the body are tightly regulated which appears to be done only by digestive absorption, as there is no known regulating mechanism for elimination of iron. The amount of free iron is also kept to a minimum thanks to binding to transferrin for transport, and to ferritin for storage. Recent research has put emphasis on the possible role of excess iron in the brain in several degenerative diseases. Iron deficiency in the central nervous system is known to cause motor impairment and cognitive deficits; more recently, it has been suggested that it may play a role in the pathophysiology of the restless leg syndrome. 2001 Harcourt Publishers Ltd'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_023',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 405}]}],\n",
       "   'context': 'Periodic limb movement disorder : a clinical and polysomnographic study. Periodic limb movement disorder (PLMD) is one of the commonest neurological disorders and causes significant disability, if left untreated. However, it is rarely diagnosed in clinical practice, probably due to lack of awareness and/or lack of necessary diagnostic facilities. Restless leg syndrome (RLS), aging, pregnancy, uraemia, iron deficiency, polyneuropathy are some of the common causes of secondary PLMD. Clinical presentation, polysomnographic findings and management of six patients of PLMD have been discussed in this report.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_024',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 626}]}],\n",
       "   'context': '[Restless leg syndrome. Diagnosis and treatment]. INTRODUCTION: The restless legs syndrome is characterized by an unpleasant sensation in the legs which causes an imperative need to move the legs and is therefore considered to be a disorder of movement. When it appears before going to sleep, it may interfere with falling asleep and lead to a sleep-deficit. DEVELOPMENT AND CONCLUSIONS: It is a clinical condition with a satisfactory treatment, and improvement of the associated sleep disorder. The etiology is unknown, sometimes it is familial. The syndrome is increasingly often diagnosed, particularly in association with iron deficiency, during pregnancy, in chronic renal failure and in patients with peripheral neuropathy. Polysomnography is not necessary, unless one suspects an associated disorder of periodic leg movements. Treatment is by dopaminergic, opiate, benzodiazepine, anticonvulsant drugs or clonidine.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c000012_025',\n",
       "     'question': 'Which deficiency is the cause of restless leg syndrome?',\n",
       "     'answers': [{'text': 'iron', 'answer_start': 1435}]}],\n",
       "   'context': 'A genetic risk factor for low serum ferritin levels in Danish blood donors. BACKGROUND: Iron deficiency is a frequent side effect of blood donation. In recent years, several studies have described genetic variants associated with iron concentrations. However, the impact of these variants on iron levels is unknown in blood donors. Knowledge of genetic variants that predispose donors to iron deficiency would allow bleeding frequency and iron supplementation to be tailored to the individual donor. STUDY DESIGN AND METHODS: The genotypes of five specific single-nucleotide polymorphisms (SNPs) in three genes that have been previously associated with iron status and/or restless leg syndrome (RLS) were investigated in two groups of female blood donors. The first group had low iron stores (serum ferritin < 12 µg/L, n = 657), and the second group had normal to high iron stores (serum ferritin > 30 µg/L, n = 645). Genotype distribution for each of the SNPs was compared between the two groups. RESULTS: Homozygosity for the T-allele of BTBD9 rs9296249 was associated with lower serum ferritin. The odds ratio for low serum ferritin was 1.35 (95% confidence interval, 1.02-1.77; p = 0.03) when comparing donors with the TT genotype with donors with the CT genotype. CONCLUSION: A frequent polymorphism in BTBD9 was significantly associated with serum ferritin. This polymorphism has previously been associated with RLS, but not low iron stores in blood donors.'},\n",
       "  {'qas': [{'id': '53398855d6d3ac6a3400005b_001',\n",
       "     'question': 'What histone modification is recognized by the bromodomain?',\n",
       "     'answers': [{'text': 'acetylated lysines', 'answer_start': 232}]}],\n",
       "   'context': '3,5-dimethylisoxazoles act as acetyl-lysine-mimetic bromodomain ligands. Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription. Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code. Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties. With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance. We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains. Using X-ray crystallographic analysis, we have determined the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains. By exploiting these interactions, we have developed compound 4d, which has IC(50) values of <5 μM for the bromodomain-containing proteins BRD2(1) and BRD4(1). These compounds are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.'},\n",
       "  {'qas': [{'id': '53398855d6d3ac6a3400005b_002',\n",
       "     'question': 'What histone modification is recognized by the bromodomain?',\n",
       "     'answers': [{'text': 'acetylated lysines', 'answer_start': 116}]}],\n",
       "   'context': 'Bromodomain coactivators in cancer, obesity, type 2 diabetes, and inflammation. Double bromodomain proteins bind to acetylated lysines in histones, bringing associated histone modification and nucleosome remodeling activity to chromatin. The ability of bromodomain regulators to alter chromatin status and control gene expression has long been appreciated to be important in the development of certain human cancers. However, bromodomain proteins have now been found also to be critical, non-redundant players in diverse, non-malignant phenotypes, directing transcriptional programs that control adipogenesis, energy metabolism and inflammation. The fact that such different processes are functionally linked by the same molecular machinery suggests a common epigenetic basis to understand and interpret the origins of several important co-morbidities, such as asthma or cancer that occurs in obesity, and complex inflammatory diseases like cardiovascular disease, systemic lupus erythematosus, rheumatoid arthritis and insulin resistance that may be built on a common pro-inflammatory foundation.'},\n",
       "  {'qas': [{'id': '53398855d6d3ac6a3400005b_003',\n",
       "     'question': 'What histone modification is recognized by the bromodomain?',\n",
       "     'answers': [{'text': 'acetylated lysines', 'answer_start': 768}]}],\n",
       "   'context': 'The Gcn5 bromodomain co-ordinates nucleosome remodelling. The access of transcription factors to eukaryotic promoters often requires modification of their chromatin structure, which is accomplished by the action of two general classes of multiprotein complexes. One class contains histone acetyltransferases (HATs), such as Gcn5 in the SAGA complex, which acetylate nucleosomal histones. The second class contains ATPases, such as Swi2 in the Swi/Snf complex, which provide the energy for nucleosome remodelling. In several promoters these two complexes cooperate but their functional linkage is unknown. A protein module that is present in all nuclear HATs, the bromodomain, could provide such a link. The recently reported in vitro binding of a HAT bromodomain with acetylated lysines within H3 and H4 amino-terminal peptides indicates that this interaction may constitute a targeting step for events that follow histone acetylation. Here we use a suitable promoter to show that bromodomain residues essential for acetyl-lysine binding are not required in vivo for Gcn5-mediated histone acetylation but are fundamental for the subsequent Swi2-dependent nucleosome remodelling and consequent transcriptional activation. We show that the Gcn5 bromodomain stabilizes the Swi/Snf complex on this promoter.'},\n",
       "  {'qas': [{'id': '56d1da3b67f0cb3d66000006_001',\n",
       "     'question': 'How many periods of regulatory innovation led to the evolution of vertebrates?',\n",
       "     'answers': [{'text': 'Three', 'answer_start': 0}]}],\n",
       "   'context': 'Three periods of regulatory innovation during vertebrate evolution. The gain, loss, and modification of gene regulatory elements may underlie a substantial proportion of phenotypic changes on animal lineages. To investigate the gain of regulatory elements throughout vertebrate evolution, we identified genome-wide sets of putative regulatory regions for five vertebrates, including humans. These putative regulatory regions are conserved nonexonic elements (CNEEs), which are evolutionarily conserved yet do not overlap any coding or noncoding mature transcript. We then inferred the branch on which each CNEE came under selective constraint. Our analysis identified three extended periods in the evolution of gene regulatory elements. Early vertebrate evolution was characterized by regulatory gains near transcription factors and developmental genes, but this trend was replaced by innovations near extracellular signaling genes, and then innovations near posttranslational protein modifiers.'},\n",
       "  {'qas': [{'id': '56d1da3b67f0cb3d66000006_002',\n",
       "     'question': 'How many periods of regulatory innovation led to the evolution of vertebrates?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 668}]}],\n",
       "   'context': 'Three periods of regulatory innovation during vertebrate evolution. The gain, loss, and modification of gene regulatory elements may underlie a substantial proportion of phenotypic changes on animal lineages. To investigate the gain of regulatory elements throughout vertebrate evolution, we identified genome-wide sets of putative regulatory regions for five vertebrates, including humans. These putative regulatory regions are conserved nonexonic elements (CNEEs), which are evolutionarily conserved yet do not overlap any coding or noncoding mature transcript. We then inferred the branch on which each CNEE came under selective constraint. Our analysis identified three extended periods in the evolution of gene regulatory elements. Early vertebrate evolution was characterized by regulatory gains near transcription factors and developmental genes, but this trend was replaced by innovations near extracellular signaling genes, and then innovations near posttranslational protein modifiers.'},\n",
       "  {'qas': [{'id': '56d1da3b67f0cb3d66000006_003',\n",
       "     'question': 'How many periods of regulatory innovation led to the evolution of vertebrates?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 668}]}],\n",
       "   'context': 'Three periods of regulatory innovation during vertebrate evolution. The gain, loss, and modification of gene regulatory elements may underlie a substantial proportion of phenotypic changes on animal lineages. To investigate the gain of regulatory elements throughout vertebrate evolution, we identified genome-wide sets of putative regulatory regions for five vertebrates, including humans. These putative regulatory regions are conserved nonexonic elements (CNEEs), which are evolutionarily conserved yet do not overlap any coding or noncoding mature transcript. We then inferred the branch on which each CNEE came under selective constraint. Our analysis identified three extended periods in the evolution of gene regulatory elements. Early vertebrate evolution was characterized by regulatory gains near transcription factors and developmental genes, but this trend was replaced by innovations near extracellular signaling genes, and then innovations near posttranslational protein modifiers.'},\n",
       "  {'qas': [{'id': '56d1da3b67f0cb3d66000006_004',\n",
       "     'question': 'How many periods of regulatory innovation led to the evolution of vertebrates?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 668}]}],\n",
       "   'context': 'Three periods of regulatory innovation during vertebrate evolution. The gain, loss, and modification of gene regulatory elements may underlie a substantial proportion of phenotypic changes on animal lineages. To investigate the gain of regulatory elements throughout vertebrate evolution, we identified genome-wide sets of putative regulatory regions for five vertebrates, including humans. These putative regulatory regions are conserved nonexonic elements (CNEEs), which are evolutionarily conserved yet do not overlap any coding or noncoding mature transcript. We then inferred the branch on which each CNEE came under selective constraint. Our analysis identified three extended periods in the evolution of gene regulatory elements. Early vertebrate evolution was characterized by regulatory gains near transcription factors and developmental genes, but this trend was replaced by innovations near extracellular signaling genes, and then innovations near posttranslational protein modifiers.'},\n",
       "  {'qas': [{'id': '56d1da3b67f0cb3d66000006_005',\n",
       "     'question': 'How many periods of regulatory innovation led to the evolution of vertebrates?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 668}]}],\n",
       "   'context': 'Three periods of regulatory innovation during vertebrate evolution. The gain, loss, and modification of gene regulatory elements may underlie a substantial proportion of phenotypic changes on animal lineages. To investigate the gain of regulatory elements throughout vertebrate evolution, we identified genome-wide sets of putative regulatory regions for five vertebrates, including humans. These putative regulatory regions are conserved nonexonic elements (CNEEs), which are evolutionarily conserved yet do not overlap any coding or noncoding mature transcript. We then inferred the branch on which each CNEE came under selective constraint. Our analysis identified three extended periods in the evolution of gene regulatory elements. Early vertebrate evolution was characterized by regulatory gains near transcription factors and developmental genes, but this trend was replaced by innovations near extracellular signaling genes, and then innovations near posttranslational protein modifiers.'},\n",
       "  {'qas': [{'id': '56d1da3b67f0cb3d66000006_006',\n",
       "     'question': 'How many periods of regulatory innovation led to the evolution of vertebrates?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 668}]}],\n",
       "   'context': 'Three periods of regulatory innovation during vertebrate evolution. The gain, loss, and modification of gene regulatory elements may underlie a substantial proportion of phenotypic changes on animal lineages. To investigate the gain of regulatory elements throughout vertebrate evolution, we identified genome-wide sets of putative regulatory regions for five vertebrates, including humans. These putative regulatory regions are conserved nonexonic elements (CNEEs), which are evolutionarily conserved yet do not overlap any coding or noncoding mature transcript. We then inferred the branch on which each CNEE came under selective constraint. Our analysis identified three extended periods in the evolution of gene regulatory elements. Early vertebrate evolution was characterized by regulatory gains near transcription factors and developmental genes, but this trend was replaced by innovations near extracellular signaling genes, and then innovations near posttranslational protein modifiers.'},\n",
       "  {'qas': [{'id': '56d1da3b67f0cb3d66000006_007',\n",
       "     'question': 'How many periods of regulatory innovation led to the evolution of vertebrates?',\n",
       "     'answers': [{'text': 'three', 'answer_start': 668}]}],\n",
       "   'context': 'Three periods of regulatory innovation during vertebrate evolution. The gain, loss, and modification of gene regulatory elements may underlie a substantial proportion of phenotypic changes on animal lineages. To investigate the gain of regulatory elements throughout vertebrate evolution, we identified genome-wide sets of putative regulatory regions for five vertebrates, including humans. These putative regulatory regions are conserved nonexonic elements (CNEEs), which are evolutionarily conserved yet do not overlap any coding or noncoding mature transcript. We then inferred the branch on which each CNEE came under selective constraint. Our analysis identified three extended periods in the evolution of gene regulatory elements. Early vertebrate evolution was characterized by regulatory gains near transcription factors and developmental genes, but this trend was replaced by innovations near extracellular signaling genes, and then innovations near posttranslational protein modifiers.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_002',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 137}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_003',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 204}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_004',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 437}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_005',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 469}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_006',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 553}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_007',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'Thyroid', 'answer_start': 614}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_008',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 831}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_009',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 881}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_010',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 914}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_011',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'Thyroid', 'answer_start': 1044}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_012',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1120}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_013',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1224}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_014',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1309}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_015',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1352}]}],\n",
       "   'context': 'Thyroid hypoplasia as a cause of congenital hypothyroidism in Williams syndrome. In the Williams-Beuren syndrome (WBS), disorders of the thyroid function and morphology have been reported and programs of thyroid screening and surveillance are recommended. However, the frequency of biochemical thyroid assessment, particularly in the first year of life, is being debated. In this report we describe an infant with WBS and congenital hypothyroidism, due to an important thyroid hypoplasia. The patient, a 1-month-old female, negative at primary neonatal thyroid screening, was referred to our hospital for dyspnea. Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentration (10.21 pmol/l; normal range: 10.29-24.45 pmol/l). Ultrasound examination of the neck showed a significant thyroid hypoplasia, whereas (99m)Tc-pertechnetate thyroid scintigraphy evidenced a thyroid gland in normal position, with reduced shape and overall weak fixation. Therefore, treatment with L-thyroxinewas started. Thyroid hypoplasia is a frequent characteristic of WBS and abnormalities of thyroid function are common in patients with this feature. Therefore, the possibility of congenital hypothyroidism should always be taken into consideration too and, even if congenital hypothyroidism neonatal screening is negative, thyroid (morphology and function) evaluation should be regularly assessed when the diagnosis is made and, thereafter, every year in the first years of life.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_016',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 669}]}],\n",
       "   'context': 'Thyroid morphology and subclinical hypothyroidism in children and adolescents with Williams syndrome. OBJECTIVE: To verify the prevalence of morpho-volumetric and functional thyroid abnormalities in young patients with Williams syndrome (WS). STUDY DESIGN: Ninety-two patients with WS (49 boys and 43 girls, 0.2-17.2 years of age) underwent evaluation of thyroid function by means of thyroid-stimulating hormone (TSH), fT3, and fT4 measurement. Thyroid ultrasonography was performed in 37 patients. Thyroid antibodies (thyroid peroxidase and thyroglobulin) were measured in all patients with abnormal thyroid function tests. RESULTS: None of our patients had overt hypothyroidism; 29 patients (31.5%) had subclinical hypothyroidism. Thyroid antibodies were absent in all patients. The prevalence of patients with subclinical hypothyroidism was significantly higher in the younger patients. Ultrasonography revealed morphological or volumetric abnormalities of the thyroid gland in 67.5% of patients; these abnormalities were more frequently observed in the older children. CONCLUSIONS: Subclinical hypothyroidism is a frequent but stable finding in young children with WS. The great majority of patients with WS >10 years, either with normal or hypoplastic thyroid, have normal thyroid function. Therefore, we suggest yearly monitoring of thyroid function and sonographic studies at least once in patients with WS. Treatment should be reserved for the patients with overt hypothyroidism or for those whose thyroid function shows signs of progressive deterioration.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_017',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 721}]}],\n",
       "   'context': 'Thyroid morphology and subclinical hypothyroidism in children and adolescents with Williams syndrome. OBJECTIVE: To verify the prevalence of morpho-volumetric and functional thyroid abnormalities in young patients with Williams syndrome (WS). STUDY DESIGN: Ninety-two patients with WS (49 boys and 43 girls, 0.2-17.2 years of age) underwent evaluation of thyroid function by means of thyroid-stimulating hormone (TSH), fT3, and fT4 measurement. Thyroid ultrasonography was performed in 37 patients. Thyroid antibodies (thyroid peroxidase and thyroglobulin) were measured in all patients with abnormal thyroid function tests. RESULTS: None of our patients had overt hypothyroidism; 29 patients (31.5%) had subclinical hypothyroidism. Thyroid antibodies were absent in all patients. The prevalence of patients with subclinical hypothyroidism was significantly higher in the younger patients. Ultrasonography revealed morphological or volumetric abnormalities of the thyroid gland in 67.5% of patients; these abnormalities were more frequently observed in the older children. CONCLUSIONS: Subclinical hypothyroidism is a frequent but stable finding in young children with WS. The great majority of patients with WS >10 years, either with normal or hypoplastic thyroid, have normal thyroid function. Therefore, we suggest yearly monitoring of thyroid function and sonographic studies at least once in patients with WS. Treatment should be reserved for the patients with overt hypothyroidism or for those whose thyroid function shows signs of progressive deterioration.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_018',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'Thyroid', 'answer_start': 70}]}],\n",
       "   'context': 'Thyroid anomalies in Williams syndrome: investigation of 95 patients. Thyroid involvement in Williams syndrome (WS) was recently reported in two small groups of patients, both showing an increased prevalence of elevation of TSH serum concentration; in one of the two reports, 70% of the patients demonstrated a hypoplasia of thyroid gland as well. In our institution, we currently follow a large population of WS patients who periodically undergo a multispecialist clinical evaluation that includes ultrasound evaluation of the thyroid gland, and levels of FT3, FT4, TSH, and anti-thyroid antibodies. Here, we report on the prevalence of thyroid structural and functional anomalies, in a population of 95 WS patients, half of them followed for more than 5 years. Our study confirms the increased incidence of both elevated TSH serum values (37.9% in our sample) and thyroid gland hypoplasia (74.7%). Moreover, we demonstrated that TSH elevation declines with age. For this reason, we suggest that a complete thyroid evaluation be performed in every patient with WS, and that this medical complication should be periodically searched for in follow-up visits.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_019',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 325}]}],\n",
       "   'context': 'Thyroid anomalies in Williams syndrome: investigation of 95 patients. Thyroid involvement in Williams syndrome (WS) was recently reported in two small groups of patients, both showing an increased prevalence of elevation of TSH serum concentration; in one of the two reports, 70% of the patients demonstrated a hypoplasia of thyroid gland as well. In our institution, we currently follow a large population of WS patients who periodically undergo a multispecialist clinical evaluation that includes ultrasound evaluation of the thyroid gland, and levels of FT3, FT4, TSH, and anti-thyroid antibodies. Here, we report on the prevalence of thyroid structural and functional anomalies, in a population of 95 WS patients, half of them followed for more than 5 years. Our study confirms the increased incidence of both elevated TSH serum values (37.9% in our sample) and thyroid gland hypoplasia (74.7%). Moreover, we demonstrated that TSH elevation declines with age. For this reason, we suggest that a complete thyroid evaluation be performed in every patient with WS, and that this medical complication should be periodically searched for in follow-up visits.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_020',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 866}]}],\n",
       "   'context': 'Thyroid anomalies in Williams syndrome: investigation of 95 patients. Thyroid involvement in Williams syndrome (WS) was recently reported in two small groups of patients, both showing an increased prevalence of elevation of TSH serum concentration; in one of the two reports, 70% of the patients demonstrated a hypoplasia of thyroid gland as well. In our institution, we currently follow a large population of WS patients who periodically undergo a multispecialist clinical evaluation that includes ultrasound evaluation of the thyroid gland, and levels of FT3, FT4, TSH, and anti-thyroid antibodies. Here, we report on the prevalence of thyroid structural and functional anomalies, in a population of 95 WS patients, half of them followed for more than 5 years. Our study confirms the increased incidence of both elevated TSH serum values (37.9% in our sample) and thyroid gland hypoplasia (74.7%). Moreover, we demonstrated that TSH elevation declines with age. For this reason, we suggest that a complete thyroid evaluation be performed in every patient with WS, and that this medical complication should be periodically searched for in follow-up visits.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_021',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1307}]}],\n",
       "   'context': 'Thyroid function and morphology in patients affected by Williams syndrome. OBJECTIVE: To evaluate the prevalence of abnormalities of thyroid function and morphology in a cohort of patients with Williams syndrome (WS). METHODS: Serum concentrations of free-T3, free-T4, TSH, thyroperoxidase antibodies (TPOA) and thyroglobulin antibodies (TgA), as well as ultrasonographic data, of 20 patients with WS (12 females and eight males), aged 1.7-34.9 years, were evaluated. RESULTS: Three cases (15%) of subclinical hypothyroidism were identified. Overt hypothyroidism was diagnosed in two cases (10%). Thyroid antibodies were negative in all patients. Fourteen patients (70%) showed thyroid hypoplasia involving the entire gland. In these patients, the left thyroid lobe appeared usually, but not significantly, reduced compared with the right thyroid lobe. One patient (5%) showed thyroid hemiagenesis. Only five patients (25%) showed a thyroid with normal volume, and of these five, one patient showed marked thyroid hypoplasia of the left lobe. In all WS patients with diagnosis of subclinical or overt hypothyroidism, thyroid hypoplasia was detected. No cases of subclinical or overt hypothyroidism were found in WS with normal thyroid volume. CONCLUSIONS: This study confirms the presence of alterations of thyroid function in WS and also suggests the frequent occurrence of abnormalities of thyroid morphology in these patients. Patients with WS should be monitored for thyroid function and a thyroid ultrasound screening should be considered, especially in those patients with changes in thyroid function.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_022',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1392}]}],\n",
       "   'context': 'Thyroid function and morphology in patients affected by Williams syndrome. OBJECTIVE: To evaluate the prevalence of abnormalities of thyroid function and morphology in a cohort of patients with Williams syndrome (WS). METHODS: Serum concentrations of free-T3, free-T4, TSH, thyroperoxidase antibodies (TPOA) and thyroglobulin antibodies (TgA), as well as ultrasonographic data, of 20 patients with WS (12 females and eight males), aged 1.7-34.9 years, were evaluated. RESULTS: Three cases (15%) of subclinical hypothyroidism were identified. Overt hypothyroidism was diagnosed in two cases (10%). Thyroid antibodies were negative in all patients. Fourteen patients (70%) showed thyroid hypoplasia involving the entire gland. In these patients, the left thyroid lobe appeared usually, but not significantly, reduced compared with the right thyroid lobe. One patient (5%) showed thyroid hemiagenesis. Only five patients (25%) showed a thyroid with normal volume, and of these five, one patient showed marked thyroid hypoplasia of the left lobe. In all WS patients with diagnosis of subclinical or overt hypothyroidism, thyroid hypoplasia was detected. No cases of subclinical or overt hypothyroidism were found in WS with normal thyroid volume. CONCLUSIONS: This study confirms the presence of alterations of thyroid function in WS and also suggests the frequent occurrence of abnormalities of thyroid morphology in these patients. Patients with WS should be monitored for thyroid function and a thyroid ultrasound screening should be considered, especially in those patients with changes in thyroid function.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_023',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 751}]}],\n",
       "   'context': 'New case of thyroid dysgenesis and clinical signs of hypothyroidism in Williams syndrome. The authors report a female presenting with congenital heart defects, liver hemangiomas, and facial dysmorphisms admitted to hospital at 3 months of age because of feeding difficulties and poor growth. She had hypotonia and large tongue, \"coarse\" face, and umbilical hernia in presence of complex congenital cardiovascular malformations. In spite of normal neonatal screening we performed serum levels of thyroid hormones. Thyrotropin level was very high (>50 microU/ml; normal value 0.2-4 microU/ml), while serum free T(3) (FT3) and free T(4) (FT4) levels were normal (FT3 3.6 pg/ml, normal value 2.8-5.6 pg/ml; FT4 11.6 pg/ml, normal value 6.6-14 pg/ml); antithyroid autoantibodies were absent. Thyroid scintigraphy with sodium 99m Tc pertechnetate showed a small ectopic thyroid located in sublingual position, so treatment with L-thyroxine 37.5 microg/24 hr was started with rapid improvement of the clinical picture. At 17 months of age the patient developed the complete characteristic phenotype of Williams syndrome (WS); the clinical diagnosis was proven by fluorescent in situ hybridization (FISH) analysis which showed hemizygous deletion of the elastin gene on chromosome 7. Recently a case of thyroid hemiagenesis in a child with WS has been reported; our patient underscores the association of hypothyroidism and WS. Moreover, our case shows that clinical manifestations of hypothyroidism may be present and the treatment may be necessary as it is in isolated congenital hypothyroidism.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_024',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1295}]}],\n",
       "   'context': 'New case of thyroid dysgenesis and clinical signs of hypothyroidism in Williams syndrome. The authors report a female presenting with congenital heart defects, liver hemangiomas, and facial dysmorphisms admitted to hospital at 3 months of age because of feeding difficulties and poor growth. She had hypotonia and large tongue, \"coarse\" face, and umbilical hernia in presence of complex congenital cardiovascular malformations. In spite of normal neonatal screening we performed serum levels of thyroid hormones. Thyrotropin level was very high (>50 microU/ml; normal value 0.2-4 microU/ml), while serum free T(3) (FT3) and free T(4) (FT4) levels were normal (FT3 3.6 pg/ml, normal value 2.8-5.6 pg/ml; FT4 11.6 pg/ml, normal value 6.6-14 pg/ml); antithyroid autoantibodies were absent. Thyroid scintigraphy with sodium 99m Tc pertechnetate showed a small ectopic thyroid located in sublingual position, so treatment with L-thyroxine 37.5 microg/24 hr was started with rapid improvement of the clinical picture. At 17 months of age the patient developed the complete characteristic phenotype of Williams syndrome (WS); the clinical diagnosis was proven by fluorescent in situ hybridization (FISH) analysis which showed hemizygous deletion of the elastin gene on chromosome 7. Recently a case of thyroid hemiagenesis in a child with WS has been reported; our patient underscores the association of hypothyroidism and WS. Moreover, our case shows that clinical manifestations of hypothyroidism may be present and the treatment may be necessary as it is in isolated congenital hypothyroidism.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_025',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1401}]}],\n",
       "   'context': 'New case of thyroid dysgenesis and clinical signs of hypothyroidism in Williams syndrome. The authors report a female presenting with congenital heart defects, liver hemangiomas, and facial dysmorphisms admitted to hospital at 3 months of age because of feeding difficulties and poor growth. She had hypotonia and large tongue, \"coarse\" face, and umbilical hernia in presence of complex congenital cardiovascular malformations. In spite of normal neonatal screening we performed serum levels of thyroid hormones. Thyrotropin level was very high (>50 microU/ml; normal value 0.2-4 microU/ml), while serum free T(3) (FT3) and free T(4) (FT4) levels were normal (FT3 3.6 pg/ml, normal value 2.8-5.6 pg/ml; FT4 11.6 pg/ml, normal value 6.6-14 pg/ml); antithyroid autoantibodies were absent. Thyroid scintigraphy with sodium 99m Tc pertechnetate showed a small ectopic thyroid located in sublingual position, so treatment with L-thyroxine 37.5 microg/24 hr was started with rapid improvement of the clinical picture. At 17 months of age the patient developed the complete characteristic phenotype of Williams syndrome (WS); the clinical diagnosis was proven by fluorescent in situ hybridization (FISH) analysis which showed hemizygous deletion of the elastin gene on chromosome 7. Recently a case of thyroid hemiagenesis in a child with WS has been reported; our patient underscores the association of hypothyroidism and WS. Moreover, our case shows that clinical manifestations of hypothyroidism may be present and the treatment may be necessary as it is in isolated congenital hypothyroidism.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_026',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'Thyroid', 'answer_start': 0}]}],\n",
       "   'context': 'Thyroid hemiagenesis and elevated thyrotropin levels in a child with Williams syndrome. A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies. Thyroid ultrasonography and scintigraphy showed hemiagenesis of the left lobe and no evidence of ectopic tissue. TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2]. These abnormalities are similar to those seen in patients with hypothalamic hypothyroidism. Thyroid function is not a recognized manifestation of WS and is not routinely investigated. However, abnormalities of the hypothalamic-pituitary-thyroid (HPT) axis and thyroid dysgenesis have been found in other WS cases. Genes mapping at 7q11.23, contiguous to the chromosomal region deleted in most WS patients, may be involved in the development of the thyroid gland, contributing to the complex phenotype of WS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_027',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 201}]}],\n",
       "   'context': 'Thyroid hemiagenesis and elevated thyrotropin levels in a child with Williams syndrome. A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies. Thyroid ultrasonography and scintigraphy showed hemiagenesis of the left lobe and no evidence of ectopic tissue. TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2]. These abnormalities are similar to those seen in patients with hypothalamic hypothyroidism. Thyroid function is not a recognized manifestation of WS and is not routinely investigated. However, abnormalities of the hypothalamic-pituitary-thyroid (HPT) axis and thyroid dysgenesis have been found in other WS cases. Genes mapping at 7q11.23, contiguous to the chromosomal region deleted in most WS patients, may be involved in the development of the thyroid gland, contributing to the complex phenotype of WS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_028',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 248}]}],\n",
       "   'context': 'Thyroid hemiagenesis and elevated thyrotropin levels in a child with Williams syndrome. A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies. Thyroid ultrasonography and scintigraphy showed hemiagenesis of the left lobe and no evidence of ectopic tissue. TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2]. These abnormalities are similar to those seen in patients with hypothalamic hypothyroidism. Thyroid function is not a recognized manifestation of WS and is not routinely investigated. However, abnormalities of the hypothalamic-pituitary-thyroid (HPT) axis and thyroid dysgenesis have been found in other WS cases. Genes mapping at 7q11.23, contiguous to the chromosomal region deleted in most WS patients, may be involved in the development of the thyroid gland, contributing to the complex phenotype of WS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_029',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 527}]}],\n",
       "   'context': 'Thyroid hemiagenesis and elevated thyrotropin levels in a child with Williams syndrome. A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies. Thyroid ultrasonography and scintigraphy showed hemiagenesis of the left lobe and no evidence of ectopic tissue. TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2]. These abnormalities are similar to those seen in patients with hypothalamic hypothyroidism. Thyroid function is not a recognized manifestation of WS and is not routinely investigated. However, abnormalities of the hypothalamic-pituitary-thyroid (HPT) axis and thyroid dysgenesis have been found in other WS cases. Genes mapping at 7q11.23, contiguous to the chromosomal region deleted in most WS patients, may be involved in the development of the thyroid gland, contributing to the complex phenotype of WS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_030',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 784}]}],\n",
       "   'context': 'Thyroid hemiagenesis and elevated thyrotropin levels in a child with Williams syndrome. A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies. Thyroid ultrasonography and scintigraphy showed hemiagenesis of the left lobe and no evidence of ectopic tissue. TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2]. These abnormalities are similar to those seen in patients with hypothalamic hypothyroidism. Thyroid function is not a recognized manifestation of WS and is not routinely investigated. However, abnormalities of the hypothalamic-pituitary-thyroid (HPT) axis and thyroid dysgenesis have been found in other WS cases. Genes mapping at 7q11.23, contiguous to the chromosomal region deleted in most WS patients, may be involved in the development of the thyroid gland, contributing to the complex phenotype of WS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_031',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 941}]}],\n",
       "   'context': 'Thyroid hemiagenesis and elevated thyrotropin levels in a child with Williams syndrome. A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies. Thyroid ultrasonography and scintigraphy showed hemiagenesis of the left lobe and no evidence of ectopic tissue. TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2]. These abnormalities are similar to those seen in patients with hypothalamic hypothyroidism. Thyroid function is not a recognized manifestation of WS and is not routinely investigated. However, abnormalities of the hypothalamic-pituitary-thyroid (HPT) axis and thyroid dysgenesis have been found in other WS cases. Genes mapping at 7q11.23, contiguous to the chromosomal region deleted in most WS patients, may be involved in the development of the thyroid gland, contributing to the complex phenotype of WS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_032',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 964}]}],\n",
       "   'context': 'Thyroid hemiagenesis and elevated thyrotropin levels in a child with Williams syndrome. A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies. Thyroid ultrasonography and scintigraphy showed hemiagenesis of the left lobe and no evidence of ectopic tissue. TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2]. These abnormalities are similar to those seen in patients with hypothalamic hypothyroidism. Thyroid function is not a recognized manifestation of WS and is not routinely investigated. However, abnormalities of the hypothalamic-pituitary-thyroid (HPT) axis and thyroid dysgenesis have been found in other WS cases. Genes mapping at 7q11.23, contiguous to the chromosomal region deleted in most WS patients, may be involved in the development of the thyroid gland, contributing to the complex phenotype of WS.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_033',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1102}]}],\n",
       "   'context': 'Thyroid morphology and subclinical hypothyroidism in children and adolescents with Williams syndrome. OBJECTIVE: To verify the prevalence of morpho-volumetric and functional thyroid abnormalities in young patients with Williams syndrome (WS). STUDY DESIGN: Ninety-two patients with WS (49 boys and 43 girls, 0.2-17.2 years of age) underwent evaluation of thyroid function by means of thyroid-stimulating hormone (TSH), fT3, and fT4 measurement. Thyroid ultrasonography was performed in 37 patients. Thyroid antibodies (thyroid peroxidase and thyroglobulin) were measured in all patients with abnormal thyroid function tests. RESULTS: None of our patients had overt hypothyroidism; 29 patients (31.5%) had subclinical hypothyroidism. Thyroid antibodies were absent in all patients. The prevalence of patients with subclinical hypothyroidism was significantly higher in the younger patients. Ultrasonography revealed morphological or volumetric abnormalities of the thyroid gland in 67.5% of patients; these abnormalities were more frequently observed in the older children. CONCLUSIONS: Subclinical hypothyroidism is a frequent but stable finding in young children with WS. The great majority of patients with WS >10 years, either with normal or hypoplastic thyroid, have normal thyroid function. Therefore, we suggest yearly monitoring of thyroid function and sonographic studies at least once in patients with WS. Treatment should be reserved for the patients with overt hypothyroidism or for those whose thyroid function shows signs of progressive deterioration.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_034',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1257}]}],\n",
       "   'context': 'Thyroid morphology and subclinical hypothyroidism in children and adolescents with Williams syndrome. OBJECTIVE: To verify the prevalence of morpho-volumetric and functional thyroid abnormalities in young patients with Williams syndrome (WS). STUDY DESIGN: Ninety-two patients with WS (49 boys and 43 girls, 0.2-17.2 years of age) underwent evaluation of thyroid function by means of thyroid-stimulating hormone (TSH), fT3, and fT4 measurement. Thyroid ultrasonography was performed in 37 patients. Thyroid antibodies (thyroid peroxidase and thyroglobulin) were measured in all patients with abnormal thyroid function tests. RESULTS: None of our patients had overt hypothyroidism; 29 patients (31.5%) had subclinical hypothyroidism. Thyroid antibodies were absent in all patients. The prevalence of patients with subclinical hypothyroidism was significantly higher in the younger patients. Ultrasonography revealed morphological or volumetric abnormalities of the thyroid gland in 67.5% of patients; these abnormalities were more frequently observed in the older children. CONCLUSIONS: Subclinical hypothyroidism is a frequent but stable finding in young children with WS. The great majority of patients with WS >10 years, either with normal or hypoplastic thyroid, have normal thyroid function. Therefore, we suggest yearly monitoring of thyroid function and sonographic studies at least once in patients with WS. Treatment should be reserved for the patients with overt hypothyroidism or for those whose thyroid function shows signs of progressive deterioration.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_035',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 1278}]}],\n",
       "   'context': 'Thyroid morphology and subclinical hypothyroidism in children and adolescents with Williams syndrome. OBJECTIVE: To verify the prevalence of morpho-volumetric and functional thyroid abnormalities in young patients with Williams syndrome (WS). STUDY DESIGN: Ninety-two patients with WS (49 boys and 43 girls, 0.2-17.2 years of age) underwent evaluation of thyroid function by means of thyroid-stimulating hormone (TSH), fT3, and fT4 measurement. Thyroid ultrasonography was performed in 37 patients. Thyroid antibodies (thyroid peroxidase and thyroglobulin) were measured in all patients with abnormal thyroid function tests. RESULTS: None of our patients had overt hypothyroidism; 29 patients (31.5%) had subclinical hypothyroidism. Thyroid antibodies were absent in all patients. The prevalence of patients with subclinical hypothyroidism was significantly higher in the younger patients. Ultrasonography revealed morphological or volumetric abnormalities of the thyroid gland in 67.5% of patients; these abnormalities were more frequently observed in the older children. CONCLUSIONS: Subclinical hypothyroidism is a frequent but stable finding in young children with WS. The great majority of patients with WS >10 years, either with normal or hypoplastic thyroid, have normal thyroid function. Therefore, we suggest yearly monitoring of thyroid function and sonographic studies at least once in patients with WS. Treatment should be reserved for the patients with overt hypothyroidism or for those whose thyroid function shows signs of progressive deterioration.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_036',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 514}]}],\n",
       "   'context': 'Thyroid function and morphology in patients affected by Williams syndrome. OBJECTIVE: To evaluate the prevalence of abnormalities of thyroid function and morphology in a cohort of patients with Williams syndrome (WS). METHODS: Serum concentrations of free-T3, free-T4, TSH, thyroperoxidase antibodies (TPOA) and thyroglobulin antibodies (TgA), as well as ultrasonographic data, of 20 patients with WS (12 females and eight males), aged 1.7-34.9 years, were evaluated. RESULTS: Three cases (15%) of subclinical hypothyroidism were identified. Overt hypothyroidism was diagnosed in two cases (10%). Thyroid antibodies were negative in all patients. Fourteen patients (70%) showed thyroid hypoplasia involving the entire gland. In these patients, the left thyroid lobe appeared usually, but not significantly, reduced compared with the right thyroid lobe. One patient (5%) showed thyroid hemiagenesis. Only five patients (25%) showed a thyroid with normal volume, and of these five, one patient showed marked thyroid hypoplasia of the left lobe. In all WS patients with diagnosis of subclinical or overt hypothyroidism, thyroid hypoplasia was detected. No cases of subclinical or overt hypothyroidism were found in WS with normal thyroid volume. CONCLUSIONS: This study confirms the presence of alterations of thyroid function in WS and also suggests the frequent occurrence of abnormalities of thyroid morphology in these patients. Patients with WS should be monitored for thyroid function and a thyroid ultrasound screening should be considered, especially in those patients with changes in thyroid function.'},\n",
       "  {'qas': [{'id': '530cefaaad0bf1360c00000d_037',\n",
       "     'question': 'Which hormone abnormalities are common in Williams syndrome ?',\n",
       "     'answers': [{'text': 'thyroid', 'answer_start': 552}]}],\n",
       "   'context': 'Thyroid function and morphology in patients affected by Williams syndrome. OBJECTIVE: To evaluate the prevalence of abnormalities of thyroid function and morphology in a cohort of patients with Williams syndrome (WS). METHODS: Serum concentrations of free-T3, free-T4, TSH, thyroperoxidase antibodies (TPOA) and thyroglobulin antibodies (TgA), as well as ultrasonographic data, of 20 patients with WS (12 females and eight males), aged 1.7-34.9 years, were evaluated. RESULTS: Three cases (15%) of subclinical hypothyroidism were identified. Overt hypothyroidism was diagnosed in two cases (10%). Thyroid antibodies were negative in all patients. Fourteen patients (70%) showed thyroid hypoplasia involving the entire gland. In these patients, the left thyroid lobe appeared usually, but not significantly, reduced compared with the right thyroid lobe. One patient (5%) showed thyroid hemiagenesis. Only five patients (25%) showed a thyroid with normal volume, and of these five, one patient showed marked thyroid hypoplasia of the left lobe. In all WS patients with diagnosis of subclinical or overt hypothyroidism, thyroid hypoplasia was detected. No cases of subclinical or overt hypothyroidism were found in WS with normal thyroid volume. CONCLUSIONS: This study confirms the presence of alterations of thyroid function in WS and also suggests the frequent occurrence of abnormalities of thyroid morphology in these patients. Patients with WS should be monitored for thyroid function and a thyroid ultrasound screening should be considered, especially in those patients with changes in thyroid function.'},\n",
       "  {'qas': [{'id': '550c4011a103b78016000009_001',\n",
       "     'question': 'Which residue of alpha-synuclein was found to be phosphorylated in Lewy bodies?',\n",
       "     'answers': [{'text': 'serine 129', 'answer_start': 183}]}],\n",
       "   'context': \"Ser129D mutant alpha-synuclein induces earlier motor dysfunction while S129A results in distinctive pathology in a rat model of Parkinson's disease. Alpha-synuclein phosphorylated at serine 129 (S129) is highly elevated in Parkinson's disease patients where it mainly accumulates in the Lewy bodies. Several groups have studied the role of phosphorylation at the S129 in α-synuclein in a rat model for Parkinson's disease using recombinant adeno-associated viral (rAAV) vectors. The results obtained are inconsistent and accordingly the role of S129 phosphorylation in α-synuclein toxicity remains unclear. This prompted us to re-examine the neuropathological and behavioral effects of the S129 modified α-synuclein species in vivo. For this purpose, we used two mutated forms of human α-synuclein in which the S129 was replaced either with an alanine (S129A), to block phosphorylation, or with an aspartate (S129D), to mimic phosphorylation, and compared them with the wild type α-synuclein. This approach was similar in design to previous studies, however our investigation of dopaminergic degeneration also included performing a detailed study of the α-synuclein induced pathology in the striatum and the analysis of motor deficits. Our results showed that overexpressing S129D or wild type α-synuclein resulted in an accelerated dopaminergic fiber loss as compared with S129A α-synuclein. Furthermore, the motor deficit seen in the group treated with the mutant S129D α-synuclein appeared earlier than the other two forms of α-synuclein. Conversely, S129A α-synuclein showed significantly larger pathological α-synuclein-positive inclusions, and slower dopaminergic fiber loss, when compared to the other two forms of α-synuclein, suggesting a neuroprotective effect of the mutation. When examined at long-term, all three α-synuclein forms resulted in pathological accumulations of α-synuclein in striatal fibers and dopaminergic cell death in the substantia nigra. Our data show that changes in the S129 residue of α-synuclein influence the rate of pathology and neurodegeneration, with an overall deleterious effect of exchanging S129 to a residue mimicking its phosphorylated state.\"},\n",
       "  {'qas': [{'id': '550c4011a103b78016000009_002',\n",
       "     'question': 'Which residue of alpha-synuclein was found to be phosphorylated in Lewy bodies?',\n",
       "     'answers': [{'text': 'serine 129', 'answer_start': 490}]}],\n",
       "   'context': \"The role of Ser129 phosphorylation of α-synuclein in neurodegeneration of Parkinson's disease: a review of in vivo models. Parkinson's disease is the most common neurodegenerative movement disorder. The motor impairments of Parkinson's disease are caused by the loss of dopaminergic neurons in the substantia nigra and associated with the appearance of fibrillar aggregates of α-synuclein (α-syn) called Lewy bodies. Approximately 90% of α-syn deposited in Lewy bodies is phosphorylated at serine 129 (Ser129). In contrast, only 4% or less of total α-syn is phosphorylated at this residue in the normal brain. This suggests that the accumulation of Ser129-phosphorylated α-syn leads to the formation of Lewy bodies and dopaminergic neurodegeneration in Parkinson's disease. Our laboratory and others have performed experiments using in vivo models of Parkinson's disease to elucidate the role of increased Ser129 phosphorylation in α-syn neurotoxicity. However, there has been a lack of consistency among these models. In this review, we summarize the main findings regarding the relationship between Ser129 phosphorylation and α-syn neurotoxicity, and examine the differences among models. We further discuss the role of Ser129 phosphorylation in α-syn aggregation and the future directions to test the potential of Ser129 phosphorylation as a therapeutic target for slowing the progression of Parkinson's disease.\"},\n",
       "  {'qas': [{'id': '550c4011a103b78016000009_003',\n",
       "     'question': 'Which residue of alpha-synuclein was found to be phosphorylated in Lewy bodies?',\n",
       "     'answers': [{'text': 'serine 129', 'answer_start': 160}]}],\n",
       "   'context': 'Alpha-synuclein phosphorylation controls neurotoxicity and inclusion formation in a Drosophila model of Parkinson disease. Alpha-synuclein is phosphorylated at serine 129 (Ser129) in intracellular protein aggregates called Lewy bodies. These inclusion bodies are the characteristic pathologic lesions of Parkinson disease. Here we define the role of phosphorylation of Ser129 in alpha-synuclein toxicity and inclusion formation using a Drosophila model of Parkinson disease. Mutation of Ser129 to alanine to prevent phosphorylation completely suppresses dopaminergic neuronal loss produced by expression of human alpha-synuclein. In contrast, altering Ser129 to the negatively charged residue aspartate, to mimic phosphorylation, significantly enhances alpha-synuclein toxicity. The G protein-coupled receptor kinase 2 (Gprk2) phosphorylates Ser129 in vivo and enhances alpha-synuclein toxicity. Blocking phosphorylation at Ser129 substantially increases aggregate formation. Thus Ser129 phosphorylation status is crucial in mediating alpha-synuclein neurotoxicity and inclusion formation. Because increased number of inclusion bodies correlates with reduced toxicity, inclusion bodies may protect neurons from alpha-synuclein toxicity.'},\n",
       "  {'qas': [{'id': '550c4011a103b78016000009_004',\n",
       "     'question': 'Which residue of alpha-synuclein was found to be phosphorylated in Lewy bodies?',\n",
       "     'answers': [{'text': 'serine 129', 'answer_start': 160}]}],\n",
       "   'context': 'Alpha-synuclein phosphorylation controls neurotoxicity and inclusion formation in a Drosophila model of Parkinson disease. Alpha-synuclein is phosphorylated at serine 129 (Ser129) in intracellular protein aggregates called Lewy bodies. These inclusion bodies are the characteristic pathologic lesions of Parkinson disease. Here we define the role of phosphorylation of Ser129 in alpha-synuclein toxicity and inclusion formation using a Drosophila model of Parkinson disease. Mutation of Ser129 to alanine to prevent phosphorylation completely suppresses dopaminergic neuronal loss produced by expression of human alpha-synuclein. In contrast, altering Ser129 to the negatively charged residue aspartate, to mimic phosphorylation, significantly enhances alpha-synuclein toxicity. The G protein-coupled receptor kinase 2 (Gprk2) phosphorylates Ser129 in vivo and enhances alpha-synuclein toxicity. Blocking phosphorylation at Ser129 substantially increases aggregate formation. Thus Ser129 phosphorylation status is crucial in mediating alpha-synuclein neurotoxicity and inclusion formation. Because increased number of inclusion bodies correlates with reduced toxicity, inclusion bodies may protect neurons from alpha-synuclein toxicity.'},\n",
       "  {'qas': [{'id': '550c4011a103b78016000009_005',\n",
       "     'question': 'Which residue of alpha-synuclein was found to be phosphorylated in Lewy bodies?',\n",
       "     'answers': [{'text': 'serine 129', 'answer_start': 160}]}],\n",
       "   'context': 'Alpha-synuclein phosphorylation controls neurotoxicity and inclusion formation in a Drosophila model of Parkinson disease. Alpha-synuclein is phosphorylated at serine 129 (Ser129) in intracellular protein aggregates called Lewy bodies. These inclusion bodies are the characteristic pathologic lesions of Parkinson disease. Here we define the role of phosphorylation of Ser129 in alpha-synuclein toxicity and inclusion formation using a Drosophila model of Parkinson disease. Mutation of Ser129 to alanine to prevent phosphorylation completely suppresses dopaminergic neuronal loss produced by expression of human alpha-synuclein. In contrast, altering Ser129 to the negatively charged residue aspartate, to mimic phosphorylation, significantly enhances alpha-synuclein toxicity. The G protein-coupled receptor kinase 2 (Gprk2) phosphorylates Ser129 in vivo and enhances alpha-synuclein toxicity. Blocking phosphorylation at Ser129 substantially increases aggregate formation. Thus Ser129 phosphorylation status is crucial in mediating alpha-synuclein neurotoxicity and inclusion formation. Because increased number of inclusion bodies correlates with reduced toxicity, inclusion bodies may protect neurons from alpha-synuclein toxicity.'},\n",
       "  {'qas': [{'id': '56bc9268ac7ad1001900001b_001',\n",
       "     'question': 'What molecule is targeted by brodalumab?',\n",
       "     'answers': [{'text': 'interleukin-17', 'answer_start': 127}]}],\n",
       "   'context': \"Phase 3 Studies Comparing Brodalumab with Ustekinumab in Psoriasis. BACKGROUND: Early clinical studies suggested that the anti-interleukin-17 receptor A monoclonal antibody brodalumab has efficacy in the treatment of psoriasis. METHODS: In two phase 3 studies (AMAGINE-2 and AMAGINE-3), patients with moderate-to-severe psoriasis were randomly assigned to receive brodalumab (210 mg or 140 mg every 2 weeks), ustekinumab (45 mg for patients with a body weight < 100 kg and 90 mg for patients >100 kg), or placebo. At week 12, patients receiving brodalumab were randomly assigned again to receive a brodalumab maintenance dose of 210 mg every 2 weeks or 140 mg every 2 weeks, every 4 weeks, or every 8 weeks; patients receiving ustekinumab continued to receive ustekinumab every 12 weeks, and patients receiving placebo received 210 mg of brodalumab every 2 weeks. The primary aims were to evaluate the superiority of brodalumab over placebo at week 12 with respect to at least a 75% reduction in the psoriasis area-and-severity index score (PASI 75) and a static physician's global assessment (sPGA) score of 0 or 1 (clear or almost clear skin), as well as the superiority of brodalumab over ustekinumab at week 12 with respect to a 100% reduction in PASI score (PASI 100). RESULTS: At week 12, the PASI 75 response rates were higher with brodalumab at the 210-mg and 140-mg doses than with placebo (86% and 67%, respectively, vs. 8% [AMAGINE-2] and 85% and 69%, respectively, vs. 6% [AMAGINE-3]; P<0.001); the rates of sPGA scores of 0 or 1 were also higher with brodalumab (P<0.001). The week 12 PASI 100 response rates were significantly higher with 210 mg of brodalumab than with ustekinumab (44% vs. 22% [AMAGINE-2] and 37% vs. 19% [AMAGINE-3], P<0.001). The PASI 100 response rates with 140 mg of brodalumab were 26% in AMAGINE-2 (P=0.08 for the comparison with ustekinumab) and 27% in AMAGINE-3 (P=0.007). Rates of neutropenia were higher with brodalumab and with ustekinumab than with placebo. Mild or moderate candida infections were more frequent with brodalumab than with ustekinumab or placebo. Through week 52, the rates of serious infectious episodes were 1.0 (AMAGINE-2) and 1.3 (AMAGINE-3) per 100 patient-years of exposure to brodalumab. CONCLUSIONS: Brodalumab treatment resulted in significant clinical improvements in patients with moderate-to-severe psoriasis. (Funded by Amgen; AMAGINE-2 and AMAGINE-3 ClinicalTrials.gov numbers, NCT01708603 and NCT01708629.).\"},\n",
       "  {'qas': [{'id': '56bc9268ac7ad1001900001b_002',\n",
       "     'question': 'What molecule is targeted by brodalumab?',\n",
       "     'answers': [{'text': 'interleukin-17', 'answer_start': 20}]}],\n",
       "   'context': \"Brodalumab, an anti-interleukin-17-receptor antibody for psoriasis. BACKGROUND: In this phase 2, randomized, double-blind, placebo-controlled, dose-ranging study, we assessed the efficacy and safety of brodalumab (AMG 827), a human anti-interleukin-17-receptor monoclonal antibody, for the treatment of moderate-to-severe plaque psoriasis. METHODS: We randomly assigned patients with a score of 12 or higher on the psoriasis area-and-severity index (PASI, on which scores range from 0 to 72, with higher scores indicating more severe disease) and with 10% or more of their body-surface area affected by psoriasis to receive brodalumab (70 mg, 140 mg, or 210 mg at day 1 and weeks 1, 2, 4, 6, 8, and 10 or 280 mg monthly) or placebo. The primary end point was the percentage improvement from baseline in the PASI score at week 12. Secondary end points included improvement of at least 75% and at least 90% in the PASI score and the score on the static physician's global assessment at week 12. RESULTS: A total of 198 patients underwent randomization. At week 12, the mean percentage improvements in the PASI score were 45.0% among patients receiving 70 mg of brodalumab, 85.9% among those receiving 140 mg, 86.3% among those receiving 210 mg, 76.0% among those receiving 280 mg, and 16.0% among those receiving placebo (P<0.001 for all comparisons with placebo). An improvement of at least 75% and at least 90% in the PASI score at week 12 was seen in 77% and 72%, respectively, of the patients in the 140-mg brodalumab group and in 82% and 75%, respectively, of the patients in the 210-mg group, as compared with 0% in the placebo group (P<0.001 for all comparisons). The percentage of patients with a static physician's global assessment of clear or minimal disease was 26%, 85%, 80%, and 69% with the 70-mg, 140-mg, 210-mg, and 280-mg doses, respectively, of brodalumab, as compared with 3% with placebo (P<0.01 for all comparisons with placebo). Two cases of grade 3 neutropenia were reported in the 210-mg brodalumab group. The most commonly reported adverse events in the combined brodalumab groups were nasopharyngitis (8%), upper respiratory tract infection (8%), and injection-site erythema (6%). CONCLUSIONS: Brodalumab significantly improved plaque psoriasis in this 12-week, phase 2 study. (Funded by Amgen; ClinicalTrials.gov number, NCT00975637.).\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_001',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 290}]}],\n",
       "   'context': 'Assessment of pain and itch behavior in a mouse model of neurofibromatosis type 1. UNLABELLED: Neurofibromatosis type 1 (NF1) is characterized primarily by tumor formation in the nervous system, but patients report other neurological complications including pain and itch. Individuals with NF1 harbor 1 mutated NF1 allele causing heterozygous expression in all of their cells. In mice, Nf1 heterozygosity leads to hyperexcitability of sensory neurons and hyperproliferation of mast cells, both of which could lead to increased hypersensitivity and scratching in response to noxious and pruritic stimuli. To determine whether Nf1 heterozygosity may increase pain and itch behaviors independent of secondary effects of tumor formation, we used mice with a targeted, heterozygous Nf1 gene deletion (Nf1±) that lack tumors. Nf1± mice exhibited normal baseline responses to thermal and mechanical stimuli. Moreover, similar to wild-type littermates, Nf1± mice developed inflammation-induced heat\\xa0and mechanical hypersensitivity, capsaicin-induced nocifensive behavior, histamine-dependent or -independent scratching, and chronic constriction injury-induced cold allodynia. However, Nf1±\\xa0mice exhibited an attenuated first phase of formalin-induced spontaneous behavior and expedited resolution of formalin-induced heat hypersensitivity. These results are not consistent with the hypothesis that Nf1 heterozygosity alone is sufficient to increase pain and itch sensation in mice, and they suggest that additional mechanisms may underlie reports of increased pain and itch in NF1 patients. PERSPECTIVE: This study assessed whether Nf1 heterozygosity in mice increased hypersensitivity and scratching following noxious and pruritic stimuli. Using Nf1± mice lacking tumors, this study finds no increases in pain or itch behavior, suggesting that there is no predisposition for either clinical symptom solely due to Nf1 heterozygosity.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_002',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 311}]}],\n",
       "   'context': 'Assessment of pain and itch behavior in a mouse model of neurofibromatosis type 1. UNLABELLED: Neurofibromatosis type 1 (NF1) is characterized primarily by tumor formation in the nervous system, but patients report other neurological complications including pain and itch. Individuals with NF1 harbor 1 mutated NF1 allele causing heterozygous expression in all of their cells. In mice, Nf1 heterozygosity leads to hyperexcitability of sensory neurons and hyperproliferation of mast cells, both of which could lead to increased hypersensitivity and scratching in response to noxious and pruritic stimuli. To determine whether Nf1 heterozygosity may increase pain and itch behaviors independent of secondary effects of tumor formation, we used mice with a targeted, heterozygous Nf1 gene deletion (Nf1±) that lack tumors. Nf1± mice exhibited normal baseline responses to thermal and mechanical stimuli. Moreover, similar to wild-type littermates, Nf1± mice developed inflammation-induced heat\\xa0and mechanical hypersensitivity, capsaicin-induced nocifensive behavior, histamine-dependent or -independent scratching, and chronic constriction injury-induced cold allodynia. However, Nf1±\\xa0mice exhibited an attenuated first phase of formalin-induced spontaneous behavior and expedited resolution of formalin-induced heat hypersensitivity. These results are not consistent with the hypothesis that Nf1 heterozygosity alone is sufficient to increase pain and itch sensation in mice, and they suggest that additional mechanisms may underlie reports of increased pain and itch in NF1 patients. PERSPECTIVE: This study assessed whether Nf1 heterozygosity in mice increased hypersensitivity and scratching following noxious and pruritic stimuli. Using Nf1± mice lacking tumors, this study finds no increases in pain or itch behavior, suggesting that there is no predisposition for either clinical symptom solely due to Nf1 heterozygosity.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_003',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 87}]}],\n",
       "   'context': \"A highly sensitive genetic protocol to detect NF1 mutations. Neurofibromatosis type 1 (NF1) is a hereditary disorder caused by mutations in the NF1 gene. Detecting mutation in NF1 is hindered by the gene's large size, the lack of mutation hotspots, the presence of pseudogenes, and the wide variety of possible lesions. We developed a method for detecting germline mutations by combining an original RNA-based cDNA-PCR mutation detection method and denaturing high-performance liquid chromatography (DHPLC) with multiplex ligation-dependent probe amplification (MLPA). The protocol was validated in a cohort of 56 blood samples from NF1 patients who fulfilled NIH diagnostic criteria, identifying the germline mutation in 53 cases (95% sensitivity). The efficiency and reliability of this approach facilitated detection of different types of mutations, including single-base substitutions, deletions or insertions of one to several nucleotides, microdeletions, and changes in intragenic copy number. Because mutational screening for minor lesions was performed using cDNA and the characterization of mutated alleles was performed at both the RNA and genomic DNA level, the analysis provided insight into the nature of the different mutations and their effect on NF1 mRNA splicing. After validation, we implemented the protocol as a routine test. Here we present the overall unbiased spectrum of NF1 mutations identified in 93 patients in a cohort of 105. The results indicate that this protocol is a powerful new tool for the molecular diagnosis of NF1.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_004',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 144}]}],\n",
       "   'context': \"A highly sensitive genetic protocol to detect NF1 mutations. Neurofibromatosis type 1 (NF1) is a hereditary disorder caused by mutations in the NF1 gene. Detecting mutation in NF1 is hindered by the gene's large size, the lack of mutation hotspots, the presence of pseudogenes, and the wide variety of possible lesions. We developed a method for detecting germline mutations by combining an original RNA-based cDNA-PCR mutation detection method and denaturing high-performance liquid chromatography (DHPLC) with multiplex ligation-dependent probe amplification (MLPA). The protocol was validated in a cohort of 56 blood samples from NF1 patients who fulfilled NIH diagnostic criteria, identifying the germline mutation in 53 cases (95% sensitivity). The efficiency and reliability of this approach facilitated detection of different types of mutations, including single-base substitutions, deletions or insertions of one to several nucleotides, microdeletions, and changes in intragenic copy number. Because mutational screening for minor lesions was performed using cDNA and the characterization of mutated alleles was performed at both the RNA and genomic DNA level, the analysis provided insight into the nature of the different mutations and their effect on NF1 mRNA splicing. After validation, we implemented the protocol as a routine test. Here we present the overall unbiased spectrum of NF1 mutations identified in 93 patients in a cohort of 105. The results indicate that this protocol is a powerful new tool for the molecular diagnosis of NF1.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_005',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 358}]}],\n",
       "   'context': 'Schweinfurthin A selectively inhibits proliferation and Rho signaling in glioma and neurofibromatosis type 1 tumor cells in a NF1-GRD-dependent manner. Neurofibromatosis type 1 (NF1) is the most common genetic disease affecting the nervous system. Patients typically develop many tumors over their lifetime, leading to increased morbidity and mortality. The NF1 gene, mutated in NF1, is also commonly mutated in sporadic glioblastoma multiforme (GBM). Because both NF1 and GBM are currently incurable, new therapeutic approaches are clearly needed. Natural products represent an opportunity to develop new therapies, as they have been evolutionarily selected to play targeted roles in organisms. Schweinfurthin A is a prenylated stilbene natural product that has previously shown specific inhibitory activity against brain and hematopoietic tumor lines. We show that patient-derived GBM and NF1 malignant peripheral nerve sheath tumor (MPNST) lines, as well as tumor lines derived from the Nf1-/+;Trp53-/+ (NPcis) mouse model of astrocytoma and MPNST are highly sensitive to inhibition by schweinfurthin A and its synthetic analogs. In contrast, primary mouse astrocytes are resistant to the growth inhibitory effects of schweinfurthin A, suggesting that schweinfurthin A may act specifically on tumor cells. Stable transfection of the GTPase-activating protein related domain of Nf1 into Nf1-/-;Trp53-/- astrocytoma cells confers resistance to schweinfurthin A. In addition, the profound effect of schweinfurthin A on dynamic reorganization of the actin cytoskeleton led us to discover that schweinfurthin A inhibits growth factor-stimulated Rho signaling. In summary, we have identified a class of small molecules that specifically inhibit growth of cells from both central and peripheral nervous system tumors and seem to act on NF1-deficient cells through cytoskeletal reorganization correlating to changes in Rho signaling.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_006',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 379}]}],\n",
       "   'context': 'Schweinfurthin A selectively inhibits proliferation and Rho signaling in glioma and neurofibromatosis type 1 tumor cells in a NF1-GRD-dependent manner. Neurofibromatosis type 1 (NF1) is the most common genetic disease affecting the nervous system. Patients typically develop many tumors over their lifetime, leading to increased morbidity and mortality. The NF1 gene, mutated in NF1, is also commonly mutated in sporadic glioblastoma multiforme (GBM). Because both NF1 and GBM are currently incurable, new therapeutic approaches are clearly needed. Natural products represent an opportunity to develop new therapies, as they have been evolutionarily selected to play targeted roles in organisms. Schweinfurthin A is a prenylated stilbene natural product that has previously shown specific inhibitory activity against brain and hematopoietic tumor lines. We show that patient-derived GBM and NF1 malignant peripheral nerve sheath tumor (MPNST) lines, as well as tumor lines derived from the Nf1-/+;Trp53-/+ (NPcis) mouse model of astrocytoma and MPNST are highly sensitive to inhibition by schweinfurthin A and its synthetic analogs. In contrast, primary mouse astrocytes are resistant to the growth inhibitory effects of schweinfurthin A, suggesting that schweinfurthin A may act specifically on tumor cells. Stable transfection of the GTPase-activating protein related domain of Nf1 into Nf1-/-;Trp53-/- astrocytoma cells confers resistance to schweinfurthin A. In addition, the profound effect of schweinfurthin A on dynamic reorganization of the actin cytoskeleton led us to discover that schweinfurthin A inhibits growth factor-stimulated Rho signaling. In summary, we have identified a class of small molecules that specifically inhibit growth of cells from both central and peripheral nervous system tumors and seem to act on NF1-deficient cells through cytoskeletal reorganization correlating to changes in Rho signaling.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_007',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 140}]}],\n",
       "   'context': 'Breakpoint characterization of a novel NF1 multiexonic deletion: a case showing expression of the mutated allele. Neurofibromatosis type 1 (NF1) is a common genetic disease caused by haploinsufficiency of the NF1 tumor-suppressor gene. Different pathogenetic mechanisms have been identified, with the majority (95%) causing intragenic lesions. Single or multiexon NF1 copy number changes occur in about 2% of patients, but little is known about the molecular mechanisms behind these intragenic deletions. We report here on the molecular characterization of a novel NF1 multiexonic deletion. The application of a multidisciplinary approach including multiplex ligation-dependent probe amplification, allelic segregation analysis, and fluorescent in situ hybridization allowed us to map the breakpoints in IVS27b and IVS48. Furthermore, the breakpoint junction was characterized by sequencing. Using bioinformatic analysis, we identified some recombinogenic motifs in close proximity to the centromeric and telomeric breakpoints and predicted the presence of a mutated messenger ribonucleic acid, which was deleted between exons 28 and 48 and encodes a neurofibromin that lacks some domains essential for its function. Through reverse transcriptase-polymerase chain reaction, the expression of the mutated allele was verified, showing the junction between exons 27b and 49 and, as expected, was not subjected to nonsense-mediated decay. Multiexonic deletions represent 2% of NF1 mutations, and until now, the breakpoint has been identified in only a few cases. The fine characterization of multiexonic deletions broadens the mutational repertoire of the NF1 gene, allowing for the identification of different pathogenetic mechanisms causing NF1.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_008',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 209}]}],\n",
       "   'context': 'Breakpoint characterization of a novel NF1 multiexonic deletion: a case showing expression of the mutated allele. Neurofibromatosis type 1 (NF1) is a common genetic disease caused by haploinsufficiency of the NF1 tumor-suppressor gene. Different pathogenetic mechanisms have been identified, with the majority (95%) causing intragenic lesions. Single or multiexon NF1 copy number changes occur in about 2% of patients, but little is known about the molecular mechanisms behind these intragenic deletions. We report here on the molecular characterization of a novel NF1 multiexonic deletion. The application of a multidisciplinary approach including multiplex ligation-dependent probe amplification, allelic segregation analysis, and fluorescent in situ hybridization allowed us to map the breakpoints in IVS27b and IVS48. Furthermore, the breakpoint junction was characterized by sequencing. Using bioinformatic analysis, we identified some recombinogenic motifs in close proximity to the centromeric and telomeric breakpoints and predicted the presence of a mutated messenger ribonucleic acid, which was deleted between exons 28 and 48 and encodes a neurofibromin that lacks some domains essential for its function. Through reverse transcriptase-polymerase chain reaction, the expression of the mutated allele was verified, showing the junction between exons 27b and 49 and, as expected, was not subjected to nonsense-mediated decay. Multiexonic deletions represent 2% of NF1 mutations, and until now, the breakpoint has been identified in only a few cases. The fine characterization of multiexonic deletions broadens the mutational repertoire of the NF1 gene, allowing for the identification of different pathogenetic mechanisms causing NF1.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_009',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 141}]}],\n",
       "   'context': 'Is pulmonary arterial hypertension in neurofibromatosis type 1 secondary to a plexogenic arteriopathy? BACKGROUND: Neurofibromatosis type 1 (NF1) is a common disorder of dysregulated tissue growth secondary to mutations in the tumor suppressor gene NF1. Pulmonary arterial hypertension (PAH) in patients with NF1 is hypothesized to be secondary to an underlying vasculopathy. METHODS: We describe the entity we term NF1-associated PAH (NF1-PAH) in four new patients and update the data on four previously published reports of patients with PAH and NF1. We performed genetic testing of the bone morphogenic protein receptor 2 (BMPR2) gene, which mutated in 70% of patients with familial PAH and approximately 25% of patients with idiopathic PAH. We report, for the first time, pathologic findings in the autopsy-obtained lung of one patient with NF1-PAH. RESULTS: Patients with NF1-PAH have a generally poor long-term prognosis. In four patients, we observed the mosaic pattern of lung attenuation on a CT scan of the chest, a radiographic finding that can be consistent with an underlying vasculopathy. No mutations or rearrangements in the BMPR2 gene were found. We observed complex plexiform lesions in the one available autopsy specimen. Similar lesions are a hallmark of plexogenic pulmonary arteriopathy and are associated with several severe types of PAH. (Plexiform lesions should not be confused with plexiform neurofibromas, which are distinctive tumors seen in NF1.) CONCLUSIONS: Our findings suggest that NF1 should be considered as being \"associated with PAH as outlined in the Revised Clinical Classification of Pulmonary Hypertension. Understanding the mechanism of PAH in NF1 may inform the pathogenesis of PAH, NF1-PAH itself, and other NF1-associated vasculopathies. The pulmonary vasculature should now be included among the arterial beds affected by NF1 vasculopathy.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_010',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 249}]}],\n",
       "   'context': 'Is pulmonary arterial hypertension in neurofibromatosis type 1 secondary to a plexogenic arteriopathy? BACKGROUND: Neurofibromatosis type 1 (NF1) is a common disorder of dysregulated tissue growth secondary to mutations in the tumor suppressor gene NF1. Pulmonary arterial hypertension (PAH) in patients with NF1 is hypothesized to be secondary to an underlying vasculopathy. METHODS: We describe the entity we term NF1-associated PAH (NF1-PAH) in four new patients and update the data on four previously published reports of patients with PAH and NF1. We performed genetic testing of the bone morphogenic protein receptor 2 (BMPR2) gene, which mutated in 70% of patients with familial PAH and approximately 25% of patients with idiopathic PAH. We report, for the first time, pathologic findings in the autopsy-obtained lung of one patient with NF1-PAH. RESULTS: Patients with NF1-PAH have a generally poor long-term prognosis. In four patients, we observed the mosaic pattern of lung attenuation on a CT scan of the chest, a radiographic finding that can be consistent with an underlying vasculopathy. No mutations or rearrangements in the BMPR2 gene were found. We observed complex plexiform lesions in the one available autopsy specimen. Similar lesions are a hallmark of plexogenic pulmonary arteriopathy and are associated with several severe types of PAH. (Plexiform lesions should not be confused with plexiform neurofibromas, which are distinctive tumors seen in NF1.) CONCLUSIONS: Our findings suggest that NF1 should be considered as being \"associated with PAH as outlined in the Revised Clinical Classification of Pulmonary Hypertension. Understanding the mechanism of PAH in NF1 may inform the pathogenesis of PAH, NF1-PAH itself, and other NF1-associated vasculopathies. The pulmonary vasculature should now be included among the arterial beds affected by NF1 vasculopathy.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_011',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 1059}]}],\n",
       "   'context': 'Analysis of mitochondrial DNA in discordant monozygotic twins with neurofibromatosis type 1. Neurofibromatosis type 1 (NF1) is the most frequent neurocutaneous disorder with autosomal dominant inheritance. Phenotype variability is high ranging from merely several café-au-lait spots to malignant peripheral nerve sheath tumors or severe disfigurement through plexiform neurofibromas. Identification of genetic factors that modify the NF1 phenotype would contribute to the understanding of NF1 pathophysiology and improve patient counselling. As even monozygotic (MZ) twins with NF1 may differ phenotypically, we wondered whether these variations might be inherited in a non-Mendelian fashion. Mitochondrial DNA (mtDNA) is inherited extrachromosomally through the cytoplasm of the oocyte and often harbours heteroplasmic sequence variations. At the time of blastomere separation, these variants may be skewedly distributed and effect phenotypic differences. Because of their co-localization with the tumor suppressor protein neurofibromin, which is mutated in NF1, mitochondria were particular attractive candidates for investigation. MtDNA was extracted from nucleated blood cells of four pairs of discordant MZ twins with NF1 and from cutaneous neurofibromas of one twin pair. We sequenced the entire mitochondrial genome and determined the state of heteroplasmy by investigating a microsatellite region of the mitochondrial D-loop (D310-tract). The clinical diagnosis was confirmed in all patients by detection of pathogenic mutations in the NF1 gene. Monozygosity was verified by genotyping. However, we did not detect evidence for mtDNA sequence differences or for different degrees of heteroplasmy between individuals of the same twin pair. The phenotypic discordance of MZ twins with NF1 cannot be explained by skewed distribution of mtDNA mutations or polymorphisms.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_012',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 195}]}],\n",
       "   'context': 'Functional analysis of splicing mutations in exon 7 of NF1 gene. BACKGROUND: Neurofibromatosis type 1 is one of the most common autosomal dominant disorders, affecting about 1:3,500 individuals. NF1 exon 7 displays weakly defined exon-intron boundaries, and is particularly prone to missplicing. METHODS: In this study we investigated the expression of exon 7 transcripts using bioinformatic identification of splicing regulatory sequences, and functional minigene analysis of four sequence changes [c.910C>T (R304X), c.945G>A/c.946C>A (Q315Q/L316M), c.1005T>C (N335N)] identified in exon 7 of three different NF1 patients. RESULTS: Our results detected the presence of three exonic splicing enhancers (ESEs) and one putative exonic splicing silencer (ESS) element. The wild type minigene assay resulted in three alternative isoforms, including a transcript lacking NF1 exon 7 (NF1DeltaE7). Both the wild type and the mutated constructs shared NF1DeltaE7 in addition to the complete messenger, but displayed a different ratio between the two transcripts. In the presence of R304X and Q315Q/L316M mutations, the relative proportion between the different isoforms is shifted toward the expression of NF1DeltaE7, while in the presence of N335N variant, the NF1DeltaE7 expression is abolished. CONCLUSION: In conclusion, it appears mandatory to investigate the role of each nucleotide change within the NF1 coding sequence, since a significant proportion of NF1 exon 7 mutations affects pre-mRNA splicing, by disrupting exonic splicing motifs and modifying the delicate balance between aberrantly and correctly spliced transcripts.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_013',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 759}]}],\n",
       "   'context': 'Mitotic recombination as evidence of alternative pathogenesis of gastrointestinal stromal tumours in neurofibromatosis type 1. BACKGROUND: Neurofibromatosis type 1 (NF1) is a neurocutaneous disorder resulting in the growth of a variety of tumours, and is inherited in an autosomal dominant pattern. Gastrointestinal stromal tumours (GISTs) are mesenchymal tumours that commonly harbour oncogenic mutations in KIT or PDGFRA and are thought to arise from the interstitial cells of Cajal (ICC; the pacemaker cells of the gut). AIM: To characterise two patients with NF1 and GISTs. METHODS: Two patients were genotyped for germline mutations in NF1. GISTs from both patients were genotyped for somatic mutations in KIT and PDGFRA. Loss of heterozygosity (LOH) of NF1 in one GIST was assessed by genotyping seven microsatellite markers spanning 2.39 Mb of the NF1 locus in the tumour and in genomic DNA. The known germline mutation in NF1 was confirmed in GIST DNA by sequencing. The copy number of the mutated NF1 allele was determined by multiplex ligand-dependent probe amplification. RESULTS: GISTs from both patients were of wild type for mutations in KIT and PDGFRA. In the GIST with adequate DNA, all seven markers were informative and showed LOH at the NF1 locus; sequencing of NF1 from that GIST showed no wild-type sequence, suggesting that it was lost in the tumour. Multiplex ligand-dependent probe amplification analysis showed that two copies of all NF1 exons were present. CONCLUSIONS: This is the first evidence of mitotic recombination resulting in a reduction to homozygosity of a germline NF1 mutation in an NF1-associated GIST. We hypothesise that the LOH of NF1 and lack of KIT and PDGFRA mutations are evidence of an alternative pathogenesis in NF1-associated GISTs.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_014',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 165}]}],\n",
       "   'context': 'Mitotic recombination as evidence of alternative pathogenesis of gastrointestinal stromal tumours in neurofibromatosis type 1. BACKGROUND: Neurofibromatosis type 1 (NF1) is a neurocutaneous disorder resulting in the growth of a variety of tumours, and is inherited in an autosomal dominant pattern. Gastrointestinal stromal tumours (GISTs) are mesenchymal tumours that commonly harbour oncogenic mutations in KIT or PDGFRA and are thought to arise from the interstitial cells of Cajal (ICC; the pacemaker cells of the gut). AIM: To characterise two patients with NF1 and GISTs. METHODS: Two patients were genotyped for germline mutations in NF1. GISTs from both patients were genotyped for somatic mutations in KIT and PDGFRA. Loss of heterozygosity (LOH) of NF1 in one GIST was assessed by genotyping seven microsatellite markers spanning 2.39 Mb of the NF1 locus in the tumour and in genomic DNA. The known germline mutation in NF1 was confirmed in GIST DNA by sequencing. The copy number of the mutated NF1 allele was determined by multiplex ligand-dependent probe amplification. RESULTS: GISTs from both patients were of wild type for mutations in KIT and PDGFRA. In the GIST with adequate DNA, all seven markers were informative and showed LOH at the NF1 locus; sequencing of NF1 from that GIST showed no wild-type sequence, suggesting that it was lost in the tumour. Multiplex ligand-dependent probe amplification analysis showed that two copies of all NF1 exons were present. CONCLUSIONS: This is the first evidence of mitotic recombination resulting in a reduction to homozygosity of a germline NF1 mutation in an NF1-associated GIST. We hypothesise that the LOH of NF1 and lack of KIT and PDGFRA mutations are evidence of an alternative pathogenesis in NF1-associated GISTs.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_015',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 689}]}],\n",
       "   'context': 'Inducible and transmissible genetic events and pediatric tumors of the nervous system. Tumors of the nervous system most often occur in both children and adults as sporadic events with no family history of the disease, but they are also among the clinical manifestations of a significant number of familial cancer syndromes, including familial retinoblastoma, neurofibromatosis 1 and 2, tuberous sclerosis, and Cowden, Turcot, Li-Fraumeni and nevoid basal cell carcinoma (Gorlin) syndromes. All of these syndromes involve transmissible genetic risk resulting from loss of a functional allele, or inheritance of a structurally defective allele, of a specific gene. These genes include RB1, NF1, NF2, TSC1, TSC2, TP53, PTEN, APC, hMLH1, hPSM2, and PTCH, most of which function as tumor suppressor genes. The same genes are also observed in mutated and inactive forms, or are deleted, in tumor cells in sporadic cases of the same tumors. The nature of the mutational events that give rise to these inactivated alleles suggests a possible role of environmental mutagens in their causation. However, only external ionizing radiation at high doses is clearly established as an environmental cause of brain, nerve and meningeal tumors in humans. Transplacental carcinogenesis studies in rodents and other species emphasize the extraordinary susceptibility of the developing mammalian nervous system to carcinogenesis, but the inverse relationship of latency to dose suggests that low transplacental exposures to genotoxicants are more likely to result in brain tumors late in life, rather than in childhood. While not all neurogenic tumor-related genes in humans have similar effects in experimental rodents, genetically engineered mice (GEM) increasingly provide useful insights into the combined effects of multiple tumor suppressor genes and of gene-environment interactions in the genesis of brain tumors, especially pediatric brain tumors such as medulloblastoma.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_016',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 332}]}],\n",
       "   'context': \"Pancreatic endocrine tumors are a rare manifestation of the neurofibromatosis type 1 phenotype: molecular analysis of a malignant insulinoma in a NF-1 patient. The tumorigenesis of sporadic endocrine tumors is still not fully understood. It is well known that patients with von Recklinghausen syndrome (NF-1) (OMIM 162200) carrying NF1 germline mutations are predisposed to endocrine tumors including pheochromocytomas and duodenal somatostatinomas. It is unclear, however, whether the rarely reported occurrence of pancreatic insulinomas in NF-1 patients represents a coincidental finding or whether insulinomas are a rare manifestation of the NF-1 syndrome. To determine the potential association between the NF-1 syndrome and pancreatic endocrine tumors, we analyzed a NF-1 patient with a well-differentiated pancreatic endocrine carcinoma for NF1 mutation, allelic loss of the NF1 gene and its expression in peripheral blood and tumor cells. The germline mutation c. 499 del TGTT known in the family was confirmed by polymerase chain reaction (PCR) and direct sequencing of exon 4 in DNA extracted from peripheral blood. Loss of heterozygosity (LOH) analysis of the NF1 gene was carried out using 3 intragenic microsatellite markers on 17q11.2. RNA expression was examined by reverse transcription and a consecutive PCR spanning intron 3 of the NF1 gene including the mutated site in exon 4. Immunohistochemistry was used to analyze NF-1 protein expression. Mutation analysis of peripheral blood leukocytes confirmed the 4 base pair deletion in exon 4 starting at codon 167 (499 del TGTT). LOH analysis of tumor tissue revealed retention of both NF1 alleles. While reverse transcriptase-PCR of peripheral blood showed bi-allelic expression of both the wild-type NF1 and the mutated form, reverse transcriptase-PCR of tumor extracts demonstrated expression of the mutated but not the wild-type NF1 allele. Additionally, neurofibromin, the NF1 gene product, was absent in the tumor tissue of the NF-1 patient. These results show that the wild-type NF1 transcrips and protein are reduced, in the reported insulinoma, supposedly by epigenetic mechanisms. This provides strong evidence that there is a relationship between von Recklinghausen disease and the patient's insulinoma. In this line, insulinomas may be viewed as a rare manifestation of the NF-1 syndrome. Furthermore, the NF1 gene must be considered as a candidate tumor suppressor gene for sporadic insulinomas and probably other pancreatic endocrine tumors.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_017',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 136}]}],\n",
       "   'context': \"Novel NF1 gene mutation in a Japanese patient with neurofibromatosis type 1 and a gastrointestinal stromal tumor. Many mutations of the NF1 gene have been reported in patients with neurofibromatosis type 1 (NF1); however, there have been no documented NF1 gene mutations in Japanese NF1 patients. In the present study, we used the polymerase chain reaction (PCR) and DNA sequencing analysis to characterize the NF1 gene in a 53-year-old Japanese patient with NF1 who suffered from neurofibroma, pheochromocytoma, and gastrointestinal stromal tumor (GIST). Direct sequence analyses revealed a single base substitution in the splicing donor site of intron 6 (IVS6 888+1, G --> A) in one NF1 allele, resulting in an altered splice site (ss) in the mutated allele. Splicing at the cryptic 5' ss in the mutated allele generated mRNA with an insertion of 60 nucleotides. In addition, we screened for mutations in exons 9, 11, 13, and 17 of the c-kit gene in GIST and the succinate dehydrogenase subunit D (SDHD) gene in the pheochromocytoma, but we did not detect any somatic mutations. We report here the first case of an NF1 patient with four neoplasms: neurofibroma, pheochromocytoma, astrocytoma and GIST. Our results suggest that the molecular pathogenesis of GISTs in NF1 patients is different from that in non-NF1 patients.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_018',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 207}]}],\n",
       "   'context': \"Novel NF1 gene mutation in a Japanese patient with neurofibromatosis type 1 and a gastrointestinal stromal tumor. Many mutations of the NF1 gene have been reported in patients with neurofibromatosis type 1 (NF1); however, there have been no documented NF1 gene mutations in Japanese NF1 patients. In the present study, we used the polymerase chain reaction (PCR) and DNA sequencing analysis to characterize the NF1 gene in a 53-year-old Japanese patient with NF1 who suffered from neurofibroma, pheochromocytoma, and gastrointestinal stromal tumor (GIST). Direct sequence analyses revealed a single base substitution in the splicing donor site of intron 6 (IVS6 888+1, G --> A) in one NF1 allele, resulting in an altered splice site (ss) in the mutated allele. Splicing at the cryptic 5' ss in the mutated allele generated mRNA with an insertion of 60 nucleotides. In addition, we screened for mutations in exons 9, 11, 13, and 17 of the c-kit gene in GIST and the succinate dehydrogenase subunit D (SDHD) gene in the pheochromocytoma, but we did not detect any somatic mutations. We report here the first case of an NF1 patient with four neoplasms: neurofibroma, pheochromocytoma, astrocytoma and GIST. Our results suggest that the molecular pathogenesis of GISTs in NF1 patients is different from that in non-NF1 patients.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_019',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 6}]}],\n",
       "   'context': \"Novel NF1 gene mutation in a Japanese patient with neurofibromatosis type 1 and a gastrointestinal stromal tumor. Many mutations of the NF1 gene have been reported in patients with neurofibromatosis type 1 (NF1); however, there have been no documented NF1 gene mutations in Japanese NF1 patients. In the present study, we used the polymerase chain reaction (PCR) and DNA sequencing analysis to characterize the NF1 gene in a 53-year-old Japanese patient with NF1 who suffered from neurofibroma, pheochromocytoma, and gastrointestinal stromal tumor (GIST). Direct sequence analyses revealed a single base substitution in the splicing donor site of intron 6 (IVS6 888+1, G --> A) in one NF1 allele, resulting in an altered splice site (ss) in the mutated allele. Splicing at the cryptic 5' ss in the mutated allele generated mRNA with an insertion of 60 nucleotides. In addition, we screened for mutations in exons 9, 11, 13, and 17 of the c-kit gene in GIST and the succinate dehydrogenase subunit D (SDHD) gene in the pheochromocytoma, but we did not detect any somatic mutations. We report here the first case of an NF1 patient with four neoplasms: neurofibroma, pheochromocytoma, astrocytoma and GIST. Our results suggest that the molecular pathogenesis of GISTs in NF1 patients is different from that in non-NF1 patients.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_020',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 460}]}],\n",
       "   'context': 'No aberrant methylation of neurofibromatosis 1 gene (NF1) promoter in pilocytic astrocytoma in childhood. Tumors of the central nervous system are the most frequent solid tumors in childhood. With 30-40% of this heterogenous group, low-grade astrocytomas represent the most common subtype. Neurofibromatosis type 1 (NF1) is strongly associated with the development of pilocytic astrocytoma (PA), frequently appearing as optic glioma. Neurofibromatosis 1 gene (NF1 ) fulfills the criteria of a tumor suppressor gene and is deleted or mutated heterozygously in patients with NF1. This suggests an involvement in the development of PA. To clarify whether silencing of NF1 by promoter methylation plays a role in PA and especially in optic glioma, the authors investigated the methylation status in 30 PA, 6 of which had optic glioma. However, no methylation was found at the NF1 promoter region in PA. To rule out that silencing of NF1 by promoter methylation is restricted to higher-grade astrocytomas, 15 pediatric WHO II degree and IV degree astrocytomas were analyzed: 12 astrocytomas II and 3 glioblastomas displayed no NF1 promoter methylation. The authors conclude that NF1 silencing by methylation plays no role in low-grade astrocytoma.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_021',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 573}]}],\n",
       "   'context': 'No aberrant methylation of neurofibromatosis 1 gene (NF1) promoter in pilocytic astrocytoma in childhood. Tumors of the central nervous system are the most frequent solid tumors in childhood. With 30-40% of this heterogenous group, low-grade astrocytomas represent the most common subtype. Neurofibromatosis type 1 (NF1) is strongly associated with the development of pilocytic astrocytoma (PA), frequently appearing as optic glioma. Neurofibromatosis 1 gene (NF1 ) fulfills the criteria of a tumor suppressor gene and is deleted or mutated heterozygously in patients with NF1. This suggests an involvement in the development of PA. To clarify whether silencing of NF1 by promoter methylation plays a role in PA and especially in optic glioma, the authors investigated the methylation status in 30 PA, 6 of which had optic glioma. However, no methylation was found at the NF1 promoter region in PA. To rule out that silencing of NF1 by promoter methylation is restricted to higher-grade astrocytomas, 15 pediatric WHO II degree and IV degree astrocytomas were analyzed: 12 astrocytomas II and 3 glioblastomas displayed no NF1 promoter methylation. The authors conclude that NF1 silencing by methylation plays no role in low-grade astrocytoma.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_022',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 73}]}],\n",
       "   'context': 'Regulating heart development: the role of Nf1. Neurofibromatosis type 1 (NF1) is one of the most common human genetic disorders and is associated with significant morbidity and mortality. The gene responsible for this disorder, NF1, encodes neurofibromin, which can function to down-regulate ras activity. Mutations that inactivate NF7 result in elevated levels of ras signaling and increased cell proliferation in some tissues. NF7 functions as a tumor suppressor gene; patients inherit one mutated copy and are believed to acquire a \"second hit\" in tissues that go on to form benign or malignant tumors. NF7 is expressed widely, yet certain tissues are more susceptible to growth dysregulation in NF1 patients. Cardiovascular defects also contribute to NF1, though the cause remains unclear. In a recent study, we used tissue-specific gene inactivation in mice to study the role of neurofibromin in heart development. A further understanding of neurofibromin function will help to elucidate the pathophysiology of NF1 and will also lead to a better understanding of cell cycle regulation and ras pathways in specific cell types. Finally, we comment on how similar genetic strategies can be used in mice to study the role of additional signaling pathways involved in heart development.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_023',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 228}]}],\n",
       "   'context': 'Regulating heart development: the role of Nf1. Neurofibromatosis type 1 (NF1) is one of the most common human genetic disorders and is associated with significant morbidity and mortality. The gene responsible for this disorder, NF1, encodes neurofibromin, which can function to down-regulate ras activity. Mutations that inactivate NF7 result in elevated levels of ras signaling and increased cell proliferation in some tissues. NF7 functions as a tumor suppressor gene; patients inherit one mutated copy and are believed to acquire a \"second hit\" in tissues that go on to form benign or malignant tumors. NF7 is expressed widely, yet certain tissues are more susceptible to growth dysregulation in NF1 patients. Cardiovascular defects also contribute to NF1, though the cause remains unclear. In a recent study, we used tissue-specific gene inactivation in mice to study the role of neurofibromin in heart development. A further understanding of neurofibromin function will help to elucidate the pathophysiology of NF1 and will also lead to a better understanding of cell cycle regulation and ras pathways in specific cell types. Finally, we comment on how similar genetic strategies can be used in mice to study the role of additional signaling pathways involved in heart development.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_024',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 453}]}],\n",
       "   'context': 'The neurofibromatoses: when less is more. The study of cancer predisposition syndromes presents unique opportunities to gain insights into the genetic events associated with tumor pathogenesis. Individuals with two inherited cancer syndromes, neurofibromatosis 1 (NF1) and neurofibromatosis 2 (NF2), develop both benign and malignant tumors. The corresponding genes mutated in these two disorders encode tumor suppressor proteins, termed neurofibromin (NF1) and merlin (NF2), which function in novel ways to regulate cell growth and differentiation. Neurofibromin inhibits cell proliferation, at least in part, by modulating mitogenic pathway signaling through inactivation of p21-ras. In contrast, merlin may act as a membrane-associated molecular switch that regulates cell-cell and cell-matrix signals transduced by cell surface receptors. Significant progress in our understanding of the genetics and biology of NF1 and NF2 has elucidated the roles of these genes in tumor initiation and progression.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_025',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'Nf1', 'answer_start': 668}]}],\n",
       "   'context': 'Nf1;Trp53 mutant mice develop glioblastoma with evidence of strain-specific effects. Astrocytomas are the leading cause of brain cancer in humans. Because these tumours are highly infiltrative, current treatments that rely on targeting the tumour mass are often ineffective. A mouse model for astrocytoma would be a powerful tool for dissecting tumour progression and testing therapeutics. Mouse models of astrocytoma have been designed to express oncogenic proteins in astrocytes, but have had limited success due to low tumour penetrance or limited tumour progression. We present here a mouse model of astrocytomas involving mutation of two tumour-suppressor genes, Nf1 and Trp53. Humans with mutations in NF1 develop neurofibromatosis type I (NF1) and have increased risk of optic gliomas, astrocytomas and glioblastomas. The TP53 tumour suppressor is often mutated in a subset of astrocytomas that develop at a young age and progress slowly to glioblastoma (termed secondary glioblastomas, in contrast to primary glioblastomas that develop rapidly de novo). This mouse model shows a range of astrocytoma stages, from low-grade astrocytoma to glioblastoma multiforme, and may accurately model human secondary glioblastoma involving TP53 loss. This is the first reported mouse model of astrocytoma initiated by loss of tumour suppressors, rather than overexpression of transgenic oncogenes.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_026',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 708}]}],\n",
       "   'context': 'Nf1;Trp53 mutant mice develop glioblastoma with evidence of strain-specific effects. Astrocytomas are the leading cause of brain cancer in humans. Because these tumours are highly infiltrative, current treatments that rely on targeting the tumour mass are often ineffective. A mouse model for astrocytoma would be a powerful tool for dissecting tumour progression and testing therapeutics. Mouse models of astrocytoma have been designed to express oncogenic proteins in astrocytes, but have had limited success due to low tumour penetrance or limited tumour progression. We present here a mouse model of astrocytomas involving mutation of two tumour-suppressor genes, Nf1 and Trp53. Humans with mutations in NF1 develop neurofibromatosis type I (NF1) and have increased risk of optic gliomas, astrocytomas and glioblastomas. The TP53 tumour suppressor is often mutated in a subset of astrocytomas that develop at a young age and progress slowly to glioblastoma (termed secondary glioblastomas, in contrast to primary glioblastomas that develop rapidly de novo). This mouse model shows a range of astrocytoma stages, from low-grade astrocytoma to glioblastoma multiforme, and may accurately model human secondary glioblastoma involving TP53 loss. This is the first reported mouse model of astrocytoma initiated by loss of tumour suppressors, rather than overexpression of transgenic oncogenes.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_027',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 746}]}],\n",
       "   'context': 'Nf1;Trp53 mutant mice develop glioblastoma with evidence of strain-specific effects. Astrocytomas are the leading cause of brain cancer in humans. Because these tumours are highly infiltrative, current treatments that rely on targeting the tumour mass are often ineffective. A mouse model for astrocytoma would be a powerful tool for dissecting tumour progression and testing therapeutics. Mouse models of astrocytoma have been designed to express oncogenic proteins in astrocytes, but have had limited success due to low tumour penetrance or limited tumour progression. We present here a mouse model of astrocytomas involving mutation of two tumour-suppressor genes, Nf1 and Trp53. Humans with mutations in NF1 develop neurofibromatosis type I (NF1) and have increased risk of optic gliomas, astrocytomas and glioblastomas. The TP53 tumour suppressor is often mutated in a subset of astrocytomas that develop at a young age and progress slowly to glioblastoma (termed secondary glioblastomas, in contrast to primary glioblastomas that develop rapidly de novo). This mouse model shows a range of astrocytoma stages, from low-grade astrocytoma to glioblastoma multiforme, and may accurately model human secondary glioblastoma involving TP53 loss. This is the first reported mouse model of astrocytoma initiated by loss of tumour suppressors, rather than overexpression of transgenic oncogenes.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_028',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 129}]}],\n",
       "   'context': 'Three different premature stop codons lead to skipping of exon 7 in neurofibromatosis type I patients. Neurofibromatosis type 1 (NF1) is an autosomal dominant genetic disorder affecting one in 3,500 individuals. The mutation rate in the NF1 gene is one of the highest known for human genes. Compared to other methods, the protein truncation test (PTT) provides improved efficiency in detecting NF1 mutations which are dispersed throughout the gene which spans 350 kilobases of genomic DNA. We have applied the PTT and subsequent sequence analysis of cloned cDNA to identify mutations in NF1 patients. We report here the identification of two novel (W336X and Q315X), and one recurrent (R304X) mutation located in exon 7 and show that all three premature termination codons lead to skipping of exon 7 in a proportion of the transcripts derived from the mutated allele. Possible mutation-induced alterations of the RNA secondary structure and their impact on skipping of exon 7 of the NF1 gene are explored and discussed.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_029',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 237}]}],\n",
       "   'context': 'Three different premature stop codons lead to skipping of exon 7 in neurofibromatosis type I patients. Neurofibromatosis type 1 (NF1) is an autosomal dominant genetic disorder affecting one in 3,500 individuals. The mutation rate in the NF1 gene is one of the highest known for human genes. Compared to other methods, the protein truncation test (PTT) provides improved efficiency in detecting NF1 mutations which are dispersed throughout the gene which spans 350 kilobases of genomic DNA. We have applied the PTT and subsequent sequence analysis of cloned cDNA to identify mutations in NF1 patients. We report here the identification of two novel (W336X and Q315X), and one recurrent (R304X) mutation located in exon 7 and show that all three premature termination codons lead to skipping of exon 7 in a proportion of the transcripts derived from the mutated allele. Possible mutation-induced alterations of the RNA secondary structure and their impact on skipping of exon 7 of the NF1 gene are explored and discussed.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_030',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 227}]}],\n",
       "   'context': 'Minor lesion mutational spectrum of the entire NF1 gene does not explain its high mutability but points to a functional domain upstream of the GAP-related domain. More than 500 unrelated patients with neurofibromatosis type 1 (NF1) were screened for mutations in the NF1 gene. For each patient, the whole coding sequence and all splice sites were studied for aberrations, either by the protein truncation test (PTT), temperature-gradient gel electrophoresis (TGGE) of genomic PCR products, or, most often, by direct genomic sequencing (DGS) of all individual exons. A total of 301 sequence variants, including 278 bona fide pathogenic mutations, were identified. As many as 216 or 183 of the genuine mutations, comprising 179 or 161 different ones, can be considered novel when compared to the recent findings of Upadhyaya and Cooper, or to the NNFF mutation database. Mutation-detection efficiencies of the various screening methods were similar: 47.1% for PTT, 53.7% for TGGE, and 54.9% for DGS. Some 224 mutations (80.2%) yielded directly or indirectly premature termination codons. These mutations showed even distribution over the whole gene from exon 1 to exon 47. Of all sequence variants determined in our study, <20% represent C-->T or G-->A transitions within a CpG dinucleotide, and only six different mutations also occur in NF1 pseudogenes, with five being typical C-->T transitions in a CpG. Thus, neither frequent deamination of 5-methylcytosines nor interchromosomal gene conversion may account for the high mutation rate of the NF1 gene. As opposed to the truncating mutations, the 28 (10.1%) missense or single-amino-acid-deletion mutations identified clustered in two distinct regions, the GAP-related domain (GRD) and an upstream gene segment comprising exons 11-17. The latter forms a so-called cysteine/serine-rich domain with three cysteine pairs suggestive of ATP binding, as well as three potential cAMP-dependent protein kinase (PKA) recognition sites obviously phosphorylated by PKA. Coincidence of mutated amino acids and those conserved between human and Drosophila strongly suggest significant functional relevance of this region, with major roles played by exons 12a and 15 and part of exon 16.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_031',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 267}]}],\n",
       "   'context': 'Minor lesion mutational spectrum of the entire NF1 gene does not explain its high mutability but points to a functional domain upstream of the GAP-related domain. More than 500 unrelated patients with neurofibromatosis type 1 (NF1) were screened for mutations in the NF1 gene. For each patient, the whole coding sequence and all splice sites were studied for aberrations, either by the protein truncation test (PTT), temperature-gradient gel electrophoresis (TGGE) of genomic PCR products, or, most often, by direct genomic sequencing (DGS) of all individual exons. A total of 301 sequence variants, including 278 bona fide pathogenic mutations, were identified. As many as 216 or 183 of the genuine mutations, comprising 179 or 161 different ones, can be considered novel when compared to the recent findings of Upadhyaya and Cooper, or to the NNFF mutation database. Mutation-detection efficiencies of the various screening methods were similar: 47.1% for PTT, 53.7% for TGGE, and 54.9% for DGS. Some 224 mutations (80.2%) yielded directly or indirectly premature termination codons. These mutations showed even distribution over the whole gene from exon 1 to exon 47. Of all sequence variants determined in our study, <20% represent C-->T or G-->A transitions within a CpG dinucleotide, and only six different mutations also occur in NF1 pseudogenes, with five being typical C-->T transitions in a CpG. Thus, neither frequent deamination of 5-methylcytosines nor interchromosomal gene conversion may account for the high mutation rate of the NF1 gene. As opposed to the truncating mutations, the 28 (10.1%) missense or single-amino-acid-deletion mutations identified clustered in two distinct regions, the GAP-related domain (GRD) and an upstream gene segment comprising exons 11-17. The latter forms a so-called cysteine/serine-rich domain with three cysteine pairs suggestive of ATP binding, as well as three potential cAMP-dependent protein kinase (PKA) recognition sites obviously phosphorylated by PKA. Coincidence of mutated amino acids and those conserved between human and Drosophila strongly suggest significant functional relevance of this region, with major roles played by exons 12a and 15 and part of exon 16.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_032',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 411}]}],\n",
       "   'context': 'Selective disactivation of neurofibromin GAP activity in neurofibromatosis type 1. Neurofibromatosis type 1 (NF1) is a common familial tumour syndrome with multiple clinical features such as neurofibromas, café-au-lait spots (CLS), iris Lisch nodules, axillary freckling, optic glioma, specific bone lesions and an increased risk of malignant tumours. It is caused by a wide spectrum of mutations affecting the NF1 gene. Most mutations result in the loss of one allele at the DNA, mRNA or protein level and thus in the loss of any function of the gene product neurofibromin. The idea of the simultaneous loss of several different neurofibromin functions has been postulated to explain the pleiotropic effects of its loss. However, we have identified a novel missense mutation in a family with a classical multi-symptomatic NF1 phenotype, including a malignant schwannoma, that specifically abolishes the Ras-GTPase-activating function of neurofibromin. In this family, Arg1276 had mutated into proline. Based on complex biochemical studies as well as the analysis of the crystal structure of the GTPase-activating protein (GAP) domain of p120GAP in the presence of Ras, we unequivocally identified this amino acid as the arginine finger of the neurofibromin GAP-related domain (GRD)-the most essential catalytic element for RasGAP activity. Here, we present data demonstrating that the mutation R1276P, unlike previously reported missense mutations of the GRD region, does not impair the secondary and tertiary protein structure. It neither reduces the level of cellular neurofibromin nor influences its binding to Ras substantially, but it does completely disable GAP activity. Our findings provide direct evidence that failure of neurofibromin GAP activity is the critical element of NF1 pathogenesis. Thus, therapeutic approaches aimed at the reduction of Ras.GTP levels in neural crest-derived cells can be expected to relieve most of the NF1 symptoms.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_033',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 109}]}],\n",
       "   'context': 'Selective disactivation of neurofibromin GAP activity in neurofibromatosis type 1. Neurofibromatosis type 1 (NF1) is a common familial tumour syndrome with multiple clinical features such as neurofibromas, café-au-lait spots (CLS), iris Lisch nodules, axillary freckling, optic glioma, specific bone lesions and an increased risk of malignant tumours. It is caused by a wide spectrum of mutations affecting the NF1 gene. Most mutations result in the loss of one allele at the DNA, mRNA or protein level and thus in the loss of any function of the gene product neurofibromin. The idea of the simultaneous loss of several different neurofibromin functions has been postulated to explain the pleiotropic effects of its loss. However, we have identified a novel missense mutation in a family with a classical multi-symptomatic NF1 phenotype, including a malignant schwannoma, that specifically abolishes the Ras-GTPase-activating function of neurofibromin. In this family, Arg1276 had mutated into proline. Based on complex biochemical studies as well as the analysis of the crystal structure of the GTPase-activating protein (GAP) domain of p120GAP in the presence of Ras, we unequivocally identified this amino acid as the arginine finger of the neurofibromin GAP-related domain (GRD)-the most essential catalytic element for RasGAP activity. Here, we present data demonstrating that the mutation R1276P, unlike previously reported missense mutations of the GRD region, does not impair the secondary and tertiary protein structure. It neither reduces the level of cellular neurofibromin nor influences its binding to Ras substantially, but it does completely disable GAP activity. Our findings provide direct evidence that failure of neurofibromin GAP activity is the critical element of NF1 pathogenesis. Thus, therapeutic approaches aimed at the reduction of Ras.GTP levels in neural crest-derived cells can be expected to relieve most of the NF1 symptoms.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_034',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 220}]}],\n",
       "   'context': 'Confirmation of a double-hit model for the NF1 gene in benign neurofibromas. Neurofibroma is a benign tumor that arises from small or large nerves. This neoplastic lesion is a common feature of neurofibromatosis type 1 (NF1), one of the most common autosomal dominant disorders. The NF1 gene codes for a protein called \"neurofibromin.\" It possesses a region that shares a high homology with the family of GTPase-activating proteins, which are negative regulators of RAS function and thereby control cell growth and differentiation. The evidence points to the NF1 gene being a tumor-suppressor gene. NF1 patients also have an increased incidence of certain malignant tumors that are believed to follow the \"two hit\" hypothesis, with one allele constitutionally inactivated and the other somatically mutated. Recently, somatic loss of heterozygosity (LOH) has been described for neurofibromas, and mutations in both copies of the NF1 gene have been reported for a dermal neurofibroma. The aim of our study was the analysis of the NF1 locus in benign neurofibromas in NF1 patients. We performed LOH analysis on 60 neurofibromas belonging to 17 patients, 9 of them with family history of the disease and 8 of them sporadic. We have analyzed five intragenic NF1 markers and six extragenic markers, and we have found LOH in 25% of the neurofibromas (corresponding to 53% of the patients). In addition, we found that in the neurofibromas of patients from familial cases the deletions occurred in the allele that is not transmitted with the disease, indicating that both copies of the NF1 gene were inactivated in these tumors. Therefore, the recent reports mentioned above, together with our findings, strongly support the double inactivation of the NF1 gene in benign neurofibromas.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_035',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 283}]}],\n",
       "   'context': 'Confirmation of a double-hit model for the NF1 gene in benign neurofibromas. Neurofibroma is a benign tumor that arises from small or large nerves. This neoplastic lesion is a common feature of neurofibromatosis type 1 (NF1), one of the most common autosomal dominant disorders. The NF1 gene codes for a protein called \"neurofibromin.\" It possesses a region that shares a high homology with the family of GTPase-activating proteins, which are negative regulators of RAS function and thereby control cell growth and differentiation. The evidence points to the NF1 gene being a tumor-suppressor gene. NF1 patients also have an increased incidence of certain malignant tumors that are believed to follow the \"two hit\" hypothesis, with one allele constitutionally inactivated and the other somatically mutated. Recently, somatic loss of heterozygosity (LOH) has been described for neurofibromas, and mutations in both copies of the NF1 gene have been reported for a dermal neurofibroma. The aim of our study was the analysis of the NF1 locus in benign neurofibromas in NF1 patients. We performed LOH analysis on 60 neurofibromas belonging to 17 patients, 9 of them with family history of the disease and 8 of them sporadic. We have analyzed five intragenic NF1 markers and six extragenic markers, and we have found LOH in 25% of the neurofibromas (corresponding to 53% of the patients). In addition, we found that in the neurofibromas of patients from familial cases the deletions occurred in the allele that is not transmitted with the disease, indicating that both copies of the NF1 gene were inactivated in these tumors. Therefore, the recent reports mentioned above, together with our findings, strongly support the double inactivation of the NF1 gene in benign neurofibromas.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_036',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 90}]}],\n",
       "   'context': '[Molecular-genetic aspects of neurofibromatosis]. Two forms of neurofibromatosis, type 1 (NF1) and type 2 (NF2) are connected with genes localized on chromosomes 17 and 22, respectively. The genes that are inactivated in neurofibromatosis code for the proteins neurofibromine and merline, respectively. Since inactivation leads to neoplasia, they are called tumour suppressor genes. Neurofibromine shows resemblances to proteins that serve to inactivate oncogenes. Merline has a relationship with proteins that connect the cytoskeleton and the cell membrane. The precise function of the proteins is still unknown. The NF1 gene is characterized by extraordinarily high sensitivity to mutation; half the NF1 patients have not inherited the disease. In the familial form of neurofibromatosis, a mutated gene is inherited and the normal allele in the tumour is inactivated, making tumour growth possible. In the sporadic form of neurofibromatosis, both normal alleles are inactivated locally in the tissue so that a tumour develops in that place.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_037',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 611}]}],\n",
       "   'context': 'Site and sequence specific DNA methylation in the neurofibromatosis (NF1) gene includes C5839T: the site of the recurrent substitution mutation in exon 31. CpG dinucleotides provide hotspots for transitional mutations in a variety of genes, some leading to genetic diseases in humans. Although this phenomenon is attributed to cytosine methylation at such sites, direct and specific observations of CpG methylation at the sites of recurrent mutations are lacking. We have used a bisulfite genomic sequencing method to analyze DNA methylation within three representative exons from the neurofibromatosis type 1 (NF1) gene, well recognized for its high frequency of spontaneous mutations. We observed that the cytosine methylation within NF1 exons 28, 29, and 31 is restricted to CpG dinucleotides, including the CpG dinucleotide present at the site of the recurrent NF1 mutation (C5839T; also referred to as R1947X). At several sites, clone-specific methylation differences were also observed. Our results provide experimental evidence for the hypothesis that methylatable CpGs in the NF1 gene contribute to spontaneous germline mutations associated with this gene, by showing that DNA methylation does occur at all CpGs contained within these representative NF1 exons. As well, the DNA methylation seen at the common mutation site in exon 31 may explain why this site is frequently mutated. Methylation-dependent mutagenesis may also provide a basis for some somatic (second hit) mutations which disable the normal allele and result in the development of NF1 associated symptoms.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_038',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 639}]}],\n",
       "   'context': '[Heterogeneity of GTPase-activating proteins for Ras in the regulation of Ras signal transduction pathway]. The proto-oncogene ras is an essential gene for the growth and the differentiation for various types of cells. Ras, ras gene product, is a GTP binding protein which controls the signal transduction by GTP hydrolysis. The ras gene is frequently activated by point mutations in various types of human cancers, which results in a decrease in the GTPase activity of its product. A GTPase-activating protein p120 (p120GAP) was identified as a factor which stimulates the GTPase of normal ras gene product p21 but not of the mutated. An NF1 gene was identified as a gene whose loss of function causes an onset of human disorder, neurofibromatosis type I. The NF1 gene encodes a protein which contains a region with a similarity to the catalytic domain of p120GAP. We recently purified a novel Ras GAP whose molecular weight and immunogenecity are different from those of p120GAP and NF1. We named the novel mammalian Ras GAP as Gap1m. Isolation and sequencing of Gap1m cDNA revealed that Gap1m is indeed a novel Ras GAP. We also succeeded in isolation of another novel Ras GAP gene, GapIII/Gap1IP4BP, which is closely related to Gap1m. Recently, it is shown that GapIII/Gap1IP4BP binds inositol-tetrakis phosphate compounds. The overview of these Ras GAP molecules is described.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_039',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 149}]}],\n",
       "   'context': \"Molecular characterization of the breakpoints of a 12-kb deletion in the NF1 gene in a family showing germ-line mosaicism. Neurofibromatosis type 1 (NF1) is caused by deletions, insertions, translocations, and point mutations in the NF1 gene, which spans 350 kb on the long arm of human chromosome 17. Although several point mutations have been described, large molecular abnormalities have rarely been characterized in detail. We describe here the molecular breakpoints of a 12-kb deletion of the NF1 gene, which is responsible for the NF1 phenotype in a kindred with two children affected because of germline mosaicism in the unaffected father, who has the mutation in 10% of his spermatozoa. The mutation spans introns 31-39, removing 12,021 nt and inserting 30 bp, of which 19 bp are a direct repetition of a sequence located in intron 31, just 4 bp before the 5' breakpoint. The 5' and 3' breakpoints contain the sequence TATTTTA, which could be involved in the generation of the deletion. The most plausible explanation for the mechanism involved in the generation of this 12-kb deletion is homologous/nonhomologous recombination. Since sperm of the father does not contain the corresponding insertion of the 12-kb deleted sequence, this deletion could have occurred within the NF1 chromosome through loop formation. RNA from lymphocytes of one of the NF1 patients showed similar levels of the mutated and normal transcripts, suggesting that the NF1-mRNA from mutations causing frame shifts of the reading frame or stop codons in this gene is not degraded during its processing. The mutation was not detected in fresh lymphocytes from the unaffected father by PCR analysis, supporting the case for true germ-line mosaicism.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_040',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 233}]}],\n",
       "   'context': \"Molecular characterization of the breakpoints of a 12-kb deletion in the NF1 gene in a family showing germ-line mosaicism. Neurofibromatosis type 1 (NF1) is caused by deletions, insertions, translocations, and point mutations in the NF1 gene, which spans 350 kb on the long arm of human chromosome 17. Although several point mutations have been described, large molecular abnormalities have rarely been characterized in detail. We describe here the molecular breakpoints of a 12-kb deletion of the NF1 gene, which is responsible for the NF1 phenotype in a kindred with two children affected because of germline mosaicism in the unaffected father, who has the mutation in 10% of his spermatozoa. The mutation spans introns 31-39, removing 12,021 nt and inserting 30 bp, of which 19 bp are a direct repetition of a sequence located in intron 31, just 4 bp before the 5' breakpoint. The 5' and 3' breakpoints contain the sequence TATTTTA, which could be involved in the generation of the deletion. The most plausible explanation for the mechanism involved in the generation of this 12-kb deletion is homologous/nonhomologous recombination. Since sperm of the father does not contain the corresponding insertion of the 12-kb deleted sequence, this deletion could have occurred within the NF1 chromosome through loop formation. RNA from lymphocytes of one of the NF1 patients showed similar levels of the mutated and normal transcripts, suggesting that the NF1-mRNA from mutations causing frame shifts of the reading frame or stop codons in this gene is not degraded during its processing. The mutation was not detected in fresh lymphocytes from the unaffected father by PCR analysis, supporting the case for true germ-line mosaicism.\"},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_041',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 127}]}],\n",
       "   'context': 'Analysis of mutations of neurofibromatosis type 1 gene and N-ras gene in acute myelogenous leukemia. Neurofibromatosis type 1 (NF1) gene is a tumor suppressor gene, and the NF1 gene product, neurofibromin, can downregulate the N-ras gene. Because the N-ras gene is often mutated in acute myelogenous leukemia (AML), we wondered if the NF1 gene might be mutated in those AML samples not having N-ras mutations. We investigated the mutational status of the N-ras gene and the FLR exon of codons 1371-1423 of the open reading frame of the full-length NF1 cDNA, which has a strong homology with the mammalian ras GTPase-activating protein (GAP), especially for a stretch of three consecutive amino acids (F, L, R), by single-strand conformation polymorphism analysis and direct sequencing in samples from patients with AML. Of 48 AML patients, 10 (21%) had point (missense) mutations of the N-ras gene involving codons 12, 13 and 61. However, mutations in the FLR exon of the NF1 gene were not detected in any of the AML samples. We also examined the difference of clinical response to induction therapy between AML patients with and without N-ras mutation. A significantly lower rate of complete remission was noted in individuals with N-ras gene mutations. These results suggest that mutation of the NF1 gene, at least in the FLR exon, is very rare in AML and the NF1 gene probably is not a functional complement of the N-ras gene mutation. The presence of N-ras gene mutation may be associated with a lower clinical response to antileukemic therapy.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_042',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 173}]}],\n",
       "   'context': 'Analysis of mutations of neurofibromatosis type 1 gene and N-ras gene in acute myelogenous leukemia. Neurofibromatosis type 1 (NF1) gene is a tumor suppressor gene, and the NF1 gene product, neurofibromin, can downregulate the N-ras gene. Because the N-ras gene is often mutated in acute myelogenous leukemia (AML), we wondered if the NF1 gene might be mutated in those AML samples not having N-ras mutations. We investigated the mutational status of the N-ras gene and the FLR exon of codons 1371-1423 of the open reading frame of the full-length NF1 cDNA, which has a strong homology with the mammalian ras GTPase-activating protein (GAP), especially for a stretch of three consecutive amino acids (F, L, R), by single-strand conformation polymorphism analysis and direct sequencing in samples from patients with AML. Of 48 AML patients, 10 (21%) had point (missense) mutations of the N-ras gene involving codons 12, 13 and 61. However, mutations in the FLR exon of the NF1 gene were not detected in any of the AML samples. We also examined the difference of clinical response to induction therapy between AML patients with and without N-ras mutation. A significantly lower rate of complete remission was noted in individuals with N-ras gene mutations. These results suggest that mutation of the NF1 gene, at least in the FLR exon, is very rare in AML and the NF1 gene probably is not a functional complement of the N-ras gene mutation. The presence of N-ras gene mutation may be associated with a lower clinical response to antileukemic therapy.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_043',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 262}]}],\n",
       "   'context': 'Functional significance of lysine 1423 of neurofibromin and characterization of a second site suppressor which rescues mutations at this residue and suppresses RAS2Val-19-activated phenotypes. Lysine 1423 of neurofibromin (neurofibromatosis type I gene product [NF1]) plays a crucial role in the function of NF1. Mutations of this lysine were detected in samples from a neurofibromatosis patient as well as from cancer patients. To further understand the significance of this residue, we have mutated it to all possible amino acids. Functional assays using yeast ira complementation have revealed that lysine is the only amino acid that produced functional NF1. Quantitative analyses of different mutant proteins have suggested that their GTPase-activating protein (GAP) activity is drastically reduced as a result of a decrease in their Ras affinity. Such a requirement for a specific residue is not observed in the case of other conserved residues within the GAP-related domain. We also report that another residue, phenylalanine 1434, plays an important role in NF1 function. This was first indicated by the finding that defective NF1s due to an alteration of lysine 1423 to other amino acids can be rescued by a second site intragenic mutation at residue 1434. The mutation partially restored GAP activity in the lysine mutant. When the mutation phenylalanine 1434 to serine was introduced into a wild-type NF1 protein, the resulting protein acquired the ability to suppress activated phenotypes of RAS2Val-19 cells. This suppression, however, does not involve Ras interaction, since the phenylalanine mutant does not stimulate the intrinsic GTPase activity of RAS2Val-19 protein and does not have an increased affinity for Ras proteins.'},\n",
       "  {'qas': [{'id': '5343fc1aaeec6fbd07000003_044',\n",
       "     'question': 'Which is the gene mutated in type 1 neurofibromatosis?',\n",
       "     'answers': [{'text': 'NF1', 'answer_start': 308}]}],\n",
       "   'context': 'Functional significance of lysine 1423 of neurofibromin and characterization of a second site suppressor which rescues mutations at this residue and suppresses RAS2Val-19-activated phenotypes. Lysine 1423 of neurofibromin (neurofibromatosis type I gene product [NF1]) plays a crucial role in the function of NF1. Mutations of this lysine were detected in samples from a neurofibromatosis patient as well as from cancer patients. To further understand the significance of this residue, we have mutated it to all possible amino acids. Functional assays using yeast ira complementation have revealed that lysine is the only amino acid that produced functional NF1. Quantitative analyses of different mutant proteins have suggested that their GTPase-activating protein (GAP) activity is drastically reduced as a result of a decrease in their Ras affinity. Such a requirement for a specific residue is not observed in the case of other conserved residues within the GAP-related domain. We also report that another residue, phenylalanine 1434, plays an important role in NF1 function. This was first indicated by the finding that defective NF1s due to an alteration of lysine 1423 to other amino acids can be rescued by a second site intragenic mutation at residue 1434. The mutation partially restored GAP activity in the lysine mutant. When the mutation phenylalanine 1434 to serine was introduced into a wild-type NF1 protein, the resulting protein acquired the ability to suppress activated phenotypes of RAS2Val-19 cells. This suppression, however, does not involve Ras interaction, since the phenylalanine mutant does not stimulate the intrinsic GTPase activity of RAS2Val-19 protein and does not have an increased affinity for Ras proteins.'},\n",
       "  {'qas': [{'id': '516be1d6298dcd4e5100006a_001',\n",
       "     'question': 'What is the mode of inheritance of nemaline myopathy?',\n",
       "     'answers': [{'text': 'Autosomal recessive', 'answer_start': 477}]}],\n",
       "   'context': 'Genotype-phenotype correlations in nemaline myopathy caused by mutations in the genes for nebulin and skeletal muscle alpha-actin. We present comparisons of the clinical pictures in a series of 60 patients with nemaline myopathy in whom mutations had been identified in the genes for nebulin or skeletal muscle alpha-actin. In the patients with nebulin mutations, the typical form of nemaline myopathy predominated, while severe, mild or intermediate forms were less frequent. Autosomal recessive inheritance had been verified or appeared likely in all nebulin cases. In the patients with actin mutations, the severe form of nemaline myopathy was the most common, but some had the mild or typical form, and a few showed other associated features such as intranuclear rods or actin accumulation. Most cases were sporadic, but in addition there were instances of both autosomal dominant and autosomal recessive inheritance, while two families showed mosaicism for dominant mutations. Although no specific phenotype was found to be associated with mutations in either gene, clinical and histological features together with pedigree data may be used in guiding mutation detection. Finding the causative mutation(s) determines the mode of inheritance and permits prenatal diagnosis if requested, but will not as such permit prognostication.'},\n",
       "  {'qas': [{'id': '516be1d6298dcd4e5100006a_002',\n",
       "     'question': 'What is the mode of inheritance of nemaline myopathy?',\n",
       "     'answers': [{'text': 'autosomal dominant', 'answer_start': 866}]}],\n",
       "   'context': 'Genotype-phenotype correlations in nemaline myopathy caused by mutations in the genes for nebulin and skeletal muscle alpha-actin. We present comparisons of the clinical pictures in a series of 60 patients with nemaline myopathy in whom mutations had been identified in the genes for nebulin or skeletal muscle alpha-actin. In the patients with nebulin mutations, the typical form of nemaline myopathy predominated, while severe, mild or intermediate forms were less frequent. Autosomal recessive inheritance had been verified or appeared likely in all nebulin cases. In the patients with actin mutations, the severe form of nemaline myopathy was the most common, but some had the mild or typical form, and a few showed other associated features such as intranuclear rods or actin accumulation. Most cases were sporadic, but in addition there were instances of both autosomal dominant and autosomal recessive inheritance, while two families showed mosaicism for dominant mutations. Although no specific phenotype was found to be associated with mutations in either gene, clinical and histological features together with pedigree data may be used in guiding mutation detection. Finding the causative mutation(s) determines the mode of inheritance and permits prenatal diagnosis if requested, but will not as such permit prognostication.'},\n",
       "  {'qas': [{'id': '516be1d6298dcd4e5100006a_003',\n",
       "     'question': 'What is the mode of inheritance of nemaline myopathy?',\n",
       "     'answers': [{'text': 'autosomal recessive', 'answer_start': 889}]}],\n",
       "   'context': 'Genotype-phenotype correlations in nemaline myopathy caused by mutations in the genes for nebulin and skeletal muscle alpha-actin. We present comparisons of the clinical pictures in a series of 60 patients with nemaline myopathy in whom mutations had been identified in the genes for nebulin or skeletal muscle alpha-actin. In the patients with nebulin mutations, the typical form of nemaline myopathy predominated, while severe, mild or intermediate forms were less frequent. Autosomal recessive inheritance had been verified or appeared likely in all nebulin cases. In the patients with actin mutations, the severe form of nemaline myopathy was the most common, but some had the mild or typical form, and a few showed other associated features such as intranuclear rods or actin accumulation. Most cases were sporadic, but in addition there were instances of both autosomal dominant and autosomal recessive inheritance, while two families showed mosaicism for dominant mutations. Although no specific phenotype was found to be associated with mutations in either gene, clinical and histological features together with pedigree data may be used in guiding mutation detection. Finding the causative mutation(s) determines the mode of inheritance and permits prenatal diagnosis if requested, but will not as such permit prognostication.'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_001',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 72}]}],\n",
       "   'context': 'A novel mutation in the endosomal Na+/H+ exchanger NHE6 (SLC9A6) causes Christianson syndrome with electrical status epilepticus during slow-wave sleep (ESES). Mutations in the solute carrier family 9, subfamily A member 6 (SLC9A6) gene, encoding the endosomal Na+/H+ exchanger 6 (NHE6) are associated with Christianson syndrome, a syndromic form of X-linked intellectual disability characterized by microcephaly, severe global developmental delay, autistic behavior, early onset seizures and ataxia. In a 7-year-old boy with characteristic clinical and neuroimaging features of Christianson syndrome and epileptic encephalopathy with continuous spikes and waves during sleep, we identified a novel splice site mutation (IVS10-1G>A) in SLC9A6. These findings expand the clinical spectrum of the syndrome and indicate NHE6 dysfunction as a new cause of electrical status epilepticus during slow-wave sleep (ESES).'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_002',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 307}]}],\n",
       "   'context': 'A novel mutation in the endosomal Na+/H+ exchanger NHE6 (SLC9A6) causes Christianson syndrome with electrical status epilepticus during slow-wave sleep (ESES). Mutations in the solute carrier family 9, subfamily A member 6 (SLC9A6) gene, encoding the endosomal Na+/H+ exchanger 6 (NHE6) are associated with Christianson syndrome, a syndromic form of X-linked intellectual disability characterized by microcephaly, severe global developmental delay, autistic behavior, early onset seizures and ataxia. In a 7-year-old boy with characteristic clinical and neuroimaging features of Christianson syndrome and epileptic encephalopathy with continuous spikes and waves during sleep, we identified a novel splice site mutation (IVS10-1G>A) in SLC9A6. These findings expand the clinical spectrum of the syndrome and indicate NHE6 dysfunction as a new cause of electrical status epilepticus during slow-wave sleep (ESES).'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_003',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 54}]}],\n",
       "   'context': 'Genetic and phenotypic diversity of NHE6 mutations in Christianson syndrome. OBJECTIVE: Recently, Christianson syndrome (CS) has been determined to be caused by mutations in the X-linked Na(+) /H(+) exchanger 6 (NHE6). We aimed to determine the diagnostic criteria and mutational spectrum for CS. METHODS: Twelve independent pedigrees (14 boys, age = 4-19 years) with mutations in NHE6 were administered standardized research assessments, and mutations were characterized. RESULTS: The mutational spectrum was composed of 9 single nucleotide variants, 2 indels, and 1 copy number variation deletion. All mutations were protein-truncating or splicing mutations. We identified 2 recurrent mutations (c.1498 c>t, p.R500X; and c.1710 g>a, p.W570X). Otherwise, all mutations were unique. In our study, 7 of 12 mutations (58%) were de novo, in contrast to prior literature wherein mutations were largely inherited. We also report prominent neurological, medical, and behavioral symptoms. All CS participants were nonverbal and had intellectual disability, epilepsy, and ataxia. Many had prior diagnoses of autism and/or Angelman syndrome. Other neurologic symptoms included eye movement abnormalities (79%), postnatal microcephaly (92%), and magnetic resonance imaging evidence of cerebellar atrophy (33%). Regression was noted in 50%, with recurrent presentations involving loss of words and/or the ability to walk. Medical symptoms, particularly gastrointestinal symptoms, were common. Height and body mass index measures were below normal ranges in most participants. Behavioral symptoms included hyperkinetic behavior (100%), and a majority exhibited high pain threshold. INTERPRETATION: This is the largest cohort of independent CS pedigrees reported. We propose diagnostic criteria for CS. CS represents a novel neurogenetic disorder with general relevance to autism, intellectual disability, Angelman syndrome, epilepsy, and regression.'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_004',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 98}]}],\n",
       "   'context': 'Genetic and phenotypic diversity of NHE6 mutations in Christianson syndrome. OBJECTIVE: Recently, Christianson syndrome (CS) has been determined to be caused by mutations in the X-linked Na(+) /H(+) exchanger 6 (NHE6). We aimed to determine the diagnostic criteria and mutational spectrum for CS. METHODS: Twelve independent pedigrees (14 boys, age = 4-19 years) with mutations in NHE6 were administered standardized research assessments, and mutations were characterized. RESULTS: The mutational spectrum was composed of 9 single nucleotide variants, 2 indels, and 1 copy number variation deletion. All mutations were protein-truncating or splicing mutations. We identified 2 recurrent mutations (c.1498 c>t, p.R500X; and c.1710 g>a, p.W570X). Otherwise, all mutations were unique. In our study, 7 of 12 mutations (58%) were de novo, in contrast to prior literature wherein mutations were largely inherited. We also report prominent neurological, medical, and behavioral symptoms. All CS participants were nonverbal and had intellectual disability, epilepsy, and ataxia. Many had prior diagnoses of autism and/or Angelman syndrome. Other neurologic symptoms included eye movement abnormalities (79%), postnatal microcephaly (92%), and magnetic resonance imaging evidence of cerebellar atrophy (33%). Regression was noted in 50%, with recurrent presentations involving loss of words and/or the ability to walk. Medical symptoms, particularly gastrointestinal symptoms, were common. Height and body mass index measures were below normal ranges in most participants. Behavioral symptoms included hyperkinetic behavior (100%), and a majority exhibited high pain threshold. INTERPRETATION: This is the largest cohort of independent CS pedigrees reported. We propose diagnostic criteria for CS. CS represents a novel neurogenetic disorder with general relevance to autism, intellectual disability, Angelman syndrome, epilepsy, and regression.'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_005',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 0}]}],\n",
       "   'context': 'Christianson syndrome protein NHE6 modulates TrkB endosomal signaling required for neuronal circuit development. Neuronal arborization is regulated by cell-autonomous and nonautonomous mechanisms including endosomal signaling via BDNF/TrkB. The endosomal Na⁺/H⁺ exchanger 6 (NHE6) is mutated in a new autism-related disorder. NHE6 functions to permit proton leak from endosomes, yet the mechanisms causing disease are unknown. We demonstrate that loss of NHE6 results in overacidification of the endosomal compartment and attenuated TrkB signaling. Mouse brains with disrupted NHE6 display reduced axonal and dendritic branching, synapse number, and circuit strength. Site-directed mutagenesis shows that the proton leak function of NHE6 is required for neuronal arborization. We find that TrkB receptor colocalizes to NHE6-associated endosomes. TrkB protein and phosphorylation are reduced in NHE6 mutant neurons in response to BDNF signaling. Finally, exogenous BDNF rescues defects in neuronal arborization. We propose that NHE6 mutation leads to circuit defects that are in part due to impoverished neuronal arborization that may be treatable by enhanced TrkB signaling.'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_006',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 98}]}],\n",
       "   'context': 'Genetic and phenotypic diversity of NHE6 mutations in Christianson syndrome. OBJECTIVE: Recently, Christianson syndrome (CS) has been determined to be caused by mutations in the X-linked Na(+) /H(+) exchanger 6 (NHE6). We aimed to determine the diagnostic criteria and mutational spectrum for CS. METHODS: Twelve independent pedigrees (14 boys, age = 4-19 years) with mutations in NHE6 were administered standardized research assessments, and mutations were characterized. RESULTS: The mutational spectrum was composed of 9 single nucleotide variants, 2 indels, and 1 copy number variation deletion. All mutations were protein-truncating or splicing mutations. We identified 2 recurrent mutations (c.1498 c>t, p.R500X; and c.1710 g>a, p.W570X). Otherwise, all mutations were unique. In our study, 7 of 12 mutations (58%) were de novo, in contrast to prior literature wherein mutations were largely inherited. We also report prominent neurological, medical, and behavioral symptoms. All CS participants were nonverbal and had intellectual disability, epilepsy, and ataxia. Many had prior diagnoses of autism and/or Angelman syndrome. Other neurologic symptoms included eye movement abnormalities (79%), postnatal microcephaly (92%), and magnetic resonance imaging evidence of cerebellar atrophy (33%). Regression was noted in 50%, with recurrent presentations involving loss of words and/or the ability to walk. Medical symptoms, particularly gastrointestinal symptoms, were common. Height and body mass index measures were below normal ranges in most participants. Behavioral symptoms included hyperkinetic behavior (100%), and a majority exhibited high pain threshold. INTERPRETATION: This is the largest cohort of independent CS pedigrees reported. We propose diagnostic criteria for CS. CS represents a novel neurogenetic disorder with general relevance to autism, intellectual disability, Angelman syndrome, epilepsy, and regression.'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_007',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 72}]}],\n",
       "   'context': 'A novel mutation in the endosomal Na+/H+ exchanger NHE6 (SLC9A6) causes Christianson syndrome with electrical status epilepticus during slow-wave sleep (ESES). Mutations in the solute carrier family 9, subfamily A member 6 (SLC9A6) gene, encoding the endosomal Na+/H+ exchanger 6 (NHE6) are associated with Christianson syndrome, a syndromic form of X-linked intellectual disability characterized by microcephaly, severe global developmental delay, autistic behavior, early onset seizures and ataxia. In a 7-year-old boy with characteristic clinical and neuroimaging features of Christianson syndrome and epileptic encephalopathy with continuous spikes and waves during sleep, we identified a novel splice site mutation (IVS10-1G>A) in SLC9A6. These findings expand the clinical spectrum of the syndrome and indicate NHE6 dysfunction as a new cause of electrical status epilepticus during slow-wave sleep (ESES).'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_008',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 307}]}],\n",
       "   'context': 'A novel mutation in the endosomal Na+/H+ exchanger NHE6 (SLC9A6) causes Christianson syndrome with electrical status epilepticus during slow-wave sleep (ESES). Mutations in the solute carrier family 9, subfamily A member 6 (SLC9A6) gene, encoding the endosomal Na+/H+ exchanger 6 (NHE6) are associated with Christianson syndrome, a syndromic form of X-linked intellectual disability characterized by microcephaly, severe global developmental delay, autistic behavior, early onset seizures and ataxia. In a 7-year-old boy with characteristic clinical and neuroimaging features of Christianson syndrome and epileptic encephalopathy with continuous spikes and waves during sleep, we identified a novel splice site mutation (IVS10-1G>A) in SLC9A6. These findings expand the clinical spectrum of the syndrome and indicate NHE6 dysfunction as a new cause of electrical status epilepticus during slow-wave sleep (ESES).'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_009',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 98}]}],\n",
       "   'context': 'Genetic and phenotypic diversity of NHE6 mutations in Christianson syndrome. OBJECTIVE: Recently, Christianson syndrome (CS) has been determined to be caused by mutations in the X-linked Na(+) /H(+) exchanger 6 (NHE6). We aimed to determine the diagnostic criteria and mutational spectrum for CS. METHODS: Twelve independent pedigrees (14 boys, age = 4-19 years) with mutations in NHE6 were administered standardized research assessments, and mutations were characterized. RESULTS: The mutational spectrum was composed of 9 single nucleotide variants, 2 indels, and 1 copy number variation deletion. All mutations were protein-truncating or splicing mutations. We identified 2 recurrent mutations (c.1498 c>t, p.R500X; and c.1710 g>a, p.W570X). Otherwise, all mutations were unique. In our study, 7 of 12 mutations (58%) were de novo, in contrast to prior literature wherein mutations were largely inherited. We also report prominent neurological, medical, and behavioral symptoms. All CS participants were nonverbal and had intellectual disability, epilepsy, and ataxia. Many had prior diagnoses of autism and/or Angelman syndrome. Other neurologic symptoms included eye movement abnormalities (79%), postnatal microcephaly (92%), and magnetic resonance imaging evidence of cerebellar atrophy (33%). Regression was noted in 50%, with recurrent presentations involving loss of words and/or the ability to walk. Medical symptoms, particularly gastrointestinal symptoms, were common. Height and body mass index measures were below normal ranges in most participants. Behavioral symptoms included hyperkinetic behavior (100%), and a majority exhibited high pain threshold. INTERPRETATION: This is the largest cohort of independent CS pedigrees reported. We propose diagnostic criteria for CS. CS represents a novel neurogenetic disorder with general relevance to autism, intellectual disability, Angelman syndrome, epilepsy, and regression.'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_010',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 307}]}],\n",
       "   'context': 'A novel mutation in the endosomal Na+/H+ exchanger NHE6 (SLC9A6) causes Christianson syndrome with electrical status epilepticus during slow-wave sleep (ESES). Mutations in the solute carrier family 9, subfamily A member 6 (SLC9A6) gene, encoding the endosomal Na+/H+ exchanger 6 (NHE6) are associated with Christianson syndrome, a syndromic form of X-linked intellectual disability characterized by microcephaly, severe global developmental delay, autistic behavior, early onset seizures and ataxia. In a 7-year-old boy with characteristic clinical and neuroimaging features of Christianson syndrome and epileptic encephalopathy with continuous spikes and waves during sleep, we identified a novel splice site mutation (IVS10-1G>A) in SLC9A6. These findings expand the clinical spectrum of the syndrome and indicate NHE6 dysfunction as a new cause of electrical status epilepticus during slow-wave sleep (ESES).'},\n",
       "  {'qas': [{'id': '56d860ad51531f7e33000002_011',\n",
       "     'question': 'Which syndrome is NHE6 associated with?',\n",
       "     'answers': [{'text': 'Christianson syndrome', 'answer_start': 307}]}],\n",
       "   'context': 'A novel mutation in the endosomal Na+/H+ exchanger NHE6 (SLC9A6) causes Christianson syndrome with electrical status epilepticus during slow-wave sleep (ESES). Mutations in the solute carrier family 9, subfamily A member 6 (SLC9A6) gene, encoding the endosomal Na+/H+ exchanger 6 (NHE6) are associated with Christianson syndrome, a syndromic form of X-linked intellectual disability characterized by microcephaly, severe global developmental delay, autistic behavior, early onset seizures and ataxia. In a 7-year-old boy with characteristic clinical and neuroimaging features of Christianson syndrome and epileptic encephalopathy with continuous spikes and waves during sleep, we identified a novel splice site mutation (IVS10-1G>A) in SLC9A6. These findings expand the clinical spectrum of the syndrome and indicate NHE6 dysfunction as a new cause of electrical status epilepticus during slow-wave sleep (ESES).'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_001',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 223}]}],\n",
       "   'context': \"Nuclear and neuritic distribution of serine-129 phosphorylated alpha-synuclein in transgenic mice. Parkinson's disease and dementia with Lewy bodies are very frequent neurological disorders of the elderly. Mutations in the alpha-synuclein (alphaSYN) gene cause Parkinson's disease, often associated with dementia. Neuropathologically these diseases are characterized by the presence of Lewy bodies and Lewy neurites, intraneuronal inclusions mostly composed of alphaSYN protein fibrils. Moreover, alphaSYN is phosphorylated at S129 (phospho-serine-129 [PSer129]) in neuropathological lesions. Using our (Thy1)-[A30P]alphaSYN transgenic mouse model that develops age-dependent impairment in fear conditioning behavior, we investigated PSer129 immunostaining in the brain. We found distinct staining patterns using new, sensitive monoclonal antibodies. Somal and nuclear PSer129 immunoreactivity increased with age in hippocampal and cortical areas as well as the lateral/basolateral amygdalar nuclei and was present also in young, pre-symptomatic mice, but not wild-type controls. The tendency of PSer129 immunostaining to accumulate in the nucleus was confirmed in cell culture. (Thy1)-[A30P]alphaSYN transgenic mice further developed age-dependent, specific neuritic/terminal alphaSYN pathology in the medial parts of the central amygdalar nucleus and one of its projection areas, the lateral hypothalamus. Interestingly, this type of PSer129 neuropathology was thioflavine S negative, unlike the Lewy-like neuropathology present in the brain stem of (Thy1)-[A30P]alphaSYN mice. Thus, alphaSYN becomes phosphorylated in distinct parts of the brain in this alpha-synucleinopathy mouse model, showing age-dependent increases of nuclear PSer129 in cortical brain areas and the formation of neuritic/terminal PSer129 neuropathology with variable amyloid quality within the fear conditioning circuitry and the brain stem.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_002',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 707}]}],\n",
       "   'context': 'Lewy bodies in progressive supranuclear palsy represent an independent disease process. Progressive supranuclear palsy (PSP) is a neurodegenerative tauopathy characterized by Parkinsonism, vertical gaze palsy, and early falls. Lewy bodies (LBs) are detected in approximately 10% of PSP cases, but there is little information on the relationship of LBs to tau pathology. We determined the frequency of LBs in a large series of autopsy-confirmed cases of PSP and studied the density and distribution of LBs, including Parkinson disease stage, in cases with LBs (PSP/LBD). PSP/LBD was compared with pure LB disease (LBD), including assessment of neuronal loss in key brainstem nuclei. Immunohistochemistry for alpha-synuclein revealed LBs in 31 of 290 PSP cases (11%). One case had multiple system atrophy in addition to PSP and was excluded from further study along with 2 PSP/LBD cases with concurrent Alzheimer disease. The 29 cases of PSP/LBD were compared with 30 cases of PSP and 24 cases of LBD. The age, sex, brain weight, Braak neurofibrillary tangle (NFT) stage, as well as counts of NFTs and senile plaques were not different among PSP, LBD, and PSP/LBD, but disease duration was longer in LBD. The Parkinson disease stage was similar, but the density of LBs in most subcortical nuclei tended to be greater in LBD than in PSP/LBD. In contrast, substantia nigra neuronal loss was greater in PSP/LBD than both PSP and LBD. Double immunostaining demonstrated alpha-synuclein and tau in different neurons with few exceptions. The findings suggest that LBs in PSP are similar in distribution to those in LBD and independent of tau pathology. The greater density of LBs in LBD compared with PSP/LBD may be the result of longer disease duration in LBD, whereas greater neuronal loss in the substantia nigra in PSP/LBD may be the result of vulnerability of this brain region to both disease processes.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_003',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 620}]}],\n",
       "   'context': 'The nigro-striatal and nigro-amygdaloid pathways undergo different degeneration processes in brains of dementia with Lewy bodies. We immunohistochemically investigated the degeneration processes of the nigro-striatal and nigro-amygdaloid pathways and the relationship between the loss of dopaminergic neurons and Lewy bodies (LB) formation in the substantia nigra using 15 autopsied cases of dementia with Lewy bodies (DLB). The number of tyrosine hydroxylase (TH)-positive neurons in the substantia nigra and TH-positive axonal terminals in the putamen decreased with a specific pattern. The substantia nigra possessed alpha-synuclein-positive LB-bearing neurons that were almost evenly distributed, while the putamen exhibited diffuse or granular alpha-synuclein-immunostaining. Most of the granular stains were positive for anti-phosphorylated alpha-synuclein antibody, whereas the diffuse stains were negative. These findings suggest that the axonal terminals in the putamen undergo abnormal alpha-synuclein accumulation, but may not always originate from LB-bearing neurons in the substantia nigra. The central amygdaloid nucleus contained anti-alpha-synuclein- and -phosphorylated alpha-synuclein-positive dystrophic axonal terminals, the degree of which was greater for cases with granular staining in the putamen, and which was proportional to the number of alpha-synuclein-positive neurons in the substantia nigra. Thus, the axonal terminals in the central amygdaloid nucleus may have originated from LB-bearing neurons in the substantia nigra. The results of the present study indicate that the nigro-striatal and nigro-amygdaloid pathways undergo different degeneration processes in DLB, and suggest that the degeneration of the nigro-amygdaloid pathway more strongly reflects LB formation in dopaminergic neurons of the substantia nigra than that of the nigro-striatal pathway. In addition, they indicate that there is no direct relationship between the loss of dopaminergic neurons and LB formation in the substantia nigra.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_004',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 749}]}],\n",
       "   'context': 'The nigro-striatal and nigro-amygdaloid pathways undergo different degeneration processes in brains of dementia with Lewy bodies. We immunohistochemically investigated the degeneration processes of the nigro-striatal and nigro-amygdaloid pathways and the relationship between the loss of dopaminergic neurons and Lewy bodies (LB) formation in the substantia nigra using 15 autopsied cases of dementia with Lewy bodies (DLB). The number of tyrosine hydroxylase (TH)-positive neurons in the substantia nigra and TH-positive axonal terminals in the putamen decreased with a specific pattern. The substantia nigra possessed alpha-synuclein-positive LB-bearing neurons that were almost evenly distributed, while the putamen exhibited diffuse or granular alpha-synuclein-immunostaining. Most of the granular stains were positive for anti-phosphorylated alpha-synuclein antibody, whereas the diffuse stains were negative. These findings suggest that the axonal terminals in the putamen undergo abnormal alpha-synuclein accumulation, but may not always originate from LB-bearing neurons in the substantia nigra. The central amygdaloid nucleus contained anti-alpha-synuclein- and -phosphorylated alpha-synuclein-positive dystrophic axonal terminals, the degree of which was greater for cases with granular staining in the putamen, and which was proportional to the number of alpha-synuclein-positive neurons in the substantia nigra. Thus, the axonal terminals in the central amygdaloid nucleus may have originated from LB-bearing neurons in the substantia nigra. The results of the present study indicate that the nigro-striatal and nigro-amygdaloid pathways undergo different degeneration processes in DLB, and suggest that the degeneration of the nigro-amygdaloid pathway more strongly reflects LB formation in dopaminergic neurons of the substantia nigra than that of the nigro-striatal pathway. In addition, they indicate that there is no direct relationship between the loss of dopaminergic neurons and LB formation in the substantia nigra.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_005',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 217}]}],\n",
       "   'context': 'Colocalization of tau and alpha-synuclein epitopes in Lewy bodies. The major protein constituent of Lewy bodies (LBs), the pathological hallmark of Parkinson disease and dementia with Lewy bodies, is considered to be alpha-synuclein, but other proteins, in particular the microtubule-associated protein tau, have been implicated in the pathogenesis of LBs. Tau is the major structural component of neurofibrillary tangles (NFTs). Both direct immunochemical studies of partially purified LBs and indirect immunohistochemical studies have suggested that LBs may contain tau, but most of these studies were based upon a single tau antibody, and immunologic cross-reactivity was not completely excluded. To gain insight into the relation between tau and alpha-synuclein in LBs, double immunostaining was performed in Lewy body cases with a rabbit polyclonal antibody to alpha-synuclein and a panel of monoclonal antibodies to phospho- and nonphospho-tau epitopes (Alz50, CP9, CP13, PG5, TG3, PHFI) that spanned the length of the tau molecule. Tau-immunoreactive LBs were present in the medulla in 80% of the cases, irrespective of Braak stage. All tau antibodies recognized at least some LBs, arguing against nonspecific antibody cross-reactivity. In most lesions the tau immunostaining was present at the periphery of the LB. The phospho-tau antibody, TG3, detected more LBs than any of the other tau antibodies. The proportion of LBs with tau immunoreactivity was greatest in neurons vulnerable to NETs, such as those in the locus ceruleus and basal nucleus of Meynert, and least in neurons resistant to NFTs, such as the dorsal motor nucleus of the vagus in the medulla. The present results suggest that tau may coaggregate with alpha-synuclein in LBs, especially in neuronal populations vulnerable to both NFTs and LBs.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_006',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 225}]}],\n",
       "   'context': 'Glial involvement in diffuse Lewy body disease. Diffuse Lewy body disease (DLBD) is characterized by the presence of Lewy bodies (LB) in the neurons and neurites of cortical, subcortical, and brain stem structures. Recently, alpha-synuclein (alphaS) has been found to be a central constituent of LB. In DLBD, abnormal accumulation of alphaS has been reported in both neurons and glia, but studies on glial lesions in DLBD have been limited. We examined in detail the constituents and distribution of glial lesions in eight patients with DLBD and report the pathogenesis of the glial lesions. alphaS-positive neuronal cytoplasmic inclusions (NI), neuropil threads (NT), and coiled bodies (CB) showed similar immunostaining profiles. Without pretreatment, NI, NT, and CB were detected by all antibodies against alphaS. The immunostaining profile of star-like astrocytes (SLA) was quite different from those of NI, NT, and CB. A few SLA were stained by an antibody against the non-Abeta component portion of alphaS without pretreatment, but formic acid pretreatment dramatically enhanced SLA immunoreactivity. SLA and CB were found in all eight brains with DLBD. SLA were scarce in the brain stem, but there were hundreds of SLA per visual field at x100 magnification in the temporal cortex of most cases, while CB were found diffusely in both the cerebral cortex and brain stem, similar to NI. This suggests that the pathogenesis of SLA is different from those of NI and CB.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_007',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 616}]}],\n",
       "   'context': \"Alpha-synuclein is not a requisite component of synaptic boutons in the adult human central nervous system. It is increasingly clear that the normal protein alpha-synuclein is in some manner closely associated with presynaptic components of select neuronal types within the adult human central nervous system (CNS) and, in addition, that in its pathologically altered state alpha-synuclein aggregates selectively in the form of filamentous inclusion bodies during certain progressive neurodegenerative disorders, such as familial and sporadic Parkinson's disease. By having the antibody AFshp raised specifically to alpha-synuclein to label Parkinson disease-specific Lewy bodies and Lewy neurites as well as synaptic boutons containing the unaltered protein, an initial attempt is made to map the overall distribution pattern and describe the staining behavior of the immunoreactive punctae in select regions of the prosencephalon. Neocortical immunolabeling is most prominent in the prodigious, but incompletely myelinated, association fields and faintest in the heavily myelinated primary motor and primary sensory fields, with the premotor and first order sensory association areas occupying an intermediate position. Of the thalamic grays evaluated, those containing powerfully myelinated fiber tracts (e.g. centrum medianum, habenular complex) show the weakest immunolabeling, whereas, less sturdily myelinated structures are highly immunoreactive. The fact that the immunostaining spectrum for normal alpha-synuclein is so broad, together with the fact that some thalamic sites actually are immunonegative leads to the following conclusions (1) alpha-synuclein, although present in the synaptic boutons of many nerve cells in the adult human CNS, is by no means ubiquitous there, and (2) neuronal types lacking the normal protein cannot generate the Parkinson's disease-specific filamentous pathology.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_008',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 1279}]}],\n",
       "   'context': \"Widespread expression of alpha-synuclein and tau immunoreactivity in Hallervorden-Spatz syndrome with protracted clinical course. Hallervorden-Spatz syndrome (HSS) is a rare autosomal recessive disorder clinically characterized by extrapyramidal signs and progressive dementia. In a typical case, the clinical symptoms become apparent during late childhood, and usually the course is protracted over a decade or more. We recently had an opportunity to study the brains of two cases of HSS with a clinical course of over 30 years. Case 1 was a 44-year-old female and case 2 was a 37-year-old male. Grossly, the brains showed severe fronto-temporal lobar atrophy with abundant spheroids and mild iron deposits in the globus pallidus, associated with features of motor neuron disease. In addition, there was diffuse sponginess in the atrophic cortex as well as widespread Alzheimer's neurofibrillary tangles (NFTs) and Lewy bodies (LBs) in the cortical and subcortical regions, including the spinal cord. Ultrastructurally, NFTs were composed of paired helical filaments, and LBs of central dense cores with radiating fibrils. Discrete immunostaining was demonstrated in NFTs and neuropil threads with various antibodies against phosphorylated tau, and in LBs with antibody against alpha-synuclein. In addition, diffuse, overlapping immunoreactivity of alpha-synuclein and phosphorylated tau was seen within the cytoplasm of many neurons. However, when LBs and NFTs coexisted within the same neurons, they were clearly segregated. The findings of our present cases as well as those reported in the literature may indicate that simultaneous and extensive occurrence of abnormal phosphorylation of tau and accumulation of alpha-synuclein may constitute cardinal pathological features of HSS with protracted clinical course.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_009',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 157}]}],\n",
       "   'context': \"Distinctive neuropathology revealed by alpha-synuclein antibodies in hereditary parkinsonism and dementia linked to chromosome 4p. The identification of the alpha-synuclein gene on chromosome 4q as a locus for familial Lewy-body parkinsonism and of alpha-synuclein as a component of Lewy bodies has heralded a new era in the study of Parkinson's disease. We have identified a large family with Lewy body parkinsonism linked to a novel locus on chromosome 4p15 that does not have a mutation in the alpha-synuclein gene. Here we report the clinical and neuropathological findings in an individual from this family and describe unusual high molecular weight alpha-synuclein-immunoreactive proteins in brain homogenates from brain regions with the most marked neuropathology. Distinctive histopathology was revealed with alpha-synuclein immunostaining, including pleomorphic Lewy bodies, synuclein-positive glial inclusions and widespread, severe neuritic dystrophy. We also discuss the relationship of this familial disorder to a Lewy body disease clinical spectrum, ranging from Parkinson's disease to dementia with psychosis.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_010',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 249}]}],\n",
       "   'context': \"Distinctive neuropathology revealed by alpha-synuclein antibodies in hereditary parkinsonism and dementia linked to chromosome 4p. The identification of the alpha-synuclein gene on chromosome 4q as a locus for familial Lewy-body parkinsonism and of alpha-synuclein as a component of Lewy bodies has heralded a new era in the study of Parkinson's disease. We have identified a large family with Lewy body parkinsonism linked to a novel locus on chromosome 4p15 that does not have a mutation in the alpha-synuclein gene. Here we report the clinical and neuropathological findings in an individual from this family and describe unusual high molecular weight alpha-synuclein-immunoreactive proteins in brain homogenates from brain regions with the most marked neuropathology. Distinctive histopathology was revealed with alpha-synuclein immunostaining, including pleomorphic Lewy bodies, synuclein-positive glial inclusions and widespread, severe neuritic dystrophy. We also discuss the relationship of this familial disorder to a Lewy body disease clinical spectrum, ranging from Parkinson's disease to dementia with psychosis.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_011',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'Alpha-synuclein', 'answer_start': 0}]}],\n",
       "   'context': \"Alpha-synuclein cortical Lewy bodies correlate with dementia in Parkinson's disease. BACKGROUND: Dementia is a frequent complication of idiopathic parkinsonism or PD, usually occurring later in the protracted course of the illness. The primary site of neuropathologic change in PD is the substantia nigra, but the neuropathologic and molecular basis of dementia in PD is less clear. Although Alzheimer's pathology has been a frequent finding, recent advances in immunostaining of alpha-synuclein have suggested the possible importance of cortical Lewy bodies (CLBs) in the brains of demented patients with PD. METHODS: The brains of 22 demented and 20 nondemented patients with a clinical and neuropathologic diagnosis of PD were evaluated with standard neuropathologic techniques. In addition, CLBs and dystrophic neurites were identified immunohistochemically with antibodies specific for alpha-synuclein and ubiquitin; plaques and tangles were identified by staining with thioflavine S. Associations between dementia status and pathologic markers were tested with logistic regression. RESULTS: CLBs positive for alpha-synuclein are highly sensitive (91%) and specific (90%) neuropathologic markers of dementia in PD and slightly more sensitive than ubiquitin-positive CLBs. They are better indicators of dementia than neurofibrillary tangles, amyloid plaques, or dystrophic neurites. CONCLUSION: CLBs detected by alpha-synuclein antibodies in patients with PD are a more sensitive and specific correlate of dementia than the presence of Alzheimer's pathology, which was present in a minority of the cases in this series.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_012',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 480}]}],\n",
       "   'context': \"Alpha-synuclein cortical Lewy bodies correlate with dementia in Parkinson's disease. BACKGROUND: Dementia is a frequent complication of idiopathic parkinsonism or PD, usually occurring later in the protracted course of the illness. The primary site of neuropathologic change in PD is the substantia nigra, but the neuropathologic and molecular basis of dementia in PD is less clear. Although Alzheimer's pathology has been a frequent finding, recent advances in immunostaining of alpha-synuclein have suggested the possible importance of cortical Lewy bodies (CLBs) in the brains of demented patients with PD. METHODS: The brains of 22 demented and 20 nondemented patients with a clinical and neuropathologic diagnosis of PD were evaluated with standard neuropathologic techniques. In addition, CLBs and dystrophic neurites were identified immunohistochemically with antibodies specific for alpha-synuclein and ubiquitin; plaques and tangles were identified by staining with thioflavine S. Associations between dementia status and pathologic markers were tested with logistic regression. RESULTS: CLBs positive for alpha-synuclein are highly sensitive (91%) and specific (90%) neuropathologic markers of dementia in PD and slightly more sensitive than ubiquitin-positive CLBs. They are better indicators of dementia than neurofibrillary tangles, amyloid plaques, or dystrophic neurites. CONCLUSION: CLBs detected by alpha-synuclein antibodies in patients with PD are a more sensitive and specific correlate of dementia than the presence of Alzheimer's pathology, which was present in a minority of the cases in this series.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_013',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 1115}]}],\n",
       "   'context': \"Alpha-synuclein cortical Lewy bodies correlate with dementia in Parkinson's disease. BACKGROUND: Dementia is a frequent complication of idiopathic parkinsonism or PD, usually occurring later in the protracted course of the illness. The primary site of neuropathologic change in PD is the substantia nigra, but the neuropathologic and molecular basis of dementia in PD is less clear. Although Alzheimer's pathology has been a frequent finding, recent advances in immunostaining of alpha-synuclein have suggested the possible importance of cortical Lewy bodies (CLBs) in the brains of demented patients with PD. METHODS: The brains of 22 demented and 20 nondemented patients with a clinical and neuropathologic diagnosis of PD were evaluated with standard neuropathologic techniques. In addition, CLBs and dystrophic neurites were identified immunohistochemically with antibodies specific for alpha-synuclein and ubiquitin; plaques and tangles were identified by staining with thioflavine S. Associations between dementia status and pathologic markers were tested with logistic regression. RESULTS: CLBs positive for alpha-synuclein are highly sensitive (91%) and specific (90%) neuropathologic markers of dementia in PD and slightly more sensitive than ubiquitin-positive CLBs. They are better indicators of dementia than neurofibrillary tangles, amyloid plaques, or dystrophic neurites. CONCLUSION: CLBs detected by alpha-synuclein antibodies in patients with PD are a more sensitive and specific correlate of dementia than the presence of Alzheimer's pathology, which was present in a minority of the cases in this series.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_014',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 1416}]}],\n",
       "   'context': \"Alpha-synuclein cortical Lewy bodies correlate with dementia in Parkinson's disease. BACKGROUND: Dementia is a frequent complication of idiopathic parkinsonism or PD, usually occurring later in the protracted course of the illness. The primary site of neuropathologic change in PD is the substantia nigra, but the neuropathologic and molecular basis of dementia in PD is less clear. Although Alzheimer's pathology has been a frequent finding, recent advances in immunostaining of alpha-synuclein have suggested the possible importance of cortical Lewy bodies (CLBs) in the brains of demented patients with PD. METHODS: The brains of 22 demented and 20 nondemented patients with a clinical and neuropathologic diagnosis of PD were evaluated with standard neuropathologic techniques. In addition, CLBs and dystrophic neurites were identified immunohistochemically with antibodies specific for alpha-synuclein and ubiquitin; plaques and tangles were identified by staining with thioflavine S. Associations between dementia status and pathologic markers were tested with logistic regression. RESULTS: CLBs positive for alpha-synuclein are highly sensitive (91%) and specific (90%) neuropathologic markers of dementia in PD and slightly more sensitive than ubiquitin-positive CLBs. They are better indicators of dementia than neurofibrillary tangles, amyloid plaques, or dystrophic neurites. CONCLUSION: CLBs detected by alpha-synuclein antibodies in patients with PD are a more sensitive and specific correlate of dementia than the presence of Alzheimer's pathology, which was present in a minority of the cases in this series.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_015',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-Synuclein', 'answer_start': 0}]}],\n",
       "   'context': 'alpha-Synuclein immunoreactivity in dementia with Lewy bodies: morphological staging and comparison with ubiquitin immunostaining. alpha-Synuclein is a presynaptic protein recently identified as a specific component of Lewy bodies (LB) and Lewy neurites. The aim of this study was to assess the morphology and distribution of alpha-synuclein immunoreactivity in cases of dementia with LB (DLB), and to compare alpha-synuclein with ubiquitin immunostaining. We examined substantia nigra, paralimbic regions (entorhinal cortex, cingulate gyrus, insula and hippocampus), and neocortex (frontal and occipital association cortices) with double alpha-synuclein and ubiquitin immunostaining in 25 cases meeting neuropathological criteria for DLB. alpha-Synuclein immunostaining was more specific than ubiquitin immunostaining in that it differentiated LB from globose tangles. It was also slightly more sensitive, staining 4-5% more intracytoplasmic structures, especially diffuse alpha-synuclein deposits that were ubiquitin negative. In addition to LB, alpha-synuclein staining showed filiform and globose neurites in the substantia nigra, CA2-3 regions of the hippocampus, and entorhinal cortex. A spectrum of alpha-synuclein staining was seen in substantia nigra: from diffuse \"cloud-like\" inclusions to aggregated intracytoplasmic inclusions with variable ubiquitin staining to classic LB. We hypothesize that these represent different stages in LB formation.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_016',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-Synuclein', 'answer_start': 131}]}],\n",
       "   'context': 'alpha-Synuclein immunoreactivity in dementia with Lewy bodies: morphological staging and comparison with ubiquitin immunostaining. alpha-Synuclein is a presynaptic protein recently identified as a specific component of Lewy bodies (LB) and Lewy neurites. The aim of this study was to assess the morphology and distribution of alpha-synuclein immunoreactivity in cases of dementia with LB (DLB), and to compare alpha-synuclein with ubiquitin immunostaining. We examined substantia nigra, paralimbic regions (entorhinal cortex, cingulate gyrus, insula and hippocampus), and neocortex (frontal and occipital association cortices) with double alpha-synuclein and ubiquitin immunostaining in 25 cases meeting neuropathological criteria for DLB. alpha-Synuclein immunostaining was more specific than ubiquitin immunostaining in that it differentiated LB from globose tangles. It was also slightly more sensitive, staining 4-5% more intracytoplasmic structures, especially diffuse alpha-synuclein deposits that were ubiquitin negative. In addition to LB, alpha-synuclein staining showed filiform and globose neurites in the substantia nigra, CA2-3 regions of the hippocampus, and entorhinal cortex. A spectrum of alpha-synuclein staining was seen in substantia nigra: from diffuse \"cloud-like\" inclusions to aggregated intracytoplasmic inclusions with variable ubiquitin staining to classic LB. We hypothesize that these represent different stages in LB formation.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_017',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-Synuclein', 'answer_start': 740}]}],\n",
       "   'context': 'alpha-Synuclein immunoreactivity in dementia with Lewy bodies: morphological staging and comparison with ubiquitin immunostaining. alpha-Synuclein is a presynaptic protein recently identified as a specific component of Lewy bodies (LB) and Lewy neurites. The aim of this study was to assess the morphology and distribution of alpha-synuclein immunoreactivity in cases of dementia with LB (DLB), and to compare alpha-synuclein with ubiquitin immunostaining. We examined substantia nigra, paralimbic regions (entorhinal cortex, cingulate gyrus, insula and hippocampus), and neocortex (frontal and occipital association cortices) with double alpha-synuclein and ubiquitin immunostaining in 25 cases meeting neuropathological criteria for DLB. alpha-Synuclein immunostaining was more specific than ubiquitin immunostaining in that it differentiated LB from globose tangles. It was also slightly more sensitive, staining 4-5% more intracytoplasmic structures, especially diffuse alpha-synuclein deposits that were ubiquitin negative. In addition to LB, alpha-synuclein staining showed filiform and globose neurites in the substantia nigra, CA2-3 regions of the hippocampus, and entorhinal cortex. A spectrum of alpha-synuclein staining was seen in substantia nigra: from diffuse \"cloud-like\" inclusions to aggregated intracytoplasmic inclusions with variable ubiquitin staining to classic LB. We hypothesize that these represent different stages in LB formation.'},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_018',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 279}]}],\n",
       "   'context': \"Abnormal distribution of the non-Abeta component of Alzheimer's disease amyloid precursor/alpha-synuclein in Lewy body disease as revealed by proteinase K and formic acid pretreatment. The precursor of the non-Abeta component of Alzheimer's disease amyloid (NACP) (also known as alpha-synuclein) is a presynaptic terminal molecule that abnormally accumulates in the plaques of Alzheimer's disease (AD) and in the Lewy bodies (LBs) of Lewy body variant of AD, diffuse Lewy body disease, and Parkinson's disease. To better understand the distribution of NACP/alpha-synuclein and its fragments in the LB-bearing neurons and neurites, as well as to clarify the patterns of NACP/alpha-synuclein compartmentalization, we studied NACP/alpha-synuclein immunoreactivity using antibodies against the C-terminal, N-terminal, and NAC regions after Proteinase K and formic acid treatment in the cortex of patients with LBs. Furthermore, studies of the subcellular localization of NACP/alpha-synuclein within LB-bearing neurons were performed by immunogold electron microscopy. These studies showed that the N-terminal antibody immunolabeled the LBs and dystrophic neurites with great intensity and, to a lesser extent, the synapses. In contrast, the C-terminal antibody strongly labeled the synapses and, to a lesser extent, the LBs and dystrophic neurites. Whereas Proteinase K treatment enhanced NACP/alpha-synuclein immunoreactivity with the C-terminal antibody, it diminished the N-terminal NACP/alpha-synuclein immunoreactivity. Furthermore, formic acid enhanced LB and dystrophic neurite labeling with both the C- and N-terminal antibodies. In addition, whereas without pretreatment only slight anti-NAC immunoreactivity was found in the LBs, formic acid pretreatment revealed an extensive anti-NAC immunostaining of LBs, plaques, and glial cells. Ultrastructural analysis revealed that NACP/alpha-synuclein immunoreactivity was diffusely distributed within the amorphous electrodense material in the LBs and as small clusters in the filaments of LBs and neurites. These results support the view that aggregated NACP/alpha-synuclein might play an important role in the pathogenesis of disorders associated with LBs.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_019',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 557}]}],\n",
       "   'context': \"Abnormal distribution of the non-Abeta component of Alzheimer's disease amyloid precursor/alpha-synuclein in Lewy body disease as revealed by proteinase K and formic acid pretreatment. The precursor of the non-Abeta component of Alzheimer's disease amyloid (NACP) (also known as alpha-synuclein) is a presynaptic terminal molecule that abnormally accumulates in the plaques of Alzheimer's disease (AD) and in the Lewy bodies (LBs) of Lewy body variant of AD, diffuse Lewy body disease, and Parkinson's disease. To better understand the distribution of NACP/alpha-synuclein and its fragments in the LB-bearing neurons and neurites, as well as to clarify the patterns of NACP/alpha-synuclein compartmentalization, we studied NACP/alpha-synuclein immunoreactivity using antibodies against the C-terminal, N-terminal, and NAC regions after Proteinase K and formic acid treatment in the cortex of patients with LBs. Furthermore, studies of the subcellular localization of NACP/alpha-synuclein within LB-bearing neurons were performed by immunogold electron microscopy. These studies showed that the N-terminal antibody immunolabeled the LBs and dystrophic neurites with great intensity and, to a lesser extent, the synapses. In contrast, the C-terminal antibody strongly labeled the synapses and, to a lesser extent, the LBs and dystrophic neurites. Whereas Proteinase K treatment enhanced NACP/alpha-synuclein immunoreactivity with the C-terminal antibody, it diminished the N-terminal NACP/alpha-synuclein immunoreactivity. Furthermore, formic acid enhanced LB and dystrophic neurite labeling with both the C- and N-terminal antibodies. In addition, whereas without pretreatment only slight anti-NAC immunoreactivity was found in the LBs, formic acid pretreatment revealed an extensive anti-NAC immunostaining of LBs, plaques, and glial cells. Ultrastructural analysis revealed that NACP/alpha-synuclein immunoreactivity was diffusely distributed within the amorphous electrodense material in the LBs and as small clusters in the filaments of LBs and neurites. These results support the view that aggregated NACP/alpha-synuclein might play an important role in the pathogenesis of disorders associated with LBs.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_020',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 674}]}],\n",
       "   'context': \"Abnormal distribution of the non-Abeta component of Alzheimer's disease amyloid precursor/alpha-synuclein in Lewy body disease as revealed by proteinase K and formic acid pretreatment. The precursor of the non-Abeta component of Alzheimer's disease amyloid (NACP) (also known as alpha-synuclein) is a presynaptic terminal molecule that abnormally accumulates in the plaques of Alzheimer's disease (AD) and in the Lewy bodies (LBs) of Lewy body variant of AD, diffuse Lewy body disease, and Parkinson's disease. To better understand the distribution of NACP/alpha-synuclein and its fragments in the LB-bearing neurons and neurites, as well as to clarify the patterns of NACP/alpha-synuclein compartmentalization, we studied NACP/alpha-synuclein immunoreactivity using antibodies against the C-terminal, N-terminal, and NAC regions after Proteinase K and formic acid treatment in the cortex of patients with LBs. Furthermore, studies of the subcellular localization of NACP/alpha-synuclein within LB-bearing neurons were performed by immunogold electron microscopy. These studies showed that the N-terminal antibody immunolabeled the LBs and dystrophic neurites with great intensity and, to a lesser extent, the synapses. In contrast, the C-terminal antibody strongly labeled the synapses and, to a lesser extent, the LBs and dystrophic neurites. Whereas Proteinase K treatment enhanced NACP/alpha-synuclein immunoreactivity with the C-terminal antibody, it diminished the N-terminal NACP/alpha-synuclein immunoreactivity. Furthermore, formic acid enhanced LB and dystrophic neurite labeling with both the C- and N-terminal antibodies. In addition, whereas without pretreatment only slight anti-NAC immunoreactivity was found in the LBs, formic acid pretreatment revealed an extensive anti-NAC immunostaining of LBs, plaques, and glial cells. Ultrastructural analysis revealed that NACP/alpha-synuclein immunoreactivity was diffusely distributed within the amorphous electrodense material in the LBs and as small clusters in the filaments of LBs and neurites. These results support the view that aggregated NACP/alpha-synuclein might play an important role in the pathogenesis of disorders associated with LBs.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_021',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 728}]}],\n",
       "   'context': \"Abnormal distribution of the non-Abeta component of Alzheimer's disease amyloid precursor/alpha-synuclein in Lewy body disease as revealed by proteinase K and formic acid pretreatment. The precursor of the non-Abeta component of Alzheimer's disease amyloid (NACP) (also known as alpha-synuclein) is a presynaptic terminal molecule that abnormally accumulates in the plaques of Alzheimer's disease (AD) and in the Lewy bodies (LBs) of Lewy body variant of AD, diffuse Lewy body disease, and Parkinson's disease. To better understand the distribution of NACP/alpha-synuclein and its fragments in the LB-bearing neurons and neurites, as well as to clarify the patterns of NACP/alpha-synuclein compartmentalization, we studied NACP/alpha-synuclein immunoreactivity using antibodies against the C-terminal, N-terminal, and NAC regions after Proteinase K and formic acid treatment in the cortex of patients with LBs. Furthermore, studies of the subcellular localization of NACP/alpha-synuclein within LB-bearing neurons were performed by immunogold electron microscopy. These studies showed that the N-terminal antibody immunolabeled the LBs and dystrophic neurites with great intensity and, to a lesser extent, the synapses. In contrast, the C-terminal antibody strongly labeled the synapses and, to a lesser extent, the LBs and dystrophic neurites. Whereas Proteinase K treatment enhanced NACP/alpha-synuclein immunoreactivity with the C-terminal antibody, it diminished the N-terminal NACP/alpha-synuclein immunoreactivity. Furthermore, formic acid enhanced LB and dystrophic neurite labeling with both the C- and N-terminal antibodies. In addition, whereas without pretreatment only slight anti-NAC immunoreactivity was found in the LBs, formic acid pretreatment revealed an extensive anti-NAC immunostaining of LBs, plaques, and glial cells. Ultrastructural analysis revealed that NACP/alpha-synuclein immunoreactivity was diffusely distributed within the amorphous electrodense material in the LBs and as small clusters in the filaments of LBs and neurites. These results support the view that aggregated NACP/alpha-synuclein might play an important role in the pathogenesis of disorders associated with LBs.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_022',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 1885}]}],\n",
       "   'context': \"Abnormal distribution of the non-Abeta component of Alzheimer's disease amyloid precursor/alpha-synuclein in Lewy body disease as revealed by proteinase K and formic acid pretreatment. The precursor of the non-Abeta component of Alzheimer's disease amyloid (NACP) (also known as alpha-synuclein) is a presynaptic terminal molecule that abnormally accumulates in the plaques of Alzheimer's disease (AD) and in the Lewy bodies (LBs) of Lewy body variant of AD, diffuse Lewy body disease, and Parkinson's disease. To better understand the distribution of NACP/alpha-synuclein and its fragments in the LB-bearing neurons and neurites, as well as to clarify the patterns of NACP/alpha-synuclein compartmentalization, we studied NACP/alpha-synuclein immunoreactivity using antibodies against the C-terminal, N-terminal, and NAC regions after Proteinase K and formic acid treatment in the cortex of patients with LBs. Furthermore, studies of the subcellular localization of NACP/alpha-synuclein within LB-bearing neurons were performed by immunogold electron microscopy. These studies showed that the N-terminal antibody immunolabeled the LBs and dystrophic neurites with great intensity and, to a lesser extent, the synapses. In contrast, the C-terminal antibody strongly labeled the synapses and, to a lesser extent, the LBs and dystrophic neurites. Whereas Proteinase K treatment enhanced NACP/alpha-synuclein immunoreactivity with the C-terminal antibody, it diminished the N-terminal NACP/alpha-synuclein immunoreactivity. Furthermore, formic acid enhanced LB and dystrophic neurite labeling with both the C- and N-terminal antibodies. In addition, whereas without pretreatment only slight anti-NAC immunoreactivity was found in the LBs, formic acid pretreatment revealed an extensive anti-NAC immunostaining of LBs, plaques, and glial cells. Ultrastructural analysis revealed that NACP/alpha-synuclein immunoreactivity was diffusely distributed within the amorphous electrodense material in the LBs and as small clusters in the filaments of LBs and neurites. These results support the view that aggregated NACP/alpha-synuclein might play an important role in the pathogenesis of disorders associated with LBs.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_023',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 2110}]}],\n",
       "   'context': \"Abnormal distribution of the non-Abeta component of Alzheimer's disease amyloid precursor/alpha-synuclein in Lewy body disease as revealed by proteinase K and formic acid pretreatment. The precursor of the non-Abeta component of Alzheimer's disease amyloid (NACP) (also known as alpha-synuclein) is a presynaptic terminal molecule that abnormally accumulates in the plaques of Alzheimer's disease (AD) and in the Lewy bodies (LBs) of Lewy body variant of AD, diffuse Lewy body disease, and Parkinson's disease. To better understand the distribution of NACP/alpha-synuclein and its fragments in the LB-bearing neurons and neurites, as well as to clarify the patterns of NACP/alpha-synuclein compartmentalization, we studied NACP/alpha-synuclein immunoreactivity using antibodies against the C-terminal, N-terminal, and NAC regions after Proteinase K and formic acid treatment in the cortex of patients with LBs. Furthermore, studies of the subcellular localization of NACP/alpha-synuclein within LB-bearing neurons were performed by immunogold electron microscopy. These studies showed that the N-terminal antibody immunolabeled the LBs and dystrophic neurites with great intensity and, to a lesser extent, the synapses. In contrast, the C-terminal antibody strongly labeled the synapses and, to a lesser extent, the LBs and dystrophic neurites. Whereas Proteinase K treatment enhanced NACP/alpha-synuclein immunoreactivity with the C-terminal antibody, it diminished the N-terminal NACP/alpha-synuclein immunoreactivity. Furthermore, formic acid enhanced LB and dystrophic neurite labeling with both the C- and N-terminal antibodies. In addition, whereas without pretreatment only slight anti-NAC immunoreactivity was found in the LBs, formic acid pretreatment revealed an extensive anti-NAC immunostaining of LBs, plaques, and glial cells. Ultrastructural analysis revealed that NACP/alpha-synuclein immunoreactivity was diffusely distributed within the amorphous electrodense material in the LBs and as small clusters in the filaments of LBs and neurites. These results support the view that aggregated NACP/alpha-synuclein might play an important role in the pathogenesis of disorders associated with LBs.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_024',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 125}]}],\n",
       "   'context': \"Nigral and cortical Lewy bodies and dystrophic nigral neurites in Parkinson's disease and cortical Lewy body disease contain alpha-synuclein immunoreactivity. A mutation in the alpha-synuclein gene has recently been linked to some cases of familial Parkinson's disease (PD). We characterized the expression of this presynaptic protein in the midbrain, striatum, and temporal cortex of control, PD, and dementia with Lewy bodies (DLB) brain. Control brain showed punctate pericellular immunostaining. PD brain demonstrated alpha-synuclein immunoreactivity in nigral Lewy bodies, pale bodies and abnormal neurites. Rare neuronal soma in PD brain were immunoreactive for alpha-synuclein. DLB cases demonstrated these findings as well as alpha-synuclein immunoreactivity in cortical Lewy bodies and CA2-3 neurites. These results suggest that, even in sporadic cases, there is an early and direct role for alpha-synuclein in the pathogenesis of PD and the neuropathologically related disorder DLB.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_025',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 522}]}],\n",
       "   'context': \"Nigral and cortical Lewy bodies and dystrophic nigral neurites in Parkinson's disease and cortical Lewy body disease contain alpha-synuclein immunoreactivity. A mutation in the alpha-synuclein gene has recently been linked to some cases of familial Parkinson's disease (PD). We characterized the expression of this presynaptic protein in the midbrain, striatum, and temporal cortex of control, PD, and dementia with Lewy bodies (DLB) brain. Control brain showed punctate pericellular immunostaining. PD brain demonstrated alpha-synuclein immunoreactivity in nigral Lewy bodies, pale bodies and abnormal neurites. Rare neuronal soma in PD brain were immunoreactive for alpha-synuclein. DLB cases demonstrated these findings as well as alpha-synuclein immunoreactivity in cortical Lewy bodies and CA2-3 neurites. These results suggest that, even in sporadic cases, there is an early and direct role for alpha-synuclein in the pathogenesis of PD and the neuropathologically related disorder DLB.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_026',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 734}]}],\n",
       "   'context': \"Nigral and cortical Lewy bodies and dystrophic nigral neurites in Parkinson's disease and cortical Lewy body disease contain alpha-synuclein immunoreactivity. A mutation in the alpha-synuclein gene has recently been linked to some cases of familial Parkinson's disease (PD). We characterized the expression of this presynaptic protein in the midbrain, striatum, and temporal cortex of control, PD, and dementia with Lewy bodies (DLB) brain. Control brain showed punctate pericellular immunostaining. PD brain demonstrated alpha-synuclein immunoreactivity in nigral Lewy bodies, pale bodies and abnormal neurites. Rare neuronal soma in PD brain were immunoreactive for alpha-synuclein. DLB cases demonstrated these findings as well as alpha-synuclein immunoreactivity in cortical Lewy bodies and CA2-3 neurites. These results suggest that, even in sporadic cases, there is an early and direct role for alpha-synuclein in the pathogenesis of PD and the neuropathologically related disorder DLB.\"},\n",
       "  {'qas': [{'id': '53189656b166e2b80600001c_027',\n",
       "     'question': 'Against which protein is the antibody used for immonostaining of Lewy bodies raised?',\n",
       "     'answers': [{'text': 'alpha-synuclein', 'answer_start': 901}]}],\n",
       "   'context': \"Nigral and cortical Lewy bodies and dystrophic nigral neurites in Parkinson's disease and cortical Lewy body disease contain alpha-synuclein immunoreactivity. A mutation in the alpha-synuclein gene has recently been linked to some cases of familial Parkinson's disease (PD). We characterized the expression of this presynaptic protein in the midbrain, striatum, and temporal cortex of control, PD, and dementia with Lewy bodies (DLB) brain. Control brain showed punctate pericellular immunostaining. PD brain demonstrated alpha-synuclein immunoreactivity in nigral Lewy bodies, pale bodies and abnormal neurites. Rare neuronal soma in PD brain were immunoreactive for alpha-synuclein. DLB cases demonstrated these findings as well as alpha-synuclein immunoreactivity in cortical Lewy bodies and CA2-3 neurites. These results suggest that, even in sporadic cases, there is an early and direct role for alpha-synuclein in the pathogenesis of PD and the neuropathologically related disorder DLB.\"},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_001',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 68}]}],\n",
       "   'context': 'Delamanid: a review of its use in patients with multidrug-resistant tuberculosis. Delamanid (Deltyba(®)), a nitroimidazo-oxazole derivative, is a new anti-tuberculosis (TB) drug which exhibits potent in vitro and in vivo antitubercular activity against drug-susceptible and -resistant strains of Mycobacterium tuberculosis. It is approved in several countries, including Japan and those of the EU, for use as part of an appropriate combination regimen in adults with multidrug-resistant tuberculosis (MDR-TB) when an effective treatment regimen cannot otherwise be composed due to resistance or tolerability. In a robust phase\\xa0II trial in adult patients with MDR-TB, oral delamanid 100\\xa0mg twice daily for 2\\xa0months plus an optimized background regimen improved sputum culture conversion rates to a significantly greater extent than placebo. In a 6-month extension study, long-term ( < 8\\xa0months) treatment with delamanid was associated with a higher incidence of favourable outcomes (i.e. cured or completed all treatment) than short-term ( < 2\\xa0months) treatment, with an accompanying reduction inunfavourable outcomes as defined by the WHO (i.e. pre-specified proportion of TB-positive sputum cultures, death or treatment discontinuation for > 2\\xa0months without medical approval). Delamanid was not associated with clinically relevant drug-drug interactions, including with antiretroviral drugs and those commonly used in treating TB. Delamanid was generally well tolerated in patients with MDR-TB, with gastrointestinal adverse events and insomnia reported most commonly. Although the incidence of QT interval prolongation was higher with delamanid-based therapy, it was not associated with clinical symptoms such as syncope and arrhythmia. In conclusion, delamanid is a useful addition to the treatment options currently available for patients with MDR-TB.'},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_002',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 155}]}],\n",
       "   'context': 'Delamanid: a review of its use in patients with multidrug-resistant tuberculosis. Delamanid (Deltyba(®)), a nitroimidazo-oxazole derivative, is a new anti-tuberculosis (TB) drug which exhibits potent in vitro and in vivo antitubercular activity against drug-susceptible and -resistant strains of Mycobacterium tuberculosis. It is approved in several countries, including Japan and those of the EU, for use as part of an appropriate combination regimen in adults with multidrug-resistant tuberculosis (MDR-TB) when an effective treatment regimen cannot otherwise be composed due to resistance or tolerability. In a robust phase\\xa0II trial in adult patients with MDR-TB, oral delamanid 100\\xa0mg twice daily for 2\\xa0months plus an optimized background regimen improved sputum culture conversion rates to a significantly greater extent than placebo. In a 6-month extension study, long-term ( < 8\\xa0months) treatment with delamanid was associated with a higher incidence of favourable outcomes (i.e. cured or completed all treatment) than short-term ( < 2\\xa0months) treatment, with an accompanying reduction inunfavourable outcomes as defined by the WHO (i.e. pre-specified proportion of TB-positive sputum cultures, death or treatment discontinuation for > 2\\xa0months without medical approval). Delamanid was not associated with clinically relevant drug-drug interactions, including with antiretroviral drugs and those commonly used in treating TB. Delamanid was generally well tolerated in patients with MDR-TB, with gastrointestinal adverse events and insomnia reported most commonly. Although the incidence of QT interval prolongation was higher with delamanid-based therapy, it was not associated with clinical symptoms such as syncope and arrhythmia. In conclusion, delamanid is a useful addition to the treatment options currently available for patients with MDR-TB.'},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_003',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 310}]}],\n",
       "   'context': 'Delamanid: a review of its use in patients with multidrug-resistant tuberculosis. Delamanid (Deltyba(®)), a nitroimidazo-oxazole derivative, is a new anti-tuberculosis (TB) drug which exhibits potent in vitro and in vivo antitubercular activity against drug-susceptible and -resistant strains of Mycobacterium tuberculosis. It is approved in several countries, including Japan and those of the EU, for use as part of an appropriate combination regimen in adults with multidrug-resistant tuberculosis (MDR-TB) when an effective treatment regimen cannot otherwise be composed due to resistance or tolerability. In a robust phase\\xa0II trial in adult patients with MDR-TB, oral delamanid 100\\xa0mg twice daily for 2\\xa0months plus an optimized background regimen improved sputum culture conversion rates to a significantly greater extent than placebo. In a 6-month extension study, long-term ( < 8\\xa0months) treatment with delamanid was associated with a higher incidence of favourable outcomes (i.e. cured or completed all treatment) than short-term ( < 2\\xa0months) treatment, with an accompanying reduction inunfavourable outcomes as defined by the WHO (i.e. pre-specified proportion of TB-positive sputum cultures, death or treatment discontinuation for > 2\\xa0months without medical approval). Delamanid was not associated with clinically relevant drug-drug interactions, including with antiretroviral drugs and those commonly used in treating TB. Delamanid was generally well tolerated in patients with MDR-TB, with gastrointestinal adverse events and insomnia reported most commonly. Although the incidence of QT interval prolongation was higher with delamanid-based therapy, it was not associated with clinical symptoms such as syncope and arrhythmia. In conclusion, delamanid is a useful addition to the treatment options currently available for patients with MDR-TB.'},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_004',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 26}]}],\n",
       "   'context': \"Delamanid when other anti-tuberculosis-treatment regimens failed due to resistance or tolerability. INTRODUCTION: The limited availability of effective drugs causes difficulties in the management of multidrug-resistant tuberculosis (MDR-TB) and novel therapeutic agents are needed. Delamanid , a new nitro-hydro-imidazooxazole derivative, inhibits mycolic acid synthesis. This review covers the efficacy and safety of delamanid for MDR-TB. AREA COVERED: This paper reviews the pharmacological profile of delamanid and the results of clinical trials evaluating its efficacy for treating MDR-TB in combination with other anti-TB drugs. The drug's safety and tolerability profiles are also considered. EXPERT OPINION: Delamanid showed potent activity against drug-susceptible and -resistant Mycobacterium tuberculosis in both in vitro and in vivo studies. In clinical trials, the drug showed significant early bactericidal activity in pulmonary TB patients, and increased culture conversion after 2 months of treatment in combination with an optimized background regimen in MDR-TB patients. In addition, decreased mortality was observed in MDR-TB patients who received > 6 months of delamanid treatment. The drug was generally tolerable, but QT prolongation should be monitored carefully using electrocardiograms and potassium levels. Therefore, delamanid could be used as part of an appropriate combination regimen for pulmonary MDR-TB in adult patients when an effective treatment regimen cannot otherwise be composed for reasons of resistance or tolerability.\"},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_005',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 802}]}],\n",
       "   'context': \"Delamanid when other anti-tuberculosis-treatment regimens failed due to resistance or tolerability. INTRODUCTION: The limited availability of effective drugs causes difficulties in the management of multidrug-resistant tuberculosis (MDR-TB) and novel therapeutic agents are needed. Delamanid , a new nitro-hydro-imidazooxazole derivative, inhibits mycolic acid synthesis. This review covers the efficacy and safety of delamanid for MDR-TB. AREA COVERED: This paper reviews the pharmacological profile of delamanid and the results of clinical trials evaluating its efficacy for treating MDR-TB in combination with other anti-TB drugs. The drug's safety and tolerability profiles are also considered. EXPERT OPINION: Delamanid showed potent activity against drug-susceptible and -resistant Mycobacterium tuberculosis in both in vitro and in vivo studies. In clinical trials, the drug showed significant early bactericidal activity in pulmonary TB patients, and increased culture conversion after 2 months of treatment in combination with an optimized background regimen in MDR-TB patients. In addition, decreased mortality was observed in MDR-TB patients who received > 6 months of delamanid treatment. The drug was generally tolerable, but QT prolongation should be monitored carefully using electrocardiograms and potassium levels. Therefore, delamanid could be used as part of an appropriate combination regimen for pulmonary MDR-TB in adult patients when an effective treatment regimen cannot otherwise be composed for reasons of resistance or tolerability.\"},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_006',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 44}]}],\n",
       "   'context': 'Delamanid for multidrug-resistant pulmonary tuberculosis. BACKGROUND: Delamanid (OPC-67683), a nitro-dihydro-imidazooxazole derivative, is a new antituberculosis medication that inhibits mycolic acid synthesis and has shown potent in vitro and in vivo activity against drug-resistant strains of Mycobacterium tuberculosis. METHODS: In this randomized, placebo-controlled, multinational clinical trial, we assigned 481 patients (nearly all of whom were negative for the human immunodeficiency virus) with pulmonary multidrug-resistant tuberculosis to receive delamanid, at a dose of 100 mg twice daily (161 patients) or 200 mg twice daily (160 patients), or placebo (160 patients) for 2 months in combination with a background drug regimen developed according to World Health Organization guidelines. Sputum cultures were assessed weekly with the use of both liquid broth and solid medium; sputum-culture conversion was defined as a series of five or more consecutive cultures that were negative for growth of M. tuberculosis. The primary efficacy end point was the proportion of patients with sputum-culture conversion in liquid broth medium at 2 months. RESULTS: Among patients who received a background drug regimen plus 100 mg of delamanid twice daily, 45.4% had sputum-culture conversion in liquid broth at 2 months, as compared with 29.6% of patients who received a background drug regimen plus placebo (P=0.008). Likewise, as compared with the placebo group, the group that received the background drug regimen plus 200 mg of delamanid twice daily had a higher proportion of patients with sputum-culture conversion (41.9%, P=0.04). The findings were similar with assessment of sputum-culture conversion in solid medium. Most adverse events were mild to moderate in severity and were evenly distributed across groups. Although no clinical events due to QT prolongation on electrocardiography were observed, QT prolongation was reported significantly more frequently in the groups that received delamanid. CONCLUSIONS: Delamanid was associated with an increase in sputum-culture conversion at 2 months among patients with multidrug-resistant tuberculosis. This finding suggests that delamanid could enhance treatment options for multidrug-resistant tuberculosis. (Funded by Otsuka Pharmaceutical Development and Commercialization; ClinicalTrials.gov number, NCT00685360.).'},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_007',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 149}]}],\n",
       "   'context': 'Delamanid for multidrug-resistant pulmonary tuberculosis. BACKGROUND: Delamanid (OPC-67683), a nitro-dihydro-imidazooxazole derivative, is a new antituberculosis medication that inhibits mycolic acid synthesis and has shown potent in vitro and in vivo activity against drug-resistant strains of Mycobacterium tuberculosis. METHODS: In this randomized, placebo-controlled, multinational clinical trial, we assigned 481 patients (nearly all of whom were negative for the human immunodeficiency virus) with pulmonary multidrug-resistant tuberculosis to receive delamanid, at a dose of 100 mg twice daily (161 patients) or 200 mg twice daily (160 patients), or placebo (160 patients) for 2 months in combination with a background drug regimen developed according to World Health Organization guidelines. Sputum cultures were assessed weekly with the use of both liquid broth and solid medium; sputum-culture conversion was defined as a series of five or more consecutive cultures that were negative for growth of M. tuberculosis. The primary efficacy end point was the proportion of patients with sputum-culture conversion in liquid broth medium at 2 months. RESULTS: Among patients who received a background drug regimen plus 100 mg of delamanid twice daily, 45.4% had sputum-culture conversion in liquid broth at 2 months, as compared with 29.6% of patients who received a background drug regimen plus placebo (P=0.008). Likewise, as compared with the placebo group, the group that received the background drug regimen plus 200 mg of delamanid twice daily had a higher proportion of patients with sputum-culture conversion (41.9%, P=0.04). The findings were similar with assessment of sputum-culture conversion in solid medium. Most adverse events were mild to moderate in severity and were evenly distributed across groups. Although no clinical events due to QT prolongation on electrocardiography were observed, QT prolongation was reported significantly more frequently in the groups that received delamanid. CONCLUSIONS: Delamanid was associated with an increase in sputum-culture conversion at 2 months among patients with multidrug-resistant tuberculosis. This finding suggests that delamanid could enhance treatment options for multidrug-resistant tuberculosis. (Funded by Otsuka Pharmaceutical Development and Commercialization; ClinicalTrials.gov number, NCT00685360.).'},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_008',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 309}]}],\n",
       "   'context': 'Delamanid for multidrug-resistant pulmonary tuberculosis. BACKGROUND: Delamanid (OPC-67683), a nitro-dihydro-imidazooxazole derivative, is a new antituberculosis medication that inhibits mycolic acid synthesis and has shown potent in vitro and in vivo activity against drug-resistant strains of Mycobacterium tuberculosis. METHODS: In this randomized, placebo-controlled, multinational clinical trial, we assigned 481 patients (nearly all of whom were negative for the human immunodeficiency virus) with pulmonary multidrug-resistant tuberculosis to receive delamanid, at a dose of 100 mg twice daily (161 patients) or 200 mg twice daily (160 patients), or placebo (160 patients) for 2 months in combination with a background drug regimen developed according to World Health Organization guidelines. Sputum cultures were assessed weekly with the use of both liquid broth and solid medium; sputum-culture conversion was defined as a series of five or more consecutive cultures that were negative for growth of M. tuberculosis. The primary efficacy end point was the proportion of patients with sputum-culture conversion in liquid broth medium at 2 months. RESULTS: Among patients who received a background drug regimen plus 100 mg of delamanid twice daily, 45.4% had sputum-culture conversion in liquid broth at 2 months, as compared with 29.6% of patients who received a background drug regimen plus placebo (P=0.008). Likewise, as compared with the placebo group, the group that received the background drug regimen plus 200 mg of delamanid twice daily had a higher proportion of patients with sputum-culture conversion (41.9%, P=0.04). The findings were similar with assessment of sputum-culture conversion in solid medium. Most adverse events were mild to moderate in severity and were evenly distributed across groups. Although no clinical events due to QT prolongation on electrocardiography were observed, QT prolongation was reported significantly more frequently in the groups that received delamanid. CONCLUSIONS: Delamanid was associated with an increase in sputum-culture conversion at 2 months among patients with multidrug-resistant tuberculosis. This finding suggests that delamanid could enhance treatment options for multidrug-resistant tuberculosis. (Funded by Otsuka Pharmaceutical Development and Commercialization; ClinicalTrials.gov number, NCT00685360.).'},\n",
       "  {'qas': [{'id': '56bc7d71ac7ad10019000018_010',\n",
       "     'question': 'Which disease can be treated with Delamanid?',\n",
       "     'answers': [{'text': 'tuberculosis', 'answer_start': 2253}]}],\n",
       "   'context': 'Delamanid for multidrug-resistant pulmonary tuberculosis. BACKGROUND: Delamanid (OPC-67683), a nitro-dihydro-imidazooxazole derivative, is a new antituberculosis medication that inhibits mycolic acid synthesis and has shown potent in vitro and in vivo activity against drug-resistant strains of Mycobacterium tuberculosis. METHODS: In this randomized, placebo-controlled, multinational clinical trial, we assigned 481 patients (nearly all of whom were negative for the human immunodeficiency virus) with pulmonary multidrug-resistant tuberculosis to receive delamanid, at a dose of 100 mg twice daily (161 patients) or 200 mg twice daily (160 patients), or placebo (160 patients) for 2 months in combination with a background drug regimen developed according to World Health Organization guidelines. Sputum cultures were assessed weekly with the use of both liquid broth and solid medium; sputum-culture conversion was defined as a series of five or more consecutive cultures that were negative for growth of M. tuberculosis. The primary efficacy end point was the proportion of patients with sputum-culture conversion in liquid broth medium at 2 months. RESULTS: Among patients who received a background drug regimen plus 100 mg of delamanid twice daily, 45.4% had sputum-culture conversion in liquid broth at 2 months, as compared with 29.6% of patients who received a background drug regimen plus placebo (P=0.008). Likewise, as compared with the placebo group, the group that received the background drug regimen plus 200 mg of delamanid twice daily had a higher proportion of patients with sputum-culture conversion (41.9%, P=0.04). The findings were similar with assessment of sputum-culture conversion in solid medium. Most adverse events were mild to moderate in severity and were evenly distributed across groups. Although no clinical events due to QT prolongation on electrocardiography were observed, QT prolongation was reported significantly more frequently in the groups that received delamanid. CONCLUSIONS: Delamanid was associated with an increase in sputum-culture conversion at 2 months among patients with multidrug-resistant tuberculosis. This finding suggests that delamanid could enhance treatment options for multidrug-resistant tuberculosis. (Funded by Otsuka Pharmaceutical Development and Commercialization; ClinicalTrials.gov number, NCT00685360.).'},\n",
       "  {'qas': [{'id': '56d3346cf22319765a000008_001',\n",
       "     'question': 'What is situs inversus?',\n",
       "     'answers': [{'text': 'Situs inversus totalis is a rare congenital anomaly with a complete mirror image of the thoracic and abdominal organs.',\n",
       "       'answer_start': 419}]}],\n",
       "   'context': 'When hepatoma rupture happens in situs inversus totalis: side matters. The authors reported a 73-year-old alcoholic man with previously-unrecognized situs inversus totalis suffering from left upper quadrant pain. Acute myocardial infarction was diagnosed and coronary angioplasty was performed immediately. However, the massive bleeding from the previously-unfound hepatomas caused hypovolemic shock and fatal outcome. Situs inversus totalis is a rare congenital anomaly with a complete mirror image of the thoracic and abdominal organs. Although being considered a benign entity, it would disturb diagnosis-making of the visceral diseases owing to the altered anatomy. To our knowledge, the coexistence of the coronary artery disease and ruptured hepatomas in situs inversus totalis, as in our patient, is never described. Recognition of any situs anomalies in time is the key to avoid misdiagnosis, inappropriate managements, and unwanted consequences.'},\n",
       "  {'qas': [{'id': '55411d9f379ddf3f47000001_001',\n",
       "     'question': 'Which is the most typical peptide sequence responsible for retrieval of endoplasmic reticulum (ER) lumenal proteins from the Golgi apparatus?',\n",
       "     'answers': [{'text': 'The carboxyl-terminal Lys-Asp-Glu-Leu (KDEL)',\n",
       "       'answer_start': 135}]}],\n",
       "   'context': 'Localization of the Lys, Asp, Glu, Leu tetrapeptide receptor to the Golgi complex and the intermediate compartment in mammalian cells. The carboxyl-terminal Lys-Asp-Glu-Leu (KDEL), or a closely-related sequence, is important for ER localization of both lumenal as well as type II membrane proteins. This sequence functions as a retrieval signal at post-ER compartment(s), but the exact compartment(s) where the retrieval occurs remains unresolved. With an affinity-purified antibody against the carboxyl-terminal sequence of the mammalian KDEL receptor, we have investigated its subcellular localization using immunogold labeling on thawed cryosections of different tissues, such as mouse spermatids and rat pancreas, as well as HeLa, Vero, NRK, and mouse L cells. We show that rab1 is an excellent marker of the intermediate compartment, and we use this marker, as well as budding profiles of the mouse hepatitis virus (MHV) in cells infected with this virus, to identify this compartment. Our results demonstrate that the KDEL receptor is concentrated in the intermediate compartment, as well as in the Golgi stack. Lower but significant labeling was detected in the rough ER. In general, only small amounts of the receptor were detected on the trans side of the Golgi stack, including the trans-Golgi network (TGN) of normal cells and tissues. However, some stress conditions, such as infection with vaccinia virus or vesicular stomatitis virus, as well as 20 degrees C or 43 degrees C treatment, resulted in a significant shift of the distribution towards the trans-TGN side of the Golgi stack. This shift could be quantified in HeLa cells stably expressing a TGN marker. No significant labeling was detected in structures distal to the TGN under all conditions tested. After GTP gamma S treatment of permeabilized cells, the receptor was detected in the beta-COP-containing buds/vesicles that accumulate after this treatment, suggesting that these vesicles may transport the receptor between compartments. We propose that retrieval of KDEL-containing proteins occurs at multiple post-ER compartments up to the TGN along the exocytotic pathway, and that within this pathway, the amounts of the receptor in different compartments varies according to physiological conditions.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_001',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 407}]}],\n",
       "   'context': 'Oral glucocorticoid-sparing effect of mepolizumab in eosinophilic asthma. BACKGROUND: Many patients with severe asthma require regular treatment with oral glucocorticoids despite the use of high-dose inhaled therapy. However, the regular use of systemic glucocorticoids can result in serious and often irreversible adverse effects. Mepolizumab, a humanized monoclonal antibody that binds to and inactivates interleukin-5, has been shown to reduce asthma exacerbations in patients with severe eosinophilic asthma. METHODS: In a randomized, double-blind trial involving 135 patients with severe eosinophilic asthma, we compared the glucocorticoid-sparing effect of mepolizumab (at a dose of 100 mg) with that of placebo administered subcutaneously every 4 weeks for 20 weeks. The primary outcome was the degree of reduction in the glucocorticoid dose (90 to 100% reduction, 75 to less than 90% reduction, 50 to less than 75% reduction, more than 0 to less than 50% reduction, or no decrease in oral glucocorticoid dose, a lack of asthma control during weeks 20 to 24, or withdrawal from treatment). Other outcomes included the rate of asthma exacerbations, asthma control, and safety. RESULTS: The likelihood of a reduction in the glucocorticoid-dose stratum was 2.39 times greater in the mepolizumab group than in the placebo group (95% confidence interval, 1.25 to 4.56; P=0.008). The median percentage reduction from baseline in the glucocorticoid dose was 50% in the mepolizumab group, as compared with no reduction in the placebo group (P=0.007). Despite receiving a reduced glucocorticoid dose, patients in the mepolizumab group, as compared with those in the placebo group, had a relative reduction of 32% in the annualized rate of exacerbations (1.44 vs. 2.12, P=0.04) and a reduction of 0.52 points with respect to asthma symptoms (P=0.004), as measured on the Asthma Control Questionnaire 5 (in which the minimal clinically important difference is 0.5 points). The safety profile of mepolizumab was similar to that of placebo. CONCLUSIONS: In patients requiring daily oral glucocorticoid therapy to maintain asthma control, mepolizumab had a significant glucocorticoid-sparing effect, reduced exacerbations, and improved control of asthma symptoms. (Funded by GlaxoSmithKline; SIRIUS ClinicalTrials.gov number, NCT01691508.).'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_002',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 628}]}],\n",
       "   'context': \"Mepolizumab treatment in patients with severe eosinophilic asthma. BACKGROUND: Some patients with severe asthma have frequent exacerbations associated with persistent eosinophilic inflammation despite continuous treatment with high-dose inhaled glucocorticoids with or without oral glucocorticoids. METHODS: In this randomized, double-blind, double-dummy study, we assigned 576 patients with recurrent asthma exacerbations and evidence of eosinophilic inflammation despite high doses of inhaled glucocorticoids to one of three study groups. Patients were assigned to receive mepolizumab, a humanized monoclonal antibody against interleukin-5, which was administered as either a 75-mg intravenous dose or a 100-mg subcutaneous dose, or placebo every 4 weeks for 32 weeks. The primary outcome was the rate of exacerbations. Other outcomes included the forced expiratory volume in 1 second (FEV1) and scores on the St. George's Respiratory Questionnaire (SGRQ) and the 5-item Asthma Control Questionnaire (ACQ-5). Safety was also assessed. RESULTS: The rate of exacerbations was reduced by 47% (95% confidence interval [CI], 29 to 61) among patients receiving intravenous mepolizumab and by 53% (95% CI, 37 to 65) among those receiving subcutaneous mepolizumab, as compared with those receiving placebo (P<0.001 for both comparisons). Exacerbations necessitating an emergency department visit or hospitalization were reduced by 32% in the group receiving intravenous mepolizumab and by 61% in the group receiving subcutaneous mepolizumab. At week 32, the mean increase from baseline in FEV1 was 100 ml greater in patients receiving intravenous mepolizumab than in those receiving placebo (P=0.02) and 98 ml greater in patients receiving subcutaneous mepolizumab than in those receiving placebo (P=0.03). The improvement from baseline in the SGRQ score was 6.4 points and 7.0 points greater in the intravenous and subcutaneous mepolizumab groups, respectively, than in the placebo group (minimal clinically important change, 4 points), and the improvement in the ACQ-5 score was 0.42 points and 0.44 points greater in the two mepolizumab groups, respectively, than in the placebo group (minimal clinically important change, 0.5 points) (P<0.001 for all comparisons). The safety profile of mepolizumab was similar to that of placebo. CONCLUSIONS: Mepolizumab administered either intravenously or subcutaneously significantly reduced asthma exacerbations and was associated with improvements in markers of asthma control. (Funded by GlaxoSmithKline; MENSA ClinicalTrials.gov number, NCT01691521.).\"},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_003',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 17}]}],\n",
       "   'context': 'Efficacy of anti-interleukin-5 therapy with mepolizumab in patients with asthma: a meta-analysis of randomized placebo-controlled trials. BACKGROUND: Interleukin (IL)-5 is believed to be a key cytokine in eosinophil inflammatory infiltration in asthma. Previous clinical trials have evaluated the efficacy and safety of mepolizumab, a monoclonal antibody against IL-5, in patients with asthma. However, most of these studies were small, the conclusions were inconsistent, and the precise effects are therefore debatable. METHODS: A meta-analysis of randomized placebo-controlled trials was conducted to evaluate the effect of intravenous infusion of mepolizumab on clinical outcomes in patients with asthma. Trials were searched in PubMed, Embase, Web of Science, Cochrane CENTRAL, Scopus, reviews, and reference lists of relevant articles. The outcome variables analyzed included eosinophil counts in blood and sputum, airways outcome measures, exacerbations, asthma control, and quality of life scores. RESULTS: Seven studies met final inclusion criteria (total n = 1131). From the pooled analyses, mepolizumab significantly reduced eosinophils in blood (MD -0.29×10(9)/L, 95% CI -0.44 to -0.14×10(9)/L, P = 0.0001) and sputum (MD -6.05%, 95% CI -9.34 to -2.77%, P = 0.0003). Mepolizumab was also associated with significantly decreased exacerbation risk than placebo (OR 0.30, 95%CI 0.13 to 0.67, P = 0.004), and with a significant improvement in the scores on the Asthma Quality of Life Questionnaire (AQLQ) (MD 0.26, 95% CI 0.03 to 0.49, P = 0.03) in patients with eosinophilic asthma. There were no statistical differences between the groups with respect to FEV1, PEF, or histamine PC20 (all P>0.05), and a non-significant trend for improvement in scores on the Juniper Asthma Control Questionnaire (JACQ) (MD -0.21, 95% CI -0.43 to 0.01, P = 0.06) in the mepolizumab group was observed. CONCLUSIONS: Mepolizumab reduces the risk of exacerbations and improves quality of life in patients with eosinophilic asthma, but no significant improvement in lung function outcomes was observed. Further research is required to establish the possible role of anti-IL-5 as a therapy for asthma.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_004',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 529}]}],\n",
       "   'context': 'Mepolizumab treatment for asthma. INTRODUCTION: Five percent of asthmatics have severe symptoms despite high doses of inhaled (ICS) or additional oral corticosteroids (OCS): these patients have high morbidity, risk for asthma death, and account for half of asthma healthcare spending. A subgroup (20 - 40%) of these has persistent airway eosinophilia and frequent exacerbations. Mepolizumab is a humanized monoclonal antibody that blocks binding of the key cytokine implicated specifically in eosinophil maturation and survival, interleukin-5, to its receptor. AREAS COVERED: Pharmacology, Phase I/IIa and Phase II/III studies of mepolizumab for asthma. Mepolizumab depleted blood and sputum eosinophils and partially reduced airway and bone marrow eosinophils. It also reduced airway remodeling. In unselected patients with moderate/severe asthma there was no clinically significant effect on lung function, but a trend to reduced exacerbation rates. When patients were selected for persistent sputum eosinophilia despite high-dose ICS/OCS, and frequent exacerbations, mepolizumab reduced exacerbations by 50%. EXPERT OPINION: Mepolizumab can reduce exacerbation rates in the severe asthma cohort who have eosinophilic airway inflammation despite corticosteroid treatment. This may be 30% of severe asthmatics and represents a new and important treatment option. Further studies need to confirm efficacy and indications for asthma (and other eosinophilic airway disease), and to examine clinical consequences of reducing remodeling.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_005',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 1517}]}],\n",
       "   'context': 'Eosinophilic oesophagitis: epidemiology, pathogenesis and management. Eosinophilic oesophagitis (EE) is a clinico-pathological entity recognized with increased frequency in children and adults. It is an atopic disease involving ingested and inhaled allergens. A pathological eosinophilic infiltrate is diagnosed by finding > 15 eosinophils per high-powered field on oesophageal mucosal biopsies. This infiltrate may result in a narrowed oesophageal lumen. It does not involve the stomach or duodenum. Children commonly present with abdominal pain, vomiting and dysphagia. Presentation in adults is with dysphagia, heartburn, chest pain or impaction of a food bolus in the oesophagus. There is often a history of allergy (asthma, hay fever, eczema). A male predominance (70% in adults) is unexplained. Distinctive endoscopic features are linear furrows, mucosal rings and white papules, and the narrowed lumen may be appreciated. Although EE and gastro-oesophageal reflux disease are separate entities, there is a significant overlap of the conditions. Treatment options include nonpharmacological approaches including an elimination or elemental diet, and/ or medications, chiefly with corticosteroids. The topical administration of fluticasone propionate has been demonstrated to improve symptoms and mobilize the pathological infiltrate of eosinophils. There has been a variable effect with the leukotriene receptor antagonist montelukast and promising early results with mepolizumab, a monoclonal antibody against interleukin-5. The long-term efficacy of topical corticosteroids has not been well studied and most patients experience recurrent symptoms when treatment is completed. Currently, repeated short courses of topical corticosteroids are utilized. Acid suppression by a proton pump inhibitor may be considered in view of the overlap between EE and gastro-oesophageal reflux disease.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_006',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 62}]}],\n",
       "   'context': 'Pharmacokinetics and pharmacodynamics of mepolizumab, an anti-interleukin-5 monoclonal antibody. Mepolizumab is a fully humanized monoclonal antibody (IgG1/κ) targeting human interleukin-5 (IL-5), a key haematopoietin needed for eosinophil development and function. Mepolizumab blocks human IL-5 from binding to the α-chain of the IL-5 receptor complex on the eosinophil cell surface, thereby inhibiting IL-5 signalling. The pharmacokinetics of mepolizumab have been evaluated in clinical studies at doses of 0.05-10 mg/kg and at 250 mg, 750 mg and 1500 mg. Mepolizumab was eliminated slowly, with mean initial and terminal phase half-life values of approximately 2 and 20 days, respectively. Plasma clearance ranged from 0.064 to 0.163 mL/h/kg and steady-state volume of distribution ranged from 49 to 93 mL/kg. Pharmacokinetics were dose proportional and time independent. Estimates based on a two-compartment intravenous infusion model from patients with asthma or healthy subjects following single doses predicted mepolizumab plasma concentrations in multiple-dose studies involving patients with hypereosinophilic syndrome (HES), asthma or eosinophilic oesophagitis. The absolute bioavailability of mepolizumab was 64-75% following subcutaneous injection and 81% following intramuscular injection. Peripheral blood eosinophil levels decreased in healthy subjects and patients with HES, asthma, eosinophilic oesophagitis or atopic dermatitis after intravenous mepolizumab infusion and subcutaneous injection. Reductions in eosinophil counts in oesophagus, sputum, skin, bone marrow, nasal lavage fluid and/or bronchial mucosa after treatment with mepolizumab were observed in placebo-controlled studies in various indications. The relationship between percentage change from baseline in blood eosinophils and mepolizumab plasma concentrations was described by an indirect pharmacological response model. The estimated maximal decrease in eosinophil count was approximately 85% from baseline and the half-maximal inhibitory concentration (IC50) was approximately 0.45 μg/mL.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_007',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 175}]}],\n",
       "   'context': 'Pharmacokinetics and pharmacodynamics of mepolizumab, an anti-interleukin-5 monoclonal antibody. Mepolizumab is a fully humanized monoclonal antibody (IgG1/κ) targeting human interleukin-5 (IL-5), a key haematopoietin needed for eosinophil development and function. Mepolizumab blocks human IL-5 from binding to the α-chain of the IL-5 receptor complex on the eosinophil cell surface, thereby inhibiting IL-5 signalling. The pharmacokinetics of mepolizumab have been evaluated in clinical studies at doses of 0.05-10 mg/kg and at 250 mg, 750 mg and 1500 mg. Mepolizumab was eliminated slowly, with mean initial and terminal phase half-life values of approximately 2 and 20 days, respectively. Plasma clearance ranged from 0.064 to 0.163 mL/h/kg and steady-state volume of distribution ranged from 49 to 93 mL/kg. Pharmacokinetics were dose proportional and time independent. Estimates based on a two-compartment intravenous infusion model from patients with asthma or healthy subjects following single doses predicted mepolizumab plasma concentrations in multiple-dose studies involving patients with hypereosinophilic syndrome (HES), asthma or eosinophilic oesophagitis. The absolute bioavailability of mepolizumab was 64-75% following subcutaneous injection and 81% following intramuscular injection. Peripheral blood eosinophil levels decreased in healthy subjects and patients with HES, asthma, eosinophilic oesophagitis or atopic dermatitis after intravenous mepolizumab infusion and subcutaneous injection. Reductions in eosinophil counts in oesophagus, sputum, skin, bone marrow, nasal lavage fluid and/or bronchial mucosa after treatment with mepolizumab were observed in placebo-controlled studies in various indications. The relationship between percentage change from baseline in blood eosinophils and mepolizumab plasma concentrations was described by an indirect pharmacological response model. The estimated maximal decrease in eosinophil count was approximately 85% from baseline and the half-maximal inhibitory concentration (IC50) was approximately 0.45 μg/mL.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_008',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 5}]}],\n",
       "   'context': 'Anti-interleukin-5 antibody treatment (mepolizumab) in active eosinophilic oesophagitis: a randomised, placebo-controlled, double-blind trial. OBJECTIVE: Eosinophilic oesophagitis (EoO) is a clinicopathological condition defined by proton pump inhibitor-refractory oesophageal symptoms combined with oesophageal eosinophilia. The pharmacodynamic effect of mepolizumab (a humanised anti-interleukin-5 monoclonal antibody) in EoO was evaluated. METHODS: Eleven adults with active EoO (>20 peak eosinophil number/high power field (hpf) and dysphagia) were randomised to 750 mg of mepolizumab (n = 5) or placebo (n = 6) and received two intravenous infusions, 1 week apart. Those not in complete remission (<5 peak eosinophil number/hpf) after 8 weeks received two further doses 4 weeks apart, 1500 mg of mepolizumab or placebo. The effect of mepolizumab was assessed clinically, endoscopically, histologically, and via blood and tissue biomarkers. RESULTS: As assessed by immunofluorescence, a marked reduction of mean oesophageal eosinophilia (p = 0.03) was seen in the mepolizumab group (-54%) compared with the placebo group (-5%) 4 weeks after initiation of treatment. No further reduction of eosinophil numbers was observed in response to the two additional infusions in either group. Mepolizumab reduced tenascin C (p = 0.033) and transforming growth factor beta1 (p = 0.05) expression in the oesophageal epithelial layer 13 weeks after initiation of treatment. Clinically, limited improvement of symptoms was seen, although a trend was seen between 4 and 13 weeks after initiation of mepolizumab treatment. Mepolizumab was well tolerated. CONCLUSIONS: Mepolizumab significantly reduced eosinophil numbers in oesophageal tissues in adult patients with active EoO, and changes in the expression of molecules associated with oesophageal remodelling were reversed. Minimal clinical improvement was achieved in a subgroup of patients with EoO. Mepolizumab had an acceptable safety profile, even at the high 1500 mg dose level. TRIAL REGISTRATION NUMBER: NCT00274703.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_009',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 386}]}],\n",
       "   'context': 'Anti-interleukin-5 antibody treatment (mepolizumab) in active eosinophilic oesophagitis: a randomised, placebo-controlled, double-blind trial. OBJECTIVE: Eosinophilic oesophagitis (EoO) is a clinicopathological condition defined by proton pump inhibitor-refractory oesophageal symptoms combined with oesophageal eosinophilia. The pharmacodynamic effect of mepolizumab (a humanised anti-interleukin-5 monoclonal antibody) in EoO was evaluated. METHODS: Eleven adults with active EoO (>20 peak eosinophil number/high power field (hpf) and dysphagia) were randomised to 750 mg of mepolizumab (n = 5) or placebo (n = 6) and received two intravenous infusions, 1 week apart. Those not in complete remission (<5 peak eosinophil number/hpf) after 8 weeks received two further doses 4 weeks apart, 1500 mg of mepolizumab or placebo. The effect of mepolizumab was assessed clinically, endoscopically, histologically, and via blood and tissue biomarkers. RESULTS: As assessed by immunofluorescence, a marked reduction of mean oesophageal eosinophilia (p = 0.03) was seen in the mepolizumab group (-54%) compared with the placebo group (-5%) 4 weeks after initiation of treatment. No further reduction of eosinophil numbers was observed in response to the two additional infusions in either group. Mepolizumab reduced tenascin C (p = 0.033) and transforming growth factor beta1 (p = 0.05) expression in the oesophageal epithelial layer 13 weeks after initiation of treatment. Clinically, limited improvement of symptoms was seen, although a trend was seen between 4 and 13 weeks after initiation of mepolizumab treatment. Mepolizumab was well tolerated. CONCLUSIONS: Mepolizumab significantly reduced eosinophil numbers in oesophageal tissues in adult patients with active EoO, and changes in the expression of molecules associated with oesophageal remodelling were reversed. Minimal clinical improvement was achieved in a subgroup of patients with EoO. Mepolizumab had an acceptable safety profile, even at the high 1500 mg dose level. TRIAL REGISTRATION NUMBER: NCT00274703.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_010',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 459}]}],\n",
       "   'context': 'Mepolizumab for prednisone-dependent asthma with sputum eosinophilia. BACKGROUND: Eosinophilic inflammation, which may be a consequence of interleukin-5 action, is a characteristic feature of some forms of asthma. However, in three previous clinical trials involving patients with asthma, blockade of this cytokine did not result in a significant improvement in outcomes. We studied the prednisone-sparing effect of mepolizumab, a monoclonal antibody against interleukin-5, in a rare subgroup of patients who have sputum eosinophilia and airway symptoms despite continued treatment with prednisone. Secondary objectives were to examine its effect on the number of eosinophils in sputum and blood, symptoms, and airflow limitation. METHODS: In this randomized, double-blind, parallel-group trial involving patients with persistent sputum eosinophilia and symptoms despite prednisone treatment, we assigned 9 patients to receive mepolizumab (administered in five monthly infusions of 750 mg each) and 11 patients to receive placebo. RESULTS: There were 12 asthma exacerbations in 10 patients who received placebo, 9 of whom had sputum eosinophilia at the time of exacerbation. In comparison, only one patient who received mepolizumab had an asthma exacerbation, and this episode was not associated with sputum eosinophilia (P=0.002). Patients who received mepolizumab were able to reduce their prednisone dose by a mean (+/-SD) of 83.8+/-33.4% of their maximum possible dose, as compared with 47.7+/-40.5% in the placebo group (P=0.04). The use of mepolizumab was associated with a significant decrease in the number of sputum and blood eosinophils. Improvements in eosinophil numbers, asthma control, and forced expiratory volume in 1 second were maintained for 8 weeks after the last infusion. There were no serious adverse events. CONCLUSIONS: Mepolizumab reduced the number of blood and sputum eosinophils and allowed prednisone sparing in patients who had asthma with sputum eosinophilia despite prednisone treatment. (ClinicalTrials.gov number, NCT00292877.)'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_011',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 634}]}],\n",
       "   'context': 'Mepolizumab and exacerbations of refractory eosinophilic asthma. BACKGROUND: Exacerbations of asthma are associated with substantial morbidity and mortality and with considerable use of health care resources. Preventing exacerbations remains an important goal of therapy. There is evidence that eosinophilic inflammation of the airway is associated with the risk of exacerbations. METHODS: We conducted a randomized, double-blind, placebo-controlled, parallel-group study of 61 subjects who had refractory eosinophilic asthma and a history of recurrent severe exacerbations. Subjects received infusions of either mepolizumab, an anti-interleukin-5 monoclonal antibody (29 subjects), or placebo (32) at monthly intervals for 1 year. The primary outcome measure was the number of severe exacerbations per subject during the 50-week treatment phase. Secondary outcomes included a change in asthma symptoms, scores on the Asthma Quality of Life Questionnaire (AQLQ, in which scores range from 1 to 7, with lower values indicating more severe impairment and a change of 0.5 unit considered to be clinically important), forced expiratory volume in 1 second (FEV(1)) after use of a bronchodilator, airway hyperresponsiveness, and eosinophil counts in the blood and sputum. RESULTS: Mepolizumab was associated with significantly fewer severe exacerbations than placebo over the course of 50 weeks (2.0 vs. 3.4 mean exacerbations per subject; relative risk, 0.57; 95% confidence interval [CI], 0.32 to 0.92; P=0.02) and with a significant improvement in the score on the AQLQ (mean increase from baseline, 0.55 vs. 0.19; mean difference between groups, 0.35; 95% CI, 0.08 to 0.62; P=0.02). Mepolizumab significantly lowered eosinophil counts in the blood (P<0.001) and sputum (P=0.002). There were no significant differences between the groups with respect to symptoms, FEV(1) after bronchodilator use, or airway hyperresponsiveness. The only serious adverse events reported were hospitalizations for acute severe asthma. CONCLUSIONS: Mepolizumab therapy reduces exacerbations and improves AQLQ scores in patients with refractory eosinophilic asthma. The results of our study suggest that eosinophils have a role as important effector cells in the pathogenesis of severe exacerbations of asthma in this patient population. (Current Controlled Trials number, ISRCTN75169762.)'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_012',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 559}]}],\n",
       "   'context': 'Treatment of patients with the hypereosinophilic syndrome with mepolizumab. BACKGROUND: The hypereosinophilic syndrome is a group of diseases characterized by persistent blood eosinophilia, defined as more than 1500 cells per microliter with end-organ involvement and no recognized secondary cause. Although most patients have a response to corticosteroids, side effects are common and can lead to considerable morbidity. METHODS: We conducted an international, randomized, double-blind, placebo-controlled trial evaluating the safety and efficacy of an anti-interleukin-5 monoclonal antibody, mepolizumab, in patients with the hypereosinophilic syndrome. Patients were negative for the FIP1L1-PDGFRA fusion gene and required prednisone monotherapy, 20 to 60 mg per day, to maintain a stable clinical status and a blood eosinophil count of less than 1000 per microliter. Patients received either intravenous mepolizumab or placebo while the prednisone dose was tapered. The primary end point was the reduction of the prednisone dose to 10 mg or less per day for 8 or more consecutive weeks. RESULTS: The primary end point was reached in 84% of patients in the mepolizumab group, as compared with 43% of patients in the placebo group (hazard ratio, 2.90; 95% confidence interval [CI], 1.59 to 5.26; P<0.001) with no increase in clinical activity of the hypereosinophilic syndrome. A blood eosinophil count of less than 600 per microliter for 8 or more consecutive weeks was achieved in 95% of patients receiving mepolizumab, as compared with 45% of patients receiving placebo (hazard ratio, 3.53; 95% CI, 1.94 to 6.45; P<0.001). Serious adverse events occurred in seven patients receiving mepolizumab (14 events, including one death; mean [+/-SD] duration of exposure, 6.7+/-1.9 months) and in five patients receiving placebo (7 events; mean duration of exposure, 4.3+/-2.6 months). CONCLUSIONS: Our study shows that treatment with mepolizumab, an agent designed to target eosinophils, can result in corticosteroid-sparing for patients negative for FIP1L1-PDGFRA who have the hypereosinophilic syndrome. (ClinicalTrials.gov number, NCT00086658 [ClinicalTrials.gov].).'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_013',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 75}]}],\n",
       "   'context': \"Mepolizumab: 240563, anti-IL-5 monoclonal antibody - GlaxoSmithKline, anti-interleukin-5 monoclonal antibody - GlaxoSmithKline, SB 240563. Mepolizumab is an anti-interleukin-5 monoclonal antibody that is in clinical trials with GlaxoSmithKline (GSK) for the treatment of severe asthma, nasal polyposis and hypereosinophilic syndrome and eosinophilic oesophagitis (the latter two indications are classed as eosinophilia in the phase table). Interleukin-5 stimulates the production, activation and maturation of eosinophils. Since mepolizumab inhibits interleukin-5 and has a long terminal half-life, treatment with mepolizumab causes a sustained reduction in the numbers of circulating eosinophils. Thus, mepolizumab may be a useful therapeutic agent for the treatment of conditions characterized by increased levels of eosinophils. Hypereosinophilic syndrome is a rare idiopathic disease with broad clinical signs and symptoms that is diagnosed based on a persistent blood eosinophil count of >1500 cells, various end-organ damages (including skin, heart, lung, nervous system and digestive system), and with exclusion of known secondary causes of hypereosinophilia. Mepolizumab is in clinical trials for the treatment of hypereosinophilic syndrome, eosinophilc oesophagitis, severe asthma (in patients with airway eosinophilia) and nasal polyposis. GlaxoSmithKline (GSK) has completed enrolment in a phase II study of mepolizumab in 20 patients with symptomatic eosinophilic bronchitis with or without asthma in Canada. The randomized, double-blind, placebo-controlled study is evaluating the effects of intravenous mepolizumab on asthma control, airway eosinophilia and the degree to which concomitant corticosteroid treatment can be reduced (NCT00292877). In previous clinical studies, including trials in the EU and US, mepolizumab has shown a lack of effect on allergen-induced airway responses and inflammation despite a significant reduction in blood and sputum eosinophil levels.A randomized, double-blind, placebo-controlled, multicentre, phase III study of mepolizumab over 9 months in 85 patients with hypereosinophilic syndrome was completed in 2006. All patients have been offered, and continued in, a phase III, open-label, long-term extension study of mepolizumab. Enrolment in this study was completed in September 2006.A phase III, compassionate use trial of mepalizumab (NCT00244686) in patients with hypereosinophilic syndrome was ongoing in October 2007 in the US. Patients who have significant clinical disease but are unresponsive to traditional treatment and those who have demonstrated clinical benefit from previous anti-IL-5 treatment are eligible to take part in the trial. Mepolizumab received orphan drug status for first-line treatment in patients with hypereosinophilic syndrome in the US and the EU in 2004. Mepolizumab is also in phase I/II clinical development for the treatment of eosinophilic oesophagitis. A phase I/II trial (NCT00358449) began in August 2006 in the US, Australia, the UK and Canada, and will enrol approximately 72 paediatric patients with eosinophilic oesophagitis. The randomized, parallel-group clinical trial will evaluate the safety, tolerability, pharmacokinetics and pharmacodynamics of intravenous mepolizumab for 12 weeks. In September 2006, GSK completed enrolment in a phase I/II study of mepolizumab for the treatment of eosinophilic oesophagitis in ten adult patients in Switzerland (NCT00274703). The randomized, double-blind, placebo-controlled study will evaluate the pharmacokinetics, pharmacodynamics, safety and tolerability of IV mepolizumab.A phase I/II trial of mepolizumab in four patients with eosinophilic oesophagitis conducted by Cincinnati Children's hospital found the monoclonal antibody was safe and effective. Brigham and Women's Hospital in association with GSK is conducting a phase I/II trial of mepolizumab in patients with Churg-Strauss Syndrome (CSS) in the US. The trial, which started in September 2007, will evaluate the potential of mepolizumab to reduce the need for corticosteroid therapy in patients with CSS (NCT00527566). CSS, otherwise known as allergic granulomatosis, is defined by patients with asthma, eosinophilia and vasculitis.\"},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_014',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 162}]}],\n",
       "   'context': \"Mepolizumab: 240563, anti-IL-5 monoclonal antibody - GlaxoSmithKline, anti-interleukin-5 monoclonal antibody - GlaxoSmithKline, SB 240563. Mepolizumab is an anti-interleukin-5 monoclonal antibody that is in clinical trials with GlaxoSmithKline (GSK) for the treatment of severe asthma, nasal polyposis and hypereosinophilic syndrome and eosinophilic oesophagitis (the latter two indications are classed as eosinophilia in the phase table). Interleukin-5 stimulates the production, activation and maturation of eosinophils. Since mepolizumab inhibits interleukin-5 and has a long terminal half-life, treatment with mepolizumab causes a sustained reduction in the numbers of circulating eosinophils. Thus, mepolizumab may be a useful therapeutic agent for the treatment of conditions characterized by increased levels of eosinophils. Hypereosinophilic syndrome is a rare idiopathic disease with broad clinical signs and symptoms that is diagnosed based on a persistent blood eosinophil count of >1500 cells, various end-organ damages (including skin, heart, lung, nervous system and digestive system), and with exclusion of known secondary causes of hypereosinophilia. Mepolizumab is in clinical trials for the treatment of hypereosinophilic syndrome, eosinophilc oesophagitis, severe asthma (in patients with airway eosinophilia) and nasal polyposis. GlaxoSmithKline (GSK) has completed enrolment in a phase II study of mepolizumab in 20 patients with symptomatic eosinophilic bronchitis with or without asthma in Canada. The randomized, double-blind, placebo-controlled study is evaluating the effects of intravenous mepolizumab on asthma control, airway eosinophilia and the degree to which concomitant corticosteroid treatment can be reduced (NCT00292877). In previous clinical studies, including trials in the EU and US, mepolizumab has shown a lack of effect on allergen-induced airway responses and inflammation despite a significant reduction in blood and sputum eosinophil levels.A randomized, double-blind, placebo-controlled, multicentre, phase III study of mepolizumab over 9 months in 85 patients with hypereosinophilic syndrome was completed in 2006. All patients have been offered, and continued in, a phase III, open-label, long-term extension study of mepolizumab. Enrolment in this study was completed in September 2006.A phase III, compassionate use trial of mepalizumab (NCT00244686) in patients with hypereosinophilic syndrome was ongoing in October 2007 in the US. Patients who have significant clinical disease but are unresponsive to traditional treatment and those who have demonstrated clinical benefit from previous anti-IL-5 treatment are eligible to take part in the trial. Mepolizumab received orphan drug status for first-line treatment in patients with hypereosinophilic syndrome in the US and the EU in 2004. Mepolizumab is also in phase I/II clinical development for the treatment of eosinophilic oesophagitis. A phase I/II trial (NCT00358449) began in August 2006 in the US, Australia, the UK and Canada, and will enrol approximately 72 paediatric patients with eosinophilic oesophagitis. The randomized, parallel-group clinical trial will evaluate the safety, tolerability, pharmacokinetics and pharmacodynamics of intravenous mepolizumab for 12 weeks. In September 2006, GSK completed enrolment in a phase I/II study of mepolizumab for the treatment of eosinophilic oesophagitis in ten adult patients in Switzerland (NCT00274703). The randomized, double-blind, placebo-controlled study will evaluate the pharmacokinetics, pharmacodynamics, safety and tolerability of IV mepolizumab.A phase I/II trial of mepolizumab in four patients with eosinophilic oesophagitis conducted by Cincinnati Children's hospital found the monoclonal antibody was safe and effective. Brigham and Women's Hospital in association with GSK is conducting a phase I/II trial of mepolizumab in patients with Churg-Strauss Syndrome (CSS) in the US. The trial, which started in September 2007, will evaluate the potential of mepolizumab to reduce the need for corticosteroid therapy in patients with CSS (NCT00527566). CSS, otherwise known as allergic granulomatosis, is defined by patients with asthma, eosinophilia and vasculitis.\"},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_015',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 499}]}],\n",
       "   'context': 'A study to evaluate safety and efficacy of mepolizumab in patients with moderate persistent asthma. RATIONALE: Accumulation of eosinophils in the bronchial mucosa of individuals with asthma is considered to be a central event in the pathogenesis of asthma. In animal models, airway eosinophil recruitment and airway hyperresponsiveness in response to allergen challenge are reduced by specific targeting of interleukin-5. A previous small dose-finding study found that mepolizumab, a humanized anti-interleukin-5 monoclonal antibody, had no effect on allergen challenge in humans. OBJECTIVES: To investigate the effect of three intravenous infusions of mepolizumab, 250 or 750 mg at monthly intervals, on clinical outcome measures in 362 patients with asthma experiencing persistent symptoms despite inhaled corticosteroid therapy (400-1,000 mug of beclomethasone or equivalent). METHODS: Multicenter, randomized, double-blind, placebo-controlled study. MEASUREMENTS AND MAIN RESULTS: Morning peak expiratory flow, forced expiratory volume in 1 second, daily beta(2)-agonist use, symptom scores, exacerbation rates, and quality of life measures. Sputum eosinophil levels were also measured in a subgroup of 37 individuals. Mepolizumab was associated with a significant reduction in blood and sputum eosinophils in both treatment groups (blood, P < 0.001 for both doses; sputum, P = 0.006 for 250 mg and P = 0.004 for 750 mg). There were no statistically significant changes in any of the clinical end points measured. There was a nonsignificant trend for decrease in exacerbation rates in the mepolizumab 750-mg treatment group (P = 0.065). CONCLUSIONS: Mepolizumab treatment does not appear to add significant clinical benefit in patients with asthma with persistent symptoms despite inhaled corticosteroid therapy. Further studies are needed to investigate the effect of mepolizumab on exacerbation rates, using protocols specifically tailored to patients with asthma with persistent airway eosinophilia.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_016',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 18}]}],\n",
       "   'context': 'No effect of anti-interleukin-5 therapy (mepolizumab) on the atopy patch test in atopic dermatitis patients. BACKGROUND: The atopy patch test (APT) is an in vivo model to study the induction of eczema by inhalant allergens in atopic dermatitis (AD) patients. Mepolizumab is a monoclonal antibody to interleukin-5, which reduces peripheral blood eosinophils. Previously, we reported that mepolizumab treatment did not result in clinical improvement in AD. The current study investigates the effect of mepolizumab therapy on the APT in the same patients. METHODS: Mepolizumab treatment was given at days 0 and 7 in a double-blind placebo-controlled design. The APT was applied at days -2, 0, 14 and 28. Clinical evaluation of each APT was conducted 48 h after application at days 0, 2, 16 and 30. Skin biopsies were taken at days 0, 2 and 16 for eosinophil counts. RESULTS: The mepolizumab-treated group showed no significant reduction in macroscopic outcome of the APT. Tissue eosinophils were reduced in the mepolizumab-treated group at day 16 compared with placebo; however, this was not significant. CONCLUSION: Mepolizumab therapy cannot prevent the eczematous reaction induced by the APT. Furthermore, the influx of tissue eosinophil numbers in the APT is not significantly inhibited after mepolizumab treatment compared with placebo, despite a significant reduction in peripheral blood eosinophils.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_017',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 299}]}],\n",
       "   'context': 'No effect of anti-interleukin-5 therapy (mepolizumab) on the atopy patch test in atopic dermatitis patients. BACKGROUND: The atopy patch test (APT) is an in vivo model to study the induction of eczema by inhalant allergens in atopic dermatitis (AD) patients. Mepolizumab is a monoclonal antibody to interleukin-5, which reduces peripheral blood eosinophils. Previously, we reported that mepolizumab treatment did not result in clinical improvement in AD. The current study investigates the effect of mepolizumab therapy on the APT in the same patients. METHODS: Mepolizumab treatment was given at days 0 and 7 in a double-blind placebo-controlled design. The APT was applied at days -2, 0, 14 and 28. Clinical evaluation of each APT was conducted 48 h after application at days 0, 2, 16 and 30. Skin biopsies were taken at days 0, 2 and 16 for eosinophil counts. RESULTS: The mepolizumab-treated group showed no significant reduction in macroscopic outcome of the APT. Tissue eosinophils were reduced in the mepolizumab-treated group at day 16 compared with placebo; however, this was not significant. CONCLUSION: Mepolizumab therapy cannot prevent the eczematous reaction induced by the APT. Furthermore, the influx of tissue eosinophil numbers in the APT is not significantly inhibited after mepolizumab treatment compared with placebo, despite a significant reduction in peripheral blood eosinophils.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_018',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 316}]}],\n",
       "   'context': \"Anti-IL-5 recombinant humanized monoclonal antibody (mepolizumab) for the treatment of atopic dermatitis. BACKGROUND: Eosinophils may play an important role in the pathogenesis of atopic dermatitis (AD). Interleukin-5 is essential for eosinophil growth, differentiation and migration. A monoclonal antibody to human interleukin-5 (mepolizumab) was developed for atopic diseases. This study was designed to study the effect of mepolizumab in AD. METHODS: Two single doses of 750 mg mepolizumab, given 1 week apart, were studied in patients with moderate to severe AD using a randomized, placebo-controlled parallel group design. The primary endpoint of 'success' to treatment was defined as the percentage of patients with at least 'marked improvement' after 2 weeks as assessed by the Physician's Global Assessment of Improvement (PGA). Furthermore, SCORing AD (SCORAD), pruritus scoring, number of blood eosinophils and serum thymus and activation-regulated chemokine (TARC) values served as secondary endpoints. Fluticasone propionate cream 0.05%, once daily could be used as rescue medication from day 16 if no improvement was recorded. RESULTS: Eighteen patients received mepolizumab and 22 placebo treatment. Peripheral blood eosinophil numbers were significantly reduced in the treatment group compared with placebo (P < 0.05). No clinical success was reached by PGA assessment (P = 0.115), SCORAD (P = 0.293), pruritus scoring and TARC values in the mepolizumab-treated group compared with placebo. However, modest improvement (<50% improvement) assessed by PGA was scored significantly more in the mepolizumab-treated group compared with placebo (P < 0.05). CONCLUSION: Two single doses of 750 mg mepolizumab did not result in clinical success in patients with AD, despite a significant decrease in peripheral blood eosinophils.\"},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_019',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 5}]}],\n",
       "   'context': 'Anti-interleukin-5 (mepolizumab) therapy for hypereosinophilic syndromes. BACKGROUND: IL-5 is a cytokine critically involved in regulating several aspects of eosinophils including their production, activation, and tissue recruitment. As such, IL-5 may be involved in the pathogenesis of hypereosinophilic syndromes, a group of poorly treated diverse disorders characterized by sustained peripheral blood and/or tissue eosinophilia. OBJECTIVE: We aimed to assess the safety and efficacy of a humanized blocking monoclonal antibody against IL-5 (mepolizumab) in patients with several forms of hyper-eosinophilic syndromes. METHODS: We performed an open-label trial of anti-IL-5 in which 3 intravenous doses (10 mg/kg, maximum 750 mg) were administered at 4-week intervals to 4 patients with hypereosinophilic syndromes (defined by peripheral blood and/or tissue eosinophilia). The effects of treatment on safety, eosinophil levels (in peripheral blood and/or diseased tissue), pulmonary function, and quality of life were measured over a 28-week period. RESULTS: Anti-IL-5 was well tolerated in all patients and lowered peripheral blood eosinophil counts despite ongoing systemic glucocorticoid therapy. The decline in circulating eosinophil counts was sustained for at least 12 weeks after the last dose of anti-IL-5. In addition, anti-IL-5 improved clinical and quality of life measurements. In one patient with striking tissue eosinophilia (eosinophilic esophagitis), anti-IL-5 resulted in a 10-fold reduction in tissue eosinophil levels. CONCLUSIONS: These results suggest that anti-IL-5 is safe, effective in lowering eosinophil levels, and has potential glucocorticoid-sparing effects in patients with a variety of hyper-eosinophilic syndromes. As such, anti-IL-5 may have significant therapeutic potential for hypereosinophilic syndromes.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_020',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 175}]}],\n",
       "   'context': 'Pharmacokinetics and pharmacodynamics of mepolizumab, an anti-interleukin-5 monoclonal antibody. Mepolizumab is a fully humanized monoclonal antibody (IgG1/κ) targeting human interleukin-5 (IL-5), a key haematopoietin needed for eosinophil development and function. Mepolizumab blocks human IL-5 from binding to the α-chain of the IL-5 receptor complex on the eosinophil cell surface, thereby inhibiting IL-5 signalling. The pharmacokinetics of mepolizumab have been evaluated in clinical studies at doses of 0.05-10 mg/kg and at 250 mg, 750 mg and 1500 mg. Mepolizumab was eliminated slowly, with mean initial and terminal phase half-life values of approximately 2 and 20 days, respectively. Plasma clearance ranged from 0.064 to 0.163 mL/h/kg and steady-state volume of distribution ranged from 49 to 93 mL/kg. Pharmacokinetics were dose proportional and time independent. Estimates based on a two-compartment intravenous infusion model from patients with asthma or healthy subjects following single doses predicted mepolizumab plasma concentrations in multiple-dose studies involving patients with hypereosinophilic syndrome (HES), asthma or eosinophilic oesophagitis. The absolute bioavailability of mepolizumab was 64-75% following subcutaneous injection and 81% following intramuscular injection. Peripheral blood eosinophil levels decreased in healthy subjects and patients with HES, asthma, eosinophilic oesophagitis or atopic dermatitis after intravenous mepolizumab infusion and subcutaneous injection. Reductions in eosinophil counts in oesophagus, sputum, skin, bone marrow, nasal lavage fluid and/or bronchial mucosa after treatment with mepolizumab were observed in placebo-controlled studies in various indications. The relationship between percentage change from baseline in blood eosinophils and mepolizumab plasma concentrations was described by an indirect pharmacological response model. The estimated maximal decrease in eosinophil count was approximately 85% from baseline and the half-maximal inhibitory concentration (IC50) was approximately 0.45 μg/mL.'},\n",
       "  {'qas': [{'id': '54d907c84b1fd0d33c000008_021',\n",
       "     'question': 'Which molecule is targeted by a monoclonal antibody Mepolizumab?',\n",
       "     'answers': [{'text': 'interleukin-5', 'answer_start': 175}]}],\n",
       "   'context': 'Pharmacokinetics and pharmacodynamics of mepolizumab, an anti-interleukin-5 monoclonal antibody. Mepolizumab is a fully humanized monoclonal antibody (IgG1/κ) targeting human interleukin-5 (IL-5), a key haematopoietin needed for eosinophil development and function. Mepolizumab blocks human IL-5 from binding to the α-chain of the IL-5 receptor complex on the eosinophil cell surface, thereby inhibiting IL-5 signalling. The pharmacokinetics of mepolizumab have been evaluated in clinical studies at doses of 0.05-10 mg/kg and at 250 mg, 750 mg and 1500 mg. Mepolizumab was eliminated slowly, with mean initial and terminal phase half-life values of approximately 2 and 20 days, respectively. Plasma clearance ranged from 0.064 to 0.163 mL/h/kg and steady-state volume of distribution ranged from 49 to 93 mL/kg. Pharmacokinetics were dose proportional and time independent. Estimates based on a two-compartment intravenous infusion model from patients with asthma or healthy subjects following single doses predicted mepolizumab plasma concentrations in multiple-dose studies involving patients with hypereosinophilic syndrome (HES), asthma or eosinophilic oesophagitis. The absolute bioavailability of mepolizumab was 64-75% following subcutaneous injection and 81% following intramuscular injection. Peripheral blood eosinophil levels decreased in healthy subjects and patients with HES, asthma, eosinophilic oesophagitis or atopic dermatitis after intravenous mepolizumab infusion and subcutaneous injection. Reductions in eosinophil counts in oesophagus, sputum, skin, bone marrow, nasal lavage fluid and/or bronchial mucosa after treatment with mepolizumab were observed in placebo-controlled studies in various indications. The relationship between percentage change from baseline in blood eosinophils and mepolizumab plasma concentrations was described by an indirect pharmacological response model. The estimated maximal decrease in eosinophil count was approximately 85% from baseline and the half-maximal inhibitory concentration (IC50) was approximately 0.45 μg/mL.'},\n",
       "  {'qas': [{'id': '550342a8f8aee20f27000002_001',\n",
       "     'question': 'Which is the major symptom of the Doose syndrome?',\n",
       "     'answers': [{'text': 'myoclonic astatic epilepsy',\n",
       "       'answer_start': 1000}]}],\n",
       "   'context': '[Influence of ketogenic diet on the clinical effects and electroencephalogram features in 31 children with pharmacoresistant epileptic encephalopathy]. OBJECTIVE: To investigate the effect of ketogenic diet (KD) on the clinical and electroencephalogram features in children with pharmacoresistant epileptic encephalopathy. METHOD: Thirty-one children (19 boys, 12 girls) aged 7 months to 7 years (mean 2 years 5 month) with epilepsy refractory to conventional antiepileptic drugs (AEDs) were included in this study. In addition to their original AED treatment, the children were assigned to different ketogenic diets based on their age. The prospective electro-clinical assessment was performed prior to the KD and then one week, one month and again 3 months after the initiation of therapy, respectively. RESULT: The reduction of seizure frequency in 52%, 68% and 71% of all patients exceeded 50% one week, one month and three months after KD treatment respectively. KD is particularly effective in myoclonic astatic epilepsy (MAE; Doose Syndrome) and West syndrome with 100% and 81.25% of the patients having a greater than 50% seizure reduction, respectively. After 3 months of KD treatment, more than 2/3 patients experienced a reduction in interictal epileptiform discharges (IEDs) and improvement in EEG background. CONCLUSION: The clinical and electroencephalographic improvement confirms that KD is beneficial in children with refractory epilepsy.'},\n",
       "  {'qas': [{'id': '550342a8f8aee20f27000002_002',\n",
       "     'question': 'Which is the major symptom of the Doose syndrome?',\n",
       "     'answers': [{'text': 'Myoclonic astatic epilepsy', 'answer_start': 0}]}],\n",
       "   'context': \"Myoclonic astatic epilepsy (Doose syndrome) - a lamotrigine responsive epilepsy? PURPOSE: Myoclonic astatic epilepsy (MAE, Doose syndrome) is a difficult to treat idiopathic generalized epilepsy of early childhood. MAE frequently shows the course of an epileptic encephalopathy and may result in permanent cognitive impairment. Systematic analyses on clinical effects of different AED combinations are still needed. The purpose of our study was to analyze the therapeutic effect of adjunctive lamotrigine (LTG) in pharmacoresistant MAE patients. PATIENTS AND METHODS: In an exploratory, retrospective study, 10 pharmacoresistant MAE patients were included who had been admitted to the Northern German Epilepsy Center between 07/2007 and 12/2010 and had been treated with LTG. Documentation was performed with the electronic seizure diary Epivista. A total observation period of 32 weeks was defined: 8-week 'pre LTG treatment phase' (before starting with LTG), 16-week 'titration phase' (starting with very low LTG doses), 8-week 'follow-up phase'. Seizure frequency, medication and adverse events were extracted from the electronic diary and evaluated in each particular patient. The individual reduction of seizure frequency per day was defined as primary outcome variable. Additionally, a dose-effect-relationship was analyzed for each patient. RESULTS: Six out of ten patients were seizure free during the follow-up phase. Statistical analysis indicated a significant seizure reduction in seven patients at follow-up compared to the pre LTG treatment phase. Seizure frequency did not significantly decrease in two patients and increased in one patient. A significant relationship between seizure frequency per day and LTG dosage during titration and follow-up phase could be demonstrated in nine patients. Group statistics using the exact Wilcoxon test revealed a significant reduction in seizure frequency (p = 0.049, two-sided). CONCLUSION: Our data provide evidence that adjunctive LTG is an eligible therapeutic option for the treatment of pharmacoresistant MAE and encourage further prospective studies to verify this observation.\"},\n",
       "  {'qas': [{'id': '550342a8f8aee20f27000002_003',\n",
       "     'question': 'Which is the major symptom of the Doose syndrome?',\n",
       "     'answers': [{'text': 'Myoclonic astatic epilepsy', 'answer_start': 90}]}],\n",
       "   'context': \"Myoclonic astatic epilepsy (Doose syndrome) - a lamotrigine responsive epilepsy? PURPOSE: Myoclonic astatic epilepsy (MAE, Doose syndrome) is a difficult to treat idiopathic generalized epilepsy of early childhood. MAE frequently shows the course of an epileptic encephalopathy and may result in permanent cognitive impairment. Systematic analyses on clinical effects of different AED combinations are still needed. The purpose of our study was to analyze the therapeutic effect of adjunctive lamotrigine (LTG) in pharmacoresistant MAE patients. PATIENTS AND METHODS: In an exploratory, retrospective study, 10 pharmacoresistant MAE patients were included who had been admitted to the Northern German Epilepsy Center between 07/2007 and 12/2010 and had been treated with LTG. Documentation was performed with the electronic seizure diary Epivista. A total observation period of 32 weeks was defined: 8-week 'pre LTG treatment phase' (before starting with LTG), 16-week 'titration phase' (starting with very low LTG doses), 8-week 'follow-up phase'. Seizure frequency, medication and adverse events were extracted from the electronic diary and evaluated in each particular patient. The individual reduction of seizure frequency per day was defined as primary outcome variable. Additionally, a dose-effect-relationship was analyzed for each patient. RESULTS: Six out of ten patients were seizure free during the follow-up phase. Statistical analysis indicated a significant seizure reduction in seven patients at follow-up compared to the pre LTG treatment phase. Seizure frequency did not significantly decrease in two patients and increased in one patient. A significant relationship between seizure frequency per day and LTG dosage during titration and follow-up phase could be demonstrated in nine patients. Group statistics using the exact Wilcoxon test revealed a significant reduction in seizure frequency (p = 0.049, two-sided). CONCLUSION: Our data provide evidence that adjunctive LTG is an eligible therapeutic option for the treatment of pharmacoresistant MAE and encourage further prospective studies to verify this observation.\"},\n",
       "  {'qas': [{'id': '550342a8f8aee20f27000002_004',\n",
       "     'question': 'Which is the major symptom of the Doose syndrome?',\n",
       "     'answers': [{'text': 'myoclonic astatic epilepsy',\n",
       "       'answer_start': 136}]}],\n",
       "   'context': \"Dissecting the genetic basis of myoclonic-astatic epilepsy. Herman Doose first described the generalized childhood epilepsy syndrome of myoclonic astatic epilepsy (MAE) in 1970, attributing a genetic cause from this first description. However, although the International League Against Epilepsy (ILAE) defined criteria for MAE in 1989, the diagnostic boundaries of the syndrome continue to be debated. Moreover, 40 years since Doose's first description of MAE, although a genetic predisposition is acknowledged and many studies have demonstrated familial aggregation of seizures within MAE families, the actual genetic determinants of MAE still remain unknown. Although initially thought to be within the same spectrum as severe myoclonic epilepsy of infancy, the exclusion of SCN1A mutations in non-generalized epilepsy with febrile seizures plus (GEFS+) MAE cases has confirmed the genetic distinction of MAE. In this critical review, we shall trace the historical evolution of concepts around MAE and its distinction from Lennox-Gastaut syndrome, review the described phenotypic features of MAE from updated studies that will allow its distinction from other overlap epilepsy syndromes, review the evidence of genetic influences and clues for genetic heterogeneity, and discuss strategies that may be helpful in elucidating the etiology of MAE in light of current genetic techniques.\"},\n",
       "  {'qas': [{'id': '550342a8f8aee20f27000002_005',\n",
       "     'question': 'Which is the major symptom of the Doose syndrome?',\n",
       "     'answers': [{'text': 'myoclonic astatic epilepsy',\n",
       "       'answer_start': 295}]}],\n",
       "   'context': 'One novel Dravet syndrome causing mutation and one recurrent MAE causing mutation in SCN1A gene. Mutations in SCN1A gene, encoding the voltage-gated sodium channel α1-subunit, are found to be associated with severe myoclonic epilepsy in infancy or Dravet syndrome (DS), but only rarely with the myoclonic astatic epilepsy (MAE, or Doose syndrome). We report on two patients with SCN1A mutations and severe epilepsy within the spectrum of generalized epilepsy with febrile seizures plus syndrome (GEFS+), the phenotypes being consistent with DS and MAE, respectively. Analysis of SCN1A revealed a heterozygous de novo frameshift mutation (c.4205_4208delGAAA) in the patient with DS, and a recurrent missense mutation (c.3521C>G) in that suffering from MAE. The missense mutation has been reported in patients with neurological diseases of various manifestations, which suggests that this variability is likely to result from the modifying effects of other genetic or environmental factors. DS phenotype has been mainly found associated with truncation mutations, while predominantly missense mutations and very few prematurely terminating substitutions have been reported in GEFS+ patients.'},\n",
       "  {'qas': [{'id': '550342a8f8aee20f27000002_006',\n",
       "     'question': 'Which is the major symptom of the Doose syndrome?',\n",
       "     'answers': [{'text': 'myoclonic astatic epilepsy',\n",
       "       'answer_start': 179}]}],\n",
       "   'context': 'The natural history of myoclonic astatic epilepsy (Doose syndrome) and Lennox-Gastaut syndrome. The purpose of this article is to present a short review of the natural history of myoclonic astatic epilepsy (MAE; Doose syndrome) and the Lennox-Gastaut syndrome (LGS). In the 1989 classification of the International League Against Epilepsy (ILAE, 1989), MAE and LGS were initially included in group 2.2: \"Cryptogenic or symptomatic generalized epilepsies and syndromes.\" The subsequent classification of the Proposed Diagnostic Scheme for People with Epileptic Seizures and with Epilepsy (see Ref. 8) placed MAE in axis 3 in the \"generalized epilepsy\" group and LGS, severe myoclonic epilepsy of infancy (SMEI or Dravet syndrome) and atypical benign partial epilepsy/pseudo-Lennox syndrome (ABPE/PLS) in the \"epileptic encephalopathy\" group. The semiology of MAE and LGS and their differential diagnosis from SMEI and ABPE/PLS are described. Before the onset of SMEI, MAE, and ABPE/PLS, the development of the child is usually normal. In contrast, in LGS, development is frequently retarded at the onset, depending on the etiopathogenesis of the underlying brain disease. The course of MAE is highly variable with regard to seizure outcome (complete remission in some cases, persistent epilepsy in others) and cognitive development (normal or delayed). The course of LGS and SMEI is generally poor, both with regard to the epilepsy and to the cognitive development whereas the course and seizure outcome of ABPE/PLS is favorable; the patients will be seizure-free at puberty. However, the neuropsychological outcome is less favorable; most patients remain mentally retarded.'},\n",
       "  {'qas': [{'id': '550342a8f8aee20f27000002_007',\n",
       "     'question': 'Which is the major symptom of the Doose syndrome?',\n",
       "     'answers': [{'text': 'myoclonic astatic epilepsy', 'answer_start': 39}]}],\n",
       "   'context': 'Video-EEG analysis of drop seizures in myoclonic astatic epilepsy of early childhood (Doose syndrome). We studied 36 drop seizures in 5 patients with myoclonic astatic epilepsy of early childhood (MAEE) with simultaneous split-screen video recording and polygraph. Sixteen were falling attacks and 20 were either less severe attacks exhibiting only deep head nodding or seizures equivalent to drop attacks in terms of ictal pattern but recorded in the supine position. All seizures except those that occurred in patients in the supine position showed sudden momentary head dropping or collapse of the whole body downward. Recovery to the preictal position was observed in 0.3-1 s. As a result of carefully repeated observations, the 36 seizures were classified as myoclonic flexor type in 9, myoclonic atonic type in 2, and atonic type, with and without transient preceding symptoms in the remaining 25. The MF seizure was characterized by sudden forward flexion of the head and trunk as well as both arms, which caused the patient to fall. In the myoclonic atonic seizure, patients showed brief myoclonic flexor spasms, immediately followed by atonic falling. The AT seizure showed abrupt atonic falling, with and without transient preceding facial expression change and/or twitching of extremities. The ictal EEGs of all 36 seizures exhibited generalized bilaterally synchronous single or multiple spike(s) and wave discharges. Atonic drop attacks appear to be a common cause of ictal epileptic falling in MAEE.'},\n",
       "  {'qas': [{'id': '56bdc79bef6e394741000001_001',\n",
       "     'question': 'Which eye condition is managed by the athens protocol?',\n",
       "     'answers': [{'text': 'Keratoconus', 'answer_start': 0}]}],\n",
       "   'context': 'Keratoconus management: long-term stability of topography-guided normalization combined with high-fluence CXL stabilization (the Athens Protocol). PURPOSE: To investigate refractive, topometric, pachymetric, and visual rehabilitation changes induced by anterior surface normalization for keratoconus by partial topography-guided excimer laser ablation in conjunction with accelerated, high-fluence cross-linking. METHODS: Two hundred thirty-one keratoconic cases subjected to the Athens Protocol procedure were studied for visual acuity, keratometry, pachymetry, and anterior surface irregularity indices up to 3 years postoperatively by Scheimpflug imaging (Oculus Optikgeräte GmbH, Wetzlar, Germany). RESULTS: Mean visual acuity changes at 3 years postoperatively were +0.38 ± 0.31 (range: -0.34 to +1.10) for uncorrected distance visual acuity and +0.20 ± 0.21 (range: -0.32 to +0.90) for corrected distance visual acuity. Mean K1 (flat meridian) keratometric values were 46.56 ± 3.83 diopters (D) (range: 39.75 to 58.30 D) preoperatively, 44.44 ± 3.97 D (range: 36.10 to 55.50 D) 1 month postoperatively, and 43.22 ± 3.80 D (range: 36.00 to 53.70 D) up to 3 years postoperatively. The average Index of Surface Variance was 98.48 ± 43.47 (range: 17 to 208) pre-operatively and 76.80 ± 38.41 (range: 7 to 190) up to 3 years postoperatively. The average Index of Height Decentration was 0.091 ± 0.053 μm (range: 0.006 to 0.275 μm) preoperatively and 0.057 ± 0.040 μm (range: 0.001 to 0.208 μm) up to 3 years postoperatively. Mean thinnest corneal thickness was 451.91 ± 40.02 μm (range: 297 to 547 μm) preoperatively, 353.95 ± 53.90 μm (range: 196 to 480 μm) 1 month postoperatively, and 370.52 ± 58.21 μm (range: 218 to 500 μm) up to 3 years postoperatively. CONCLUSIONS: The Athens Protocol to arrest keratectasia progression and improve corneal regularity demonstrates safe and effective results as a keratoconus management option. Progressive potential for long-term flattening validates using caution in the surface normalization to avoid overcorrection.'},\n",
       "  {'qas': [{'id': '56bdc79bef6e394741000001_002',\n",
       "     'question': 'Which eye condition is managed by the athens protocol?',\n",
       "     'answers': [{'text': 'keratoconus', 'answer_start': 1905}]}],\n",
       "   'context': 'Keratoconus management: long-term stability of topography-guided normalization combined with high-fluence CXL stabilization (the Athens Protocol). PURPOSE: To investigate refractive, topometric, pachymetric, and visual rehabilitation changes induced by anterior surface normalization for keratoconus by partial topography-guided excimer laser ablation in conjunction with accelerated, high-fluence cross-linking. METHODS: Two hundred thirty-one keratoconic cases subjected to the Athens Protocol procedure were studied for visual acuity, keratometry, pachymetry, and anterior surface irregularity indices up to 3 years postoperatively by Scheimpflug imaging (Oculus Optikgeräte GmbH, Wetzlar, Germany). RESULTS: Mean visual acuity changes at 3 years postoperatively were +0.38 ± 0.31 (range: -0.34 to +1.10) for uncorrected distance visual acuity and +0.20 ± 0.21 (range: -0.32 to +0.90) for corrected distance visual acuity. Mean K1 (flat meridian) keratometric values were 46.56 ± 3.83 diopters (D) (range: 39.75 to 58.30 D) preoperatively, 44.44 ± 3.97 D (range: 36.10 to 55.50 D) 1 month postoperatively, and 43.22 ± 3.80 D (range: 36.00 to 53.70 D) up to 3 years postoperatively. The average Index of Surface Variance was 98.48 ± 43.47 (range: 17 to 208) pre-operatively and 76.80 ± 38.41 (range: 7 to 190) up to 3 years postoperatively. The average Index of Height Decentration was 0.091 ± 0.053 μm (range: 0.006 to 0.275 μm) preoperatively and 0.057 ± 0.040 μm (range: 0.001 to 0.208 μm) up to 3 years postoperatively. Mean thinnest corneal thickness was 451.91 ± 40.02 μm (range: 297 to 547 μm) preoperatively, 353.95 ± 53.90 μm (range: 196 to 480 μm) 1 month postoperatively, and 370.52 ± 58.21 μm (range: 218 to 500 μm) up to 3 years postoperatively. CONCLUSIONS: The Athens Protocol to arrest keratectasia progression and improve corneal regularity demonstrates safe and effective results as a keratoconus management option. Progressive potential for long-term flattening validates using caution in the surface normalization to avoid overcorrection.'},\n",
       "  {'qas': [{'id': '56bdc79bef6e394741000001_003',\n",
       "     'question': 'Which eye condition is managed by the athens protocol?',\n",
       "     'answers': [{'text': 'cornea blindness due to severe corneal scarring',\n",
       "       'answer_start': 450}]}],\n",
       "   'context': 'The management of cornea blindness from severe corneal scarring, with the Athens Protocol (transepithelial topography-guided PRK therapeutic remodeling, combined with same-day, collagen cross-linking). PURPOSE: To evaluate the safety and efficacy of combined transepithelial topography-guided photorefractive keratectomy (PRK) therapeutic remodeling, combined with same-day, collagen cross-linking (CXL). This protocol was used for the management of cornea blindness due to severe corneal scarring. METHODS: A 57-year-old man had severe corneal blindness in both eyes. Both corneas had significant central scars attributed to a firework explosion 45 years ago, when the patient was 12 years old. Corrected distance visual acuity (CDVA) was 20/100 both eyes (OU) with refraction: +4.00, -4.50 at 135° in the right eye and +3.50, -1.00 at 55° in the left. Respective keratometries were: 42.3, 60.4 at 17° and 35.8, 39.1 at 151.3°. Cornea transplantation was the recommendation by multiple cornea specialists as the treatment of choice. We decided prior to considering a transplant to employ the Athens Protocol (combined topography-guided partial PRK and CXL) in the right eye in February 2010 and in the left eye in September 2010. The treatment plan for both eyes was designed on the topography-guided wavelight excimer laser platform. RESULTS: Fifteen months after the right eye treatment, the right cornea had improved translucency and was topographically stable with uncorrected distance visual acuity (UDVA) 20/50 and CDVA 20/40 with refraction +0.50, -2.00 at 5°. We noted a similar outcome after similar treatment applied in the left eye with UDVA 20/50 and CDVA 20/40 with -0.50, -2.00 at 170° at the 8-month follow-up. CONCLUSION: In this case, the introduction of successful management of severe cornea abnormalities and scarring with the Athens Protocol may provide an effective alternative to other existing surgical or medical options.'},\n",
       "  {'qas': [{'id': '56bdc79bef6e394741000001_004',\n",
       "     'question': 'Which eye condition is managed by the athens protocol?',\n",
       "     'answers': [{'text': 'keratoconus', 'answer_start': 1905}]}],\n",
       "   'context': 'Keratoconus management: long-term stability of topography-guided normalization combined with high-fluence CXL stabilization (the Athens Protocol). PURPOSE: To investigate refractive, topometric, pachymetric, and visual rehabilitation changes induced by anterior surface normalization for keratoconus by partial topography-guided excimer laser ablation in conjunction with accelerated, high-fluence cross-linking. METHODS: Two hundred thirty-one keratoconic cases subjected to the Athens Protocol procedure were studied for visual acuity, keratometry, pachymetry, and anterior surface irregularity indices up to 3 years postoperatively by Scheimpflug imaging (Oculus Optikgeräte GmbH, Wetzlar, Germany). RESULTS: Mean visual acuity changes at 3 years postoperatively were +0.38 ± 0.31 (range: -0.34 to +1.10) for uncorrected distance visual acuity and +0.20 ± 0.21 (range: -0.32 to +0.90) for corrected distance visual acuity. Mean K1 (flat meridian) keratometric values were 46.56 ± 3.83 diopters (D) (range: 39.75 to 58.30 D) preoperatively, 44.44 ± 3.97 D (range: 36.10 to 55.50 D) 1 month postoperatively, and 43.22 ± 3.80 D (range: 36.00 to 53.70 D) up to 3 years postoperatively. The average Index of Surface Variance was 98.48 ± 43.47 (range: 17 to 208) pre-operatively and 76.80 ± 38.41 (range: 7 to 190) up to 3 years postoperatively. The average Index of Height Decentration was 0.091 ± 0.053 μm (range: 0.006 to 0.275 μm) preoperatively and 0.057 ± 0.040 μm (range: 0.001 to 0.208 μm) up to 3 years postoperatively. Mean thinnest corneal thickness was 451.91 ± 40.02 μm (range: 297 to 547 μm) preoperatively, 353.95 ± 53.90 μm (range: 196 to 480 μm) 1 month postoperatively, and 370.52 ± 58.21 μm (range: 218 to 500 μm) up to 3 years postoperatively. CONCLUSIONS: The Athens Protocol to arrest keratectasia progression and improve corneal regularity demonstrates safe and effective results as a keratoconus management option. Progressive potential for long-term flattening validates using caution in the surface normalization to avoid overcorrection.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_001',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 712}]}],\n",
       "   'context': 'Gender differences in Latin-American patients with rheumatoid arthritis. BACKGROUND: Data on the effect of gender in rheumatoid arthritis (RA) in non-Caucasian populations is scarce. Latin America and the Caribbean (LAC) is a large population with unique characteristics, including high admixture. OBJECTIVE: Our aim was to examine the effect of gender in patients with RA in LAC. METHODS: This was a 2-phase study. First we conducted a cross-sectional and analytical study in which 1128 consecutive Colombian patients with RA were assessed. Second, a systematic review of the literature was done to evaluate the effect of gender in LAC patients with RA. RESULTS: Our results show a high prevalence of RA in LAC women with a ratio of 5.2 women per man. Colombian women with RA are more at risk of having an early age at onset and developing polyautoimmunity and abdominal obesity, and they perform more household duties than their male counterparts. However, male gender was associated with the presence of extra-articular manifestations. Of a total of 641 potentially relevant articles, 38 were considered for final analysis, in which several factors and outcomes related to gender were identified. CONCLUSIONS: RA in LAC women is not only more common but presents with some clinical characteristics that differ from RA presentation in men. Some of those characteristics could explain the high rates of disability and worse prognosis observed in women with RA in LAC.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_002',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 738}]}],\n",
       "   'context': 'Gender differences in Latin-American patients with rheumatoid arthritis. BACKGROUND: Data on the effect of gender in rheumatoid arthritis (RA) in non-Caucasian populations is scarce. Latin America and the Caribbean (LAC) is a large population with unique characteristics, including high admixture. OBJECTIVE: Our aim was to examine the effect of gender in patients with RA in LAC. METHODS: This was a 2-phase study. First we conducted a cross-sectional and analytical study in which 1128 consecutive Colombian patients with RA were assessed. Second, a systematic review of the literature was done to evaluate the effect of gender in LAC patients with RA. RESULTS: Our results show a high prevalence of RA in LAC women with a ratio of 5.2 women per man. Colombian women with RA are more at risk of having an early age at onset and developing polyautoimmunity and abdominal obesity, and they perform more household duties than their male counterparts. However, male gender was associated with the presence of extra-articular manifestations. Of a total of 641 potentially relevant articles, 38 were considered for final analysis, in which several factors and outcomes related to gender were identified. CONCLUSIONS: RA in LAC women is not only more common but presents with some clinical characteristics that differ from RA presentation in men. Some of those characteristics could explain the high rates of disability and worse prognosis observed in women with RA in LAC.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_003',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 1223}]}],\n",
       "   'context': 'Gender differences in Latin-American patients with rheumatoid arthritis. BACKGROUND: Data on the effect of gender in rheumatoid arthritis (RA) in non-Caucasian populations is scarce. Latin America and the Caribbean (LAC) is a large population with unique characteristics, including high admixture. OBJECTIVE: Our aim was to examine the effect of gender in patients with RA in LAC. METHODS: This was a 2-phase study. First we conducted a cross-sectional and analytical study in which 1128 consecutive Colombian patients with RA were assessed. Second, a systematic review of the literature was done to evaluate the effect of gender in LAC patients with RA. RESULTS: Our results show a high prevalence of RA in LAC women with a ratio of 5.2 women per man. Colombian women with RA are more at risk of having an early age at onset and developing polyautoimmunity and abdominal obesity, and they perform more household duties than their male counterparts. However, male gender was associated with the presence of extra-articular manifestations. Of a total of 641 potentially relevant articles, 38 were considered for final analysis, in which several factors and outcomes related to gender were identified. CONCLUSIONS: RA in LAC women is not only more common but presents with some clinical characteristics that differ from RA presentation in men. Some of those characteristics could explain the high rates of disability and worse prognosis observed in women with RA in LAC.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_004',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 1447}]}],\n",
       "   'context': 'Gender differences in Latin-American patients with rheumatoid arthritis. BACKGROUND: Data on the effect of gender in rheumatoid arthritis (RA) in non-Caucasian populations is scarce. Latin America and the Caribbean (LAC) is a large population with unique characteristics, including high admixture. OBJECTIVE: Our aim was to examine the effect of gender in patients with RA in LAC. METHODS: This was a 2-phase study. First we conducted a cross-sectional and analytical study in which 1128 consecutive Colombian patients with RA were assessed. Second, a systematic review of the literature was done to evaluate the effect of gender in LAC patients with RA. RESULTS: Our results show a high prevalence of RA in LAC women with a ratio of 5.2 women per man. Colombian women with RA are more at risk of having an early age at onset and developing polyautoimmunity and abdominal obesity, and they perform more household duties than their male counterparts. However, male gender was associated with the presence of extra-articular manifestations. Of a total of 641 potentially relevant articles, 38 were considered for final analysis, in which several factors and outcomes related to gender were identified. CONCLUSIONS: RA in LAC women is not only more common but presents with some clinical characteristics that differ from RA presentation in men. Some of those characteristics could explain the high rates of disability and worse prognosis observed in women with RA in LAC.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_005',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 1070}]}],\n",
       "   'context': 'Disease progression and treatment responses in a prospective DMARD-naive seropositive early rheumatoid arthritis cohort: does gender matter? OBJECTIVE: To assess gender differences in disease characteristics and treatment responses over time in a disease-modifying antirheumatic drug (DMARD)-naive seropositive early rheumatoid arthritis (RA) cohort. METHODS: Patients with polyarticular disease who were DMARD-naive and had seropositive early RA (< 14 months) were recruited by the Western Consortium of Practicing Rheumatologists. Each patient was examined at study entry, after 6 and 12 months, and yearly thereafter. Clinical and demographic data were collected. We investigated gender differences in baseline disease characteristics and treatment using chi-squared, Mann-Whitney U, and t tests. We used generalized estimating equations (GEE) models for repeated measures to examine whether the rate of change of specific disease outcomes during the first 2 years after DMARD initiation was significantly influenced by gender. RESULTS: At baseline, men (n = 67) and women (n = 225) had similar disease activity and radiographic damage; men, however, had significantly worse erosion, while women had worse joint space narrowing. Despite similar treatment, women had worse disease progression over the 2-year followup, as assessed by trends in Disease Activity Score 28/erythrocyte sedimentation rate (DAS28-ESR4), physician global scores, and tender joint counts. In the GEE model, gender was significantly associated with the rate of change of DAS28-ESR4 scores (p = 0.009), although not independently associated with disease activity. Self-reported measures (Health Assessment Questionnaire-Disability Index, patient global scores, fatigue, pain) were worse among women at baseline and throughout the study period. Men were more likely to achieve remission. CONCLUSION: At baseline, men and women had similar disease activity and joint damage. Responses to treatment over time were better among men in this prebiologic era; women had worse progression despite similar treatment.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_006',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 2029}]}],\n",
       "   'context': 'Disease progression and treatment responses in a prospective DMARD-naive seropositive early rheumatoid arthritis cohort: does gender matter? OBJECTIVE: To assess gender differences in disease characteristics and treatment responses over time in a disease-modifying antirheumatic drug (DMARD)-naive seropositive early rheumatoid arthritis (RA) cohort. METHODS: Patients with polyarticular disease who were DMARD-naive and had seropositive early RA (< 14 months) were recruited by the Western Consortium of Practicing Rheumatologists. Each patient was examined at study entry, after 6 and 12 months, and yearly thereafter. Clinical and demographic data were collected. We investigated gender differences in baseline disease characteristics and treatment using chi-squared, Mann-Whitney U, and t tests. We used generalized estimating equations (GEE) models for repeated measures to examine whether the rate of change of specific disease outcomes during the first 2 years after DMARD initiation was significantly influenced by gender. RESULTS: At baseline, men (n = 67) and women (n = 225) had similar disease activity and radiographic damage; men, however, had significantly worse erosion, while women had worse joint space narrowing. Despite similar treatment, women had worse disease progression over the 2-year followup, as assessed by trends in Disease Activity Score 28/erythrocyte sedimentation rate (DAS28-ESR4), physician global scores, and tender joint counts. In the GEE model, gender was significantly associated with the rate of change of DAS28-ESR4 scores (p = 0.009), although not independently associated with disease activity. Self-reported measures (Health Assessment Questionnaire-Disability Index, patient global scores, fatigue, pain) were worse among women at baseline and throughout the study period. Men were more likely to achieve remission. CONCLUSION: At baseline, men and women had similar disease activity and joint damage. Responses to treatment over time were better among men in this prebiologic era; women had worse progression despite similar treatment.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_007',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 1708}]}],\n",
       "   'context': \"Gender, body mass index and rheumatoid arthritis disease activity: results from the QUEST-RA Study. OBJECTIVES: To investigate whether body mass index (BMI), as a proxy for body fat, influences rheumatoid arthritis (RA) disease activity in a gender-specific manner. METHODS: Consecutive patients with RA were enrolled from 25 countries into the QUEST-RA program between 2005 and 2008. Clinical and demographic data were collected by treating rheumatologists and by patient self-report. Distributions of Disease Activity Scores (DAS28), BMI, age, and disease duration were assessed for each country and for the entire dataset; mean values between genders were compared using Student's t-tests. An association between BMI and DAS28 was investigated using linear regression, adjusting for age, disease duration and country. RESULTS: A total of 5,161 RA patients (4,082 women and 1,079 men) were included in the analyses. Overall, women were younger, had longer disease duration, and higher DAS28 scores than men, but BMI was similar between genders. The mean DAS28 scores increased with increasing BMI from normal to overweight and obese, among women, whereas the opposite trend was observed among men. Regression results showed BMI (continuous or categorical) to be associated with DAS28. Compared to the normal BMI range, being obese was associated with a larger difference in mean DAS28 (0.23, 95% CI: 0.11, 0.34) than being overweight (0.12, 95% CI: 0.03, 0.21); being underweight was not associated with disease activity. These associations were more pronounced among women, and were not explained by any single component of the DAS28. CONCLUSIONS: BMI appears to be associated with RA disease activity in women, but not in men.\"},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_008',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 866}]}],\n",
       "   'context': \"Gender, body mass index and rheumatoid arthritis disease activity: results from the QUEST-RA Study. OBJECTIVES: To investigate whether body mass index (BMI), as a proxy for body fat, influences rheumatoid arthritis (RA) disease activity in a gender-specific manner. METHODS: Consecutive patients with RA were enrolled from 25 countries into the QUEST-RA program between 2005 and 2008. Clinical and demographic data were collected by treating rheumatologists and by patient self-report. Distributions of Disease Activity Scores (DAS28), BMI, age, and disease duration were assessed for each country and for the entire dataset; mean values between genders were compared using Student's t-tests. An association between BMI and DAS28 was investigated using linear regression, adjusting for age, disease duration and country. RESULTS: A total of 5,161 RA patients (4,082 women and 1,079 men) were included in the analyses. Overall, women were younger, had longer disease duration, and higher DAS28 scores than men, but BMI was similar between genders. The mean DAS28 scores increased with increasing BMI from normal to overweight and obese, among women, whereas the opposite trend was observed among men. Regression results showed BMI (continuous or categorical) to be associated with DAS28. Compared to the normal BMI range, being obese was associated with a larger difference in mean DAS28 (0.23, 95% CI: 0.11, 0.34) than being overweight (0.12, 95% CI: 0.03, 0.21); being underweight was not associated with disease activity. These associations were more pronounced among women, and were not explained by any single component of the DAS28. CONCLUSIONS: BMI appears to be associated with RA disease activity in women, but not in men.\"},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_009',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 704}]}],\n",
       "   'context': 'Influence of gender on assessments of disease activity and function in early rheumatoid arthritis in relation to radiographic joint damage. OBJECTIVE: To evaluate gender differences in score on 28-joint Disease Activity Score (DAS28), Health Assessment Questionnaire (HAQ) and Signals Of Functional Impairment (SOFI) and to relate these scores to radiographic joint destruction. METHODS: In all, 549 patients with early RA (62% women) from the BARFOT (for \"Better Anti-Rheumatic FarmacOTherapy\") study were included. At baseline, 1, 2 and 5 years DAS28, HAQ and SOFI scoring, and radiographs of hands and feet were performed. The radiographs were scored using the van der Heijde-Sharp score. RESULTS: In women the DAS28 was significantly higher than in men due to higher scores for general health and tender joints. Likewise, HAQ and VAS pain were rated significantly higher in women. The SOFI score was worse in men during the first 2 years, depending on higher upper limb scores. Total Sharp score (TotSharp), erosion score and joint space narrowing score did not differ between the sexes at any time point. The DAS28 area under the curve (AUC) correlated significantly with TotSharp at 5 years in both genders (r = 0.316, r = 0.313) mainly owing to swollen joints and erythrocyte sedimentation rate (ESR). The SOFI AUC correlated significantly with TotSharp in women (r = 0.135 to 0.220) but not in men. CONCLUSIONS: Despite a similar degree of radiographic joint destruction women had, compared with men, worse scores for DAS28 and HAQ, possibly due to higher pain perception and less muscular strength and perhaps because men overestimate their functional capacity.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_010',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 878}]}],\n",
       "   'context': 'Influence of gender on assessments of disease activity and function in early rheumatoid arthritis in relation to radiographic joint damage. OBJECTIVE: To evaluate gender differences in score on 28-joint Disease Activity Score (DAS28), Health Assessment Questionnaire (HAQ) and Signals Of Functional Impairment (SOFI) and to relate these scores to radiographic joint destruction. METHODS: In all, 549 patients with early RA (62% women) from the BARFOT (for \"Better Anti-Rheumatic FarmacOTherapy\") study were included. At baseline, 1, 2 and 5 years DAS28, HAQ and SOFI scoring, and radiographs of hands and feet were performed. The radiographs were scored using the van der Heijde-Sharp score. RESULTS: In women the DAS28 was significantly higher than in men due to higher scores for general health and tender joints. Likewise, HAQ and VAS pain were rated significantly higher in women. The SOFI score was worse in men during the first 2 years, depending on higher upper limb scores. Total Sharp score (TotSharp), erosion score and joint space narrowing score did not differ between the sexes at any time point. The DAS28 area under the curve (AUC) correlated significantly with TotSharp at 5 years in both genders (r = 0.316, r = 0.313) mainly owing to swollen joints and erythrocyte sedimentation rate (ESR). The SOFI AUC correlated significantly with TotSharp in women (r = 0.135 to 0.220) but not in men. CONCLUSIONS: Despite a similar degree of radiographic joint destruction women had, compared with men, worse scores for DAS28 and HAQ, possibly due to higher pain perception and less muscular strength and perhaps because men overestimate their functional capacity.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_011',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 161}]}],\n",
       "   'context': 'The effects of gender and sex hormones on outcome in rheumatoid arthritis. Disease patterns in RA vary between the sexes; the condition is more commonly seen in women, who exhibit a more aggressive disease and a poorer long-term outcome. Men, however, are more likely than women to die from extra-articular complications of rheumatoid disease. This chapter discusses the outcome and mortality studies that substantiate these conclusions and then examines the possible mechanisms that may account for them, including the HLA system, seropositivity, compliance, response to therapy and pain threshold. In particular, sex and sex hormones emerge as independent risk factors in rheumatoid disease. The epidemiological evidence points towards a peak age of onset of RA at the time of the menopause in women and towards later in life in men. Premenopausal women may fare better than postmenopausal women with RA. The possible protective effects of the oral contraceptive pill and the dramatic amelioration with pregnancy are well documented. In vivo and in vitro studies have demonstrated that sex hormones interfere with a number of the putative processes involved in the pathogenesis of RA, including immunoregulation, interaction with inflammatory mediators and the cytokine system, and direct effects on cartilage itself. All these observations point towards the importance of gonadal hormones. However, trials on the potential therapeutic use of sex hormones in RA are limited and, as yet, disappointing. Further work is necessary to determine whether the roles of sex hormones are as central protagonists or just supporting cast in the complex arena of rheumatoid disease.'},\n",
       "  {'qas': [{'id': '5118dd1305c10fae75000001_012',\n",
       "     'question': 'Is Rheumatoid Arthritis more common in men or women?',\n",
       "     'answers': [{'text': 'women', 'answer_start': 851}]}],\n",
       "   'context': 'Early rheumatoid arthritis: does gender influence disease expression? OBJECTIVE: To investigate whether gender is an independent factor associated with disease expression in early rheumatoid arthritis (RA) patients. METHODS: 438 patients with early RA (disease duration less than one year) were studied. They all were patients with early RA who presented at the Rheumatology Clinic of the University Hospital of Ioannina during the period 1991-2000. All patients fulfilled the American College of Rheumatology criteria for RA. The demographic, clinical, laboratory, radiological and therapeutic characteristics of the disease at diagnosis, and at the last follow-up were analyzed according to gender. RESULTS: We studied 312 women and 126 men with early RA. The female to male ratio was 2.5:1 and the mean age at diagnosis was 49.4 +/- 14.9 years for women and 55.3 +/-15.6 years for men (P < 0.0003). Women had a longer duration of follow-up (P < 0.0003). There were no differences between genders in the general symptoms or the simmetricity of joint involvement at at disease onset. However at disease onset women had a higher erythrocyte sedimentation rate (ESR) (> 30 mm/1st hour), although there were no significant differences between the two groups concerninig the rest of the clinical, laboratory and radiological findings. At the last follow-up women still had a higher ESR (>30 min/1st hour), but no significant differences were found between the two groups concerning the rest of the parameters investigated independently of the follow-up duration. Finally, women and men showed the same degree of radiological changes and functional ability and were treated similarly except for the more frequent use of hydroxychloroquine in women. CONCLUSION: It seems that gender does not signficantly influence the expression of RA.'},\n",
       "  {'qas': [{'id': '534ebb59288f4dae47000004_001',\n",
       "     'question': 'Which methyl-CpG-binding protein when mutant becomes the hallmark for Rett syndrome?',\n",
       "     'answers': [{'text': 'methyl-CpG-binding protein 2 (MeCP2)',\n",
       "       'answer_start': 230}]}],\n",
       "   'context': 'A TrkB small molecule partial agonist rescues TrkB phosphorylation deficits and improves respiratory function in a mouse model of Rett syndrome. Rett syndrome (RTT) results from loss-of-function mutations in the gene encoding the methyl-CpG-binding protein 2 (MeCP2) and is characterized by abnormal motor, respiratory and autonomic control, cognitive impairment, autistic-like behaviors and increased risk of seizures. RTT patients and Mecp2-null mice exhibit reduced expression of brain-derived neurotrophic factor (BDNF), which has been linked in mice to increased respiratory frequency, a hallmark of RTT. The present study was undertaken to test the hypotheses that BDNF deficits in Mecp2 mutants are associated with reduced activation of the BDNF receptor, TrkB, and that pharmacologic activation of TrkB would improve respiratory function. We characterized BDNF protein expression, TrkB activation and respiration in heterozygous female Mecp2 mutant mice (Het), a model that recapitulates the somatic mosaicism for mutant MECP2 found in typical RTT patients, and evaluated the ability of a small molecule TrkB agonist, LM22A-4, to ameliorate biochemical and functional abnormalities in these animals. We found that Het mice exhibit (1) reduced BDNF expression and TrkB activation in the medulla and pons and (2) breathing dysfunction, characterized by increased frequency due to periods of tachypnea, and increased apneas, as in RTT patients. Treatment of Het mice with LM22A-4 for 4 weeks rescued wild-type levels of TrkB phosphorylation in the medulla and pons and restored wild-type breathing frequency. These data provide new insight into the role of BDNF signaling deficits in the pathophysiology of RTT and highlight TrkB as a possible therapeutic target in this disease.'},\n",
       "  {'qas': [{'id': '534ebb59288f4dae47000004_002',\n",
       "     'question': 'Which methyl-CpG-binding protein when mutant becomes the hallmark for Rett syndrome?',\n",
       "     'answers': [{'text': 'methyl-CpG-binding protein 2 (Mecp2)',\n",
       "       'answer_start': 316}]}],\n",
       "   'context': 'Breathing dysfunction in Rett syndrome: understanding epigenetic regulation of the respiratory network. Severely arrhythmic breathing is a hallmark of Rett syndrome (RTT) and profoundly affects quality of life for patients and their families. The last decade has seen the identification of the disease-causing gene, methyl-CpG-binding protein 2 (Mecp2) and the development of mouse models that phenocopy many aspects of the human syndrome, including breathing dysfunction. Recent studies have begun to characterize the breathing phenotype of Mecp2 mutant mice and to define underlying electrophysiological and neurochemical deficits. The picture that is emerging is one of defects in synaptic transmission throughout the brainstem respiratory network associated with abnormal expression in several neurochemical signaling systems, including brain-derived neurotrophic factor (BDNF), biogenic amines and gamma-amino-butyric acid (GABA). Based on such findings, potential therapeutic strategies aimed at improving breathing by targeting deficits in neurochemical signaling are being explored. This review details our current understanding of respiratory dysfunction and underlying mechanisms in RTT with a particular focus on insights gained from mouse models.'},\n",
       "  {'qas': [{'id': '534ebb59288f4dae47000004_003',\n",
       "     'question': 'Which methyl-CpG-binding protein when mutant becomes the hallmark for Rett syndrome?',\n",
       "     'answers': [{'text': 'methyl-CpG-binding protein 2 (MECP2)',\n",
       "       'answer_start': 484}]}],\n",
       "   'context': 'Rett syndrome: methyl-CpG-binding protein 2 mutations and phenotype-genotype correlations. Rett syndrome (RTT) is an X-linked dominant neurodevelopmental disorder that manifests in females, typically after the first year of life. It is a leading cause of mental retardation and autistic behavior in girls and women; a hallmark of the disease is incessant hand movements in the form of wringing, twisting, or clapping. It was recently discovered that RTT is caused by mutations in the methyl-CpG-binding protein 2 (MECP2) gene. MECP2 assists in the transcriptional silencing process via DNA methylation; we hypothesize that disruption of this gene alters the normal developmental expression of various other genes, some of which must account for the peculiar neurologic phenotype of RTT. Molecular studies have identified MECP2 mutations in up to 80% of classic RTT patients; mutation type has some effect on the phenotypic manifestation of RTT, but the pattern of X inactivation seems to determine phenotypic severity. Favorable (skewed) X inactivation can so spare a patient from the effects of mutant MECP2 that they display only the mildest learning disability or no phenotype at all. The unmitigated impact of mutant MECP2 can be inferred from the few males who have been born into RTT kindreds with such severe neonatal encephalopathy that they did not survive their second year. MECP2 mutations thus manifest in a far broader array of phenotypes than classic RTT. This discovery should prove helpful in diagnosing cases of mild learning disability or severe neonatal encephalopathies of unknown cause and also should provide insight into the pathogenesis of RTT.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_001',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 236}]}],\n",
       "   'context': 'Identification of recombinant alleles using quantitative real-time PCR implications for Gaucher disease. Pseudogenes, resulting from duplications of functional genes, contribute to the functional complexity of their parental genes. The glucocerebrosidase gene (GBA), located in a gene-rich region on chromosome 1q 21, is mutated in Gaucher disease. The presence of contiguous, highly homologous pseudogenes for both GBA and metaxin 1 at this locus increases the likelihood of DNA rearrangement. We describe a facile method to identify and analyze recombinant alleles in patients with Gaucher disease. Genomic DNA from 20 patients with recombinant GBA alleles and five controls was evaluated to identify DNA rearrangements or copy number variation using six probes specific for either the GBA gene or pseudogene. Quantitative real-time PCR was performed on genomic DNA, and Southern blot analyses using HincII together with sequencing confirmed the real-time results. Both GBA fusions and duplications could be detected. Different sites of crossover were identified, and alleles resulting from gene conversion could be distinguished from reciprocal recombinant alleles. Quantitative real-time PCR is a sensitive and rapid method to detect fusions and duplications in patients with recombinant GBA alleles. This technique is more sensitive, faster, and cheaper than Southern blot analysis, and can be used in diagnostic laboratories, and to detect other recombinant alleles within the genome.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_002',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 240}]}],\n",
       "   'context': 'Expression of human Gaucher disease gene GBA generates neurodevelopmental defects and ER stress in Drosophila eye. Gaucher disease (GD) is the most common of the lysosomal storage disorders and is caused by defects in the GBA gene encoding glucocerebrosidase (GlcCerase). The accumulation of its substrate, glucocylceramide (GlcCer) is considered the main cause of GD. We found here that the expression of human mutated GlcCerase gene (hGBA) that is associated with neuronopathy in GD patients causes neurodevelopmental defects in Drosophila eyes. The data indicate that endoplasmic reticulum (ER) stress was elevated in Drosophila eye carrying mutated hGBAs by using of the ER stress markers dXBP1 and dBiP. We also found that Ambroxol, a potential pharmacological chaperone for mutated hGBAs, can alleviate the neuronopathic phenotype through reducing ER stress. We demonstrate a novel mechanism of neurodevelopmental defects mediated by ER stress through expression of mutants of human GBA gene in the eye of Drosophila.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_003',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 138}]}],\n",
       "   'context': \"[Genetics of Gaucher's disease. Genotype-phenotype correlation]. Gaucher's disease (GD) results from a deficiency of the lysosomal enzyme glucocerebrosidase and, in very rare occasions, a deficiency of its activator, the saposin C. The complexity of identification and characterization of mutations in the gene of glucocerebrosidase (GBA1) is caused by a great amount of mutated alleles, the existence of a highly homologous pseudogene and its location in a very rich zone in genes, which promotes the presence of complex alleles. Although genotype-phenotype correlations in EG are not completely established, there are a series of generalities, as the mutation c.1226A>G (N370S) is often associated with a certain degree of neuroprotection and the homozygosity for the c.1448T>C (L444P) mutation presents with neurological symptoms.\"},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_004',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 127}]}],\n",
       "   'context': 'Global gene expression profile progression in Gaucher disease mouse models. BACKGROUND: Gaucher disease is caused by defective glucocerebrosidase activity and the consequent accumulation of glucosylceramide. The pathogenic pathways resulting from lipid laden macrophages (Gaucher cells) in visceral organs and their abnormal functions are obscure. RESULTS: To elucidate this pathogenic pathway, developmental global gene expression analyses were conducted in distinct Gba1 point-mutated mice (V394L/V394L and D409 V/null). About 0.9 to 3% of genes had altered expression patterns ( > ± 1.8 fold change), representing several categories, but particularly macrophage activation and immune response genes. Time course analyses (12 to 28 wk) of INFγ-regulated pro-inflammatory (13) and IL-4-regulated anti-inflammatory (11) cytokine/mediator networks showed tissue differential profiles in the lung and liver of the Gba1 mutant mice, implying that the lipid-storage macrophages were not functionally inert. The time course alterations of the INFγ and IL-4 pathways were similar, but varied in degree in these tissues and with the Gba1 mutation. CONCLUSIONS: Biochemical and pathological analyses demonstrated direct relationships between the degree of tissue glucosylceramides and the gene expression profile alterations. These analyses implicate IFNγ-regulated pro-inflammatory and IL-4-regulated anti-inflammatory networks in differential disease progression with implications for understanding the Gaucher disease course and pathophysiology.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_005',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 136}]}],\n",
       "   'context': 'In silico and functional studies of the regulation of the glucocerebrosidase gene. In Gaucher disease (GD), the inherited deficiency of glucocerebrosidase results in the accumulation of glucocerebroside within lysosomes. Although almost 300 mutations in the glucocerebrosidase gene (GBA) have been identified, the ability to predict phenotype from genotype is quite limited. In this study, we sought to examine potential GBA transcriptional regulatory elements for variants that contribute to phenotypic diversity. Specifically, we generated the genomic sequence for the orthologous genomic region ( approximately 39.4kb) encompassing GBA in eight non-human mammals. Computational comparisons of the resulting sequences, using human sequence as the reference, allowed the identification of multi-species conserved sequences (MCSs). Further analyses predicted the presence of two putative clusters of transcriptional regulatory elements upstream and downstream of GBA, containing five and three transcription factor-binding sites (TFBSs), respectively. A firefly luciferase (Fluc) reporter construct containing sequence flanking the GBA gene was used to test the functional consequences of altering these conserved sequences. The predicted TFBSs were individually altered by targeted mutagenesis, resulting in enhanced Fluc expression for one site and decreased expression for seven others sites. Gel-shift assays confirmed the loss of nuclear-protein binding for several of the mutated constructs. These identified conserved non-coding sequences flanking GBA could play a role in the transcriptional regulation of the gene contributing to the complexity underlying the phenotypic diversity seen in GD.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_006',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 110}]}],\n",
       "   'context': 'Mutation analysis and genotype/phenotype relationships of Gaucher disease patients in Spain. Mutations in the glucocerebrosidase (GBA) gene cause Gaucher disease (GD). The aim of this study was to characterise the GBA mutations and analyze genotype/phenotype relationships in 193 unrelated patients from the Spanish GD Registry. We have identified 98.7% of the mutated GBA alleles, finding 56 different GBA mutations and 66 genotypes causing GD in Spain: 47 previously described mutations and 9 novel mutations (4 missense R395C, R463H, W312R and V398I, 1 nonsense R359X, 4 frameshift c.708delC, c.1214-1215delGC, c.1439-1445del7 and c.42-65del24). The most prevalent mutations were N370S and L444P, accounting for 68.7% of the mutated alleles. A wide phenotypic difference was observed within each genotypic group, and 9% of diagnosed type 1 patients developed neurological involvement including parkisonism, tremor, hypoacusia and eye movements. All of these findings indicate that there is a significant genotypic heterogeneity that explains the huge phenotypic variation among Spanish GD patients.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_007',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 702}]}],\n",
       "   'context': 'Miglustat (NB-DNJ) works as a chaperone for mutated acid beta-glucosidase in cells transfected with several Gaucher disease mutations. Gaucher disease (GD) is a disorder of glycosphinglipid metabolism caused by deficiency of lysosomal acid beta-glucosidase (GC), resulting in progressive deposition of glucosylceramide in macrophages. The glucose analogue, N-butyl-deoxynojirimycin (NB-DNJ, Miglustat), is an inhibitor of the ceramide-specific glucosyltransferase (CSG) which catalyzes the first step of glycosphingolipids biosynthesis and is currently approved for the oral treatment of type 1 GD. Using site-directed mutagenesis, we constructed plasmids containing wild-type and several mutations in glucocerebrosidase (GBA) gene. The plasmids were transfected into COS-7 cells and stable transfected cell lines were obtained by geneticin (G418) selection. Cells were cultured during 6 days with medium with or without 10 microM NB-DNJ. The addition of NB-DNJ to COS-7 cell medium leads to 1.3-, 2.1-, 2.3-, 3.6-, and 9.9-fold increase in the activity of S364R, wild-type, N370S, V15M, and M123T GC, respectively. However, no significant changes were observed in the activity of the L444P, L336P, and S465del mutated proteins, but a small decrease in the rare P266L variant was observed. These results suggest that NB-DNJ, in addition to the inhibitory effect on CSG, also works as a \"chemical chaperone\", increasing the activity of acid beta-glucosidase of wild-type and several GC mutated proteins, including the most frequent N370S mutation. The specific location of the Miglustat binding site in GC is unknown. Potential binding sites in the enzyme have been searched for using computational molecular docking. The searching strategy identified three potential GC binding sites for Miglustat, one being the substrate-binding site of the enzyme, which was the best-ranked site by AutoDock program. Therefore, it is possible that Miglustat exerts its chaperoning activity on acid beta-glucosidase by acting as an inhibitor bound at the active site. This increase on the activity of the acid beta-glucosidase would imply that Miglustat is not only a substrate reducer but also an inhibitor of the GC degradation, with very promising clinical implications for the treatment of GD patients.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_008',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 241}]}],\n",
       "   'context': 'Expression and functional characterization of mutated glucocerebrosidase alleles causing Gaucher disease in Spanish patients. BACKGROUND: Gaucher disease (GD) is a heterogeneous disease characterized by an impaired activity of the lysosomal glucocerebrosidase. This heterogeneity is attributed in part to the existence of a large number of mutations in the corresponding gene. SUBJECTS AND METHODS: To establish genotype-phenotype relationships, we analyzed the residual enzyme activities of six naturally occurring mutations found in Spanish population in the glucocerebrosidase gene [c.160G > A (V15M), c.485T>C (M123T), c.914C>T (P266L), c.1124T>C (L336P), c.1207A>C (S364R) and c.1510-1512delTCT (S465del)]. The mutated genes were subcloned into the mammalian expression vector pCR 3.1 and expressed by transient transfection in COS cells. The enzymatic activity of the expressed protein were measured and compared with the wild-type human glucocerebrosidase cDNA. Also, two previously alleles, c.1226A>G (N370S) and c.1448T>C (L444P), were used for comparative purposes. RESULTS: The residual activity of the expressed proteins using the synthetic substrate (4-methylumbelliferyl-beta-D-glucopyranoside, 4MU-Glu) ranged from 5.5% (for the 3-bp deletion) to 42.7% (for S364R mutation) of the activity of the wild-type enzyme. CONCLUSION: The present analyses may help to better understand the molecular basis and the pathogenesis of Gaucher disease. However, results of expression of mutated enzymes are necessary but not sufficient to explain the ultimate clinical outcome of Gaucher disease.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_009',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'Glucocerebrosidase', 'answer_start': 182}]}],\n",
       "   'context': \"Co-morbidity in Gaucher's disease results of a nationwide enquiry in Spain. SHORT INTRODUCTION: Gaucher's disease (GD) is an autosomal recessive disease produced by mutations of the Glucocerebrosidase gene. Carriers are considered to be healthy subjects because there is no manifestation of the disease, but they show signs of macrophage disfunction. The aim of the study was to determine if GD patients and non affected carriers risk suffering other diseases when compared to healthy non-carrier relatives. DESIGN: Epidemiologic study of historic cohorts. The fact that they have one or two mutated alleles has been considered to be the risk factor leading to other conditions (Dementia, Parkinson disease, Ischemic stroke, Ischemic heart disease, Non rheumatic valvular disease, Cancer hematological and non-hematological, Pulmonary fibrosis, Tuberculosis, Gallstones and Schizophrenia). All people, patients, carriers and healthy controls shared the same genetical background and environmental influence. - Patients and relatives enrolled on the Spanish Gaucher Disease Registry were evaluated. STATISTICS: For the Relative-Risk calculation the Mantel-Haenszel test was applied. Yates' correction was used when size sample was too small. A value of p <0.05 was accepted for statistical significance. RESULTS: 370 people, from 79 different families, were surveyed. We received evaluable information from 45 families (56%), totalling 258 people (69%): 59 healthy subjects (Mean age 32. 20, RANGE: 10-85; M 57.63%/F 42.37%), 132 carriers (Mean age 35.91, RANGE: 1-79; M 56.82%/F 43.18%) and 67 patients (Mean age 32.16, Range: 1-76; M 44.78%/F 55.22%. - Relative Risk of suffering any disease with regard to Gaucher's status: Patient vs Healthy 9.69 (95% Confidence interval [CI] 2.00-63.99; p 0.0006). Patient vs Carrier 3.74 (CI 1.53-9.27; p 0.001); Carrier vs Healthy 2.59 (CI 0. 52-12.50; p 0.21). Relative Risk of suffering any disease with regard to sex was 3.96 for female patients (CI 1.01-16.75; p 0.02) and 1.34 for female carriers (CI 0.27-6.75; p = 0.68). CONCLUSION: As a group, Gaucher's patients seem to have a greater risk of suffering other common unrelated diseases than carriers or healthy relatives. This excess of risk is particularly higher among female patients and can not be explained in terms of differences in age. Carrier status doesn't seem to highten the risk of suffering other diseases.\"},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_010',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 213}]}],\n",
       "   'context': 'Recurrence of the D409H mutation in Spanish Gaucher disease patients: description of a new homozygous patient and haplotype analysis. Gaucher disease results, in most patients, from mutations in the gene encoding glucocerebrosidase. Mutation D409H is the third most frequent in Spanish patients, accounting for 5.7% of all mutated alleles. This allele is associated mainly with the neurological forms of the disease. Recently, homozygosity for the D409H mutation has been associated with a particular phenotype, including specific cardiovascular symptoms. Here we report a second Spanish patient bearing the D409H/D409H genotype with a very early manifestation of the disease. The patient started enzyme replacement therapy at 3 months of age. A common origin for the Spanish D409H alleles was ruled out by haplotype analysis using an internal polymorphism of the glucocerebrosidase gene and two external microsatellite markers.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_011',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 397}]}],\n",
       "   'context': 'Type I Gaucher disease due to homozygosity for the 259T mutation in a Bedouin patient. A 26-year-old Bedouin with moderate thrombocytopenia and enlarged spleen and liver was diagnosed as having type I Gaucher disease based on the presence of Gaucher cells in the bone marrow biopsy and enzymatic determination of glucocerebrosidase activity. Molecular analysis excluded 10 common mutations in the glucocerebrosidase gene. Homozygosity for the C --> T mutation in nucleotide 259 of the cDNA (1763 genomic) was detected by digestion with restriction enzyme StyI after an amplification of a portion of exon 3 by mismatched primers. This is the first known case of homozygosity for this mutation. The fact that it produces a very mild phenotype, confirms a previous suggestion that 259T can be classified as a \"mild\" mutation. Association of the 259T mutation with the \"Pv 1.1 +\" haplotype is consistent with a common origin of the mutated alleles.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_012',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 425}]}],\n",
       "   'context': \"Molecular analysis of Gaucher disease: distribution of eight mutations and the complete gene deletion in 27 patients from Germany. Gaucher disease is the most common lysosomal storage disease with a high prevalence in the Ashkenazi Jewish population but it is also present in other populations. The presence of eight mutations (1226G, 1448C, IVS2+1. 84GG, 1504T, 1604T, 1342C and 1297T) and the complete deletion of the beta-glucocerebrosidase gene was investigated in 25 unrelated non-Jewish patients with Gaucher's disease in Germany. In the Jewish population, three of these mutations account for more than 90% of all mutated alleles. In addition, relatives of two patients were included in our study. Restriction fragment length polymorphism analysis and sequencing of PCR products obtained from DNA of peripheral blood leukocytes was performed for mutation analysis. Gene deletion was detected by comparison of radioactively labelled PCR fragments of both the functional beta-glucocerebrosidase gene and the pseudogene. Among the unrelated patients, 50 alleles were investigated and the mutations identified in 35 alleles (70%), whereas 15 alleles (30%) remained unidentified. The most prevalent mutation in our group of patients was the 1226G (370Asn-->Ser) mutation, accounting for 18 alleles (36%), followed by the 1448C (444Lcu-->Pro) mutation, that was found in 12 alleles (24%). A complete gene deletion was present in two alleles (4%). The IVS1+2 (splicing mutation), the 1504T (463Arg-->Cys) as well as the 1342C (409Asp-->His) mutations were each present in one allele (2%). None of the alleles carried the 84GG (frame-shift), 1604A (496Arg-->His) or the 1297T (394Val-->Leu) mutation. This distribution is different from the Ashkenazi Jewish population but is similar to other Caucasian groups like the Spanish and Portuguese populations. Our results confirm the variability of mutation patterns in Gaucher patients of different ethnic origin. All patients were divided into nine groups according to their genotype and their clinical status was related to the individual genotype. Genotype/phenotype characteristics of the 1226G, 1448C, and 1342C mutations of previous studies were confirmed by our results.\"},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_013',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 167}]}],\n",
       "   'context': 'Expression of mutated glucocerebrosidase alleles in human cells. Gaucher disease is a heterogeneous disease characterized by impaired activity of the lysosomal enzyme glucocerebrosidase. This heterogeneity is attributed to a large number of mutations in the corresponding gene. In order to test the biochemical properties of some mutations prevalent among Israeli populations, the normal human glucocerebrosidase cDNA and cDNAs carrying mutations N370S, L444P, D409H, recTL, recNcil, P415R and 84GG were coupled to the T7 RNA polymerase promoter in a vaccinia virus-derived expression vector (pTM-1). Recombinant viruses were produced and used to infect human tissue culture cells. RNA and protein stability, recognition by anti-glucocerebrosidase monoclonal antibodies and intracellular enzymatic activity were measured. The results demonstrated that the D409H allele directed synthesis of cytoplasmic RNA with decreased stability compared with its normal counterpart or other mutated forms. The D409H and L444P mutated proteins had lower stability than that of their normal counterpart, while the recNcil-mutated protein was more stable. Only glucocerebrosidase forms harboring leucine at position 444 were recognized by the anti-glucocerebrosidase monoclonal antibodies used (8E4 and 2C7). Measurements of enzymatic activity of the recombinant proteins in cells loaded with a fluorescent glucosylceramide demonstrated that the N370S mutated enzyme had activity similar to that of the normal enzyme. The other mutated enzymes exhibited varying degrees of activities, generally corresponding to the phenotypes with which they are associated. The results presented demonstrate the use of the vaccinia virus-derived expression system and of loading living cells with fluorescent substrate as efficient tools for studying mutants in Gaucher disease and in other lysosomal diseases.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_014',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 137}]}],\n",
       "   'context': 'The glucocerebrosidase D409H mutation in Gaucher disease. Gaucher disease, resulting from the decreased activity of the lysosomal enzyme glucocerebrosidase, is the most prevalent sphingolipid storage disease. Due to considerable heterogeneity of phenotypic expression, it has been subdivided into the nonneurological type 1 disease, and types 2 and 3, the neurological types. We describe homozygosity for the D409H mutation within the glucocerebrosidase gene associated with a unique form of type 3 Gaucher disease. Twelve patients, originating from three Arab sibships, were found to be homozygous for the D409H mutation. They all presented with oculomotor apraxia and a progressive cardiac valve defect with minimal organomegaly. When expressed in human cells in tissue culture, using the T7/EMC/vaccinia virus hybrid expression system, we were able to demonstrate that the mRNA carrying the D409H mutation was less stable than the normal counterpart. Pulse-chase experiments demonstrated that the mutated protein exhibited lower stability than the normal counterpart. Its activity toward the artificial substrate 4-methyl umbelliferyl glucopyranoside was similar to that of the mutated enzymes carrying the N370S or the L444P mutations. However, in loading experiments using lissamine-rhodamine conjugated glucosyl ceramide as a substrate, the recombinant mutated protein carrying the D409H mutation exhibited 28.63 +/- 6.05% of the activity exhibited by the normal enzyme. L444P and N370S mutations exhibited 51.90 +/- 7.16 and 115.75 +/- 12.64% of normal enzyme activity, respectively. Loading of cells homozygous for the D409H mutation demonstrated 10. 05% of the activity shown by normal cells. L444P and N370S homozygous cells demonstrated 25.3 and 98.5% of foreskin fibroblast glucocerebrosidase activity, respectively. We demonstrate that homozygosity for the D409H mutation is a unique case of a peculiar phenotype associated with a specific intracellular glucocerebrosidase activity.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_015',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 164}]}],\n",
       "   'context': 'A rare G6490-->A substitution at the last nucleotide of exon 10 of the glucocerebrosidase gene in two unrelated Italian Gaucher patients. Mutation screening of the glucocerebrosidase gene by SSCP analysis revealed an abnormal pattern of exon 10 in two unrelated Italian Gaucher patients. Direct sequencing of the mutated samples identified a G6490-->A transition. The same mutation has been described before in a Japanese patient with Gaucher disease type III. The clinical phenotype of our patients was type I in one whose second allele carried the N370S mutation and type II in the other one with a L444P mutation. In this latter the G6490-->A substitution cancels a normal Msp I site, while on the opposite chromosome the T6433-->C mutation (L444P) introduces a new Msp I site. Thus, digestion with Msp I of the amplified exon 10 is a useful method for identifying the two mutations simultaneously.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_016',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 219}]}],\n",
       "   'context': 'Homozygous presence of the crossover (fusion gene) mutation identified in a type II Gaucher disease fetus: is this analogous to the Gaucher knock-out mouse model? Gaucher disease (GD) is an inherited deficiency of beta-glucocerebrosidase (EC 3.1.2.45, gene symbol GBA). In type I GD, the CNS is not involved (nonneuronopathic), whereas in type II GD (acute neuronopathic) CNS involvement is early and rapidly progressive, while in type III GD (subacute neuronopathic) CNS involvement occurs later and is slowly progressive. The T6433C (L444P) substitution is prevalent in type GD II. It may occur alone as a single base-pair mutation but often is found as part of a complex allele containing additional GBA nucleotide substitutions, G6468C (A456P) and G6482C (V460V), without (recNciI) or with (recTL) G5957C (D409H). This complex allele is presumed to have formed by recombination (crossover, fusion) of the structural gene with the pseudogene, which contains the mutated sequences. Two complex alleles have never been demonstrated to coexist in any individual. We devised a selective PCR method for the specific amplification of the normal and/or fusion gene. Using this procedure we demonstrated the fusion gene in homozygous form for the first time, in a Macedonian/Ashkenazi Jewish GD type II fetus. Both parents were carriers of the recombination. This was confirmed by direct sequence analysis. A previous conceptus in this family was stillborn at 36 weeks, with features of severe type II GD. Neonates showing a severe clinical phenotype, analogous to the early neonatal lethal disease occurring in mice homozygous for a null allele produced by targeted disruption of GBA, have been described elsewhere, but the specific mutations in these cases have not yet been characterized.(ABSTRACT TRUNCATED AT 250 WORDS)'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_017',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 179}]}],\n",
       "   'context': 'Gaucher disease: N370S glucocerebrosidase gene frequency in the Portuguese population. In the Portuguese population the most frequent form of Gaucher disease is type 1. The N370S glucocerebrosidase gene mutation accounts for 63% of mutated alleles. The frequency of this mutation was accurately determined in the Portuguese population, which does not present an Ashkenazi Jewish genetic background. A gene frequency of 0.0043, with 95% confidence limits between 0.0023 and 0.0063, was obtained studying the genomic DNA of 2000 blood cards randomly sampled from the national neonatal screening program. On the basis of this frequency a significantly high number of homozygotes for the N370S mutation should be expected in the Portuguese population. This finding supports the idea that the majority of homozygotes for this mutation present a very mild clinical phenotype and remain undiagnosed.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_018',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 175}]}],\n",
       "   'context': 'The N370S mutation in the glucocerebrosidase gene of Portuguese type 1 Gaucher patients: linkage to the PvuII polymorphism. The mutation N370S accounts for 63% of the mutated glucocerebrosidase alleles of Portuguese type 1 Gaucher patients. It has been shown previously that this mutation is linked to the Pv1.1- form of the PvuII polymorphism and suggested that the N370S mutation in glucocerebrosidase alleles has an Ashkenazi Jewish origin. We have found that in Portuguese type 1 Gaucher patients this mutation is also invariably associated with the Pv1.1- haplotype, despite the fact that there is no evidence of Ashkenazi Jewish background in this population.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_019',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 153}]}],\n",
       "   'context': 'Prevalence of nine mutations among Jewish and non-Jewish Gaucher disease patients. The frequency of nine different mutated alleles known to occur in the glucocerebrosidase gene was determined in 247 Gaucher patients, of whom 176 were of Jewish extraction, 2 were Jewish with one converted parent, and 69 were of non-Jewish origin. DNA was prepared from peripheral blood, active glucocerebrosidase sequences were amplified by using the PCR technique, and the mutations were identified by using the allele-specific oligonucleotide hybridization method. The N37OS mutation appeared in 69.77% of the mutated alleles in Jewish patients and in 22.86% of the mutated alleles in non-Jews. The 84GG mutation, which has not been found so far among non-Jewish patients, existed in 10.17% of the disease alleles among Jewish patients. The IVS + 1 mutation constituted 2.26% of the disease alleles among Jewish patients and 1.43% among the non-Jewish patients. RecTL, a complex allele containing four single-base-pair changes, occurred in 2.26% of the alleles in Jewish patients and was found in two (1.43%) of the patients of non-Jewish extraction. Another complex allele, designated \"RecNciI\" and containing three single-point mutations, appeared in 7.8% of alleles of non-Jewish patients and in only two (0.56%) of the Jewish families. The prevalence of the L444P mutation among non-Jewish Gaucher patients was 31.43%, while its prevalence among Jewish patients was only 4.24%. The prevalence of two other point mutations--D409H and R463C--was 5.00% and 3.57%, respectively, among non-Jewish patients and was not found among the Jewish Gaucher patient population. The prevalence of the R496H mutation, found so far only among Jewish patients, was 1.13%. The results presented demonstrate that seven mutations identify 90.40% of the mutations among Jewish patients and that these seven mutations allow diagnosis of only 73.52% of the non-Jewish patients. Identification of additional mutant alleles will enhance the accuracy of carrier detection.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_020',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 272}]}],\n",
       "   'context': 'Mutation analysis of an Ashkenazi Jewish family with Gaucher disease in three successive generations. Seven members of an Ashkenazi Jewish family with Gaucher disease in 3 successive generations were tested for the presence of the 2 common mutations known to occur in the glucocerebrosidase gene. Genomic DNA from blood or skin fibroblasts of relatives was amplified by using the PCR technique and individual mutations identified by oligonucleotides specific to the mutated sequences. Four individuals were homozygous for a mutation at amino acid 370 (370 mutation) known to occur only in type 1 disease. The other 3 affected relatives were compound heterozygotes for this mutation and for a mutation at amino acid 444 (NciI mutation) which, in the homozygous state, is associated with neurological disease. Clinical severity was more marked in the compound heterozygotes than in the homozygotes. Since the mutation is present in Ashkenazim, molecular diagnosis in families which carry the NciI mutation should prove useful in assessing their risk of the neurologic forms of Gaucher disease.'},\n",
       "  {'qas': [{'id': '532f55fed6d3ac6a34000036_021',\n",
       "     'question': 'What is the gene mutated in the Gaucher disease?',\n",
       "     'answers': [{'text': 'glucocerebrosidase', 'answer_start': 134}]}],\n",
       "   'context': 'Characterization of mutations in Gaucher patients by cDNA cloning. Mutated cDNA clones containing the entire coding sequence of human glucocerebrosidase were isolated from libraries originated from Gaucher patients. Sequence analysis of a mutated cDNA derived from a type II Gaucher patient revealed a C-to-G transversion causing a substitution of an arginine for a proline at residue 415. This change creates a new cleavage site for the enzyme HhaI in the mutated cDNA. Allele-specific oligonucleotide hybridization made it possible to show that this mutation exists in the genomic DNA of the patient. From a cDNA library originated from a type I Gaucher patient, a mutated allele was cloned that contains a T-to-C transition causing a substitution of proline for leucine at residue 444 and creating a new NciI site. This mutation is identical to that described by S. Tsuji and colleagues in genomic DNA from type I, type II, and type III patients. Since the new NciI site generates RFLP, it was used to test the existence of this mutated allele in several Gaucher patients by Southern blot analysis. This allele was found in type I (Jewish and non-Jewish), type II, and type III Gaucher patients. These findings led us to conclude that the patient suffering from type II disease (denoted GM1260) carried both mutations described above. Any one of the amino acid changes described reduces the glucocerebrosidase activity as tested by transfection of COS cells with expression vectors harboring the mutated cDNAs. The base changes in the two mutated cDNAs do not affect the electrophoretic mobility of the corresponding polypeptides on an SDS polyacrylamide gel.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_001',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 132}]}],\n",
       "   'context': 'Patient with craniosynostosis and marfanoid phenotype (Shprintzen-Goldberg syndrome) and cloverleaf skull. Marfanoid phenotype with craniosynostosis (Shprintzen-Goldberg syndrome) is a rare disorder previously described in only 5 patients. We report on the sixth known patient with this condition. The findings which distinguish our patient from others reported previously are that she was ascertained prenatally as having a cloverleaf skull; this is the first female patient described with this condition. Postnatally, she presented with arachnodactyly, camptodactyly, and clover-leaf skull. Imaging studies of the brain documented microcephaly with malformed brain, hydrocephaly, and hypoplasia of the corpus callosum. She also had choanal atresia and stenosis, a clinical finding previously reported only once, in this disorder.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_002',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 42}]}],\n",
       "   'context': 'Mutation in fibrillin-1 and the Marfanoid-craniosynostosis (Shprintzen-Goldberg) syndrome. Recent reports have described a distinct and recurrent pattern of systemic malformation that associates craniosynostosis and neurodevelopmental abnormalities with many clinical features of the Marfan syndrome (MFS), an autosomal dominant disorder of the extracellular microfibril caused by defects in the gene encoding fibrillin-1, FBN1 (ref. 8). Additional common findings include other craniofacial anomalies, hypotonia, obstructive apnea, foot deformity, and congenital weakness of the abdominal wall. So far, only 11 cases have been reported precluding the assignment of definitive diagnostic criteria. While it remains unclear whether these cases represent a discrete clinical entity with a single aetiology, they have been pragmatically grouped under the rubric Marfanoid-craniosynostosis or Shprintzen-Goldberg syndrome (SGS). Because of the significant clinical overlap between MFS and SGS, we proposed that they may be caused by allelic mutations. We now report two SGS patients who harbour mutations in FBN1. While it remains unclear whether these mutations are sufficient for the clinical expression of the entire SGS phenotype, these data suggest a role for fibrillin-1 in early craniofacial and central nervous system development. Our recent observation that FBN1 transcript is expressed as early as the 8-cell stage of human embryogenesis is consistent with this hypothesis.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_003',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 237}]}],\n",
       "   'context': 'Radiographic findings in Shprintzen-Goldberg syndrome. We report the case of a Japanese boy whose dysmorphic features were consistent with those of Shprintzen-Goldberg syndrome. The radiological features were characterized by late-onset craniosynostosis, arachnodactyly, undermodeling of short tubular bones, mildly undermodeled and slightly bowed long bones, twisted ribs and tall vertebral bodies with elongated neural arches. Apart from the craniosynostosis, these skeletal changes resembled those of frontometaphyseal dysplasia, a well-known craniotubular dysplasia. Shprintzen-Goldberg syndrome also shares many clinical features with frontometaphyseal dysplasia.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_004',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 128}]}],\n",
       "   'context': 'Shprintzen-Goldberg syndrome: a clinical analysis. Shprintzen-Goldberg syndrome is one of a group of disorders characterized by craniosynostosis and marfanoid habitus. Eleven cases were reported previously. We present 4 new patients and review one of the patients of the original report of Shprintzen and Goldberg [1982: J Craniofac Genet Dev Biol 2:65-74], 15 years later. The clinical and radiologic findings on our patients are compared with those of the previously reported patients and also with those of Furlong et al. [1987: Am J Med Genet 26:599-604] and Lacombe and Battin [1993: Clin Dysmorphol 2: 220-224], who share many of the characteristics of Shprintzen-Goldberg syndrome. Some of the clinical data are helpful in determining if the patients of Furlong et al. [1987: Am J Med Genet 26:599-604] and Lacombe and Battin [1993: Clin Dysmorphol 2: 220-224] have a separate syndrome or represent a variant of Shprintzen-Goldberg syndrome. However, radiologic investigations appear to be more specific, since an abnormality of the first and second cervical vertebrae, hydrocephalus, dilatation of the lateral ventricles, and a Chiari-I malformation of the brain were found only in the patients with Shprintzen-Goldberg syndrome. The apparently diagnostic findings of the 15 patients with this syndrome may be helpful in differentiating between Shprintzen-Goldberg syndrome and other syndromes with craniosynostosis and marfanoid habitus.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_005',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 187}]}],\n",
       "   'context': 'Shprintzen-Goldberg syndrome: case report. The Shprintzen-Goldberg syndrome is an extremely rare syndrome with a characteristic face. This is one of a group of disorders characterized by craniosynostosis and marfanoid features. The aim of this study was to present a new sporadic case of the syndrome and describe in detail the findings at the maxillofacial region.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_006',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 158}]}],\n",
       "   'context': 'Shprintzen-Goldberg syndrome: fourteen new patients and a clinical analysis. The Shprintzen-Goldberg syndrome (SGS) is a disorder of unknown cause comprising craniosynostosis, a marfanoid habitus and skeletal, neurological, cardiovascular, and connective-tissue anomalies. There are no pathognomonic signs of SGS and diagnosis depends on recognition of a characteristic combination of anomalies. Here, we describe 14 persons with SGS and compare their clinical findings with those of 23 previously reported individuals, including two families with more than one affected individual. Our analysis suggests that there is a characteristic facial appearance, with more than two thirds of all individuals having hypertelorism, down-slanting palpebral fissures, a high-arched palate, micrognathia, and apparently low-set and posteriorly rotated ears. Other commonly reported manifestations include hypotonia in at least the neonatal period, developmental delay, and inguinal or umbilical hernia. The degree of reported intellectual impairment ranges from mild to severe. The most common skeletal manifestations in SGS were arachnodactyly, pectus deformity, camptodactyly, scoliosis, and joint hypermobility. None of the skeletal signs alone is specific for SGS. Our study includes 14 mainly German individuals with SGS evaluated over a period of 10 years. Given that only 23 other persons with SGS have been reported to date worldwide, we suggest that SGS may be more common than previously assumed.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_007',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 199}]}],\n",
       "   'context': 'Shprintzen-Goldberg syndrome associated with a novel missense mutation in TGFBR2. Shprintzen-Goldberg syndrome (SGS) is a rare disorder characterized by a Marfan-like habitus, mental retardation and craniosynostosis. Cardiac abnormalities, such as aortic root dilation have also been noted as well as several skeletal abnormalities. Its nosological status is unclear as it is hard to delineate SGS from similar disorders, such as Furlong, Marfan type II, Camurati-Engelmann and Loeys-Dietz syndromes. It has been suggested that these conditions represent a phenotypical spectrum associated with aberrant TGF-beta signalling. In support of this notion, we found a novel TGFBR2 missense mutation in a patient with features of SGS.'},\n",
       "  {'qas': [{'id': '5519110f622b19434500000c_008',\n",
       "     'question': 'Which disease is included as an additional feature in the Goldberg-Shprintzen syndrome?',\n",
       "     'answers': [{'text': 'craniosynostosis', 'answer_start': 147}]}],\n",
       "   'context': 'Surgical treatment for scoliosis in patients with Shprintzen-Goldberg syndrome. BACKGROUND: Shprintzen-Goldberg syndrome (SGS) is characterized by craniosynostosis and marfanoid habitus. The clinical findings of SGS include neurological, cardiovascular, connective tissue, and skeletal abnormalities. Among these skeletal findings, developmental scoliosis is recognized in half of all patients with SGS. However, no earlier reports have described the surgical treatment of scoliosis associated with SGS. METHODS: Four patients (2 boys and 2 girls; mean age at the time of surgery, 7.3±4.4 y) with SGS who underwent surgical treatment for progressive scoliosis were reviewed. The radiologic findings, operative findings, and perioperative complications were evaluated. RESULTS: The mean preoperative Cobb angle was 102.8±16.9 degrees. The curve patterns were a double curve in 2 cases and a triple curve in 2 cases. Local kyphosis at the thoracolumbar area was recognized in all the cases with a mean kyphosis angle of 49±16 degrees. Growing rod procedures were performed in 2 patients, and posterior correction and fusion were performed in 2 patients. The mean correction rate was 45% in the patients who underwent the growing rod procedures at the time of growing rod placement and 51% in the patients who underwent posterior correction and fusion. Dislodgement of the proximal anchors occurred in 3 of the 4 patients. One patient developed pseudoarthrosis. Two patients developed deep wound infections, and implant removal was necessary in 1 patient. CONCLUSIONS: Surgical treatment for scoliosis in patients with SGS was associated with a high incidence of perioperative and postoperative complications including implant dislodgements and deep wound infections attributable to poor bone quality and a thin body habitus, which are characteristic clinical features of this syndrome. Careful preoperative surgical planning and postoperative care are critical for the surgical treatment of scoliosis associated with SGS, especially in infants requiring multiple surgeries. LEVEL OF EVIDENCE: Level IV.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_001',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'LEKTI', 'answer_start': 230}]}],\n",
       "   'context': 'Identification by in silico and in vitro screenings of small organic molecules acting as reversible inhibitors of kallikreins. Netherton syndrome is caused by loss-of-function mutations in SPINK5 encoding the Kazal-type inhibitor LEKTI-1 leading to dysregulation of proteolytic cascades involving several kallikreins. We used both structure-based and ligand-based virtual screening computations to identify commercially available non-covalent inhibitors of human kallikrein 5 (hK5), a serine protease (trypsin-like) that plays a central role in the initiation of the molecular cascades leading to the Netherton syndrome phenotype. The efficacy and mechanism of inhibition of the identified new families of organic compounds were analyzed not only for hK5 but also on other proteases implicated in the cascades (hK7, hK14 and matriptase). These inhibitors are nontoxic on healthy human keratinocytes and are structurally different from traditional serine protease inhibitors validating their potential utility as initial hits to control proteolytic disorders observed in dermatological pathologies such as Netherton syndrome.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_002',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'LEKTI', 'answer_start': 298}]}],\n",
       "   'context': 'Phase I study protocol for ex-vivo lentiviral gene therapy for the inherited skin disease, Netherton Syndrome. Netherton syndrome (NS) is a serious inherited skin disorder caused by mutations in the gene SPINK5 (serine protease inhibitor Kazal type 5) which encodes for a serine protease inhibitor LEKTI (lymphoepithelial Kazal type-related inhibitor). Patients with NS have defective keratinization, hair shaft defects, recurrent infections, atopy and a predisposition to skin malignancies. Historically, one in ten infants has died before their first birthday. Currently there are no proven treatments to cure this condition. A SIN-lentiviral vector encoding the codon optimized SPINK5 gene under the control of a 572bp element derived from the human involucrin promoter (INVO) can confer compartment specific LEKTI expression in NS keratinocytes with restoration of normal skin architecture. Here we detail a study protocol for a phase I trial for feasibility and safety evaluations of autologous epidermal sheets generated from ex-vivo gene corrected keratinocyte stem cells, which will be grafted onto patients with mutation proven NS.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_003',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'lymphoepithelial Kazal type-related inhibitor',\n",
       "       'answer_start': 305}]}],\n",
       "   'context': 'Phase I study protocol for ex-vivo lentiviral gene therapy for the inherited skin disease, Netherton Syndrome. Netherton syndrome (NS) is a serious inherited skin disorder caused by mutations in the gene SPINK5 (serine protease inhibitor Kazal type 5) which encodes for a serine protease inhibitor LEKTI (lymphoepithelial Kazal type-related inhibitor). Patients with NS have defective keratinization, hair shaft defects, recurrent infections, atopy and a predisposition to skin malignancies. Historically, one in ten infants has died before their first birthday. Currently there are no proven treatments to cure this condition. A SIN-lentiviral vector encoding the codon optimized SPINK5 gene under the control of a 572bp element derived from the human involucrin promoter (INVO) can confer compartment specific LEKTI expression in NS keratinocytes with restoration of normal skin architecture. Here we detail a study protocol for a phase I trial for feasibility and safety evaluations of autologous epidermal sheets generated from ex-vivo gene corrected keratinocyte stem cells, which will be grafted onto patients with mutation proven NS.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_004',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'LEKTI', 'answer_start': 343}]}],\n",
       "   'context': 'Netherton syndrome associated with growth hormone deficiency. Netherton syndrome (NS) is a rare autosomal recessive disorder characterized by ichthyosiform scaling, hair abnormalities, and variable atopic features. Mutations in the serine protease inhibitor Kazal type 5 (SPINK5) gene leading to lymphoepithelial Kazal-type-related inhibitor (LEKTI) deficiency cause NS. Growth retardation is a classic feature of NS, but growth hormone (GH) deficiency with subsequent response to GH therapy is not documented in the literature. It is proposed that a lack of inhibition of proteases due to a deficiency of LEKTI in the pituitary gland leads to the overprocessing of human GH in NS. Herein we report three patients with NS who had growth retardation associated with GH deficiency and responded well to GH therapy.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_005',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'LEKTI', 'answer_start': 511}]}],\n",
       "   'context': 'Netherton syndrome and its multifaceted defective protein LEKTI. Netherton syndrome (NS, OMIM 256500) is a rare autosomal recessive disorder manifesting with congenital ichthyosis, a specific hair shaft abnormality named trichorrhexis invaginata, and atopic manifestations. Because of severe complications frequently occurring in the neonatal period, NS prognosis can be poor in infancy. NS is due to loss-of-function mutations in the SPINK5 gene and to the consequent lack of expression of its encoded protein LEKTI in the skin and all stratified epithelial tissues. Following the identification of the NS causative gene and protein, specific diagnostic tools have been developed, thus breaking up the challenge of distinguishing NS from other congenital ichthyoses with overlapping features, and from severe, early-onset forms of atopic dermatitis or psoriasis. Intensive efforts to extend the knowledge into the pathomechanisms of NS have also been made. However, NS management is still problematic due to the lack of specific treatment and unmet needs. This overview summarizes the current state of the art in NS research with an emphasis on the progress made toward disease-specific innovative therapy development.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_006',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'LEKTI', 'answer_start': 365}]}],\n",
       "   'context': 'Netherton syndrome: skin inflammation and allergy by loss of protease inhibition. Netherton syndrome (NS) is a rare autosomal recessive skin disease with severe skin inflammation and scaling, a specific hair shaft defect and constant allergic manifestations. NS is caused by loss-of-function mutations in SPINK5 (serine protease inhibitor of kazal type 5) encoding LEKTI-1 (lympho-epithelial kazal type related inhibitor type 5) expressed in stratified epithelia. In vitro and in vivo studies in murine models and in NS patients have cast light on the pathogenesis of the disease and shown that LEKTI deficiency results in unopposed kallikrein-related peptidase 5 (KLK5) and KLK7 activities and to the overactivity of a new epidermal protease, elastase 2 (ELA2). Two main cascades initiated by KLK5 activity have emerged. One results in desmoglein 1 degradation and desmosome cleavage leading to stratum corneum detachment. KLK5 also activates KLK7 and ELA2, which contribute to a defective skin barrier. This facilitates allergen and microbe penetration and generates danger signals leading to caspase 1 activation and the production of active interleukin-1β. In parallel, KLK5 activates a specific cascade of allergy and inflammation by activating protease-activated receptor-2 (PAR-2) receptors. PAR-2 activation triggers the production of the major pro-Th2 cytokine TSLP (thymic stromal lymphopoietin) and several inflammatory cytokines, including tumour necrosis factor-α. Levels of thymus and activation-regulated chemokine (TARC) and macrophage-derived chemokine (MDC) also contribute to allergy in a PAR-2-independent manner. Patient investigations have confirmed these abnormalities and revealed a wide spectrum of disease expression, sometimes associated with residual LEKTI expression. These results have demonstrated that the tight regulation of epidermal protease activity is essential for skin homeostasis and identified new targets for therapeutic intervention. They also provide a link with atopic dermatitis through deregulated protease activity, as recently supported by functional studies of the E420K LEKTI variant.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_007',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'LEKTI', 'answer_start': 184}]}],\n",
       "   'context': 'Proteolytic activation cascade of the Netherton syndrome-defective protein, LEKTI, in the epidermis: implications for skin homeostasis. Lympho-epithelial Kazal-type-related inhibitor (LEKTI) is the defective protein of the ichthyosiform condition Netherton syndrome (NS). Strongly expressed in the most differentiated epidermal layers, LEKTI is a serine protease inhibitor synthesized as three different high-molecular-weight precursors, which are rapidly processed into shorter fragments and secreted extracellularly. LEKTI polypeptides interact with several proteases to regulate skin barrier homeostasis as well as inflammatory and/or immunoallergic responses. Here, by combining antibody mapping, N-terminal sequencing, and site-specific mutagenesis, we defined the amino-acid sequence of most of the LEKTI polypeptides physiologically generated in human epidermis. We also identified three processing intermediates not described so far. Hence, a proteolytic cascade model for LEKTI activation is proposed. We then pinpointed the most effective fragments against the desquamation-related kallikreins (KLKs) and we proved that LEKTI is involved in stratum corneum shedding as some of its polypeptides inhibit the KLK-mediated proteolysis of desmoglein-1. Finally, we quantified the individual LEKTI fragments in the uppermost epidermis, showing that the ratios between LEKTI polypeptides and active KLK5 are compatible with a fine-tuned inhibition. These findings are relevant both to the understanding of skin homeostasis regulation and to the design of novel therapeutic strategies for NS.'},\n",
       "  {'qas': [{'id': '54ff45966ad7dcbc12000010_008',\n",
       "     'question': 'Which protein is causing Netherton syndrome?',\n",
       "     'answers': [{'text': 'LEKTI', 'answer_start': 162}]}],\n",
       "   'context': 'Matriptase initiates activation of epidermal pro-kallikrein and disease onset in a mouse model of Netherton syndrome. Deficiency in the serine protease inhibitor LEKTI is the etiological origin of Netherton syndrome, which causes detachment of the stratum corneum and chronic inflammation. Here we show that the membrane protease matriptase initiates Netherton syndrome in a LEKTI-deficient mouse model by premature activation of a pro-kallikrein cascade. Auto-activation of pro-inflammatory pro-kallikrein-related peptidases that are associated with stratum corneum detachment was either low or undetectable, but they were efficiently activated by matriptase. Ablation of matriptase from LEKTI-deficient mice dampened inflammation, eliminated aberrant protease activity, prevented detachment of the stratum corneum, and improved the barrier function of the epidermis. These results uncover a pathogenic matriptase-pro-kallikrein pathway that could operate in several human skin and inflammatory diseases.'},\n",
       "  {'qas': [{'id': '550c3754a103b78016000007_001',\n",
       "     'question': 'Which disease of the central nervous system is characterized by the presence of Lewy bodies?',\n",
       "     'answers': [{'text': \"Parkinson's disease (PD)\", 'answer_start': 30}]}],\n",
       "   'context': \"SUMO and Parkinson's disease. Parkinson's disease (PD) is one of the most common degenerative disorders of the central nervous system that produces motor and non-motor symptoms. The majority of cases are idiopathic and characterized by the presence of Lewy bodies containing fibrillar α-synuclein. Small ubiquitin-related modifier (SUMO) immunoreactivity was observed among others in cases with PD. Key disease-associated proteins are SUMO-modified, linking this posttranslational modification to neurodegeneration. SUMOylation and SUMO-mediated mechanisms have been intensively studied in recent years, revealing nuclear and extranuclear functions for SUMO in a variety of cellular processes, including the regulation of transcriptional activity, modulation of signal transduction pathways, and response to cellular stress. This points to a role for SUMO more than just an antagonist to ubiquitin and proteasomal degradation. The identification of risk and age-at-onset gene loci was a breakthrough in PD and promoted the understanding of molecular mechanisms in the pathology. PD has been increasingly linked with mitochondrial dysfunction and impaired mitochondrial quality control. Interestingly, SUMO is involved in many of these processes and up-regulated in response to cellular stress, further emphasizing the importance of SUMOylation in physiology and disease.\"},\n",
       "  {'qas': [{'id': '550c3754a103b78016000007_002',\n",
       "     'question': 'Which disease of the central nervous system is characterized by the presence of Lewy bodies?',\n",
       "     'answers': [{'text': \"Parkinson's disease (PD)\", 'answer_start': 126}]}],\n",
       "   'context': \"LRRK2 as a Potential Genetic Modifier of Synucleinopathies: Interlacing the Two Major Genetic Factors of Parkinson's Disease. Parkinson's disease (PD) and related Lewy body diseases are characterized by deposition of α-synuclein aggregates in both the central nervous system and peripheral nervous system. Synucleinopathy lesions spread to larger brain areas as the disease progresses, and prion-like cell-to-cell transmission of aggregated α-synuclein is thought to be the underlying mechanism for this pathological spreading. LRRK2 is another protein linked to the pathogenesis of PD, and its presence in Lewy bodies has attracted much attention as to whether LRRK2 and α-synuclein interplay during the pathogenesis of PD. However, the relationship between these two crucial proteins still remains unclear. In this review article, we will discuss the current state of knowledge in terms of how these proteins cause the disease and provide the hypothetical mechanisms by which LRRK2 might modify the generation and progression of synucleinopathy.\"},\n",
       "  {'qas': [{'id': '550c3754a103b78016000007_003',\n",
       "     'question': 'Which disease of the central nervous system is characterized by the presence of Lewy bodies?',\n",
       "     'answers': [{'text': \"Parkinson's disease (PD)\", 'answer_start': 158}]}],\n",
       "   'context': \"An update on the rotenone models of Parkinson's disease: their ability to reproduce the features of clinical disease and model gene-environment interactions. Parkinson's disease (PD) is the second most common neurodegenerative disorder that is characterized by two major neuropathological hallmarks: the degeneration of dopaminergic neurons in the substantia nigra (SN) and the presence of Lewy bodies in the surviving SN neurons, as well as other regions of the central and peripheral nervous system. Animal models have been invaluable tools for investigating the underlying mechanisms of the pathogenesis of PD and testing new potential symptomatic, neuroprotective and neurorestorative therapies. However, the usefulness of these models is dependent on how precisely they replicate the features of clinical PD with some studies now employing combined gene-environment models to replicate more of the affected pathways. The rotenone model of PD has become of great interest following the seminal paper by the Greenamyre group in 2000 (Betarbet et al., 2000). This paper reported for the first time that systemic rotenone was able to reproduce the two pathological hallmarks of PD as well as certain parkinsonian motor deficits. Since 2000, many research groups have actively used the rotenone model worldwide. This paper will review rotenone models, focusing upon their ability to reproduce the two pathological hallmarks of PD, motor deficits, extranigral pathology and non-motor symptoms. We will also summarize the recent advances in neuroprotective therapies, focusing on those that investigated non-motor symptoms and review rotenone models used in combination with PD genetic models to investigate gene-environment interactions.\"},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_001',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 308}]}],\n",
       "   'context': 'Bach1-dependent and -independent regulation of heme oxygenase-1 in keratinocytes. Bach1 is a member of the basic leucine zipper transcription factor family, and the Bach1/small Maf heterodimer specifically represses transcriptional activity directed by the Maf recognition element (MARE). Because Bach1 is a repressor of the oxidative stress response, we examined the function(s) of Bach1 in keratinocytes subjected to oxidative stress. Oxidative stress induced by H(2)O(2) led to an increase in MARE activity and expression of heme oxygenase-1 (HO-1), an inducible antioxidant defense enzyme. Bach1 depletion by small interfering RNAs or by deletion of Bach1 enhanced HO-1 expression in the absence of H(2)O(2), indicating that Bach1 is a critical repressor of HO-1 in keratinocytes. Although Bach1-deficient or -reduced keratinocytes expressed higher levels of HO-1 than control cells in response to H(2)O(2), Bach1 down-regulation did not attenuate the production of reactive oxygen species by H(2)O(2). In contrast, Bach1 overexpression abolished HO-1 induction by H(2)O(2), which led to increased reactive oxygen species accumulation. HO-1 was induced during keratinocyte differentiation, but MARE activity did not change during differentiation. Furthermore, Bach1 overexpression did not inhibit differentiation-associated induction of HO-1 expression, suggesting that HO-1 induction in differentiation is independent of Bach1. Thus, in response to oxidative stress, Bach1 regulates the oxidation state through the negative control of HO-1 expression prior to terminal keratinocyte differentiation. However, Bach1-mediated repression is negated during keratinocyte differentiation.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_002',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 2277}]}],\n",
       "   'context': 'Induction of hemeoxygenase-1 expression after inhibition of hemeoxygenase activity promotes inflammation and worsens ischemic brain damage in mice. Hemeoxygenase (HO) is an enzymatic system that degrades heme. HO-1 is an inducible isoform whereas HO-2 is constitutive. Stroke strongly induces HO-1 expression but the underlying mechanisms are not fully elucidated. Cytokines that are up-regulated after ischemia, like interleukin (IL)-10, can induce HO-1 gene expression, which is positively regulated by the transcriptional activator nuclear factor erythroid 2-related factor 2 (Nrf2) and negatively regulated by the transcriptional repressor breast cancer type 1 susceptibility protein (BRCA1) associated C-terminal helicase 1 (Bach-1). While Nrf2 is activated after ischemia and drugs promoting Nrf2 activation increase HO-1 and are beneficial, the involvement of Bach-1 is unknown. Here we investigated mechanisms involved in HO-1 induction and evaluated the effects of HO activity inhibition in mouse permanent middle cerebral artery occlusion (pMCAO). HO-1 was induced after ischemia in IL-10-deficient mice suggesting that post-ischemic HO-1 induction was IL-10-independent. Attenuation of Bach-1 gene repression after ischemia was associated to enhanced HO-1 induction. Administration of the HO activity inhibitor zinc proto-porphyrin IX (ZnPP) i.p. 24h before pMCAO exacerbated ischemia-induced tumor necrosis factor-α (TNF-α) and IL-1β, nitro-oxidative stress, and the presence of neutrophils at 8h, and increased infarct volume at day 4. However, ZnPP did not worsen ischemic damage when given 30min before pMCAO. ZnPP induced HO-1 expression in the cerebral vasculature at 24h, when it was still detected by high-performance liquid chromatography (HPLC) in plasma. While ZnPP was not found in brain tissue extracts of controls, it could be detected after ischemia, supporting that a small fraction of the injected drug can reach the tissue following blood-brain barrier breakdown. The deleterious effect of inhibiting HO activity in ischemia became apparent in the presence of ZnPP-induced HO-1, which is known to exert effects independent of its enzymatic activity. In conclusion, HO-1 induction after ischemia was associated to down-regulation of transcriptional repressor Bach-1, and induction of HO-1 when HO enzymatic activity was inhibited was related to worst outcome after brain ischemia.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_003',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 566}]}],\n",
       "   'context': \"The let-7 microRNA enhances heme oxygenase-1 by suppressing Bach1 and attenuates oxidant injury in human hepatocytes. The let-7 microRNA (miRNA) plays important roles in human liver development and diseases such as hepatocellular carcinoma, liver fibrosis and hepatitis wherein oxidative stress accelerates the progression of these diseases. To date, the role of the let-7 miRNA family in modulation of heme oxygenase 1 (HMOX1), a key cytoprotective enzyme, remains unknown. Our aims were to determine whether let-7 miRNA directly regulates Bach1, a transcriptional repressor of the HMOX1 gene, and whether indirect up-regulation of HMOX1 by let-7 miRNA attenuates oxidant injury in human hepatocytes. The effects of let-7 miRNA on Bach1 and HMOX1 gene expression in Huh-7 and HepG2 cells were determined by real-time qRT-PCR, Western blot, and luciferase reporter assays. Dual luciferase reporter assays revealed that let-7b, let-7c, or miR-98 significantly decreased Bach1 3'-untranslated region (3'-UTR)-dependent luciferase activity but not mutant Bach1 3'-UTR-dependent luciferase activity, whereas mutant let-7 miRNA containing base complementarity with mutant Bach1 3'-UTR restored its effect on mutant reporter activity. let-7b, let-7c, or miR-98 down-regulated Bach1 protein levels by 50-70%, and subsequently up-regulated HMOX1 gene expression by 3-4 fold, compared with non-specific controls. Furthermore, Huh-7 cells transfected with let-7b, let-7c or miR-98 mimic showed increased resistance against oxidant injury induced by tert-butyl-hydroperoxide (tBuOOH), whereas the protection was abrogated by over-expression of Bach1. In conclusion, let-7 miRNA directly acts on the 3'-UTR of Bach1 and negatively regulates expression of this protein, and thereby up-regulates HMOX1 gene expression. Over-expression of the let-7 miRNA family members may represent a novel approach to protecting human hepatocytes from oxidant injury.\"},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_004',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 1063}]}],\n",
       "   'context': \"Novel insights into the regulation of antioxidant-response-element-mediated gene expression by electrophiles: induction of the transcriptional repressor BACH1 by Nrf2. A central mechanism in cellular defence against oxidative or electrophilic stress is mediated by transcriptional induction of genes via the ARE (antioxidant-response element), a cis-acting sequence present in the regulatory regions of genes involved in the detoxification and elimination of reactive oxidants and electrophiles. The ARE binds different bZIP (basic-region leucine zipper) transcription factors, most notably Nrf2 (nuclear factor-erythroid 2-related factor 2) that functions as a transcriptional activator via heterodimerization with small Maf proteins. Although ARE activation by Nrf2 is relatively well understood, the mechanisms by which ARE-mediated signalling is down-regulated are poorly known. Transcription factor BACH1 [BTB (broad-complex, tramtrack and bric-a-brac) and CNC (cap'n'collar protein) homology 1] binds to ARE-like sequences, functioning as a transcriptional repressor in a subset of ARE-regulated genes, thus antagonizing the activator function of Nrf2. In the present study, we have demonstrated that BACH1 itself is regulated by Nrf2 as it is induced by Nrf2 overexpression and by Nrf2-activating agents in an Nrf2-dependent manner. Furthermore, a functional ARE site was identified at +1411 from the transcription start site of transcript variant 2 of BACH1. We conclude that BACH1 is a bona fide Nrf2 target gene and that induction of BACH1 by Nrf2 may serve as a feedback-inhibitory mechanism for ARE-mediated gene regulation.\"},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_005',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 131}]}],\n",
       "   'context': 'Inhibition of Bach1 ameliorates indomethacin-induced intestinal injury in mice. BTB and CNC homolog 1 (Bach1) is a transcriptional repressor of heme oxygenase-1 (HO-1). It plays an important role in the feedback regulation of HO-1 expression, which protects cells from various insults including oxidative stress and inflammatory cytokines. However, the role of Bach1 in intestinal inflammation remains unclear. In this study, the role of Bach1 in intestinal mucosal injury was elucidated using 8-week-old female C57BL/6 (wild-type) and homozygous Bach1-deficient C57BL/6 mice. Intestinal mucosal injuries induced by a single subcutaneous administration of indomethacin were evaluated macroscopically, histologically, and biochemically. Mucosal protein content and chemokine mRNA levels were determined by real-time PCR. Our results showed that the indomethacin-induced intestinal injury was remarkably improved in Bach1-deficient mice. Histological examination showed that the area of injured lesion was decreased in Bach1-deficient mice compared to wild-type mice. Administration of indomethacin induced expression of inflammatory chemokines such as KC, MIP1alpha and MCP1, which was suppressed in Bach1-deficient mice. Myeloperoxidase activity in the intestinal mucosa was also significantly decreased in Bach1-deficient mice. Additionally, Bach1 deficiency enhanced immunopositivity of HO-1 in the intestinal mucosa after indomethacin administration. Disruption of the Bach1 gene thus caused inhibition of mucosal injury, indicating that inhibition of Bach1 may be a novel therapeutic strategy for treating indomethacin-induced intestinal injury.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_006',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 505}]}],\n",
       "   'context': 'Expression of heme oxygenase-1 in human leukemic cells and its regulation by transcriptional repressor Bach1. Heme oxygenase (HO)-1 has anti-oxidative, anti-inflammatory, and anti-apoptotic activities. However, little is known about the regulation of HO-1 in human primary acute myeloid leukemia (AML) cells. Here we investigated the expression of HO-1 in primary and established AML cells as well as other types of leukemic cells and normal monocytes, and its regulatory mechanism by the transcriptional repressor, BTB and CNC homology 1 (Bach1), and the activator, nuclear factor erythroid-derived 2 related factor 2 (Nrf2). Leukemic cell lines such as U937 expressed little HO-1, whereas most freshly isolated AML cells and monocytes expressed substantial amounts of HO-1, along with Bach1 and Nrf2. When U937 cells were treated with phorbol myristate acetate (PHA) or gamma-interferon, they significantly expressed both HO-1 and Bach1, like primary AML cells. Treatment with lipopolysaccharide (LPS) enhanced HO-1 expression in U937 cells but suppressed it in primary monocytes and PMA-treated U937 cells. In HO-1-expressing cells, Bach1 was localized in the cytoplasm, but Nrf2 was localized in the nuclei. Chromatin immunoprecipitation assay of these cells revealed the preferential binding of Nrf2 over Bach1 to Maf-recognition elements, the enhancer regions of the HO-1 gene. The downregulation of the HO-1 gene with siRNA increased a cytotoxic effect of an anticancer drug on primary AML cells, whereas the downregulation of Bach1 increased HO-1 expression, leading to enhanced survival. These and other results show that Bach1 plays a critical role in regulating HO-1 gene expression in AML cells and its expression suppresses their survival by downregulating HO-1 expression. Thus, functional upregulation of Bach1 is a potential strategy for antileukemic therapy.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_007',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 282}]}],\n",
       "   'context': \"MicroRNA-196 represses Bach1 protein and hepatitis C virus gene expression in human hepatoma cells expressing hepatitis C viral proteins. UNLABELLED: Hepatitis C virus (HCV) directly induces oxidative stress and liver injury. Bach1, a basic leucine zipper mammalian transcriptional repressor, negatively regulates heme oxygenase 1 (HMOX1), a key cytoprotective enzyme that has antioxidant and anti-inflammatory activities. microRNAs (miRNAs) are small noncoding RNAs ( approximately 22 nt) that are important regulators of gene expression. Whether and how miRNAs regulate Bach1 or HCV are largely unknown. The aims of this study were to determine whether miR-196 regulates Bach1, HMOX1, and/or HCV gene expression. HCV replicon cell lines (Con1 and 9-13) of the Con1 isolate and J6/JFH1-based HCV cell culture system were used in this study. The effects of miR-196 mimic on Bach1, HMOX1, and HCV RNA, and protein levels were measured by way of quantitative real-time polymerase chain reaction (qRT-PCR) and Western blotting, respectively. The Dual Glo Luciferase Assay System was used to determine reporter activities. miR-196 mimic significantly down-regulated Bach1 and up-regulated HMOX1 gene expression and inhibited HCV expression. Dual luciferase reporter assays demonstrated that transfection of miR-196 mimic resulted in a significant decrease in Bach1 3'-untranslated region (UTR)-dependent luciferase activity but not in mutant Bach1 3'-UTR-dependent luciferase activity. Moreover, there was no detectable effect of mutant miR-196 on Bach1 3'-UTR-dependent luciferase activity. CONCLUSION: miR-196 directly acts on the 3'-UTR of Bach1 messenger RNA and translationally represses the expression of this protein, and up-regulates HMOX1. miR-196 also inhibits HCV expression in HCV replicon cell lines (genotype 1b) and in J6/JFH1 (genotype 2a) HCV cell culture system. Thus, miR-196 plays a role in both HMOX1/Bach1 expression and the regulation of HCV expression in human hepatocytes. Overexpression of miR-196 holds promise as a potential novel strategy to prevent or ameliorate hepatitis C infection, and to protect against liver injury in chronic HCV infection.\"},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_008',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 115}]}],\n",
       "   'context': 'Genetic ablation of the Bach1 gene reduces hyperoxic lung injury in mice: role of IL-6. Bach1 is a transcriptional repressor of the heme oxygenase (HO)-1 gene. Bach1-null (Bach1(-/-)) mice are reported to be protected from myocardial ischemia/reperfusion injury; however, the effect of Bach1 disruption on another oxidative stress model of hyperoxic lung injury has yet to be determined. To investigate the role of Bach1 in hyperoxic lung injury, Bach1(-/-) mice and wild-type (WT) mice were exposed to 90% O(2). During hyperoxic exposure, the survival of Bach1(-/-) mice was significantly longer than that of WT mice. However, the administration of zinc protoporphyrin, an inhibitor of HO-1 activity, did not change the mortality in either of the mice, thus suggesting that this protective effect was not mediated by an HO-1 overexpression in Bach1(-/-) mice. The indices of lung injury in the lungs of Bach1(-/-) mice were lower than those of WT mice; unexpectedly, however, the levels of IL-6 in bronchoalveolar lavage (BAL) fluid from Bach1(-/-) mice were significantly higher than those of WT mice. Interestingly, the intrapulmonary administration of small interfering RNA against IL-6 was shown to reduce the IL-6 levels in BAL fluids and shorten the survival in Bach1(-/-) mice during hyperoxic exposure. In addition, a chromatin immunoprecipitation analysis revealed the binding of Bach1 to the IL-6 promoter and its detachment after oxidative stress. Considering the previous observation that the transgenic mice overexpressing IL-6 are protected from hyperoxic lung injury, these results therefore indicate that IL-6 mediates an increased survival in Bach1(-/-) mice during hyperoxic exposure.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_009',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 400}]}],\n",
       "   'context': 'Genetic ablation of transcription repressor Bach1 reduces neural tissue damage and improves locomotor function after spinal cord injury in mice. Heme oxygenase (HO)-1 is an inducible cytoprotective enzyme that degrades heme to iron, carbon monoxide (CO), and biliverdin, the latter two of which are thought to mediate the anti-inflammatory and antioxidant actions of HO-1. Bach1 is a transcriptional repressor of the HO-1 gene (Hmox-1). Previous reports have demonstrated that the genetic ablation of Bach1 engenders an increased HO-1 expression and a marked reduction in the degree of oxidative tissue damage in vivo. However, the function of Bach1 in spinal cord injury is still not understood. In the present study, we examined whether Bach1 deficiency increases HO-1 expression and reduces neural tissue damage in a spinal cord injury model using Bach1 knock-out (KO) mice and wild-type (WT) mice. The expression of HO-1 protein in the spinal cord was significantly higher in the Bach1 KO mice than in the WT mice before and after injury. The KO mice also had significantly higher Basso mouse scale scores for locomotor function and larger areas of spared white matter than the WT mice at 6 weeks after injury. Neuronal loss and apoptotic cell death in the injured spinal cord was significantly reduced in the KO mice in comparison to the WT mice. These results suggest that Bach1 deficiency engenders a constitutively higher expression of HO-1 and a dramatic increase in cytoprotection against spinal cord injury.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_010',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 482}]}],\n",
       "   'context': 'Regulation of heme oxygenase-1 by transcription factor Bach1 in the mouse brain. Oxidative stress has been implicated in tissue damage from traumatic brain injury. Heme oxygenase-1 (HO-1) is an inducible enzyme that degrades prooxidant heme to radical-scavenging biliverdin/bilirubin in order to protect cells from oxidative stress. Although HO-1 is induced after induction of brain damage, the regulatory mechanism of HO-1 in the brain is still unclear. Bach1 is a transcriptional repressor of the HO-1 gene, and plays a critical role in tissue protection from oxidative stress by reperfusion injury of the myocardium. In this study, we examined the role of Bach1 in HO-1 regulation of the various brain sites by investigating the expression of Bach1 and HO-1 in brain tissues of mice bearing Bach1-deficient (Bach1(-/-)) or wild-type (Bach1(+/+)) genes. While the expression levels of Bach1 mRNA in the olfactory bulb were significantly higher than other brain areas, those at the cortex showed the lowest activity. Bach1(-/-) mice showed significantly higher HO-1 mRNA expression levels than Bach1(+/+) mice in all brain sites studied. Moreover, higher induction of HO-1 was observed around damaged tissues after cold injury in Bach1(-/-) than Bach1(+/+) mice. Thus, Bach1 plays an important role in regulating the constitutive and inducible expression levels of HO-1 in the brain. Although a significantly higher level of HO-1 was observed in Bach1(-/-) than Bach1(+/+) mice, genetic ablation of the Bach1 gene failed to show any tissue protective effect after cold injury was inflicted on the cortex.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_011',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 320}]}],\n",
       "   'context': 'Heme oxygenase-1 induction by NRF2 requires inactivation of the transcriptional repressor BACH1. Oxidative stress activates the transcription factor NRF2, which in turn binds cis-acting antioxidant response element (ARE) enhancers and induces expression of protective antioxidant genes. In contrast, the transcriptional repressor BACH1 binds ARE-like enhancers in cells naïve to oxidative stress and antagonizes NRF2 binding until it becomes inactivated by pro-oxidants. Here, we describe the dynamic roles of BACH1 and NRF2 in the transcription of the heme oxygenase-1 (HMOX1) gene. HMOX1 induction, elicited by arsenite-mediated oxidative stress, follows inactivation of BACH1 and precedes activation of NRF2. BACH1 repression is dominant over NRF2-mediated HMOX1 transcription and inactivation of BACH1 is a prerequisite for HMOX1 induction. In contrast, thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1. By comparing the expression levels of HMOX1 with TXNRD1, we show that nuclear accumulation of NRF2 is not necessary for HMOX1 induction; rather, BACH1 inactivation permits NRF2 already present in the nucleus at low basal levels to bind the HMOX1 promoter and elicit HMOX1 induction. Thus, BACH1 confers an additional level of regulation to ARE-dependent genes that reveals a new dimension to the oxidative stress response.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_012',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 188}]}],\n",
       "   'context': 'Bach1, a heme-dependent transcription factor, reveals presence of multiple heme binding sites with distinct coordination structure. The mammalian transcription factor Bach1 functions as a repressor of the enhancers of heme oxygenase-1 (HO-1) gene (Hmox-1) by forming heterodimers with the small Maf proteins such as MafK. The transcription of Hmox-1 is regulated by the substrate of HO-1, heme. Heme induces expression of Hmox-1 in part by inhibiting the binding of Bach1 to the enhancers and inducing the nuclear export of Bach1. A dipeptide motif of cysteine and proline (CP motif) in Bach1 is essential for the heme-mediated regulation. In this study, we show that five molecules of heme bind to Bach1 by the heme-titration assay. The Bach1-heme complex exhibits an absorption spectrum with a major Soret peak at 371 nm and Raman band at 343 cm(-1) in high amounts of heme and a spectrum containing the major Soret peak at 423 nm at low heme concentrations. The spectroscopic characterization indicates that Bach1 has two kinds of heme-binding sites with different coordination structures. Mutagenesis studies have established that four molecules of heme bind to the cysteine residues of four CP motifs in the C terminus of Bach1. These results raise the possibility that two separated activities of Bach1, DNA-binding and nuclear export, are regulated by heme binding at the different CP motifs of Bach1 respectively, but not by cooperative heme-binding.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_013',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 814}]}],\n",
       "   'context': 'Heme: a versatile signaling molecule controlling the activities of diverse regulators ranging from transcription factors to MAP kinases. Heme (iron protoporphyrin IX) is an essential molecule for numerous living organisms. Not only does it serve as a prosthetic group in enzymes, it also acts as a signaling molecule that controls diverse molecular and cellular processes ranging from signal transduction to protein complex assembly. Deficient heme synthesis or function impacts the hematopoietic, hepatic and nervous systems in humans. Recent studies have revealed a series of heme-regulated transcription factors and signal transducers including Hap1, a heme-activated transcription factor that mediates the effects of oxygen on gene transcription in the yeast Saccharomyces cerevisiae; Bach1, a transcriptional repressor that is negatively regulated by heme in mammalian cells; IRR, an iron regulatory protein that mediates the iron-dependant regulation of heme synthesis in the bacterium Bradyrhizobium japonicum; and heme-regulated inhibitor, an eucaryotic initiation factor 2alpha kinase that coordinates protein synthesis with heme availability in reticulocytes. In this review, we summarize the current knowledge about how heme controls the activity of these transcriptional regulators and signal transducers, and discuss diseases associated with defective heme synthesis, degradation and function.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_014',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 40}]}],\n",
       "   'context': 'Redox regulation of the transcriptional repressor Bach1. Bach1 is a transcriptional repressor of heme oxygenase-1, one of the most inducible phase 2 proteins. Bach1 binds in conjunction with a small Maf protein to tandem repeats of the antioxidant response element (ARE) and quenches the target gene expression. On the other hand, the transactivator Nrf2 binds and up-regulates the ARE-governed gene expression. By using a sulfhydryl oxidizing agent, diamide, here we provide evidence which indicates that the Bach1 function is regulated by the redox state. Diamide showed restricted Nrf2 nuclear translocation and ARE-driven reporter activity but reversed the ARE transcriptional activity suppressed by ectopically expressed Bach1. Substitution of the conserved cysteine residue in the DNA binding domain of Bach1 to serine (C574S mutant) caused a refractory response to the diamide-mediated reactivation of the Bach1-suppressed reporter activity. Moreover, diamide induced cytoplasmic translocation of the GFP-Bach1 fusion protein but failed to translocate the fusion protein consisting of the C574S mutant. These data suggest that redox regulation of Bach1 is an alternative mechanism to induce multiple ARE-governed genes.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_015',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 1497}]}],\n",
       "   'context': 'Transgenic expression of BACH1 transcription factor results in megakaryocytic impairment. Both nuclear factor erythroid 2 45 kDa subunit (p45) and BTB and CNC homolog 1 (Bach) transcription factors can form dimers with one of the small Maf proteins, and these heterodimers bind to the musculoaponeurotic fibrosarcoma oncogene (Maf) recognition element (MARE). MARE is known to act as a critical cis-regulatory element of erythroid and megakaryocytic genes. Although detailed analyses of p45-null mutant mice and small maf compound mutant mice revealed that these factors are both critical for platelet production, the functional contributions of Bach1 and the relationship or redundancy between Bach1 and p45 in megakaryocytes remain to be clarified. To address these issues, we generated transgenic lines of mice bearing human BACH1 cDNA under the control of the GATA-1 locus hematopoietic regulatory domain. The transgenic mouse lines showed significant thrombocytopenia associated with impaired maturation of the megakaryocytes, and they developed myelofibrosis. The megakaryocytes in the transgenic mice exhibited reduced proplatelet formation, and the modal ploidy class of megakaryocytes was 2N, indicating the impairment of endomitosis. Transcription of the p45 target genes was down-regulated and we indeed found that BACH1 binds to the thromboxane synthase gene, one of the target genes for p45 in megakaryocytes. These findings thus provide evidence that BACH1 acts as a transcriptional repressor in the regulation of MARE-dependent genes in megakaryocytes.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_016',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 215}]}],\n",
       "   'context': 'Heme-dependent up-regulation of the alpha-globin gene expression by transcriptional repressor Bach1 in erythroid cells. The transcriptional factor Bach1 forms a heterodimer with small Maf family, and functions as a repressor of the Maf recognition element (MARE) in vivo. To investigate the involvement of Bach1 in the heme-dependent regulation of the expression of the alpha-globin gene, human erythroleukemia K562 cells were cultured with succinylacetone (SA), a heme biosynthetic inhibitor, and the level of alpha-globin mRNA was examined. A decrease of alpha-globin mRNA was observed in SA-treated cells, which was restored by the addition of hemin. The heme-dependent expression of alpha-globin occurred at the transcriptional level since the expression of human alpha-globin gene promoter-reporter gene containing hypersensitive site-40 (HS-40) was decreased when K562 cells were cultured with SA. Hemin treatment restored the decrease of the promoter activity by SA. The regulation of the HS-40 activity by heme was dependent on the NF-E2/AP-1 (NA) site, which is similar to MARE. The NA site-binding activity of Bach1 in K562 increased upon SA-treatment, and the increase was diminished by the addition of hemin. The transient expression of Bach1 and mutated Bach1 lacking CP motifs suppressed the HS-40 activity, and cancellation of the repressor activity by hemin was observed when wild-type Bach1 was expressed. The expression of NF-E2 strengthened the restoration of the Bach1-effect by hemin. Interestingly, nuclear localization of Bach1 increased when cells were treated with SA, while hemin induced the nuclear export of Bach1. These results indicated that heme plays an important role in the induction of alpha-globin gene expression through disrupting the interaction of Bach1 and the NA site in HS-40 enhancer in erythroid cells.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_017',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 112}]}],\n",
       "   'context': 'Heme regulates gene expression by triggering Crm1-dependent nuclear export of Bach1. Bach1 is a transcriptional repressor of heme oxygenase-1 and beta-globin genes, both of which are known to be transcriptionally induced by heme. To test the hypothesis that heme regulates the activity of Bach1, we expressed wild type and mutated versions of Bach1 together with or without its heterodimer partner MafK in human 293T and GM02063 cells and examined their subcellular localization. Inhibition of heme synthesis enhanced the nuclear accumulation of Bach1, whereas treating cells with hemin resulted in nuclear exclusion of Bach1. While the cadmium-inducible nuclear export signal (NES) of Bach1 was dispensable for the heme response, a region containing two of the heme-binding motifs was found to be critical for the heme-induced nuclear exclusion. This region functioned as a heme-regulated NES dependent on the exporter Crm1. These results extend the regulatory roles for heme in protein sorting, and suggest that Bach1 transduces metabolic activity into gene expression.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_018',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 1063}]}],\n",
       "   'context': 'The heme oxygenase dilemma in cellular homeostasis: new insights for the feedback regulation of heme catabolism. Heme must be synthesized and degraded within an individual nucleated cell. Heme degradation is catalyzed by the two isozymes of heme oxygenase, heme oxygenase-1 (HO-1) and HO-2, eventually yielding biliverdin/bilirubin, CO, and iron. These products possess important physiological roles but are potentially toxic to cells. Characteristically, human HO-1 contains no Cys residues, whereas HO-2 contains the potential heme-binding motifs of the Cys-Pro dipeptide. Expression of HO-1 is inducible or repressible, depending on cell types or cellular microenvironments, but expression levels of HO-2 are fairly constant. Thus, the main regulation of heme catabolism is a problem of the balance between induction and repression of HO-1. Notably, HO-1 expression is induced by heme in all mammalian cells examined, but is repressed by hypoxia in certain types of cultured human cells. The recent discovery of Bach1 as a heme-regulated and hypoxia-inducible repressor for transcription of the HO-1 gene has provided a missing link in the feedback control of heme catabolism. On the other hand, the human HO-1 gene promoter contains the (GT)n repeat polymorphism and a single nucleotide polymorphism (-427A --> T), both of which may contribute to fine-tuning of the transcription. Importantly, long (GT)n alleles are associated with susceptibility to smoking-induced emphysema or coronary artery disease, but may provide with resistance to cerebral malaria. The latter finding suggests a novel therapeutic strategy with inhibitors of HO-1 for the treatment of cerebral malaria. We discuss the potential regulatory role of Bach1 and HO-2 in heme catabolism and update the understanding of the regulation of HO-1 expression.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_019',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 59}]}],\n",
       "   'context': 'Cadmium induces nuclear export of Bach1, a transcriptional repressor of heme oxygenase-1 gene. The export of certain nuclear proteins is involved in the regulation of various nuclear functions, including transcription. In some cases, the export of target proteins is induced upon environmental or cellular cues, resulting in conditional gene expression. The small Maf proteins appear to be critical regulators of heme oxygenase (HO)-1, an anti-oxidant defense enzyme that degrades heme into iron, carbon monoxide, and biliverdin. Although ho-1 is repressed by Bach1/small Maf heterodimers, it is activated by Nrf2/small Maf heterodimers, indicating that Bach1 and Nrf2 compete with each other. We anticipated that the nuclear concentration of Bach1 might be regulated to ensure that the entire system effectively responds to various stimuli. We carried out detailed domain analysis of Bach1 in an effort to understand how various inducers of HO-1 inactivate Bach1. We show here that cadmium, a strong inducer of HO-1, activates the nuclear export of Bach1. This cadmium-induced export of Bach1 was mediated in trans by its C-terminal region that is conserved between Bach1 and Bach2. The nuclear export of Bach2 was also induced by cadmium, indicating that the cadmium responsibility is shared between Bach1 and Bach2. The nuclear export of Bach1 was dependent on Crm1/Exportin-1 as well as the extracellular signal-regulated kinase-1/2 (ERK1/2) activity. These results indicate that the nuclear export of Bach1 constitutes an important regulatory mechanism to relieve the Bach1-mediated repression of genes such as ho-1.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_020',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 705}]}],\n",
       "   'context': 'Bach1 functions as a hypoxia-inducible repressor for the heme oxygenase-1 gene in human cells. Heme oxygenase 1 (HO-1) catalyzes heme breakdown, eventually releasing iron, carbon monoxide, and bilirubin IXalpha. HO-1 is induced by its substrate heme and various environmental factors, which represents a protective response against oxidative stresses. Here we show that hypoxia represses HO-1 expression in three human cell types but induces it in rat, bovine, and monkey cells, indicating the inter-species difference in the hypoxic regulation of HO-1 expression. The hypoxia-mediated repression of HO-1 expression is consistently associated with the induction of Bach1, a heme-regulated transcriptional repressor, in human cells. Bach1 is a basic leucine zipper protein, forming a heterodimer with a small Maf protein. Expression of HO-1 was also reduced in human cells when exposed to interferon-gamma or an iron chelator desferrioxamine, each of which induced Bach1 expression. In contrast, induction of HO-1 expression by CoCl(2) is associated with reduced expression of Bach1 mRNA. Thus, expression of HO-1 and Bach1 is inversely regulated. We have identified a Maf recognition element in the human HO-1 gene that is required for repression of a reporter gene by hypoxia and targeted by Bach1. Therefore, Bach1 functions as a hypoxia-inducible repressor for the HO-1 gene, thereby contributing to fine-tuning of oxygen homeostasis in human cells.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_021',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 372}]}],\n",
       "   'context': 'Hemoprotein Bach1 regulates enhancer availability of heme oxygenase-1 gene. Heme oxygenase-1 (HO-1) protects cells from various insults including oxidative stress. Transcriptional activators, including the Nrf2/Maf heterodimer, have been the focus of studies on the inducible expression of ho-1. Here we show that a heme-binding factor, Bach1, is a critical physiological repressor of ho-1. Bach1 bound to the multiple Maf recognition elements (MAREs) of ho-1 enhancers with MafK in vitro and repressed their activity in vivo, while heme abrogated this repressor function of Bach1 by inhibiting its binding to the ho-1 enhancers. Gene targeting experiments in mice revealed that, in the absence of Bach1, ho-1 became expressed constitutively at high levels in various tissues under normal physiological conditions. By analyzing bach1/nrf2 compound-deficient mice, we documented antagonistic activities of Bach1 and Nrf2 in several tissues. Chromatin immunoprecipitation revealed that small Maf proteins participate in both repression and activation of ho-1. Thus, regulation of ho-1 involves a direct sensing of heme levels by Bach1 (by analogy to lac repressor sensitivity to lactose), generating a simple feedback loop whereby the substrate effects repressor-activator antagonism.'},\n",
       "  {'qas': [{'id': '52fa6ac72059c6d71c000055_022',\n",
       "     'question': 'Is the transcriptional regulator BACH1 an activator or a repressor?',\n",
       "     'answers': [{'text': 'repressor', 'answer_start': 94}]}],\n",
       "   'context': 'Heme mediates derepression of Maf recognition element through direct binding to transcription repressor Bach1. Heme controls expression of genes involved in the synthesis of globins and heme. The mammalian transcription factor Bach1 functions as a repressor of the Maf recognition element (MARE) by forming antagonizing hetero-oligomers with the small Maf family proteins. We show here that heme binds specifically to Bach1 and regulates its DNA-binding activity. Deletion studies demonstrated that a heme-binding region of Bach1 is confined within its C-terminal region that possesses four dipeptide cysteine-proline (CP) motifs. Mutations in all of the CP motifs of Bach1 abolished its interaction with heme. The DNA-binding activity of Bach1 as a MafK hetero-oligomer was markedly inhibited by heme in gel mobility shift assays. The repressor activity of Bach1 was lost upon addition of hemin in transfected cells. These results suggest that increased levels of heme inactivate the repressor Bach1, resulting in induction of a host of genes with MARES:'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_001',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 19}]}],\n",
       "   'context': 'Suvorexant: a dual orexin receptor antagonist for the treatment of sleep onset and sleep maintenance insomnia. OBJECTIVE: To review the efficacy, safety, and pharmacology data available for suvorexant and determine its role in therapy as compared with other agents available for the treatment of insomnia. DATA SOURCES: A PubMed search using the terms suvorexant and MK-4305 (the original name given to suvorexant during early trials) was conducted in December 2014 to identify initial literature sources. No time frame was used for exclusion of older trials. STUDY SELECTION AND DATA EXTRACTION: Animal studies and trials written in a language other than English were excluded. Abstracts of the remaining trials were evaluated for determination of relevance to this review. References from these studies along with suvorexant prescriber information were used to identify additional literature. DATA SYNTHESIS: Three randomized, double-blind, placebo-controlled clinical trials were identified showing suvorexant to be safe, effective, and tolerable for the treatment of insomnia. After 4 weeks of therapy, relative to placebo, the 10- and 20-mg doses improved subjective total sleep time (22.3 and 49.9 minutes, respectively), wake after sleep onset (-21.4 and -28.1 minutes), and latency to persistent sleep (-2.3 and -22.3 minutes). CONCLUSION: Suvorexant is the first dual orexin receptor antagonist approved for the treatment of insomnia. Clinical trials have shown that it is relatively safe and effective for the treatment of both sleep onset and sleep maintenance at doses of 20 mg or less. Higher doses were studied but not approved because of concerns for next-day somnolence and effects on driving. Further studies are needed to assess this medication in patients with a history of addiction, because they were excluded from clinical trials, as well as to compare suvorexant with other insomnia medications available because no head-to-head studies have yet been conducted. However, its novel mechanism of action and theoretically lower addiction liability make suvorexant an appealing new option.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_002',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 1377}]}],\n",
       "   'context': 'Suvorexant: a dual orexin receptor antagonist for the treatment of sleep onset and sleep maintenance insomnia. OBJECTIVE: To review the efficacy, safety, and pharmacology data available for suvorexant and determine its role in therapy as compared with other agents available for the treatment of insomnia. DATA SOURCES: A PubMed search using the terms suvorexant and MK-4305 (the original name given to suvorexant during early trials) was conducted in December 2014 to identify initial literature sources. No time frame was used for exclusion of older trials. STUDY SELECTION AND DATA EXTRACTION: Animal studies and trials written in a language other than English were excluded. Abstracts of the remaining trials were evaluated for determination of relevance to this review. References from these studies along with suvorexant prescriber information were used to identify additional literature. DATA SYNTHESIS: Three randomized, double-blind, placebo-controlled clinical trials were identified showing suvorexant to be safe, effective, and tolerable for the treatment of insomnia. After 4 weeks of therapy, relative to placebo, the 10- and 20-mg doses improved subjective total sleep time (22.3 and 49.9 minutes, respectively), wake after sleep onset (-21.4 and -28.1 minutes), and latency to persistent sleep (-2.3 and -22.3 minutes). CONCLUSION: Suvorexant is the first dual orexin receptor antagonist approved for the treatment of insomnia. Clinical trials have shown that it is relatively safe and effective for the treatment of both sleep onset and sleep maintenance at doses of 20 mg or less. Higher doses were studied but not approved because of concerns for next-day somnolence and effects on driving. Further studies are needed to assess this medication in patients with a history of addiction, because they were excluded from clinical trials, as well as to compare suvorexant with other insomnia medications available because no head-to-head studies have yet been conducted. However, its novel mechanism of action and theoretically lower addiction liability make suvorexant an appealing new option.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_003',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 35}]}],\n",
       "   'context': 'Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant. The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) respond to orexin neuropeptides in the central nervous system to regulate sleep and other behavioural functions in humans. Defects in orexin signalling are responsible for the human diseases of narcolepsy and cataplexy; inhibition of orexin receptors is an effective therapy for insomnia. The human OX2 receptor (OX2R) belongs to the β branch of the rhodopsin family of GPCRs, and can bind to diverse compounds including the native agonist peptides orexin-A and orexin-B and the potent therapeutic inhibitor suvorexant. Here, using lipid-mediated crystallization and protein engineering with a novel fusion chimaera, we solved the structure of the human OX2R bound to suvorexant at 2.5\\xa0Å resolution. The structure reveals how suvorexant adopts a π-stacked horseshoe-like conformation and binds to the receptor deep in the orthosteric pocket, stabilizing a network of extracellular salt bridges and blocking transmembrane helix motions necessary for activation. Computational docking suggests how other classes of synthetic antagonists may interact with the receptor at a similar position in an analogous π-stacked fashion. Elucidation of the molecular architecture of the human OX2R expands our understanding of peptidergic GPCR ligand recognition and will aid further efforts to modulate orexin signalling for therapeutic ends.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_004',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 613}]}],\n",
       "   'context': 'Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant. The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) respond to orexin neuropeptides in the central nervous system to regulate sleep and other behavioural functions in humans. Defects in orexin signalling are responsible for the human diseases of narcolepsy and cataplexy; inhibition of orexin receptors is an effective therapy for insomnia. The human OX2 receptor (OX2R) belongs to the β branch of the rhodopsin family of GPCRs, and can bind to diverse compounds including the native agonist peptides orexin-A and orexin-B and the potent therapeutic inhibitor suvorexant. Here, using lipid-mediated crystallization and protein engineering with a novel fusion chimaera, we solved the structure of the human OX2R bound to suvorexant at 2.5\\xa0Å resolution. The structure reveals how suvorexant adopts a π-stacked horseshoe-like conformation and binds to the receptor deep in the orthosteric pocket, stabilizing a network of extracellular salt bridges and blocking transmembrane helix motions necessary for activation. Computational docking suggests how other classes of synthetic antagonists may interact with the receptor at a similar position in an analogous π-stacked fashion. Elucidation of the molecular architecture of the human OX2R expands our understanding of peptidergic GPCR ligand recognition and will aid further efforts to modulate orexin signalling for therapeutic ends.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_005',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 626}]}],\n",
       "   'context': 'Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant. The orexin (also known as hypocretin) G protein-coupled receptors (GPCRs) respond to orexin neuropeptides in the central nervous system to regulate sleep and other behavioural functions in humans. Defects in orexin signalling are responsible for the human diseases of narcolepsy and cataplexy; inhibition of orexin receptors is an effective therapy for insomnia. The human OX2 receptor (OX2R) belongs to the β branch of the rhodopsin family of GPCRs, and can bind to diverse compounds including the native agonist peptides orexin-A and orexin-B and the potent therapeutic inhibitor suvorexant. Here, using lipid-mediated crystallization and protein engineering with a novel fusion chimaera, we solved the structure of the human OX2R bound to suvorexant at 2.5\\xa0Å resolution. The structure reveals how suvorexant adopts a π-stacked horseshoe-like conformation and binds to the receptor deep in the orthosteric pocket, stabilizing a network of extracellular salt bridges and blocking transmembrane helix motions necessary for activation. Computational docking suggests how other classes of synthetic antagonists may interact with the receptor at a similar position in an analogous π-stacked fashion. Elucidation of the molecular architecture of the human OX2R expands our understanding of peptidergic GPCR ligand recognition and will aid further efforts to modulate orexin signalling for therapeutic ends.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_006',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'Orexin', 'answer_start': 0}]}],\n",
       "   'context': 'Orexin receptor antagonism for treatment of insomnia: a randomized clinical trial of suvorexant. OBJECTIVE: To assess the utility of orexin receptor antagonism as a novel approach to treating insomnia. METHODS: We evaluated suvorexant, an orexin receptor antagonist, for treating patients with primary insomnia in a randomized, double-blind, placebo-controlled, 2-period (4 weeks per period) crossover polysomnography study. Patients received suvorexant (10 mg [n = 62], 20 mg [n = 61], 40 mg [n = 59], or 80 mg [n = 61]) in one period and placebo (n = 249) in the other. Polysomnography was performed on night 1 and at the end of week 4 of each period. The coprimary efficacy end points were sleep efficiency on night 1 and end of week 4. Secondary end points were wake after sleep onset and latency to persistent sleep. RESULTS: Suvorexant showed significant (p values <0.01) dose-related improvements vs placebo on the coprimary end points of sleep efficiency at night 1 and end of week 4. Dose-related effects were also observed for sleep induction (latency to persistent sleep) and maintenance (wake after sleep onset). Suvorexant was generally well tolerated. CONCLUSIONS: The data suggest that orexin receptor antagonism offers a novel approach to treating insomnia. CLASSIFICATION OF EVIDENCE: This study provides Class I evidence that suvorexant improves sleep efficiency over 4 weeks in nonelderly adult patients with primary insomnia.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_007',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 46}]}],\n",
       "   'context': 'Promotion of sleep by suvorexant-a novel dual orexin receptor antagonist. Orexins/hypocretins are key neuropeptides responsible for regulating central arousal and reward circuits. Two receptors respond to orexin signaling, orexin 1 receptor (OX(1)R) and orexin 2 receptor (OX(2)R) with partially overlapping nervous system distributions. Genetic studies suggest orexin receptor antagonists could be therapeutic for insomnia and other disorders with disruptions of sleep and wake. Suvorexant (MK-4305) is a potent, selective, and orally bioavailable antagonist of OX(1)R and OX(2)R currently under clinical investigation as a novel therapy for insomnia. Examination of Suvorexant in radioligand binding assays using tissue from transgenic rats expressing the human OX(2)R found nearly full receptor occupancy (>90%) at plasma exposures of 1.1 μM. Dosed orally Suvorexant significantly and dose-dependently reduced locomotor activity and promoted sleep in rats (10, 30, and 100 mg/kg), dogs (1 and 3 mg/kg), and rhesus monkeys (10 mg/kg). Consistent cross-species sleep/wake architecture changes produced by Suvorexant highlight a unique opportunity to develop dual orexin antagonists as a novel therapy for insomnia.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_008',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 958}]}],\n",
       "   'context': 'CNS drug development: lessons from the development of ondansetron, aprepitant, ramelteon, varenicline, lorcaserin, and suvorexant. Part I. This column is the first in a two-part series exploring lessons for psychiatric drug development that can be learned from the development of six central nervous system drugs with novel mechanisms of action over the past 25 years. Part 1 presents a brief overview of the neuroscience that supported the development of each drug, including the rationale for selecting a) the target, which in each case was a receptor for a specific neurotransmitter system, and b) the indication, which was based on an understanding of the role that target played in a specific neural circuit in the brain. The neurotransmitter systems on which the development of these agents were based included serotonin for ondansetron and lorcaserin, dopamine for varenicline, substance P (or neurokinin) for aprepitant, melatonin for ramelteon, and orexin for suvorexant. The indications were chemotherapy-induced nausea and vomiting for ondansetron and aprepitant, smoking cessation for varenicline, weight loss for lorcaserin, and insomnia for suvorexant and ramelteon.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_009',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 19}]}],\n",
       "   'context': 'Suvorexant, a dual orexin receptor antagonist for the management of insomnia. Suvorexant, a dual orexin receptor antagonist for the management of insomnia.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_010',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 19}]}],\n",
       "   'context': 'Suvorexant, a dual orexin receptor antagonist for the management of insomnia. Suvorexant, a dual orexin receptor antagonist for the management of insomnia.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_011',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 56}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_012',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 69}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_013',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 152}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_014',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 165}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_015',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 304}]}],\n",
       "   'context': 'Suvorexant: something new for sleep? Orexin, also called hypocretin, is a neuropeptide that acts on central nervous system receptors to promote arousal. Suvorexant, its receptor antagonist, generates interest as a medication to treat insomnia. Suvorexant helps in decreasing wakefulness by counteracting orexin activity. Its low side effect potential may offer considerable benefit. Compared with other sleep aids, diminished drowsiness and less cognitive dysfunction is an advantage. Now approved for clinical use, an apparent lack of rebound insomnia or drug dependence potential might make suvorexant a good choice pharmacotherapy for patients with insomnia.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_016',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 56}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_017',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 69}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_018',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 152}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  {'qas': [{'id': '56c1f003ef6e394741000039_019',\n",
       "     'question': 'What molecule is targeted by suvorexant?',\n",
       "     'answers': [{'text': 'orexin', 'answer_start': 165}]}],\n",
       "   'context': 'Small-molecule antagonists of the orexin receptors. The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved. The scope of this article is to review the small molecule orexin receptor antagonist patent literature between January 2012 and January 2014.'},\n",
       "  ...],\n",
       " 'title': 'BioASQ6b'}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df['data'][0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
